{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "from key import KEY as OPENAI_API_KEY\n",
    "from key import SS_KEY as SS_API_KEY\n",
    "from urllib.request import Request, urlopen\n",
    "\n",
    "import json\n",
    "import os\n",
    "import pickle\n",
    "import time\n",
    "import requests\n",
    "from bs4 import BeautifulSoup"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# With Semantic Scholar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Directly define the API key (Reminder: Securely handle API keys in production environments)\n",
    "api_key = SS_API_KEY\n",
    "\n",
    "# Define headers with API key\n",
    "headers = {'x-api-key': api_key}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_paper_data(paper_id):\n",
    "  url = 'https://api.semanticscholar.org/graph/v1/paper/' + paper_id\n",
    "\n",
    "  # Define which details about the paper you would like to receive in the response\n",
    "  paper_data_query_params = {'fields': 'title,citationCount'}\n",
    "\n",
    "  # Send the API request and store the response in a variable\n",
    "  response = requests.get(url, params=paper_data_query_params)\n",
    "  if response.status_code == 200:\n",
    "    return response.json()\n",
    "  else:\n",
    "    return None\n",
    "\n",
    "def download_file(url, filename):\n",
    "  try:\n",
    "    response = requests.get(url=url, headers={'User-Agent': 'Mozilla/5.0'})\n",
    "\n",
    "    with open(filename, 'wb') as file:\n",
    "      file.write(response.content)\n",
    "    print(f\"File downloaded successfully: {filename}\")\n",
    "  \n",
    "  except requests.exceptions.RequestException as e:\n",
    "    print(f\"Error occurred: {e}\")\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'paperId': 'e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e', 'title': 'Large-scale simultaneous measurement of epitopes and transcriptomes in single cells', 'citationCount': 1713}\n"
     ]
    }
   ],
   "source": [
    "# Define the API endpoint URL\n",
    "url = 'https://api.semanticscholar.org/graph/v1/paper/search'\n",
    "\n",
    "# More specific query parameter\n",
    "query_params = {'query': 'Large-scale simultaneous measurement of epitopes and transcriptomes in single cells', }\n",
    "\n",
    "# Send the API request\n",
    "search_response = requests.get(url, params=query_params, headers=headers)\n",
    "\n",
    "# Check if the request was successful (status code 200)\n",
    "if search_response.status_code == 200:\n",
    "  search_response = search_response.json()\n",
    "\n",
    "  # Retrieve the paper id corresponding to the 1st result in the list\n",
    "  paper_id = search_response['data'][0]['paperId']\n",
    "\n",
    "  # Retrieve the paper details corresponding to this paper id using the function we defined earlier.\n",
    "  paper_details = get_paper_data(paper_id)\n",
    "  print(paper_details)\n",
    "else:\n",
    "  # Handle potential errors or non-200 responses\n",
    "  print(f\"Relevance Search Request failed with status code {search_response.status_code}: {search_response.text}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define the directory name\n",
    "out_dir = 'citing_paper_abstracts'\n",
    "\n",
    "# Create the directory if it doesn't exist\n",
    "if not os.path.exists(out_dir):\n",
    "    os.makedirs(out_dir)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=0&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a83dbce941baa359f9c950fa30e353ef39f9d564\",\n",
      "        \"title\": \"Single-cell genomics-based immune and disease monitoring in blood malignancies\",\n",
      "        \"abstract\": \"Achieving long-term disease control using therapeutic immunomodulation is a long-standing concept with a strong tradition in blood malignancies. Besides allogeneic hematopoietic stem cell transplantation that continues to provide potentially curative treatment for otherwise challenging diagnoses, recent years have seen impressive progress in immunotherapies for leukemias and lymphomas with immune checkpoint blockade, bispecific monoclonal antibodies, and CAR T cell therapies. Despite their success, non-response, relapse, and immune toxicities remain frequent, thus prioritizing the elucidation of the underlying mechanisms and identifying predictive biomarkers. The increasing availability of single-cell genomic tools now provides a system\\u2019s immunology view to resolve the molecular and cellular mechanisms of immunotherapies at unprecedented resolution. Here, we review recent studies that leverage these technological advancements for tracking immune responses, the emergence of immune resistance, and toxicities. As single-cell immune monitoring tools evolve and become more accessible, we expect their wide adoption for routine clinical applications to catalyze more precise therapeutic steering of personal immune responses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"116e8815e3f7217557ce7a5fabd86cab6afc311c\",\n",
      "        \"title\": \"CAraCAl: CAMML with the integration of chromatin accessibility\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=2&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"509301a2aefb1f4d9cd171e0ca8a1251630c0c09\",\n",
      "        \"title\": \"A human STAT3 gain-of-function variant drives local Th17 dysregulation and skin inflammation in mice.\",\n",
      "        \"abstract\": \"Germline gain-of-function (GOF) variants in STAT3 cause an inborn error of immunity associated with early-onset poly-autoimmunity and immune dysregulation. To study tissue-specific immune dysregulation, we used a mouse model carrying a missense variant (p.G421R) that causes human disease. We observed spontaneous and imiquimod (IMQ)-induced skin inflammation associated with cell-intrinsic local Th17 responses in STAT3 GOF mice. CD4+ T cells were sufficient to drive skin inflammation and showed increased Il22 expression in expanded clones. Certain aspects of disease, including increased epidermal thickness, also required the presence of STAT3 GOF in epithelial cells. Treatment with a JAK inhibitor improved skin disease without affecting local Th17 recruitment and cytokine production. These findings collectively support the involvement of Th17 responses in the development of organ-specific immune dysregulation in STAT3 GOF and suggest that the presence of STAT3 GOF in tissues is important for disease and can be targeted with JAK inhibition.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=3&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f909ca816dd7449e80641496a664533433253687\",\n",
      "        \"title\": \"Machine learning integrative approaches to advance computational immunology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=4&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ff57e84ecbbd35d561ffdcf30e77f2254ff81e1\",\n",
      "        \"title\": \"Application of omics in the diagnosis, prognosis, and treatment of acute myeloid leukemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=5&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4dd15f5fe35207f3835a17161c2591f154b337be\",\n",
      "        \"title\": \"Revisiting Cardiac Biology in the Era of Single Cell and Spatial Omics\",\n",
      "        \"abstract\": \"Throughout our lifetime, each beat of the heart requires the coordinated action of multiple cardiac cell types. Understanding cardiac cell biology, its intricate microenvironments, and the mechanisms that govern their function in health and disease are crucial to designing novel therapeutical and behavioral interventions. Recent advances in single-cell and spatial omics technologies have significantly propelled this understanding, offering novel insights into the cellular diversity and function and the complex interactions of cardiac tissue. This review provides a comprehensive overview of the cellular landscape of the heart, bridging the gap between suspension-based and emerging in situ approaches, focusing on the experimental and computational challenges, comparative analyses of mouse and human cardiac systems, and the rising contextualization of cardiac cells within their niches. As we explore the heart at this unprecedented resolution, integrating insights from both mouse and human studies will pave the way for novel diagnostic tools and therapeutic interventions, ultimately improving outcomes for patients with cardiovascular diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=6&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8464769ca50f6ca1c5901860e5083f42b15dfc80\",\n",
      "        \"title\": \"Lymphatic vessel transit seeds cytotoxic resident memory T cells in skin draining lymph nodes\",\n",
      "        \"abstract\": \"Lymphatic transport shapes the homeostatic immune repertoire of lymph nodes (LNs). LN-resident memory T cells (TRMs) play an important role in site-specific immune memory, yet how LN TRMs form de novo after viral infection remains unclear. Here, we tracked the anatomical distribution of antiviral CD8+ T cells as they seeded skin and LN TRMs using a model of vaccinia virus\\u2013induced skin infection. LN TRMs localized to the draining LNs (dLNs) of infected skin, and their formation depended on the lymphatic egress of effector CD8+ T cells from the skin, already poised for residence. Effector CD8+ T cell transit through skin was required to populate LN TRMs in dLNs, a process reinforced by antigen encounter in skin. Furthermore, LN TRMs were protective against viral rechallenge in the absence of circulating memory T cells. These data suggest that a subset of tissue-infiltrating CD8+ T cells egress from tissues during viral clearance and establish a layer of regional protection in the dLN basin. T cells egress out of virally infected skin via afferent lymphatic vessels and form protective, long-lived lymph node\\u2013resident memory. Editor\\u2019s summary CD8+ resident memory T cells (TRMs) play an important role in host protection after antigen reencounter. TRMs are often retained in nonlymphoid tissues but can also reside in draining lymph nodes (dLNs). Heim et al. investigate how TRMs are established in dLNs using a mouse model of localized viral skin infection. Egress of antigen experienced CD8+ effector T cells through infected skin and transport via the dermal lymphatic vessels was required for the establishment of TRMs in the dLN. TRMs were poised for cytotoxicity and provided antigen-specific protection in the dLNs after viral rechallenge. These findings highlight the importance of CD8+ T cell tissue egress and lymphatic transport in the establishment of dLN TRMs. \\u2014Hannah Isles\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=7&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1df6a4cf543a0560450286d3f4b92862850706f3\",\n",
      "        \"title\": \"A comprehensive exploration of schistosomiasis: Global impact, molecular characterization, drug discovery, artificial intelligence and future prospects\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=8&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"74cab8838c66fcb02867306feabbfcea61989a8e\",\n",
      "        \"title\": \"The chromatin landscape of pathogenic transcriptional cell states in rheumatoid arthritis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=9&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0f85eb398e8b54d5e458d4eefdac28452536073d\",\n",
      "        \"title\": \"Pleiotropy, a feature or a bug? Toward co-ordinating plant growth, development, and environmental responses through engineering plant hormone signaling.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=10&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"480567bb55e580af5f22e324481c693a5881a9b8\",\n",
      "        \"title\": \"Unveiling gene regulatory networks during cellular state transitions without linkage across time points\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=11&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"19693decc494525f37afd96f053a86ae31041a37\",\n",
      "        \"title\": \"Multi-modal domain adaptation for revealing spatial functional landscape from spatially resolved transcriptomics\",\n",
      "        \"abstract\": \"Abstract Spatially resolved transcriptomics (SRT) has emerged as a powerful tool for investigating gene expression in spatial contexts, providing insights into the molecular mechanisms underlying organ development and disease pathology. However, the expression sparsity poses a computational challenge to integrate other modalities (e.g. histological images and spatial locations) that are simultaneously captured in SRT datasets for spatial clustering and variation analyses. In this study, to meet such a challenge, we propose multi-modal domain adaption for spatial transcriptomics (stMDA), a novel multi-modal unsupervised domain adaptation method, which integrates gene expression and other modalities to reveal the spatial functional landscape. Specifically, stMDA first learns the modality-specific representations from spatial multi-modal data using multiple neural network architectures and then aligns the spatial distributions across modal representations to integrate these multi-modal representations, thus facilitating the integration of global and spatially local information and improving the consistency of clustering assignments. Our results demonstrate that stMDA outperforms existing methods in identifying spatial domains across diverse platforms and species. Furthermore, stMDA excels in identifying spatially variable genes with high prognostic potential in cancer tissues. In conclusion, stMDA as a new tool of multi-modal data integration provides a powerful and flexible framework for analyzing SRT datasets, thereby advancing our understanding of intricate biological systems.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=12&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5c40535c9050ce3c82ac47b66f526ad6e897a633\",\n",
      "        \"title\": \"A hybrid demultiplexing strategy that improves performance and robustness of cell hashing\",\n",
      "        \"abstract\": \"Abstract Cell hashing, a nucleotide barcode-based method that allows users to pool multiple samples and demultiplex in downstream analysis, has gained widespread popularity in single-cell sequencing due to its compatibility, simplicity, and cost-effectiveness. Despite these advantages, the performance of this method remains unsatisfactory under certain circumstances, especially in experiments that have imbalanced sample sizes or use many hashtag antibodies. Here, we introduce a hybrid demultiplexing strategy that increases accuracy and cell recovery in multi-sample single-cell experiments. This approach correlates the results of cell hashing and genetic variant clustering, enabling precise and efficient cell identity determination without additional experimental costs or efforts. In addition, we developed HTOreader, a demultiplexing tool for cell hashing that improves the accuracy of cut-off calling by avoiding the dominance of negative signals in experiments with many hashtags or imbalanced sample sizes. When compared to existing methods using real-world datasets, this hybrid approach and HTOreader consistently generate reliable results with increased accuracy and cell recovery.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=13&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7781b34bb88cf7e1ff362551c83be38cae406e93\",\n",
      "        \"title\": \"DMF-scMT-seq linking methylome and transcriptome within single cells with digital microfluidics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=14&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"87c1dd8275ef4f7c5d44a91d710e521a862947d6\",\n",
      "        \"title\": \"Microfluidic Biochips for Single-Cell Isolation and Single-Cell Analysis of Multiomics and Exosomes.\",\n",
      "        \"abstract\": \"Single-cell multiomic and exosome analyses are potent tools in various fields, such as cancer research, immunology, neuroscience, microbiology, and drug development. They facilitate the in-depth exploration of biological systems, providing insights into disease mechanisms and aiding in treatment. Single-cell isolation, which is crucial for single-cell analysis, ensures reliable cell isolation and quality control for further downstream analyses. Microfluidic chips are small lightweight systems that facilitate efficient and high-throughput single-cell isolation and real-time single-cell analysis on- or off-chip. Therefore, most current single-cell isolation and analysis technologies are based on the single-cell microfluidic technology. This review offers comprehensive guidance to researchers across different fields on the selection of appropriate microfluidic chip technologies for single-cell isolation and analysis. This review describes the design principles, separation mechanisms, chip characteristics, and cellular effects of various microfluidic chips available for single-cell isolation. Moreover, this review highlights the implications of using this technology for subsequent analyses, including single-cell multiomic and exosome analyses. Finally, the current challenges and future prospects of microfluidic chip technology are outlined for multiplex single-cell isolation and multiomic and exosome analyses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=15&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d8d82f80881014fe86e4009139138d6bbe50d4af\",\n",
      "        \"title\": \"High-throughput mechanical phenotyping and transcriptomics of single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=16&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"355b16246c494eaf22edcc6b83823cfec65e569a\",\n",
      "        \"title\": \"Protocol to perform integrative analysis of high-dimensional single-cell multimodal data using an interpretable deep learning technique\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=17&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"437d8fb2ed5fef16b519a03021e97096821bc25f\",\n",
      "        \"title\": \"Commonly used software tools produce conflicting and overly-optimistic AUPRC values\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=18&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5db920e880a4014581a488bdf1bc4dfd44c5ecc0\",\n",
      "        \"title\": \"Single\\u2010cell proteomics by mass spectrometry: Advances and implications in cancer research\",\n",
      "        \"abstract\": \"Cancer harbours extensive proteomic heterogeneity. Inspired by the prior success of single\\u2010cell RNA sequencing (scRNA\\u2010seq) in characterizing minute transcriptomics heterogeneity in cancer, researchers are now actively searching for information regarding the proteomics counterpart. Therefore recently, single\\u2010cell proteomics by mass spectrometry (SCP) has rapidly developed into state\\u2010of\\u2010the\\u2010art technology to cater the need. This review aims to summarize application of SCP in cancer research, while revealing current development progress of SCP technology. The review also aims to contribute ideas into research gaps and future directions, ultimately promoting the application of SCP in cancer research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=19&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"db435c055031c0e7de5fe7b0291e5e113197df8a\",\n",
      "        \"title\": \"Single-cell transcriptomic analyses reveal distinct immune cell contributions to epithelial barrier dysfunction in checkpoint inhibitor colitis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=20&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ffe7cf06c25fa23f822a7ec22cb2708b47ed6529\",\n",
      "        \"title\": \"A review of single-cell transcriptomics and epigenomics studies in maternal and child health.\",\n",
      "        \"abstract\": \"Single-cell sequencing technologies enhance our understanding of cellular dynamics throughout pregnancy. We outlined the workflow of single-cell sequencing techniques and reviewed single-cell studies in maternal and child health. We conducted a literature review of single cell studies on maternal and child health using PubMed. We summarized the findings from 16 single-cell atlases of the human and mammalian placenta across gestational stages and 31 single-cell studies on maternal exposures and complications including infection, obesity, diet, gestational diabetes, pre-eclampsia, environmental exposure and preterm birth. Single-cell studies provides insights on novel cell types in placenta and cell type-specific marks associated with maternal exposures and complications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=21&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"48629c8d7eca851d9ce0efe4143ca443bc7200d5\",\n",
      "        \"title\": \"The future of rapid and automated single-cell data analysis using reference mapping\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=22&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"24aa7c05d7ca2982d87b74d1e91554342f8b0253\",\n",
      "        \"title\": \"Clinical and CSF single-cell profiling of post-COVID-19 cognitive impairment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=23&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b704a243e8b850d45ac1caf5f1d85083d6a9087c\",\n",
      "        \"title\": \"scDM: A deep generative method for cell surface protein prediction with diffusion model.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=24&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eca98ed5a503c7237317f28e9afa67d36d84ffb1\",\n",
      "        \"title\": \"Single Cell Multiomics Systems Biology for Molecular Toxicity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=25&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2ef89f091f7e64a0d10da4c061e87edb2ac480dd\",\n",
      "        \"title\": \"Integrating population and single-cell variations in vaccine responses identifies a naturally adjuvanted human immune setpoint.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=26&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1618d2ad5a2dd0ea10bb34600f1e40cfde4fd0fc\",\n",
      "        \"title\": \"Skin in the game: a review of single-cell and spatial transcriptomics in dermatological research\",\n",
      "        \"abstract\": \"Abstract Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) are two emerging research technologies that uniquely characterize gene expression microenvironments on a cellular or subcellular level. The skin, a clinically accessible tissue composed of diverse, essential cell populations, serves as an ideal target for these high-resolution investigative approaches. Using these tools, researchers are assembling a compendium of data and discoveries in healthy skin as well as a range of dermatologic pathophysiologies, including atopic dermatitis, psoriasis, and cutaneous malignancies. The ongoing advancement of single-cell approaches, coupled with anticipated decreases in cost with increased adoption, will reshape dermatologic research, profoundly influencing disease characterization, prognosis, and ultimately clinical practice.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=27&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"835158e91bc64c2d1574d9ac7ed9df9ae13b424a\",\n",
      "        \"title\": \"Spatial genomics: mapping human steatotic liver disease.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=28&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"993829cbd35c7a91a5a3e3856388fd2b6383c0fd\",\n",
      "        \"title\": \"TIM-3+ CD8 T cells with a terminally exhausted phenotype retain functional capacity in hematological malignancies\",\n",
      "        \"abstract\": \"Chronic antigen stimulation is thought to generate dysfunctional CD8 T cells. Here, we identify a CD8 T cell subset in the bone marrow tumor microenvironment that, despite an apparent terminally exhausted phenotype (TPHEX), expressed granzymes, perforin, and IFN-\\u03b3. Concurrent gene expression and DNA accessibility revealed that genes encoding these functional proteins correlated with BATF expression and motif accessibility. IFN-\\u03b3+ TPHEX effectively killed myeloma with comparable efficacy to transitory effectors, and disease progression correlated with numerical deficits in IFN-\\u03b3+ TPHEX. We also observed IFN-\\u03b3+ TPHEX within CD19-targeted chimeric antigen receptor T cells, which killed CD19+ leukemia cells. An IFN-\\u03b3+ TPHEX gene signature was recapitulated in TEX cells from human cancers, including myeloma and lymphoma. Here, we characterize a TEX subset in hematological malignancies that paradoxically retains function and is distinct from dysfunctional TEX found in chronic viral infections. Thus, IFN-\\u03b3+ TPHEX represent a potential target for immunotherapy of blood cancers. A subset of CD8 T cells with a terminally exhausted epigenetic profile expressed IFN-\\u03b3, granzymes, and perforin in bone marrow. Editor\\u2019s Summary Chronic antigen stimulation can drive CD8 T cell exhaustion and dysfunction among multiple T cell subsets. Using a murine myeloma model, Minnie et al. characterized a terminally exhausted CD8 T cell subset in the bone marrow tumor microenvironment that expressed not only transcription and epigenetic factors associated with exhaustion but also effector molecules, including IFN-\\u03b3, granzymes, and perforin. This subset, IFN-\\u03b3+ TPHEX (IFN-\\u03b3\\u2013secreting, phenotypically exhausted T cells), could kill myeloma cells, and IFN-\\u03b3+ TPHEX differentiated from CD19-targeted chimeric antigen receptor T cells could kill CD19+ tumor cells. An analogous IFN-\\u03b3+ TPHEX subset was also detected in patients with myeloma and lymphoma. Together, these findings suggest that IFN-\\u03b3+ TPHEX have dysfunctional features but retain functionality, facilitating antitumor responses. \\u2014Christiana Fogg\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=29&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"08ce9aebac485044abc75c7461cdf1ad07ed009e\",\n",
      "        \"title\": \"Unravelling B cell heterogeneity: insights into flow cytometry-gated B cells from single-cell multi-omics data\",\n",
      "        \"abstract\": \"Introduction B cells play a pivotal role in adaptive immunity which has been extensively characterised primarily via flow cytometry-based gating strategies. This study addresses the discrepancies between flow cytometry-defined B cell subsets and their high-confidence molecular signatures using single-cell multi-omics approaches. Methods By analysing multi-omics single-cell data from healthy individuals and patients across diseases, we characterised the level and nature of cellular contamination within standard flow cytometric-based gating, resolved some of the ambiguities in the literature surrounding unconventional B cell subsets, and demonstrated the variable effects of flow cytometric-based gating cellular heterogeneity across diseases. Results We showed that flow cytometric-defined B cell populations are heterogenous, and the composition varies significantly between disease states thus affecting the implications of functional studies performed on these populations. Importantly, this paper draws caution on findings about B cell selection and function of flow cytometric-sorted populations, and their roles in disease. As a solution, we developed a simple tool to identify additional markers that can be used to increase the purity of flow-cytometric gated immune cell populations based on multi-omics data (AlliGateR). Here, we demonstrate that additional non-linear CD20, CD21 and CD24 gating can increase the purity of both na\\u00efve and memory populations. Discussion These findings underscore the need to reconsider B cell subset definitions within the literature and propose leveraging single-cell multi-omics data for refined characterisation. We show that single-cell multi-omics technologies represent a powerful tool to bridge the gap between surface marker-based annotations and the intricate molecular characteristics of B cell subsets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=30&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"deaf8285c9024cd4667907e0834ab43c8740b6fd\",\n",
      "        \"title\": \"Next-Generation Sequencing-Based Spatial Transcriptomics: A Perspective from Barcoding Chemistry\",\n",
      "        \"abstract\": \"Gene expression profiling of tissue cells with spatial context is in high demand to reveal cell types, locations, and intercellular or molecular interactions for physiological and pathological studies. With rapid advances in barcoding chemistry and sequencing chemistry, spatially resolved transcriptome (SRT) techniques have emerged to quantify spatial gene expression in tissue samples by correlating transcripts with their spatial locations using diverse strategies. These techniques provide both physical tissue structure and molecular characteristics and are poised to revolutionize many fields, such as developmental biology, neuroscience, oncology, and histopathology. In this context, this Perspective focuses on next-generation sequencing-based SRT methods, particularly highlighting spatial barcoding chemistry. It delves into optically manipulated spatial indexing methods and DNA array-barcoded spatial indexing methods by exploring current advances, challenges, and future development directions in this nascent field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=31&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b930688cffe12e5aa750a2573fee2bffb90cc4a8\",\n",
      "        \"title\": \"Highly Multiplexing, Throughput and Efficient Single-Cell Protein Analysis with Digital Microfluidics.\",\n",
      "        \"abstract\": \"Proteins as crucial components of cells are responsible for the majority of cellular processes. Sensitive and efficient protein detection enables a more accurate and comprehensive investigation of cellular phenotypes and life activities. Here, a protein sequencing method with high multiplexing, high throughput, high cell utilization, and integration based on digital microfluidics (DMF-Protein-seq) is proposed, which transforms protein information into DNA sequencing readout via DNA-tagged antibodies and labels single cells with unique cell barcodes. In a 184-electrode DMF-Protein-seq system, \\u22481800 cells are simultaneously detected per experimental run. The digital microfluidics device harnessing low-adsorbed hydrophobic surface and contaminants-isolated reaction space supports high cell utilization (>90%) and high mapping reads (>90%) with the input cells ranging from 140 to 2000. This system leverages split&pool strategy on the DMF chip for the first time to overcome DMF platform restriction in cell analysis throughput and replace the traditionally tedious bench-top combinatorial barcoding. With the benefits of high efficiency and sensitivity in protein analysis, the system offers great potential for cell classification and drug monitoring based on protein expression at the single-cell level.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=32&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c877dd7ffe0e1dd7ecdd6428dbfa8f3b83e2ad7\",\n",
      "        \"title\": \"Advances in the joint profiling technologies of 5mC and 5hmC\",\n",
      "        \"abstract\": \"DNA cytosine methylation, a crucial epigenetic modification, involves the dynamic interplay of 5-methylcytosine (5mC) and its oxidized form, 5-hydroxymethylcytosine (5hmC), generated by ten-eleven translocation (TET) DNA dioxygenases. This process is central to regulating gene expression, influencing critical biological processes such as development, disease progression, and aging. Recognizing the distinct functions of 5mC and 5hmC, researchers often employ restriction enzyme-based or chemical treatment methods for their simultaneous measurement from the same genomic sample. This enables a detailed understanding of the relationship between these modifications and their collective impact on cellular function. This review focuses on summarizing the technologies for detecting 5mC and 5hmC together but also discusses the limitations and potential future directions in this evolving field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=33&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"34036d9f94d2ecce98511ebb71ba5defbf42d883\",\n",
      "        \"title\": \"Molecular and cellular mechanisms of selective vulnerability in neurodegenerative diseases.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=34&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"190cae77328960dc1c2dc6db9aabc248954b727d\",\n",
      "        \"title\": \"CITEViz: interactively classify cell populations in CITE-Seq via a flow cytometry-like gating workflow using R-Shiny\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=35&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"42b804e07eede558bf9d27f96e89be9f753098ae\",\n",
      "        \"title\": \"The Role of Immune Cells in DKD: Mechanisms and Targeted Therapies\",\n",
      "        \"abstract\": \"Abstract Diabetic kidney disease (DKD), is a common microvascular complication and a major cause of death in patients with diabetes. Disorders of immune cells and immune cytokines can accelerate DKD development of in a number of ways. As the kidney is composed of complex and highly differentiated cells, the interactions among different cell types and immune cells play important regulatory roles in disease development. Here, we summarize the latest research into the molecular mechanisms underlying the interactions among various immune and renal cells in DKD. In addition, we discuss the most recent studies related to single cell technology and bioinformatics analysis in the field of DKD. The aims of our review were to explore immune cells as potential therapeutic targets in DKD and provide some guidance for future clinical treatments.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=36&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ed74691b564b16ca43c12171633ded1dddeac1e\",\n",
      "        \"title\": \"Applications of single-cell multi-omics in liver cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=37&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"093ad3365acf9e31de630d4879cfb88796ad30b7\",\n",
      "        \"title\": \"Type I interferons induce an epigenetically distinct memory B cell subset in chronic viral infection.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=38&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ad968bdb06208dc485a3f2c3b6681567223d4925\",\n",
      "        \"title\": \"Glutamine antagonist DRP-104 suppresses tumor growth and enhances response to checkpoint blockade in KEAP1 mutant lung cancer\",\n",
      "        \"abstract\": \"Loss-of-function mutations in KEAP1 frequently occur in lung cancer and are associated with poor prognosis and resistance to standard of care treatment, highlighting the need for the development of targeted therapies. We previously showed that KEAP1 mutant tumors consume glutamine to support the metabolic rewiring associated with NRF2-dependent antioxidant production. Here, using preclinical patient-derived xenograft models and antigenic orthotopic lung cancer models, we show that the glutamine antagonist prodrug DRP-104 impairs the growth of KEAP1 mutant tumors. We find that DRP-104 suppresses KEAP1 mutant tumors by inhibiting glutamine-dependent nucleotide synthesis and promoting antitumor T cell responses. Using multimodal single-cell sequencing and ex vivo functional assays, we demonstrate that DRP-104 reverses T cell exhaustion, decreases Tregs, and enhances the function of CD4 and CD8 T cells, culminating in an improved response to anti-PD1 therapy. Our preclinical findings provide compelling evidence that DRP-104, currently in clinical trials, offers a promising therapeutic approach for treating patients with KEAP1 mutant lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=39&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c906b0c0f91029683ae56162b70289b0c21f1fd6\",\n",
      "        \"title\": \"Fast and flexible profiling of chromatin accessibility and total RNA expression in single nuclei using Microwell-seq3\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=40&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c5e4e4a24fe2597b5b651953cfda606e092999d7\",\n",
      "        \"title\": \"Peripheral priming induces plastic transcriptomic and proteomic responses in circulating neutrophils required for pathogen containment\",\n",
      "        \"abstract\": \"Neutrophils rapidly respond to inflammation and infection, but to which degree their functional trajectories after mobilization from the bone marrow are shaped within the circulation remains vague. Experimental limitations have so far hampered neutrophil research in human disease. Here, using innovative fixation and single-cell\\u2013based toolsets, we profile human and murine neutrophil transcriptomes and proteomes during steady state and bacterial infection. We find that peripheral priming of circulating neutrophils leads to dynamic shifts dominated by conserved up-regulation of antimicrobial genes across neutrophil substates, facilitating pathogen containment. We show the TLR4/NF-\\u03baB signaling\\u2013dependent up-regulation of canonical neutrophil activation markers like CD177/NB-1 during acute inflammation, resulting in functional shifts in vivo. Blocking de novo RNA synthesis in circulating neutrophils abrogates these plastic shifts and prevents the adaptation of antibacterial neutrophil programs by up-regulation of distinct effector molecules upon infection. These data underline transcriptional plasticity as a relevant mechanism of functional neutrophil reprogramming during acute infection to foster bacterial containment within the circulation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=41&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7c071256f41c71cbb6749c18a25dbdfc6b5eef32\",\n",
      "        \"title\": \"The Alzheimer's disease risk gene BIN1 regulates activity-dependent gene expression in human-induced glutamatergic neurons.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=42&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f0ec07d73b8f4945d615c9c5cb3b5a7b06249760\",\n",
      "        \"title\": \"An immunophenotype-coupled transcriptomic atlas of human hematopoietic progenitors\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=43&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5127a064566cd61fe45bfec1d4ccbf03592dc87a\",\n",
      "        \"title\": \"Mapping human hematopoiesis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=44&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1bd12fff91ebae7308db0ea6b1a4fbd5bec90f0a\",\n",
      "        \"title\": \"Opto-combinatorial indexing enables high-content transcriptomics by linking cell images and transcriptomes.\",\n",
      "        \"abstract\": \"We introduce a simple integrated analysis method that links cellular phenotypic behaviour with single-cell RNA sequencing (scRNA-seq) by utilizing a combination of optical indices from cells and hydrogel beads. With our method, the combinations, referred to as joint colour codes, enable the link via matching the optical combinations measured by conventional epi-fluorescence microscopy with the concatenated DNA molecular barcodes created by cell-hydrogel bead pairs and sequenced by next-generation sequencing. We validated our approach by demonstrating an accurate link between the cell image and scRNA-seq with mixed species experiments, longitudinal cell tagging by electroporation and lipofection, and gene expression analysis. Furthermore, we extended our approach to multiplexed chemical transcriptomics, which enabled us to identify distinct phenotypic behaviours in HeLa cells treated with various concentrations of paclitaxel, and determine the corresponding gene regulation associated with the formation of a multipolar spindle.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=45&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e3752e5e3453fd6592061c7ce2e1f7c29dafb4c0\",\n",
      "        \"title\": \"The proteomic landscape of microglia in health and disease\",\n",
      "        \"abstract\": \"Microglia are the resident immune cells of the central nervous system (CNS) and as such play crucial roles in regulating brain homeostasis. Their presence in neurodegenerative diseases is known, with neurodegeneration-associated risk genes heavily expressed in microglia, highlighting their importance in contributing to disease pathogenesis. Transcriptomics studies have uncovered the heterogeneous landscape of microglia in health and disease, identifying important disease-associated signatures such as DAM, and insight into both the regional and temporal diversity of microglia phenotypes. Quantitative mass spectrometry methods are ever increasing in the field of neurodegeneration, utilised as ways to identify disease biomarkers and to gain deeper understanding of disease pathology. Proteins are the main mechanistic indicators of cellular function, yet discordance between transcript and proteomic findings has highlighted the need for in-depth proteomic phenotypic and functional analysis to fully understand disease kinetics at the cellular and molecular level. This review details the current progress of using proteomics to define microglia biology, the relationship between gene and protein expression in microglia, and the future of proteomics and emerging methods aiming to resolve heterogeneous cell landscapes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=46&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b5e50e159f56d4368eba9352a843433ea9999ce9\",\n",
      "        \"title\": \"Photosensitive Hydrogel with Temperature\\u2010Controlled Reversible Nano\\u2010Apertures for Single\\u2010Cell Protein Analysis\",\n",
      "        \"abstract\": \"Single cell western blot (scWB) is one of the most important methods for cellular heterogeneity profiling. However, current scWB based on conventional photoactive polyacrylamide hydrogel material suffers from the tradeoff between in\\u2010gel probing and separation resolution. Here, a highly sensitive temperature\\u2010controlled single\\u2010cell western blotting (tc\\u2010scWB) method is introduced, which is based on a thermo/photo\\u2010dualistic\\u2010sensitive polyacrylamide hydrogel, namely acrylic acid\\u2010functionalized graphene oxide (AFGO) assisted, N\\u2010isopropylacrylamide modified polyacrylamide (ANP) hydrogel. The ANP hydrogel is contracted at high\\u2010temperature to constrain protein band diffusion during microchip electrophoretic separation, while the gel aperture is expanded under low\\u2010temperature for better antibody penetration into the hydrogel. The tc\\u2010scWB method enables the separation and profiling of small\\u2010molecule\\u2010weight proteins with highly crosslinked gel (12% T) in SDS\\u2010PAGE. The tc\\u2010scWB is demonstrated on three metabolic and ER stress\\u2010specific proteins (CHOP, MDH2 and FH) in four pancreatic cell subtypes, revealing the expression of key enzymes in the Krebs cycle is upregulated with enhanced ER stress. It is found that ER stress can regulate crucial enzyme (MDH2 and FH) activities of metabolic cascade in cancer cells, boosting aerobic respiration to attenuate the Warburg effect and promote cell apoptosis. The tc\\u2010scWB is a general toolbox for the analysis of low\\u2010abundance small\\u2010molecular functional proteins at the single\\u2010cell level.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=47&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"243f718013f32667911045b4484ee057369e5d95\",\n",
      "        \"title\": \"scmFormer Integrates Large\\u2010Scale Single\\u2010Cell Proteomics and Transcriptomics Data by Multi\\u2010Task Transformer\",\n",
      "        \"abstract\": \"Transformer\\u2010based models have revolutionized single cell RNA\\u2010seq (scRNA\\u2010seq) data analysis. However, their applicability is challenged by the complexity and scale of single\\u2010cell multi\\u2010omics data. Here a novel single\\u2010cell multi\\u2010modal/multi\\u2010task transformer (scmFormer) is proposed to fill up the existing blank of integrating single\\u2010cell proteomics with other omics data. Through systematic benchmarking, it is demonstrated that scmFormer excels in integrating large\\u2010scale single\\u2010cell multimodal data and heterogeneous multi\\u2010batch paired multi\\u2010omics data, while preserving shared information across batchs and distinct biological information. scmFormer achieves 54.5% higher average F1 score compared to the second method in transferring cell\\u2010type labels from single\\u2010cell transcriptomics to proteomics data. Using COVID\\u201019 datasets, it is presented that scmFormer successfully integrates over 1.48 million cells on a personal computer. Moreover, it is also proved that scmFormer performs better than existing methods on generating the unmeasured modality and is well\\u2010suited for spatial multi\\u2010omic data. Thus, scmFormer is a powerful and comprehensive tool for analyzing single\\u2010cell multi\\u2010omics data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=48&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1cab8ca82831e076712598c6f3d3660e69cf6963\",\n",
      "        \"title\": \"Nanoscale Methods for Longitudinal Extraction of Intracellular Contents.\",\n",
      "        \"abstract\": \"Longitudinal analysis of intracellular contents including gene and protein expression is crucial for deciphering the fundamentally dynamic nature of cells. This offers invaluable insights into complex tissue composition and behavior, and drive progress in disease diagnosis, biomarker discovery and drug development. Traditional longitudinal analysis workflows, involving the destruction of cells at various timepoints, limit insights to singular moments and fail to account for cellular heterogeneity. Current non-destructive approaches, like temporal modelling with single-cell RNA-sequencing and live-cell fluorescence imaging, either rely on biological assumptions or possess risk of cellular perturbation. Recent advances in nanoscale technologies for non-destructive intracellular content extraction offer a promising solution to these challenges. These novel methods work at the nanoscale to non-destructively access cellular membranes and can be broadly classified into three mechanisms: tip-facilitated aspiration, membrane-based and probe-based methods. This perspective focuses on these emerging nanotechnologies for repeated intracellular content extraction. We discuss their potential in longitudinal analysis, address the critical requirements for effective repeated sampling, and explore the suitability of each technique for various applications. Furthermore, we highlight unresolved challenges in repeated sampling to encourage further research in this growing field. This article is protected by copyright. All rights reserved.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=49&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4dbf09f5a0d201167c04f789f70422e18569024b\",\n",
      "        \"title\": \"ScLinear predicts protein abundance at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=50&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"53dc4e61196676fccebe5736ed5c71abb19b7509\",\n",
      "        \"title\": \"Omics approaches in asthma research: Challenges and opportunities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=51&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d347f856ed2b8596320c4970f71d463269011db3\",\n",
      "        \"title\": \"Nanopore DNA sequencing technologies and their applications towards single-molecule proteomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=52&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3ed4bd804de506b917002f3b61c371e9d30526b1\",\n",
      "        \"title\": \"Systems analysis of innate and adaptive immunity in Long COVID.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=53&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"44d251020aad0d8e1ce473dd0b188538bde342ab\",\n",
      "        \"title\": \"Advances in single-cell omics and multiomics for high-resolution molecular profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=54&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f59f0ab2b5f90c4f2619bb22c97499be09ceb61c\",\n",
      "        \"title\": \"A commentary on multi-omics data integration in systems vaccinology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=55&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1e5119c2fdc359050826c9062026e0940ba99a3e\",\n",
      "        \"title\": \"Computational prediction of protein interactions in single cells by proximity sequencing\",\n",
      "        \"abstract\": \"Proximity sequencing (Prox-seq) simultaneously measures gene expression, protein expression and protein complexes on single cells. Using information from dual-antibody binding events, Prox-seq infers surface protein dimers at the single-cell level. Prox-seq provides multi-dimensional phenotyping of single cells in high throughput, and was recently used to track the formation of receptor complexes during cell signaling and discovered a novel interaction between CD9 and CD8 in na\\u00efve T cells. The distribution of protein abundance can affect identification of protein complexes in a complicated manner in dual-binding assays like Prox-seq. These effects are difficult to explore with experiments, yet important for accurate quantification of protein complexes. Here, we introduce a physical model of Prox-seq and computationally evaluate several different methods for reducing background noise when quantifying protein complexes. Furthermore, we developed an improved method for analysis of Prox-seq data, which resulted in more accurate and robust quantification of protein complexes. Finally, our Prox-seq model offers a simple way to investigate the behavior of Prox-seq data under various biological conditions and guide users toward selecting the best analysis method for their data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=56&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75171c00a4082ecc12b7a39d6905867fe37bb791\",\n",
      "        \"title\": \"Pooled CRISPR screening of high-content cellular phenotypes using ghost cytometry\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=57&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"46895db9a5c159b2fde261487eafffc14a0b6cc9\",\n",
      "        \"title\": \"Unique challenges and best practices for single cell transcriptomic analysis in toxicology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=58&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3ab002e5ce2fad70308adc6d437ad28ca8a3dc49\",\n",
      "        \"title\": \"Harnessing HetHydrogel: A Universal Platform to Dropletize Single-Cell Multiomics.\",\n",
      "        \"abstract\": \"A universal platform is developed for dropletizing single cell plate-based multiomic assays, consisting of three main pillars: a miniaturized open Heterogeneous Hydrogel reactor (abbreviated HetHydrogel) for multi-step biochemistry, its tunable permeability that allows Tn5 tagmentation, and single cell droplet barcoding. Through optimizing the HetHydrogel manufacturing procedure, the chemical composition, and cell permeation conditions, simultaneous high-throughput mitochondrial DNA genotyping and chromatin profiling at the single-cell level are demonstrated using a mixed-species experiment. This platform offers a powerful way to investigate the genotype-phenotype relationships of various mtDNA mutations in biological processes. The HetHydrogel platform is believed to have the potential to democratize droplet technologies, upgrading a whole range of plate-based single cell assays to high throughput format.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=59&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"36424ca7de1500c606a2cf9d549368c552183e7d\",\n",
      "        \"title\": \"Anti-TIGIT antibody improves PD-L1 blockade through myeloid and Treg cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=60&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ec7fac452f7fbce1c51964f811d10b3c057e988a\",\n",
      "        \"title\": \"Tracking the immune response profiles elicited by the BNT162b2 vaccine in COVID-19 unexperienced and experienced individuals.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=61&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dab7e584844142296aa486a4d66503712883e2da\",\n",
      "        \"title\": \"scGIST: gene panel design for spatial transcriptomics with prioritized gene sets\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=62&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e7b8fda72ae154fbc2ae5a993a853fad1ad4deeb\",\n",
      "        \"title\": \"The rise of single\\u2010cell transcriptomics in yeast\",\n",
      "        \"abstract\": \"The field of single\\u2010cell omics has transformed our understanding of biological processes and is constantly advancing both experimentally and computationally. One of the most significant developments is the ability to measure the transcriptome of individual cells by single\\u2010cell RNA\\u2010seq (scRNA\\u2010seq), which was pioneered in higher eukaryotes. While yeast has served as a powerful model organism in which to test and develop transcriptomic technologies, the implementation of scRNA\\u2010seq has been significantly delayed in this organism, mainly because of technical constraints associated with its intrinsic characteristics, namely the presence of a cell wall, a small cell size and little amounts of RNA. In this review, we examine the current technologies for scRNA\\u2010seq in yeast and highlight their strengths and weaknesses. Additionally, we explore opportunities for developing novel technologies and the potential outcomes of implementing single\\u2010cell transcriptomics and extension to other modalities. Undoubtedly, scRNA\\u2010seq will be invaluable for both basic and applied yeast research, providing unique insights into fundamental biological processes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=63&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"087f193fb87c0a4071ae9ef39114ccf1e161e1b4\",\n",
      "        \"title\": \"High-dimensional covariate-augmented overdispersed poisson factor model.\",\n",
      "        \"abstract\": \"The current Poisson factor models often assume that the factors are unknown, which overlooks the explanatory potential of certain observable covariates. This study focuses on high dimensional settings, where the number of the count response variables and/or covariates can diverge as the sample size increases. A covariate-augmented overdispersed Poisson factor model is proposed to jointly perform a high-dimensional Poisson factor analysis and estimate a large coefficient matrix for overdispersed count data. A group of identifiability conditions is provided to theoretically guarantee computational identifiability. We incorporate the interdependence of both response variables and covariates by imposing a low-rank constraint on the large coefficient matrix. To address the computation challenges posed by nonlinearity, two high-dimensional latent matrices, and the low-rank constraint, we propose a novel variational estimation scheme that combines Laplace and Taylor approximations. We also develop a criterion based on a singular value ratio to determine the number of factors and the rank of the coefficient matrix. Comprehensive simulation studies demonstrate that the proposed method outperforms the state-of-the-art methods in estimation accuracy and computational efficiency. The practical merit of our method is demonstrated by an application to the CITE-seq dataset. A flexible implementation of our proposed method is available in the R package COAP.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=64&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d7030df889e86763551a0e1e6b6327c9693ffa7a\",\n",
      "        \"title\": \"Multimodal and spatially resolved profiling identifies distinct patterns of T cell infiltration in nodal B cell lymphoma entities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=65&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"16e8adcaddc78e6e880d5ef2dfff2a807b77d7f3\",\n",
      "        \"title\": \"Single-cell and spatial multi-omics highlight effects of anti-integrin therapy across cellular compartments in ulcerative colitis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=66&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2001b1f912032cb4ea73677f3811c24d70d1eee7\",\n",
      "        \"title\": \"Single-cell and spatial transcriptomics: Bridging current technologies with long-read sequencing.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=67&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4d1fcaf22431bfa5f1c69ab81630ff61b4557346\",\n",
      "        \"title\": \"Toward universal cell embeddings: integrating single-cell RNA-seq datasets across species with SATURN.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=68&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4ca5ee4274358b63389a125acf86d54f720f018\",\n",
      "        \"title\": \"Simultaneous single-cell analysis of 5mC and 5hmC with SIMPLE-seq.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=69&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ac8d7d097098d0984b1253208a0fc25df8f8b2d7\",\n",
      "        \"title\": \"Functional genomics in inborn errors of immunity\",\n",
      "        \"abstract\": \"Inborn errors of immunity (IEI) comprise a diverse spectrum of 485 disorders as recognized by the International Union of Immunological Societies Committee on Inborn Error of Immunity in 2022. While IEI are monogenic by definition, they illuminate various pathways involved in the pathogenesis of polygenic immune dysregulation as in autoimmune or autoinflammatory syndromes, or in more common infectious diseases that may not have a significant genetic basis. Rapid improvement in genomic technologies has been the main driver of the accelerated rate of discovery of IEI and has led to the development of innovative treatment strategies. In this review, we will explore various facets of IEI, delving into the distinctions between PIDD and PIRD. We will examine how Mendelian inheritance patterns contribute to these disorders and discuss advancements in functional genomics that aid in characterizing new IEI. Additionally, we will explore how emerging genomic tools help to characterize new IEI as well as how they are paving the way for innovative treatment approaches for managing and potentially curing these complex immune conditions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=70&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b0462b137e3f86e27c50899378b070b6a645953b\",\n",
      "        \"title\": \"Building and analyzing metacells in single-cell genomics data\",\n",
      "        \"abstract\": \"The advent of high-throughput single-cell genomics technologies has fundamentally transformed biological sciences. Currently, millions of cells from complex biological tissues can be phenotypically profiled across multiple modalities. The scaling of computational methods to analyze such data is a constant challenge and tools need to be regularly updated, if not redesigned, to cope with ever-growing numbers of cells. Over the last few years, metacells have been introduced to reduce the size and complexity of single-cell genomics data while preserving biologically relevant information. Here, we review recent studies that capitalize on the concept of metacells \\u2013 and the many variants in nomenclature that have been used. We further outline how and when metacells should (or should not) be used to study single-cell genomics data and what should be considered when analyzing such data at the metacell level. To facilitate the exploration of metacells, we provide a comprehensive tutorial on construction and analysis of metacells from single-cell RNA-seq data (https://github.com/GfellerLab/MetacellAnalysisTutorial) as well as a fully integrated pipeline to rapidly build, visualize and evaluate metacells with different methods (https://github.com/GfellerLab/MetacellAnalysisToolkit).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=71&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b69f3afd84504a2820dbe3293869d0616f13a171\",\n",
      "        \"title\": \"Multiplex protein imaging in tumour biology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=72&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aa7927a48f400a75850ac017348a2f56c30a379a\",\n",
      "        \"title\": \"Preservation of scRNA-Seq Libraries Using Existing Inactivation Protocols\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing has soared in popularity in recent years. The ability to deeply profile the states of individual cells during the course of disease or infection has helped to expand our knowledge of coordinated responses. However, significant challenges arise when performing this analysis in high containment settings such as biosafety level 3 (BSL-3), BSL-3+ and BSL-4. Working in containment is necessary for many important pathogens, such as Ebola virus, Marburg virus, Lassa virus, Nipah and Hendra viruses. Since standard operating procedures (SOPs) for inactivation are extensive and may compromise sample integrity, we tested whether the removal of single-cell sequencing libraries from containment laboratories using existing inactivation protocols for nucleic acid extraction (Trizol, RLT buffer, or AVL buffer) was feasible. We have demonstrated that the inactivation does not affect sample quality and can work with existing methods for inactivation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=73&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cf422ba886b3a9e64b3c787431a7ac1b038f67f0\",\n",
      "        \"title\": \"Analytical Workflows for Single\\u2010Cell Multiomic Data Using the BD Rhapsody Platform\",\n",
      "        \"abstract\": \"The conversion of raw sequencing reads to biologically relevant data in high\\u2010throughput single\\u2010cell RNA sequencing experiments is a complex and involved process. Drawing meaning from thousands of individual cells to provide biological insight requires ensuring not only that the data are of the highest quality but also that the signal can be separated from noise. In this article, we describe a detailed analytical workflow, including six pipelines, that allows high\\u2010quality data analysis in single\\u2010cell multiomics. \\u00a9 2024 The Authors. Current Protocols published by Wiley Periodicals LLC.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=74&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"edbf3731e6f507025179048425acbec8d82d56cf\",\n",
      "        \"title\": \"Development and application of transcriptomics technologies in plant science\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=75&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"29609cc1c485e71f7565dd9f51ffacb229ccb5e3\",\n",
      "        \"title\": \"An iron rheostat controls hematopoietic stem cell fate.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=76&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abf4498c28c7b20674fcce6b63cfd866e495620c\",\n",
      "        \"title\": \"Harnessing the Transcriptional Signatures of CAR-T-Cells and Leukemia/Lymphoma Using Single-Cell Sequencing Technologies\",\n",
      "        \"abstract\": \"Chimeric antigen receptor (CAR)-T-cell therapy has greatly improved outcomes for patients with relapsed or refractory hematological malignancies. However, challenges such as treatment resistance, relapse, and severe toxicity still hinder its widespread clinical application. Traditional transcriptome analysis has provided limited insights into the complex transcriptional landscape of both leukemia cells and engineered CAR-T-cells, as well as their interactions within the tumor microenvironment. However, with the advent of single-cell sequencing techniques, a paradigm shift has occurred, providing robust tools to unravel the complexities of these factors. These techniques enable an unbiased analysis of cellular heterogeneity and molecular patterns. These insights are invaluable for precise receptor design, guiding gene-based T-cell modification, and optimizing manufacturing conditions. Consequently, this review utilizes modern single-cell sequencing techniques to clarify the transcriptional intricacies of leukemia cells and CAR-Ts. The aim of this manuscript is to discuss the potential mechanisms that contribute to the clinical failures of CAR-T immunotherapy. We examine the biological characteristics of CAR-Ts, the mechanisms that govern clinical responses, and the intricacies of adverse events. By exploring these aspects, we hope to gain a deeper understanding of CAR-T therapy, which will ultimately lead to improved clinical outcomes and broader therapeutic applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=77&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a33e4149fa8ff0018e0a9ef6907e0df8d79a81a3\",\n",
      "        \"title\": \"Dynamics of Whole Transcriptome Analysis (WTA) and Surface markers expression (AbSeq) in Immune Cells of COVID-19 Patients and Recovered captured through Single Cell Genomics\",\n",
      "        \"abstract\": \"Introduction Single-cell multi-omics studies, such as multidimensional transcriptomics (whole transcriptomic analysis, WTA), and surface marker analysis (antibody sequencing, AbSeq), have turned out to be valuable techniques that offer inaccessible possibilities for single-cell profiling of mRNA, lncRNA, and proteins. Methods We used this technique to understand the dynamics of mRNA and protein-level differences in healthy, COVID-19-infected and recovered individuals using peripheral blood mononuclear cells (PBMCs). Our results demonstrate that compared to mRNA expression, protein abundance is a better indicator of the disease state. Results We demonstrate that compared to mRNA expression, protein abundance is a better indicator of the disease state. We observed high levels of cell identity and regulatory markers, CD3E, CD4, CD8A, CD5, CD7, GITR, and KLRB1 in healthy individuals, whereas markers related to cell activation, CD38, CD28, CD69, CD62L, CD14, and CD16 elevated in the SARS-CoV-2 infected patients at both WTA and AbSeq levels. Curiously, in recovered individuals, there was a high expression of cytokine and chemokine receptors (CCR5, CCR7, CCR4, CXCR3, and PTGRD2). We also observed variations in the expression of markers within cell populations under different states. Discussion Furthermore, our study emphasizes the significance of employing an oligo-based method (AbSeq) that can help in diagnosis, prognosis, and protection from disease/s by identifying cell surface markers that are unique to different cell types or states. It also allows simultaneous study of a vast array of markers, surpassing the constraints of techniques like FACS to query the vast repertoire of proteins.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=78&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0c10a733ab4e0fa1a6e2f292a09f600b638741c2\",\n",
      "        \"title\": \"Mapping cancer biology in space: applications and perspectives on spatial omics for oncology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=79&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5a99347a921a6c3f4e9b390ca5818cb383f52e28\",\n",
      "        \"title\": \"SuperCellCyto: enabling efficient analysis of large scale cytometry datasets\",\n",
      "        \"abstract\": \"Advancements in cytometry technologies have enabled quantification of up to 50 proteins across millions of cells at single cell resolution. Analysis of cytometry data routinely involves tasks such as data integration, clustering, and dimensionality reduction. While numerous tools exist, many require extensive run times when processing large cytometry data containing millions of cells. Existing solutions, such as random subsampling, are inadequate as they risk excluding rare cell subsets. To address this, we propose SuperCellCyto, an R package that builds on the SuperCell tool which groups highly similar cells into supercells. SuperCellCyto is available on GitHub (https://github.com/phipsonlab/SuperCellCyto).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=80&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"500ae47d905efb51faa1d3311946d6d8b9fc7782\",\n",
      "        \"title\": \"Deciphering metabolic crosstalk in context: lessons from inflammatory diseases.\",\n",
      "        \"abstract\": \"Metabolism plays a crucial role in regulating the function of immune cells in both health and disease, with altered metabolism contributing to the pathogenesis of cancer and many inflammatory diseases. The local microenvironment has a profound impact on the metabolism of immune cells. Therefore, immunological and metabolic heterogeneity as well as the spatial organization of cells in tissues should be taken into account when studying immunometabolism. Here, we highlight challenges of investigating metabolic communication. Additionally, we review the capabilities and limitations of current technologies for studying metabolism in inflamed microenvironments, including single-cell omics techniques, flow cytometry-based methods (Met-Flow, single-cell energetic metabolism by profiling translation inhibition (SCENITH)), cytometry by time of flight (CyTOF), cellular indexing of transcriptomes and epitopes by sequencing (CITE-Seq), and mass spectrometry imaging. Considering the importance of metabolism in regulating immune cells in diseased states, we also discuss the applications of metabolomics in clinical research, as well as some hurdles to overcome to implement these techniques in standard clinical practice. Finally, we provide a flowchart to assist scientists in designing effective strategies to unravel immunometabolism in disease-relevant contexts.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=81&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"72e84df18ec1ec9b260610853787e6e5fea29799\",\n",
      "        \"title\": \"InClust+: the deep generative framework with mask modules for multimodal data integration, imputation, and cross-modal generation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=82&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"379cca82e76f530db74540586a5ba73fddff4d1c\",\n",
      "        \"title\": \"Mosaic integration and knowledge transfer of single-cell multimodal data with MIDAS.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=83&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cfef4959c6214fa58658b1f70aa4f0e242904807\",\n",
      "        \"title\": \"scMMT: a multi-use deep learning approach for cell annotation, protein prediction and embedding in single-cell RNA-seq data\",\n",
      "        \"abstract\": \"Abstract Accurate cell type annotation in single-cell RNA-sequencing data is essential for advancing biological and medical research, particularly in understanding disease progression and tumor microenvironments. However, existing methods are constrained by single feature extraction approaches, lack of adaptability to immune cell types with similar molecular profiles but distinct functions and a failure to account for the impact of cell label noise on model accuracy, all of which compromise the precision of annotation. To address these challenges, we developed a supervised approach called scMMT. We proposed a novel feature extraction technique to uncover more valuable information. Additionally, we constructed a multi-task learning framework based on the GradNorm method to enhance the recognition of challenging immune cells and reduce the impact of label noise by facilitating mutual reinforcement between cell type annotation and protein prediction tasks. Furthermore, we introduced logarithmic weighting and label smoothing mechanisms to enhance the recognition ability of rare cell types and prevent model overconfidence. Through comprehensive evaluations on multiple public datasets, scMMT has demonstrated state-of-the-art performance in various aspects including cell type annotation, rare cell identification, dropout and label noise resistance, protein expression prediction and low-dimensional embedding representation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=84&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7a5879aae61729dbdf11e90813c14f8a1f1a2bb9\",\n",
      "        \"title\": \"scMLC: an accurate and robust multiplex community detection method for single-cell multi-omics data\",\n",
      "        \"abstract\": \"Abstract Clustering cells based on single-cell multi-modal sequencing technologies provides an unprecedented opportunity to create high-resolution cell atlas, reveal cellular critical states and study health and diseases. However, effectively integrating different sequencing data for cell clustering remains a challenging task. Motivated by the successful application of Louvain in scRNA-seq data, we propose a single-cell multi-modal Louvain clustering framework, called scMLC, to tackle this problem. scMLC builds multiplex single- and cross-modal cell-to-cell networks to capture modal-specific and consistent information between modalities and then adopts a robust multiplex community detection method to obtain the reliable cell clusters. In comparison with 15 state-of-the-art clustering methods on seven real datasets simultaneously measuring gene expression and chromatin accessibility, scMLC achieves better accuracy and stability in most datasets. Synthetic results also indicate that the cell-network-based integration strategy of multi-omics data is superior to other strategies in terms of generalization. Moreover, scMLC is flexible and can be extended to single-cell sequencing data with more than two modalities.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=85&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f5376dad15cea1a79db65820ef8afba7613bb17\",\n",
      "        \"title\": \"Single-cell RNA sequencing in melanoma: what have we learned so far?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=86&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"42e66169e75a66e1b7ef1eba9f6a814f2dee1cc0\",\n",
      "        \"title\": \"Spatial multi-omics: novel tools to study the complexity of cardiovascular diseases\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=87&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"369e3b5507411daf0c608272915791758f7e6d74\",\n",
      "        \"title\": \"Understanding Spondyloarthritis Pathogenesis: The Promise of Single-Cell Profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=88&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"432a5607b99f76f9475cb7889787af9d24865c62\",\n",
      "        \"title\": \"Tuft cells and fibroblasts promote thymus regeneration through ILC2-mediated type 2 immune response\",\n",
      "        \"abstract\": \"The thymus is a primary lymphoid organ that is essential for the establishment of adaptive immunity through generation of immunocompetent T cells. In response to various stress signals, the thymus undergoes acute but reversible involution. However, the mechanisms governing its recovery are incompletely understood. Here, we used a dexamethasone-induced acute thymic involution mouse model to investigate how thymic hematopoietic cells (excluding T cells) contribute to thymic regeneration. scRNA-seq analysis revealed marked transcriptional and cellular changes in various thymic populations and highlighted thymus-resident innate lymphoid cells type 2 (ILC2) as a key cell type involved in the response to damage. We identified that ILC2 are activated by the alarmins IL-25 and IL-33 produced in response to tissue damage by thymic tuft cells and fibroblasts, respectively. Moreover, using mouse models deficient in either tuft cells and/or IL-33, we found that these alarmins are required for effective thymus regeneration after dexamethasone-induced damage. We also demonstrate that upon their damage-dependent activation, thymic ILC2 produce several effector molecules linked to tissue regeneration, such as amphiregulin and IL-13, which in turn promote thymic epithelial cell differentiation. Collectively, our study elucidates a previously undescribed role for thymic tuft cells and fibroblasts in thymus regeneration through activation of the type 2 immune response. Thymus regeneration upon glucocorticoid-induced acute involution is orchestrated by IL-25\\u2013 and IL-33\\u2013dependent activation of ILC2. Editor\\u2019s summary The thymus undergoes transient but reversible involution in response to stress stimuli, but the molecular mechanisms mediating thymic regeneration after involution are not well defined. Nevo et al. performed scRNA-seq on the thymic non\\u2013T cell compartment after dexamethasone-induced acute thymic involution (ATI) in mice. Dexamethasone-induced ATI promoted a type 2 immune response driven by thymic resident ILC2s that were activated in response to tissue damage by IL-25 produced by thymic tuft cells and IL-33 produced by fibroblasts. ILC2s facilitated tissue regeneration by producing effector molecules including AREG, IL-13, CSF2, and IL-5, which promoted mTEC differentiation. Together, these findings identify a thymic tuft cell\\u2013fibroblast\\u2013ILC2 axis required for thymic regeneration after ATI. \\u2014Hannah Isles\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=89&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"06983be76b6f54f1443576fc0a4a49df88e69494\",\n",
      "        \"title\": \"Memory B cell subsets have divergent developmental origins that are coupled to distinct imprinted epigenetic states.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=90&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6cecc501e3e2e6ba5a64cd5db82a2cb9d049262c\",\n",
      "        \"title\": \"A fast, scalable and versatile tool for analysis of single-cell omics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=91&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6f737b7061c49ccf8af9a5a89b12c7b1ef8374cd\",\n",
      "        \"title\": \"Single Cell Sequencing Technology and Its Application in Alzheimer's Disease.\",\n",
      "        \"abstract\": \"Alzheimer's disease (AD) involves degeneration of cells in the brain. Due to insidious onset and slow progression, AD is often not diagnosed until it gets progressed to a more severe stage. The diagnosis and treatment of AD has been a challenge. In recent years, high-throughput sequencing technologies have exhibited advantages in exploring the pathogenesis of diseases. However, the types of cells of the central nervous system are complex and traditional bulk sequencing cannot reflect their heterogeneity. Single-cell sequencing technology enables study at the individual cell level and has an irreplaceable advantage in the study of complex diseases. In recent years, this field has expanded rapidly and several types of single-cell sequencing technologies have emerged, including transcriptomics, epigenomics, genomics and proteomics. This review article provides an overview of these single-cell sequencing technologies and their application in AD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=92&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"22a2e8c818e78e3700f942a2cb736537926411a1\",\n",
      "        \"title\": \"The post-septic peripheral myeloid compartment reveals unexpected diversity in myeloid-derived suppressor cells\",\n",
      "        \"abstract\": \"Sepsis engenders distinct host immunologic changes that include the expansion of myeloid-derived suppressor cells (MDSCs). These cells play a physiologic role in tempering acute inflammatory responses but can persist in patients who develop chronic critical illness. The origins and lineage of these MDSC subpopulations were previously assumed to be discrete and unidirectional; however, these cells exhibit a dynamic phenotype with considerable plasticity. Using Cellular Indexing of Transcriptomes and Epitopes by Sequencing followed by transcriptomic analysis, we identify a unique lineage and differentiation pathway for MDSCs after sepsis and describe a novel MDSC subpopulation. Additionally, we report that the heterogeneous response of the myeloid compartment of blood to sepsis is dependent on clinical outcome.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=93&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a3d667b9dcc5967eab68fbf6247c52b5a6fd8bdb\",\n",
      "        \"title\": \"Protocol to study the inheritance and propagation of non-genetically encoded states using barcode decay lineage tracing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=94&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8065b4253c672ec9b400a3b0808022449f244fed\",\n",
      "        \"title\": \"Redefining Mucosal Inflammation with Spatial Genomics\",\n",
      "        \"abstract\": \"The human oral mucosa contains one of the most complex cellular systems that are essential for normal physiology and defense against a wide variety of local pathogens. Evolving techniques and experimental systems have helped refine our understanding of this complex cellular network. Current single-cell RNA sequencing methods can resolve subtle differences between cell types and states, thus providing a great tool for studying the molecular and cellular repertoire of the oral mucosa in health and disease. However, it requires the dissociation of tissue samples, which means that the interrelationships between cells are lost. Spatial transcriptomic methods bypass tissue dissociation and retain this spatial information, thereby allowing gene expression to be assessed across thousands of cells within the context of tissue structural organization. Here, we discuss the contribution of spatial technologies in shaping our understanding of this complex system. We consider the impact on identifying disease cellular neighborhoods and how space defines cell state. We also discuss the limitations and future directions of spatial sequencing technologies with recent advances in machine learning. Finally, we offer a perspective on open questions about mucosal homeostasis that these technologies are well placed to address.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=95&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"56e29f0f9d47313bbd01ad390e57385bea48abb8\",\n",
      "        \"title\": \"Single-cell joint profiling of multiple epigenetic proteins and gene transcription\",\n",
      "        \"abstract\": \"Sculpting the epigenome with a combination of histone modifications and transcription factor occupancy determines gene transcription and cell fate specification. Here, we first develop uCoTarget, utilizing a split-pool barcoding strategy for realizing ultrahigh-throughput single-cell joint profiling of multiple epigenetic proteins. Through extensive optimization for sensitivity and multimodality resolution, we demonstrate that uCoTarget enables simultaneous detection of five histone modifications (H3K27ac, H3K4me3, H3K4me1, H3K36me3, and H3K27me3) in 19,860 single cells. We applied uCoTarget to the in vitro generation of hematopoietic stem/progenitor cells (HSPCs) from human embryonic stem cells, presenting multimodal epigenomic profiles in 26,418 single cells. uCoTarget reveals establishment of pairing of HSPC enhancers (H3K27ac) and promoters (H3K4me3) and RUNX1 engagement priming for H3K27ac activation along the HSPC path. We then develop uCoTargetX, an expansion of uCoTarget to simultaneously measure transcriptome and multiple epigenome targets. Together, our methods enable generalizable, versatile multimodal profiles for reconstructing comprehensive epigenome and transcriptome landscapes and analyzing the regulatory interplay at single-cell level.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=96&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6eda2a8a6741ec806ba91e242a3b620e6b9cb5dd\",\n",
      "        \"title\": \"Integrative genotyping of cancer and immune phenotypes by long-read sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=97&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a7b3fb344e6f52680e75a103e792aea8dd797f7\",\n",
      "        \"title\": \"Single-cell sequencing analysis within biologically relevant dimensions.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=98&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fad6c8da21c63152ac5bf074039f06b02d625cf1\",\n",
      "        \"title\": \"The use of single-cell RNA-seq to study heterogeneity at varying levels of virus\\u2013host interactions\",\n",
      "        \"abstract\": \"The outcome of viral infection depends on the diversity of the infecting viral population and the heterogeneity of the cell population that is infected. Until almost a decade ago, the study of these dynamic processes during viral infection was challenging and limited to certain targeted measurements. Presently, with the use of single-cell sequencing technology, the complex interface defined by the interactions of cells with infecting virus can now be studied across the breadth of the transcriptome in thousands of individual cells simultaneously. In this review, we will describe the use of single-cell RNA sequencing (scRNA-seq) to study the heterogeneity of viral infections, ranging from individual virions to the immune response between infected individuals. In addition, we highlight certain key experimental limitations and methodological decisions that are critical to analyzing scRNA-seq data at each scale.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=99&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b42b88ec7b230c2f6b9e807172b2099d110875ed\",\n",
      "        \"title\": \"Advancing Immunotoxicology with Single-Cell Sequencing: Challenges and Progress Defining Mechanisms of Arsenic Toxicity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=100&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"320183e77a4971c34d8d2247660dace06ca299d4\",\n",
      "        \"title\": \"Cell fate decision in erythropoiesis: insights from multi-omics studies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=101&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcfe399e37a3cc358ebe0b7f62aa9d6370f43447\",\n",
      "        \"title\": \"Lack of affinity signature for germinal center cells that have initiated plasma cell differentiation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=102&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b3b8836e93df8f4913a4b9c166c804eac48c100d\",\n",
      "        \"title\": \"Single-Cell Epigenomics and Proteomics Methods Integrated in Multiomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=103&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a0231dfbcdf39edb9114fa4e7100cd941ce4a95\",\n",
      "        \"title\": \"Immunomodulatory effects and improved outcomes with cisplatin- versus carboplatin-based chemotherapy plus atezolizumab in urothelial cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=104&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4309c30212166f718c2152369fc98cd19ba625e0\",\n",
      "        \"title\": \"Single cell omics in tracing cellular heterogeneity of drug-induced liver injury: technological landscape and prospective application\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=105&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ee2740249c1f1e2704176d8ab63a807e625c08b\",\n",
      "        \"title\": \"Omics Approaches Unveiling the Biology of Human Atherosclerotic Plaques.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=106&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5a222cb41559c34093761a3e698fe26dc38378ab\",\n",
      "        \"title\": \"Single-cell analysis of the epitranscriptome: RNA modifications under the microscope\",\n",
      "        \"abstract\": \"ABSTRACT The identification of mechanisms capable of modifying genetic information by the addition of covalent RNA modifications distinguishes a level of complexity in gene expression which challenges key long-standing concepts of RNA biology. One of the current challenges of molecular biology is to properly understand the molecular functions of these RNA modifications, with more than 170 different ones having been identified so far. However, it has not been possible to map specific RNA modifications at a single-cell resolution until very recently. This review will highlight the technological advances in single-cell methodologies aimed at assessing and testing the biological function of certain RNA modifications, focusing on m6A. These advances have allowed for the development of novel strategies that enable the study of the \\u2018epitranscriptome\\u2019. Nevertheless, despite all these improvements, many challenges and difficulties still need fixing for these techniques to work efficiently.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=107&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21daeb558ecf74c3e4f5c8481b533f165b6ac0d2\",\n",
      "        \"title\": \"B cell phylogenetics in the single cell era.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=108&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"36a5cd715042ea403b5cf8f909f3c5e8b9c06c26\",\n",
      "        \"title\": \"Mapping Cell Atlases at the Single\\u2010Cell Level\",\n",
      "        \"abstract\": \"Recent advancements in single\\u2010cell technologies have led to rapid developments in the construction of cell atlases. These atlases have the potential to provide detailed information about every cell type in different organisms, enabling the characterization of cellular diversity at the single\\u2010cell level. Global efforts in developing comprehensive cell atlases have profound implications for both basic research and clinical applications. This review provides a broad overview of the cellular diversity and dynamics across various biological systems. In addition, the incorporation of machine learning techniques into cell atlas analyses opens up exciting prospects for the field of integrative biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=109&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ddf68a2d4180c9e543c66108eb477c42cb32dff5\",\n",
      "        \"title\": \"Computational Design of Synthetic Optical Barcodes in Microdroplets\",\n",
      "        \"abstract\": \"Barcodes are useful for identifying objects across time, space, and information modalities. However, materializing and decoding optical and multimodal barcodes on microscopic objects remains difficult despite the increasing need for multiplexed cell analysis. Here, a computational design of randomly combinatorial is presented, yet decodable barcodes in microdroplets. The design is based on a novel Real2Sim2Real framework: it first collects experimental images of optically distinct microparticles, then simulates massive combinatorial images by randomly assembling the imaged particles to train a neural network\\u2010based decoder. It is demonstrated that the decoder, even though trained via simulation, accurately identifies the randomly assembled particles in real hydrogel microdroplets. It also shows that the microdroplets with an additional DNA barcoding functionality are applicable to individually link independently measured microscopic images and transcriptome profiles of pooled single cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=110&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c073b23b9e16d09b5be96d9e17b26e0ba9a79d33\",\n",
      "        \"title\": \"Lineage tracing of T cell differentiation from T-iPSC by 2D feeder-free culture and 3D organoid culture\",\n",
      "        \"abstract\": \"Introduction T cells induced from induced pluripotent stem cells(iPSCs) derived from antigen-specific T cells (T-iPS-T cells) are an attractive tool for T cell immunotherapy. The induction of cytotoxic T-iPS-T cells is well established in feeder-free condition for the aim of off-the-shelf production, however, the induction of helper T-iPS-T cells remains challenging. Methods We analyzed T-iPS-T cells matured in 3D organoid culture at different steps in the culture process at the single-cell level. T-iPS-T cell datasets were merged with an available human thymocyte dataset based in single-cell RNA sequencing (scRNA-seq). Particularly, we searched for genes crucial for generation CD4+ T-iPS-T cells by comparing T-iPS-T cells established in 2D feeder-free or 3D organoid culture. Results The scRNA-seq data indicated that T-iPS-T cells are similar to T cells transitioning to human thymocytes, with SELENOW, GIMAP4, 7, SATB1, SALMF1, IL7R, SYTL2, S100A11, STAT1, IFITM1, LZTFL1 and SOX4 identified as candidate genes for the 2D feeder-free induction of CD4+ T-iPS-T cells. Discussion This study provides single cell transcriptome datasets of iPS-T cells and leads to further analysis for CD4+ T cell generation from T-iPSCs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=111&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"003d1a42b684b738d0e832ef9898e112724d1bc0\",\n",
      "        \"title\": \"Associating growth factor secretions and transcriptomes of single cells in nanovials using SEC-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=112&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f099a2e0908dae5eca51b3123d9857d2094123fb\",\n",
      "        \"title\": \"Loss of function of XBP1 splicing activity of IRE1\\u03b1 favors B cell tolerance breakdown.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=113&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cca726d10e071ba4a3ecf8dbc5e4e63ec7ad2f57\",\n",
      "        \"title\": \"scGEMOC, A Graph Embedded Contrastive Learning Single-cell Multiomics Clustering Model\",\n",
      "        \"abstract\": \"Recent advancements in single-cell multiomics sequencing create new research opportunities but also pose challenges, particularly in cell clustering. One major challenge is feature fusion. Early fusion models are robust but ignore the unique distributions of omics and cannot handle various omic dimensions. Most current clustering methods use late fusion, employing independent encoders for each omic. However, the extracted omic features belong to different latent spaces, leading to difficulties in aligning omics. Additionally, current cell clustering methods do not incorporate prior biological knowledge, such as interactions within and across omics, which has been shown plays a key role in defining cell types.To address these shortcomings, we propose a novel, scalable, end-to-end clustering method, called single-cell graph embedding multiomics cluster (scGEMOC). scGEMOC utilizes prior biological knowledge to represent inter- and intra-omics connections as a heterogeneous graph. It applies graph embedding to aggregate omics interaction data as a pseudo omic and employs contrastive learning for effectively aligning omics in the latent space. We evaluated scGEMOC on three public datasets against five state-of-the-art baseline models. scGEMOC achieves superior clustering performance compared to the baseline models on all datasets. An ablation study confirms the significant contribution of each component and identifies the most impactful one.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=114&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"08e487c1fdf8e6e964c20ddc0c472b66556aaf17\",\n",
      "        \"title\": \"Quantifying the impact of uninformative features on the performance of supervised classification and dimensionality reduction algorithms\",\n",
      "        \"abstract\": \"Machine learning approaches have become critical tools in data mining and knowledge discovery, especially when attempting to uncover relationships in high-dimensional data. However, researchers have noticed that a large fraction of features in high-dimensional datasets are commonly uninformative (too noisy or irrelevant). Because optimal feature selection is an NP-hard task, it is essential to understand how uninformative features impact the performance of machine learning algorithms. Here, we conduct systematic experiments on algorithms from a wide range of taxonomy families using synthetic datasets with different numbers of uninformative features and different numbers of patterns to be learned. Upon visual inspection, we classify these algorithms into four groups with varying robustness against uninformative features. For the algorithms in three of the groups, we find that when the number of uninformative features exceeds the number of data instances per pattern to be learned, the algorithms fail to learn the patterns. Finally, we investigate whether increasing the distinguishability of patterns or adding training instances can mitigate the effect of uninformative features. Surprisingly, we find that uninformative features still cause algorithms to suffer big losses in performance, even when patterns should be easily distinguishable. Analyses of real-world data show that our conclusions hold beyond the synthetic datasets we study systematically.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=115&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0edf94bfa1919fbf0dd5eebaaca37bc99a8515a5\",\n",
      "        \"title\": \"High-capacity sample multiplexing for single cell chromatin accessibility profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=116&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17acf471a92fb3014696179342bba660f465dc23\",\n",
      "        \"title\": \"SingleScan: a comprehensive resource for single-cell sequencing data processing and mining\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=117&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"13f7be85402225c13e2270b064c1d54fbdbeeb21\",\n",
      "        \"title\": \"Systems immunology spanning tumors, lymph nodes, and periphery\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=118&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f6384e9c45fff6f0222a6e17bde48c5f163944a0\",\n",
      "        \"title\": \"Recent advances and future prospects in direct cardiac reprogramming\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=119&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a67c2acaff9ead9be3f999d9a968431531145cdf\",\n",
      "        \"title\": \"Single-cell transcriptomics identifies adipose tissue CD271+ progenitors for enhanced angiogenesis in limb ischemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=120&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c88405295aca5d458d38062f5cec09e5122fc7e2\",\n",
      "        \"title\": \"Differential in vivo labeling with barcoded antibodies allows for simultaneous transcriptomic profiling of airway, lung tissue and intravascular immune cells\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) is the state-of-the-art approach to study transcriptomic signatures in individual cells in respiratory health and disease. However, classical scRNA-seq approaches provide no spatial information and are performed using either bronchoalveolar lavage fluid (BAL) or lung single cell suspensions to assess transcript levels in airway and tissue immune cells, respectively. Herein we describe a simple method to simultaneously characterize transcriptomic features of airway, lung parenchymal and intravascular immune cells based on differential in vivo labeling with barcoded antibodies. In addition to gaining basic spatial information, this approach allows for direct comparison of cells within different anatomical compartments. Furthermore, this method provides a time- and cost-effective alternative to classical scRNA-seq where lung and BAL samples are processed individually, reducing animal and reagent use. We demonstrate the feasibility of this approach in a preclinical mouse model of bacterial lung infection comparing airway, parenchymal and vasculature neutrophils early after infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=121&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"907864b6d4601e1812130db32895864a74a1bea2\",\n",
      "        \"title\": \"Deciphering tumor-infiltrating dendritic cells in the single-cell era\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=122&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6ee11ba0f40cbbd768f28de5912e890b93e41583\",\n",
      "        \"title\": \"Fragment-sequencing unveils local tissue microenvironments at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=123&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e194ced4ad6789e9affc9b74f7c7ac6562a04123\",\n",
      "        \"title\": \"Beyond single cells: microfluidics empowering multiomics analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=124&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5b7742e952f19a4f3478f5ca258b3269c83ee568\",\n",
      "        \"title\": \"Simultaneous Profiling of Chromosome Conformation and Gene Expression in Single Cells\",\n",
      "        \"abstract\": \"Rapid development in single-cell chromosome conformation capture technologies has provided valuable insights into the importance of spatial genome architecture for gene regulation. However, a long-standing technical gap remains in the simultaneous characterization of three-dimensional genomes and transcriptomes in the same cell. We have described an assay named Hi-C and RNA-seq employed simultaneously (HiRES), which integrates in situ reverse transcription and chromosome conformation capture (3C) for the parallel analysis of chromatin organization and gene expression. Here, we provide a detailed implementation of the assay, using mouse embryos and cerebral cortices as examples. The versatility of this method extends beyond these two samples, with the potential to be used in various other cell types. Key features \\u2022 A multi-omics sequencing approach to profile 3D genome structure and gene expression simultaneously in single cells. \\u2022 Compatible with animal tissues. \\u2022 One-tube amplification of both DNA and RNA components. \\u2022 Requires three days to complete.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=125&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fdd41eb63f2560e1f1e85995f238894cf6eee42c\",\n",
      "        \"title\": \"The transcriptional and phenotypic characteristics that define alveolar macrophage subsets in acute hypoxemic respiratory failure\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=126&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc51366d62c303e0d0ba51da03f6bf0c256bc5e3\",\n",
      "        \"title\": \"The use of gene expression datasets in feature selection research: 20\\u2009years of inherent bias?\",\n",
      "        \"abstract\": \"Feature selection algorithms are frequently employed in preprocessing machine learning pipelines applied to biological data to identify relevant features. The use of feature selection in gene expression studies began at the end of the 1990s with the analysis of human cancer microarray datasets. Since then, gene expression technology has been perfected, the Human Genome Project has been completed, new microarray platforms have been created and discontinued, and RNA\\u2010seq has gradually replaced microarrays. However, most feature selection methods in the last two decades were designed, evaluated, and validated on the same datasets from the microarray technology's infancy. In this review of over 1200 publications regarding feature selection and gene expression, published between 2010 and 2020, we found that 57% of the publications used at least one outdated dataset, 23% used only outdated data, and 32% did not cite data sources. Other issues include referencing databases that are no longer available, the slow adoption of RNA\\u2010seq datasets, and bias toward human cancer data, even for methods designed for a broader scope. In the most popular datasets, some being 23\\u2009years old, mislabeled samples, experimental biases, distribution shifts, and the absence of classification challenges are common. These problems are more predominant in publications with computer science backgrounds compared to publications from biology and can lead to inaccurate and misleading biological results.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=127&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7a4a460e6979c8a40ed9229e7f2454a0f92de3cb\",\n",
      "        \"title\": \"Enhancing Single-Cell Western Blotting Sensitivity Using Diffusive Analyte Blotting and Antibody Conjugate Amplification.\",\n",
      "        \"abstract\": \"While there are many techniques to achieve highly sensitive, multiplex detection of RNA and DNA from single cells, detecting protein content often suffers from low limits of detection and throughput. Miniaturized, high-sensitivity Western blots on single cells (scWesterns) are attractive because they do not require advanced instrumentation. By physically separating analytes, scWesterns also uniquely mitigate limitations to target protein multiplexing posed by the affinity reagent performance. However, a fundamental limitation of scWesterns is their limited sensitivity for detecting low-abundance proteins, which arises from transport barriers posed by the separation gel against detection species. Here we address the sensitivity by decoupling the electrophoretic separation medium from the detection medium. We transfer scWestern separations to a nitrocellulose blotting medium with distinct mass transfer advantages over traditional in-gel probing, yielding a 5.9-fold improvement in the limit of detection. We next amplify probing of blotted proteins with enzyme-antibody conjugates, which are incompatible with traditional in-gel probing to achieve further improvement in the limit of detection to 1000 molecules, a 120-fold improvement. This enables us to detect 100% of cells in an EGFP-expressing population using fluorescently tagged and enzyme-conjugated antibodies compared to 84.5% of cells using in-gel detection. These results suggest the compatibility of nitrocellulose-immobilized scWesterns with a variety of affinity reagents\\u2500not previously accessible for in-gel use\\u2500for further signal amplification and detection of low-abundance targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=128&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8562976230a6a84ec54122eb3d3ee5d9b4e18162\",\n",
      "        \"title\": \"S100A8\\u2010enriched microglia populate the brain of tau\\u2010seeded and accelerated aging mice\",\n",
      "        \"abstract\": \"Long considered to fluctuate between pro- and anti-inflammatory states, it has now become evident that microglia occupy a variegated phenotypic landscape with relevance to aging and neurodegeneration. However, whether specific microglial subsets converge in or contribute to both processes that eventually affect brain function is less clear. To investigate this, we analyzed microglial heterogeneity in a tauopathy mouse model (K18-seeded P301L) and an accelerated aging model (senescence accelerated mouse prone 8, SAMP8) using cellular indexing of transcriptomes and epitopes by sequencing. We found that widespread tau pathology in K18-seeded P301L mice caused a significant change in the number and morphology of microglia, but only a mild overrepresentation of disease-associated microglia. At the cell population-level, we observed a marked upregulation of the calprotectin-encoding genes S100a8 and S100a9. In 9-months-old SAMP8 mice, we identified a unique microglial subpopulation that showed partial similarity with the disease-associated microglia phenotype and was additionally characterized by a high expression of the same calprotectin gene set. Immunostaining for S100A8 revealed that this population was enriched in the hippocampus, correlating with the cognitive impairment observed in this model. However, incomplete colocalization between their residence and markers of neuronal loss suggests regional specificity. Importantly, S100A8-positive microglia were also retrieved in brain biopsies of human AD and tauopathy patients as well as in a biopsy of an aged individual without reported pathology. Thus, the emergence of S100A8-positive microglia portrays a conspicuous commonality between accelerated aging and tauopathy progression, which may have relevance for ensuing brain dysfunction. Graphical abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=129&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4d39a038b9832742cdde827e4ba27c85352b4d34\",\n",
      "        \"title\": \"SPDB: a comprehensive resource and knowledgebase for proteomic data at the single-cell resolution\",\n",
      "        \"abstract\": \"Abstract The single-cell proteomics enables the direct quantification of protein abundance at the single-cell resolution, providing valuable insights into cellular phenotypes beyond what can be inferred from transcriptome analysis alone. However, insufficient large-scale integrated databases hinder researchers from accessing and exploring single-cell proteomics, impeding the advancement of this field. To fill this deficiency, we present a comprehensive database, namely Single-cell Proteomic DataBase (SPDB, https://scproteomicsdb.com/), for general single-cell proteomic data, including antibody-based or mass spectrometry-based single-cell proteomics. Equipped with standardized data process and a user-friendly web interface, SPDB provides unified data formats for convenient interaction with downstream analysis, and offers not only dataset-level but also protein-level data search and exploration capabilities. To enable detailed exhibition of single-cell proteomic data, SPDB also provides a module for visualizing data from the perspectives of cell metadata or protein features. The current version of SPDB encompasses 133 antibody-based single-cell proteomic datasets involving more than 300 million cells and over 800 marker/surface proteins, and 10 mass spectrometry-based single-cell proteomic datasets involving more than 4000 cells and over 7000 proteins. Overall, SPDB is envisioned to be explored as a useful resource that will facilitate the wider research communities by providing detailed insights into proteomics from the single-cell perspective.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=130&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"050c2c5d16d078ae203f939fe649a5efa8db7261\",\n",
      "        \"title\": \"Footprint of pancreas infiltrating and circulating immune cells throughout type 1 diabetes development\",\n",
      "        \"abstract\": \"Introduction Type 1 diabetes (T1D) is defined by immune cell infiltration of the pancreas, in particular the islets of Langerhans, referred to as insulitis, which is especially prominent during the early disease stages in association with decreased beta cell mass. An in-depth understanding of the dynamics and phenotype of the immune cells infiltrating the pancreas and the accompanying changes in their profiles in peripheral blood during T1D development is critical to generate novel preventive and therapeutic approaches, as well as to find biomarkers for the disease process. Methods Using multi-parameter flow cytometry, we explored the dynamic changes of immune cells infiltrating the pancreas and the pancreatic draining lymph nodes (PLN), compared to those in peripheral blood in female and male non-obese diabetic (NOD) mice during T1D progression. Results The early stages of T1D development were characterized by an influx of innate dendritic cells and neutrophils in the pancreas. While dendritic cells seemed to move in and out (to the PLN), neutrophils accumulated during the pre-symptomatic phase and reached a maximum at 8 weeks of age, after which their numbers declined. During disease progression, CD4+ and CD8+ T cells appeared to continuously migrate from the PLN to the pancreas, which coincided with an increase in beta cell autoimmunity and insulitis severity, and a decline in insulin content. At 12 weeks of age, CD4+ and especially CD8+ T cells in the pancreas showed a dramatic shift from na\\u00efve to effector memory phenotype, in contrast to the PLN, where most of these cells remained na\\u00efve. A large proportion of pancreas infiltrating CD4+ T cells were na\\u00efve, indicating that antigenic stimulation was not necessary to traffic and invade the pancreas. Interestingly, a pre-effector-like T cell dominated the peripheral blood. These cells were intermediates between na\\u00efve and effector memory cells as identified by single cell RNA sequencing and might be a potential novel therapeutic target. Conclusion These time- and tissue-dependent changes in the dynamics and functional states of CD4+ and CD8+ T cells are essential steps in our understanding of the disease process in NOD mice and need to be considered for the interpretation and design of disease-modifying therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=131&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"65de58c8c7c3c6e1a30672362d403010133d5d27\",\n",
      "        \"title\": \"Insights gained from single-cell analysis of chimeric antigen receptor T-cell immunotherapy in cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=132&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5b4c63c66c2199e6439d8a7eafc0e320bfa1f20b\",\n",
      "        \"title\": \"A discrete \\u2018early-responder\\u2019 stromal-cell subtype orchestrates immunocyte recruitment to injured tissue\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=133&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"93f58c21bccff103fca3d18829a3cfbcaf00fb10\",\n",
      "        \"title\": \"A holistic approach to understanding immune-mediated inflammatory diseases: bioinformatic tools to integrate omics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=134&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"22d509f45712a799b31d9f381f463d741c57182a\",\n",
      "        \"title\": \"scapGNN: A graph neural network\\u2013based framework for active pathway and gene module inference from single-cell multi-omics data\",\n",
      "        \"abstract\": \"Although advances in single-cell technologies have enabled the characterization of multiple omics profiles in individual cells, extracting functional and mechanistic insights from such information remains a major challenge. Here, we present scapGNN, a graph neural network (GNN)-based framework that creatively transforms sparse single-cell profile data into the stable gene\\u2013cell association network for inferring single-cell pathway activity scores and identifying cell phenotype\\u2013associated gene modules from single-cell multi-omics data. Systematic benchmarking demonstrated that scapGNN was more accurate, robust, and scalable than state-of-the-art methods in various downstream single-cell analyses such as cell denoising, batch effect removal, cell clustering, cell trajectory inference, and pathway or gene module identification. scapGNN was developed as a systematic R package that can be flexibly extended and enhanced for existing analysis processes. It provides a new analytical platform for studying single cells at the pathway and network levels.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=135&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6221d2c75057b3ff85c854e6c978b1efbb2c4292\",\n",
      "        \"title\": \"Mechanisms of multi-omics and network pharmacology to explain traditional chinese medicine for vascular cognitive impairment: A narrative review.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=136&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b04abf261511754198855a6745e7a5a386a5a32c\",\n",
      "        \"title\": \"Simultaneous deep transcriptome and proteome profiling in a single mouse oocyte.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=137&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cb9d3a7c4985a14079d6dc7f87bbb873bcc11a90\",\n",
      "        \"title\": \"Next-generation forward genetic screens: uniting high-throughput perturbations with single-cell analysis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=138&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"94dbd5f6a76a9343ac4e55fbcb30f03c148ed915\",\n",
      "        \"title\": \"Predicting patient outcomes after treatment with immune checkpoint blockade: A review of biomarkers derived from diverse data modalities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=139&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6127216893bc310a3582e53555a11819bfd63637\",\n",
      "        \"title\": \"A simple joint detection platform for high-throughput single-cell heterogeneity screening.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=140&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"64becb6e231a2b170e5696a0ee25560522644fd3\",\n",
      "        \"title\": \"Deciphering single-cell protein secretion and gene expressions by constructing cell-antibody conjugates.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=141&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3903803f0eeeb05a54a857f2e6068657008a81b7\",\n",
      "        \"title\": \"Dissecting viral infections, one cell at a time, by single-cell technologies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=142&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ad9b1d80b9afe42565037e0637717fa8287167a3\",\n",
      "        \"title\": \"Revealing spatial multimodal heterogeneity in tissues with SpaTrio\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=143&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd4b309a1f6cc9a6a681db0325789431fcc886f3\",\n",
      "        \"title\": \"Early cellular mechanisms of type I interferon-driven susceptibility to tuberculosis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=144&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"716d7e0b5972b0c0a0270ad68efd1468857e4252\",\n",
      "        \"title\": \"CRISPR screening in hematology research: from bulk to single-cell level\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=145&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b27eb8af433b707dd8ff95d5d2df995b2a1f0fe\",\n",
      "        \"title\": \"Systematic evaluation of AML-associated antigens identifies anti-U5 SNRNP200 therapeutic antibodies for the treatment of acute myeloid leukemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=146&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cdd9b1fbe2d9b52dc67e5c39d3f24ae16b886fe4\",\n",
      "        \"title\": \"Generation of circulating autoreactive pre-plasma cells fueled by na\\u00efve B cells in celiac disease\",\n",
      "        \"abstract\": \"Autoantibodies against the enzyme transglutaminase 2 (TG2) are characteristic of celiac disease (CeD), and TG2-specific IgA plasma cells are abundant in gut biopsies of patients. We here describe the corresponding population of autoreactive B cells in blood. Circulating TG2-specific IgA cells were found in untreated patients on a gluten-containing diet but not in controls. They were clonally related to TG2-specific small intestinal plasma cells, and they expressed gut-homing molecules, indicating that they are plasma cell precursors. Unlike other IgA-switched cells, the TG2-specific cells were negative for CD27, placing them in the double negative (IgD-CD27-) category. They had a plasmablast or activated memory B-cell phenotype, and they harbored fewer variable region mutations than other IgA cells. Based on their similarity to na\\u00efve B cells, we propose that autoreactive IgA cells in CeD are generated mainly through chronic recruitment of na\\u00efve B cells via an extrafollicular response involving gluten-specific CD4+ T cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=147&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9607aefd0333a207a26a36b8e6e4912023a5870\",\n",
      "        \"title\": \"Surfaceome analysis of extracellular vesicles from senescent cells uncovers uptake repressor DPP4\",\n",
      "        \"abstract\": \"Significance Vast numbers of senescence-associated EVs (S-EVs) are released and carry different contents than EVs from proliferating cells (P-EVs), suggesting that they may harbor important factors critical for the senescence microenvironment and pathologies associated with age. We characterized the proteins, especially surface proteins, on P-EVs and S-EVs, and identified DPP4 as a specific marker on the surface of S-EVs. Although S-EVs were taken up by macrophages, they were refractory to uptake by many other cell types, including cancer cells. DPP4 on the surface of S-EVs appeared to function as a signal to prevent S-EVs internalization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=148&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cb29261a346a4c5f0c5d6b30dc8e36607850b56e\",\n",
      "        \"title\": \"Gene regulatory network reconstruction: harnessing the power of single-cell multi-omic data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=149&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ebe30368cb4aa8408b0f957a1cb1d5b3482fc532\",\n",
      "        \"title\": \"CellSTAR: a comprehensive resource for single-cell transcriptomic annotation\",\n",
      "        \"abstract\": \"Abstract Large-scale studies of single-cell sequencing and biological experiments have successfully revealed expression patterns that distinguish different cell types in tissues, emphasizing the importance of studying cellular heterogeneity and accurately annotating cell types. Analysis of gene expression profiles in these experiments provides two essential types of data for cell type annotation: annotated references and canonical markers. In this study, the first comprehensive database of single-cell transcriptomic annotation resource (CellSTAR) was thus developed. It is unique in (a) offering the comprehensive expertly annotated reference data for annotating hundreds of cell types for the first time and (b) enabling the collective consideration of reference data and marker genes by incorporating tens of thousands of markers. Given its unique features, CellSTAR is expected to attract broad research interests from the technological innovations in single-cell transcriptomics, the studies of cellular heterogeneity & dynamics, and so on. It is now publicly accessible without any login requirement at: https://idrblab.org/cellstar.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=150&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"741a80b104d4b5345d17f5af3879fbdbd5091094\",\n",
      "        \"title\": \"Cardiac cellular diversity and functionality in cardiac repair by single-cell transcriptomics\",\n",
      "        \"abstract\": \"Cardiac repair after myocardial infarction (MI) is orchestrated by multiple intrinsic mechanisms in the heart. Identifying cardiac cell heterogeneity and its effect on processes that mediate the ischemic myocardium repair may be key to developing novel therapeutics for preventing heart failure. With the rapid advancement of single-cell transcriptomics, recent studies have uncovered novel cardiac cell populations, dynamics of cell type composition, and molecular signatures of MI-associated cells at the single-cell level. In this review, we summarized the main findings during cardiac repair by applying single-cell transcriptomics, including endogenous myocardial regeneration, myocardial fibrosis, angiogenesis, and the immune microenvironment. Finally, we also discussed the integrative analysis of spatial multi-omics transcriptomics and single-cell transcriptomics. This review provided a basis for future studies to further advance the mechanism and development of therapeutic approaches for cardiac repair.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=151&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2edc5189a9c39a5946767089cce6ef789891aa6a\",\n",
      "        \"title\": \"Identifying Cell Types in Single-Cell Multimodal Omics Data via Joint Embedding Learning\",\n",
      "        \"abstract\": \"Emerging single-cell technologies profile different modalities of data in the same cell, providing opportunities to study cellular population and cell development at a res-olution that was previously inaccessible. The first and most fundamental step in analyzing single-cell multimodal data is the identification of the cell types in the data using clustering analysis and classification. However, combining different data modalities for the classification task in multimodal data remains a computational challenge. We propose an approach for identifying cell types in multimodal omics data via joint dimensionality reduction. We first introduce a general framework that extends loss based dimensionality reduction methods such as nonnegative matrix factorization and UMAP to multimodal omics data. Our approach can learn the relative contribution of each modality to a concise representation of cellular identity that enhances discriminative features and decreases the effect of noisy features. The precise representation of the multimodal data in a low dimensional space improves the predictivity of classification methods. In our experiments using both synthetic and real data, we show that our framework produces unified embeddings that agree with known cell types and allows the predictive algorithms to annotate the cell types more accurately than state-of-the-art classification methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=152&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"729717bbd26f4afe1ec67816635597c33e75220e\",\n",
      "        \"title\": \"Complete miRNA-15/16 loss in mice promotes hematopoietic progenitor expansion and a myeloid-biased hyperproliferative state\",\n",
      "        \"abstract\": \"Significance Alterations in miR-15/16 can lead to hematological malignancies. Loss of a single cluster occurs frequently in chronic lymphocytic leukemia, but unexpectedly, loss of both clusters in mice promotes a myeloproliferative disorder resembling acute myeloid leukemia (AML). We show that the complete absence of miR-15/16 in multipotent progenitors (MPPs) results in expanded numbers and altered functions of this subpopulation of hematopoietic cells, contributing to myeloid-skewing and a hyperproliferative state. Since a molecular signature of miR-15/16 deletion in murine MPPs correlates with poor outcome in AML patients, our findings appear important for leukemia biology and as a potentially valuable biomarker in blood cancers. This study demonstrates that miR-15/16 plays an important role at the apex of hematopoiesis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=153&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0da9489c4f5ce0fc7d7f369e2df5e888012f0f4a\",\n",
      "        \"title\": \"Single-cell analysis technologies for cancer research: from tumor-specific single cell discovery to cancer therapy\",\n",
      "        \"abstract\": \"Single-cell sequencing (SCS) technology is changing our understanding of cellular components, functions, and interactions across organisms, because of its inherent advantage of avoiding noise resulting from genotypic and phenotypic heterogeneity across numerous samples. By directly and individually measuring multiple molecular characteristics of thousands to millions of single cells, SCS technology can characterize multiple cell types and uncover the mechanisms of gene regulatory networks, the dynamics of transcription, and the functional state of proteomic profiling. In this context, we conducted systematic research on SCS techniques, including the fundamental concepts, procedural steps, and applications of scDNA, scRNA, scATAC, scCITE, and scSNARE methods, focusing on the unique clinical advantages of SCS, particularly in cancer therapy. We have explored challenging but critical areas such as circulating tumor cells (CTCs), lineage tracing, tumor heterogeneity, drug resistance, and tumor immunotherapy. Despite challenges in managing and analyzing the large amounts of data that result from SCS, this technique is expected to reveal new horizons in cancer research. This review aims to emphasize the key role of SCS in cancer research and promote the application of single-cell technologies to cancer therapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=154&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f3f0d13b82d7893248e77d711213d4e1c80c14ea\",\n",
      "        \"title\": \"Single-cell sequencing technology applied to epigenetics for the study of tumor heterogeneity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=155&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"225fd694519e863b7d76d7043920c5337bb49d1b\",\n",
      "        \"title\": \"The latent HIV reservoir: current advances in genetic sequencing approaches\",\n",
      "        \"abstract\": \"ABSTRACT Multiple cellular HIV reservoirs in diverse anatomical sites can undergo clonal expansion and persist for years despite suppressive antiretroviral therapy, posing a major barrier toward an HIV cure. Commonly adopted assays to assess HIV reservoir size mainly consist of PCR-based measures of cell-associated total proviral DNA, intact proviruses and transcriptionally competent provirus (viral RNA), flow cytometry and microscopy-based methods to measure translationally competent provirus (viral protein), and quantitative viral outgrowth assay, the gold standard to measure replication-competent provirus; yet no assay alone can provide a comprehensive view of the total HIV reservoir or its dynamics. Furthermore, the detection of extant provirus by these measures does not preclude defects affecting replication competence. An accurate measure of the latent reservoir is essential for evaluating the efficacy of HIV cure strategies. Recent approaches have been developed, which generate proviral sequence data to create a more detailed profile of the latent reservoir. These sequencing approaches are valuable tools to understand the complex multicellular processes in a diverse range of tissues and cell types and have provided insights into the mechanisms of HIV establishment and persistence. These advancements over previous sequencing methods have allowed multiplexing and new assays have emerged, which can document transcriptional activity, chromosome accessibility, and in-depth cellular phenotypes harboring latent HIV, enabling the characterization of rare infected cells across restrictive sites such as the brain. In this manuscript, we provide a review of HIV sequencing-based assays adopted to address challenges in quantifying and characterizing the latent HIV reservoir.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=156&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"efa0f01b8c9533aff5455bdf8dafd802b533a326\",\n",
      "        \"title\": \"Systematic identification of gene combinations to target in innate immune cells to enhance T cell activation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=157&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"15ce7dd674754feb7026ebe9ffb62a8a98da2e2c\",\n",
      "        \"title\": \"Sequencing-based methods for single-cell multi-omics studies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=158&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a2f4e636c59805915d3273e7feceff13f15b097b\",\n",
      "        \"title\": \"Single-cell sequencing, spatial transcriptome ad periodontitis: Rethink pathogenesis and classification.\",\n",
      "        \"abstract\": \"OBJECTIVE\\nThis narrative review illuminates on the application of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) in periodontitis and highlights the probability of relating cell population and gene signatures to the pathogenesis of the disease for a better diagnosis.\\n\\n\\nMETHODS\\nAn electronic search of the literature in the PubMed database for the keywords, \\\"single cell sequencing\\\" OR \\\"spatial transcriptomics\\\" and \\\"periodontitis\\\" OR \\\"gingiva\\\" OR \\\"oral mucosa\\\" yielded 486 research articles and reviews. After filtering duplicates and careful curation, 22 papers conducted in humans were retained.\\n\\n\\nRESULTS\\nThe molecular mechanisms underlying periodontitis are complex and involve the interaction of multiple cells and various gene expressions. Most residing cells in periodontal tissues participate in maintaining homeostasis and health, while in addition to infiltrating immune cells contribute to the fight against the bacterial insult.\\n\\n\\nCONCLUSION\\nscRNA-seq and ST have provided new insights into the cellular and molecular changes associated with periodontitis for a better diagnosis and clinical outcome. New functions of cells and genes are revealed with these techniques; however, no cells or gene signatures are attributed to periodontitis so far.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=159&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4aa73cdb819d4a8d6b0d0d3a34371dc3c0f8b9b6\",\n",
      "        \"title\": \"Spatial transcriptomics in neuroscience\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=160&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"721ae67a7b818bcbaff49304874dc106ef0ea620\",\n",
      "        \"title\": \"Single-cell and spatial analyses reveal a tradeoff between murine mammary proliferation and lineage programs associated with endocrine cues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=161&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e5160f6aafbf28a3704b5c1b93cf0f679a19e16a\",\n",
      "        \"title\": \"Emerging multianalyte biosensors for the simultaneous detection of protein and nucleic acid biomarkers.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=162&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3c8a1a747a76a03f4cffe814b990e720fc2752d3\",\n",
      "        \"title\": \"Comparison of high-throughput single-cell RNA-seq methods for ex vivo drug screening\",\n",
      "        \"abstract\": \"Functional precision medicine (FPM) aims to optimize patient-specific drug selection based on the unique characteristics of their cancer cells. Recent advancements in high throughput ex vivo drug profiling have accelerated interest in FPM. Here, we present a proof-of-concept study for an integrated experimental system that incorporates ex vivo treatment response with a single-cell gene expression output enabling barcoding of several drug conditions in one single-cell sequencing experiment. We demonstrate this through a proof-of-concept investigation focusing on the glucocorticoid-resistant acute lymphoblastic leukemia (ALL) E/R+ Reh cell line. Three different single-cell transcriptome sequencing (scRNA-seq) approaches were evaluated, each exhibiting high cell recovery and accurate tagging of distinct drug conditions. Notably, our comprehensive analysis revealed variations in library complexity, sensitivity (gene detection), and differential gene expression detection across the methods. Despite these differences, we identified a substantial transcriptional response to fludarabine, a highly relevant drug for treating high-risk ALL, which was consistently recapitulated by all three methods. These findings highlight the potential of our integrated approach for studying drug responses at the single-cell level and emphasize the importance of method selection in scRNA-seq studies. Finally, our data encompassing 27,327 cells are freely available to extend to future scRNA-seq methodological comparisons.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=163&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1510c7b89ff0e9d80ef16472d7e32baacc8b2f55\",\n",
      "        \"title\": \"Flow cytometry analysis of protein expression using antibody\\u2010derived tags followed by CITE\\u2010Seq\",\n",
      "        \"abstract\": \"Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE\\u2010Seq) is a single\\u2010cell phenotyping method that uses antibody\\u2010derived tags (ADTs) to quantitatively detect cell surface protein expression and generate transcriptomic data at the single\\u2010cell level. Despite the increased popularity of this technique to study cellular heterogeneity and dynamics, detailed methods on how to choose ADT markers and ensuring reagent performance in biological relevant systems prior to sequencing is not available. Here we describe a novel and easy\\u2010to\\u2010use multiplex flow proxy assay in which multiple protein markers can be measured simultaneously using a combination of ADT reagents and dye\\u2010oligo conjugates by flow cytometry. Using dye\\u2010oligo conjugates with sequences complementary to the ADT reagents, we can achieve specific binding and evaluate protein marker expression in a multiplex way. This quality control assay is useful for guiding ADT marker choice and confirming protein expression prior to sequencing. Importantly, the labeled cells can be directly isolated based on the specific fluorescence from dye\\u2010oligo conjugates using a flow cytometry cell sorter and processed for downstream single\\u2010cell multiomics. Using this streamlined workflow, we sorted natural killer cells and T cells efficiently using only ADT and dye\\u2010oligo reagents, avoiding the possibility of decreased marker resolution from co\\u2010staining cells with ADT and fluorescent antibodies. This novel workflow provides a viable option for improving ADT marker choice and cell sorting efficiency, allowing subsequent CITE\\u2010Seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=164&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"778741373bebed817ea0c9d686ce7947b42ca719\",\n",
      "        \"title\": \"FEAST: A flow cytometry-based toolkit for interrogating microglial engulfment of synaptic and myelin proteins\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=165&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2eceb1880a0ad7d511fef0d4c8b28e43a4f595db\",\n",
      "        \"title\": \"Liquid biopsy: creating opportunities in brain space\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=166&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b8bd08dee73190e11d52e9942bfb613b978d5853\",\n",
      "        \"title\": \"Sample enrichment for single-nucleus sequencing using concanavalin A-conjugated magnetic beads\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=167&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"615f274cbb56add5fcdb16648e44e0af1020cc9f\",\n",
      "        \"title\": \"Multimodal single-cell datasets characterize antigen-specific CD8+ T cells across SARS-CoV-2 vaccination and infection\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=168&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"510d3cffac0140584f027275ab28f02b0d4e416f\",\n",
      "        \"title\": \"DISCERN: deep single-cell expression reconstruction for improved cell clustering and cell subtype and state detection\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=169&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"be0729438d238df2722af1780a830fa97a2a667f\",\n",
      "        \"title\": \"Baseline immune states (BIS) associated with vaccine responsiveness and factors that shape the BIS.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=170&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"38649e115328dfe8becd96a613be3ef8fb60555f\",\n",
      "        \"title\": \"Integration of spatial and single-cell data across modalities with weakly linked features.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=171&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c36edf4981f2bea447fa6ae6a6864b61b9d0fb04\",\n",
      "        \"title\": \"scDecouple: decoupling cellular response from infected proportion bias in scCRISPR-seq\",\n",
      "        \"abstract\": \"Motivation scCRISPR-seq is an emerging high-throughput CRISPR screening technology that combines CRISPR screening with single-cell sequencing technologies. It provides rich information on the mechanism of gene regulation. However, when scCRISPR-seq is applied in a population of heterogeneous cells, the true cellular response to perturbation is coupled with infected proportion bias of guide RNAs (gRNAs) across different cell clusters. The mixing of these effects introduces noise into scCRISPR-seq data analysis and thus obstacles to relevant studies. Results We developed scDecouple to decouple true cellular response of perturbation from the influence of infected proportion bias. scDecouple first models the distribution of gene expression profiles in perturbed cells and then iteratively finds the maximum likelihood of cell cluster proportions as well as the cellular response for each gRNA. We demonstrated its performance in a series of simulation experiments. By applying scDecouple to real scCRISPR-seq data, we found that scDecouple enhances the identification of biologically perturbation-related genes. scDecouple can benefit scCRISPR-seq data analysis, especially in the case of heterogeneous samples or complex gRNA libraries. Availability scDecouple is freely available at https://github.com/MengQiuchen/scDecouple. Contact weilei92@tsinghua.edu.cn, yinqingl@tsinghua.edu.cn\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=172&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a352bfca37f2a020b6091da13d2d08fc2bd37dcb\",\n",
      "        \"title\": \"Single-cell RNA sequencing technology in human spermatogenesis: Progresses and perspectives\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=173&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe8d2224e980facd501628f4f413d784dbdeee3e\",\n",
      "        \"title\": \"Mechanisms of kidney fibrosis and routes towards therapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=174&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b8c2ec61be676888f54b53cbdb50de7b4fa2aae4\",\n",
      "        \"title\": \"A Roadmap for a Consensus Human Skin Cell Atlas and Single-Cell Data Standardization.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=175&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"36243fab94ffe699e924912d354673042fc35a42\",\n",
      "        \"title\": \"Integrated Multimodal Omics and Dietary Approaches for the Management of Neurodegeneration\",\n",
      "        \"abstract\": \"Neurodegenerative diseases, such as Alzheimer\\u2019s disease and Parkinson\\u2019s disease, are caused by a combination of multiple events that damage neuronal function. A well-characterized biomarker of neurodegeneration is the accumulation of proteinaceous aggregates in the brain. However, the gradually worsening symptoms of neurodegenerative diseases are unlikely to be solely due to the result of a mutation in a single gene, but rather a multi-step process involving epigenetic changes. Recently, it has been suggested that a fraction of epigenetic alternations may be correlated to neurodegeneration in the brain. Unlike DNA mutations, epigenetic alterations are reversible, and therefore raise the possibilities for therapeutic intervention, including dietary modifications. Additionally, reactive oxygen species may contribute to the pathogenesis of Alzheimer\\u2019s disease and Parkinson\\u2019s disease through epigenetic alternation. Given that the antioxidant properties of plant-derived phytochemicals are likely to exhibit pleiotropic effects against ROS-mediated epigenetic alternation, dietary intervention may be promising for the management of neurodegeneration in these diseases. In this review, the state-of-the-art applications using single-cell multimodal omics approaches, including epigenetics, and dietary approaches for the identification of novel biomarkers and therapeutic approaches for the treatment of neurodegenerative diseases are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=176&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f31fb94b272c12cc40bbada60fc07a87565fd794\",\n",
      "        \"title\": \"Identification of cell-type-specific genes in multimodal single-cell data using deep neural network algorithm\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=177&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f140990f353cdd1ffeb89ac6a15a1d75c29db786\",\n",
      "        \"title\": \"Non-Mass Spectrometric Targeted Single-Cell Metabolomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=178&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"24a7e63887407c565fd0111dbb56c13e17e8c1ee\",\n",
      "        \"title\": \"Decoding Human Biology and Disease Using Single-cell Omics Technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=179&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"385db8da8e48e68b85238a181f4adbefa75679a3\",\n",
      "        \"title\": \"DNA-GPS: A theoretical framework for optics-free spatial genomics and synthesis of current methods.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=180&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"64526f4fa1db4e38687303d509a2fcc12c90662f\",\n",
      "        \"title\": \"Barcoding intracellular reverse transcription enables high-throughput phenotype-coupled T cell receptor analyses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=181&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"29ceec90b1bb8a5396c087649415ee37dac2c855\",\n",
      "        \"title\": \"Identification, sorting and profiling of functional killer cells via the capture of fluorescent target-cell lysate\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=182&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ce5b2cfc1f04cd03bd105e0ed0a0ae48d9eaa2b6\",\n",
      "        \"title\": \"Changes in subcutaneous white adipose tissue cellular composition and molecular programs underlie glucose intolerance in persons with HIV\",\n",
      "        \"abstract\": \"Introduction Subcutaneous adipose tissue (SAT) is a critical regulator of systemic metabolic homeostasis. Persons with HIV (PWH) have an increased risk of metabolic diseases and significant alterations in the SAT immune environment compared with the general population. Methods We generated a comprehensive single-cell multi-omic SAT atlas to characterize cellular compositional and transcriptional changes in 59 PWH across a spectrum of metabolic health. Results Glucose intolerance was associated with increased lipid-associated macrophages, CD4+ and CD8+ T effector memory cells, and decreased perivascular macrophages. We observed a coordinated intercellular regulatory program which enriched for genes related to inflammation and lipid-processing across multiple cell types as glucose intolerance increased. Increased CD4+ effector memory tissue-resident cells most strongly associated with altered expression of adipocyte genes critical for lipid metabolism and cellular regulation. Intercellular communication analysis demonstrated enhanced pro-inflammatory and pro-fibrotic signaling between immune cells and stromal cells in PWH with glucose intolerance compared with non-diabetic PWH. Lastly, while cell type-specific gene expression among PWH with diabetes was globally similar to HIV-negative individuals with diabetes, we observed substantially divergent intercellular communication pathways. Discussion These findings suggest a central role of tissue-resident immune cells in regulating SAT inflammation among PWH with metabolic disease, and underscore unique mechanisms that may converge to promote metabolic disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=183&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c998492ce0513f8bdc2e8960dc87cca2ae228dc\",\n",
      "        \"title\": \"Multiplexing cortical brain organoids for the longitudinal dissection of developmental traits at single cell resolution\",\n",
      "        \"abstract\": \"The combination of brain organoid and single cell omic technologies holds transformative potential to dissect human neurobiology at high resolution and with mechanistic precision. Delivering this promise in the context of human neurodiversity, physiological and pathological alike, requires however a major leap in scalability, given the need for experimental designs that include multiple individuals and, prospectively, population cohorts. To lay the foundation for this, we implemented and benchmarked complementary strategies to multiplex brain organoids. Following an extended longitudinal design with a uniquely informative set of timepoints, we pooled cells from different induced pluripotent stem cell lines either during organoids generation (upstream multiplexing in mosaic models) or before single cell-RNAseq library preparation (downstream multiplexing). We developed a new method, SCanSNP, and an aggregated call to deconvolve organoids cell identities, overcoming current criticalities in doublets prediction and low quality cells identification and improving accuracy over state of the art algorithms. Integrating single cell transcriptomes and analysing cell types across neurodevelopmental stages and multiplexing modalities, we validated the feasibility of both multiplexing methods in charting neurodevelopmental trajectories at high resolution, linking their specificity to genetic variation between individual lines. Together, this multiplexing suite of experimental and computational methods provides an enabling resource for disease modelling at scale and paves the way towards an in vitro epidemiology paradigm.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=184&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d471a7b78d029ef66cca8d22014df53b92e2c51a\",\n",
      "        \"title\": \"How single-cell techniques help us look into lung cancer heterogeneity and immunotherapy\",\n",
      "        \"abstract\": \"Lung cancer patients tend to have strong intratumoral and intertumoral heterogeneity and complex tumor microenvironment, which are major contributors to the efficacy of and drug resistance to immunotherapy. From a new perspective, single-cell techniques offer an innovative way to look at the intricate cellular interactions between tumors and the immune system and help us gain insights into lung cancer and its response to immunotherapy. This article reviews the application of single-cell techniques in lung cancer, with focuses directed on the heterogeneity of lung cancer and the efficacy of immunotherapy. This review provides both theoretical and experimental information for the future development of immunotherapy and personalized treatment for the management of lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=185&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d360b66147861f435f9206499506d717ed2f84ef\",\n",
      "        \"title\": \"Spatial transcriptomics: recent developments and insights in respiratory research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=186&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2df9ce2919bb07656a86cae4b08c843720c1622d\",\n",
      "        \"title\": \"CellBender removes technical artifacts from single-cell RNA sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=187&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d98e718dd53f4b1d8d84e5893ad296938117a2fa\",\n",
      "        \"title\": \"Resolving the spatial architecture of myeloma and its microenvironment at the single-cell level\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=188&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b6102407a1712207ca92e57d47d374374489dca6\",\n",
      "        \"title\": \"Platelet factors attenuate inflammation and rescue cognition in ageing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=189&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"27103c8d17be52494d94c6136735e7b715e647b9\",\n",
      "        \"title\": \"Single-cell multiomic analysis of thymocyte development reveals drivers of CD4+ T cell and CD8+ T cell lineage commitment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=190&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e0853361e076c797e482f4f78df3e3c9a692dc7c\",\n",
      "        \"title\": \"Dynamic thresholding and tissue dissociation optimization for CITE-seq identifies differential surface protein abundance in metastatic melanoma\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=191&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"83b425a44b0d6efda3a55b2203901e8c63d97daa\",\n",
      "        \"title\": \"Omics data integration in computational biology viewed through the prism of machine learning paradigms\",\n",
      "        \"abstract\": \"Important quantities of biological data can today be acquired to characterize cell types and states, from various sources and using a wide diversity of methods, providing scientists with more and more information to answer challenging biological questions. Unfortunately, working with this amount of data comes at the price of ever-increasing data complexity. This is caused by the multiplication of data types and batch effects, which hinders the joint usage of all available data within common analyses. Data integration describes a set of tasks geared towards embedding several datasets of different origins or modalities into a joint representation that can then be used to carry out downstream analyses. In the last decade, dozens of methods have been proposed to tackle the different facets of the data integration problem, relying on various paradigms. This review introduces the most common data types encountered in computational biology and provides systematic definitions of the data integration problems. We then present how machine learning innovations were leveraged to build effective data integration algorithms, that are widely used today by computational biologists. We discuss the current state of data integration and important pitfalls to consider when working with data integration tools. We eventually detail a set of challenges the field will have to overcome in the coming years.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=192&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c096af5d1f0b5c2b419c1bba44c4a6547f0c736f\",\n",
      "        \"title\": \"Single-cell transcriptomics identifies prothymosin \\u03b1 restriction of HIV-1 in vivo\",\n",
      "        \"abstract\": \"Host restriction factors play key roles in innate antiviral defense, but it remains poorly understood which of them restricts HIV-1 in vivo. Here, we used single-cell transcriptomic analysis to identify host factors associated with HIV-1 control during acute infection by correlating host gene expression with viral RNA abundance within individual cells. Wide sequencing of cells from one participant with the highest plasma viral load revealed that intracellular viral RNA transcription correlates inversely with expression of the gene PTMA, which encodes prothymosin \\u03b1. This association was genome-wide significant (Padjusted < 0.05) and was validated in 28 additional participants from Thailand and the Americas with HIV-1 CRF01_AE and subtype B infections, respectively. Overexpression of prothymosin \\u03b1 in vitro confirmed that this cellular factor inhibits HIV-1 transcription and infectious virus production. Our results identify prothymosin \\u03b1 as a host factor that restricts HIV-1 infection in vivo, which has implications for viral transmission and cure strategies. Description Profiling of host and viral transcripts in the same single cells isolated during acute infection identified prothymosin \\u03b1 as restricting HIV-1. Editor\\u2019s summary Identifying factors that restrict human immunodeficiency virus (HIV)\\u20131 replication in vivo may inform potential cure strategies for people living with HIV-1. Here, Geretz et al. analyzed transcripts from both the host and virus within the same single cells to identify host transcripts associated with lower HIV-1 RNA. Using samples from a person living with HIV-1, the authors identified prothymosin \\u03b1 as one such host restriction factor, which they validated in 28 additional participants. Further, overexpression of prothymosin \\u03b1 was sufficient to restrict HIV-1 replication in vitro. These data demonstrate that prothymosin \\u03b1 is a host restriction factor and suggest that it may represent a target for HIV-1 cure strategies. \\u2014Courtney Malo\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=193&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8dcdd8a94493412ca72f4e74379540797b4731d8\",\n",
      "        \"title\": \"Game changers in science and technology - now and beyond\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=194&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7d33200da3bb127e996f40f5ed95cdb068038757\",\n",
      "        \"title\": \"Single\\u2010cell transcriptome sequencing of B\\u2010cell heterogeneity and tertiary lymphoid structure predicts breast cancer prognosis and neoadjuvant therapy efficacy\",\n",
      "        \"abstract\": \"Abstract Background Breast cancer (BC) is a highly heterogeneous disease, and although immunotherapy has recently increased patient survival in a number of solid and hematologic malignancies, most BC subtypes respond poorly to immune checkpoint blockade therapy (ICB). B cells, particularly those that congregate in tertiary lymphoid structures (TLS), play a significant role in antitumour immunity. However, B\\u2010cell heterogeneity at single\\u2010cell resolution and its clinical significance with TLS in BC need to be explored further. Methods Primary tumour lesions and surrounding normal tissues were taken from 14 BC patients, totaling 124,587 cells, for single\\u2010cell transcriptome sequencing and bioinformatics analysis. Results Based on the usual markers, the single\\u2010cell transcriptome profiles were classified into various clusters. A thorough single\\u2010cell study was conducted with a focus on tumour\\u2010infiltrating B cells (TIL\\u2010B) and tumour\\u2010associated neutrophils (TAN). TIL\\u2010B was divided into five clusters, and unusual cell types, such as follicular B cells, which are strongly related to immunotherapy efficacy, were identified. In BC, TAN and TIL\\u2010B infiltration are positively correlated, and at the same time, compared with TLS\\u2010high, TAN and TIL\\u2010B in TLS\\u2010low group are significantly positively correlated. Conclusions In conclusion, our study highlights the heterogeneity of B cells in BC, explains how B cells and TLS contribute significantly to antitumour immunity at both the single\\u2010cell and clinical level, and offers a straightforward marker for TLS called CD23. These results will offer more pertinent information on the applicability and effectiveness of tumour immunotherapy for BC.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=195&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e803bf84e9d199c26f714978a58fd509d047ec99\",\n",
      "        \"title\": \"Recent progress in co-detection of single-cell transcripts and proteins\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=196&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d22089751fd73c35d261ad18968db92095ab7ae\",\n",
      "        \"title\": \"Encoding Thiol Metabolite Information into a DNA Sequence by Disulfide Exchange Reaction on Oligodeoxynucleotides for Parallel Analysis of the Metabolite and mRNA\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=197&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc89af801ce372728da2a9d16be5e3c1564db830\",\n",
      "        \"title\": \"mRNA vaccination boosts S-specific T cell memory and promotes expansion of CD45RAint TEMRA-like CD8+ T cells in COVID-19 recovered individuals\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=198&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"73e65c6cb5b560ce40423f99d7e5f4d0dbca5af2\",\n",
      "        \"title\": \"Spatial Transcriptomic Technologies\",\n",
      "        \"abstract\": \"Spatial transcriptomic technologies enable measurement of expression levels of genes systematically throughout tissue space, deepening our understanding of cellular organizations and interactions within tissues as well as illuminating biological insights in neuroscience, developmental biology and a range of diseases, including cancer. A variety of spatial technologies have been developed and/or commercialized, differing in spatial resolution, sensitivity, multiplexing capability, throughput and coverage. In this paper, we review key enabling spatial transcriptomic technologies and their applications as well as the perspective of the techniques and new emerging technologies that are developed to address current limitations of spatial methodologies. In addition, we describe how spatial transcriptomics data can be integrated with other omics modalities, complementing other methods in deciphering cellar interactions and phenotypes within tissues as well as providing novel insight into tissue organization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=199&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"56b9def0c0270e473e011012f78e10d0eae6daad\",\n",
      "        \"title\": \"Single-cell multi-omics topic embedding reveals cell-type-specific and COVID-19 severity-related immune signatures\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=200&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"efb65982acf5a5dbac1d6ea6879aaaa17856dd9c\",\n",
      "        \"title\": \"Implications of tumour heterogeneity on cancer evolution and therapy resistance: lessons from breast cancer\",\n",
      "        \"abstract\": \"Tumour heterogeneity is pervasive amongst many cancers and leads to disease progression, and therapy resistance. In this review, using breast cancer as an exemplar, we focus on the recent advances in understanding the interplay between tumour cells and their microenvironment using single cell sequencing and digital spatial profiling technologies. Further, we discuss the utility of lineage tracing methodologies in pre\\u2010clinical models of breast cancer, and how these are being used to unravel new therapeutic vulnerabilities and reveal biomarkers of breast cancer progression. \\u00a9 2023 The Pathological Society of Great Britain and Ireland.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=201&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9d274a6e1c1fed62c0385651a392e85177c43f8\",\n",
      "        \"title\": \"Mutations from patients with IPEX ported to mice reveal different patterns of FoxP3 and Treg dysfunction\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=202&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18e265d68772af5d6563e2ff8ce3dd53ee177643\",\n",
      "        \"title\": \"Lymphatic endothelial transcription factor Tbx1 promotes an immunosuppressive microenvironment to facilitate post-myocardial infarction repair.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=203&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"efa6e29335c790c4b3ddbd99d26aea86dcdeb430\",\n",
      "        \"title\": \"Single-cell assignment using multiple-adversarial domain adaptation network with large-scale references\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=204&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3de1d9ad3127be34a8dcdba1d9c0ede67b79e4bd\",\n",
      "        \"title\": \"hadge: a comprehensive pipeline for donor deconvolution in single-cell studies\",\n",
      "        \"abstract\": \"Single cell multiplexing techniques (cell hashing and genetic multiplexing) allow to combine multiple samples, thereby optimizing sample processing and reducing batch effects. Cell hashing conjugates antibody-tags or chemical-oligonucleotides to cell membranes, while genetic multiplexing allows to mix genetically diverse samples and relies on aggregation of RNA reads at known genomic coordinates. We developed hadge (hashing deconvolution combined with genotype information), a Nextflow pipeline that combines 12 methods to perform both hashing- and genotype-based deconvolution. We propose a joint deconvolution strategy combining the best performing methods and we demonstrate how this approach leads to recovery of previously discarded cells in a nuclei hashing of fresh-frozen brain tissue.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=205&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a8d8a3eaa014e8451472cfd98c007072267dbb11\",\n",
      "        \"title\": \"Exploring the optimization of autoencoder design for imputing single-cell RNA sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=206&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50c5dd72c00b4f94ce4228d73efd74a678e82eb0\",\n",
      "        \"title\": \"Protein-DNA Conjugates with a Discrete Number of Oligonucleotide Strands for Highly Reproducible Protein Quantification by the DNA Proximity Assay.\",\n",
      "        \"abstract\": \"Protein-oligonucleotide conjugates are increasingly used as detection probes in biological applications such as proximity sensing and spatial biology. The preparation of high-quality conjugate probes as starting reagents is critical for achieving good and consistent performance, which we demonstrate via the DNA proximity assay (DPA) for the one-pot quantification of protein targets. We first established a complete conjugation and anion-exchange chromatography purification workflow to reproducibly obtain pure subpopulations of protein probes carrying a discrete number of oligonucleotide strands. A systematic study using the purified conjugate sub-populations confirmed that the order of conjugate (number of oligonucleotides per protein) and its purity (the absence of the unconjugated antibody) were important for ensuring optimal and reproducible assay performance. The streamlined workflow was then successfully used to conjugate a pair of universal DPA initiator oligonucleotides onto a wide range of binders including antibodies, nanobodies, and antigens which enabled the versatile detection of different types of proteins such as cytokines, total antibodies, and specific antibody isotypes. The good assay robustness (the inter-assay coefficient of variation lower than 5%) and linear calibration curve was achieved across all targets with just a single mix-and-incubate reaction step and a short reaction time of 30 min. We anticipate the streamlined protein-oligonucleotide probe preparation workflow developed in this work to have broad utility across applications leveraging the specificity of protein bio-recognition with the programmability of DNA hybridization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=207&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"33c76aba753fa580f905a608e8be4e733bbcb232\",\n",
      "        \"title\": \"Applications of multi\\u2010omics analysis in human diseases\",\n",
      "        \"abstract\": \"Abstract Multi\\u2010omics usually refers to the crossover application of multiple high\\u2010throughput screening technologies represented by genomics, transcriptomics, single\\u2010cell transcriptomics, proteomics and metabolomics, spatial transcriptomics, and so on, which play a great role in promoting the study of human diseases. Most of the current reviews focus on describing the development of multi\\u2010omics technologies, data integration, and application to a particular disease; however, few of them provide a comprehensive and systematic introduction of multi\\u2010omics. This review outlines the existing technical categories of multi\\u2010omics, cautions for experimental design, focuses on the integrated analysis methods of multi\\u2010omics, especially the approach of machine learning and deep learning in multi\\u2010omics data integration and the corresponding tools, and the application of multi\\u2010omics in medical researches (e.g., cancer, neurodegenerative diseases, aging, and drug target discovery) as well as the corresponding open\\u2010source analysis tools and databases, and finally, discusses the challenges and future directions of multi\\u2010omics integration and application in precision medicine. With the development of high\\u2010throughput technologies and data integration algorithms, as important directions of multi\\u2010omics for future disease research, single\\u2010cell multi\\u2010omics and spatial multi\\u2010omics also provided a detailed introduction. This review will provide important guidance for researchers, especially who are just entering into multi\\u2010omics medical research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=208&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"179c71b93b9b45a515d77e1423eca1aa10e512ba\",\n",
      "        \"title\": \"A Review of Single-Cell RNA-Seq Annotation, Integration, and Cell\\u2013Cell Communication\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) has emerged as a powerful tool for investigating cellular biology at an unprecedented resolution, enabling the characterization of cellular heterogeneity, identification of rare but significant cell types, and exploration of cell\\u2013cell communications and interactions. Its broad applications span both basic and clinical research domains. In this comprehensive review, we survey the current landscape of scRNA-seq analysis methods and tools, focusing on count modeling, cell-type annotation, data integration, including spatial transcriptomics, and the inference of cell\\u2013cell communication. We review the challenges encountered in scRNA-seq analysis, including issues of sparsity or low expression, reliability of cell annotation, and assumptions in data integration, and discuss the potential impact of suboptimal clustering and differential expression analysis tools on downstream analyses, particularly in identifying cell subpopulations. Finally, we discuss recent advancements and future directions for enhancing scRNA-seq analysis. Specifically, we highlight the development of novel tools for annotating single-cell data, integrating and interpreting multimodal datasets covering transcriptomics, epigenomics, and proteomics, and inferring cellular communication networks. By elucidating the latest progress and innovation, we provide a comprehensive overview of the rapidly advancing field of scRNA-seq analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=209&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"51e416973743931afa470edb9f0e4068ab323100\",\n",
      "        \"title\": \"Application of single-cell omics in inflammatory bowel disease\",\n",
      "        \"abstract\": \"Over the past decade, the advent of single cell RNA-sequencing has revolutionized the approach in cellular transcriptomics research. The current technology offers an unbiased platform to understand how genotype correlates to phenotype. Single-cell omics applications in gastrointestinal (GI) research namely inflammatory bowel disease (IBD) has become popular in the last few years with multiple publications as single-cell omics techniques can be applied directly to the target organ, the GI tract at the tissue level. Through examination of mucosal tissue and peripheral blood in IBD, the recent boom in single cell research has identified a myriad of key immune players from enterocytes to tissue resident memory T cells, and explored functional heterogeneity within cellular subsets previously unreported. As we begin to unravel the complex mucosal immune system in states of health and disease like IBD, the power of exploration through single-cell omics can change our approach to translational research. As novel techniques evolve through multiplexing single-cell omics and spatial transcriptomics come to the forefront, we can begin to fully comprehend the disease IBD and better design targets of treatment. In addition, hopefully these techniques can ultimately begin to identify biomarkers of therapeutic response and answer clinically relevant questions in how to tailor individual therapy to patients through personalized medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=210&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6921a1fc74f5741523ccc95195f4618b70ea90ee\",\n",
      "        \"title\": \"Contrastively generative self-expression model for single-cell and spatial multimodal data\",\n",
      "        \"abstract\": \"Advances in single-cell multi-omics technology provide an unprecedented opportunity to fully understand cellular heterogeneity. However, integrating omics data from multiple modalities is challenging due to the individual characteristics of each measurement. Here, to solve such a problem, we propose a contrastive and generative deep self-expression model, called single-cell multimodal self-expressive integration (scMSI), which integrates the heterogeneous multimodal data into a unified manifold space. Specifically, scMSI first learns each omics-specific latent representation and self-expression relationship to consider the characteristics of different omics data by deep self-expressive generative model. Then, scMSI combines these omics-specific self-expression relations through contrastive learning. In such a way, scMSI provides a paradigm to integrate multiple omics data even with weak relation, which effectively achieves the representation learning and data integration into a unified framework. We demonstrate that scMSI provides a cohesive solution for a variety of analysis tasks, such as integration analysis, data denoising, batch correction and spatial domain detection. We have applied scMSI on various single-cell and spatial multimodal datasets to validate its high effectiveness and robustness in diverse data types and application scenarios.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=211&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bf42a0e0719037f7f0ec1b7ca232f605c5d3605f\",\n",
      "        \"title\": \"Tutorial: a statistical genetics guide to identifying HLA alleles driving complex disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=212&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3ae1bb7f9eb76f909b17c04c8d905ac16cdca5d5\",\n",
      "        \"title\": \"Statistical and machine learning methods for immunoprofiling based on single-cell data\",\n",
      "        \"abstract\": \"ABSTRACT Immunoprofiling has become a crucial tool for understanding the complex interactions between the immune system and diseases or interventions, such as therapies and vaccinations. Immune response biomarkers are critical for understanding those relationships and potentially developing personalized intervention strategies. Single-cell data have emerged as a promising source for identifying immune response biomarkers. In this review, we discuss the current state-of-the-art methods for immunoprofiling, including those for reducing the dimensionality of high-dimensional single-cell data and methods for clustering, classification, and prediction. We also draw attention to recent developments in data integration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=213&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b3e8145b657751c347d7e02af9f38564e6abff68\",\n",
      "        \"title\": \"Understanding virus\\u2013host interactions in tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=214&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eac3d02689279866046bcadabd7535bdc1c277ae\",\n",
      "        \"title\": \"A universal tool for predicting differentially active features in single-cell and spatial genomics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=215&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd158c031c4140eee517dca9e81d3aba3c6cb746\",\n",
      "        \"title\": \"Unifying comprehensive genomics and transcriptomics in individual cells to illuminate oncogenic and drug resistance mechanisms\",\n",
      "        \"abstract\": \"Discovering transcriptional variation in the absence of underlying genomic contributions hinders understanding of molecular mechanisms of disease. To assess this coordination in individual cells, we leveraged a new workflow, ResolveOME, exploiting the attributes of primary template-directed amplification (PTA) to enable accurate, complete-genome assessment of single-nucleotide variation in conjunction with full-transcript RNA-seq. In cultured AML cells resistant to the FLT3 inhibitor quizartinib, we uncovered a FLT3 missense mutation and matched transcript upregulation of AXL signal transduction and enhancer factor modulation driving resistance. In primary breast cancer cells, we detected oncogenic PIK3CA N345K mutations and heterogeneous classes of chromosomal loss and were empowered to interpret these genotypes with the crucial knowledge of cell identity and state derived from the transcriptome. The study reinforces the plasticity of the genome in conjunction with expected transcriptional modulation, leading to combinatorial alterations that affect cellular evolution that can be identified through application of this workflow to individual cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=216&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"87c9b2b0b22d21b7e36cacc2808be4973aaca884\",\n",
      "        \"title\": \"Single-cell multiomic understanding of HIV-1 reservoir at epigenetic, transcriptional, and protein levels\",\n",
      "        \"abstract\": \"Purpose of review The success of HIV-1 eradication strategies relies on in-depth understanding of HIV-1-infected cells. However, HIV-1-infected cells are extremely heterogeneous and rare. Single-cell multiomic approaches resolve the heterogeneity and rarity of HIV-1-infected cells. Recent findings Advancement in single-cell multiomic approaches enabled HIV-1 reservoir profiling across the epigenetic (ATAC-seq), transcriptional (RNA-seq), and protein levels (CITE-seq). Using HIV-1 RNA as a surrogate, ECCITE-seq identified enrichment of HIV-1-infected cells in clonally expanded cytotoxic CD4+ T cells. Using HIV-1 DNA PCR-activated microfluidic sorting, FIND-seq captured the bulk transcriptome of HIV-1 DNA+ cells. Using targeted HIV-1 DNA amplification, PheP-seq identified surface protein expression of intact versus defective HIV-1-infected cells. Using ATAC-seq to identify HIV-1 DNA, ASAP-seq captured transcription factor activity and surface protein expression of HIV-1 DNA+ cells. Combining HIV-1 mapping by ATAC-seq and HIV-1 RNA mapping by RNA-seq, DOGMA-seq captured the epigenetic, transcriptional, and surface protein expression of latent and transcriptionally active HIV-1-infected cells. To identify reproducible biological insights and authentic HIV-1-infected cells and avoid false-positive discovery of artifacts, we reviewed current practices of single-cell multiomic experimental design and bioinformatic analysis. Summary Single-cell multiomic approaches may identify innovative mechanisms of HIV-1 persistence, nominate therapeutic strategies, and accelerate discoveries.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=217&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b925f560249d7b4364ce538112517b7d3f7d99be\",\n",
      "        \"title\": \"Progress in single-cell multimodal sequencing and multi-omics data integration\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=218&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9ff596f86b2545c2cc4600841d003bf4679344cf\",\n",
      "        \"title\": \"Application of single-cell RNA sequencing methods to develop B cell targeted treatments for autoimmunity\",\n",
      "        \"abstract\": \"The COVID-19 pandemic coincided with several transformative advances in single-cell analysis. These new methods along with decades of research and trials with antibody therapeutics and RNA based technologies allowed for highly effective vaccines and treatments to be produced at astonishing speeds. While these tools were initially focused on models of infection, they also show promise in an autoimmune setting. Self-reactive B cells play important roles as antigen-presenting cells and cytokine and autoantibody producers for many autoimmune diseases. Yet, current therapies to target autoreactive B cells deplete all B cells irrespective of their pathogenicity. Development of self-reactive B cell targeting therapies that would spare non-pathogenic B cells are needed to treat disease while allowing effective immune responses to other ailments. Single-cell RNA sequencing (scRNA-seq) approaches will aid in identification of the pathogenic self-reactive B cells operative in autoimmunity and help with development of more favorable precision targeted therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=219&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6accc806c275f23300f7765c23bb1c68eb1ba49d\",\n",
      "        \"title\": \"The Contribution of Multiplexing Single Cell RNA Sequencing in Acute Myeloid Leukemia\",\n",
      "        \"abstract\": \"Decades ago, the treatment for acute myeloid leukemia relied on cytarabine and anthracycline. However, advancements in medical research have introduced targeted therapies, initially employing monoclonal antibodies such as ant-CD52 and anti-CD123, and subsequently utilizing specific inhibitors that target molecular mutations like anti-IDH1, IDH2, or FLT3. The challenge lies in determining the role of these therapeutic options, considering the inherent tumor heterogeneity associated with leukemia diagnosis and the clonal drift that this type of tumor can undergo. Targeted drugs necessitate an examination of various therapeutic targets at the individual cell level rather than assessing the entire population. It is crucial to differentiate between the prognostic value and therapeutic potential of a specific molecular target, depending on whether it is found in a terminally differentiated cell with limited proliferative potential or a stem cell with robust capabilities for both proliferation and self-renewal. However, this cell-by-cell analysis is accompanied by several challenges. Firstly, the scientific aspect poses difficulties in comparing different single cell analysis experiments despite efforts to standardize the results through various techniques. Secondly, there are practical obstacles as each individual cell experiment incurs significant financial costs and consumes a substantial amount of time. A viable solution lies in the ability to process multiple samples simultaneously, which is a distinctive feature of the cell hashing technique. In this study, we demonstrate the applicability of the cell hashing technique for analyzing acute myeloid leukemia cells. By comparing it to standard single cell analysis, we establish a strong correlation in various parameters such as quality control, gene expression, and the analysis of leukemic blast markers in patients. Consequently, this technique holds the potential to become an integral part of the biological assessment of acute myeloid leukemia, contributing to the personalized and optimized management of the disease, particularly in the context of employing targeted therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=220&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1aeac38f48baa72651f0febdfb904782ddb0e7b3\",\n",
      "        \"title\": \"Liver in infections: a single-cell and spatial transcriptomics perspective\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=221&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ccfac1e9787dbdba0e8d30fb38e6cc608ae8670b\",\n",
      "        \"title\": \"Characterizing control of memory CD8 T cell differentiation by BTB-ZF transcription factor Zbtb20\",\n",
      "        \"abstract\": \"This work characterizes the transcriptional and epigenetic control of memory CD8 T cell differentiation by the BTB-ZF family transcription factor Zbtb20 using single cell RNA and ATAC sequencing and identifies direct genomic targets of Zbtb20 using CUT&RUN. Members of the BTB-ZF transcription factor family regulate the immune system. Our laboratory identified that family member Zbtb20 contributes to the differentiation, recall responses, and metabolism of CD8 T cells. Here, we report a characterization of the transcriptional and epigenetic signatures controlled by Zbtb20 at single-cell resolution during the effector and memory phases of the CD8 T cell response. Without Zbtb20, transcriptional programs associated with memory CD8 T cell formation were up-regulated throughout the CD8 T response. A signature of open chromatin was associated with genes controlling T cell activation, consistent with the known impact on differentiation. In addition, memory CD8 T cells lacking Zbtb20 were characterized by open chromatin regions with overrepresentation of AP-1 transcription factor motifs and elevated RNA- and protein-level expressions of the corresponding AP-1 components. Finally, we describe motifs and genomic annotations from the DNA targets of Zbtb20 in CD8 T cells identified by cleavage under targets and release under nuclease (CUT&RUN). Together, these data establish the transcriptional and epigenetic networks contributing to the control of CD8 T cell responses by Zbtb20.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=222&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"520d8a93f393a4734fc9a3e17651de0abc95bedc\",\n",
      "        \"title\": \"Single-cell analysis of signalling and transcriptional responses to type I interferons\",\n",
      "        \"abstract\": \"Type I interferons (IFNs) play crucial roles in antiviral defence, autoinflammation and cancer immunity. The human genome encodes 17 different type I IFNs that all signal through the same receptor. Non-redundant functions have been reported for some type I IFNs. However, whether different type I IFNs induce different responses remains largely unknown. We stimulated human peripheral blood mononuclear cells (PBMCs) with recombinant type I IFNs to address this question in multiple types of primary cells. We analysed signalling responses by mass cytometry and changes in gene expression by bulk and single-cell RNA sequencing. We found cell-type specific changes in the phosphorylation of STAT transcription factors and in the gene sets induced and repressed upon type I IFN exposure. We further report that the magnitude of these responses varied between different type I IFNs, whilst qualitatively different responses to type I IFN subtypes were not apparent. Taken together, we provide a rich resource mapping signalling responses and IFN-regulated genes in immune cells. Highlights Mass cytometry and scRNAseq analysis of human PBMCs stimulated with type I IFNs Cell type-specific phosphorylation of STAT proteins and induction of ISGs Different type I IFNs induce qualitatively similar responses that vary in magnitude Identification of ten core ISGs, up-regulated by all cell types in response to all type I IFNs In brief Rigby et al. provide a single-cell map of signalling and transcriptomic responses to type I IFNs in ex vivo stimulated human PBMCs. Different cell types responded in unique ways but differences between different type I IFNs were only quantitative. These rich datasets are available via an easy-to-use interactive interface (https://rehwinkellab.shinyapps.io/ifnresource/). Graphical abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=223&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"67f558d5fd83f34b7bd25a209b7e3e88d4bb7c0d\",\n",
      "        \"title\": \"Integration of single\\u2010cell transcriptome and proteome technologies: Toward spatial resolution levels\",\n",
      "        \"abstract\": \"Cells are basic building blocks of life with vast heterogeneity. Nowadays, the rapid development of single\\u2010cell multiomics (scMulti\\u2010Omics) has facilitated comprehensive understanding of gene regulatory networks, cellular characteristics, and temporal dynamics. However, simultaneous analysis of transcriptome and proteome at single\\u2010cell level still faces huge challenges due to their differences in molecular modalities. Recent technological advances in single\\u2010cell manipulations, barcoding, and ultrasensitive instrument recently offer unprecedented opportunities for the co\\u2010profiling of genes and proteins. In this review, multiple types of single\\u2010cell isolation, lysis, and molecular separation technologies are first introduced. Second, various approaches for co\\u2010measurement of transcriptome and proteome in single\\u2010cells are summarized, with their advantages, limitations, and capacity for targeted or unbiased deep analysis. Then we highlight the cutting\\u2010edge spatial multiomics methodologies that operate at the single\\u2010cell or subcellular resolution level, providing a comprehensive understanding of cell function and heterogeneity within the tissue spatial environment. The emerging biomedical applications of multiomics are also discussed. Finally, the challenges and prospects of this field are proposed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=224&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f193a7709fd1c25874cd4df2c3168f5220ee457\",\n",
      "        \"title\": \"A structured evaluation of cryopreservation in generating single-cell transcriptomes from cerebrospinal fluid\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=225&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"48f2eb9f388592c54e971e859f9299dee61295dc\",\n",
      "        \"title\": \"Tracking B cell responses to the SARS-CoV-2 mRNA-1273 vaccine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=226&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5620c313d3de7eb83b0a521106618b6dbd7c410e\",\n",
      "        \"title\": \"The promising application of cell-cell interaction analysis in cancer from single-cell and spatial transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=227&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b854844f564241c2ebb80a3125c1754c57616a08\",\n",
      "        \"title\": \"Graded expression of the chemokine receptor CX3CR1 marks differentiation states of human and murine T cells and enables cross-species interpretation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=228&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aa0f109bfac3b49d81b7a1197301f22b73d3617a\",\n",
      "        \"title\": \"Scalable Integration of Multiomic Single Cell Data Using Generative Adversarial Networks\",\n",
      "        \"abstract\": \"Single cell profiling has become a common practice to investigate the complexity of tissues, organs and organisms. Recent technological advances are expanding our capabilities to profile various molecular layers beyond the transcriptome such as, but not limited to, the genome, the epigenome and the proteome. Depending on the experimental procedure, these data can be obtained from separate assays or from the very same cells. Despite development of computational methods for data integration is an active research field, most of the available strategies have been devised for the joint analysis of two modalities and cannot accommodate a high number of them. To solve this problem, we here propose a multiomic data integration framework based on Wasserstein Generative Adversarial Networks (MOWGAN) suitable for the analysis of paired or unpaired data with high number of modalities (>2). At the core of our strategy is a single network trained on all modalities together, limiting the computational burden when many molecular layers are evaluated. Source code of our framework is available at https://github.com/vgiansanti/MOWGAN.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=229&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fad690bfc40aeb049d7a33dd1d93a4edaf73ade6\",\n",
      "        \"title\": \"Robust joint clustering of multi-omics single-cell data via multi-modal high-order neighborhood Laplacian matrix optimization\",\n",
      "        \"abstract\": \"Abstract Motivation Simultaneous profiling of multi-omics single-cell data represents exciting technological advancements for understanding cellular states and heterogeneity. Cellular indexing of transcriptomes and epitopes by sequencing allowed for parallel quantification of cell-surface protein expression and transcriptome profiling in the same cells; methylome and transcriptome sequencing from single cells allows for analysis of transcriptomic and epigenomic profiling in the same individual cells. However, effective integration method for mining the heterogeneity of cells over the noisy, sparse, and complex multi-modal data is in growing need. Results In this article, we propose a multi-modal high-order neighborhood Laplacian matrix optimization framework for integrating the multi-omics single-cell data: scHoML. Hierarchical clustering method was presented for analyzing the optimal embedding representation and identifying cell clusters in a robust manner. This novel method by integrating high-order and multi-modal Laplacian matrices would robustly represent the complex data structures and allow for systematic analysis at the multi-omics single-cell level, thus promoting further biological discoveries. Availability and implementation Matlab code is available at https://github.com/jianghruc/scHoML.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=230&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"04d6c3719cd3b3cb35f42e961fbf5de3eb2d18bd\",\n",
      "        \"title\": \"Recent Approaches to Design and Analysis of Electrical Impedance Systems for Single Cells Using Machine Learning\",\n",
      "        \"abstract\": \"Individual cells have many unique properties that can be quantified to develop a holistic understanding of a population. This can include understanding population characteristics, identifying subpopulations, or elucidating outlier characteristics that may be indicators of disease. Electrical impedance measurements are rapid and label-free for the monitoring of single cells and generate large datasets of many cells at single or multiple frequencies. To increase the accuracy and sensitivity of measurements and define the relationships between impedance and biological features, many electrical measurement systems have incorporated machine learning (ML) paradigms for control and analysis. Considering the difficulty capturing complex relationships using traditional modelling and statistical methods due to population heterogeneity, ML offers an exciting approach to the systemic collection and analysis of electrical properties in a data-driven way. In this work, we discuss incorporation of ML to improve the field of electrical single cell analysis by addressing the design challenges to manipulate single cells and sophisticated analysis of electrical properties that distinguish cellular changes. Looking forward, we emphasize the opportunity to build on integrated systems to address common challenges in data quality and generalizability to save time and resources at every step in electrical measurement of single cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=231&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"819bf8e22ebff4cb7143554f3ca7d05e3ff1e41a\",\n",
      "        \"title\": \"Microtechnologies for single-cell and spatial multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=232&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0181a9a1770ed8ecb2d363ed95f36c06fb2c639e\",\n",
      "        \"title\": \"TCR-independent CD137 (4\\u20131BB) signaling promotes CD8+-exhausted T cell proliferation and terminal differentiation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=233&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b3395570012336e5e03d197d10f34649a27ba8e3\",\n",
      "        \"title\": \"Understanding tumour endothelial cell heterogeneity and function from single-cell omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=234&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd302cb3ba54d7307419cb70998063b7be9b8d3d\",\n",
      "        \"title\": \"Early cellular mechanism of type I interferon-driven susceptibility to tuberculosis\",\n",
      "        \"abstract\": \"Mycobacterium tuberculosis (Mtb) causes 1.6 million deaths annually. Active tuberculosis correlates with a neutrophil-driven type I interferon (IFN) signature, but the cellular mechanisms underlying tuberculosis pathogenesis remain poorly understood. We found interstitial macrophages (IMs) and plasmacytoid dendritic cells (pDCs) are dominant producers of type I IFN during Mtb infection in mice and non-human primates, and pDCs localize near human Mtb granulomas. Depletion of pDCs reduces Mtb burdens, implicating pDCs in tuberculosis pathogenesis. During IFN-driven disease, we observe abundant DNA-containing neutrophil extracellular traps (NETs) known to activate pDCs. Cell type-specific disruption of the type I IFN receptor suggests IFNs act on IMs to inhibit Mtb control. Single cell RNA-seq indicates type I IFN-responsive cells are defective in their response to IFN\\u03b3, a cytokine critical for Mtb control. We propose pDC-derived type I IFNs act on IMs to drive bacterial replication, further neutrophil recruitment, and active tuberculosis disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=235&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b87f8ef92a1477546b85910028101628a17aad80\",\n",
      "        \"title\": \"Deconvolution reveals compositional differences in high-grade serous ovarian cancer subtypes\",\n",
      "        \"abstract\": \"Ovarian cancer is a deadly disease with few effective therapies. The most common form is high-grade serous ovarian cancer (HGSOC). Transcriptomic subtypes of HGSOC have shown promise in characterizing tumor heterogeneity and are associated with survival. Gene expression signatures for the subtypes suggest variation in stromal cell types in the tumor microenvironment (TME). Here, we characterize the TME composition of HGSOC on a population scale by performing deconvolution on bulk transcriptomic data. We use comprehensive cell type profiles from 164 HGSOC tumor samples from two independent reference datasets, in order to compare cell type proportions across and within bulk transcriptomic datasets, and assess their alignment to the subtypes proposed by The Cancer Genome Atlas. We also assess the relationship between tumor composition and clinical outcomes. Our results suggest that HGSOC transcriptomic subtypes are driven by TME composition, specifically fibroblast and immune cell content, and we propose a modified HGSOC subtype model informed by cell composition.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=236&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b3c278f593f9336c6390da1d47ad57dad8931ec3\",\n",
      "        \"title\": \"Innovations advancing our understanding of microglia in Alzheimer's disease: From in vitro to in vivo models\",\n",
      "        \"abstract\": \"Microglia have been implicated in Alzheimer's disease (AD) pathogenesis through the identification of risk factor genes that are specifically or predominantly expressed in this cell type. Additional evidence suggests that microglia undergo dramatic morphological and phenotypic state changes during AD progression, as observed in human post\\u2010mortem tissue and animal model research. Although valuable, these studies are often hampered by either representing one time point in human tissue (end point) or because of the lack of conservation between species of microglial transcriptomes, proteomes and cell states. Thus, the development and application of novel human model systems have been beneficial in the study of microglia in neurodegeneration. Recent innovations include the use of human pluripotent stem cell (hPSC)\\u2010derived microglia in 2D or 3D culture systems, the transdifferentiation of microglia from patient monocytes and the xenotransplantation of hPSC\\u2010derived microglia into mouse brains. This review summarizes the recent innovations that have advanced our understanding of microglia in AD, through the use of single\\u2010cell RNA sequencing, hPSC\\u2010derived microglia culture within brain organoids and xenotransplantation into mouse brain. Through outlining the strengths and limitations of these approaches, we provide recommendations that will aid future endeavours in advancing our understanding of the complex role of microglia in AD onset and progression.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=237&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"624109a70870b6565629489009f71c17bec28eba\",\n",
      "        \"title\": \"Enhancing single-cell western blotting sensitivity using diffusive analyte blotting and antibody conjugate amplification\",\n",
      "        \"abstract\": \"While there are many techniques to achieve highly sensitive, multiplex detection of RNA and DNA from single cells, detecting protein contents often suffers from low limits of detection and throughput. Miniaturized, high-sensitivity western blots on single cells (scWesterns) are attractive since they do not require advanced instrumentation. By physically separating analytes, scWesterns also uniquely mitigate limitations to target protein multiplexing posed by affinity reagent performance. However, a fundamental limitation of scWesterns is their limited sensitivity for detecting low-abundance proteins, which arises from transport barriers posed by the separation gel against detection species. Here we address sensitivity by decoupling the electrophoretic separation medium from the detection medium. We transfer scWestern separations to a nitrocellulose blotting medium with distinct mass transfer advantages over traditional in-gel probing, yielding a 5.9-fold improvement in limit of detection. We next amplify probing of blotted proteins with enzyme-antibody conjugates which are incompatible with traditional in-gel probing to achieve further improvement in the limit of detection to 103 molecules, a 520-fold improvement. This enables us to detect 85% and 100% of cells in an EGFP-expressing population using fluorescently tagged and enzyme-conjugated antibodies respectively, compared to 47% of cells using in-gel detection. These results suggest compatibility of nitrocellulose-immobilized scWesterns with a variety of affinity reagents \\u2014 not previously accessible for in-gel use \\u2014 for further signal amplification and detection of low abundance targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=238&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=239&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=240&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=241&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=242&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=243&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=244&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=245&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=246&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7414df74826fb1c20ddb3c111b81edd428741931\",\n",
      "        \"title\": \"Chemoenzymatic Measurement of LacNAc in Single-Cell Multiomics Reveals It as a Cell-Surface Indicator of Glycolytic Activity of CD8+ T Cells.\",\n",
      "        \"abstract\": \"Despite the rich information about the physiological state of a cell encoded in the dynamic changes of cell-surface glycans, chemical methods to capture specific glycan epitopes at the single-cell level are quite limited. Here, we report a chemoenzymatic method for the single-cell detection of N-acetyllactosamine (LacNAc) by labeling LacNAc with a specific DNA barcode. The chemoenzymatic labeling does not alter the transcriptional status of immune cells and is compatible with multiple scRNA-seq platforms. Integrated analysis of LacNAc and the transcriptome of T cells at the single-cell level reveals that the amount of cell-surface LacNAc is significantly upregulated in activated CD8+ T cells but maintained at basal levels in resting CD8+ T cells (i.e., naive and central memory T cells). Further analysis confirms that LacNAc levels are positively correlated with the glycolytic activity of CD8+ T cells during differentiation. Taken together, our study demonstrates the feasibility of the chemoenzymatic detection of cell-surface glycan in single-cell RNA sequencing-based multiomics with TCR sequence and cell-surface epitope information (i.e., scTCR and CITE-seq), and provides a new way to characterize the biological role of glycan in diverse physiological states.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=247&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=248&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=249&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=250&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=251&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=252&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=253&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=254&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"46468db9b852faa29b9aa1946a2792257367a7c5\",\n",
      "        \"title\": \"Studying stochastic systems biology of the cell with single-cell genomics data\",\n",
      "        \"abstract\": \"Recent experimental developments in genome-wide RNA quantification hold considerable promise for systems biology. However, rigorously probing the biology of living cells requires a unified mathematical framework that accounts for single-molecule biological stochasticity in the context of technical variation associated with genomics assays. We review models for a variety of RNA transcription processes, as well as the encapsulation and library construction steps of microfluidics-based single-cell RNA sequencing, and present a framework to integrate these phenomena by the manipulation of generating functions. Finally, we use simulated scenarios and biological data to illustrate the implications and applications of the approach.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=255&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=256&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=257&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cdfd919ed1d81447cf18ed7c09336b1fc22e02bb\",\n",
      "        \"title\": \"Responsiveness to Vedolizumab Therapy in Ulcerative Colitis is Associated With Alterations in Immune Cell-Cell Communications.\",\n",
      "        \"abstract\": \"BACKGROUND\\nUlcerative colitis (UC) and Crohn's disease are 2 types of inflammatory bowel disease (IBD), a group of chronic digestive disorders caused by aberrant immune responses to intestinal microbes. Although changes in the composition of immune cell subsets in the context of IBD have been previously described, the interactions and communication among cells are less well understood. Moreover, the precise mechanisms of action underlying many biologic therapies, including the anti-\\u03b14\\u03b27 integrin antagonist vedolizumab, remain incompletely understood. Our study aimed to explore possible additional mechanisms through which vedolizumab acts.\\n\\n\\nMETHODS\\nWe performed cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) on peripheral blood and colon immune cells derived from patients with ulcerative colitis treated with the anti-\\u03b14\\u03b27 integrin antagonist vedolizumab. We applied a previously published computational approach, NicheNet, to predict immune cell-cell interactions, revealing putative ligand-receptor pairs and key transcriptional changes downstream of these cell-cell communications (CCC).\\n\\n\\nRESULTS\\nWe observed decreased proportions of T helper 17 (TH17) cells in UC patients who responded to vedolizumab and therefore focused the study on identifying cell-cell communications and signals of TH17 cells with other immune cells. For example, we observed that colon TH17 cells from vedolizumab nonresponders were predicted to have a greater degree of interactions with classical monocytes compared with responders, whereas colon TH17 cells from vedolizumab responders exhibited more interactions with myeloid dendritic cells compared with nonresponders.\\n\\n\\nCONCLUSIONS\\nOverall, our results indicate that efforts to elucidate cell-cell communications among immune and nonimmune cell types may increase the mechanistic understanding of current and investigational therapies for IBD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=258&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=259&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=260&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"737d07b4cf99c2b7b8318493e71fd21985e7aba8\",\n",
      "        \"title\": \"Stabilized mosaic single-cell data integration using unshared features\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=261&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=262&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=263&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=264&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=265&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=266&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=267&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e82c817fba8aeb2faf5dc89dda0f308b46b77631\",\n",
      "        \"title\": \"Cellular dynamics of distinct skeletal cells and the development of osteosarcoma\",\n",
      "        \"abstract\": \"Bone contributes to the maintenance of vital biological activities. At the cellular level, multiple types of skeletal cells, including skeletal stem and progenitor cells (SSPCs), osteoblasts, chondrocytes, marrow stromal cells, and adipocytes, orchestrate skeletal events such as development, aging, regeneration, and tumorigenesis. Osteosarcoma (OS) is a primary malignant tumor and the main form of bone cancer. Although it has been proposed that the cellular origins of OS are in osteogenesis-related skeletal lineage cells with cancer suppressor gene mutations, its origins have not yet been fully elucidated because of a poor understanding of whole skeletal cell diversity and dynamics. Over the past decade, the advent and development of single-cell RNA sequencing analyses and mouse lineage-tracing approaches have revealed the diversity of skeletal stem and its lineage cells. Skeletal stem cells (SSCs) in the bone marrow endoskeletal region have now been found to efficiently generate OS and to be robust cells of origin under p53 deletion conditions. The identification of SSCs may lead to a more limited redefinition of bone marrow mesenchymal stem/stromal cells (BM-MSCs), and this population has been thought to contain cells from which OS originates. In this mini-review, we discuss the cellular diversity and dynamics of multiple skeletal cell types and the origin of OS in the native in vivo environment in mice. We also discuss future challenges in the study of skeletal cells and OS.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=268&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=269&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f792223b2b7bef56b572b4a9f219e9aba30202e0\",\n",
      "        \"title\": \"Spatial transcriptomics in human skin research\",\n",
      "        \"abstract\": \"Spatial transcriptomics is a revolutionary technique that enables researchers to characterise tissue architecture and localisation of gene expression. A plethora of technologies that map gene expression are currently being developed, aiming to facilitate spatially resolved, high\\u2010dimensional assessment of gene transcription in the context of human skin research. Knowing which gene is expressed by which cell and in which location within skin, facilitates understanding of skin function and dysfunction in both health and disease. In this review, we summarise the available spatial transcriptomic methods and we describe their application to a broad spectrum of dermatological diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=270&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"940a7d8fa8ab490b11a4f5eef1ca4f9a8c49d91b\",\n",
      "        \"title\": \"Deciphering and advancing CAR T-cell therapy with single-cell sequencing technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=271&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2bf94b2e912122c16951567b78e35c813e84d9eb\",\n",
      "        \"title\": \"Cytokinopathy with aberrant cytotoxic lymphocytes and pro-fibrotic myeloid response in SARS-CoV-2 mRNA vaccine-associated myocarditis\",\n",
      "        \"abstract\": \"Rare immune-mediated cardiac tissue inflammation can occur after vaccination, including after SARS-CoV-2 mRNA vaccines. However, the underlying immune cellular and molecular mechanisms driving this pathology remain poorly understood. Here, we investigated a cohort of patients who developed myocarditis and/or pericarditis with elevated troponin, B-type natriuretic peptide, and C-reactive protein levels as well as cardiac imaging abnormalities shortly after SARS-CoV-2 mRNA vaccination. Contrary to early hypotheses, patients did not demonstrate features of hypersensitivity myocarditis, nor did they have exaggerated SARS-CoV-2\\u2013specific or neutralizing antibody responses consistent with a hyperimmune humoral mechanism. We additionally found no evidence of cardiac-targeted autoantibodies. Instead, unbiased systematic immune serum profiling revealed elevations in circulating interleukins (IL-1\\u03b2, IL-1RA, and IL-15), chemokines (CCL4, CXCL1, and CXCL10), and matrix metalloproteases (MMP1, MMP8, MMP9, and TIMP1). Subsequent deep immune profiling using single-cell RNA and repertoire sequencing of peripheral blood mononuclear cells during acute disease revealed expansion of activated CXCR3+ cytotoxic T cells and NK cells, both phenotypically resembling cytokine-driven killer cells. In addition, patients displayed signatures of inflammatory and profibrotic CCR2+ CD163+ monocytes, coupled with elevated serum-soluble CD163, that may be linked to the late gadolinium enhancement on cardiac MRI, which can persist for months after vaccination. Together, our results demonstrate up-regulation in inflammatory cytokines and corresponding lymphocytes with tissue-damaging capabilities, suggesting a cytokine-dependent pathology, which may further be accompanied by myeloid cell\\u2013associated cardiac fibrosis. These findings likely rule out some previously proposed mechanisms of mRNA vaccine\\u2013-associated myopericarditis and point to new ones with relevance to vaccine development and clinical care. Description Cytokine-driven killer lymphocytes and inflammatory monocytes characterize myopericarditis after SARS-CoV-2 mRNA vaccination. Immunopathology signatures in myocarditis Myocarditis and/or pericarditis are rare adverse cardiac events observed after SARS-CoV-2 mRNA vaccination with a predilection for adolescent and young adult males. To investigate the pathogenesis of myopericarditis in this setting, Barmada and Klein et al. used unbiased immune profiling techniques to search for immune signatures that distinguished patients who developed myopericarditis from healthy vaccinated controls. Immune events associated with myopericarditis included elevated systemic levels of cytokines, an increased frequency of activated T and NK cells, and induction of inflammatory monocytes with profibrotic features. Neither immune targeting of cardiac autoantigens nor enhanced clonal expansion of B and T lymphocytes was detected. These findings provide deeper insights into the chain of events that can rarely lead to myopericarditis in the mRNA vaccine setting. \\u2014IRW\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=272&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=273&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=274&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=275&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"680f1360fc2bb5928c386f3bc1d87be25a177f4c\",\n",
      "        \"title\": \"Single-Cell Analysis in Immuno-Oncology\",\n",
      "        \"abstract\": \"The complexity of the cellular and non-cellular milieu surrounding human tumors plays a decisive role in the course and outcome of disease. The high variability in the distribution of the immune and non-immune compartments within the tumor microenvironments (TME) among different patients governs the mode of their response or resistance to current immunotherapeutic approaches. Through deciphering this diversity, one can tailor patients\\u2019 management to meet an individual\\u2019s needs. Single-cell (sc) omics technologies have given a great boost towards this direction. This review gathers recent data about how multi-omics profiling, including the utilization of single-cell RNA sequencing (scRNA-seq), assay for transposase-accessible chromatin with sequencing (scATAC-seq), T-cell receptor sequencing (scTCR-seq), mass, tissue-based, or microfluidics cytometry, and related bioinformatics tools, contributes to the high-throughput assessment of a large number of analytes at single-cell resolution. Unravelling the exact TCR clonotype of the infiltrating T cells or pinpointing the classical or novel immune checkpoints across various cell subsets of the TME provide a boost to our comprehension of adaptive immune responses, their antigen specificity and dynamics, and grant suggestions for possible therapeutic targets. Future steps are expected to merge high-dimensional data with tissue localization data, which can serve the investigation of novel multi-modal biomarkers for the selection and/or monitoring of the optimal treatment from the current anti-cancer immunotherapeutic armamentarium.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=276&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"759a3355ced7b55e8427e8f8cdf235f9c52a4e15\",\n",
      "        \"title\": \"Multiplexed single-cell transcriptome analysis reveals molecular characteristics of monkey pluripotent stem cell lines\",\n",
      "        \"abstract\": \"Efforts have been made to establish various human pluripotent stem cell lines. However, such methods have not yet been duplicated in non-human primate cells. Here, we introduce a multiplexed single-cell sequencing technique to profile the molecular features of monkey pluripotent stem cells in published culture conditions. The results demonstrate suboptimized maintenance of pluripotency and show that the selected signaling pathways for resetting human stem cells can also be interpreted for establishing monkey cell lines. Overall, this work legitimates the translation of novel human cell line culture conditions to monkey cells and provides guidance for exploring chemical cocktails for monkey stem cell line derivation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=277&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a2a841801ca42c9a0822b968d2f6860477b8d01a\",\n",
      "        \"title\": \"Clinical Cytometry for Platelets and Platelet Disorders.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=278&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=279&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"31eb6d979cebf62c8b1dc0d54cd8afe768c1d3ea\",\n",
      "        \"title\": \"Efficient Generation of Paired Single\\u2010Cell Multiomics Profiles by Deep Learning\",\n",
      "        \"abstract\": \"Recent advances in single\\u2010cell sequencing technology have made it possible to measure multiple paired omics simultaneously in a single cell such as cellular indexing of transcriptomes and epitopes by sequencing (CITE\\u2010seq) and single\\u2010nucleus chromatin accessibility and mRNA expression sequencing (SNARE\\u2010seq). However, the widespread application of these single\\u2010cell multiomics profiling technologies has been limited by their experimental complexity, noise in nature, and high cost. In addition, single\\u2010omics sequencing technologies have generated tremendous and high\\u2010quality single\\u2010cell datasets but have yet to be fully utilized. Here, single\\u2010cell multiomics generation (scMOG), a deep learning\\u2010based framework to generate single\\u2010cell assay for transposase\\u2010accessible chromatin (ATAC) data in silico is developed from experimentally available single\\u2010cell RNA\\u2010seq measurements and vice versa. The results demonstrate that scMOG can accurately perform cross\\u2010omics generation between RNA and ATAC, and generate paired multiomics data with biological meanings when one omics is experimentally unavailable and out of training datasets. The generated ATAC, either alone or in combination with measured RNA, exhibits equivalent or superior performance to that of the experimentally measured counterparts throughout multiple downstream analyses. scMOG is also applied to human lymphoma data, which proves to be more effective in identifying tumor samples than the experimentally measured ATAC data. Finally, the performance of scMOG is investigated in other omics such as proteomics and it still shows robust performance on surface protein generation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=280&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=281&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b2291347a53175aabbd884d54981e2fc4c79c92\",\n",
      "        \"title\": \"Multiplex Protein Imaging through PACIFIC: Photoactive Immunofluorescence with Iterative Cleavage\",\n",
      "        \"abstract\": \"Multiplex protein imaging technologies enable deep phenotyping and provide rich spatial information about biological samples. Existing methods have shown great success but also harbored trade-offs between various pros and cons, underscoring the persisting necessity to expand the imaging toolkits. Here we present PACIFIC: photoactive immunofluorescence with iterative cleavage, a new modality of multiplex protein imaging methods. PACIFIC achieves iterative multiplexing by implementing photocleavable fluorophores for antibody labeling with one-step spin-column purification. PACIFIC requires no specialized instrument, no DNA encoding, or chemical treatments. We demonstrate that PACIFIC can resolve cellular heterogeneity in both formalin-fixed paraffin-embedded (FFPE) samples and fixed cells. To further highlight how PACIFIC assists discovery, we integrate PACIFIC with live-cell tracking and identify phosphor-p70S6K as a critical driver that governs U87 cell mobility. Considering the cost, flexibility, and compatibility, we foresee that PACIFIC can confer deep phenotyping capabilities to anyone with access to traditional immunofluorescence platforms.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=282&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=283&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b5659ea2de3954e960ca5a688015f74ed3314c5\",\n",
      "        \"title\": \"Immune heterogeneity in cardiovascular diseases from a single-cell perspective\",\n",
      "        \"abstract\": \"A variety of immune cell subsets occupy different niches in the cardiovascular system, causing changes in the structure and function of the heart and vascular system, and driving the progress of cardiovascular diseases (CVDs). The immune cells infiltrating the injury site are highly diverse and integrate into a broad dynamic immune network that controls the dynamic changes of CVDs. Due to technical limitations, the effects and molecular mechanisms of these dynamic immune networks on CVDs have not been fully revealed. With recent advances in single-cell technologies such as single-cell RNA sequencing, systematic interrogation of the immune cell subsets is feasible and will provide insights into the way we understand the integrative behavior of immune populations. We no longer lightly ignore the role of individual cells, especially certain highly heterogeneous or rare subpopulations. We summarize the phenotypic diversity of immune cell subsets and their significance in three CVDs of atherosclerosis, myocardial ischemia and heart failure. We believe that such a review could enhance our understanding of how immune heterogeneity drives the progression of CVDs, help to elucidate the regulatory roles of immune cell subsets in disease, and thus guide the development of new immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=284&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"67a3873b8dd2e3a905bd2336b0246afd78d18e96\",\n",
      "        \"title\": \"ALS is imprinted in the chromatin accessibility of blood cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=285&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=286&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=287&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d1d6430241cc4c6e75530a2736fdf549e8bd8108\",\n",
      "        \"title\": \"Droplet-based microfluidics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=288&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3303680721d874315410cefd6c818f0cf58ccfdc\",\n",
      "        \"title\": \"Proximal tubule responses to injury: interrogation by single-cell transcriptomics\",\n",
      "        \"abstract\": \"Purpose of review Acute kidney injury (AKI) occurs in approximately 10\\u201315% of patients admitted to hospital and is associated with adverse clinical outcomes. Despite recent advances, management of patients with AKI is still mainly supportive, including the avoidance of nephrotoxins, volume and haemodynamic management and renal replacement therapy. A better understanding of the renal response to injury is the prerequisite to overcome current limitations in AKI diagnostics and therapy. Recent findings Single-cell technologies provided new opportunities to study the complexity of the kidney and have been instrumental for rapid advancements in the understanding of the cellular and molecular mechanisms of AKI. Summary We provide an update on single-cell technologies and we summarize the recent discoveries on the cellular response to injury in proximal tubule cells from the early response in AKI, to the mechanisms of tubule repair and the relevance of maladaptive tubule repair in the transition to chronic kidney disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=289&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=290&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=291&limit=1\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=292&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"226f682c916b9f671ae9abb38641847e1992d4f8\",\n",
      "        \"title\": \"Carcinoma-associated fibroblast-like tumor cells remodel the Ewing sarcoma tumor microenvironment\",\n",
      "        \"abstract\": \"Tumor heterogeneity is a major driver of cancer progression. In epithelial-derived malignancies, carcinoma-associated fibroblasts (CAFs) contribute to tumor heterogeneity by depositing extracellular matrix (ECM) proteins that dynamically remodel the tumor microenvironment (TME). Ewing sarcomas (EwS) are histologically monomorphous, mesenchyme-derived tumors that are devoid of CAFs. Here we identify a previously uncharacterized subpopulation of transcriptionally distinct EwS tumor cells that deposit pro-tumorigenic ECM. Single cell analyses revealed that these CAF-like cells differ from bulk EwS cells by their upregulation of a matrisome-rich gene signature that is normally repressed by EWS::FLI1, the oncogenic fusion transcription factor that underlies EwS pathogenesis. Further, our studies showed that ECM-depositing tumor cells express the cell surface marker CD73, allowing for their isolation ex vivo and detection in situ. Spatial profiling of tumor xenografts and patient biopsies demonstrated that CD73+ EwS cells and tumor cell-derived ECM are prevalent along tumor borders and invasive fronts. Importantly, despite loss of EWS::FLI1-mediated gene repression, CD73+ EwS cells retain expression of EWS::FLI1 and the fusion-activated gene signature, as well as tumorigenic and proliferative capacities. Thus, EwS tumor cells can be reprogrammed to adopt CAF-like properties and these transcriptionally and phenotypically distinct cell subpopulations contribute to tumor heterogeneity by remodeling the TME.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=293&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"51a63cbe11e62d4cf615e3af473527b5ba5c8f2e\",\n",
      "        \"title\": \"Unraveling the heterogeneity of cholangiocarcinoma and identifying biomarkers and therapeutic strategies with single-cell sequencing technology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=294&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2b4936365ef59dc5b74aff8aa5964c9ecb7bc20c\",\n",
      "        \"title\": \"Targeting CXCL16 and STAT1 augments immune checkpoint blockade therapy in triple-negative breast cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=295&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6458dabea2566cca45d67410992978f120e974b7\",\n",
      "        \"title\": \"Advanced analysis and applications of single-cell transcriptome sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=296&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c58139b9b5726fb311956d1054ae57a5b65cb7df\",\n",
      "        \"title\": \"Dynamics and specificities of T cells in cancer immunotherapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=297&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6f21ed485a0412a21a5f0a1968a1e46806b3150\",\n",
      "        \"title\": \"Single-cell measurement of dynamic protein secretion and transcriptome\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=298&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9b9e42852ef9a632cb8f5188b9f748a0fc888b8d\",\n",
      "        \"title\": \"The Chromatin Landscape of Pathogenic Transcriptional Cell States in Rheumatoid Arthritis\",\n",
      "        \"abstract\": \"Synovial tissue inflammation is the hallmark of rheumatoid arthritis (RA). Recent work has identified prominent pathogenic cell states in inflamed RA synovial tissue, such as T peripheral helper cells; however, the epigenetic regulation of these states has yet to be defined. We measured genome-wide open chromatin at single cell resolution from 30 synovial tissue samples, including 12 samples with transcriptional data in multimodal experiments. We identified 24 chromatin classes and predicted their associated transcription factors, including a CD8+ GZMK+ class associated with EOMES and a lining fibroblast class associated with AP-1. By integrating an RA tissue transcriptional atlas, we found that the chromatin classes represented \\u2018superstates\\u2019 corresponding to multiple transcriptional cell states. Finally, we demonstrated the utility of this RA tissue chromatin atlas through the associations between disease phenotypes and chromatin class abundance as well as the nomination of classes mediating the effects of putatively causal RA genetic variants.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=299&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"336089405221802740a9c470400d41c16b3c8bb5\",\n",
      "        \"title\": \"Germline-encoded amino acid-binding motifs drive immunodominant public antibody responses\",\n",
      "        \"abstract\": \"Despite the vast diversity of the antibody repertoire, infected individuals often mount antibody responses to precisely the same epitopes within antigens. The immunological mechanisms underpinning this phenomenon remain unknown. By mapping 376 immunodominant \\u201cpublic epitopes\\u201d at high resolution and characterizing several of their cognate antibodies, we concluded that germline-encoded sequences in antibodies drive recurrent recognition. Systematic analysis of antibody-antigen structures uncovered 18 human and 21 partially overlapping mouse germline-encoded amino acid\\u2013binding (GRAB) motifs within heavy and light V gene segments that in case studies proved critical for public epitope recognition. GRAB motifs represent a fundamental component of the immune system\\u2019s architecture that promotes recognition of pathogens and leads to species-specific public antibody responses that can exert selective pressure on pathogens. Description Antibodies GRAB their targets Different individuals will reproducibly generate antibodies against the same epitopes after exposure to a pathogen. However, the mechanisms driving the production of these \\u201cpublic antibodies\\u201d continue to be poorly understood. Shrock et al. used a phage display platform called VirScan to glean additional insights into this process. They found that many human V gene segments contained germline-encoded amino acid\\u2013binding (GRAB) motifs that bind particular amino acids and are essential for antibody recognition. Moreover, a comparison of mouse and human GRAB motifs uncovered only a partial overlap, which may explain the distinct public epitopes targeted by different species. \\u2014STS Antibody genes have GRAB motifs that bind specific amino acids and drive reproducible responses to immunodominant epitopes. INTRODUCTION Antibodies are generated by a DNA recombination mechanism occurring in the immunoglobulin heavy and light chain genes in which modular VDJ (for heavy) and VJ (for light) gene segments are combinatorially assembled. The vast complexity of the antibody repertoire allows many species to generate antibodies against virtually any protein. However, when different individuals are exposed to a given pathogen they often mount antibody responses to the same precise protein regions\\u2014or epitopes\\u2014from the pathogen. The mechanisms underlying these recurrent antibody responses to immunodominant \\u201cpublic epitopes\\u201d are not well understood. RATIONALE We set out to identify a collection of immunodominant public epitopes that would allow us to study mechanisms underlying recurrent antibody responses. We employed VirScan\\u2014a phage display platform programmed to display peptides covering the entire human virome\\u2014to identify the epitopes of antiviral antibodies from a large cohort of individuals in a high-throughput manner. Additionally, we isolated B cell receptors from different individuals that bound to model public epitopes in order to investigate their determinants of specificity. Finally, we performed a systematic analysis of antibody\\u2013antigen structures from the Protein Data Bank (PDB) to search for recurrent modes of antigen recognition. RESULTS We mapped 376 immunodominant public epitopes from 51 viral species to single-amino-acid resolution. Antibodies from different individuals that recognized the same public epitope often (i) shared light chain isotype (kappa or lambda) and (ii) bound the same precise critical residues in the epitope. Public epitopes showed biased amino acid composition, including a striking enrichment of lysine at the borders of public epitopes recognized by antibodies with lambda light chains. We examined 50 B cell receptors recognizing three model public epitopes in detail and observed conserved V gene segment usage but almost no conservation of heavy chain CDR3 sequences, indicating that key specificity determinants lay within the V gene segments themselves. Structural analysis of antibody\\u2013antigen complexes in the PDB uncovered 18 human V gene segments that harbor germline-encoded amino acid\\u2013binding (GRAB) motifs that specifically bind to particular amino acids. Among these were a family of six closely related lambda V gene segments with similar GRAB motifs specific for border lysines. We confirmed that the GRAB motifs we identified were critical for antibody recognition of two model public epitopes. Analysis of murine antibody\\u2013antigen structures revealed 21 V gene segment\\u2013encoded GRAB motifs that only partially overlapped with the human GRAB motifs, which may explain why there is little overlap between the public epitopes recognized across species. Thus, there appears to be a structural basis underlying the notable convergence in humoral immune responses to immunodominant public epitopes across humans and the differing public epitope selection across species. CONCLUSION Recurrent antibody responses to immunodominant public epitopes are a general feature of humoral immunity. We propose that they are driven by GRAB motifs, a germline-encoded component of the architecture of the antibody repertoire that predisposes antibodies to recognize particular structures and thus influences epitope selection and composition. Public epitopes likely arise in part because they are best aligned for recognition by GRAB motifs and can thus be bound by a relatively large precursor pool of B cells. GRAB motifs may have evolved to ensure efficient antibody responses to pathogens; the recurrent responses they engender across populations likely exert selective pressure on pathogens and influence host\\u2013pathogen coevolution. Uncovering origins of public antibody responses. VirScan reveals shared antibody responses to certain regions of viral proteins (\\u201cpublic epitopes\\u201d) across human populations. Heatmap color represents the strength of the antibody response in each individual (columns) to each peptide of a viral protein (rows). An analysis of antibody\\u2013antigen structures reveals that antibody genes harbor germline-encoded motifs that specifically bind certain amino acids and drive recognition of public epitopes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=300&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e951b240611008fa715157f9f55aaea3f22b4f67\",\n",
      "        \"title\": \"High-dimensional immune cell profiling of cerebrospinal fluid from patients with metastatic breast cancer and leptomeningeal disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=301&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3b9f94e764c8d950027fb05b0ad4d90d8d52c269\",\n",
      "        \"title\": \"Spatial Transcriptomics: Technical Aspects of Recent Developments and Their Applications in Neuroscience and Cancer Research\",\n",
      "        \"abstract\": \"Spatial transcriptomics is a newly emerging field that enables high\\u2010throughput investigation of the spatial localization of transcripts and related analyses in various applications for biological systems. By transitioning from conventional biological studies to \\u201cin situ\\u201d biology, spatial transcriptomics can provide transcriptome\\u2010scale spatial information. Currently, the ability to simultaneously characterize gene expression profiles of cells and relevant cellular environment is a paradigm shift for biological studies. In this review, recent progress in spatial transcriptomics and its applications in neuroscience and cancer studies are highlighted. Technical aspects of existing technologies and future directions of new developments (as of March 2023), computational analysis of spatial transcriptome data, application notes in neuroscience and cancer studies, and discussions regarding future directions of spatial multi\\u2010omics and their expanding roles in biomedical applications are emphasized.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=302&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa09d58167206ec28ba15a532a5e0d98008adda2\",\n",
      "        \"title\": \"exFINDER: identify external communication signals using single-cell transcriptomics data\",\n",
      "        \"abstract\": \"Abstract Cells make decisions through their communication with other cells and receiving signals from their environment. Using single-cell transcriptomics, computational tools have been developed to infer cell\\u2013cell communication through ligands and receptors. However, the existing methods only deal with signals sent by the measured cells in the data, the received signals from the external system are missing in the inference. Here, we present exFINDER, a method that identifies such external signals received by the cells in the single-cell transcriptomics datasets by utilizing the prior knowledge of signaling pathways. In particular, exFINDER can uncover external signals that activate the given target genes, infer the external signal-target signaling network (exSigNet), and perform quantitative analysis on exSigNets. The applications of exFINDER to scRNA-seq datasets from different species demonstrate the accuracy and robustness of identifying external signals, revealing critical transition-related signaling activities, inferring critical external signals and targets, clustering signal-target paths, and evaluating relevant biological events. Overall, exFINDER can be applied to scRNA-seq data to reveal the external signal-associated activities and maybe novel cells that send such signals.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=303&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e640cc5cc7733c9a978cab9ee3ce957015aff699\",\n",
      "        \"title\": \"Recent revelations and future directions using single-cell technologies in chronic lymphocytic leukemia\",\n",
      "        \"abstract\": \"Chronic lymphocytic leukemia (CLL) is a clinically and biologically heterogeneous disease with varying outcomes. In the last decade, the application of next-generation sequencing technologies has allowed extensive mapping of disease-specific genomic, epigenomic, immunogenetic, and transcriptomic signatures linked to CLL pathogenesis. These technologies have improved our understanding of the impact of tumor heterogeneity and evolution on disease outcome, although they have mostly been performed on bulk preparations of nucleic acids. As a further development, new technologies have emerged in recent years that allow high-resolution mapping at the single-cell level. These include single-cell RNA sequencing for assessment of the transcriptome, both of leukemic and non-malignant cells in the tumor microenvironment; immunogenetic profiling of B and T cell receptor rearrangements; single-cell sequencing methods for investigation of methylation and chromatin accessibility across the genome; and targeted single-cell DNA sequencing for analysis of copy-number alterations and single nucleotide variants. In addition, concomitant profiling of cellular subpopulations, based on protein expression, can also be obtained by various antibody-based approaches. In this review, we discuss different single-cell sequencing technologies and how they have been applied so far to study CLL onset and progression, also in response to treatment. This latter aspect is particularly relevant considering that we are moving away from chemoimmunotherapy to targeted therapies, with a potentially distinct impact on clonal dynamics. We also discuss new possibilities, such as integrative multi-omics analysis, as well as inherent limitations of the different single-cell technologies, from sample preparation to data interpretation using available bioinformatic pipelines. Finally, we discuss future directions in this rapidly evolving field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=304&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ab764c5c5f524af1e3f6ba252045a78a2876aa57\",\n",
      "        \"title\": \"Multi-omic analyses in immune cell development with lessons learned from T cell development\",\n",
      "        \"abstract\": \"Traditionally, flow cytometry has been the preferred method to characterize immune cells at the single-cell level. Flow cytometry is used in immunology mostly to measure the expression of identifying markers on the cell surface, but\\u2014with good antibodies\\u2014can also be used to assess the expression of intracellular proteins. The advent of single-cell RNA-sequencing has paved the road to study immune development at an unprecedented resolution. Single-cell RNA-sequencing studies have not only allowed us to efficiently chart the make-up of heterogeneous tissues, including their most rare cell populations, it also increasingly contributes to our understanding how different omics modalities interplay at a single cell resolution. Particularly for investigating the immune system, this means that these single-cell techniques can be integrated to combine and correlate RNA and protein data at the single-cell level. While RNA data usually reveals a large heterogeneity of a given population identified solely by a combination of surface protein markers, the integration of different omics modalities at a single cell resolution is expected to greatly contribute to our understanding of the immune system.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=305&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0e6e6470ece1c7eff14742c3bd35f4af7987ea28\",\n",
      "        \"title\": \"Single-cell multi-omics sequencing and its application in tumor heterogeneity.\",\n",
      "        \"abstract\": \"In recent years, the emergence and development of single-cell sequencing technologies have provided unprecedented opportunities to analyze deoxyribonucleic acid, ribonucleic acid and proteins at single-cell resolution. The advancements and reduced costs of high-throughput technologies allow for parallel sequencing of multiple molecular layers from a single cell, providing a comprehensive insight into the biological state and behavioral mechanisms of cells through the integration of genomics, transcriptomics, epigenomics and proteomics information. Researchers are actively working to further improve the cost-effectiveness, stability and high-throughput capabilities of single-cell multi-omics sequencing technologies and exploring their potential in precision medicine through clinical diagnostics. This review aims to survey the cutting-edge advancements in single-cell multi-omics sequencing, summarizing the representative technologies and their applications in profiling complex diseases, with a particular focus on tumors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=306&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fc42eedb665aa29d9ff7b766dfeb23c7f8909a02\",\n",
      "        \"title\": \"Multi-omic single cell sequencing: Overview and opportunities for kidney disease therapeutic development\",\n",
      "        \"abstract\": \"Single cell sequencing technologies have rapidly advanced in the last decade and are increasingly applied to gain unprecedented insights by deconstructing complex biology to its fundamental unit, the individual cell. First developed for measurement of gene expression, single cell sequencing approaches have evolved to allow simultaneous profiling of multiple additional features, including chromatin accessibility within the nucleus and protein expression at the cell surface. These multi-omic approaches can now further be applied to cells in situ, capturing the spatial context within which their biology occurs. To extract insights from these complex datasets, new computational tools have facilitated the integration of information across different data types and the use of machine learning approaches. Here, we summarize current experimental and computational methods for generation and integration of single cell multi-omic datasets. We focus on opportunities for multi-omic single cell sequencing to augment therapeutic development for kidney disease, including applications for biomarkers, disease stratification and target identification.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=307&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d0c930590c0c0327fe698a6e5667ee7800ed6f1\",\n",
      "        \"title\": \"Slide-tags: scalable, single-nucleus barcoding for multi-modal spatial genomics\",\n",
      "        \"abstract\": \"Recent technological innovations have enabled the high-throughput quantification of gene expression and epigenetic regulation within individual cells, transforming our understanding of how complex tissues are constructed. Missing from these measurements, however, is the ability to routinely and easily spatially localise these profiled cells. We developed a strategy, Slide-tags, in which single nuclei within an intact tissue section are \\u2018tagged\\u2019 with spatial barcode oligonucleotides derived from DNA-barcoded beads with known positions. These tagged nuclei can then be used as input into a wide variety of single-nucleus profiling assays. Application of Slide-tags to the mouse hippocampus positioned nuclei at less than 10 micron spatial resolution, and delivered whole-transcriptome data that was indistinguishable in quality from ordinary snRNA-seq. To demonstrate that Slide-tags can be applied to a wide variety of human tissues, we performed the assay on brain, tonsil, and melanoma. We revealed cell-type-specific spatially varying gene expression across cortical layers and spatially contextualised receptor-ligand interactions driving B-cell maturation in lymphoid tissue. A major benefit of Slide-tags is that it is easily adaptable to virtually any single-cell measurement technology. As proof of principle, we performed multiomic measurements of open chromatin, RNA, and T-cell receptor sequences in the same cells from metastatic melanoma. We identified spatially distinct tumour subpopulations to be differentially infiltrated by an expanded T-cell clone and undergoing cell state transition driven by spatially clustered accessible transcription factor motifs. Slide-tags offers a universal platform for importing the compendium of established single-cell measurements into the spatial genomics repertoire.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=308&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"01e57478dd3edf1cbdebaf62c460cd68afd92156\",\n",
      "        \"title\": \"The Neurovasculome: Key Roles in Brain Health and Cognitive Impairment: A Scientific Statement From the American Heart Association/American Stroke Association\",\n",
      "        \"abstract\": \"Background: Preservation of brain health has emerged as a leading public health priority for the aging world population. Advances in neurovascular biology have revealed an intricate relationship among brain cells, meninges, and the hematic and lymphatic vasculature (the neurovasculome) that is highly relevant to the maintenance of cognitive function. In this scientific statement, a multidisciplinary team of experts examines these advances, assesses their relevance to brain health and disease, identifies knowledge gaps, and provides future directions. Methods: Authors with relevant expertise were selected in accordance with the American Heart Association conflict-of-interest management policy. They were assigned topics pertaining to their areas of expertise, reviewed the literature, and summarized the available data. Results: The neurovasculome, composed of extracranial, intracranial, and meningeal vessels, as well as lymphatics and associated cells, subserves critical homeostatic functions vital for brain health. These include delivering O2 and nutrients through blood flow and regulating immune trafficking, as well as clearing pathogenic proteins through perivascular spaces and dural lymphatics. Single-cell omics technologies have unveiled an unprecedented molecular heterogeneity in the cellular components of the neurovasculome and have identified novel reciprocal interactions with brain cells. The evidence suggests a previously unappreciated diversity of the pathogenic mechanisms by which disruption of the neurovasculome contributes to cognitive dysfunction in neurovascular and neurodegenerative diseases, providing new opportunities for the prevention, recognition, and treatment of these conditions. Conclusions: These advances shed new light on the symbiotic relationship between the brain and its vessels and promise to provide new diagnostic and therapeutic approaches for brain disorders associated with cognitive dysfunction.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=309&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fb74c5d83d10f2e10cd6685e18d84c4586504b10\",\n",
      "        \"title\": \"Multiomics Empowers Predictive Pancreatic Cancer Immunotherapy.\",\n",
      "        \"abstract\": \"Advances in cancer immunotherapy, particularly immune checkpoint inhibitors, have dramatically improved the prognosis for patients with metastatic melanoma and other previously incurable cancers. However, patients with pancreatic ductal adenocarcinoma (PDAC) generally do not respond to these therapies. PDAC is exceptionally difficult to treat because of its often late stage at diagnosis, modest mutation burden, and notoriously complex and immunosuppressive tumor microenvironment. Simultaneously interrogating features of cancer, immune, and other cellular components of the PDAC tumor microenvironment is therefore crucial for identifying biomarkers of immunotherapeutic resistance and response. Notably, single-cell and multiomics technologies, along with the analytical tools for interpreting corresponding data, are facilitating discoveries of the systems-level cellular and molecular interactions contributing to the overall resistance of PDAC to immunotherapy. Thus, in this review, we will explore how multiomics and single-cell analyses provide the unprecedented opportunity to identify biomarkers of resistance and response to successfully sensitize PDAC to immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=310&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4560f82abf42999628706331c6a5f82ba1130dd1\",\n",
      "        \"title\": \"scJVAE: A novel method for integrative analysis of multimodal single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=311&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a74c62ce4a48c5f174ad2239466a46e5c29ffcb\",\n",
      "        \"title\": \"The T-cell-directed vaccine BNT162b4 encoding conserved non-spike antigens protects animals from severe SARS-CoV-2 infection\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=312&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b403877e42603411e241f865600b3601d91d5800\",\n",
      "        \"title\": \"Accelerating the understanding of cancer biology through the lens of genomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=313&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b017cc33f5bd4d74e05da7940223e9324ba2797a\",\n",
      "        \"title\": \"Sequence-Based Platforms for Discovering Biomarkers in Liquid Biopsy of Non-Small-Cell Lung Cancer\",\n",
      "        \"abstract\": \"Simple Summary A lack of sensitive biomarkers hinders lung cancer detection and monitoring, resulting in late diagnosis and missed treatment response. Liquid biopsies have recently emerged as a promising method of detecting biomarkers in lung cancer patients without the need to conduct invasive procedures. New approaches for biomarker discovery have emerged due to advances in high-throughput sequencing technologies and bioinformatics tools. In this review, we comprehensively describe established and emerging methods for identifying lung cancer biomarkers using liquid biopsy. Furthermore, we highlight advanced bioinformatics tools and methods for processing NGS data, as well as recently developed software tailored for liquid biopsy biomarker detection. Abstract Lung cancer detection and monitoring are hampered by a lack of sensitive biomarkers, which results in diagnosis at late stages and difficulty in tracking response to treatment. Recent developments have established liquid biopsies as promising non-invasive methods for detecting biomarkers in lung cancer patients. With concurrent advances in high-throughput sequencing technologies and bioinformatics tools, new approaches for biomarker discovery have emerged. In this article, we survey established and emerging biomarker discovery methods using nucleic acid materials derived from bodily fluids in the context of lung cancer. We introduce nucleic acid biomarkers extracted from liquid biopsies and outline biological sources and methods of isolation. We discuss next-generation sequencing (NGS) platforms commonly used to identify novel biomarkers and describe how these have been applied to liquid biopsy. We highlight emerging biomarker discovery methods, including applications of long-read sequencing, fragmentomics, whole-genome amplification methods for single-cell analysis, and whole-genome methylation assays. Finally, we discuss advanced bioinformatics tools, describing methods for processing NGS data, as well as recently developed software tailored for liquid biopsy biomarker detection, which holds promise for early diagnosis of lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=314&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9721bacbaab9f9921cf46aefd29ca11be5dd9be2\",\n",
      "        \"title\": \"Multi-view clustering by CPS-merge analysis with application to multimodal single-cell data\",\n",
      "        \"abstract\": \"Multi-view data can be generated from diverse sources, by different technologies, and in multiple modalities. In various fields, integrating information from multi-view data has pushed the frontier of discovery. In this paper, we develop a new approach for multi-view clustering, which overcomes the limitations of existing methods such as the need of pooling data across views, restrictions on the clustering algorithms allowed within each view, and the disregard for complementary information between views. Our new method, called CPS-merge analysis, merges clusters formed by the Cartesian product of single-view cluster labels, guided by the principle of maximizing clustering stability as evaluated by CPS analysis. In addition, we introduce measures to quantify the contribution of each view to the formation of any cluster. CPS-merge analysis can be easily incorporated into an existing clustering pipeline because it only requires single-view cluster labels instead of the original data. We can thus readily apply advanced single-view clustering algorithms. Importantly, our approach accounts for both consensus and complementary effects between different views, whereas existing ensemble methods focus on finding a consensus for multiple clustering results, implying that results from different views are variations of one clustering structure. Through experiments on single-cell datasets, we demonstrate that our approach frequently outperforms other state-of-the-art methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=315&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"708909ebb03b35e3e5164da95989e8cb340bcb36\",\n",
      "        \"title\": \"A marker gene-based method for identifying the cell-type of origin from single-cell RNA sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=316&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7324f3e28dc723fec77b21d6f4649e7230ad61a3\",\n",
      "        \"title\": \"Deciphering the tumour immune microenvironment cell by cell\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=317&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4a73897d6556e39ef156992b8959aa4b9e592deb\",\n",
      "        \"title\": \"Best practices for single-cell analysis across modalities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=318&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f00b015cac7e9368a0afb1c60fde3d74680bec96\",\n",
      "        \"title\": \"kNN-Res: Residual Neural Network with kNN-Graph coherence for point cloud registration\",\n",
      "        \"abstract\": \"In this paper, we present a residual neural network-based method for point set registration that preserves the topological structure of the target point set. Similar to coherent point drift (CPD), the registration (alignment) problem is viewed as the movement of data points sampled from a target distribution along a regularized displacement vector field. While the coherence constraint in CPD is stated in terms of local motion coherence, the proposed regularization term relies on a global smoothness constraint as a proxy for preserving local topology. This makes CPD less flexible when the deformation is locally rigid but globally non-rigid as in the case of multiple objects and articulate pose registration. A Jacobian-based cost function and geometric-aware statistical distances are proposed to mitigate these issues. The latter allows for measuring misalignment between the target and the reference. The justification for the k-Nearest Neighbour(kNN) graph preservation of target data, when the Jacobian cost is used, is also provided. Further, to tackle the registration of high-dimensional point sets, a constant time stochastic approximation of the Jacobian cost is introduced. The proposed method is illustrated on several 2-dimensional toy examples and tested on high-dimensional flow Cytometry datasets where the task is to align two distributions of cells whilst preserving the kNN-graph in order to preserve the biological signal of the transformed data. The implementation of the proposed approach is available at https://github.com/MuhammadSaeedBatikh/kNN-Res_Demo/ under the MIT license.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=319&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0d229da0b51713393e833f296902c3179e8bc63d\",\n",
      "        \"title\": \"Single-cell profiling to explore pancreatic cancer heterogeneity, plasticity and response to therapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=320&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7601a74cfc8e50199242de6ea2c121c7eb4d7671\",\n",
      "        \"title\": \"Temporal multimodal single-cell profiling of native hematopoiesis illuminates altered differentiation trajectories with age.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=321&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"091591c6a6f7620dcb7ec0a93ee17b1a19b7b747\",\n",
      "        \"title\": \"Opportunities for High-plex Spatial Transcriptomics in Solid Organ Transplantation.\",\n",
      "        \"abstract\": \"The last 5 y have seen the development and widespread adoption of high-plex spatial transcriptomic technology. This technique detects and quantifies mRNA transcripts in situ, meaning that transcriptomic signatures can be sampled from specific cells, structures, lesions, or anatomical regions while conserving the physical relationships that exist within complex tissues. These methods now frequently implement next-generation sequencing, enabling the simultaneous measurement of many targets, up to and including the whole mRNA transcriptome. To date, spatial transcriptomics has been foremost used in the fields of neuroscience and oncology, but there is potential for its use in transplantation sciences. Transplantation has a clear dependence on biopsies for diagnosis, monitoring, and research. Transplant patients represent a unique cohort with multiple organs of interest, clinical courses, demographics, and immunosuppressive regimens. Obtaining high complexity data on the disease processes underlying rejection, tolerance, infection, malignancy, and injury could identify new opportunities for therapeutic intervention and biomarker identification. In this review, we discuss currently available spatial transcriptomic technologies and how they can be applied to transplantation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=322&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"48b031325f37c4d00731ef4e96d895580f334753\",\n",
      "        \"title\": \"Multiscale integration of human and single-cell variations reveals unadjuvanted vaccine high responders are naturally adjuvanted\",\n",
      "        \"abstract\": \"Advances in multimodal single cell analysis can empower high-resolution dissection of human vaccination responses. The resulting data capture multiple layers of biological variations, including molecular and cellular states, vaccine formulations, inter- and intra-subject differences, and responses unfolding over time. Transforming such data into biological insight remains a major challenge. Here we present a systematic framework applied to multimodal single cell data obtained before and after influenza vaccination without adjuvants or pandemic H5N1 vaccination with the AS03 adjuvant. Our approach pinpoints responses shared across or unique to specific cell types and identifies adjuvant specific signatures, including pro-survival transcriptional states in B lymphocytes that emerged one day after vaccination. We also reveal that high antibody responders to the unadjuvanted vaccine have a distinct baseline involving a rewired network of cell type specific transcriptional states. Remarkably, the status of certain innate immune cells in this network in high responders of the unadjuvanted vaccine appear \\\"naturally adjuvanted\\\": they resemble phenotypes induced early in the same cells only by vaccination with AS03. Furthermore, these cell subsets have elevated frequency in the blood at baseline and increased cell-intrinsic phospho-signaling responses after LPS stimulation ex vivo in high compared to low responders. Our findings identify how variation in the status of multiple immune cell types at baseline may drive robust differences in innate and adaptive responses to vaccination and thus open new avenues for vaccine development and immune response engineering in humans.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=323&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"11013bca9ea2ef981e7ca937265aa244ead22957\",\n",
      "        \"title\": \"Shaping the brain vasculature in development and disease in the single-cell era\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=324&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"19efaf8c2000f99267163fc3a9900dc59aba106d\",\n",
      "        \"title\": \"Phenotypic and spatial heterogeneity of brain myeloid cells after stroke is associated with cell ontogeny, tissue damage, and brain connectivity\",\n",
      "        \"abstract\": \"Acute stroke causes substantial mortality and morbidity and provokes extensive changes to myeloid immune cell populations in the brain that may be targets for limiting brain damage and enhancing brain repair. The most effective immunomodulatory approaches will require precise manipulation of discrete myeloid cell phenotypes in time and space to avoid harmful effects of indiscriminate neuroimmune perturbation. We sought to define how stroke alters the composition and phenotypes of mononuclear myeloid cells with particular attention to how cell ontogeny and spatial organisation combine to expand myeloid cell diversity across the brain after stroke. Multiple reactive microglial states and dual monocyte-derived populations contributed to an extensive repertoire of myeloid cells in post-stroke brain. We identified important overlap and distinctions among different cell types and states that involved ontogeny- and spatial-related properties. Notably, brain connectivity with infarcted tissue underpinned the pattern of local and remote altered cell accumulation and reactivity. Our discoveries suggest a global but anatomically-governed brain myeloid cell response to stroke that comprises diverse phenotypes arising through intrinsic cell ontogeny factors interacting with exposure to spatially-organised brain damage and neuroaxonal cues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=325&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"379cf38c252830dc90d182f0b43591a44e4a8f83\",\n",
      "        \"title\": \"A universal framework for single-cell multi-omics data integration with graph convolutional networks\",\n",
      "        \"abstract\": \"Abstract Single-cell omics data are growing at an unprecedented rate, whereas effective integration of them remains challenging due to different sequencing methods, quality, and expression pattern of each omics data. In this study, we propose a universal framework for the integration of single-cell multi-omics data based on graph convolutional network (GCN-SC). Among the multiple single-cell data, GCN-SC usually selects one data with the largest number of cells as the reference and the rest as the query dataset. It utilizes mutual nearest neighbor algorithm to identify cell-pairs, which provide connections between cells both within and across the reference and query datasets. A GCN algorithm further takes the mixed graph constructed from these cell-pairs to adjust count matrices from the query datasets. Finally, dimension reduction is performed by using non-negative matrix factorization before visualization. By applying GCN-SC on six datasets, we show that GCN-SC can effectively integrate sequencing data from multiple single-cell sequencing technologies, species or different omics, which outperforms the state-of-the-art methods, including Seurat, LIGER, GLUER and Pamona.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=326&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bcb83da0022dbc3c89ec05eca413c22aa00b5cf3\",\n",
      "        \"title\": \"InClust+: the multimodal version of inClust for multimodal data integration, imputation, and cross modal generation\",\n",
      "        \"abstract\": \"With the development of single-cell technology, many cell traits (e.g. gene expression, chromatin accessibility, DNA methylation) can be measured. Furthermore, the multi-omic profiling technology could jointly measure two or more traits in a single cell simultaneously. In order to process the various data accumulated rapidly, computational methods for multimodal data integration are needed. Previously, we developed inClust, a flexible all-in deep generative framework for transcriptome data. Here, we extend the applicability of inClust into the realm of multimodal data by adding two mask modules: an input-mask module in front of the encoder and an output-mask module behind the decoder. We call this augmented model inClust+, and apply it to various multimodal data. InClust+ was first used to integrate scRNA and MERFISH data from similar cell populations and to impute MERFISH data based on scRNA data. Then, inClust+ is shown to have the capability to integrate a multimodal data contain scRNA and scATAC or two multimodal CITE datasets with batch effect. Finally, inClust+ is used to integrate a monomodal scRNA dataset and two multimodal CITE datasets, and generate the missing modality of surface protein in monomodal scRNA data. In the above examples, the performance of inClust+ is better than or comparable to the most recent tools to the corresponding task, which prove inClust+ is a suitable framework for handling multimodal data. Meanwhile, the successful implementation of mask in inClust+ means that it can be applied to other deep learning methods with similar encoder-decoder architecture to broaden the application scope of these models.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=327&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"857a20b13f18759c5815214d5a2ee7eba608944c\",\n",
      "        \"title\": \"RNA regulation in brain function and disease 2022 (NeuroRNA): A conference report\",\n",
      "        \"abstract\": \"Recent research integrates novel technologies and methods from the interface of RNA biology and neuroscience. This advancing integration of both fields creates new opportunities in neuroscience to deepen the understanding of gene expression programs and their regulation that underlies the cellular heterogeneity and physiology of the central nervous system. Currently, transcriptional heterogeneity can be studied in individual neural cell types in health and disease. Furthermore, there is an increasing interest in RNA technologies and their application in neurology. These aspects were discussed at an online conference that was shortly named NeuroRNA.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=328&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c0ff113045ca0d562829c064a4aa48f820f4797b\",\n",
      "        \"title\": \"Droplet-based forward genetic screening of astrocyte\\u2013microglia cross-talk\",\n",
      "        \"abstract\": \"Cell\\u2013cell interactions in the central nervous system play important roles in neurologic diseases. However, little is known about the specific molecular pathways involved, and methods for their systematic identification are limited. Here, we developed a forward genetic screening platform that combines CRISPR-Cas9 perturbations, cell coculture in picoliter droplets, and microfluidic-based fluorescence-activated droplet sorting to identify mechanisms of cell\\u2013cell communication. We used SPEAC-seq (systematic perturbation of encapsulated associated cells followed by sequencing), in combination with in vivo genetic perturbations, to identify microglia-produced amphiregulin as a suppressor of disease-promoting astrocyte responses in multiple sclerosis preclinical models and clinical samples. Thus, SPEAC-seq enables the high-throughput systematic identification of cell\\u2013cell communication mechanisms. Description Dissecting cell-cell interactions Systematically determining cell-cell interactions, particularly in the brain, is a technical challenge. Wheeler et al. developed a microfluidic platform for the identification of mechanisms of cell-cell communication. SPEAC-seq (systematic perturbation of encapsulated associated cells followed by sequencing) allowed the sorting of pairs of cells co-cultured in droplets. Studying astrocyte-microglia pairs, SPEAC-seq identified microglial-derived amphiregulin as a negative regulator of astrocyte nuclear factor kappa B activation. In multiple sclerosis patients and animal models, astrocyte-derived interleukin-33 activated a receptor in microglia, inducing amphiregulin expression and dampening disease-promoting astrocyte responses. SPEAC-seq provides a powerful tool to systematically investigate the mechanisms of cell-cell communication. \\u2014SMH A high-throughput platform enables the identification of cell-cell interaction mechanisms in forward genetic screens.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=329&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8a99fea4f63727f78af57197a1e0dd5425dbe0ec\",\n",
      "        \"title\": \"Selection of antibody-binding covalent aptamers\",\n",
      "        \"abstract\": \"Aptamers are oligonucleotides with antibody-like binding function, selected from large combinatorial libraries. In this study, we modified a DNA aptamer library with N-hydroxysuccinimide esters, enabling covalent reactivity with cognate proteins. We selected for the ability to bind to mouse monoclonal antibodies, resulting in the isolation of two distinct covalent binding motifs. The covalent aptamers are specific for the Fc region of mouse monoclonal IgG1 and are cross-reactive with mouse IgG2a and other IgGs. Investigation into the covalent reactivity of the aptamers revealed a dependence on micromolar concentrations of Cu2+ ions which can be explained by residual catalyst remaining after modification of the aptamer library. The aptamers were successfully used as adapters in the formation of antibody-oligonucleotide conjugates (AOCs) for use in detection of HIV protein p24 and super-resolution imaging of actin. This work introduces a new method for the site-specific modification of native monoclonal antibodies and may be useful in applications requiring AOCs or other antibody conjugates.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=330&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ebcadc44d6e5d0aae7007daf6e8417b8d872dd5c\",\n",
      "        \"title\": \"Standardization of flow cytometry and cell sorting to enable a transcriptomic analysis in a multi-site sarcoidosis study\",\n",
      "        \"abstract\": \"The contribution and regulation of various CD4+ T cell lineages that occur with remitting vs progressive courses in sarcoidosis are poorly understood. We developed a multiparameter flow cytometry panel to sort these CD4+ T cell lineages followed by measurement of their functional potential using RNA-sequencing analysis at six-month intervals across multiple study sites. To obtain good quality RNA for sequencing, we relied on chemokine receptor expression to identify and sort lineages. To minimize gene expression changes induced by perturbations of T cells and avoid protein denaturation caused by freeze/thaw cycles, we optimized our protocols using freshly isolated samples at each study site. To accomplish this study, we had to overcome significant standardization challenges across multiple sites. Here, we detail standardization considerations for cell processing, flow staining, data acquisition, sorting parameters, and RNA quality control analysis that were performed as part of the NIH-sponsored, multi-center study, BRonchoscopy at Initial sarcoidosis diagnosis Targeting longitudinal Endpoints (BRITE). After several rounds of iterative optimization, we identified the following aspects as critical for successful standardization: 1) alignment of PMT voltages across sites using CS&T/rainbow bead technology; 2) a single template created in the cytometer program that was used by all sites to gate cell populations during data acquisition and cell sorting; 3) use of standardized lyophilized flow cytometry staining cocktails to reduce technical error during processing; 4) development and implementation of a standardized Manual of Procedures. After standardization of cell sorting, we were able to determine the minimum number of sorted cells necessary for next generation sequencing through analysis of RNA quality and quantity from sorted T cell populations. Overall, we found that implementing a multi-parameter cell sorting with RNA-seq analysis clinical study across multiple study sites requires iteratively tested standardized procedures to ensure comparable and high-quality results.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=331&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59a0da2761871e64931a9a4d3afe50105a9e5d84\",\n",
      "        \"title\": \"COVID-19db linkage maps of cell surface proteins and transcription factors in immune cells\",\n",
      "        \"abstract\": \"The highly contagious SARS-CoV-2 and its associated disease (COVID-19) are a threat to global public health and economies. To develop effective treatments for COVID-19, we must understand the host cell types, cell states and regulators associated with infection and pathogenesis such as dysregulated transcription factors (TFs) and surface proteins, including signalling receptors. To link cell surface proteins with TFs, we recently developed SPaRTAN (Single-cell Proteomic and RNA-based Transcription factor Activity Network) by integrating parallel single-cell proteomic and transcriptomic data based on Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) and gene cis-regulatory information. We apply SPaRTAN to CITE-seq datasets from patients with varying degrees of COVID-19 severity and healthy controls to identify the associations between surface proteins and TFs in host immune cells. Here, we present COVID-19db of Immune Cell States (https://covid19db.streamlit.app/), a web server containing cell surface protein expression, SPaRTAN-inferred TF activities, and their associations with major host immune cell types. The data include four high-quality COVID-19 CITE-seq datasets with a toolset for userfriendly data analysis and visualization. We provide interactive surface protein and TF visualizations across major immune cell types for each dataset, allowing comparison between various patient severity groups for the discovery of potential therapeutic targets and diagnostic biomarkers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=332&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"594980745005b136bb5d45d497a3467c793ec819\",\n",
      "        \"title\": \"The next generation of single-cell sequencing methods can be microfluidics-free\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=333&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6806e1d9e0ecb11140f4b399776a6092d49259e1\",\n",
      "        \"title\": \"mTORC1 upregulates B7-H3/CD276 to inhibit antitumor T cells and drive tumor immune evasion\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=334&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"223611cdb8c0b4d8ffcd92e221af105082e4ffb3\",\n",
      "        \"title\": \"Methods and applications for single-cell and spatial multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=335&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0a3ebeaafcef96372f54b58d7374e0cfa67805c6\",\n",
      "        \"title\": \"A unified framework of realistic in silico data generation and statistical model inference for single-cell and spatial omics\",\n",
      "        \"abstract\": \"In the single-cell and spatial omics field, computational challenges include method benchmarking, data interpretation, and in silico data generation. To address these challenges, we propose an all-in-one statistical simulator, scDesign3, to generate realistic single-cell and spatial omics data, including various cell states, experimental designs, and feature modalities, by learning interpretable parameters from real datasets. Furthermore, using a unified probabilistic model for single-cell and spatial omics data, scDesign3 can infer biologically meaningful parameters, assess the goodness-of-fit of inferred cell clusters, trajectories, and spatial locations, and generate in silico negative and positive controls for benchmarking computational tools.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=336&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e2d11f7feed6ff1e7b8c935380997a78ccd95728\",\n",
      "        \"title\": \"Type I interferon dynamics determines memory B cell epigenetic identity in chronic viral infection\",\n",
      "        \"abstract\": \"Memory B cells are key providers of long-lived immunity against infectious disease, yet in chronic viral infection they do not produce effective protection. How chronic viral infection disrupts memory B cell development, and whether such changes are reversible, remains unknown. Here, we uncover type-I interferon (IFN-I) dynamics as a key determinant in shaping chronic memory B cell development. Through single-cell (sc)ATAC-sequencing and scRNA-sequencing, we identified a unique memory subset enriched for IFN-stimulated genes (ISGs) during chronic lymphocytic choriomeningitis virus infection. Blockade of IFNAR-1 early in infection transformed the chromatin landscape of chronic memory B cells, decreasing accessibility at ISG-inducing transcription factor binding motifs and inducing a phenotypic change in the dominating memory B cell subset. However, timing was critical, with memory B cells resistant to intervention after 4 weeks post-infection. Together, our research identifies a key mechanism to instruct memory B cell identity during viral infection. One Sentence Summary: IFN dynamics in chronic versus acute viral infection determines memory B cell development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=337&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fef948f0900f8de073cd749e618aa28ca0dadf4d\",\n",
      "        \"title\": \"Gene function and cell surface protein association analysis based on single-cell multiomics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=338&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58e4344c93b7e9f3c34568f898f4044adf2fb7f5\",\n",
      "        \"title\": \"Strategies for optimizing CITE-seq for human islets and other tissues\",\n",
      "        \"abstract\": \"Defining the immunological landscape of human tissue is an important area of research, but challenges include the impact of tissue disaggregation on cell phenotypes and the low abundance of immune cells in many tissues. Here, we describe methods to troubleshoot and standardize Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) for studies involving enzymatic digestion of human tissue. We tested epitope susceptibility of 92 antibodies commonly used to differentiate immune lineages and cell states on human peripheral blood mononuclear cells following treatment with an enzymatic digestion cocktail used to isolate islets. We observed CD4, CD8a, CD25, CD27, CD120b, CCR4, CCR6, and PD1 display significant sensitivity to enzymatic treatment, effects that often could not be overcome with alternate antibodies. Comparison of flow cytometry-based CITE-seq antibody titrations and sequencing data supports that for the majority of antibodies, flow cytometry accurately predicts optimal antibody concentrations for CITE-seq. Comparison by CITE-seq of immune cells in enzymatically digested islet tissue and donor-matched spleen not treated with enzymes revealed little digestion-induced epitope cleavage, suggesting increased sensitivity of CITE-seq and/or that the islet structure may protect resident immune cells from enzymes. Within islets, CITE-seq identified immune cells difficult to identify by transcriptional signatures alone, such as distinct tissue-resident T cell subsets, mast cells, and innate lymphoid cells (ILCs). Collectively this study identifies strategies for the rational design and testing of CITE-seq antibodies for single-cell studies of immune cells within islets and other tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=339&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"53fa280d893b917c19212a67bccb8a510ee6604a\",\n",
      "        \"title\": \"Apoptotic cell fragments locally activate tingible body macrophages in the germinal center\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=340&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"52d07778e382f7ee911842252b76ee12a4fcf8c4\",\n",
      "        \"title\": \"Single-cell proteomics enabled by next-generation sequencing or mass spectrometry\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=341&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ef9aa7deece5932eaeee7259b337e5ee745fd6e2\",\n",
      "        \"title\": \"Synthetic enforcement of STING signaling in cancer cells appropriates the immune microenvironment for checkpoint inhibitor therapy\",\n",
      "        \"abstract\": \"Immune checkpoint inhibitors (ICIs) enhance anticancer immunity by releasing repressive signals into tumor microenvironments (TMEs). To be effective, ICIs require preexisting immunologically \\u201chot\\u201d niches for tumor antigen presentation and lymphocyte recruitment. How the mutational landscape of cancer cells shapes these immunological niches remains poorly defined. We found in human and murine colorectal cancer (CRC) models that the superior antitumor immune response of mismatch repair (MMR)\\u2013deficient CRC required tumor cell\\u2013intrinsic activation of cGAS-STING signaling triggered by genomic instability. Subsequently, we synthetically enforced STING signaling in CRC cells with intact MMR signaling using constitutively active STING variants. Even in MMR-proficient CRC, genetically encoded gain-of-function STING was sufficient to induce cancer cell\\u2013intrinsic interferon signaling, local activation of antigen-presenting cells, recruitment of effector lymphocytes, and sensitization of previously \\u201ccold\\u201d TMEs to ICI therapy in vivo. Thus, our results introduce a rational strategy for modulating cancer cell\\u2013intrinsic programs via engineered STING enforcement to sensitize resistant tumors to ICI responsiveness.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=342&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cefae8ff1232fed861a928b738b39a19ace93e9a\",\n",
      "        \"title\": \"New technologies to study helminth development and host-parasite interactions.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=343&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a0c339f6d938e33019e89ce1fa6caaed04c274c8\",\n",
      "        \"title\": \"Omics-based approaches to guide the design of biomaterials\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=344&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"64346d25276876fd8b2a1b7df7929b2d2c570ae6\",\n",
      "        \"title\": \"Human T\\u00a0cell generation is restored in CD3\\u03b4 severe combined immunodeficiency through adenine base editing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=345&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"537c8321b551cf8de0ff7ee27359718cf28acc0e\",\n",
      "        \"title\": \"Single-cell protein-DNA interactomics and multiomics tools for deciphering genome regulation\",\n",
      "        \"abstract\": \": The emergence of single-cell genomic and transcriptomic sequencing accelerates the development of single-cell epigenomic technologies, providing an unprecedented opportunity for decoding cell fate decisions largely encoded in the epigenome. Recent advances in single-cell multimodality epigenomic technologies facilitate directly interrogating the regulatory relationship between multi-layer molecular information in the same cell. In this review, we discuss recent progress in development of single-cell multimodality epigenomic technologies and applications in elucidating cellular diversifications in development and diseases, with a focus on protein-DNA interactomics and regulatory links between epigenome and transcriptome. Further, we provide perspective on the future direction of single-cell multiomics tool development as well as challenges facing ahead\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=346&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d51fce765e285a9d116fd16ff01adf59818941b\",\n",
      "        \"title\": \"A neutrophil response linked to tumor control in immunotherapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=347&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6bc4729ad1fde5030f03eff2b45a21f91d50de20\",\n",
      "        \"title\": \"An update on renal fibrosis: from mechanisms to therapeutic strategies with a focus on extracellular vesicles\",\n",
      "        \"abstract\": \"The increasing prevalence of chronic kidney disease (CKD) is a major global public health concern. Despite the complicated pathogenesis of CKD, renal fibrosis represents the most common pathological condition, comprised of progressive accumulation of extracellular matrix in the diseased kidney. Over the last several decades, tremendous progress in understanding the mechanism of renal fibrosis has been achieved, and corresponding potential therapeutic strategies targeting fibrosis-related signaling pathways are emerging. Importantly, extracellular vesicles (EVs) contribute significantly to renal inflammation and fibrosis by mediating cellular communication. Increasing evidence suggests the potential of EV-based therapy in renal inflammation and fibrosis, which may represent a future direction for CKD therapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=348&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5749cc20178d2f0cd4c1b8a9cceb259c54c54a47\",\n",
      "        \"title\": \"Identification of CD8+ T-Cell\\u2013Immune Cell Communications in Ileal Crohn's Disease\",\n",
      "        \"abstract\": \"INTRODUCTION: Crohn's disease (CD) is a major subtype of inflammatory bowel disease (IBD), a spectrum of chronic intestinal disorders caused by dysregulated immune responses to gut microbiota. Although transcriptional and functional changes in a number of immune cell types have been implicated in the pathogenesis of IBD, the cellular interactions and signals that drive these changes have been less well-studied. METHODS: We performed Cellular Indexing of Transcriptomes and Epitopes by sequencing on peripheral blood, colon, and ileal immune cells derived from healthy subjects and patients with CD. We applied a previously published computational approach, NicheNet, to predict immune cell types interacting with CD8+ T-cell subsets, revealing putative ligand-receptor pairs and key transcriptional changes downstream of these cell-cell communications. RESULTS: As a number of recent studies have revealed a potential role for CD8+ T-cell subsets in the pathogenesis of IBD, we focused our analyses on identifying the interactions of CD8+ T-cell subsets with other immune cells in the intestinal tissue microenvironment. We identified ligands and signaling pathways that have implicated in IBD, such as interleukin-1\\u03b2, supporting the validity of the approach, along with unexpected ligands, such as granzyme B, which may play previously unappreciated roles in IBD. DISCUSSION: Overall, these findings suggest that future efforts focused on elucidating cell-cell communications among immune and nonimmune cell types may further our understanding of IBD pathogenesis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=349&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2809e74a5d8ede703a86c330cac93223382a9142\",\n",
      "        \"title\": \"Tn5 DNA Transposase in Multi-Omics Research\",\n",
      "        \"abstract\": \"Tn5 transposase use in biotechnology has substantially advanced the sequencing applications of genome-wide analysis of cells. This is mainly due to the ability of Tn5 transposase to efficiently transpose DNA essentially randomly into any target DNA without the aid of other factors. This concise review is focused on the advances in Tn5 applications in multi-omics technologies, genome-wide profiling, and Tn5 hybrid molecule creation. The possibilities of other transposase uses are also discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=350&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ed9ea9f6930f407618fe7356527cb9e240c63ccb\",\n",
      "        \"title\": \"BdLT-Seq as a barcode decay-based method to unravel lineage-linked transcriptome plasticity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=351&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff434ebeaf40136aea38cdc5819d4019b98283dd\",\n",
      "        \"title\": \"High-plex protein and whole transcriptome co-mapping at cellular resolution with spatial CITE-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=352&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6fff73266dfc869ce31bc8cf6a6650a3345f6c7e\",\n",
      "        \"title\": \"Ensemble deep learning of embeddings for clustering multimodal single-cell omics data\",\n",
      "        \"abstract\": \"Motivation Recent advances in multimodal single-cell omics technologies enable multiple modalities of molecular attributes, such as gene expression, chromatin accessibility, and protein abundance, to be profiled simultaneously at a global level in individual cells. While the increasing availability of multiple data modalities is expected to provide a more accurate clustering and characterisation of cells, the development of computational methods that are capable of extracting information embedded across data modalities is still in its infancy. Results We propose SnapCCESS for clustering cells by integrating data modalities in multimodal singlecell omics data using an unsupervised ensemble deep learning framework. By creating snapshots of embeddings of multimodality using variational autoencoders, SnapCCESS can be coupled with various clustering algorithms for generating consensus clustering of cells. We applied SnapCCESS with several clustering algorithms to various datasets generated from popular multimodal single-cell omics technologies. Our results demonstrate that SnapCCESS is effective and more efficient than conventional ensemble deep learning-based clustering methods and outperforms other state-of-the-art multimodal embedding generation methods in integrating data modalities for clustering cells. The improved clustering of cells from SnapCCESS will pave the way for more accurate characterisation of cell identity and types, an essential step for various downstream analyses of multimodal single-cell omics data. Availability and implementation SnapCCESS is implemented as a Python package and is freely available from https://github.com/yulijia/SnapCCESS.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=353&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"afda9a2fd54fa7719143fa3e760af81fce30cabe\",\n",
      "        \"title\": \"Recent advances in microfluidics for single-cell functional proteomics.\",\n",
      "        \"abstract\": \"Single-cell proteomics (SCP) reveals phenotypic heterogeneity by profiling individual cells, their biological states and functional outcomes upon signaling activation that can hardly be probed via other omics characterizations. This has become appealing to researchers as it enables an overall more holistic view of biological details underlying cellular processes, disease onset and progression, as well as facilitates unique biomarker identification from individual cells. Microfluidic-based strategies have become methods of choice for single-cell analysis because they allow facile assay integrations, such as cell sorting, manipulation, and content analysis. Notably, they have been serving as an enabling technology to improve the sensitivity, robustness, and reproducibility of recently developed SCP methods. Critical roles of microfluidics technologies are expected to further expand rapidly in advancing the next phase of SCP analysis to reveal more biological and clinical insights. In this review, we will capture the excitement of the recent achievements of microfluidics methods for both targeted and global SCP, including efforts to enhance the proteomic coverage, minimize sample loss, and increase multiplexity and throughput. Furthermore, we will discuss the advantages, challenges, applications, and future prospects of SCP.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=354&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"62f586e322d7b74568f07819c1602f5f78d38a5f\",\n",
      "        \"title\": \"siVAE: interpretable deep generative models for single-cell transcriptomes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=355&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"94e46123d565e78279e6a122ef34ae06325aeb17\",\n",
      "        \"title\": \"Single-cell mapping of combinatorial target antigens for CAR switches using logic gates\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=356&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ecd3b1a06c492d1674b7d667bfb0de7740d09038\",\n",
      "        \"title\": \"Mitochondrial single-cell ATAC-seq for high-throughput multi-omic detection of mitochondrial genotypes and chromatin accessibility\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=357&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"05b4c59a8072f8cb51c04010e97e22e486ba6706\",\n",
      "        \"title\": \"Single-cell transcriptomics is revolutionizing the improvement of plant biotechnology research: recent advances and future opportunities\",\n",
      "        \"abstract\": \"Abstract Single-cell approaches are a promising way to obtain high-resolution transcriptomics data and have the potential to revolutionize the study of plant growth and development. Recent years have seen the advent of unprecedented technological advances in the field of plant biology to study the transcriptional information of individual cells by single-cell RNA sequencing (scRNA-seq). This review focuses on the modern advancements of single-cell transcriptomics in plants over the past few years. In addition, it also offers a new insight of how these emerging methods will expedite advance research in plant biotechnology in the near future. Lastly, the various technological hurdles and inherent limitations of single-cell technology that need to be conquered to develop such outstanding possible knowledge gain is critically analyzed and discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=358&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d305a709d594805c582b9a05e5a702a8bfecdd2\",\n",
      "        \"title\": \"Single-cell clonal tracking of persistent T-cells in allogeneic hematopoietic stem cell transplantation\",\n",
      "        \"abstract\": \"The critical balance between intended and adverse effects in allogeneic hematopoietic stem cell transplantation (alloHSCT) depends on the fate of individual donor T-cells. To this end, we tracked \\u03b1\\u03b2T-cell clonotypes during stem cell mobilization treatment with granulocyte-colony stimulating factor (G-CSF) in healthy donors and for six months during immune reconstitution after transfer to transplant recipients. More than 250 \\u03b1\\u03b2T-cell clonotypes were tracked from donor to recipient. These clonotypes consisted almost exclusively of CD8+ effector memory T cells (CD8TEM), which exhibited a different transcriptional signature with enhanced effector and cytotoxic functions compared to other CD8TEM. Importantly, these distinct and persisting clonotypes could already be delineated in the donor. We confirmed these phenotypes on the protein level and their potential for selection from the graft. Thus, we identified a transcriptional signature associated with persistence and expansion of donor T-cell clonotypes after alloHSCT that may be exploited for personalized graft manipulation strategies in future studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=359&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a74fc1c2f669b0303abfd6c709a46903cb9ef9b6\",\n",
      "        \"title\": \"Paired single-cell multi-omics data integration with Mowgli\",\n",
      "        \"abstract\": \"The profiling of multiple molecular layers from the same set of cells has recently become possible. There is thus a growing need for multi-view learning methods able to jointly analyze these data. We here present Multi-Omics Wasserstein inteGrative anaLysIs (Mowgli), a novel method for the integration of paired multi-omics data with any type and number of omics. Of note, Mowgli combines integrative Nonnegative Matrix Factorization (NMF) and Optimal Transport (OT), enhancing at the same time the clustering performance and interpretability of integrative NMF. We apply Mowgli to multiple paired single-cell multi-omics data profiled with 10X Multiome, CITE-seq and TEA-seq. Our in depth benchmark demonstrates that Mowgli\\u2019s performance is competitive with the state-of-the-art in cell clustering and superior to the state-of-the-art once considering biological interpretability. Mowgli is implemented as a Python package seamlessly integrated within the scverse ecosystem and it is available at http://github.com/cantinilab/mowgli.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=360&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b614ec4d05718fb7e2673ce121c666a0b492e3b3\",\n",
      "        \"title\": \"iPoLNG\\u2014An unsupervised model for the integrative analysis of single-cell multiomics data\",\n",
      "        \"abstract\": \"Single-cell multiomics technologies, where the transcriptomic and epigenomic profiles are simultaneously measured in the same set of single cells, pose significant challenges for effective integrative analysis. Here, we propose an unsupervised generative model, iPoLNG, for the effective and scalable integration of single-cell multiomics data. iPoLNG reconstructs low-dimensional representations of the cells and features using computationally efficient stochastic variational inference by modelling the discrete counts in single-cell multiomics data with latent factors. The low-dimensional representation of cells enables the identification of distinct cell types, and the feature by factor loading matrices help characterize cell-type specific markers and provide rich biological insights on the functional pathway enrichment analysis. iPoLNG is also able to handle the setting of partial information where certain modality of the cells is missing. Taking advantage of GPU and probabilistic programming, iPoLNG is scalable to large datasets and it takes less than 15 min to implement on datasets with 20,000 cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=361&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c89483d70d3e290f71f3852bc6536f21094b53c\",\n",
      "        \"title\": \"Single-cell RNA sequencing to identify cellular heterogeneity and targets in cardiovascular diseases: from bench to bedside\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=362&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d76c65d50a2327d5c491f6f566799db4d86a5d3\",\n",
      "        \"title\": \"Locally sourced: site-specific immune barriers to metastasis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=363&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"967ab5fd8afb4345fecd15d2201560f998555566\",\n",
      "        \"title\": \"SHARE-Topic: Bayesian interpretable modeling of single-cell multi-omic data\",\n",
      "        \"abstract\": \"Single-cell sequencing technologies are providing unprecedented insights into the molecular biology of individual cells. More recently, multi-omic technologies have emerged which can simultaneously measure gene expression and the epigenomic state of the same cell, holding the promise to unlock our understanding of the epigenetic mechanisms of gene regulation. However, the sparsity and noisy nature of the data pose fundamental statistical challenges which hinder our ability to extract biological knowledge from these complex data sets. Here we propose SHARE-Topic, a Bayesian generative model of multi-omic single cell data which addresses these challenges from the point of view of topic models. SHARE-Topic identifies common patterns of co-variation between different \\u2018omic layers, providing interpretable explanations for the complexity of the data. Tested on joint ATAC and expression data, SHARE-Topic was able to provide low dimensional representations that recapitulate known biology, and to define in a principled way associations between genes and distal regulators in individual cells. We illustrate SHARE-Topic in a case study of B-cell lymphoma, studying the usage of alternative promoters in the regulation of the FOXP1 transcription factors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=364&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"66725c644d6b501a55d8dc5fe8c36bc0164c8f83\",\n",
      "        \"title\": \"Leukemic stem cells and therapy resistance in acute myeloid leukemia\",\n",
      "        \"abstract\": \"A major obstacle in the treatment of acute myeloid leukemia (AML) is refractory disease or relapse after achieving remission. The latter arises from a few therapy-resistant cells within minimal residual disease (MRD). Resistant cells with long-term self-renewal capacity that drive clonal outgrowth are referred to as leukemic stem cells (LSC). The cancer stem cell concept considers LSC as relapse-initiating cells residing at the top of each genetically defined AML subclone forming epigenetically controlled downstream hierarchies. LSC display significant phenotypic and epigenetic plasticity, particularly in response to therapy stress, which results in various mechanisms mediating treatment resistance. Given the inherent chemotherapy resistance of LSC, targeted strategies must be incorporated into first-line regimens to prevent LSC-mediated AML relapse. The combination of venetoclax and azacitidine is a promising current strategy for the treatment of AML LSC. Nevertheless, the selection of patients who would benefit either from standard chemotherapy or venetoclax + azacitidine treatment in first-line therapy has yet to be established and the mechanisms of resistance still need to be discovered and overcome. Clinical trials are currently underway that investigate LSC susceptibility to first-line therapies. The era of single-cell multi-omics has begun to uncover the complex clonal and cellular architectures and associated biological networks. This should lead to a better understanding of the highly heterogeneous AML at the inter- and intra-patient level and identify resistance mechanisms by longitudinal analysis of patients\\u2019 samples. This review discusses LSC biology and associated resistance mechanisms, potential therapeutic LSC vulnerabilities and current clinical trial activities.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=365&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e5b999dc81c4d53db458cfd9fd1d3a2fea2c1d22\",\n",
      "        \"title\": \"Biologically informed deep learning to query gene programs in single-cell atlases\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=366&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7af210342196ea5f7a65fc0eec9b87838cb3de3d\",\n",
      "        \"title\": \"Meeting report: 6th Global Forum on Tuberculosis Vaccines, 22\\u201325 February 2022, Toulouse, France\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=367&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a37682f4ef3c2cc9238ed23c30cf74c3c800c183\",\n",
      "        \"title\": \"Into the multi-omics era: Progress of T cells profiling in the context of solid organ transplantation\",\n",
      "        \"abstract\": \"T cells are the common type of lymphocyte to mediate allograft rejection, remaining long-term allograft survival impeditive. However, the heterogeneity of T cells, in terms of differentiation and activation status, the effector function, and highly diverse T cell receptors (TCRs) have thus precluded us from tracking these T cells and thereby comprehending their fate in recipients due to the limitations of traditional detection approaches. Recently, with the widespread development of single-cell techniques, the identification and characterization of T cells have been performed at single-cell resolution, which has contributed to a deeper comprehension of T cell heterogeneity by relevant detections in a single cell \\u2013 such as gene expression, DNA methylation, chromatin accessibility, surface proteins, and TCR. Although these approaches can provide valuable insights into an individual cell independently, a comprehensive understanding can be obtained when applied joint analysis. Multi-omics techniques have been implemented in characterizing T cells in health and disease, including transplantation. This review focuses on the thesis, challenges, and advances in these technologies and highlights their application to the study of alloreactive T cells to improve the understanding of T cell heterogeneity in solid organ transplantation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=368&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b4563791c3ec16487bec22d32f025c6b5a412418\",\n",
      "        \"title\": \"Single-cell multi-omic topic embedding reveals cell-type-specific and COVID-19 severity-related immune signatures\",\n",
      "        \"abstract\": \"The advent of single-cell multi-omics sequencing technology makes it possible for re-searchers to leverage multiple modalities for individual cells and explore cell heterogeneity. However, the high dimensional, discrete, and sparse nature of the data make the downstream analysis particularly challenging. Most of the existing computational methods for single-cell data analysis are either limited to single modality or lack flexibility and interpretability. In this study, we propose an interpretable deep learning method called multi-omic embedded topic model (moETM) to effectively perform integrative analysis of high-dimensional single-cell multimodal data. moETM integrates multiple omics data via a product-of-experts in the encoder for efficient variational inference and then employs multiple linear decoders to learn the multi-omic signatures of the gene regulatory programs. Through comprehensive experiments on public single-cell transcriptome and chromatin accessibility data (i.e., scRNA+scATAC), as well as scRNA and proteomic data (i.e., CITE-seq), moETM demonstrates superior performance compared with six state-of-the-art single-cell data analysis methods on seven publicly available datasets. By applying moETM to the scRNA+scATAC data in human bone marrow mononuclear cells (BMMCs), we identified sequence motifs corresponding to the transcription factors that regulate immune gene signatures. Applying moETM analysis to CITE-seq data from the COVID-19 patients revealed not only known immune cell-type-specific signatures but also composite multi-omic biomarkers of critical conditions due to COVID-19, thus providing insights from both biological and clinical perspectives.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=369&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8312ba096244d742080e56073b5c57aae29281f2\",\n",
      "        \"title\": \"Single-cell sequencing combined with machine learning reveals the mechanism of interaction between epilepsy and stress cardiomyopathy\",\n",
      "        \"abstract\": \"Background Epilepsy is a disorder that can manifest as abnormalities in neurological or physical function. Stress cardiomyopathy is closely associated with neurological stimulation. However, the mechanisms underlying the interrelationship between epilepsy and stress cardiomyopathy are unclear. This paper aims to explore the genetic features and potential molecular mechanisms shared in epilepsy and stress cardiomyopathy. Methods By analyzing the epilepsy dataset and stress cardiomyopathy dataset separately, the intersection of the two disease co-expressed differential genes is obtained, the co-expressed differential genes reveal the biological functions, the network is constructed, and the core modules are identified to reveal the interaction mechanism, the co-expressed genes with diagnostic validity are screened by machine learning algorithms, and the co-expressed genes are validated in parallel on the epilepsy single-cell data and the stress cardiomyopathy rat model. Results Epilepsy causes stress cardiomyopathy, and its key pathways are Complement and coagulation cascades, HIF-1 signaling pathway, its key co-expressed genes include SPOCK2, CTSZ, HLA-DMB, ALDOA, SFRP1, ERBB3. The key immune cell subpopulations localized by single-cell data are the T_cells subgroup, Microglia subgroup, Macrophage subgroup, Astrocyte subgroup, and Oligodendrocytes subgroup. Conclusion We believe epilepsy causing stress cardiomyopathy results from a multi-gene, multi-pathway combination. We identified the core co-expressed genes (SPOCK2, CTSZ, HLA-DMB, ALDOA, SFRP1, ERBB3) and the pathways that function in them (Complement and coagulation cascades, HIF-1 signaling pathway, JAK-STAT signaling pathway), and finally localized their key cellular subgroups (T_cells subgroup, Microglia subgroup, Macrophage subgroup, Astrocyte subgroup, and Oligodendrocytes subgroup). Also, combining cell subpopulations with hypercoagulability as well as sympathetic excitation further narrowed the cell subpopulations of related functions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=370&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd8ac615b08563ea5b1e9d694f5409f45ceeaeb1\",\n",
      "        \"title\": \"Pooled CRISPR screening of high-content cellular phenotypes by ghost cytometry\",\n",
      "        \"abstract\": \"Fast enrichment of cells based on morphological information remains a challenge, limiting genome-wide perturbation screening for diverse high-content phenotypes of cells. Here we show that multi-modal ghost cytometry-based cell sorting is applicable to fast pooled CRISPR screening for both fluorescence and label-free high-content phenotypes of millions of cells. By employing the high-content cell sorter in the fluorescence mode, we enabled the genome-wide CRISPR screening of genes that affect NF-\\u03baB nuclear translocation. Furthermore, by employing the multi-parametric, label-free mode, we performed the large-scale screening to identify a gene involved in macrophage polarization. Especially the label-free platform can enrich target phenotypes without invasive staining, preserving untouched cells for downstream assays and unlocking the potential to screen for the cellular phenotypes even when suitable markers are lacking. One-Sentence Summary Machine vision-based cell sorter enabled genome-wide perturbation screens for high-content cell phenotypes even without labeling\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=371&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd39b1f3bee4654d8c4336490698c6113d20e05b\",\n",
      "        \"title\": \"Targeting Immune-Fibroblast Crosstalk in Myocardial Infarction and Cardiac Fibrosis\",\n",
      "        \"abstract\": \"Inflammation and tissue fibrosis co-exist and are causally linked to organ dysfunction. However, the molecular mechanisms driving immune-fibroblast crosstalk in human cardiac disease remains unexplored and there are currently no therapeutics to target fibrosis. Here, we performed multi-omic single-cell gene expression, epitope mapping, and chromatin accessibility profiling in 38 donors, acutely infarcted, and chronically failing human hearts. We identified a disease-associated fibroblast trajectory marked by cell surface expression of fibroblast activator protein (FAP), which diverged into distinct myofibroblasts and pro-fibrotic fibroblast populations, the latter resembling matrifibrocytes. Pro-fibrotic fibroblasts were transcriptionally similar to cancer associated fibroblasts and expressed high levels of collagens and periostin (POSTN), thymocyte differentiation antigen 1 (THY-1), and endothelin receptor A (EDNRA) predicted to be driven by a RUNX1 gene regulatory network. We assessed the applicability of experimental systems to model tissue fibrosis and demonstrated that 3 different in vivo mouse models of cardiac injury were superior compared to cultured human heart and dermal fibroblasts in recapitulating the human disease phenotype. Ligand-receptor analysis and spatial transcriptomics predicted that interactions between C-C chemokine receptor type 2 (CCR2) macrophages and fibroblasts mediated by interleukin 1 beta (IL-1\\u03b2) signaling drove the emergence of pro-fibrotic fibroblasts within spatially defined niches. This concept was validated through in silico transcription factor perturbation and in vivo inhibition of IL-1\\u03b2 signaling in fibroblasts where we observed reduced pro-fibrotic fibroblasts, preferential differentiation of fibroblasts towards myofibroblasts, and reduced cardiac fibrosis. Herein, we show a subset of macrophages signal to fibroblasts via IL-1\\u03b2 and rewire their gene regulatory network and differentiation trajectory towards a pro-fibrotic fibroblast phenotype. These findings highlight the broader therapeutic potential of targeting inflammation to treat tissue fibrosis and restore organ function.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=372&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a288692395b530384a12dcb7af6a194a58a0c4a4\",\n",
      "        \"title\": \"Decision level integration of unimodal and multimodal single cell data with scTriangulate\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=373&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"77ecbd42cd655ff3bf5f1fb96c8c2a6eb44906c0\",\n",
      "        \"title\": \"Application of Single-Cell and Spatial Omics in Musculoskeletal Disorder Research\",\n",
      "        \"abstract\": \"Musculoskeletal disorders, including fractures, scoliosis, heterotopic ossification, osteoporosis, osteoarthritis, disc degeneration, and muscular injury, etc., can occur at any stage of human life. Understanding the occurrence and development mechanism of musculoskeletal disorders, as well as the changes in tissues and cells during therapy, might help us find targeted treatment methods. Single-cell techniques provide excellent tools for studying alterations at the cellular level of disorders. However, the application of these techniques in research on musculoskeletal disorders is still limited. This review summarizes the current single-cell and spatial omics used in musculoskeletal disorders. Cell isolation, experimental methods, and feasible experimental designs for single-cell studies of musculoskeletal system diseases have been reviewed based on tissue characteristics. Then, the paper summarizes the latest findings of single-cell studies in musculoskeletal disorders from three aspects: bone and ossification, joint, and muscle and tendon disorders. Recent discoveries about the cell populations involved in these diseases are highlighted. Furthermore, the therapeutic responses of musculoskeletal disorders, especially single-cell changes after the treatments of implants, stem cell therapies, and drugs are described. Finally, the application potential and future development directions of single-cell and spatial omics in research on musculoskeletal diseases are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=374&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"46590a953783aeaf53fa817f1608115ea3468fba\",\n",
      "        \"title\": \"Multimodal human thymic profiling reveals trajectories and cellular milieu for T agonist selection\",\n",
      "        \"abstract\": \"To prevent autoimmunity, thymocytes expressing self-reactive T cell receptors (TCRs) are negatively selected, however, divergence into tolerogenic, agonist selected lineages represent an alternative fate. As thymocyte development, selection, and lineage choices are dependent on spatial context and cell-to-cell interactions, we have performed Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) and spatial transcriptomics on paediatric human thymus. Thymocytes expressing markers of strong TCR signalling diverged from the conventional developmental trajectory prior to CD4+ or CD8+ lineage commitment, while markers of different agonist selected T cell populations (CD8\\u03b1\\u03b1(I), CD8\\u03b1\\u03b1(II), T(agonist), Treg(diff), and Treg) exhibited variable timing of induction. Expression profiles of chemokines and co-stimulatory molecules, together with spatial localisation, supported that dendritic cells, B cells, and stromal cells contribute to agonist selection, with different subsets influencing thymocytes at specific developmental stages within distinct spatial niches. Understanding factors influencing agonist T cells is needed to benefit from their immunoregulatory effects in clinical use.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=375&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"946f125589b42a8a3b042d9773446bc337d5a596\",\n",
      "        \"title\": \"sortChIC enables detailed chromatin analysis of rare cell types\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=376&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f6c66338b0f923dcd7f910b9adb7a599c5a6b20c\",\n",
      "        \"title\": \"\\u03b3\\u03b4 T cells are effectors of immunotherapy in cancers with HLA class I defects\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=377&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd1880589b54bc3674d1f84e0f781110ff022413\",\n",
      "        \"title\": \"Advancements in Oncoproteomics Technologies: Treading toward Translation into Clinical Practice\",\n",
      "        \"abstract\": \"Proteomics continues to forge significant strides in the discovery of essential biological processes, uncovering valuable information on the identity, global protein abundance, protein modifications, proteoform levels, and signal transduction pathways. Cancer is a complicated and heterogeneous disease, and the onset and progression involve multiple dysregulated proteoforms and their downstream signaling pathways. These are modulated by various factors such as molecular, genetic, tissue, cellular, ethnic/racial, socioeconomic status, environmental, and demographic differences that vary with time. The knowledge of cancer has improved the treatment and clinical management; however, the survival rates have not increased significantly, and cancer remains a major cause of mortality. Oncoproteomics studies help to develop and validate proteomics technologies for routine application in clinical laboratories for (1) diagnostic and prognostic categorization of cancer, (2) real-time monitoring of treatment, (3) assessing drug efficacy and toxicity, (4) therapeutic modulations based on the changes with prognosis and drug resistance, and (5) personalized medication. Investigation of tumor-specific proteomic profiles in conjunction with healthy controls provides crucial information in mechanistic studies on tumorigenesis, metastasis, and drug resistance. This review provides an overview of proteomics technologies that assist the discovery of novel drug targets, biomarkers for early detection, surveillance, prognosis, drug monitoring, and tailoring therapy to the cancer patient. The information gained from such technologies has drastically improved cancer research. We further provide exemplars from recent oncoproteomics applications in the discovery of biomarkers in various cancers, drug discovery, and clinical treatment. Overall, the future of oncoproteomics holds enormous potential for translating technologies from the bench to the bedside.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=378&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f54b3f28b3e808b844a335588cde594e5e79f598\",\n",
      "        \"title\": \"Robust single-cell matching and multimodal analysis using shared and distinct features\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=379&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"746ce85807d65f54f058617e1fa69c659a503e94\",\n",
      "        \"title\": \"Influenza vaccination reveals sex dimorphic imprints of prior mild COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=380&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d65dc556e76596ee68b344b3a4ffb50ebe5ead1\",\n",
      "        \"title\": \"Single-cell omics: A new direction for functional genetic research in human diseases and animal models\",\n",
      "        \"abstract\": \"Over the past decade, with the development of high-throughput single-cell sequencing technology, single-cell omics has been emerged as a powerful tool to understand the molecular basis of cellular mechanisms and refine our knowledge of diverse cell states. They can reveal the heterogeneity at different genetic layers and elucidate their associations by multiple omics analysis, providing a more comprehensive genetic map of biological regulatory networks. In the post-GWAS era, the molecular biological mechanisms influencing human diseases will be further elucidated by single-cell omics. This review mainly summarizes the development and trend of single-cell omics. This involves single-cell omics technologies, single-cell multi-omics technologies, multiple omics data integration methods, applications in various human organs and diseases, classic laboratory cell lines, and animal disease models. The review will reveal some perspectives for elucidating human diseases and constructing animal models.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=381&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a0108b7dd6c9d1ac04a4d0854c56cd89d8ac4862\",\n",
      "        \"title\": \"Single-cell computational machine learning approaches to immune-mediated inflammatory disease: New tools uncover novel fibroblast and macrophage interactions driving pathogenesis\",\n",
      "        \"abstract\": \"Recent advances in single-cell sequencing technologies call for greater computational scalability and sensitivity to analytically decompose diseased tissues and expose meaningful biological relevance in individual cells with high resolution. And while fibroblasts, one of the most abundant cell types in tissues, were long thought to display relative homogeneity, recent analytical and technical advances in single-cell sequencing have exposed wide variation and sub-phenotypes of fibroblasts of potential and apparent clinical significance to inflammatory diseases. Alongside anticipated improvements in single cell spatial sequencing resolution, new computational biology techniques have formed the technical backbone when exploring fibroblast heterogeneity. More robust models are required, however. This review will summarize the key advancements in computational techniques that are being deployed to categorize fibroblast heterogeneity and their interaction with the myeloid compartments in specific biological and clinical contexts. First, typical machine-learning-aided methods such as dimensionality reduction, clustering, and trajectory inference, have exposed the role of fibroblast subpopulations in inflammatory disease pathologies. Second, these techniques, coupled with single-cell predicted computational methods have raised novel interactomes between fibroblasts and macrophages of potential clinical significance to many immune-mediated inflammatory diseases such as rheumatoid arthritis, ulcerative colitis, lupus, systemic sclerosis, and others. Third, recently developed scalable integrative methods have the potential to map cross-cell-type spatial interactions at the single-cell level while cross-tissue analysis with these models reveals shared biological mechanisms between disease contexts. Finally, these advanced computational omics approaches have the potential to be leveraged toward therapeutic strategies that target fibroblast-macrophage interactions in a wide variety of inflammatory diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=382&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b9a841aa454ccac4cb69ec853246babd0bf6b8f5\",\n",
      "        \"title\": \"Integration of whole transcriptome spatial profiling with protein markers\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=383&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c6f74901359942f7c7bbcbee295db601b239a5b4\",\n",
      "        \"title\": \"Integrated Single-Cell (Phospho-)Protein and RNA Detection Uncovers Phenotypic Characteristics and Active Signal Transduction of Human Antibody-Secreting Cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=384&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"12332f4a8c5f59900d018287ace003f87dd19e77\",\n",
      "        \"title\": \"Specialized functions and sexual dimorphism explain the functional diversity of the myeloid populations during glioma progression.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=385&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1a6f3d7ea1fba53ab7674ff9149087ab52d788c6\",\n",
      "        \"title\": \"Single-cell technologies uncover intra-tumor heterogeneity in childhood cancers\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=386&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e8b85b14b8fe9d344a11949d2f9c080c6b5c9e21\",\n",
      "        \"title\": \"scONE-seq: A single-cell multi-omics method enables simultaneous dissection of phenotype and genotype heterogeneity from frozen tumors\",\n",
      "        \"abstract\": \"Single-cell multi-omics can provide a unique perspective on tumor cellular heterogeneity. Most previous single-cell whole-genome RNA sequencing (scWGS-RNA-seq) methods demonstrate utility with intact cells from fresh samples. Among them, many are not applicable to frozen samples that cannot produce intact single-cell suspensions. We have developed scONE-seq, a versatile scWGS-RNA-seq method that amplifies single-cell DNA and RNA without separating them from each other and hence is compatible with frozen biobanked samples. We benchmarked scONE-seq against existing methods using fresh and frozen samples to demonstrate its performance in various aspects. We identified a unique transcriptionally normal-like tumor clone by analyzing a 2-year frozen astrocytoma sample, demonstrating that performing single-cell multi-omics interrogation on biobanked tissue by scONE-seq could enable previously unidentified discoveries in tumor biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=387&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"affb3c4306455360e25fb199b9b1fa15d2b91c21\",\n",
      "        \"title\": \"Immune Profiling Reveals Decreases in Circulating Regulatory and Exhausted T\\u00a0Cells in Human Hypertension.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=388&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d399d59020f0e582e17cda7b722ba78ec10c1c67\",\n",
      "        \"title\": \"Recent advances in droplet microfluidics for single-cell analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=389&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"51303d458f2b3b6ea41c40ec5fd13c013d03a939\",\n",
      "        \"title\": \"A universal MHCII technology platform to characterize antigen-specific CD4+ T cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=390&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8cc8866036d20af6402715801b7eb09289645def\",\n",
      "        \"title\": \"Current and future perspectives of single-cell multi-omics technologies in cardiovascular research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=391&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"01d142bfda9fc5ee662cc21b198796ccf2a2bb3a\",\n",
      "        \"title\": \"Overview of single\\u2010cell RNA sequencing analysis and its application to spermatogenesis research\",\n",
      "        \"abstract\": \"Single\\u2010cell transcriptomics allows parallel analysis of multiple cell types in tissues. Because testes comprise somatic cells and germ cells at various stages of spermatogenesis, single\\u2010cell RNA sequencing is a powerful tool for investigating the complex process of spermatogenesis. However, single\\u2010cell RNA sequencing analysis needs extensive knowledge of experimental technologies and bioinformatics, making it difficult for many, particularly experimental biologists and clinicians, to use it.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=392&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4933d412f46dcc8356b6cae6d52653507f06b021\",\n",
      "        \"title\": \"The human placenta shapes the phenotype of decidual macrophages.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=393&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1eb32fe5f3be2334740e5322e75355e66ef9d576\",\n",
      "        \"title\": \"An inflammatory state remodels the immune microenvironment and improves risk stratification in acute myeloid leukemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=394&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a97374d044a6b81854956923fac50fbb7b4a7b96\",\n",
      "        \"title\": \"Lineage Reconstruction of In Vitro Identified Antigen-Specific Autoreactive B Cells from Adaptive Immune Receptor Repertoires\",\n",
      "        \"abstract\": \"The emergence, survival, growth and maintenance of autoreactive (AR) B-cell clones, the hallmark of humoral autoimmunity, leave their footprints in B-cell receptor repertoires. Collecting IgH sequences related to polyreactive (PR) ones from adaptive immune receptor repertoire (AIRR) datasets make the reconstruction and analysis of PR/AR B-cell lineages possible. We developed a computational approach, named ImmChainTracer, to extract members and to visualize clonal relationships of such B-cell lineages. Our approach was successfully applied on the IgH repertoires of patients suffering from monogenic hypomorphic RAG1 and 2 deficiency (pRD) or polygenic systemic lupus erythematosus (SLE) autoimmune diseases to identify relatives of AR IgH sequences and to track their fate in AIRRs. Signs of clonal expansion, affinity maturation and class-switching events in PR/AR and non-PR/AR B-cell lineages were revealed. An extension of our method towards B-cell expansion caused by any trigger (e.g., infection, vaccination or antibody development) may provide deeper insight into antigen specific B-lymphogenesis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=395&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5dd2f810be972a0fcbab627968f0fc990cb89623\",\n",
      "        \"title\": \"Class switch towards non-inflammatory, spike-specific IgG4 antibodies after repeated SARS-CoV-2 mRNA vaccination\",\n",
      "        \"abstract\": \"RNA vaccines are efficient preventive measures to combat the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic. High levels of neutralizing SARS-CoV-2 antibodies are an important component of vaccine-induced immunity. Shortly after the initial two mRNA vaccine doses, the immunoglobulin G (IgG) response mainly consists of the proinflammatory subclasses IgG1 and IgG3. Here, we report that several months after the second vaccination, SARS-CoV-2\\u2013specific antibodies were increasingly composed of noninflammatory IgG4, which were further boosted by a third mRNA vaccination and/or SARS-CoV-2 variant breakthrough infections. IgG4 antibodies among all spike-specific IgG antibodies rose, on average, from 0.04% shortly after the second vaccination to 19.27% late after the third vaccination. This induction of IgG4 antibodies was not observed after homologous or heterologous SARS-CoV-2 vaccination with adenoviral vectors. Single-cell sequencing and flow cytometry revealed substantial frequencies of IgG4-switched B cells within the spike-binding memory B cell population [median of 14.4%; interquartile range (IQR) of 6.7 to 18.1%] compared with the overall memory B cell repertoire (median of 1.3%; IQR of 0.9 to 2.2%) after three immunizations. This class switch was associated with a reduced capacity of the spike-specific antibodies to mediate antibody-dependent cellular phagocytosis and complement deposition. Because Fc-mediated effector functions are critical for antiviral immunity, these findings may have consequences for the choice and timing of vaccination regimens using mRNA vaccines, including future booster immunizations against SARS-CoV-2. Description Repeated SARS-CoV-2 mRNA vaccinations shift the antibody response toward IgG4 subclass with decreased Fc\\u03b3R-dependent effector activity. Anti-spike IgG4 rises from obscurity The four human IgG subclasses have distinct effector properties due to differences in binding Fc receptors and activating complement. The serum concentration of human IgG4 is normally lower than either IgG1, IgG2, or IgG3. Irrgang et al. did a longitudinal analysis of the level of spike-specific antibodies from each IgG subclass in recipients of the SARS-CoV-2 BNT162b2 mRNA vaccine. Anti-spike IgG4 as a fraction of total anti-spike IgG rose by 6 months after the second vaccination and increased further after a third vaccine dose. Serum antibody effector activity assessed by antibody-dependent phagocytosis or complement deposition was less after the third dose than after the second dose. Further studies are needed to determine how emergence of an IgG4 anti-spike response influences vaccine-induced protection from SARS-CoV-2 infection. \\u2014IRW\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=396&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"850cdb8b81523044a927321e23b028afabab424d\",\n",
      "        \"title\": \"Human Macrophage Long Intergenic Noncoding RNA, SIMALR, Suppresses Inflammatory Macrophage Apoptosis via NTN1 (Netrin-1)\",\n",
      "        \"abstract\": \"Background: Long noncoding RNAs (lncRNAs) have emerged as novel regulators of macrophage biology and inflammatory cardiovascular diseases. However, studies focused on lncRNAs in human macrophage subtypes, particularly human lncRNAs that are not conserved in rodents, are limited. Methods: Through RNA-sequencing of human monocyte\\u2013derived macrophages, we identified suppressor of inflammatory macrophage apoptosis lncRNA (SIMALR). Lipopolysaccharide/IFN\\u03b3 (interferon \\u03b3) stimulated human macrophages were treated with SIMALR antisense oligonucleotides and subjected to RNA-sequencing to investigate the function of SIMALR. Western blots, luciferase assay, and RNA immunoprecipitation were performed to validate function and potential mechanism of SIMALR. RNAscope was performed to identify SIMALR expression in human carotid atherosclerotic plaques. Results: RNA-sequencing of human monocyte\\u2013derived macrophages identified SIMALR, a human macrophage-specific long intergenic noncoding RNA that is highly induced in lipopolysaccharide/IFN\\u03b3\\u2013stimulated macrophages. SIMALR knockdown in lipopolysaccharide/IFN\\u03b3 stimulated THP1 human macrophages induced apoptosis of inflammatory macrophages, as shown by increased protein expression of cleaved PARP (poly[ADP-ribose] polymerase), caspase 9, caspase 3, and Annexin V+. RNA-sequencing of control versus SIMALR knockdown in lipopolysaccharide/IFN\\u03b3\\u2013stimulated macrophages showed Netrin-1 (NTN1) to be significantly decreased upon SIMALR knockdown. We confirmed that NTN1 knockdown in lipopolysaccharide/IFN\\u03b3\\u2013stimulated macrophages induced apoptosis. The SIMALR knockdown-induced apoptotic phenotype was rescued by adding recombinant NTN1. NTN1 promoter-luciferase reporter activity was increased in HEK293T (human embryonic kidney 293) cells treated with lentiviral overexpression of SIMALR. NTN1 promoter activity is known to require HIF1\\u03b1 (hypoxia-inducible factor 1 subunit alpha), and our studies suggest that SIMALR may interact with HIF1\\u03b1 to regulate NTN1 transcription, thereby regulating macrophages apoptosis. SIMALR was found to be expressed in macrophages in human carotid atherosclerotic plaques of symptomatic patients. Conclusions: SIMALR is a nonconserved, human macrophage lncRNA expressed in atherosclerosis that suppresses macrophage apoptosis. SIMALR partners with HIF1\\u03b1 (hypoxia-inducible factor 1 subunit alpha) to regulate NTN1, which is a known macrophage survival factor. This work illustrates the importance of interrogating the functions of human lncRNAs and exploring their translational and therapeutic potential in human atherosclerosis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=397&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f2a389e2cc76028e17d4f7105f5ac6918b277a5\",\n",
      "        \"title\": \"Watson-Crick Base-Pairing Requirements for ssDNA Recognition and Processing in Replication-Initiating HUH Endonucleases\",\n",
      "        \"abstract\": \"Circular Rep-encoding single-stranded DNA (CRESS-DNA) viruses are a ubiquitous group of viruses that infect organisms across all domains of life. These viruses negatively impact both agriculture and human health. ABSTRACT Replication-initiating HUH endonucleases (Reps) are sequence-specific nucleases that cleave and rejoin single-stranded DNA (ssDNA) during rolling-circle replication. These functions are mediated by covalent linkage of the Rep to its substrate post cleavage. Here, we describe the structures of the endonuclease domain from the Muscovy duck circovirus Rep in complex with its cognate ssDNA 10-mer with and without manganese in the active site. Structural and functional analyses demonstrate that divalent cations play both catalytic and structural roles in Reps by polarizing and positioning their substrate. Further structural comparisons highlight the importance of an intramolecular substrate Watson-Crick (WC) base pairing between the \\u22124 and +1 positions. Subsequent kinetic and functional analyses demonstrate a functional dependency on WC base pairing between these positions regardless of the pair\\u2019s identity (i.e., A\\u00b7T, T\\u00b7A, G\\u00b7C, or C\\u00b7G), highlighting a structural specificity for substrate interaction. Finally, considering how well WC swaps were tolerated in vitro, we sought to determine to what extent the canonical \\u22124T\\u00b7+1A pairing is conserved in circular Rep-encoding single-stranded DNA viruses and found evidence of noncanonical pairings in a minority of these genomes. Altogether, our data suggest that substrate intramolecular WC base pairing is a universal requirement for separation and reunion of ssDNA in Reps. IMPORTANCE Circular Rep-encoding single-stranded DNA (CRESS-DNA) viruses are a ubiquitous group of viruses that infect organisms across all domains of life. These viruses negatively impact both agriculture and human health. All members of this viral family employ a multifunctional nuclease (Rep) to initiate replication. Reps are structurally similar throughout this family, making them targets of interest for viral inhibition strategies. Here, we investigate the functional dependencies of the Rep protein from Muscovy duck circovirus for ssDNA interaction. We demonstrate that this Rep requires an intramolecular Watson-Crick base pairing for origin of replication (Ori) recognition and interaction. We show that noncognate base pair swaps are well tolerated, highlighting a local structural specificity over sequence specificity. Bioinformatic analysis found that the vast majority of CRESS-DNA Oris form base pairs in conserved positions, suggesting this pairing is a universal requirement for replication initiation in the CRESS-DNA virus family.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=398&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75513765d2557b257f3a9bc5c1bc46d59b36afbf\",\n",
      "        \"title\": \"Single cell multi-omic reference atlases of non-human primate immune tissues reveals CD102 as a biomarker for long-lived plasma cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=399&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"00db920193f84b941410e44acc47b4a6b6d5c2b6\",\n",
      "        \"title\": \"Learning cell identity in immunology, neuroscience, and cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=400&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e1dfd10a2bc23c6bf31ba13a2c0eec30d0542ff5\",\n",
      "        \"title\": \"Multiplexed mapping of chromatin features at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=401&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6897dd90a2748a280bbf7457c43d475b0cfc5379\",\n",
      "        \"title\": \"Dissecting CD8+ T cell pathology of severe SARS-CoV-2 infection by single-cell immunoprofiling\",\n",
      "        \"abstract\": \"Introduction SARS-CoV-2 infection results in varying disease severity, ranging from asymptomatic infection to severe illness. A detailed understanding of the immune response to SARS-CoV-2 is critical to unravel the causative factors underlying differences in disease severity and to develop optimal vaccines against new SARS-CoV-2 variants. Methods We combined single-cell RNA and T cell receptor sequencing with CITE-seq antibodies to characterize the CD8+ T cell response to SARS-CoV-2 infection at high resolution and compared responses between mild and severe COVID-19. Results We observed increased CD8+ T cell exhaustion in severe SARS-CoV-2 infection and identified a population of NK-like, terminally differentiated CD8+ effector T cells characterized by expression of FCGR3A (encoding CD16). Further characterization of NK-like CD8+ T cells revealed heterogeneity among CD16+ NK-like CD8+ T cells and profound differences in cytotoxicity, exhaustion, and NK-like differentiation between mild and severe disease conditions. Discussion We propose a model in which differences in the surrounding inflammatory milieu lead to crucial differences in NK-like differentiation of CD8+ effector T cells, ultimately resulting in the appearance of NK-like CD8+ T cell populations of different functionality and pathogenicity. Our in-depth characterization of the CD8+ T cell-mediated response to SARS-CoV-2 infection provides a basis for further investigation of the importance of NK-like CD8+ T cells in COVID-19 severity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=402&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f5458a1c85439714e7b79ddad2f2bc82f99de3f9\",\n",
      "        \"title\": \"Asxl1 deletion disrupts MYC and RNA polymerase II function in granulocyte progenitors\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=403&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aeb6a42778c1034bd670fd55e9c0d22d9ea08f07\",\n",
      "        \"title\": \"Progress in kidney transplantation: The role for systems immunology\",\n",
      "        \"abstract\": \"The development of systems biology represents an immense breakthrough in our ability to perform translational research and deliver personalized and precision medicine. A multidisciplinary approach in combination with use of novel techniques allows for the extraction and analysis of vast quantities of data even from the volume and source limited samples that can be obtained from human subjects. Continued advances in microfluidics, scalability and affordability of sequencing technologies, and development of data analysis tools have made the application of a multi-omics, or systems, approach more accessible for use outside of specialized centers. The study of alloimmune and protective immune responses after solid organ transplant offers innumerable opportunities for a multi-omics approach, however, transplant immunology labs are only just beginning to adopt the systems methodology. In this review, we focus on advances in biological techniques and how they are improving our understanding of the immune system and its interactions, highlighting potential applications in transplant immunology. First, we describe the techniques that are available, with emphasis on major advances that allow for increased scalability. Then, we review initial applications in the field of transplantation with a focus on topics that are nearing clinical integration. Finally, we examine major barriers to adapting these methods and discuss potential future developments.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=404&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"77e3d9b7c17eb53ca2e7e8ebcfa9cb067f80bce4\",\n",
      "        \"title\": \"Tumor-promoting function and regulatory landscape of PD-L2 in B-cell lymphoma\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=405&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17ffe6293225a913b99d8d38ee6cecfa7379edff\",\n",
      "        \"title\": \"Generation of single-cell and single-nuclei suspensions from embryonic and adult mouse brains\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=406&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1984197e248c2e525be2d7b8f32c813bc0b315f3\",\n",
      "        \"title\": \"Spatial omics technologies at multimodal and single cell/subcellular level\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=407&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eab655875ad5fd7854d72fab65dbfabd07eb3a48\",\n",
      "        \"title\": \"Clustering of single-cell multi-omics data with a multimodal deep learning method\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=408&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"74240e96df05e09fdedeb2478008ae312d9b9ee0\",\n",
      "        \"title\": \"Massively Parallel CRISPR\\u2010Based Genetic Perturbation Screening at Single\\u2010Cell Resolution\",\n",
      "        \"abstract\": \"The clustered regularly interspaced short palindromic repeats (CRISPR)\\u2010based genetic screening has been demonstrated as a powerful approach for unbiased functional genomics research. Single\\u2010cell CRISPR screening (scCRISPR) techniques, which result from the combination of single\\u2010cell toolkits and CRISPR screening, allow dissecting regulatory networks in complex biological systems at unprecedented resolution. These methods allow cells to be perturbed en masse using a pooled CRISPR library, followed by high\\u2010content phenotyping. This is technically accomplished by annotating cells with sgRNA\\u2010specific barcodes or directly detectable sgRNAs. According to the integration of distinct single\\u2010cell technologies, these methods principally fall into four categories: scCRISPR with RNA\\u2010seq, scCRISPR with ATAC\\u2010seq, scCRISPR with proteome probing, and imaging\\u2010based scCRISPR. scCRISPR has deciphered genotype\\u2013phenotype relationships, genetic regulations, tumor biological issues, and neuropathological mechanisms. This review provides insight into the technical breakthrough of scCRISPR by systematically summarizing the advancements of various scCRISPR methodologies and analyzing their merits and limitations. In addition, an application\\u2010oriented approach guide is offered to meet researchers\\u2019 individualized demands.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=409&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9206af10bd9870bd553bbcee861d0bcc4135fa95\",\n",
      "        \"title\": \"STREAK: A supervised cell surface receptor abundance estimation strategy for single cell RNA-sequencing data using feature selection and thresholded gene set scoring\",\n",
      "        \"abstract\": \"The accurate estimation of cell surface receptor abundance for single cell transcriptomics data is important for the tasks of cell type and phenotype categorization and cell-cell interaction quantification. We previously developed an unsupervised receptor abundance estimation technique named SPECK (Surface Protein abundance Estimation using CKmeans-based clustered thresholding) to address the challenges associated with accurate abundance estimation. In that paper, we concluded that SPECK results in improved concordance with Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) data relative to comparative unsupervised abundance estimation techniques using only single-cell RNA-sequencing (scRNA-seq) data. In this paper, we outline a new supervised receptor abundance estimation method called STREAK (gene Set Testing-based Receptor abundance Estimation using Adjusted distances and cKmeans thresholding) that leverages associations learned from joint scRNA-seq/CITE-seq training data and a thresholded gene set scoring mechanism to estimate receptor abundance for scRNA-seq target data. We evaluate STREAK relative to both unsupervised and supervised receptor abundance estimation techniques using two evaluation approaches on three joint scRNA-seq/CITE-seq datasets that represent two human tissue types. We conclude that STREAK outperforms other abundance estimation strategies and provides a more biologically interpretable and transparent statistical model.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=410&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d8f3759b9ef99adcc9a6eda2acc518d287386ff8\",\n",
      "        \"title\": \"The regulatory network analysis of single-cell RNA-seq and single-cell ATAC-seq data using differential equation models with perturbations\",\n",
      "        \"abstract\": \"A recent study presented a new differential equation model, MultiVelo, which simultaneously performs on chromatin accessibility and gene expression from single-cell data to reveals regulatory network of epigenomic and transcriptomic changes. However, it is unclear how regulatory network changes with perturbations. Here, we extend the ordinal differential equation models to time-delay differential equation models with controller to investigate the stability of regulatory network with perturbations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=411&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"14073de5a6868c851c8d29c8a031d5e9074f21f2\",\n",
      "        \"title\": \"An Autoimmune Transcriptional Circuit Driving Foxp3+ Regulatory T cell Dysfunction\",\n",
      "        \"abstract\": \"Autoimmune diseases, among the most common disorders of young adults, are mediated by genetic and environmental factors. While CD4+Foxp3+ regulatory T cells (Tregs) play a central role in preventing autoimmunity, the molecular mechanism underlying their dysfunction is unknown. Here, we performed comprehensive transcriptomic and epigenomic profiling of Tregs in the autoimmune disease multiple sclerosis (MS) to identify central transcriptional programs regulating human autoimmunity. We discovered that upregulation of a primate-specific short PRDM1 isoform (PRDM1-S) induces SGK1 independent from evolutionally conserved long PRDM1, leading to destabilization of Foxp3 and Treg dysfunction. This aberrant PRDM1-S/SGK1 axis is shared among other autoimmune diseases. Furthermore, by chromatin landscape profiling in MS Tregs we identified a PRDM1-S specific cis-regulatory element associated with enriched binding of AP- 1/IRF transcription factors. Our study identifies evolutionally emerged PRDM1-S and epigenetic priming of AP-1/IRF as key drivers of pathogenic Treg programs leading to human autoimmune disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=412&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f383e689366d0f7e337b14c391f037ba50b7de3f\",\n",
      "        \"title\": \"Titration of 124 antibodies using CITE-Seq on human PBMCs\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=413&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"26f9a473550c3ab9c13d0eb34bdd65aeb26f0756\",\n",
      "        \"title\": \"Proximity sequencing for single-cell measurement of protein complexes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=414&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a55ba1b7ab80d15a4e1f8ca3561733614fc2d80f\",\n",
      "        \"title\": \"Novel Molecular Insights into Leukemic Evolution of Myeloproliferative Neoplasms: A Single Cell Perspective\",\n",
      "        \"abstract\": \"Myeloproliferative neoplasms (MPNs) are clonal disorders originated by the serial acquisition of somatic mutations in hematopoietic stem/progenitor cells. The major clinical entities are represented by polycythemia vera (PV), essential thrombocythemia (ET), and primary myelofibrosis (PMF), that are caused by driver mutations affecting JAK2, MPL or CALR. Disease progression is related to molecular and clonal evolution. PV and ET can progress to secondary myelofibrosis (sMF) but can also evolve to secondary acute myeloid leukemia (sAML). PMF is associated with the highest frequency of leukemic transformation, which represents the main cause of death. sAML is associated with a dismal prognosis and clinical features that differ from those of de novo AML. The molecular landscape distinguishes sAML from de novo AML, since the most frequent hits involve TP53, epigenetic regulators, spliceosome modulators or signal transduction genes. Single cell genomic studies provide novel and accurate information about clonal architecture and mutation acquisition order, allowing the reconstruction of clonal dynamics and molecular events that accompany leukemic transformation. In this review, we examine our current understanding of the genomic heterogeneity in MPNs and how it affects disease progression and leukemic transformation. We focus on molecular events elicited by somatic mutations acquisition and discuss the emerging findings coming from single cell studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=415&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b6a1ba5e7664ee58f25810dd3d4986754ef1f98d\",\n",
      "        \"title\": \"Microfluidics-based single cell analysis: From transcriptomics to spatiotemporal multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=416&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f8fe3fc5610b7d710ff95d6777ff5d719caa9b5b\",\n",
      "        \"title\": \"A mineralizing pool of Gli1-expressing progenitors builds the tendon enthesis and demonstrates therapeutic potential.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=417&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6499659ba5a6bd751724797d221f7673cb567611\",\n",
      "        \"title\": \"Quantification of extracellular proteins, protein complexes and mRNAs in single cells by proximity sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=418&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bef6eb4ca916f44ff1290036828edc9805caf3b2\",\n",
      "        \"title\": \"Cancer cell-derived type I interferons instruct tumor monocyte polarization.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=419&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bfa1687788079f0207716140a499c9464f49427a\",\n",
      "        \"title\": \"Real-Time Characterization of Clonal Fate Decisions in Complex Leukemia Samples by Fluorescent Genetic Barcoding\",\n",
      "        \"abstract\": \"Clonal heterogeneity in acute myeloid leukemia (AML) forms the basis for treatment failure and relapse. Attempts to decipher clonal evolution and clonal competition primarily depend on deep sequencing approaches. However, this prevents the experimental confirmation of the identified disease-relevant traits on the same cell material. Here, we describe the development and application of a complex fluorescent genetic barcoding (cFGB) lentiviral vector system for the labeling and subsequent multiplex tracking of up to 48 viable AML clones by flow cytometry. This approach allowed the visualization of longitudinal changes in the in vitro growth behavior of multiplexed color-coded AML clones for up to 137 days. Functional studies of flow cytometry-enriched clones documented their stably inherited increase in competitiveness, despite the absence of growth-promoting mutations in exome sequencing data. Transplantation of aliquots of a color-coded AML cell mix into mice revealed the initial engraftment of similar clones and their subsequent differential distribution in the animals over time. Targeted RNA-sequencing of paired pre-malignant and de novo expanded clones linked gene sets associated with Myc-targets, embryonic stem cells, and RAS signaling to the foundation of clonal expansion. These results demonstrate the potency of cFGB-mediated clonal tracking for the deconvolution of verifiable driver-mechanisms underlying clonal selection in leukemia.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=420&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c6ea74d00e9c4264d419ec7f6e8cfa64a6fbeda4\",\n",
      "        \"title\": \"Use of intercellular proximity labeling to quantify and decipher cell-cell interactions directed by diversified molecular pairs\",\n",
      "        \"abstract\": \"FucoID is an intercellular proximity labeling technique for studying cell-cell interactions (CCIs) via fucosyltransferase (FT)\\u2013meditated fucosyl-biotinylation, which has been applied to probe antigen-specific dendritic cell (DC)\\u2013T cell interactions. In this system, bait cells of interest with cell surface\\u2013anchored FT are used to capture the interacting prey cells by transferring a biotin-modified substrate to prey cells. Here, we leveraged FucoID to study CCIs directed by different molecular pairs, e.g., programmed cell death protein-1(PD-1)/programmed cell death protein-ligand-1 (PD-L1), and identify unknown or little studied CCIs, e.g., the interaction of DCs and B cells. To expand the application of FucoID to complex systems, we also synthesized site-specific antibody-based FT conjugate, which substantially improves the ability of FucoID to probe molecular signatures of specific CCI when cells of interest (bait cells) cannot be purified, e.g., in clinical samples. Collectively, these studies demonstrate the general applicability of FucoID to study unknown CCIs in complex systems at a molecular resolution.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=421&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6f5da1047c09075a0776ee4d5d3a812c95bf4bc2\",\n",
      "        \"title\": \"The ectonucleotidase CD39 identifies tumor-reactive CD8+ T\\u00a0cells predictive of immune checkpoint blockade efficacy in human lung cancer.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=422&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"10d4b93cd9c8316dda5f75ac19058304704f88ae\",\n",
      "        \"title\": \"Small intestine and colon tissue-resident memory CD8+ T\\u00a0cells exhibit molecular heterogeneity and differential dependence on Eomes.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=423&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"902157c118fb0925aaa488e06c953548e822db22\",\n",
      "        \"title\": \"Artificial intelligence-based multi-omics analysis fuels cancer precision medicine.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=424&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d02424070e17ea798813b61429017100b26a0b7\",\n",
      "        \"title\": \"Bioinformatics approaches to analyzing CRISPR screen data: from dropout screens to single-cell CRISPR screens\",\n",
      "        \"abstract\": \"Background: Pooled CRISPR screen is a promising tool in drug targets or essential genes identification with the utilization of three different systems including CRISPR knockout (CRISPRko), CRISPR interference (CRISPRi) and CRISPR activation (CRISPRa). Aside from continuous improvements in technology, more and more bioinformatics methods have been developed to analyze the data obtained by CRISPR screens which facilitate better understanding of physiological effects. Results: Here, we provide an overview on the application of CRISPR screens and bioinformatics approaches to analyzing different types of CRISPR screen data. We also discuss mechanisms and underlying challenges for the analysis of dropout screens, sorting-based screens and single-cell screens. Conclusion: Different analysis approaches should be chosen based on the design of screens. This review will help community to better design novel algorithms and provide suggestions for wet-lab researchers to choose from different analysis methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=425&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"90c3c42c6af425a46e4d9aa11f49f8ffda835d67\",\n",
      "        \"title\": \"Cell type-specific changes identified by single-cell transcriptomics in Alzheimer\\u2019s disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=426&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4a2a4e74f092abfb629bb9de006f84ffcfd17c22\",\n",
      "        \"title\": \"Microfluidic single\\u2010cell multiomics analysis\",\n",
      "        \"abstract\": \"Cellular heterogeneity is essential to biological processes, such as embryonic development, cell differentiation, and the progression of disease. Recent years have seen the development of a variety of single\\u2010cell multiomics technologies that systemically codetect the genome, epigenome, transcriptome, and proteome for single\\u2010cell heterogeneity evaluation comprehensively. Microfluidics has emerged as a significant tool for single\\u2010cell multiomics techniques, enabling the analysis of the complex regulatory network associated with genome coding, epigenome regulation, and transcriptome/proteome expression in a single cell with increased detection sensitivity, accuracy, throughput, and integration. A review of state\\u2010of\\u2010the\\u2010art microfluidic single\\u2010cell multiomics analysis is presented here. Various microfluidics for isolating single cells are introduced first, highlighting their advantages, disadvantages, and applications in single\\u2010cell sequencing. Then, a comprehensive overview of microfluidic single\\u2010cell multiomics techniques is provided. In addition, a brief introduction of single\\u2010cell multiomics analysis in biological applications and clinical settings will be presented. Finally, we will conclude by discussing the future challenges and prospects of this field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=427&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"78c0cea1b81db0834b92af9665e3961d5f69de74\",\n",
      "        \"title\": \"Single-Cell Epitope-Transcriptomics Reveal Lung Stromal and Immune Cell Response Kinetics to Nanoparticle-delivered RIG-I and TLR4 Agonists\",\n",
      "        \"abstract\": \"Lung-resident and circulatory lymphoid, myeloid, and stromal cells, expressing various pattern recognition receptors (PRRs), detect pathogen and danger-associated molecular patterns (PAMPs/DAMPs), and defend against respiratory pathogens and injuries. Here, we report the early responses of murine lungs to nanoparticle-delivered PAMPs, specifically the RIG-I agonist poly-U/UC (PUUC), with or without the TLR4 agonist monophosphoryl lipid A (MPLA). Using cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq), we characterized the responses at 4 and 24 hours after intranasal administration. Within 4 hours, ribosome-associated transcripts decreased in both stromal and immune cells, followed by widespread interferon-stimulated gene (ISG) expression. Using RNA velocity, we show that lung-neutrophils dynamically regulate the synthesis of cytokines like CXCL-10, IL-1\\u03b1, and IL-1\\u03b2. Co-delivery of MPLA and PUUC increased chemokine synthesis and upregulated antimicrobial binding proteins targeting iron, manganese, and zinc in many cell types, including fibroblasts, endothelial cells, and epithelial cells. Overall, our results elucidate the early PAMP-induced cellular responses in the lung and demonstrate that stimulation of the RIG-I pathway, with or without TLR4 agonists, induces a ubiquitous microbial defense state in lung stromal and immune cells. Nanoparticle-delivered combination PAMPs may have applications in intranasal antiviral and antimicrobial therapies and prophylaxis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=428&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"687c1cd0b15caedea5d4e3ac88cffa6b86c143a8\",\n",
      "        \"title\": \"Functional analysis of structural variants in single cells using Strand-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=429&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ac47e6f88d448f4eb10ebda65459f795e99ed363\",\n",
      "        \"title\": \"Antibody binding reports spatial heterogeneities in cell membrane organization\",\n",
      "        \"abstract\": \"The spatial organization of cell membrane glycoproteins and glycolipids is critical for mediating the binding of ligands, receptors, and macromolecules on the plasma membrane. However, we currently do not have the methods to quantify the spatial heterogeneities of macromolecular crowding on live cell surfaces. In this work, we combine experiment and simulation to report crowding heterogeneities on reconstituted membranes and live cell membranes with nanometer spatial resolution. By quantifying the effective binding affinity of IgG monoclonal antibodies to engineered antigen sensors, we discovered sharp gradients in crowding within a few nanometers of the crowded membrane surface. Our measurements on human cancer cells support the hypothesis that raft-like membrane domains exclude bulky membrane proteins and glycoproteins. Our facile and high-throughput method to quantify spatial crowding heterogeneities on live cell membranes may facilitate monoclonal antibody design and provide a mechanistic understanding of plasma membrane biophysical organization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=430&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6af664fd27983eebd71bca37654f6efa8e766ef9\",\n",
      "        \"title\": \"High-content CRISPR screening in tumor immunology\",\n",
      "        \"abstract\": \"CRISPR screening is a powerful tool that links specific genetic alterations to corresponding phenotypes, thus allowing for high-throughput identification of novel gene functions. Pooled CRISPR screens have enabled discovery of innate and adaptive immune response regulators in the setting of viral infection and cancer. Emerging methods couple pooled CRISPR screens with parallel high-content readouts at the transcriptomic, epigenetic, proteomic, and optical levels. These approaches are illuminating cancer immune evasion mechanisms as well as nominating novel targets that augment T cell activation, increase T cell infiltration into tumors, and promote enhanced T cell cytotoxicity. This review details recent methodological advances in high-content CRISPR screens and highlights the impact this technology is having on tumor immunology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=431&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a1f07126630a6066a7179016d353465933063ac6\",\n",
      "        \"title\": \"Human gut microbiota-reactive DP8\\u03b1 regulatory T cells, signature and related emerging functions\",\n",
      "        \"abstract\": \"In mice, microbiota-induced Tregs both maintain intestinal homeostasis and provide resistance to immuno-pathologies in the adult. Identifying their human functional counterpart therefore represents an important goal. We discovered, in the human colonic lamina propria and blood, a FoxP3-negative IL-10-secreting Treg subset, which co-expresses CD4 and CD8\\u03b1 (hence named DP8\\u03b1) and displays a TCR-reactivity against Faecalibacterium prausnitzii, indicating a role for this symbiotic bacterium in their induction. Moreover, supporting their role in intestinal homeostasis, we previously reported both their drastic decrease in IBD patients and their protective role in vivo against intestinal inflammation, in mice. Here, we aimed at identifying the genomic, phenotypic and functional signatures of these microbiota-induced Tregs, towards delineating their physiological role(s) and clinical potential. Human F. prausnitzii-reactive DP8\\u03b1 Treg clones were derived from both the colonic lamina propria and blood. RNA-sequencing, flow cytometry and functional assays were performed to characterize their response upon activation and compare them to donor- and tissue-matched FoxP3+ Treg clones. DP8\\u03b1 Tregs exhibited a unique mixed Tr1-like/cytotoxic CD4+ T cell-profile and shared the ROR\\u03b3t and MAF master genes with mouse gut microbiota-induced FoxP3+ Tregs. We revealed their potent cytotoxic, chemotactic and IgA-promoting abilities, which were confirmed using in vitro assays. Therefore, besides their induction by a Clostridium bacterium, DP8\\u03b1 Tregs also partake master genes with mouse microbiota-induced Tregs. The present identification of their complete signature and novel functional properties, should be key in delineating the in vivo roles and therapeutic applications of these unique human microbiota-induced Tregs through their study in pathological contexts, particularly in inflammatory bowel diseases. Graphical Abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=432&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"800cbc3821dc8cb8de74b28deb86c59e74e6854e\",\n",
      "        \"title\": \"Immune biology of NSCLC revealed by single-cell technologies: implications for the development of biomarkers in patients treated with immunotherapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=433&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"35ad5480c6cc8de94a3362d2819f113c05c127c6\",\n",
      "        \"title\": \"Single-cell RNA-seq methods to interrogate virus-host interactions\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=434&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4200772247321d146f6fd3ad73390ee45ee120f7\",\n",
      "        \"title\": \"Engineered molecular sensors of cell surface crowding\",\n",
      "        \"abstract\": \"Cells mediate interactions with the extracellular environment through a crowded assembly of transmembrane proteins, glycoproteins and glycolipids on their plasma membrane. The extent to which surface crowding modulates the biophysical interactions of ligands, receptors, and other macromolecules is poorly understood due to the lack of methods to quantify surface crowding on native cell membranes. In this work, we demonstrate that physical crowding on reconstituted membranes and live cell surfaces attenuates the effective binding affinity of macromolecules such as IgG antibodies in a surface crowding-dependent manner. We combine experiment and simulation to design a crowding sensor based on this principle that provides a quantitative readout of cell surface crowding. Our measurements reveal that surface crowding decreases IgG antibody binding by 2-20 fold in live cells compared to a bare membrane surface, resulting in a cell surface osmotic pressure opposing binding of 1 - 4 kPa. Our sensors show that sialic acid, a negatively charged monosaccharide, contributes disproportionately to red blood cell surface crowding via electrostatic repulsion, despite occupying only ~1% of the total cell membrane by mass. We also observe significant differences in surface crowding for different cell types and find that expression of single oncogenes can both increase and decrease crowding, suggesting that surface crowding may be an indicator of both cell type and state. Our high-throughput, single-cell measurement of cell surface osmotic pressure may be combined with functional assays to enable further biophysical dissection of the cell surfaceome. Significance Statement Cells interact with each other and the extracellular environment through a crowded assembly of polymers on their plasma membranes. The high density of these surface polymers can generate physical crowding that impacts cell function. However, tools to quantify the extent and effect of surface crowding on live cell membranes are lacking. In this work, we design macromolecular sensors that act as direct reporters of cell surface crowding. We combine experiments on reconstituted and live cell surfaces with molecular dynamics simulations to provide a mechanistic understanding of how cell surface crowding reduces binding of soluble molecules, and we show that crowding varies significantly with cell type and is affected by oncogene expression.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=435&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"86c8004541bcdad3aa7f93d66b29da98aa7fcffc\",\n",
      "        \"title\": \"Single-cell multi-omics reveals dynamics of purifying selection of pathogenic mitochondrial DNA across human immune cells\",\n",
      "        \"abstract\": \"Cells experience intrinsic and extrinsic pressures that affect their proclivity to expand and persist in vivo. In congenital disorders caused by loss-of-function mutations in mitochondrial DNA (mtDNA), metabolic vulnerabilities may result in cell-type specific phenotypes and depletion of pathogenic alleles, contributing to purifying selection. However, the impact of pathogenic mtDNA mutations on the cellular hematopoietic landscape is not well understood. Here, we establish a multi-omics approach to quantify deletions in mtDNA alongside cell state features in single cells derived from Pearson syndrome patients. We resolve the interdependence between pathogenic mtDNA and lineage, including purifying selection against deletions in effector/memory CD8 T-cell populations and recent thymic emigrants and dynamics in other hematopoietic populations. Our mapping of lineage-specific purifying selection dynamics in primary cells from patients carrying pathogenic heteroplasmy provides a new perspective on recurrent clinical phenotypes in mitochondrial disorders, including cancer and infection, with potential broader relevance to age-related immune dysfunction.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=436&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ee3a4d735b3034a0501129fe0a9458a99f49c718\",\n",
      "        \"title\": \"Integration of CRISPR/Cas9 with artificial intelligence for improved cancer therapeutics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=437&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21039e16e3b4060193a52f7af6d8887f2eb9c5ad\",\n",
      "        \"title\": \"STAT3 gain-of-function mutations connect leukemia with autoimmune disease by pathological NKG2Dhi CD8+\\u00a0T\\u00a0cell dysregulation and accumulation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=438&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"546f72177a4e03e4538f58f13ce96c8ac15cfcce\",\n",
      "        \"title\": \"A dynamic atlas of immunocyte migration from the gut\",\n",
      "        \"abstract\": \"Dysbiosis in the gut microbiota impacts several systemic diseases. One possible mechanism is the migration of perturbed intestinal immunocytes to extra-intestinal tissues. Combining the Kaede photoconvertible mouse model and single-cell genomics, we generated a detailed map of migratory trajectories from the colon, at baseline and during intestinal and extra-intestinal inflammation. All colonic lineages emigrated from the colon in an S1P-dependent manner, dominated by B lymphocytes with a large continuous circulation of follicular B cells, which carried a gut-imprinted transcriptomic signature. T cell emigration was more selective, with distinct groups of ROR\\u03b3+ or IEL-like CD160+ cells in the spleen. Gut inflammation curtailed emigration, except for DCs disseminating to lymph nodes. Colon emigrating cells distributed differentially to tumor, skin inflammation, or arthritic synovium, the former dominated by myeloid cells in a chemokine-dependent manner. These results thus reveal specific cellular trails originating in the gut, influenced by microbiota, which can shape peripheral immunity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=439&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abfb7b86eac825ff02872fcea7a4661aa4f37a74\",\n",
      "        \"title\": \"Ascertaining cells\\u2019 synaptic connections and RNA expression simultaneously with barcoded rabies virus libraries\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=440&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"801b5ee8c98565d0a7781159d8648f137bb8020f\",\n",
      "        \"title\": \"Structure-preserving visualization for single-cell RNA-Seq profiles using deep manifold transformation with batch-correction\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=441&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3bc044df5abcbff608161495e0d963a044ff7beb\",\n",
      "        \"title\": \"Pooled screening of CAR T cells identifies diverse immune signaling domains for next-generation immunotherapies\",\n",
      "        \"abstract\": \"Chimeric antigen receptors (CARs) repurpose natural signaling components to retarget T cells to refractory cancers but have shown limited efficacy in persistent, recurrent malignancies. Here, we introduce \\u201cCAR Pooling,\\u201d a multiplexed approach to rapidly identify CAR designs with clinical potential. Forty CARs with signaling domains derived from a range of immune cell lineages were evaluated in pooled assays for their ability to stimulate critical T cell effector functions during repetitive stimulation that mimics long-term tumor antigen exposure. Several domains were identified from the tumor necrosis factor (TNF) receptor family that have been primarily associated with B cells. CD40 enhanced proliferation, whereas B cell\\u2013activating factor receptor (BAFF-R) and transmembrane activator and CAML interactor (TACI) promoted cytotoxicity. These functions were enhanced relative to clinical benchmarks after prolonged antigen stimulation, and CAR T cell signaling through these domains fell into distinct states of memory, cytotoxicity, and metabolism. BAFF-R CAR T cells were enriched for a highly cytotoxic transcriptional signature previously associated with positive clinical outcomes. We also observed that replacing the 4-1BB intracellular signaling domain with the BAFF-R signaling domain in a clinically validated B cell maturation antigen (BCMA)\\u2013specific CAR resulted in enhanced activity in a xenotransplant model of multiple myeloma. Together, these results show that CAR Pooling is a general approach for rapid exploration of CAR architecture and activity to improve the efficacy of CAR T cell therapies. Description CAR Pooling is a high-throughput approach to screening and testing CAR designs with the potential to enhance engineered T cell therapies for cancer. Test driving CAR T cells CAR T cells have already been through multiple generations, with each iteration improving signaling through the receptor. However, a strategy to broadly test multiple candidate intracellular signaling domains would allow for even better optimization. Here, Goodman et al. have developed CAR Pooling, a multiplexed approach to rapidly identify the top CAR signaling domains. The authors found using CAR Pooling that the intracellular signaling domain from BAFF-R improved CAR T cell activity; these results translated into control of tumor burden in vivo. Thus, CAR Pooling represents an effective approach to rapidly identify strategies to improve CAR T cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=442&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"97c1d9ca80bc035710dd3d03e7a90de3920a8572\",\n",
      "        \"title\": \"Multimodal and spatially resolved profiling identifies distinct patterns of T-cell infiltration in nodal B-cell lymphoma entities\",\n",
      "        \"abstract\": \"Summary T-cell-engaging immunotherapies have improved the treatment of nodal B-cell lymphoma, but responses vary highly. Future improvements of such therapies require better understanding of the variety of lymphoma-infiltrating T-cells. We employed single-cell RNA and T-cell receptor sequencing alongside quantification of surface proteins, flow cytometry and multiplexed immunofluorescence on 101 lymph nodes from healthy controls, and patients with diffuse large B-cell, mantle cell, follicular, or marginal zone lymphoma. This multimodal resource revealed entity-specific quantitative and spatial aberrations of the T-cell microenvironment. Clonal PD1+ TCF7\\u2212 but not PD1+ TCF7+ cytotoxic T-cells converged into terminally exhausted T-cells, the proportions of which were variable across entities and linked to inferior prognosis. In follicular and marginal zone lymphoma, we observed expansion of follicular helper and IKZF3+ regulatory T-cells, which were clonally related and inversely associated with tumor grading. Overall, we portray lymphoma-infiltrating T-cells with unprecedented comprehensiveness and decipher both beneficial and adverse dimensions of T-cell response.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=443&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"49037b6fa160160245c32926e84e7698375ba640\",\n",
      "        \"title\": \"JAK2V617F mutation drives vascular resident macrophages toward a pathogenic phenotype and promotes dissecting aortic aneurysm\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=444&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d5cebe2a7e72618f49ac419fd0c0398a5897981\",\n",
      "        \"title\": \"DiSCERN - Deep Single Cell Expression ReconstructioN for improved cell clustering and cell subtype and state detection\",\n",
      "        \"abstract\": \"Single cell sequencing provides detailed insights into biological processes including cell differentiation and identity. While providing deep cell-specific information, the method suffers from technical constraints, most notably a limited number of expressed genes per cell, which leads to suboptimal clustering and cell type identification. Here we present DISCERN, a novel deep generative network that reconstructs missing single cell gene expression using a reference dataset. DISCERN outperforms competing algorithms in expression inference resulting in greatly improved cell clustering, cell type and activity detection, and insights into the cellular regulation of disease. We used DISCERN to detect two unseen COVID-19-associated T cell types, cytotoxic CD4+ and CD8+ Tc2 T helper cells, with a potential role in adverse disease outcome. We utilized T cell fraction information of patient blood to classify mild or severe COVID-19 with an AUROC of 81% that can serve as a biomarker of disease stage. DISCERN can be easily integrated into existing single cell sequencing workflows and readily adapted to enhance various other biomedical data types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=445&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"82ad8e92e1078477defc9754fa568dba3c4ed24b\",\n",
      "        \"title\": \"Sphere-sequencing unveils local tissue microenvironments at single cell resolution\",\n",
      "        \"abstract\": \"The spatial organization of cells within tissues is tightly linked to their biological function. Yet, methods to probe the entire transcriptome of multiple native tissue microenvironments at single cell resolution are lacking. Here, we introduce spheresequencing, a method that enables the transcriptomic characterization of single cells within spatially distinct tissue niches. Sphere-sequencing of the mouse metastatic liver revealed previously uncharacterized zonated genes and ligand-receptor interactions enriched in different hepatic microenvironments and the metastatic niche.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=446&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ef5bd8741ca0a24e23ef67bd7791608d62b6099\",\n",
      "        \"title\": \"Tissue-specific transcriptional profiles and heterogeneity of natural killer cells and group 1 innate lymphoid cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=447&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"03688b95b05a9a694ee35edad8cc3ae8be85e6f0\",\n",
      "        \"title\": \"A review of spatial profiling technologies for characterizing the tumor microenvironment in immuno-oncology\",\n",
      "        \"abstract\": \"Interpreting the mechanisms and principles that govern gene activity and how these genes work according to -their cellular distribution in organisms has profound implications for cancer research. The latest technological advancements, such as imaging-based approaches and next-generation single-cell sequencing technologies, have established a platform for spatial transcriptomics to systematically quantify the expression of all or most genes in the entire tumor microenvironment and explore an array of disease milieus, particularly in tumors. Spatial profiling technologies permit the study of transcriptional activity at the spatial or single-cell level. This multidimensional classification of the transcriptomic and proteomic signatures of tumors, especially the associated immune and stromal cells, facilitates evaluation of tumor heterogeneity, details of the evolutionary trajectory of each tumor, and multifaceted interactions between each tumor cell and its microenvironment. Therefore, spatial profiling technologies may provide abundant and high-resolution information required for the description of clinical-related features in immuno-oncology. From this perspective, the present review will highlight the importance of spatial transcriptomic and spatial proteomics analysis along with the joint use of other sequencing technologies and their implications in cancers and immune-oncology. In the near future, advances in spatial profiling technologies will undoubtedly expand our understanding of tumor biology and highlight possible precision therapeutic targets for cancer patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=448&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4ce842919bf82cdb418b5081a12d8a66bb5fca44\",\n",
      "        \"title\": \"Integrative insights and clinical applications of single-cell sequencing in cancer immunotherapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=449&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"167c4607722d10f350c320bc36a5612e72deba00\",\n",
      "        \"title\": \"Graph representation learning in biomedicine and healthcare\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=450&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fef9f8006ef17e569122963eabc14af8e57d5817\",\n",
      "        \"title\": \"Multifactorial profiling of epigenetic landscapes at single-cell resolution using MulTI-Tag\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=451&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"080d977c1c20fd30f6c949b0ce241b261af12a09\",\n",
      "        \"title\": \"A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration with cell surface protein prediction and imputation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=452&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9df9a7247cc447f265b4870bac5a0e3374b08243\",\n",
      "        \"title\": \"AE-TPGG: a novel autoencoder-based approach for single-cell RNA-seq data imputation and dimensionality reduction\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=453&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ac74d308eac78f818c6797ed34db08edada2eed0\",\n",
      "        \"title\": \"Clonal expansion and epigenetic inheritance of long-lasting NK cell memory\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=454&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"81781c3b508ef8b29a48131f430362bef3340dc3\",\n",
      "        \"title\": \"Microfluidics-free single-cell genomics with templated emulsification\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=455&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9f725520056c563897c0cc4060f781f9f7a814b8\",\n",
      "        \"title\": \"Targeting the Immune-Fibrosis Axis in Myocardial Infarction and Heart Failure\",\n",
      "        \"abstract\": \"Cardiac fibrosis is causally linked to heart failure pathogenesis and adverse clinical outcomes. However, the precise fibroblast populations that drive fibrosis in the human heart and the mechanisms that govern their emergence remain incompletely defined. Here, we performed Cellular Indexing of Transcriptomes and Epitomes by sequencing (CITE-seq) in 22 explanted human hearts from healthy donors, acute myocardial infarction (MI), and chronic ischemic and non-ischemic cardiomyopathy patients. We identified a fibroblast trajectory marked by fibroblast activator protein (FAP) and periostin (POSTN) expression that was independent of myofibroblasts, peaked early after MI, remained elevated in chronic heart failure, and displayed a transcriptional signature consistent with fibrotic activity. We assessed the applicability of cardiac fibrosis models and demonstrated that mouse MI, angiotensin II/phenylephrine infusion, and pressure overload models were superior compared to cultured human heart and dermal fibroblasts in recapitulating cardiac fibroblast diversity including pathogenic cell states. Ligand-receptor analysis and spatial transcriptomics predicted interactions between macrophages, T cells, and fibroblasts within spatially defined niches. CCR2+ monocyte and macrophage states were the dominant source of ligands targeting fibroblasts. Inhibition of IL-1\\u03b2 signaling to cardiac fibroblasts was sufficient to suppress fibrosis, emergence, and maturation of FAP+POSTN+ fibroblasts. Herein, we identify a human fibroblast trajectory marked by FAP and POSTN expression that is associated with cardiac fibrosis and identify macrophage-fibroblast crosstalk mediated by IL-1\\u03b2 signaling as a key regulator of pathologic fibroblast differentiation and fibrosis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=456&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a25bb5bb4fe818396754f75945992fe56da25504\",\n",
      "        \"title\": \"A microfluidic Braille valve platform for on-demand production, combinatorial screening and sorting of chemically distinct droplets\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=457&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"68a90ccc63f4939a1eedcab77dd636cc52a2c710\",\n",
      "        \"title\": \"Recent advances in the field of single-cell proteomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=458&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"889af507da9e92d0c2ce8ccbb7f531baf0c8da54\",\n",
      "        \"title\": \"CMOT: Cross-Modality Optimal Transport for multimodal inference\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=459&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"92f3e22ff499c53cea0cd89076e6109b4e09c39f\",\n",
      "        \"title\": \"White Adipose Tissue Compositional and Transcriptional Patterns with Progressive Glucose Intolerance in Persons with HIV\",\n",
      "        \"abstract\": \"Subcutaneous adipose tissue (SAT) is a critical regulator of systemic metabolic homeostasis. Persons with HIV (PWH) have an increased risk of metabolic diseases and significant alterations in the SAT immune environment compared with the general population. We generated a comprehensive SAT atlas to characterize cellular compositional and transcriptional changes in 59 PWH with a spectrum of metabolic health. Glucose intolerance was associated with increased lipid-associated macrophages and CD4+ and CD8+ T effector memory cells, and decreased perivascular macrophages. We observed a coordinated intercellular regulatory program which enriched for genes related to inflammation and lipid-processing across multiple cell types as glucose intolerance increased. Increased CD4+ effector memory tissue resident cells most strongly associated with altered expression of adipocyte genes critical for lipid metabolism and cellular regulation. Many of these findings were present in a separate group of 32 diabetic HIV-negative persons, suggesting these changes are not specific to HIV.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=460&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8a9f10e8e6807cb6df0266a34c076b0d2e52193\",\n",
      "        \"title\": \"Characterization of spleen and lymph node cell types via CITE-seq and machine learning methods\",\n",
      "        \"abstract\": \"The spleen and lymph nodes are important functional organs for human immune system. The identification of cell types for spleen and lymph nodes is helpful for understanding the mechanism of immune system. However, the cell types of spleen and lymph are highly diverse in the human body. Therefore, in this study, we employed a series of machine learning algorithms to computationally analyze the cell types of spleen and lymph based on single-cell CITE-seq sequencing data. A total of 28,211 cell data (training vs. test = 14,435 vs. 13,776) involving 24 cell types were collected for this study. For the training dataset, it was analyzed by Boruta and minimum redundancy maximum relevance (mRMR) one by one, resulting in an mRMR feature list. This list was fed into the incremental feature selection (IFS) method, incorporating four classification algorithms (deep forest, random forest, K-nearest neighbor, and decision tree). Some essential features were discovered and the deep forest with its optimal features achieved the best performance. A group of related proteins (CD4, TCRb, CD103, CD43, and CD23) and genes (Nkg7 and Thy1) contributing to the classification of spleen and lymph nodes cell types were analyzed. Furthermore, the classification rules yielded by decision tree were also provided and analyzed. Above findings may provide helpful information for deepening our understanding on the diversity of cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=461&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6328132b83f4ee8cfea45beb4956b8791c4426c\",\n",
      "        \"title\": \"Spatial transcriptomics technology in cancer research\",\n",
      "        \"abstract\": \"In recent years, spatial transcriptomics (ST) technologies have developed rapidly and have been widely used in constructing spatial tissue atlases and characterizing spatiotemporal heterogeneity of cancers. Currently, ST has been used to profile spatial heterogeneity in multiple cancer types. Besides, ST is a benefit for identifying and comprehensively understanding special spatial areas such as tumor interface and tertiary lymphoid structures (TLSs), which exhibit unique tumor microenvironments (TMEs). Therefore, ST has also shown great potential to improve pathological diagnosis and identify novel prognostic factors in cancer. This review presents recent advances and prospects of applications on cancer research based on ST technologies as well as the challenges.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=462&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b63701cd0b8a2d5450e6b4817320a7b0a8b28d62\",\n",
      "        \"title\": \"Deep learning detector for high precision monitoring of cell encapsulation statistics in microfluidic droplets.\",\n",
      "        \"abstract\": \"Encapsulation of cells inside microfluidic droplets is central to several applications involving cellular analysis. Although, theoretically the encapsulation statistics are expected to follow a Poisson distribution, experimentally this may not be achieved due to lack of full control of the experimental variables and conditions. Therefore, there is a need for automatic detection of droplets and cell count enumeration within droplets so a process control feedback to adjust experimental conditions can be implemented. In this study, we use a deep learning object detector called You Only Look Once (YOLO), an influential class of object detectors with several benefits over traditional methods. This paper investigates the application of both YOLOv3 and YOLOv5 object detectors in the development of an automated droplet and cell detector. Experimental data was obtained from a microfluidic flow focusing device with a dispersed phase of cancer cells. The microfluidic device contained an expansion chamber downstream of the droplet generator, allowing for visualization and recording of cell-encapsulated droplet images. In the procedure, a droplet bounding box is predicted, then cropped from the original image for the individual cells to be detected through a separate model for further examination. The system includes a production set for additional performance analysis with Poisson statistics while providing an experimental workflow with both droplet and cell models. The training set is collected and preprocessed before labeling and applying image augmentations, allowing for a generalizable object detector. Precision and recall were utilized as a validation and test set metric, resulting in a high mean average precision (mAP) metric for an accurate droplet detector. To examine model limitations, the predictions were compared to ground truth labels, illustrating that the YOLO predictions closely matched with the droplet and cell labels. Furthermore, it is demonstrated that droplet enumeration from the YOLOv5 model is consistent with hand counted ratios and the Poisson distribution, confirming that the platform can be used in real-time experiments for cell encapsulation optimization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=463&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f12c4e0c5834bd89cad11902e32fec6cabe75ea2\",\n",
      "        \"title\": \"InfoOT: Information Maximizing Optimal Transport\",\n",
      "        \"abstract\": \"Optimal transport aligns samples across distributions by minimizing the transportation cost between them, e.g., the geometric distances. Yet, it ignores coherence structure in the data such as clusters, does not handle outliers well, and cannot integrate new data points. To address these drawbacks, we propose InfoOT, an information-theoretic extension of optimal transport that maximizes the mutual information between domains while minimizing geometric distances. The resulting objective can still be formulated as a (generalized) optimal transport problem, and can be efficiently solved by projected gradient descent. This formulation yields a new projection method that is robust to outliers and generalizes to unseen samples. Empirically, InfoOT improves the quality of alignments across benchmarks in domain adaptation, cross-domain retrieval, and single-cell alignment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=464&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6e0fc8c1b51ccc520024c76448b9974b2b98e9c\",\n",
      "        \"title\": \"Vascular endothelial cell development and diversity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=465&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"32f5627cc404ac9b1c47952942ca0f3c122765f0\",\n",
      "        \"title\": \"Characterizing the biology of primary brain tumors and their microenvironment via single-cell profiling methods\",\n",
      "        \"abstract\": \"Abstract Genomic and transcriptional heterogeneity is prevalent among the most common and aggressive primary brain tumors in children and adults. Over the past 20 years, advances in bioengineering, biochemistry and bioinformatics have enabled the development of an array of techniques to study tumor biology at single-cell resolution. The application of these techniques to study primary brain tumors has helped advance our understanding of their intra-tumoral heterogeneity and uncover new insights regarding their co-option of developmental programs and signaling from their microenvironment to promote tumor proliferation and invasion. These insights are currently being harnessed to develop new therapeutic approaches. Here we provide an overview of current single-cell techniques and discuss relevant biology and therapeutic insights uncovered by their application to primary brain tumors in children and adults.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=466&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7faa0196d334bef8e75ed42be9cf688643702d37\",\n",
      "        \"title\": \"From multitude to singularity: An up-to-date overview of scRNA-seq data generation and analysis\",\n",
      "        \"abstract\": \"Single cell RNA sequencing (scRNA-seq) is today a common and powerful technology in biomedical research settings, allowing to profile the whole transcriptome of a very large number of individual cells and reveal the heterogeneity of complex clinical samples. Traditionally, cells have been classified by their morphology or by expression of certain proteins in functionally distinct settings. The advent of next generation sequencing (NGS) technologies paved the way for the detection and quantitative analysis of cellular content. In this context, transcriptome quantification techniques made their advent, starting from the bulk RNA sequencing, unable to dissect the heterogeneity of a sample, and moving to the first single cell techniques capable of analyzing a small number of cells (1\\u2013100), arriving at the current single cell techniques able to generate hundreds of thousands of cells. As experimental protocols have improved rapidly, computational workflows for processing the data have also been refined, opening up to novel methods capable of scaling computational times more favorably with the dataset size and making scRNA-seq much better suited for biomedical research. In this perspective, we will highlight the key technological and computational developments which have enabled the analysis of this growing data, making the scRNA-seq a handy tool in clinical applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=467&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c35d0f609c3065967cad68241020098a3b938c34\",\n",
      "        \"title\": \"The expanding vistas of spatial transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=468&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"117d752d833d54e820968e4a1074d79095152539\",\n",
      "        \"title\": \"Review: Challenges and perspectives in applying single nuclei RNA-seq technology in plant biology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=469&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b1a155dfeeadde429aaaa83ccd11441ae76ecef\",\n",
      "        \"title\": \"Revealing the heterogeneity of CD4+ T cells through single-cell transcriptomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=470&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d3d362b604539702af10629b664d72cbe49e5893\",\n",
      "        \"title\": \"Tumour infiltrating B cells discriminate checkpoint blockade-induced responses.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=471&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"34877e4d9f8546b8dde21c0c8e200d7482858cd8\",\n",
      "        \"title\": \"Multimodal single\\u2010cell analysis provides novel insights on ankylosing spondylitis in females\",\n",
      "        \"abstract\": \"Dear Editor, Ankylosing spondylitis (AS) is a chronic rheumatic disease that causes disability and severe impairment in quality of life, especially in females.1,2 Based on clinical observations, diagnosing female patients with AS is challenging because of the minor radiation damage. However, little is known about how large heterogeneous circulating immune cells are involved in AS development in females.3 To overcome these limitations, single-cell-resolved gene expression profiling was used to characterize the immune cell status profile in the blood of female AS patients. To identify targets specific to female AS in heterogeneous cell populations, single-cell RNA sequencing (scRNA-seq) was performed on peripheral blood mononuclear cells (PBMCs) of three female patients with AS and five sex-matched (female) healthy control individuals (Figure 1A and Table S1). For visualization, scRNAseq data of PBMCs were integrated for unsupervised dimension-reduction clustering using uniform manifold approximation and projection (Figure 1B). Immune cell features were identified (Figure 1B,C), and the expression profile of the immune marker genes in each cluster was confirmed (Figure 1D). Furthermore, we observed a dynamic change in the composition of PBMCs from female AS patients (Figure 2A and Table S2). To explore disease-associated expression features, differential expression analysis was performed to determine the significance-filtered differentially expressed genes (DEGs) in patients and healthy controls (Table S3). The gene ontology pathway analysis of each cell type was conducted based on the differential expression analysis between the patient group (n= 3) and the healthy control group (n= 5) (Figure 2B). To further identify critical targets in female patients with AS, feature selection was performed using multiple filters that collected the common DEGs from the strictsignificance-filtered (p< .001) DEGs of each cell type in the patient group (n = 3) compared to the healthy group (n = 5) (Figure S1 and Table S3). Among the 167 DEGs in\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=472&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aacc6799962569a1306505c151736e10a381e33a\",\n",
      "        \"title\": \"Technology meets TILs: Deciphering T\\u00a0cell function in the -omics era.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=473&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"81c5851a766ce8a353e5b906a583bac5d3efde43\",\n",
      "        \"title\": \"Application of Deep Learning on Single-cell RNA Sequencing Data Analysis: A Review\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=474&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5f55e4e3c1dbf1abd740d6a10664fd5231b59870\",\n",
      "        \"title\": \"Spatial maps of T\\u00a0cell receptors and transcriptomes reveal distinct immune niches and interactions in the adaptive immune response.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=475&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a7c965e4bc8dcced10c53265d27738fdfb94e89\",\n",
      "        \"title\": \"CellCallEXT: Analysis of Ligand\\u2013Receptor and Transcription Factor Activities in Cell\\u2013Cell Communication of Tumor Immune Microenvironment\",\n",
      "        \"abstract\": \"Simple Summary CellCall is an R package tool that is used to analyze cell\\u2013cell communication based on transcription factor (TF) activities calculated by cell-type specificity of target genes and thus cannot directly handle two-condition comparisons. We developed CellCallEXT to complement CellCall. CellCallEXT can directly identify ligand\\u2013receptor (L\\u2013R) interactions that alter the expression profiles of downstream genes between two conditions, such as tumor and healthy tissue. Scoring in CellCallEXT quantitatively integrates expression of ligands, receptors, TFs, and target genes (TGs). The pathway enrichment analysis and visualization modules allow biologists to investigate how disease alters cell\\u2013cell communication. Furthermore, Reactome pathways were added into CellCallEXT to expand the L\\u2013R\\u2013TF database. Abstract (1) Background: Single-cell RNA sequencing (scRNA-seq) data are useful for decoding cell\\u2013cell communication. CellCall is a tool that is used to infer inter- and intracellular communication pathways by integrating paired ligand\\u2013receptor (L\\u2013R) and transcription factor (TF) activities from steady-state data and thus cannot directly handle two-condition comparisons. For tumor and healthy status, it can only individually analyze cells from tumor or healthy tissue and examine L\\u2013R pairs only identified in either tumor or healthy controls, but not both together. Furthermore, CellCall is highly affected by gene expression specificity in tissues. (2) Methods: CellCallEXT is an extension of CellCall that deconvolutes intercellular communication and related internal regulatory signals based on scRNA-seq. Information on Reactome was retrieved and integrated with prior knowledge of L\\u2013R\\u2013TF signaling and gene regulation datasets of CellCall. (3) Results: CellCallEXT was successfully applied to examine tumors and immune cell microenvironments and to identify the altered L\\u2013R pairs and downstream gene regulatory networks among immune cells. Application of CellCallEXT to scRNA-seq data from patients with deficiency of adenosine deaminase 2 demonstrated its ability to impute dysfunctional intercellular communication and related transcriptional factor activities. (4) Conclusions: CellCallEXT provides a practical tool to examine intercellular communication in disease based on scRNA-seq data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=476&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"23351088e0970a220edc7da95a44196a4431b4e6\",\n",
      "        \"title\": \"The industrial genomic revolution: A new era in neuroimmunology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=477&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1d31c980e2d96f346576a1a33df0d332c171fc08\",\n",
      "        \"title\": \"Lipopolysaccharide distinctively alters human microglia transcriptomes to resemble microglia from Alzheimer's disease mouse models\",\n",
      "        \"abstract\": \"ABSTRACT Alzheimer's disease (AD) is the most common form of dementia, and risk-influencing genetics implicates microglia and neuroimmunity in the pathogenesis of AD. Induced pluripotent stem cell (iPSC)-derived microglia (iPSC-microglia) are increasingly used as a model of AD, but the relevance of historical immune stimuli to model AD is unclear. We performed a detailed cross-comparison over time on the effects of combinatory stimulation of iPSC-microglia, and in particular their relevance to AD. We used single-cell RNA sequencing to measure the transcriptional response of iPSC-microglia after 24\\u2005h and 48\\u2005h of stimulation with prostaglandin E2 (PGE2) or lipopolysaccharide (LPS)+interferon gamma (IFN-\\u03b3), either alone or in combination with ATP\\u03b3S. We observed a shared core transcriptional response of iPSC-microglia to ATP\\u03b3S and to LPS+IFN-\\u03b3, suggestive of a convergent mechanism of action. Across all conditions, we observed a significant overlap, although directional inconsistency to genes that change their expression levels in human microglia from AD patients. Using a data-led approach, we identify a common axis of transcriptomic change across AD genetic mouse models of microglia and show that only LPS provokes a transcriptional response along this axis in mouse microglia and LPS+IFN-\\u03b3 in human iPSC-microglia. This article has an associated First Person interview with the first author of the paper.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=478&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"39b2c346cd76b0cf8e38e62a10d92e33353fcc1f\",\n",
      "        \"title\": \"Comprehensive single-cell transcriptional profiling defines shared and unique epithelial injury responses during kidney fibrosis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=479&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"05480c4e1165e6b5708e729661147e4b5f72bc20\",\n",
      "        \"title\": \"Type 1 interferon remodels normal and neoplastic hematopoiesis in human\",\n",
      "        \"abstract\": \"Inflammatory cytokines perturb hematopoietic stem cell (HSC) homeostasis and modulate the fitness of neoplastic HSC clones in mouse models. However, the study of cytokines in human hematopoiesis is challenging due to the concerted activities of multiple cytokines across physiologic and pathologic processes. To overcome this limitation, we leveraged serial bone marrow samples from patients with CALR-mutated myeloproliferative neoplasms who were treated with recombinant interferon-alpha (IFNa). We interrogated baseline and IFNa-treated CD34+ stem and progenitor cells using single-cell multi-omics platforms that directly link, within the same cell, the mutation status, whole transcriptomes and immunophenotyping or chromatin accessibility. We identified a novel IFNa-induced inflammatory granulocytic progenitor defined by expression and activities of RFX2/3 and AP-1 transcription factors, with evidence supporting a direct differentiation from HSCs. On the other hand, IFNa also induced a significant B-lymphoid progenitor expansion and proliferation, associated with enhanced activities of PU.1 and its co- regulator TCF3, as well as decreased accessibility of megakaryocytic-erythroid transcription factor GATA1 binding sites in HSCs. In the neoplastic hematopoiesis, the lymphoid expansion was constrained by a preferential myeloid skewing of the mutated cells, linked with increased myeloid proliferation and enhanced CEBPA and GATA1 activities compared to wildtype cells. Further, IFNa caused a downregulation of the TNFa signaling pathway, with downregulation of NFKB and AP-1 transcription factors. Thus, IFNa simultaneously initiated both \\u2013 pro-inflammatory and anti- inflammatory \\u2013 cell states within the same hematopoiesis, and its phenotypic impact varied as a function of the underlying HSC state and mutation status.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=480&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4c8585739065ea5d5d27b2fca6c5c6dee690e78\",\n",
      "        \"title\": \"Time-resolved single-cell RNA-seq using metabolic RNA labelling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=481&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cab24dd9025bd5d6951402136ce8495cc4debf6f\",\n",
      "        \"title\": \"In situ tools for chromatin structural epigenomics\",\n",
      "        \"abstract\": \"Technological progress over the past 15\\u2009years has fueled an explosion in genome\\u2010wide chromatin profiling tools that take advantage of low\\u2010cost short\\u2010read sequencing technologies to map particular chromatin features. Here, we survey the recent development of epigenomic tools that provide precise positions of chromatin proteins genome\\u2010wide in intact cells or nuclei. Some profiling tools are based on tethering Micrococcal Nuclease to chromatin proteins of interest in situ, whereas others similarly tether Tn5 transposase to integrate DNA sequencing adapters (tagmentation) and so eliminate the need for library preparation. These in situ cleavage and tagmentation tools have gained in popularity over the past few years, with many protocol enhancements and adaptations for single\\u2010cell and spatial chromatin profiling. The application of experimental and computational tools to address problems in gene regulation, eukaryotic development, and human disease are helping to define the emerging field of chromatin structural epigenomics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=482&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4e252995c652bbd88ff0fddd3bce87573b838b0\",\n",
      "        \"title\": \"Integrating Micro and Nano Technologies for Cell Engineering and Analysis: Toward the Next Generation of Cell Therapy Workflows.\",\n",
      "        \"abstract\": \"The emerging field of cell therapy offers the potential to treat and even cure a diverse array of diseases for which existing interventions are inadequate. Recent advances in micro and nanotechnology have added a multitude of single cell analysis methods to our research repertoire. At the same time, techniques have been developed for the precise engineering and manipulation of cells. Together, these methods have aided the understanding of disease pathophysiology, helped formulate corrective interventions at the cellular level, and expanded the spectrum of available cell therapeutic options. This review discusses how micro and nanotechnology have catalyzed the development of cell sorting, cellular engineering, and single cell analysis technologies, which have become essential workflow components in developing cell-based therapeutics. The review focuses on the technologies adopted in research studies and explores the opportunities and challenges in combining the various elements of cell engineering and single cell analysis into the next generation of integrated and automated platforms that can accelerate preclinical studies and translational research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=483&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"790e3a8ad8718788e35b3a0fdd34cb22338162ef\",\n",
      "        \"title\": \"A guide to systems-level immunomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=484&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1e541f2945fce225395f90e23a00232e4b8369cd\",\n",
      "        \"title\": \"Exploring glioblastoma stem cell heterogeneity: Immune microenvironment modulation and therapeutic opportunities\",\n",
      "        \"abstract\": \"Despite its growing use in cancer treatment, immunotherapy has been virtually ineffective in clinical trials for gliomas. The inherently cold tumor immune microenvironment (TIME) in gliomas, characterized by a high ratio of pro-tumor to anti-tumor immune cell infiltrates, acts as a seemingly insurmountable barrier to immunotherapy. Glioma stem cells (GSCs) within these tumors are key contributors to this cold TIME, often functioning indirectly through activation and recruitment of pro-tumor immune cell types. Furthermore, drivers of GSC plasticity and heterogeneity (e.g., reprogramming transcription factors, epigenetic modifications) are associated with induction of immunosuppressive cell states. Recent studies have identified GSC-intrinsic mechanisms, including functional mimicry of immune suppressive cell types, as key determinants of anti-tumor immune escape. In this review, we cover recent advancements in our understanding of GSC-intrinsic mechanisms that modulate GSC-TIME interactions and discuss cutting-edge techniques and bioinformatics platforms available to study immune modulation at high cellular resolution with exploration of both malignant (i.e., GSC) and non-malignant (i.e., immune) cell fractions. Finally, we provide insight into the therapeutic opportunities for targeting immunomodulatory GSC-intrinsic mechanisms to potentiate immunotherapy response in gliomas.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=485&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d926399dea8ced87050eadc70d2cfbd4ae83b204\",\n",
      "        \"title\": \"Combined multidimensional single-cell protein and RNA profiling dissects the cellular and functional heterogeneity of thymic epithelial cells\",\n",
      "        \"abstract\": \"The network of thymic stromal cells provides essential niches with unique molecular cues controlling T-cell development and selection. Recent single-cell RNA-sequencing studies uncovered a large transcriptional heterogeneity among thymic epithelial cells (TEC) demonstrating a previously unappreciated complexity. However, there are only very few cell markers that allow a comparable phenotypic identification of TEC. Here we deconvoluted by massively parallel flow cytometry and machine learning known and novel TEC phenotypes into novel subpopulations and related these by CITEseq to the corresponding TEC subtypes defined by the cells\\u2019 individual RNA profiles. This approach phenotypically identified perinatal cTEC, physically located these cells within the cortical stromal scaffold, displayed their dynamic change during the life course and revealed their exceptional efficiency in positively selecting immature thymocytes. Collectively, we have identified novel markers that allow for an unprecedented dissection of the thymus stromal complexity, the cells physical isolation and assignment of specific functions to individual TEC subpopulations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=486&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3d813e4877677f3ee85cd6ae872295bbcc272524\",\n",
      "        \"title\": \"Role of lymphoid lineage cells aberrantly expressing alarmins S100A8/A9 in determining the severity of COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=487&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c3a1dd3d1e7a130987d695229b19de3e87f2fc4\",\n",
      "        \"title\": \"Mostly natural sequencing-by-synthesis for scRNA-seq using Ultima sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=488&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f30ad59d54f4b596417e9061f50736abb2dad46b\",\n",
      "        \"title\": \"Murine fetal bone marrow does not support functional hematopoietic stem and progenitor cells until birth\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=489&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f8093363184dc44064f80eed29785d5ffaae0841\",\n",
      "        \"title\": \"Increased p53 expression induced by APR-246 reprograms tumor-associated macrophages to augment immune checkpoint blockade\",\n",
      "        \"abstract\": \"In addition to playing a major role in tumor cell biology, p53 generates a microenvironment that promotes antitumor immune surveillance via tumor-associated macrophages. We examined whether increasing p53 signaling in the tumor microenvironment influences antitumor T cell immunity. Our findings indicate that increased p53 signaling induced either pharmacologically with APR-246 (eprenetapopt) or in p53-overexpressing transgenic mice can disinhibit antitumor T cell immunity and augment the efficacy of immune checkpoint blockade. We demonstrated that increased p53 expression in tumor-associated macrophages induces canonical p53-associated functions such as senescence and activation of a p53-dependent senescence-associated secretory phenotype. This was linked with decreased expression of proteins associated with M2 polarization by tumor-associated macrophages. Our preclinical data led to the development of a clinical trial in patients with solid tumors combining APR-246 with pembrolizumab. Biospecimens from select patients participating in this ongoing trial showed that there was a suppression of M2-polarized myeloid cells and increase in T cell proliferation with therapy in those who responded to the therapy. Our findings, based on both genetic and a small molecule\\u2013based pharmacological approach, suggest that increasing p53 expression in tumor-associated macrophages reprograms the tumor microenvironment to augment the response to immune checkpoint blockade.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=490&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b2266bd0d5d381b643533fb457095341d74a4115\",\n",
      "        \"title\": \"A ROR\\u03b3t+ cell instructs gut microbiota-specific Treg cell differentiation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=491&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c73486ee3356c25d35107fba09060be1d3080c26\",\n",
      "        \"title\": \"Single cell RNA\\u2010sequencing: A powerful yet still challenging technology to study cellular heterogeneity\",\n",
      "        \"abstract\": \"Almost all biomedical research to date has relied upon mean measurements from cell populations, however it is well established that what it is observed at this macroscopic level can be the result of many interactions of several different single cells. Thus, the observable macroscopic \\u2018average\\u2019 cannot outright be used as representative of the \\u2018average cell\\u2019. Rather, it is the resulting emerging behaviour of the actions and interactions of many different cells. Single\\u2010cell RNA sequencing (scRNA\\u2010Seq) enables the comparison of the transcriptomes of individual cells. This provides high\\u2010resolution maps of the dynamic cellular programmes allowing us to answer fundamental biological questions on their function and evolution. It also allows to address medical questions such as the role of rare cell populations contributing to disease progression and therapeutic resistance. Furthermore, it provides an understanding of context\\u2010specific dependencies, namely the behaviour and function that a cell has in a specific context, which can be crucial to understand some complex diseases, such as diabetes, cardiovascular disease and cancer. Here, we provide an overview of scRNA\\u2010Seq, including a comparative review of emerging technologies and computational pipelines. We discuss the current and emerging applications and focus on tumour heterogeneity a clear example of how scRNA\\u2010Seq can provide new understanding of a complex disease. Additionally, we review the limitations and highlight the need of powerful computational pipelines and reproducible protocols for the broader acceptance of this technique in basic and clinical research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=492&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6328f83a69b8e3ce052fc3fe5d3d910e2db5d18d\",\n",
      "        \"title\": \"Big data in basic and translational cancer research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=493&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a257b4e4309c4fb337989bfd840ab31e364ccdb\",\n",
      "        \"title\": \"Reducing the Complexity of Fc Glycan Enables the Construction of Antibody Conjugates with Unexpected High Efficiency and Payload Capacity via Glycoengineering\",\n",
      "        \"abstract\": \"Glycoengineering has been demonstrated as a powerful approach to construct site-specific antibody conjugates. However, most glycoengineering strategies take of hours to days to achieve full conversion. Here, using unnatural fucosylation as an example, we show that reduce the complexity of Fc glycan dramatically boost its enzymatic conjugation. The fucosyl-biotinylation reaction on the created simple monoantennary LacNAc could be finished in 5 min while it takes 16 h on the natural biantennary LacNAc. More importantly, this unnatural fucosylation of the created simple LacNAc on antibody accommodate payloads over 15,000 Da, which allows the enzymatic construction of antibody-oligonucleotide conjugate (AOC) and bispecific antibody. Further applications of antibody conjugates in animal model and human cancer organoid indicates the translational potential of this platform, especially for constructing unlimited antibody-biomacromolecules (X) conjugate, \\u201cAXC\\u201d, as therapeutics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=494&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e46bcb8af2143f200d8bd563dd4018bdbc18dd90\",\n",
      "        \"title\": \"Taz protects hematopoietic stem cells from an aging-dependent decrease in PU.1 activity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=495&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e9f1fc15e5a9ac24e0de59e19a9939690e47e08e\",\n",
      "        \"title\": \"Chemoenzymatic Measurement of Cell-surface Glycan in Single-cell Multiomics: LacNAc as an Example\",\n",
      "        \"abstract\": \"Despite the rich information of a cell\\u2019s physiological state encoded in the dynamic changes of cell-surface glycans, methods of capturing glycosylation states at the single-cell level are quite limited. Here we report a chemoenzymatic single-cell N-acetyllactosamine (LacNAc) detection method via tagging the LacNAc with a specific DNA barcode. Compared to the lectin-based glycan detection, the chemoenzymatic labeling does not change the transcriptional status of immune cells and is more compatible with scRNA-seq. Integrated analysis of LacNAc and transcriptome of T cells at a single-cell level reveals that the quantity of cell-surface LacNAc is significantly upregulated in activated CD8+ T cells but maintained at the basal level in quiescent CD8+ T cells (i.e., naive and central memory T cells). Further analysis confirms that the LacNAc level is positively correlated to the glycolytic activity of CD8+ T cells at all statues. Taken together, our study demonstrates the feasibility of chemoenzymatic detection of cell-surface glycan in single-cell RNA sequencing-based multiomics with information of TCR sequence and cell-surface epitopes (i.e., scTCR and CITE-seq) and offers a new way to characterize the biological role of glycan in diversified physiological states.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=496&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4a238019d7855c0d778fac09a8c8e54b334705e2\",\n",
      "        \"title\": \"Hematopoietic differentiation at single-cell resolution in NPM1-mutated AML\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=497&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a135c46b77aacb2fedfb710734d0a4bbc576710\",\n",
      "        \"title\": \"Post-infusion CAR TReg cells identify patients resistant to CD19-CAR therapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=498&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"67fcefb25c23426ef978b697b5fc53e606b7eabe\",\n",
      "        \"title\": \"Advancing T cell\\u2013based cancer therapy with single-cell technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=499&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"78bed1bdffbb20abde566f8bc3651ad2cf27ddbe\",\n",
      "        \"title\": \"Tumor vessel co-option: The past & the future\",\n",
      "        \"abstract\": \"Tumor vessel co-option (VCO) is a non-angiogenic vascularization mechanism that is a possible cause of resistance to anti-angiogenic therapy (AAT). Multiple tumors are hypothesized to primarily rely on growth factor signaling-induced sprouting angiogenesis, which is often inhibited during AAT. During VCO however, tumors invade healthy tissues by hijacking pre-existing blood vessels of the host organ to secure their blood and nutrient supply. Although VCO has been described in the context of AAT resistance, the molecular mechanisms underlying this process and the profile and characteristics of co-opted vascular cell types (endothelial cells (ECs) and pericytes) remain poorly understood, resulting in the lack of therapeutic strategies to inhibit VCO (and to overcome AAT resistance). In the past few years, novel next-generation technologies (such as single-cell RNA sequencing) have emerged and revolutionized the way of analyzing and understanding cancer biology. While most studies utilizing single-cell RNA sequencing with focus on cancer vascularization have centered around ECs during sprouting angiogenesis, we propose that this and other novel technologies can be used in future investigations to shed light on tumor EC biology during VCO. In this review, we summarize the molecular mechanisms driving VCO known to date and introduce the models used to study this phenomenon to date. We highlight VCO studies that recently emerged using sequencing approaches and propose how these and other novel state-of-the-art methods can be used in the future to further explore ECs and other cell types in the VCO process and to identify potential vulnerabilities in tumors relying on VCO. A better understanding of VCO by using novel approaches could provide new answers to the many open questions, and thus pave the way to develop new strategies to control and target tumor vascularization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=500&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1c4797aade42f2aa1f0b9b0d42262bee4678fd95\",\n",
      "        \"title\": \"DEMOC: a deep embedded multi-omics learning approach for clustering single-cell CITE-seq data\",\n",
      "        \"abstract\": \"Advances in single-cell RNA sequencing (scRNA-seq) technologies has provided an unprecedent opportunity for cell-type identification. As clustering is an effective strategy towards cell-type identification, various computational approaches have been proposed for clustering scRNA-seq data. Recently, with the emergence of cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq), the cell surface expression of specific proteins and the RNA expression on the same cell can be captured, which provides more comprehensive information for cell analysis. However, existing single cell clustering algorithms are mainly designed for single-omic data, and have difficulties in handling multi-omics data with diverse characteristics efficiently. In this study, we propose a novel deep embedded multi-omics clustering with collaborative training (DEMOC) model to perform joint clustering on CITE-seq data. Our model can take into account the characteristics of transcriptomic and proteomic data, and make use of the consistent and complementary information provided by different data sources effectively. Experiment results on two real CITE-seq datasets demonstrate that our DEMOC model not only outperforms state-of-the-art single-omic clustering methods, but also achieves better and more stable performance than existing multi-omics clustering methods. We also apply our model on three scRNA-seq datasets to assess the performance of our model in rare cell-type identification, novel cell-subtype detection and cellular heterogeneity analysis. Experiment results illustrate the effectiveness of our model in discovering the underlying patterns of data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=501&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1f31003f4a9cbb37ed43eecd1ac44de7a0e9384f\",\n",
      "        \"title\": \"Probabilistic tensor decomposition extracts better latent embeddings from single-cell multiomic data\",\n",
      "        \"abstract\": \"Single-cell sequencing technology enables the simultaneous capture of multiomic data from multiple cells. The captured data can be represented by tensors, i.e., the higher-rank matrices. However, the proposed analysis tools often take the data as a collection of two-order matrices, renouncing the correspondences among the features. Consequently, we propose a probabilistic tensor decomposition framework, SCOIT, to extract embeddings from single-cell multiomic data. To deal with sparse, noisy, and heterogeneous single-cell data, we incorporate various distributions in SCOIT, including Gaussian, Poisson, and negative binomial distributions. Our framework can decompose a multiomic tensor into a cell embedding matrix, a gene embedding matrix, and an omic embedding matrix, allowing for various downstream analyses. We applied SCOIT to seven single-cell multiomic datasets from different sequencing protocols. With cell embeddings, SCOIT achieves superior performance for cell clustering compared to seven state-of-the-art tools under various metrics, demonstrating its ability to dissect cellular heterogeneity. With the gene embeddings, SCOIT enables cross-omics gene expression analysis and integrative gene regulatory network study. Furthermore, the embeddings allow cross-omics imputation simultaneously, outperforming conventional imputation methods with the Pearson correlation coefficient increased by 0.03-0.28.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=502&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bc10e17dc00673b76627070c03911d45e457d3b9\",\n",
      "        \"title\": \"SEC-seq: association of molecular signatures with antibody secretion in thousands of single human plasma cells\",\n",
      "        \"abstract\": \"Protein secretion drives many functions in vivo; however, methods to link secretions with surface markers and transcriptomes have been lacking. By accumulating secretions close to secreting cells held within cavity-containing hydrogel nanovials, we demonstrate workflows to analyze the amount of IgG secreted from single human antibody-secreting cells and link this information to surface marker expression and transcriptional profiles from the same cells. Measurements using flow cytometry and imaging flow cytometry corroborated an association between levels of IgG secretion and CD138 expression. Using oligonucleotide-labeled antibodies and droplet-based sequencing, we show that pathways encoding protein localization to the endoplasmic reticulum, NADH complex assembly, and mitochondrial respiration were most associated with high IgG secretion. Altogether, this method links secretion information to cell surface and single-cell sequencing information (SEC-seq) and enables exploration of links between genome and secretory function, laying the foundation for numerous discoveries in immunology, stem cell biology, and beyond.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=503&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c17e8b68659e8d27f1afee181826faed9f4199ee\",\n",
      "        \"title\": \"The development and function of CD11c+ atypical B cells - insights from single cell analysis\",\n",
      "        \"abstract\": \"CD11c+ T-bet+ atypical B cells (ABCs) have been identified in the context of vaccination, acute and chronic infections and autoimmune disease. However, the origins and functions of ABCs remain elusive. A major obstacle in the study of ABCs, and human MBCs more generally, has been the use of different phenotypic markers in different contexts to identify what appear to be phenotypically similar cells. Advances in single-cell RNA sequencing (scRNA-seq) technology have allowed researchers to accurately identify ABCs in different immune contexts such as diseases and tissues. Notably, recent studies utilizing single cell techniques have demonstrated ABCs are a highly conserved memory B cell lineage. This analysis has also revealed that ABCs are more abundant in ostensibly healthy donors than previously thought. Nonetheless, the normal function of these cells remains elusive. In this review, we will focus on scRNA-seq studies to discuss recent advances in our understanding about the development and functions of ABCs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=504&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8e032668167d1125cc216628fbd31113905083fe\",\n",
      "        \"title\": \"Clustering CITE-seq data with a canonical correlation-based deep learning method\",\n",
      "        \"abstract\": \"Single-cell multiomics sequencing techniques have rapidly developed in the past few years. Among these techniques, single-cell cellular indexing of transcriptomes and epitopes (CITE-seq) allows simultaneous quantification of gene expression and surface proteins. Clustering CITE-seq data have the great potential of providing us with a more comprehensive and in-depth view of cell states and interactions. However, CITE-seq data inherit the properties of scRNA-seq data, being noisy, large-dimensional, and highly sparse. Moreover, representations of RNA and surface protein are sometimes with low correlation and contribute divergently to the clustering object. To overcome these obstacles and find a combined representation well suited for clustering, we proposed scCTClust for multiomics data, especially CITE-seq data, and clustering analysis. Two omics-specific neural networks are introduced to extract cluster information from omics data. A deep canonical correlation method is adopted to find the maximumly correlated representations of two omics. A novel decentralized clustering method is utilized over the linear combination of latent representations of two omics. The fusion weights which can account for contributions of omics to clustering are adaptively updated during training. Extensive experiments over both simulated and real CITE-seq data sets demonstrated the power of scCTClust. We also applied scCTClust on transcriptome\\u2013epigenome data to illustrate its potential for generalizing.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=505&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e658def7e4cba3250c6c7a5004800b82fd1b60cc\",\n",
      "        \"title\": \"Characterizing control of memory CD8 T cell differentiation by BTB-ZF transcription factor Zbtb20\",\n",
      "        \"abstract\": \"Members of the BTB-ZF transcription factor family regulate the immune system. Our lab identified that family member Zbtb20 contributes to the differentiation, recall responses and metabolism of CD8 T cells. Here, we report a characterization of the transcriptional and epigenetic signatures controlled by Zbtb20 at single-cell resolution during the effector and memory phases of the CD8 T cell response. Without Zbtb20, transcriptional programs associated with memory CD8 T cell formation were upregulated throughout the CD8 T response. A signature of open chromatin was associated with genes controlling T cell activation, consistent with the known impact on differentiation. Additionally, memory CD8 T cells lacking Zbtb20 were characterized by open chromatin regions with overrepresentation of AP-1 transcription factor motifs and elevated RNA- and protein-level expression of the corresponding AP-1 components. Finally, we describe motifs and genomic annotations from the DNA targets of Zbtb20 in CD8 T cells identified by CUT&RUN. Together, these data establish the transcriptional and epigenetic networks contributing to the control of CD8 T cell responses by Zbtb20.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=506&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d001b32ceb7482996a522dab21dbba56c5f1cb8d\",\n",
      "        \"title\": \"Simultaneous transcriptome and proteome profiling in a single mouse oocyte with a deep single-cell multi-omics approach\",\n",
      "        \"abstract\": \"Nowadays, although single-cell multi-omics technologies are undergoing rapid development, simultaneous transcriptome and proteome analysis of a single-cell individual still faces great challenges. Here, we developed a single-cell simultaneous transcriptome and proteome (scSTAP) analysis platform based on microfluidics, high-throughput sequencing and mass spectrometry technology, to achieve deep and joint quantitative analysis of transcriptome and proteome at the single-cell level for the first time. This platform was applied to analyze single mouse oocytes at different meiotic maturation stages, reaching an average quantification depth of 19948 genes and 2663 protein groups in single mouse oocytes. This reliable quantitative two-omics dataset of single cells provided an important resource for understanding the relationship between the transcriptome and the proteome in cells. Based on the correlation analysis of RNAs and proteins in the same single cell, we demonstrated the expressive heterogeneity of transcriptome and proteome during the cellular biological process. Specially, we analyzed the meiosis regulatory network during oocyte maturation with an unprecedented depth at the single-cell level, and identified 30 transcript-protein pairs as specific oocyte maturational signatures, providing crucial insights into the regulatory features of transcription and translation during oocyte meiotic maturation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=507&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5c6ffba0879d36c0fc1483bcded55b6c4d65ffc9\",\n",
      "        \"title\": \"High-dimensional investigation of the cerebrospinal fluid to explore and monitor CNS immune responses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=508&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fbab5b91e4790a749c87806032ee74adcded6510\",\n",
      "        \"title\": \"Characterizing the role of the immune microenvironment in multiple myeloma progression at a single-cell level\",\n",
      "        \"abstract\": \"Visual Abstract Image 1\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=509&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"61904099b66f0ff34c357f72b2b0b756e1dcd7d9\",\n",
      "        \"title\": \"Onepot-Seq: capturing single-cell transcriptomes simultaneously in a continuous medium via transient localization of mRNA\",\n",
      "        \"abstract\": \"Abstract The development of single-cell RNA-seq has broadened the spectrum for biological research by providing a high-resolution analysis of cellular heterogeneity. However, the requirement for sophisticated devices for the compartmentalization of cells has limited its widespread applicability. Here, we develop Onepot-Seq, a device-free method, that harnesses the transient localization of mRNA after lysis to capture single-cell transcriptomes simultaneously in a continuous fluid medium. In mixed-species experiments, we obtained high-quality single-cell profiles. Further, cell type-specific poly(A)-conjugated antibodies allow Onepot-Seq to effectively capture target cells in complex populations. Chemical perturbations to cells can be profiled by Onepot-Seq at single-cell resolution. Onepot-Seq should allow routine transcriptional profiling at single-cell resolution, accelerating clinical and scientific discoveries in many fields of science.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=510&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eea774a75bc375d0ba63547f76ac4020fbc62603\",\n",
      "        \"title\": \"Dynamics of monocyte-derived macrophage diversity in experimental myocardial infarction\",\n",
      "        \"abstract\": \"Abstract Aims Macrophages have a critical and dual role in post-ischaemic cardiac repair, as they can foster both tissue healing and damage. Multiple subsets of tissue resident and monocyte-derived macrophages coexist in the infarcted heart, but their precise identity, temporal dynamics, and the mechanisms regulating their acquisition of discrete states are not fully understood. To address this, we used multi-modal single-cell immune profiling, combined with targeted cell depletion and macrophage fate mapping, to precisely map monocyte/macrophage transitions after experimental myocardial infarction. Methods and results We performed single-cell transcriptomic and cell-surface marker profiling of circulating and cardiac immune cells in mice challenged with acute myocardial infarction, and integrated single-cell transcriptomes obtained before and at 1, 3, 5, 7, and 11 days after infarction. Using complementary strategies of CCR2+ monocyte depletion and fate mapping of tissue resident macrophages, we determined the origin of cardiac macrophage populations. The macrophage landscape of the infarcted heart was dominated by monocyte-derived cells comprising two pro-inflammatory populations defined as Isg15hi and MHCII+Il1b+, alongside non-inflammatory Trem2hi cells. Trem2hi macrophages were observed in the ischaemic area, but not in the remote viable myocardium, and comprised two subpopulations sequentially populating the heart defined as Trem2hiSpp1hi monocyte-to-macrophage intermediates, and fully differentiated Trem2hiGdf15hi macrophages. Cardiac Trem2hi macrophages showed similarities to \\u2018lipid-associated macrophages\\u2019 found in mouse models of metabolic diseases and were observed in the human heart, indicating conserved features of this macrophage state across diseases and species. Ischaemic injury induced a shift of circulating Ly6Chi monocytes towards a Chil3hi state with granulocyte-like features, but the acquisition of the Trem2hi macrophage signature occurred in the ischaemic tissue. In vitro, macrophages acquired features of the Trem2hi signature following apoptotic-cell efferocytosis. Conclusion Our work provides a comprehensive map of monocyte/macrophage transitions in the ischaemic heart, constituting a valuable resource for further investigating how these cells may be harnessed and modulated to promote post-ischaemic heart repair.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=511&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a6b5f43766cb400c5bd804ddc212e4269ba71dd\",\n",
      "        \"title\": \"SMaSH: a scalable, general marker gene identification framework for single-cell RNA-sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=512&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"639d581e0f26bddb99e3c5de4ea131bbf83da1e4\",\n",
      "        \"title\": \"Scarf enables a highly memory-efficient analysis of large-scale single-cell genomics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=513&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d3db06ad77fb0bf21699688f72ce24785a36b918\",\n",
      "        \"title\": \"Single-cell RNA and protein profiling of immune cells from the mouse brain and its border tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=514&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c27f63db28b329b0cdb59c064e4b06dc5cec9dda\",\n",
      "        \"title\": \"Single-cell spatial explorer: easy exploration of spatial and multimodal transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=515&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1d5963f79a0eaf6b60bdd2c058621582721523c0\",\n",
      "        \"title\": \"Sample-multiplexing approaches for single-cell sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=516&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d500cdbef62eb2b135b09289f81a04f2c484e520\",\n",
      "        \"title\": \"Practical Considerations for Single\\u2010Cell Genomics\",\n",
      "        \"abstract\": \"The single\\u2010cell revolution in the field of genomics is in full bloom, with clever new molecular biology tricks appearing regularly that allow researchers to explore new modalities or scale up their projects to millions of cells and beyond. Techniques abound to measure RNA expression, DNA alterations, protein abundance, chromatin accessibility, and more, all with single\\u2010cell resolution and often in combination. Despite such a rapidly changing technology landscape, there are several fundamental principles that are applicable to the majority of experimental workflows to help users avoid pitfalls and exploit the advantages of the chosen platform. In this overview article, we describe a variety of popular single\\u2010cell genomics technologies and address some common questions pertaining to study design, sample preparation, quality control, and sequencing strategy. As the majority of relevant publications currently revolve around single\\u2010cell RNA\\u2010seq, we will prioritize this genomics modality in our discussion. \\u00a9 2022 Wiley Periodicals LLC.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=517&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58c987a0ac825a1743f47c8e320c41da151c528b\",\n",
      "        \"title\": \"Untangling the web of intratumour heterogeneity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=518&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"472e68f8388795316b1043901e0d307426cb363f\",\n",
      "        \"title\": \"Unbiased discovery of autoantibodies associated with severe COVID-19 via genome-scale self-assembled DNA-barcoded protein libraries\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=519&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ffe2a07830861ffbc3ab35674afff40c42cb548c\",\n",
      "        \"title\": \"See-N-Seq: RNA sequencing of target single cells identified by microscopy via micropatterning of hydrogel porosity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=520&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5f8106cbb25cfde2d510e80d843a85317a2a5189\",\n",
      "        \"title\": \"Future of Digital Assays to Resolve Clinical Heterogeneity of Single Extracellular Vesicles.\",\n",
      "        \"abstract\": \"Extracellular vesicles (EVs) are complex lipid membrane vehicles with variable expressions of molecular cargo, composed of diverse subpopulations that participate in the intercellular signaling of biological responses in disease. EV-based liquid biopsies demonstrate invaluable clinical potential for overhauling current practices of disease management. Yet, EV heterogeneity is a major needle-in-a-haystack challenge to translate their use into clinical practice. In this review, existing digital assays will be discussed to analyze EVs at a single vesicle resolution, and future opportunities to optimize the throughput, multiplexing, and sensitivity of current digital EV assays will be highlighted. Furthermore, this review will outline the challenges and opportunities that impact the clinical translation of single EV technologies for disease diagnostics and treatment monitoring.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=521&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2751732385ab4f40318fe55ae73957f82bc3c57b\",\n",
      "        \"title\": \"Community-driven ELIXIR activities in single-cell omics\",\n",
      "        \"abstract\": \"Single-cell omics (SCO) has revolutionized the way and the level of resolution by which life science research is conducted, not only impacting our understanding of fundamental cell biology but also providing novel solutions in cutting-edge medical research. The rapid development of single-cell technologies has been accompanied by the active development of data analysis methods, resulting in a plethora of new analysis tools and strategies every year. Such a rapid development of SCO methods and tools poses several challenges in standardization, benchmarking, computational resources and training. These challenges are in line with the activities of ELIXIR, the European coordinated infrastructure for life science data. Here, we describe the current landscape of and the main challenges in SCO data, and propose the creation of the ELIXIR SCO Community, to coordinate the efforts in order to best serve SCO researchers in Europe and beyond. The Community will build on top of national experiences and pave the way towards integrated long-term solutions for SCO research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=522&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b2c0dccbd9b4a4b150001222c35e34585ea6ac3\",\n",
      "        \"title\": \"A developmental atlas of the mouse brain by single-cell mass cytometry\",\n",
      "        \"abstract\": \"Development of the mammalian brain requires precisely controlled differentiation of neurons, glia, and nonneural cells. To investigate protein-level changes in these diverse cell types and their progenitors, we performed single-cell mass cytometry on whole brain (E11.5/E12.5) and microdissected telencephalon, diencephalon, mesencephalon, and rhombencephalon (E13.5\\u2013P4) collected at daily timepoints from C57/BL6 mice. Measuring 24,290,787 cells from 112 sample replicates with a 40-antibody panel, we quantified 85 molecularly distinct cell populations across embryonic and postnatal development, including microglia putatively phagocytosing neurites, neural cells, and myelin. Differentiation trajectory analysis also identified two separate pathways for producing oligodendrocyte precursor cells. Comparison with previous studies revealed considerable discrepancies between protein and mRNA abundances in the developing brain, demonstrating the value of protein-level measurements for identifying functional cell states. Overall, our findings demonstrate the utility of mass cytometry as a high-throughput, scalable platform for single-cell profiling of brain tissue.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=523&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cf53eb2aedc06ebe08509ffe95a0e5078ec6829a\",\n",
      "        \"title\": \"Hierarchical cell-type identifier accurately distinguishes immune-cell subtypes enabling precise profiling of tissue microenvironment with single-cell RNA-sequencing\",\n",
      "        \"abstract\": \"Single-cell RNA-seq enabled in-depth study on tissue micro-environment and immune profiling, where a crucial step is to annotate cell identity. Immune cells play key roles in many diseases while their activities are hard to track due to their diverse and highly variable nature. Existing cell-type identifiers had limited performance for this purpose. We present HiCAT, a hierarchical, marker-based cell-type identifier utilizing gene set analysis for statistical scoring for given markers. It features successive identification of major-type, minor-type and subsets utilizing subset markers structured in a three-level taxonomy tree. Comparison with manual annotation and pairwise match test showed HiCAT performs better than others in major and minor-type identification. For subset identification, since no dataset provides canonical subset annotation, we qualitatively evaluated marker expression profile, where HiCAT was the best in the sense that, with its annotation, marker genes were expressed mostly in the designated subset while rarely expressed in non-designated subsets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=524&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eb0674514e2c12cac8cacee1999e8d153b683ce5\",\n",
      "        \"title\": \"MarkerMap: nonlinear marker selection for single-cell studies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=525&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c46f974edac04c8716eba70a513b4c823b7ce4cd\",\n",
      "        \"title\": \"Magnetic Microtweezers: A Tool for High\\u2010Throughput Bioseparation in Sub\\u2010Nanoliter Droplets\",\n",
      "        \"abstract\": \"Droplet microfluidics has revolutionized the field of single\\u2010cell analysis. Production of micro\\u2010droplets at high throughput allows for rapid isolation of single cells within micro\\u2010compartments, which are then subjected to different analytical processes. However, certain operations, such as physical separation from droplets, remain difficult to implement at high throughput and single\\u2010cell level but would be highly valuable to currently expanding multiomics techniques, where several biomolecular modalities are involved. This work presents a method based on microfabricated NiFe structures, the magnetic microtweezers, to trap and extract magnetic particles from a continuous stream of sub\\u2010nanoliter droplets, and enable physical separation from single cell\\u2010based droplets. Using a physical model, simulations, and experiments, a comprehensive description of the complex particle extraction process is provided. After optimization, the magnetic microtweezers provided unprecedented particle extraction performance, allowing extraction of high loads (10\\u201020 ng) of magnetic particles from 500 pL droplets, with a capture rate close to 100% at 20 Hz. To evaluate the applicability to single\\u2010cell analysis, mRNA extraction is performed. It demonstrated around 72% specific recovery of mRNA from droplets containing purified nucleic acids, and 43% from single cells. Overall, this approach enables efficient physical separation that is compatible with existing high\\u2010throughput droplet\\u2010based single\\u2010cell workflows.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=526&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"170360fbb2dacf57ed3de3996d1d3570c0087669\",\n",
      "        \"title\": \"Defining Patient-Level Molecular Heterogeneity in Psoriasis Vulgaris Based on Single-Cell Transcriptomics\",\n",
      "        \"abstract\": \"Identifying genetic variation underlying human diseases establishes targets for therapeutic development and helps tailor treatments to individual patients. Large-scale transcriptomic profiling has extended the study of such molecular heterogeneity between patients to somatic tissues. However, the lower resolution of bulk RNA profiling, especially in a complex, composite tissue such as the skin, has limited its success. Here we demonstrate approaches to interrogate patient-level molecular variance in a chronic skin inflammatory disease, psoriasis vulgaris, leveraging single-cell RNA-sequencing of CD45+ cells isolated from active lesions. Highly psoriasis-specific transcriptional abnormalities display greater than average inter-individual variance, nominating them as potential sources of clinical heterogeneity. We find that one of these chemokines, CXCL13, demonstrates significant correlation with severity of lesions within our patient series. Our analyses also establish that genes elevated in psoriatic skin-resident memory T cells are enriched for programs orchestrating chromatin and CDC42-dependent cytoskeleton remodeling, specific components of which are distinctly correlated with and against Th17 identity on a single-cell level. Collectively, these analyses describe systematic means to dissect cell type- and patient-level differences in cutaneous psoriasis using high-resolution transcriptional profiles of human inflammatory disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=527&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c2c99ed3804f669857a4272587b562a0ce6cabc\",\n",
      "        \"title\": \"Techniques for Profiling the Cellular Immune Response and Their Implications for Interventional Oncology\",\n",
      "        \"abstract\": \"Simple Summary The prognosis of patients with different types of cancer is usually predicted with the help of the TNM staging guidelines; however, in recent years, there has been increased interest in using the primary tumor\\u2019s immune environment and characteristics to predict patient prognosis. Interventional radiologists play an important role in the evaluation of this immune microenvironment by performing image-guided biopsies that are paramount for obtaining appropriate tissue samples. The goal of this review is to describe the different techniques that are used for immune microenvironment evaluation, analysis steps for the data collected, and its relevance to patient care, including implications for patients undergoing interventional oncology procedures. Abstract In recent years there has been increased interest in using the immune contexture of the primary tumors to predict the patient\\u2019s prognosis. The tumor microenvironment of patients with cancers consists of different types of lymphocytes, tumor-infiltrating leukocytes, dendritic cells, and others. Different technologies can be used for the evaluation of the tumor microenvironment, all of which require a tissue or cell sample. Image-guided tissue sampling is a cornerstone in the diagnosis, stratification, and longitudinal evaluation of therapeutic efficacy for cancer patients receiving immunotherapies. Therefore, interventional radiologists (IRs) play an essential role in the evaluation of patients treated with systemically administered immunotherapies. This review provides a detailed description of different technologies used for immune assessment and analysis of the data collected from the use of these technologies. The detailed approach provided herein is intended to provide the reader with the knowledge necessary to not only interpret studies containing such data but also design and apply these tools for clinical practice and future research studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=528&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1f327cda1b5c6f3b0792b74e27dd5a5548bcd20d\",\n",
      "        \"title\": \"High titre neutralizing antibodies in response to SARS\\u2013CoV\\u20132 infection require RBD\\u2013specific CD4 T cells that include proliferative memory cells\",\n",
      "        \"abstract\": \"Background Long-term immunity to SARS-CoV-2 infection, including neutralizing antibodies and T cell-mediated immunity, is required in a very large majority of the population in order to reduce ongoing disease burden. Methods We have investigated the association between memory CD4 and CD8 T cells and levels of neutralizing antibodies in convalescent COVID-19 subjects. Findings Higher titres of convalescent neutralizing antibodies were associated with significantly higher levels of RBD-specific CD4 T cells, including specific memory cells that proliferated vigorously in vitro. Conversely, up to half of convalescent individuals had low neutralizing antibody titres together with a lack of receptor binding domain (RBD)-specific memory CD4 T cells. These low antibody subjects had other, non-RBD, spike-specific CD4 T cells, but with more of an inhibitory Foxp3+ and CTLA-4+ cell phenotype, in contrast to the effector T-bet+, cytotoxic granzymes+ and perforin+ cells seen in RBD-specific memory CD4 T cells from high antibody subjects. Single cell transcriptomics of antigen-specific CD4+ T cells from high antibody subjects similarly revealed heterogenous RBD-specific CD4+ T cells that comprised central memory, transitional memory and Tregs, as well as cytotoxic clusters containing diverse TCR repertoires, in individuals with high antibody levels. However, vaccination of low antibody convalescent individuals led to a slight but significant improvement in RBD-specific memory CD4 T cells and increased neutralizing antibody titres. Interpretation Our results suggest that targeting CD4 T cell epitopes proximal to and within the RBD-region should be prioritized in booster vaccines.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=529&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e32a94b8e26171991b810d1bdc1687930d44a098\",\n",
      "        \"title\": \"Human Variation in DNA Repair, Immune Function, and Cancer Risk\",\n",
      "        \"abstract\": \"DNA damage constantly threatens genome integrity, and DNA repair deficiency is associated with increased cancer risk. An intuitive and widely accepted explanation for this relationship is that unrepaired DNA damage leads to carcinogenesis due to the accumulation of mutations in somatic cells. But DNA repair also plays key roles in the function of immune cells, and immunodeficiency is an important risk factor for many cancers. Thus, it is possible that emerging links between inter-individual variation in DNA repair capacity and cancer risk are driven, at least in part, by variation in immune function, but this idea is underexplored. In this review we present an overview of the current understanding of the links between cancer risk and both inter-individual variation in DNA repair capacity and inter-individual variation in immune function. We discuss factors that play a role in both types of variability, including age, lifestyle, and environmental exposures. In conclusion, we propose a research paradigm that incorporates functional studies of both genome integrity and the immune system to predict cancer risk and lay the groundwork for personalized prevention.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=530&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fd6980b9525feacc9184f1bcbaa8cf7704c413fb\",\n",
      "        \"title\": \"Single-cell analysis of endometriosis reveals a coordinated transcriptional programme driving immunotolerance and angiogenesis across eutopic and ectopic tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=531&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f4efd852bc17d7a3b04f57aec933abe33d12cd5\",\n",
      "        \"title\": \"The emerging landscape of spatial profiling technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=532&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"020b7f533e573c24e912250d108a6d110681ff9b\",\n",
      "        \"title\": \"Recent Advances in Molecular Diagnosis of Pulmonary Fibrosis for Precision Medicine.\",\n",
      "        \"abstract\": \"Pulmonary fibrosis is a serious, progressive lung disease characterized by scarring and stiffening lung tissues, affecting the respiratory system and leading to organ failure. It is a complex disease consisting of alveolar damage, chronic inflammation, and a varying degree of lung fibrosis. Significant challenges with pulmonary fibrosis include the lack of effective means to diagnose the disease at early stages, identify patients at higher risks of progress, and assess disease progression and treatment response. Precision medicine powered by accurate molecular profiling and phenotyping could significantly improve our understanding of the disease's heterogeneity, potential biomarkers for diagnosis and prognosis, and molecular targets for treatment development. This Review discusses various translational model systems, including organoids and lung-on-a-chip systems, biomarkers in single cells and extracellular vesicles, and functional pharmacodynamic markers. We also highlight emerging sensing technologies for molecular characterization of pulmonary fibrosis and biomarker detection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=533&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2499f238cb58ac117fa7af7b8166f20b47901b55\",\n",
      "        \"title\": \"Integration of single-cell multi-omics data by regression analysis on unpaired observations\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=534&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bc7f2300d011e8d122de5e1d671b326673325e4a\",\n",
      "        \"title\": \"Leveraging Systems Immunology to Optimize Diagnosis and Treatment of Inborn Errors of Immunity\",\n",
      "        \"abstract\": \"Inborn errors of immunity (IEI) are monogenic disorders that can cause diverse symptoms, including recurrent infections, autoimmunity and malignancy. While many factors have contributed, the increased availability of next-generation sequencing has been central in the remarkable increase in identification of novel monogenic IEI over the past years. Throughout this phase of disease discovery, it has also become evident that a given gene variant does not always yield a consistent phenotype, while variants in seemingly disparate genes can lead to similar clinical presentations. Thus, it is increasingly clear that the clinical phenotype of an IEI patient is not defined by genetics alone, but is also impacted by a myriad of factors. Accordingly, we need methods to amplify our current diagnostic algorithms to better understand mechanisms underlying the variability in our patients and to optimize treatment. In this review, we will explore how systems immunology can contribute to optimizing both diagnosis and treatment of IEI patients by focusing on identifying and quantifying key dysregulated pathways. To improve mechanistic understanding in IEI we must deeply evaluate our rare IEI patients using multimodal strategies, allowing both the quantification of altered immune cell subsets and their functional evaluation. By studying representative controls and patients, we can identify causative pathways underlying immune cell dysfunction and move towards functional diagnosis. Attaining this deeper understanding of IEI will require a stepwise strategy. First, we need to broadly apply these methods to IEI patients to identify patterns of dysfunction. Next, using multimodal data analysis, we can identify key dysregulated pathways. Then, we must develop a core group of simple, effective functional tests that target those pathways to increase efficiency of initial diagnostic investigations, provide evidence for therapeutic selection and contribute to the mechanistic evaluation of genetic results. This core group of simple, effective functional tests, targeting key pathways, can then be equitably provided to our rare patients. Systems biology is thus poised to reframe IEI diagnosis and therapy, fostering research today that will provide streamlined diagnosis and treatment choices for our rare and complex patients in the future, as well as providing a better understanding of basic immunology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=535&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1a347150b83c2c618d6fa123074d8ac20a1f2f8f\",\n",
      "        \"title\": \"Protein Adsorption Loss\\u2500The Bottleneck of Single-Cell Proteomics.\",\n",
      "        \"abstract\": \"Single-cell proteomics is a promising field to provide direct yet comprehensive molecular insights into cellular functions without averaging effects. Here, we address a grand technical challenge impeding the maturation of single-cell proteomics\\u2500protein adsorption loss (PAL). Even though widely known, there is currently no quantitation on how profoundly and selectively PAL has affected single-cell proteomics. Therefore, the mitigations to this challenge have been generic, and their efficacy was only evaluated by the size of the resolved proteome with no specificity on individual proteins. We use the existing knowledge of PAL, protein expression, and the typical surface area used in single-cell proteomics to discuss the severity of protein loss. We also summarize the current solutions to this challenge and briefly review the available methods to characterize the physical and chemical properties of protein surface adsorption. By citing successful strategies in single-cell genomics for measurement errors in individual transcripts, we pinpoint the urgency to benchmark PAL at the proteome scale with individual protein resolution. Finally, orthogonal single-cell proteomic techniques that have the potential to cross validate PAL are proposed. We hope these efforts can promote the fruition of single-cell proteomics in the near future.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=536&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dde7790a6cbacc972db7bdb239d7a7d0c278d009\",\n",
      "        \"title\": \"Characterizing cis-regulatory elements using single-cell epigenomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=537&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"11847883bbdea161963289fe4a636d59575ba793\",\n",
      "        \"title\": \"Diversity in the bone marrow niche: Classic and novel strategies to uncover niche composition\",\n",
      "        \"abstract\": \"Our view on the role and composition of the bone marrow (BM) has dramatically changed over time from a simple nutrient for the bone to a highly complex multicellular tissue that sustains haematopoiesis. Among these cells, multipotent haematopoietic stem cells (HSCs), which are predominantly quiescent, possess unique self\\u2010renewal capacity and multilineage differentiation potential and replenish all blood lineages to maintain lifelong haematopoiesis. Adult HSCs reside in specialised BM niches, which support their functions. Much effort has been put into deciphering HSC niches due to their potential clinical relevance. Multiple cell types have been implicated as HSC\\u2010niche components including sinusoidal endothelium, perivascular stromal cells, macrophages, megakaryocytes, osteoblasts and sympathetic nerves. In this review we provide a historical perspective on how technical advances, from genetic mouse models to imaging and high\\u2010throughput sequencing techniques, are unveiling the plethora of molecular cues and cellular components that shape the niche and regulate HSC functions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=538&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6493975e788f0a5c37f5b630c25f38eeb49d1f5e\",\n",
      "        \"title\": \"Indirect CRISPR screening with photoconversion revealed key factors of drug resistance with cell\\u2013cell interactions\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=539&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1596cea2821e564dfdf1bed6adfac375b55a5fd0\",\n",
      "        \"title\": \"Microglia Heterogeneity: A Single-cell Concerto\",\n",
      "        \"abstract\": \"Microglia are tissue-resident macrophages of the central nervous system (CNS) that play crucial roles in development, homeostasis, and response to perturbation. Microglia react to the surrounding environment in a context-dependent manner. However, research on microglial heterogeneity is limited, given the lack of high-resolution and high-sensitivity methods. Recent studies have demonstrated the heterogeneity of microglia on a spatial-temporal scale, benefiting from the advancement of single-cell technologies. Here, we review the current knowledge about microglial diversity during physiological and pathological conditions in humans and mice.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=540&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e42c1132c23e638e534a83b2a130fbf953af3dd7\",\n",
      "        \"title\": \"HIV viral transcription and immune perturbations in the CNS of people with HIV despite ART\",\n",
      "        \"abstract\": \"People with HIV (PWH) on antiretroviral therapy (ART) experience elevated rates of neurological impairment, despite controlling for demographic factors and comorbidities, suggesting viral or neuroimmune etiologies for these deficits. Here, we apply multimodal and cross-compartmental single-cell analyses of paired cerebrospinal fluid (CSF) and peripheral blood in PWH and uninfected controls. We demonstrate that a subset of central memory CD4+ T cells in the CSF produced HIV-1 RNA, despite apparent systemic viral suppression, and that HIV-1\\u2013infected cells were more frequently found in the CSF than in the blood. Using cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq), we show that the cell surface marker CD204 is a reliable marker for rare microglia-like cells in the CSF, which have been implicated in HIV neuropathogenesis, but which we did not find to contain HIV transcripts. Through a feature selection method for supervised deep learning of single-cell transcriptomes, we find that abnormal CD8+ T cell activation, rather than CD4+ T cell abnormalities, predominated in the CSF of PWH compared with controls. Overall, these findings suggest ongoing CNS viral persistence and compartmentalized CNS neuroimmune effects of HIV infection during ART and demonstrate the power of single-cell studies of CSF to better understand the CNS reservoir during HIV infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=541&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"867e5a96834177c94866141312d2df893ed3847c\",\n",
      "        \"title\": \"Constraint-Based Reconstruction and Analyses of Metabolic Models: Open-Source Python Tools and Applications to Cancer\",\n",
      "        \"abstract\": \"The influence of metabolism on signaling, epigenetic markers, and transcription is highly complex yet important for understanding cancer physiology. Despite the development of high-resolution multi-omics technologies, it is difficult to infer metabolic activity from these indirect measurements. Fortunately, genome-scale metabolic models and constraint-based modeling provide a systems biology framework to investigate the metabolic states and define the genotype-phenotype associations by integrations of multi-omics data. Constraint-Based Reconstruction and Analysis (COBRA) methods are used to build and simulate metabolic networks using mathematical representations of biochemical reactions, gene-protein reaction associations, and physiological and biochemical constraints. These methods have led to advancements in metabolic reconstruction, network analysis, perturbation studies as well as prediction of metabolic state. Most computational tools for performing these analyses are written for MATLAB, a proprietary software. In order to increase accessibility and handle more complex datasets and models, community efforts have started to develop similar open-source tools in Python. To date there is a comprehensive set of tools in Python to perform various flux analyses and visualizations; however, there are still missing algorithms in some key areas. This review summarizes the availability of Python software for several components of COBRA methods and their applications in cancer metabolism. These tools are evolving rapidly and should offer a readily accessible, versatile way to model the intricacies of cancer metabolism for identifying cancer-specific metabolic features that constitute potential drug targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=542&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4d27a9f9fae7e378eba56194851a89edea26bd50\",\n",
      "        \"title\": \"Applications of single\\u2010cell multi\\u2010omics sequencing in deep understanding of brain diseases\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=543&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fbf89d4b0d5b609f07e0899eabca4be755374e22\",\n",
      "        \"title\": \"Multi-modal profiling of peripheral blood cells across the human lifespan reveals distinct immune cell signatures of aging and longevity\",\n",
      "        \"abstract\": \"Age-related changes in immune cell composition and functionality are associated with multimorbidity and mortality. However, many centenarians delay the onset of aging-related disease suggesting the presence of elite immunity that remains highly functional at extreme old age. To identify immune-specific patterns of aging and extreme human longevity, we analyzed novel single cell profiles from the peripheral blood mononuclear cells (PBMCs) of 7 centenarians (mean age 106) and publicly available single cell RNA-sequencing (scRNA-seq) datasets that included an additional 7 centenarians as well as 52 people at younger ages (20-89 years). The analysis confirmed known shifts in the ratio of lymphocytes to myeloid cells, and noncytotoxic to cytotoxic cell distributions with aging, but also identified significant shifts from CD4+ T cell to B cell populations in centenarians suggesting a history of exposure to natural and environmental immunogens. Our transcriptional analysis identified cell type signatures specific to exceptional longevity that included genes with age-related changes (e.g., increased expression of STK17A, a gene known to be involved in DNA damage response) as well as genes expressed uniquely in centenarians\\u2019 PBMCs (e.g., S100A4, part of the S100 protein family studied in age-related disease and connected to longevity and metabolic regulation). Collectively, these data suggest that centenarians harbor unique, highly functional immune systems that have successfully adapted to a history of insults allowing for the achievement of exceptional longevity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=544&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eba87a14bb55ca59bf327a67236324a82daa2dbe\",\n",
      "        \"title\": \"NKG7 Enhances CD8+ T Cell Synapse Efficiency to Limit Inflammation\",\n",
      "        \"abstract\": \"Cytotoxic lymphocytes are essential for anti-tumor immunity, and for effective responses to cancer immunotherapy. Natural killer cell granule protein 7 (NKG7) is expressed at high levels in cytotoxic lymphocytes infiltrating tumors from patients treated with immunotherapy, but until recently, the role of this protein in cytotoxic lymphocyte function was largely unknown. Unexpectedly, we found that highly CD8+ T cell-immunogenic murine colon carcinoma (MC38-OVA) tumors grew at an equal rate in Nkg7+/+ and Nkg7-/- littermate mice, suggesting NKG7 may not be necessary for effective CD8+ T cell anti-tumor activity. Mechanistically, we found that deletion of NKG7 reduces the ability of CD8+ T cells to degranulate and kill target cells in vitro. However, as a result of inefficient cytotoxic activity, NKG7 deficient T cells form a prolonged immune synapse with tumor cells, resulting in increased secretion of inflammatory cytokines, including tumor necrosis factor alpha (TNF). By deleting the TNF receptor, TNFR1, from MC38-OVA tumors, we demonstrate that this hyper-secretion of TNF compensates for reduced synapse-mediated cytotoxic activity against MC38-OVA tumors in vivo, via increased TNF-mediated tumor cell death. Taken together, our results demonstrate that NKG7 enhances CD8+ T cell immune synapse efficiency, which may serve as a mechanism to accelerate direct cytotoxicity and limit potentially harmful inflammatory responses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=545&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2156bb895c918be5527955f992c95102383f85aa\",\n",
      "        \"title\": \"Single-cell glycomics analysis by CyTOF-Lec reveals glycan features defining cells differentially susceptible to HIV\",\n",
      "        \"abstract\": \"High-parameter single-cell phenotyping has enabled in-depth classification and interrogation of immune cells, but to date has not allowed for glycan characterization. Here, we develop CyTOF-Lec as an approach to simultaneously characterize many protein and glycan features of human immune cells at the single-cell level. We implemented CyTOF-Lec to compare glycan features between different immune subsets from blood and multiple tissue compartments, and to characterize HIV-infected cell cultures. Using bioinformatics approaches to distinguish preferential infection of cellular subsets from viral-induced remodeling, we demonstrate that HIV upregulates the levels of cell-surface fucose and sialic acid in a cell-intrinsic manner, and that memory CD4+ T cells co-expressing high levels of fucose and sialic acid are highly susceptible to HIV infection. Sialic acid levels were found to distinguish memory CD4+ T cell subsets expressing different amounts of viral entry receptors, pro-survival factors, homing receptors, and activation markers, and to play a direct role in memory CD4+ T cells\\u2019 susceptibility to HIV infection. The ability of sialic acid to distinguish memory CD4+ T cells with different susceptibilities to HIV infection was experimentally validated through sorting experiments. Together, these results suggest that HIV remodels not only cellular proteins but also glycans, and that glycan expression can differentiate memory CD4+ T cells with vastly different susceptibility to HIV infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=546&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"277eb06e740f9ee1ffafba493532c1257b76bcbb\",\n",
      "        \"title\": \"The role of single-cell genomics in human genetics\",\n",
      "        \"abstract\": \"Single-cell sequencing is a powerful approach that can detect genetic alterations and their phenotypic consequences in the context of human development, with cellular resolution. Humans start out as single-cell zygotes and undergo fission and differentiation to develop into multicellular organisms. Before fertilisation and during development, the cellular genome acquires hundreds of mutations that propagate down the cell lineage. Whether germline or somatic in nature, some of these mutations may have significant genotypic impact and lead to diseased cellular phenotypes, either systemically or confined to a tissue. Single-cell sequencing enables the detection and monitoring of the genotype and the consequent molecular phenotypes at a cellular resolution. It offers powerful tools to compare the cellular lineage between \\u2018normal\\u2019 and \\u2018diseased\\u2019 conditions and to establish genotype-phenotype relationships. By preserving cellular heterogeneity, single-cell sequencing, unlike bulk-sequencing, allows the detection of even small, diseased subpopulations of cells within an otherwise normal tissue. Indeed, the characterisation of biopsies with cellular resolution can provide a mechanistic view of the disease. While single-cell approaches are currently used mainly in basic research, it can be expected that applications of these technologies in the clinic may aid the detection, diagnosis and eventually the treatment of rare genetic diseases as well as cancer. This review article provides an overview of the single-cell sequencing technologies in the context of human genetics, with an aim to empower clinicians to understand and interpret the single-cell sequencing data and analyses. We discuss the state-of-the-art experimental and analytical workflows and highlight current challenges/limitations. Notably, we focus on two prospective applications of the technology in human genetics, namely the annotation of the non-coding genome using single-cell functional genomics and the use of single-cell sequencing data for in silico variant prioritisation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=547&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b4138fcd96e0cee8480adf9357c007a867368a68\",\n",
      "        \"title\": \"Tools for mammalian glycoscience research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=548&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3d44673a8f915d07526b21d4e4e4fbd407662f94\",\n",
      "        \"title\": \"Decoding endoplasmic reticulum stress signals in cancer cells and antitumor immunity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=549&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"954c9533823b7cdfe429e7dd12e3eabe2787bdb1\",\n",
      "        \"title\": \"New Insights into Cerebral Vessel Disease Landscapes at Single-Cell Resolution: Pathogenetic and Therapeutic Perspectives\",\n",
      "        \"abstract\": \"Cerebrovascular diseases are a leading cause of death and disability globally. The development of new therapeutic targets for cerebrovascular diseases (e.g., ischemic, and hemorrhagic stroke, vascular dementia) is limited by a lack of knowledge of the cellular and molecular biology of health and disease conditions and the factors that cause injury to cerebrovascular structures. Here, we describe the role of advances in omics technology, particularly RNA sequencing, in studying high-dimensional, multifaceted profiles of thousands of individual blood and vessel cells at single-cell resolution. This analysis enables the dissection of the heterogeneity of diseased cerebral vessels and their atherosclerotic plaques, including the microenvironment, cell evolutionary trajectory, and immune response pathway. In animal models, RNA sequencing permits the tracking of individual cells (including immunological, endothelial, and vascular smooth muscle cells) that compose atherosclerotic plaques and their alteration under experimental settings such as phenotypic transition. We describe how single-cell RNA transcriptomics in humans allows mapping to the molecular and cellular levels of atherosclerotic plaques in cerebral arteries, tracking individual lymphocytes and macrophages, and how these data can aid in identifying novel immune mechanisms that could be exploited as therapeutic targets for cerebrovascular diseases. Single-cell multi-omics approaches will likely provide the unprecedented resolution and depth of data needed to generate clinically relevant cellular and molecular signatures for the precise treatment of cerebrovascular diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=550&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a4d39ceaff50d584555b61a94b4b28c9a5fb313c\",\n",
      "        \"title\": \"Transcriptional Regulation of the Hippo Pathway: Current Understanding and Insights from Single-Cell Technologies\",\n",
      "        \"abstract\": \"The Hippo pathway regulates tissue homeostasis in normal development and drives oncogenic processes. In this review, we extensively discuss how YAP/TAZ/TEAD cooperate with other master transcription factors and epigenetic cofactors to orchestrate a broad spectrum of transcriptional responses. Even though these responses are often context- and lineage-specific, we do not have a good understanding of how such precise and specific transcriptional control is achieved\\u2014whether they are driven by differences in TEAD paralogs, or recruitment of cofactors to tissue-specific enhancers. We believe that emerging single-cell technologies would enable a granular understanding of how the Hippo pathway influences cell fate and drives oncogenic processes, ultimately allowing us to design better pharmacological agents against TEADs and identify robust pharmacodynamics markers of Hippo pathway inhibition.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=551&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ea74be0513ac9590da492d5060059fdf31e75ace\",\n",
      "        \"title\": \"Nanoparticle single-cell multiomic readouts reveal that cell heterogeneity influences lipid nanoparticle-mediated messenger RNA delivery\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=552&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"44f8703f1e5ac450ec6238f722db1a19920d2ae0\",\n",
      "        \"title\": \"Single-cell multi-omics sequencing and its applications in studying the nervous system\",\n",
      "        \"abstract\": \"Single-cell sequencing has become one of the most powerful and popular techniques in dissecting molecular heterogeneity and modeling the cellular architecture of a biological system. During the past twenty years, the throughput of single-cell sequencing has increased from hundreds of cells to over tens of thousands of cells in parallel. Moreover, this technology has been developed from sequencing transcriptome to measure different omics such as DNA methylome, chromatin accessibility, and so on. Currently, multi-omics which can analyze different omics in the same cell is rapidly advancing. This work advances the study of many biosystems, including the nervous system. Here, we review current single-cell multi-omics sequencing techniques and describe how they improve our understanding of the nervous system. Finally, we discuss the open scientific questions in neural research that may be answered through further improvement of single-cell multi-omics sequencing technology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=553&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ce1b80cf01624956adbeccea2afed2f61cd5a4f9\",\n",
      "        \"title\": \"Single-cell sequencing: expansion, integration and translation.\",\n",
      "        \"abstract\": \"With the rapid advancement in sequencing technologies, the concept of omics has revolutionized our understanding of cellular behaviors. Conventional omics investigation approaches measure the averaged behaviors of multiple cells, which may easily hide signals represented by a small-cell cohort, urging for the development of techniques with enhanced resolution. Single-cell RNA sequencing, investigating cell transcriptomics at the resolution of a single cell, has been rapidly expanded to investigate other omics such as genomics, proteomics and metabolomics since its invention. The requirement for comprehensive understanding of complex cellular behavior has led to the integration of multi-omics and single-cell sequencing data with other layers of information such as spatial data and the CRISPR screening technique towards gained knowledge or innovative functionalities. The development of single-cell sequencing in both dimensions has rendered it a unique field that offers us a versatile toolbox to delineate complex diseases, including cancers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=554&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5271208dce12a0d7668d5a84a248b36a0270a504\",\n",
      "        \"title\": \"Myeloid reprogramming by JAK inhibition enhances checkpoint blockade therapy\",\n",
      "        \"abstract\": \"Unleashing anti-tumor T cell activity by checkpoint inhibition is effective in many cancer patients but clinical response rates remain limited. Myeloid derived suppressor cells erode antitumor lymphocyte numbers and function, and correlate with resistance to checkpoint inhibitors. By screening small molecule libraries, we identified JAK inhibitors\\u2019 ability to rescue T cell function. Despite its documented immune suppressive properties, the prototypical JAK inhibitor ruxolitinib enhanced the efficacy of immune checkpoint blockade in cancer. This effect correlated with loss of suppressive gene expression, and acquisition of immunostimulatory molecular markers and T cell stimulatory activity in myeloid cells. In preclinical models, ruxolitinib significantly improved the function and increased the total numbers of activated tumor-infiltrating NK and CD4 T cells compared to checkpoint blockade alone and the efficacy was conditional on granulocytic cells. In addition to myeloid reprogramming in the tumor, ruxolitinib blunts G-CSF signaling in the bone marrow to prevent expression of suppressive and chemotaxis genes in neutrophils. In a clinical trial of Hodgkin lymphoma patients resistant to checkpoint inhibitors, treatment with ruxolitinib significantly reduced neutrophil-to-lymphocyte ratios and levels of suppressive markers in myeloid cells but increased numbers of cytokine-producing T cells. These results support the therapeutic potential of JAK inhibition in combination with checkpoint inhibitors in cancer and highlight the potential of reshaped myeloid immunity to improve immunotherapy. One sentence summary: Ruxolitinib reshapes myeloid immunity to synergize with checkpoint inhibitors\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=555&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b7f0bc964973b158b74dfbaa90685bbc53979f8\",\n",
      "        \"title\": \"An atlas of cells in the human tonsil\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=556&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59ef699f70ecd58d09bec58e2882eeba315e676b\",\n",
      "        \"title\": \"Self-supervised learning of cell type specificity from immunohistochemical images\",\n",
      "        \"abstract\": \"Abstract Motivation Advances in bioimaging now permit in situ proteomic characterization of cell\\u2013cell interactions in complex tissues, with important applications across a spectrum of biological problems from development to disease. These methods depend on selection of antibodies targeting proteins that are expressed specifically in particular cell types. Candidate marker proteins are often identified from single-cell transcriptomic data, with variable rates of success, in part due to divergence between expression levels of proteins and the genes that encode them. In principle, marker identification could be improved by using existing databases of immunohistochemistry for thousands of antibodies in human tissue, such as the Human Protein Atlas. However, these data lack detailed annotations of the types of cells in each image. Results We develop a method to predict cell type specificity of protein markers from unlabeled images. We train a convolutional neural network with a self-supervised objective to generate embeddings of the images. Using non-linear dimensionality reduction, we observe that the model clusters images according to cell types and anatomical regions for which the stained proteins are specific. We then use estimates of cell type specificity derived from an independent single-cell transcriptomics dataset to train an image classifier, without requiring any human labelling of images. Our scheme demonstrates superior classification of known proteomic markers in kidney compared to selection via single-cell transcriptomics. Availability and implementation Code and trained model are available at www.github.com/murphy17/HPA-SimCLR. Supplementary information Supplementary data are available at Bioinformatics online.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=557&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5ce4a53d15379f25c6f18e85ae75f2df8bb17ec\",\n",
      "        \"title\": \"Single-cell transcriptomics and surface epitope detection in human brain epileptic lesions identifies pro-inflammatory signaling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=558&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dafca22040721584818c82a8b0f3686e8104ccfd\",\n",
      "        \"title\": \"A novel strategy to dissect multifaceted macrophage function in human diseases\",\n",
      "        \"abstract\": \"Macrophages are widely distributed immune cells that play central roles in a variety of physiologic and pathologic processes, including obesity and cardiovascular disease (CVD). They are highly plastic cells that execute diverse functions according to a combination of signaling and environmental cues. While macrophages have traditionally been understood to polarize to either proinflammatory M1\\u2010like or anti\\u2010inflammatory M2\\u2010like states, evidence has shown that they exist in a spectrum of states between those 2 phenotypic extremes. In obesity\\u2010related disease, M1\\u2010like macrophages exacerbate inflammation and promote insulin resistance, while M2\\u2010like macrophages reduce inflammation, promoting insulin sensitivity. However, polarization markers are expressed inconsistently in adipose tissue macrophages, and they additionally exhibit phenotypes differing from the M1/M2 paradigm. In atherosclerotic CVD, activated plaque macrophages can also exist in a range of proinflammatory or anti\\u2010inflammatory states. Some of these macrophages scavenge lipids, developing into heterogeneous foam cell populations. To better characterize the many actions of macrophages in human disease, we have designed a novel set of computational tools: MacSpectrum and AtheroSpectrum. These tools provide information on the inflammatory polarization status, differentiation, and foaming of macrophages in both human and mouse samples, allowing for better characterization of macrophage subpopulations based on their function. Using these tools, we identified disease\\u2010relevant cell states in obesity and CVD, including the novel concept that macrophage\\u2010derived foam cell formation can follow homeostatic noninflammatory or pathogenic inflammatory foaming programs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=559&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1d4d9a823e55e6843795787a89c07fd1f4772c9f\",\n",
      "        \"title\": \"Noncellular components in the liver cancer stem cell niche: Biology and potential clinical implications\",\n",
      "        \"abstract\": \"Cancer stem cells (CSCs) are now recognized as one of the major root causes of therapy failure and tumor recurrence in hepatocellular carcinoma (HCC). Early studies in the field focused primarily on the intrinsic regulators of CSC maintenance, but in recent years, mounting evidence has demonstrated the presence and role of extrinsic regulators in the tumor microenvironment (TME) in the control of liver CSCs. In addition to direct interaction with cellular components, noncellular components, including the extracellular matrix, hypoxia, nutrient deprivation, and secreted molecules within the tumor stroma and hepatitis viruses, also play a critical role in shaping the CSC niche. In this review, we highlight how various noncellular components in the TME play a role in regulating CSCs and how CSCs secrete components to interact with the TME to generate their own niche, working hand in hand to drive tumor physiology in HCC. In addition, we describe the potential clinical applications of these findings and propose perspectives on future research of noncellular components in the liver CSC niche.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=560&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6f46cd9d2c2be1d90f016d486cbac715258636cd\",\n",
      "        \"title\": \"Processing human skin samples for single-cell assays\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=561&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"573f61274871f84697f00e7f8fbbfc6f7bf8864d\",\n",
      "        \"title\": \"Subsets of Tissue CD4 T Cells Display Different Susceptibilities to HIV Infection and Death: Analysis by CyTOF and Single Cell RNA-seq\",\n",
      "        \"abstract\": \"CD4 T lymphocytes belong to diverse cellular subsets whose sensitivity or resistance to HIV-associated killing remains to be defined. Working with lymphoid cells from human tonsils, we characterized the HIV-associated depletion of various CD4 T cell subsets using mass cytometry and single-cell RNA-seq. CD4 T cell subsets preferentially killed by HIV are phenotypically distinct from those resistant to HIV-associated cell death, in a manner not fully accounted for by their susceptibility to productive infection. Preferentially-killed subsets express CXCR5 and CXCR4 while preferentially-infected subsets exhibit an activated and exhausted effector memory cell phenotype. Single-cell RNA-seq analysis reveals that the subsets of preferentially-killed cells express genes favoring abortive infection and pyroptosis. These studies emphasize a complex interplay between HIV and distinct tissue-based CD4 T cell subsets, and the important contribution of abortive infection and inflammatory programmed cell death to the overall depletion of CD4 T cells that accompanies untreated HIV infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=562&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1b4613915b15e036ad0ff613b5be9d01e040fd64\",\n",
      "        \"title\": \"Interleukin-17 governs hypoxic adaptation of injured epithelium\",\n",
      "        \"abstract\": \"Mammalian cells autonomously activate hypoxia-inducible transcription factors (HIFs) to ensure survival in low-oxygen environments. We report here that injury-induced hypoxia is insufficient to trigger HIF1\\u03b1 in damaged epithelium. Instead, multimodal single-cell and spatial transcriptomics analyses and functional studies reveal that retinoic acid\\u2013related orphan receptor \\u03b3t+ (ROR\\u03b3t+) \\u03b3\\u03b4 T cell\\u2013derived interleukin-17A (IL-17A) is necessary and sufficient to activate HIF1\\u03b1. Protein kinase B (AKT) and extracellular signal\\u2013regulated kinase 1/2 (ERK1/2) signaling proximal of IL-17 receptor C (IL-17RC) activates mammalian target of rapamycin (mTOR) and consequently HIF1\\u03b1. The IL-17A\\u2013HIF1\\u03b1 axis drives glycolysis in wound front epithelia. Epithelial-specific loss of IL-17RC, HIF1\\u03b1, or blockade of glycolysis derails repair. Our findings underscore the coupling of inflammatory, metabolic, and migratory programs to expedite epithelial healing and illuminate the immune cell\\u2013derived inputs in cellular adaptation to hypoxic stress during repair. Description Metabolic rewiring helps wounds heal Rapid healing after skin injury is essential for protecting our internal organs from external threats. Epithelial cells must cope with a wound environment low in oxygen while simultaneously multiplying and migrating to quickly repair damage. Such adaptations to oxygen deprivation have long been thought to rely on direct cellular sensing and responsiveness by activating hypoxia-inducible factor 1\\u03b1 (HIF1\\u03b1). Konieczny et al. found instead that the cytokine interleukin-17A (IL-17A), which is produced by skin-resident \\u03b3\\u03b4 T cells, facilitates the adaptation of epithelium to hypoxia through HIF1\\u03b1. This IL-17A\\u2013HIF1\\u03b1 axis activates a program of glycolytic metabolism in epithelial cells that fuels repair. These findings underscore a role for immune-derived secondary signals in expediting repair by driving cellular adaptation to chronic hypoxia. \\u2014STS Interleukin-17A\\u2013induced hypoxia-inducible factor 1 alpha drives epithelial migration and glycolytic adaptation to hypoxia during tissue repair. INTRODUCTION Injury compromises our epithelial barriers, leaving them vulnerable to external threats. Organismal survival thus impinges on rapid barrier re-establishment after damage. Immunocompromised individuals and immunodeficient animals have profound defects in epithelial repair. However, the precise mechanisms and consequences of immune-epithelial cross-talk after injury remain incompletely defined. RATIONALE Epithelial repair requires cellular adaptation to the hypoxic wound microenvironment through evolutionarily conserved hypoxia-inducible factors (HIFs). This response has long been thought to depend upon epithelial cell-autonomous sensing of oxygen deprivation and responsiveness through HIFs. However, whether and how supportive microenvironmental signals from immune cells intersect with ancient hypoxia responses during epithelial repair have yet to be explored. To address these questions, we were drawn to the skin, a primary epithelial barrier that is surveilled by immune cells and has evolved sophisticated repair mechanisms. RESULTS Using unbiased profiling of repair-associated lymphocytes, we uncovered the enrichment of heterogeneous populations of type 17 lymphocytes. Homozygous Rorgt-EGFP (GFI-KI) mice deficient in ROR\\u03b3t+ cells exhibited a significant defect in epithelial migration and wound re-epithelialization. After damage, we observed a rapid local expansion of preexisting skin-resident ROR\\u03b3t+ cells that was sufficient to drive repair. Of the myriad wound ROR\\u03b3t+ populations, spatial transcriptomics and functional studies revealed that innate-like \\u03b3\\u03b4 T cells directed tissue repair by localizing to the wound front and boosting epithelial migration. These cells dominantly produced IL-17A/F in the wound microenvironment, which signaled directly to epithelia through IL-17RC to induce a migratory program. Comparative spatial transcriptomics of ROR\\u03b3t+ cell-deficient and wild-type (WT) wound edges revealed differences in HIF1\\u03b1 signaling. WT migrating epithelia had robust nuclear HIF1\\u03b1 activation that was notably missing from ROR\\u03b3t+ cell\\u2013 and epithelial IL-17RC\\u2013deficient (Il17rcEKO) animals. Accordingly, epithelial-specific loss of HIF1\\u03b1 (Hif1aEKO) phenocopied the wound-healing defect of GFP-KI and Il17rcEKO mice. However, in contrast to GFP-KI wounds, recombinant IL-17A was unable to augment repair in Hif1aEKO mice, underscoring the necessity of this transcriptional effector mediating re-epithelialization downstream of IL-17A signaling. ROR\\u03b3t+ cell\\u2013deficient and control animals had comparable levels of wound hypoxia, indicating that the failure of ROR\\u03b3t+ cell\\u2013deficient animals to induce HIF1\\u03b1 was independent of oxygen sensing. Instead, IL-17A was sufficient to activate epithelial HIF1\\u03b1 both in vivo and in epithelial organoids. IL-17RC signaling rapidly induced activation of both ERK and AKT, which augmented HIF1\\u03b1 protein and transcript levels, but not stability, through mammalian target of rapamycin (mTOR). Acute hypoxia potently activated epithelial HIF1\\u03b1 in vitro, and this response was further boosted by IL-17A. By contrast, we uncovered a striking loss of mTOR and HIF1\\u03b1 in chronic hypoxia. Exogenous IL-17A rescued mTOR, and consequently HIF1\\u03b1, under conditions of chronic hypoxia, indicating that IL-17A acts as a second signal to enable cell adaptation to low-oxygen environments. IL-17A induced a transcriptional and functional program of glycolytic metabolism in epithelia, which was dependent upon mTOR and HIF1\\u03b1. The IL-17A\\u2013HIF1\\u03b1-dependent program of glycolysis fueled epithelial migration, and pharmacological inhibition of glycolysis impaired wound re-epithelialization. CONCLUSION Our findings upend a long-held view that hypoxia is sufficient to cell-autonomously induce HIF1\\u03b1-mediated metabolic remodeling. We illustrate that IL-17A supplied by ROR\\u03b3t+ \\u03b3\\u03b4 T cells is necessary for optimal HIF1\\u03b1 activation in the wound-edge epithelium. The IL-17A\\u2013HIF1\\u03b1 axis directed the metabolic rewiring of damaged epithelium toward a program of glycolysis to fuel migration. HIF1\\u03b1 and glycolysis are drivers of tumor progression and metastasis, raising the possibility that IL-17A or other immune-derived signals could drive these pathways in cancer. IL-17A is also central to the pathology of many autoimmune conditions, including psoriasis and inflammatory bowel disease. Thus, the IL-17A\\u2013HIF1\\u03b1 axis unveiled by our study may provide therapeutic opportunities for a range of epithelial inflammatory and metastatic diseases. IL-17 drives epithelial HIF1\\u03b1 to fuel wound repair through glycolysis. IL-17A/F supplied by expanding skin-resident ROR\\u03b3t+ \\u03b3\\u03b4 T cells is necessary for optimal HIF1\\u03b1 activation in the wound-edge epithelium, even in the presence of hypoxia. IL-17A signaling through the IL-17RC receptor rapidly induces ERK/AKT/mTOR, which augments HIF1\\u03b1 transcripts and protein. The IL-17\\u2013HIF1\\u03b1 axis directs a transcriptional and functional program of glycolysis to promote migration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=563&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8a938139abeb067fb01b8cd0549764279cb59f3\",\n",
      "        \"title\": \"A single-cell atlas of the human brain in Alzheimer\\u2019s disease and its implications for personalized drug repositioning\",\n",
      "        \"abstract\": \"Alzheimer\\u2019s disease (AD) is a neurodegenerative disease with complex pathophysiology, and AD-dysregulated pathways are inconsistent across different brain regions and patients. Although single-cell RNA sequencing (scRNA-seq) has been performed in different regions of postmortem AD brains, the common and distinct molecular features among different regions remains largely unclear. This hinders the discovery of repurposable and personalized drugs for AD. We combined four scRNA-seq datasets and systematically investigated the common and distinct cellular responses, cell subpopulations, and transcription factors involved in AD. Moreover, we explored the transcriptional heterogeneity of different AD subtypes at the single-cell level. Finally, we conducted individual-based drug repurposing analysis to explore repurposable and personalized drugs. Six major brain cell types were detected after scRNA-seq batch-effect removal and noise cells filtering. Integration with genome-wide association studies (GWAS) summary statistics demonstrated that AD-susceptible genes were mainly enriched with differentially expressed genes (DEGs) in glial cells rather than neuronal cells. While most of DEGs were regulated in opposite directions among different cell types, cell-cell communication analysis revealed several common cellular interaction events involved in neurogenesis, as well as increased cell-cell adhesion. Our comprehensive drug repositioning analysis identified new candidates for AD treatment, including trichostatin, which was predicted to be broadly applicable to different identified AD subtypes, and vorinostat, which was specific for one subtype of AD. In summary, we delineated a cell-specific atlas of the AD transcriptome. Our work illustrated strong cellular heterogeneity in AD for defining AD subtypes. The cell-specific features are important for understanding AD etiology, progression, and drug discovery.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=564&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d647de51fd34f3ee5d7d25529502db6d88d9840\",\n",
      "        \"title\": \"Gut mucosa dissociation protocols influence cell type proportions and single-cell gene expression levels\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=565&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"15040ea34ee90b4656505eb7ce670edc78af5644\",\n",
      "        \"title\": \"Characterization of naked mole\\u2010rat hematopoiesis reveals unique stem and progenitor cell patterns and neotenic traits\",\n",
      "        \"abstract\": \"Naked mole rats (NMRs) are the longest\\u2010lived rodents yet their stem cell characteristics remain enigmatic. Here, we comprehensively mapped the NMR hematopoietic landscape and identified unique features likely contributing to longevity. Adult NMRs form red blood cells in spleen and marrow, which comprise a myeloid\\u2009bias toward granulopoiesis together with decreased B\\u2010lymphopoiesis. Remarkably, youthful blood and marrow single\\u2010cell transcriptomes and cell compositions are largely maintained until at least middle age. Similar to primates, the primitive stem and progenitor cell (HSPC) compartment is marked by CD34 and THY1. Stem cell polarity is seen for Tubulin but not CDC42, and is not lost until 12\\u2009years of age. HSPC respiration rates are as low as in purified human stem cells, in concert with a strong expression signature for fatty acid metabolism. The pool of quiescent stem cells is higher than in mice, and the cell cycle of hematopoietic cells is prolonged. By characterizing the NMR hematopoietic landscape, we identified resilience phenotypes such as an increased quiescent HSPC compartment, absence of age\\u2010related decline, and neotenic traits likely geared toward longevity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=566&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"687c6c37c3648b6d0503f7d1082e21eefdc661b2\",\n",
      "        \"title\": \"Discovering Immune-Mediated Mechanisms of Gastric Carcinogenesis Through Single-Cell RNA Sequencing\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNAseq) technology is still relatively new in the field of gastric cancer immunology but gaining significant traction. This technology now provides unprecedented insights into the intratumoral and intertumoral heterogeneities at the immunological, cellular, and molecular levels. Within the last few years, a volume of publications reported the usefulness of scRNAseq technology in identifying thus far elusive immunological mechanisms that may promote and impede gastric cancer development. These studies analyzed datasets generated from primary human gastric cancer tissues, metastatic ascites fluid from gastric cancer patients, and laboratory-generated data from in vitro and in vivo models of gastric diseases. In this review, we overview the exciting findings from scRNAseq datasets that uncovered the role of critical immune cells, including T cells, B cells, myeloid cells, mast cells, ILC2s, and other inflammatory stromal cells, like fibroblasts and endothelial cells. In addition, we also provide a synopsis of the initial scRNAseq findings on the interesting epithelial cell responses to inflammation. In summary, these new studies have implicated roles for T and B cells and subsets like NKT cells in tumor development and progression. The current studies identified diverse subsets of macrophages and mast cells in the tumor microenvironment, however, additional studies to determine their roles in promoting cancer growth are needed. Some groups specifically focus on the less prevalent ILC2 cell type that may contribute to early cancer development. ScRNAseq analysis also reveals that stromal cells, e.g., fibroblasts and endothelial cells, regulate inflammation and promote metastasis, making them key targets for future investigations. While evaluating the outcomes, we also highlight the gaps in the current findings and provide an assessment of what this technology holds for gastric cancer research in the coming years. With scRNAseq technology expanding rapidly, we stress the need for periodic review of the findings and assess the available scRNAseq analytical tools to guide future work on immunological mechanisms of gastric carcinogenesis. \\u2003\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=567&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abb7218aa6062dc1fa5f83e6f9210c63270cb0fa\",\n",
      "        \"title\": \"Comparison of methods and resources for cell-cell communication inference from single-cell RNA-Seq data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=568&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d56cc7ab718dd58733360f83b0e3d2070e786510\",\n",
      "        \"title\": \"Single-cell multi-omics defines the cell-type specific impact of splicing aberrations in human hematopoietic clonal outgrowths\",\n",
      "        \"abstract\": \"RNA splicing factors are recurrently affected by alteration-of-function mutations in clonal blood disorders, highlighting the importance of splicing regulation in hematopoiesis. However, our understanding of the impact of dysregulated RNA splicing has been hampered by the inability to distinguish mutant and wildtype cells in primary patient samples, the cell-type complexity of the hematopoietic system, and the sparse and biased coverage of splice junctions by short-read sequencing typically used in single-cell RNA sequencing. To overcome these limitations, we developed GoT-Splice by integrating Genotyping of Transcriptomes (GoT) with enhanced efficiency long-read single-cell transcriptome profiling, as well as proteogenomics (with CITE-seq). This allowed for the simultaneous single-cell profiling of gene expression, cell surface protein markers, somatic mutation status, and RNA splicing. We applied GoT-Splice to bone marrow progenitors from patients with myelodysplastic syndrome (MDS) affected by mutations in the most prevalent mutated RNA splicing factor \\u2013 the core RNA splicing factor SF3B1. High-resolution mapping of SF3B1mut vs. SF3B1wt hematopoietic progenitors revealed a fitness advantage of SF3B1mut cells in the megakaryocytic-erythroid lineage, resulting in an expansion of SF3B1mut erythroid progenitor (EP) cells. SF3B1mut EP cells exhibited upregulation of genes involved in regulation of cell cycle and mRNA translation. Long-read single-cell transcriptomes revealed the previously reported increase of aberrant 3\\u2019 splicing site usage in SF3B1mut cells. However, the ability to profile splicing within individual cell populations uncovered distinct cryptic 3\\u2019 splice site usage across different progenitor populations, as well as stage-specific aberrant splicing during erythroid maturation. Lastly, as splice factor mutations occur in clonal hematopoiesis (CH) with increased risk of neoplastic transformation, we applied GoT-Splice to CH samples. These data revealed that the erythroid lineage bias, as well as cell-type specific cryptic 3\\u2019 splice site usage in SF3B1mut cells, precede overt MDS. Collectively, we present an expanded multi-omics single-cell toolkit to define the cell-type specific impact of somatic mutations on RNA splicing, from the earliest phases of clonal outgrowths to overt neoplasia, directly in human samples.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=569&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ffefb7a974ef4ebbd8e4cdf465c4890b3f88075\",\n",
      "        \"title\": \"Single-cell multi-omics integration for unpaired data by a siamese network with graph-based contrastive loss\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=570&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d4449c89a73f3eb282d04e9a77310496b68ce19d\",\n",
      "        \"title\": \"Efficient isolation of rare B cells using next-generation antigen barcoding\",\n",
      "        \"abstract\": \"The ability to efficiently isolate antigen-specific B cells in high throughput will greatly accelerate the discovery of therapeutic monoclonal antibodies (mAbs) and catalyze rational vaccine development. Traditional mAb discovery is a costly and labor-intensive process, although recent advances in single-cell genomics using emulsion microfluidics allow simultaneous processing of thousands of individual cells. Here we present a streamlined method for isolation and analysis of large numbers of antigen-specific B cells, including next generation antigen barcoding and an integrated computational framework for B cell multi-omics. We demonstrate the power of this approach by recovering thousands of antigen-specific mAbs, including the efficient isolation of extremely rare precursors of VRC01-class and IOMA-class broadly neutralizing HIV mAbs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=571&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18ce65fa88351ef9222ee853bd6199f399d11432\",\n",
      "        \"title\": \"CD39 Identifies Tumor-Reactive CD8 T cells in Patients With Lung Cancer\",\n",
      "        \"abstract\": \"The repertoire of tumor-infiltrating lymphocytes (TILs) can be vast, and many of these TILs are not endowed with tumor reactivity. While a number of reports have shown that tumor-reactive TILs express CD39, few reports have demonstrated that conversely, CD39 can be leveraged to serve as a proxy of tumor-reactive CD8 T cells. Using single-cell CITE/RNA/TCRseq, we show that CD39+ CD8 T cells in human lung cancers demonstrate transcriptional and proteomic features of exhaustion, tumor reactivity, and clonal expansion. Moreover, TCR cloning revealed that CD39 enriched for tumor-reactive CD8 T cell clones. Flow cytometry of 440 lung cancer specimens revealed that CD39 level on CD8 T cells is only weakly correlated with tumoral features that currently guide lung cancer therapy, such as histology, driver mutation, PD-L1 and tumor mutation burden. PD-1 axis blockade, but not cytotoxic chemotherapy, increased intratumoral CD39+ CD8 T cells. CD39 correlated with PD-1 expression on CD8 T cells and high pre-treatment/early-on-treatment levels were associated with improved clinical outcomes, but not immune-related adverse events, from immune checkpoint blockade therapy. This comprehensive profiling of the clinical, pathological and molecular features highlights the utility of CD39 as a proxy for tumor-reactive CD8 T cells in human lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=572&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"049dc0f656eb4a698d8047e95d8705922e1fd3f4\",\n",
      "        \"title\": \"A developmental atlas of somatosensory diversification and maturation in the dorsal root ganglia by single-cell mass cytometry\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=573&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"57aa061debd40f737724ce2767e8d7c8466e3795\",\n",
      "        \"title\": \"Spatiotemporal multiplexed immunofluorescence imaging of living cells and tissues with bioorthogonal cycling of fluorescent probes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=574&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2b84b330e96122704b1490912d22de874da5beff\",\n",
      "        \"title\": \"Single cell\\u2010 and spatial \\u2018Omics revolutionize physiology\",\n",
      "        \"abstract\": \"Single cell multi\\u2010 \\u2018Omics and Spatial Transcriptomics are prominent technological highlights of recent years, and both fields still witness a ceaseless firework of novel approaches for high resolution profiling of additional omics layers. As all life processes in organs and organisms are based on the functions of their fundamental building blocks, the individual cells and their interactions, these methods are of utmost worth for the study of physiology in health and disease. Recent discoveries on embryonic development, tumor immunology, the detailed cellular composition and function of complex tissues like for example the kidney or the brain, different roles of the same cell type in different organs, the oncogenic program of individual tumor entities, or the architecture of immunopathology in infected tissue are based on single cell and spatial transcriptomics experiments. In this review, we will give a broad overview of technological concepts for single cell and spatial analysis, showing both advantages and limitations, and illustrate their impact with some particularly impressive case studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=575&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"32e2e53e5415b63487798a9f8b0db52b5f3c0fca\",\n",
      "        \"title\": \"Decoding Expression Dynamics of Protein and Transcriptome at the Single-Cell Level in Paired Picoliter Chambers.\",\n",
      "        \"abstract\": \"Simultaneous analysis of mRNAs and proteins at the single-cell level provides information about the dynamics and correlations of gene and protein expressions in individual cells, enabling a comprehensive study of cellular heterogeneity and expression patterns. Here, we present a platform for about 1000 cellular indexing of mRNAs and membrane proteins, named multi-Paired-seq, with high cell utilization, accurate molecular measurement, and low cost. Based on hydrodynamic differential flow resistance, multi-Paired-seq largely improves cell utilization in the percentage of cells measured in population (>95%). Combined with the pump/valve structure, cell-free antibodies and mRNAs can be removed completely for highly accurate detection (R = 0.96) of protein copies. The picoliter reaction chambers allow high detection sensitivity for both mRNA transcripts and protein copies and low sequencing cost. Using multi-Paired-seq, three clusters of known breast cancer cell types are identified according to multimodal measurements, and the expression correlations between mRNAs and proteins under altered conditions are quantified. Multi-Paired-seq provides multimodal measurements at the single-cell level, which offers a new tool for cell biology, developmental biology, drug discovery, and precision medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=576&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"84fabf9dc5261ca3df659282fbb24dcc090d6cb4\",\n",
      "        \"title\": \"Single-cell antigen-specific landscape of CAR T infusion product identifies determinants of CD19-positive relapse in patients with ALL\",\n",
      "        \"abstract\": \"A notable number of acute lymphoblastic leukemia (ALL) patients develop CD19-positive relapse within 1 year after receiving chimeric antigen receptor (CAR) T cell therapy. It remains unclear if the long-term response is associated with the characteristics of CAR T cells in infusion products, hindering the identification of biomarkers to predict therapeutic outcomes. Here, we present 101,326 single-cell transcriptomes and surface protein landscape from the infusion products of 12 ALL patients. We observed substantial heterogeneity in the antigen-specific activation states, among which a deficiency of T helper 2 function was associated with CD19-positive relapse compared with durable responders (remission, >54 months). Proteomic data revealed that the frequency of early memory T cells, rather than activation or coinhibitory signatures, could distinguish the relapse. These findings were corroborated by independent functional profiling of 49 patients, and an integrative model was developed to predict the response. Our data unveil the molecular mechanisms that may inform strategies to boost specific T cell function to maintain long-term remission.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=577&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"20e30c95e14abc0f3c0f7bceea9e48ab0bff11d4\",\n",
      "        \"title\": \"A gastruloid model of the interaction between embryonic and extra-embryonic cell types\",\n",
      "        \"abstract\": \"Stem-cell derived in vitro systems, such as organoids or embryoids, hold great potential for modeling in vivo development. Full control over their initial composition, scalability, and easily measurable dynamics make those systems useful for studying specific developmental processes in isolation. Here we report the formation of gastruloids consisting of mouse embryonic stem cells (mESCs) and extraembryonic endoderm (XEN) cells. These XEN-enhanced gastruloids (XEGs) exhibit the formation of neural epithelia, which are absent in gastruloids derived from mESCs only. By single-cell RNA-seq, imaging, and differentiation experiments, we demonstrate the neural characteristics of the epithelial tissue. We further show that the mESCs induce the differentiation of the XEN cells to a visceral endoderm-like state. Finally, we demonstrate that local inhibition of WNT signaling and production of a basement membrane by the XEN cells underlie the formation of the neuroepithelial tissue. In summary, we establish XEGs to explore heterotypic cellular interactions and their developmental consequences in vitro.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=578&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8902fccb7b3ad9f0570409aad2e165c9f3846e39\",\n",
      "        \"title\": \"Integration of single-cell RNA and protein data identifies novel clinically-relevant lymphocyte phenotypes in breast cancers\",\n",
      "        \"abstract\": \"Immune cells are critical determinants of solid tumour aetiology, but the diverse phenotypes of intra-tumoural immune cells remain incompletely characterised. We applied integrated single cell RNA sequencing (scRNA-Seq) and highly multiplexed protein epitope analysis to a cohort of breast cancer samples to resolve cell states within the tumour microenvironment. We reveal novel protein markers for resting and activated tumour infiltrating lymphocytes, and show that high expression of CD103 primarily marks exhausted CD8 rather than tissue resident CD8 T-cells in human breast cancers. We identify two distinct states of activated CD4+ T follicular helper (Tfh) cells. A population resembling conventional Tfh (cTfh) cells were localised primarily to lymphoid aggregates by spatial transcriptomics. In contrast, cancer associated Tfh (caTfh) cells expressing markers of tissue residency and exhaustion co-localized with cancer foci and signalled to macrophages. Importantly, increased caTfh : cTfh ratio associated with improved disease outcome and response to checkpoint immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=579&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b68e7431dc2ccab6d012b3a5f5da86eff71d078\",\n",
      "        \"title\": \"Single cell RNA-seq by mostly-natural sequencing by synthesis\",\n",
      "        \"abstract\": \"Massively parallel single cell RNA-seq (scRNA-seq) for diverse applications, from cell atlases to functional screens, is increasingly limited by sequencing costs, and large-scale low-cost sequencing can open many additional applications, including patient diagnostics and drug screens. Here, we adapted and systematically benchmarked a newly developed, mostly-natural sequencing by synthesis method for scRNA-seq. We demonstrate successful application in four scRNA-seq case studies of different technical and biological types, including 5\\u2019 and 3\\u2019 scRNA-seq, human peripheral blood mononuclear cells from a single individual and in multiplex, as well as Perturb-Seq. Our data show comparable results to existing technology, including compatibility with state-of-the-art scRNA-seq libraries independent of the sequencing technology used \\u2013 thus providing an enhanced cost-effective path for large scale scRNA-seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=580&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f7b6c698dd4e3b2bcdfd454118f840f2159b71e\",\n",
      "        \"title\": \"Epigenetic regulation of T cell exhaustion\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=581&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8291851d68410eb4a705d94247e557fb156f8b98\",\n",
      "        \"title\": \"Glycan Profiling by Sequencing to Uncover Multicellular Communication: Launching Glycobiology in Single Cells and Microbiomes\",\n",
      "        \"abstract\": \"Glycans are essential building blocks of life that are located at the outermost surface of all cells from mammals to bacteria and even viruses. Cell surface glycans mediate multicellular communication in diverse biological processes and are useful as \\u201csurface markers\\u201d to identify cells. Various single-cell sequencing technologies have already emerged that enable the high-throughput analysis of omics information, such as transcriptome and genome profiling on a cell-by-cell basis, which has advanced our understanding of complex multicellular interactions. However, there has been no robust technology to analyze the glycome in single cells, mainly because glycans with branched and heterogeneous structures cannot be readily amplified by polymerase chain reactions like nucleic acids. We hypothesized that the generation of lectins conjugated with DNA barcodes (DNA-barcoded lectins) would enable the conversion of glycan information to gene information, which may be amplified and measured using DNA sequencers. This technology will enable the simultaneous analysis of glycan and RNA in single cells. Based on this concept, we developed a technology to analyze glycans and RNA in single cells, which was referred to as scGR-seq. Using scGR-seq, we acquired glycan and gene expression profiles of individual cells constituting heterogeneous cell populations, such as tissues. We further extended Glycan-seq to the profiling of the surface glycans of bacteria and even gut microbiota. Glycan-seq and scGR-seq are new technologies that enable us to elucidate the function of glycans in cell\\u2013cell and cell\\u2013microorganism communication, which extends glycobiology to the level of single cells and microbiomes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=582&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0fcba7b85dc07c7f7786bcffb4aa579add796cb8\",\n",
      "        \"title\": \"Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=583&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dac1d6ee6acd5a995b4b40eb0e35ca02932a23ef\",\n",
      "        \"title\": \"Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=584&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e41fbaead50671c20a065863a6949197ff9a2fce\",\n",
      "        \"title\": \"The use of single-cell multi-omics in immuno-oncology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=585&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe0ce56a54273b168d55a71bc7587f7939a189ab\",\n",
      "        \"title\": \"Breaking the Immune Complexity of the Tumor Microenvironment Using Single-Cell Technologies\",\n",
      "        \"abstract\": \"Tumors are not a simple aggregate of transformed cells but rather a complicated ecosystem containing various components, including infiltrating immune cells, tumor-related stromal cells, endothelial cells, soluble factors, and extracellular matrix proteins. Profiling the immune contexture of this intricate framework is now mandatory to develop more effective cancer therapies and precise immunotherapeutic approaches by identifying exact targets or predictive biomarkers, respectively. Conventional technologies are limited in reaching this goal because they lack high resolution. Recent developments in single-cell technologies, such as single-cell RNA transcriptomics, mass cytometry, and multiparameter immunofluorescence, have revolutionized the cancer immunology field, capturing the heterogeneity of tumor-infiltrating immune cells and the dynamic complexity of tenets that regulate cell networks in the tumor microenvironment. In this review, we describe some of the current single-cell technologies and computational techniques applied for immune-profiling the cancer landscape and discuss future directions of how integrating multi-omics data can guide a new \\u201cprecision oncology\\u201d advancement.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=586&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"01e97bab096582d703cff5156b9e732a3e0b22a8\",\n",
      "        \"title\": \"CITE-Viz: Replicating the Interactive Flow Cytometry Workflow in CITE-Seq\",\n",
      "        \"abstract\": \"Summary The rapid advancement of new genomic sequencing technology has enabled the development of multi-omic single-cell sequencing assays. These assays profile multiple modalities in the same cell and can often yield new insights not revealed with a single modality. For example, CITE-Seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) simultaneously profiles the single-cell RNA transcriptome and the surface protein expression. The extra dimension of surface protein markers can be used to further identify cell clusters \\u2013 an essential step for downstream analyses and interpretation. Additionally, multi-dimensional datasets like CITE-Seq require nuanced visualization methods to accurately assess the data. To facilitate cell cluster classification and visualization in CITE-Seq, we developed CITE-Viz. CITE-Viz is a single-cell visualization platform with a custom module that replicates the interactive flow-cytometry gating workflow. With CITE-Viz, users can investigate CITE-Seq specific quality control (QC) metrics, view multi-omic co-expression feature plots, and classify cell clusters by iteratively gating on the abundance of cell surface markers. CITE-Viz was developed to make multi-modal single-cell analysis accessible to a wide variety of biologists, with the aim to discover new insights into their data and to facilitate novel hypothesis generation. Availability and Implementation CITE-Viz installation and usage instructions can be found in the GitHub repository https://github.com/maxsonBraunLab/CITE-Viz Contact maxsonj@ohsu.edu Supplementary Information Down-sampled peripheral blood mononuclear dataset (Hao et al. 2021): https://bit.ly/3vxbhfW\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=587&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9c65b074b4707a2b04ae8651aa734fab0612772a\",\n",
      "        \"title\": \"SystemMatch: optimizing preclinical drug models to human clinical outcomes via generative latent-space matching\",\n",
      "        \"abstract\": \"Translating the relevance of preclinical models ( in vitro , animal models, or organoids) to their relevance in humans presents an important challenge during drug development. The rising abundance of single-cell genomic data from human tumors and tissue offers a new opportunity to optimize model systems by their similarity to targeted human cell types in disease. In this work, we introduce SystemMatch to assess the \\ufb01t of preclinical model systems to an in sapiens target population and to recommend experimental changes to further optimize these systems. We demonstrate this through an application to developing in vitro systems to model human tumor-derived suppressive macrophages. We show with held-out in vivo controls that our pipeline successfully ranks macrophage sub-populations by their biological similarity to the target population, and apply this analysis to rank a series of 18 in vitro macrophage systems perturbed with a variety of cytokine stimulations. We extend this analysis to predict the behavior of 66 in silico model systems generated using a perturbational autoencoder and apply a k -medoids approach to recommend a subset of these model systems for further experimental development in order to fully explore the space of possible perturbations. Through this use case, we demonstrate a novel approach to model system development to generate a system more similar to human biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=588&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8dc03172adb326b25cbaf9ac2760ef4c2fb0e5bb\",\n",
      "        \"title\": \"Deciphering Innate Immune Cell-Tumor Microenvironment Crosstalk at a Single-Cell Level\",\n",
      "        \"abstract\": \"The tumor microenvironment encompasses various innate immune cells which regulate tumor progression. Exploiting innate immune cells is a new frontier of cancer immunotherapy. However, the classical surface markers for cell-type classification cannot always well-conclude the phenotype, which will further hinge our understanding. The innate immune cells include dendritic cells, monocytes/macrophages, natural killer cells, and innate lymphoid cells. They play important roles in tumor growth and survival, in some cases promoting cancer, in other cases negating cancer. The precise characterization of innate immune cells at the single-cell level will boost the potential of cancer immunotherapy. With the development of single-cell RNA sequencing technology, the transcriptome of each cell in the tumor microenvironment can be dissected at a single-cell level, which paves a way for a better understanding of the cell type and its functions. Here, we summarize the subtypes and functions of innate immune cells in the tumor microenvironment based on recent literature on single-cell technology. We provide updates on recent achievements and prospects for how to exploit novel functions of tumor-associated innate immune cells and target them for cancer immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=589&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9f9ebfe9a79e3d055aeb29b887fdc240e34ea1b1\",\n",
      "        \"title\": \"Isolation of Viable Single Cells With High Yield and Purity Using a Small Amount of Human Kidney Tissue Biopsy\",\n",
      "        \"abstract\": \"Graphical Abstract Objective: Establishment of an efficient method of preparing human kidney single cell suspension, using a very small amount of tissue puncture. Methods: Samples of human kidney tissue puncture were cut into pieces, and then 80 \\u03bcL of the digestive enzyme were added to each punctured tissue to induce enzymatic digestion. The enzyme combination is composed of collagenases, DNase and hyaluronidase and the sample was incubated 20 min at 37\\u00b0C. The obtained cell suspension was filtered through a 70 \\u03bcm cell strainer, centrifuged at 300 g for 5 min and the supernatant was removed, then the pellet was resuspended in 3 ml of DMEM (Dulbecco\\u2019s Modified Eagle\\u2019s Medium). Cell suspension was sorted and purified by flow sorting to remove dead cells and obtain a cell suspension with higher viability rate. Results: We found that 1) diverse single cells of human kidney can be obtained by the digestive enzyme, as observed under the light microscope, with different sizes, normal cell morphology and good dispersion. 2) (2-3) \\u00d7 106 single cells can be extracted from one fresh punctured kidney tissue of about 10 mg, with a cell viability rate of more than 80%. Conclusion: In this work we generated a comprehensive and high-resolution single-cell method, which is simple and efficient for preparing single cell suspension from a minimal amount of human kidney tissue. This method can facilitate the study of renal cell biology and the pathogenesis of kidney diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=590&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcbecc077abc1c9d6ee5eaa6ff5a1ae8227759c3\",\n",
      "        \"title\": \"Bi-order multimodal integration of single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=591&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dd0e8c378e7d32897e7c65681af24d7100a03f60\",\n",
      "        \"title\": \"Landscape of helper and regulatory antitumour CD4+ T cells in melanoma\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=592&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9c4677d03bd3f263c8823bc7202658e42fb8fc10\",\n",
      "        \"title\": \"The Road to Elimination: Current State of Schistosomiasis Research and Progress Towards the End Game\",\n",
      "        \"abstract\": \"The new WHO Roadmap for Neglected Tropical Diseases targets the global elimination of schistosomiasis as a public health problem. To date, control strategies have focused on effective diagnostics, mass drug administration, complementary and integrative public health interventions. Non-mammalian intermediate hosts and other vertebrates promote transmission of schistosomiasis and have been utilized as experimental model systems. Experimental animal models that recapitulate schistosomiasis immunology, disease progression, and pathology observed in humans are important in testing and validation of control interventions. We discuss the pivotal value of these models in contributing to elimination of schistosomiasis. Treatment of schistosomiasis relies heavily on mass drug administration of praziquantel whose efficacy is comprised due to re-infections and experimental systems have revealed the inability to kill juvenile schistosomes. In terms of diagnosis, nonhuman primate models have demonstrated the low sensitivity of the gold standard Kato Katz smear technique. Antibody assays are valuable tools for evaluating efficacy of candidate vaccines, and sera from graded infection experiments are useful for evaluating diagnostic sensitivity of different targets. Lastly, the presence of Schistosomes can compromise the efficacy of vaccines to other infectious diseases and its elimination will benefit control programs of the other diseases. As the focus moves towards schistosomiasis elimination, it will be critical to integrate treatment, diagnostics, novel research tools such as sequencing, improved understanding of disease pathogenesis and utilization of experimental models to assist with evaluating performance of new approaches.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=593&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f60dff7e47e82ef6286fc2a50b0349162505f378\",\n",
      "        \"title\": \"Resolution of the curse of dimensionality in single-cell RNA sequencing data analysis\",\n",
      "        \"abstract\": \"This work formulates a noise reduction method, RECODE, which resolves the curse of dimensionality in noisy high-dimensional data, including scRNA-seq data, for effective downstream analyses. Single-cell RNA sequencing (scRNA-seq) can determine gene expression in numerous individual cells simultaneously, promoting progress in the biomedical sciences. However, scRNA-seq data are high-dimensional with substantial technical noise, including dropouts. During analysis of scRNA-seq data, such noise engenders a statistical problem known as the curse of dimensionality (COD). Based on high-dimensional statistics, we herein formulate a noise reduction method, RECODE (resolution of the curse of dimensionality), for high-dimensional data with random sampling noise. We show that RECODE consistently resolves COD in relevant scRNA-seq data with unique molecular identifiers. RECODE does not involve dimension reduction and recovers expression values for all genes, including lowly expressed genes, realizing precise delineation of cell fate transitions and identification of rare cells with all gene information. Compared with representative imputation methods, RECODE employs different principles and exhibits superior overall performance in cell-clustering, expression value recovery, and single-cell\\u2013level analysis. The RECODE algorithm is parameter-free, data-driven, deterministic, and high-speed, and its applicability can be predicted based on the variance normalization performance. We propose RECODE as a powerful strategy for preprocessing noisy high-dimensional data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=594&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"642c0c0cbdc0bcf92b9ecd171676470c99b81878\",\n",
      "        \"title\": \"Activated SUMOylation restricts MHC class I antigen presentation to confer immune evasion in cancer\",\n",
      "        \"abstract\": \"Activated SUMOylation is a hallmark of cancer. Starting from a targeted screening for SUMO-regulated immune evasion mechanisms, we identified an evolutionarily conserved function of activated SUMOylation, which attenuated the immunogenicity of tumor cells. Activated SUMOylation allowed cancer cells to evade CD8+ T cell\\u2013mediated immunosurveillance by suppressing the MHC class I (MHC-I) antigen-processing and presentation machinery (APM). Loss of the MHC-I APM is a frequent cause of resistance to cancer immunotherapies, and the pharmacological inhibition of SUMOylation (SUMOi) resulted in reduced activity of the transcriptional repressor scaffold attachment factor B (SAFB) and induction of the MHC-I APM. Consequently, SUMOi enhanced the presentation of antigens and the susceptibility of tumor cells to CD8+ T cell\\u2013mediated killing. Importantly, SUMOi also triggered the activation of CD8+ T cells and thereby drove a feed-forward loop amplifying the specific antitumor immune response. In summary, we showed that activated SUMOylation allowed tumor cells to evade antitumor immunosurveillance, and we have expanded the understanding of SUMOi as a rational therapeutic strategy for enhancing the efficacy of cancer immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=595&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1d0bc87f929df22059d0f4c8b323815095011c2b\",\n",
      "        \"title\": \"Machine learning: its challenges and opportunities in plant system biology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=596&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"41fa088fdaa32776173524f2209c26431055b0bd\",\n",
      "        \"title\": \"A human breast atlas integrating single-cell proteomics and transcriptomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=597&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58f89444483d810d5b6353219abf9f8b31812c1b\",\n",
      "        \"title\": \"Identification of a distinct NK-like hepatic T-cell population activated by NKG2C in a TCR-independent manner.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=598&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4f9c8f41d3b670eb00fb391f20cf200f8354d621\",\n",
      "        \"title\": \"Technological and computational advances driving high-throughput oncology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=599&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ec23f70da13aae8d25e8125dcd8654b79db5ed15\",\n",
      "        \"title\": \"Into the multiverse: advances in single-cell multiomic profiling.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=600&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"08d00aafb0de27255fa9fc83a9b3e1732c56b55f\",\n",
      "        \"title\": \"SINGLE CELL MULTIOMIC APPROACHES TO DISENTANGLE T CELL HETEROGENEITY.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=601&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f6c9b343d2ec842691b14bf1a6c7a9984bd6c1e\",\n",
      "        \"title\": \"Single-Cell Transcriptomics of Liver Cancer: Hype or Insights?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=602&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bf4ccb627f281e68ef883ebae57410f3bf25f9db\",\n",
      "        \"title\": \"Single-cell transcriptomic and chromatin accessibility analyses of dairy cattle peripheral blood mononuclear cells and their responses to lipopolysaccharide\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=603&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f1eae4b611a6f109adf3d370d15dbde667a5691\",\n",
      "        \"title\": \"One-Way Matching of Datasets with Low Rank Signals\",\n",
      "        \"abstract\": \"We study one-way matching of a pair of datasets with low rank signals. Under a stylized model, we \\ufb01rst derive information-theoretic limits of matching. We then show that linear assignment with projected data achieves fast rates of convergence and sometimes even minimax rate optimality for this task. The theoretical error bounds are corroborated by simulated examples. Furthermore, we illustrate practical use of the matching procedure on two single-cell data examples.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=604&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c3891d8c7c3ebdcf6746f7acdcfcece23a60916b\",\n",
      "        \"title\": \"The ImmGen consortium OpenSource T cell project\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=605&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3e29a4e961d3787b780aaa65b1982ce1e96b0527\",\n",
      "        \"title\": \"UMINT: Unsupervised Neural Network For Single Cell Multi-Omics Integration\",\n",
      "        \"abstract\": \"Multi-omics studies have enabled us to understand the mechanistic drivers behind complex disease states and progressions, thereby providing novel and actionable biological insights into health status. However, integrating data from multiple modalities is challenging due to the high dimensionality of data and noise associated with each platform. Non-overlapping features and technical batch effects in the data make the task of learning more complicated. Conventional machine learning (ML) tools are not quite effective against such data integration hazards. In addition, existing methods for single cell multi-omics integration are computationally expensive. This has encouraged the development of a novel architecture that produces a robust model for integration of high-dimensional multi-omics data, which would be capable of learning meaningful features for further downstream analysis. In this work, we have introduced a novel Unsupervised neural network for single cell Multi-omics INTegration (UMINT). UMINT serves as a promising model for integrating variable number of single cell omics layers with high dimensions, and provides substantial reduction in the number of parameters. It is capable of learning a latent low-dimensional embedding that can capture useful data characteristics. The effectiveness of UMINT has been evaluated on benchmark CITE-seq (paired RNA and surface proteins) datasets. It has outperformed existing state-of-the-art methods for multi-omics integration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=606&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9636cecf9d7c667be6aded80cf8613e162a92619\",\n",
      "        \"title\": \"Single-Cell Sequencing and Its Applications in Liver Cancer\",\n",
      "        \"abstract\": \"As one of the most lethal cancers, primary liver cancer (PLC) has high tumor heterogeneity, including the heterogeneity between cancer cells. Traditional methods which have been used to identify tumor heterogeneity for a long time are based on large mixed cell samples, and the research results usually show average level of the cell population, ignoring the heterogeneity between cancer cells. In recent years, single-cell sequencing has been increasingly applied to the studies of PLCs. It can detect the heterogeneity between cancer cells, distinguish each cell subgroup in the tumor microenvironment (TME), and also reveal the clonal characteristics of cancer cells, contributing to understand the evolution of tumor. Here, we introduce the process of single-cell sequencing, review the applications of single-cell sequencing in the heterogeneity of cancer cells, TMEs, oncogenesis, and metastatic mechanisms of liver cancer, and discuss some of the current challenges in the field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=607&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1439c35cd35c1ac0f217f154b253263f064e7077\",\n",
      "        \"title\": \"Identification of genetic variants that impact gene co-expression relationships using large-scale single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=608&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7aabbab525553242a01872b4cff1fbd3f4cb4880\",\n",
      "        \"title\": \"TNF is a potential therapeutic target to suppress prostatic inflammation and hyperplasia in autoimmune disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=609&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"af222fcea0e4154e2e86cb2b48eff7941750ba8e\",\n",
      "        \"title\": \"Classification of human chronic inflammatory skin disease based on single-cell immune profiling\",\n",
      "        \"abstract\": \"Inflammatory conditions represent the largest class of chronic skin disease, but the molecular dysregulation underlying many individual cases remains unclear. Single-cell RNA sequencing (scRNA-seq) has increased precision in dissecting the complex mixture of immune and stromal cell perturbations in inflammatory skin disease states. We single-cell\\u2013profiled CD45+ immune cell transcriptomes from skin samples of 31 patients (7 atopic dermatitis, 8 psoriasis vulgaris, 2 lichen planus (LP), 1 bullous pemphigoid (BP), 6 clinical/histopathologically indeterminate rashes, and 7 healthy controls). Our data revealed active proliferative expansion of the Treg and Trm components and universal T cell exhaustion in human rashes, with a relative attenuation of antigen-presenting cells. Skin-resident memory T cells showed the greatest transcriptional dysregulation in both atopic dermatitis and psoriasis, whereas atopic dermatitis also demonstrated recurrent abnormalities in ILC and CD8+ cytotoxic lymphocytes. Transcript signatures differentiating these rash types included genes previously implicated in T helper cell (TH2)/TH17 diatheses, segregated in unbiased functional networks, and accurately identified disease class in untrained validation data sets. These gene signatures were able to classify clinicopathologically ambiguous rashes with diagnoses consistent with therapeutic response. Thus, we have defined major classes of human inflammatory skin disease at the molecular level and described a quantitative method to classify indeterminate instances of pathologic inflammation. To make this approach accessible to the scientific community, we created a proof-of-principle web interface (RashX), where scientists and clinicians can visualize their patient-level rash scRNA-seq\\u2013derived data in the context of our TH2/TH17 transcriptional framework. Description scRNA-seq of immune cells can distinguish different inflammatory skin diseases and classify indeterminate rashes. Gene interface tool to identify your rash Many chronic skin conditions are caused by imbalances in the immune milieu of the environment, yet many of these changes remain under characterized. Here, Liu and Wang et al. used single-cell profiling of immune cells from skin samples of atopic dermatitis (AD), psoriasis vulgaris (PV), and healthy controls to profile the immune signatures of the different diseases. They found that AD and PV samples had distinct T cell resident memory, innate lymphoid cell, and CD8+ T cell gene signatures that could be validated in an external dataset. Using these signatures, they created a tool that could classify previously clinicopathologically indeterminate rash samples. This tool is now available as a web resource (RashX) for anyone to use to help further classify skin rashes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=610&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ed0617b66bea86594a66a7d73761b4a359a50420\",\n",
      "        \"title\": \"Immune cells in cardiac repair and regeneration\",\n",
      "        \"abstract\": \"ABSTRACT The immune system is fundamental to tissue homeostasis and is the first line of defense following infection, injury or disease. In the damaged heart, large numbers of immune cells are recruited to the site of injury. These cells play an integral part in both repair by scar formation and the initiation of tissue regeneration. They initially assume inflammatory phenotypes, releasing pro-inflammatory cytokines and removing dead and dying tissue, before entering a reparative stage, replacing dead muscle tissue with a non-contractile scar. In this Review, we present an overview of the innate and adaptive immune response to heart injury. We explore the kinetics of immune cell mobilization following cardiac injury and how the different innate and adaptive immune cells interact with one another and with the damaged tissue. We draw on key findings from regenerative models, providing insight into how to support a robust immune response permissible for cardiac regeneration. Finally, we consider how the latest technological developments can offer opportunities for a deeper and unbiased functional understanding of the immune response to heart disease, highlighting the importance of such knowledge as the basis for promoting regeneration following cardiac injury in human patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=611&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6e1bf980796e64350fa3e0c26fbe7508e9519260\",\n",
      "        \"title\": \"A roadmap for translational cancer glycoimmunology at single cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=612&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"11d1ebe1ab167c33f7d6b67f7cb241322856b90b\",\n",
      "        \"title\": \"A Regularized Multi-Task Learning Approach for Cell Type Detection in Single-Cell RNA Sequencing Data\",\n",
      "        \"abstract\": \"Cell type prediction is one of the most challenging goals in single-cell RNA sequencing (scRNA-seq) data. Existing methods use unsupervised learning to identify signature genes in each cluster, followed by a literature survey to look up those genes for assigning cell types. However, finding potential marker genes in each cluster is cumbersome, which impedes the systematic analysis of single-cell RNA sequencing data. To address this challenge, we proposed a framework based on regularized multi-task learning (RMTL) that enables us to simultaneously learn the subpopulation associated with a particular cell type. Learning the structure of subpopulations is treated as a separate task in the multi-task learner. Regularization is used to modulate the multi-task model (e.g., W 1, W 2, \\u2026 W t ) jointly, according to the specific prior. For validating our model, we trained it with reference data constructed from a single-cell RNA sequencing experiment and applied it to a query dataset. We also predicted completely independent data (the query dataset) from the reference data which are used for training. We have checked the efficacy of the proposed method by comparing it with other state-of-the-art techniques well known for cell type detection. Results revealed that the proposed method performed accurately in detecting the cell type in scRNA-seq data and thus can be utilized as a useful tool in the scRNA-seq pipeline.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=613&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"faf3dd31b4a17e66eb057bd3c7f7b1fe8fb7babb\",\n",
      "        \"title\": \"scTAM-seq enables targeted high-confidence analysis of DNA methylation in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=614&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ee6bc40c9b06d62385d1d1f53b60b22f5288f8de\",\n",
      "        \"title\": \"CyTOF-Lec: Single-cell Glycomics Analysis Reveals Glycan Features Defining Cells Differentially Susceptible to HIV\",\n",
      "        \"abstract\": \"High-parameter single-cell phenotyping has enabled in-depth classification and interrogation of immune cells, but to date has not allowed for glycan characterization. Here, we develop CyTOF-Lec as an approach to simultaneously characterize many protein and glycan features of human immune cells at the single-cell level. We implemented CyTOF-Lec to compare glycan features between different immune subsets from blood and multiple tissue compartments, and to characterize HIV-infected cell cultures. Using bioinformatics approaches to distinguish preferential infection of cellular subsets from viral-induced remodeling, we demonstrate that HIV upregulates the levels of cell surface fucose and sialic acid in a cell- intrinsic manner, and that memory CD4+ T cells co-expressing high levels of fucose and sialic acid are highly susceptible to HIV infection. Sialic acid levels were found to distinguish memory CD4+ T cell subsets expressing different amounts of viral entry receptors, pro-survival factors, homing receptors, and activation markers, and to play a direct role in memory CD4+ T cells\\u2019 susceptibility to HIV infection. The ability of sialic acid to distinguish memory CD4+ T cells with different susceptibilities to HIV infection was experimentally validated through sorting experiments. Together, these results suggest that HIV remodels not only cellular proteins but also glycans, and that glycan expression can differentiate memory CD4+ T cells with vastly different susceptibility to HIV infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=615&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8de25fbbd2120053741bc9072f40b510ace612fe\",\n",
      "        \"title\": \"A Toolkit for Profiling the Immune Landscape of Pediatric Central Nervous System Malignancies\",\n",
      "        \"abstract\": \"The prognosis of pediatric central nervous system (CNS) malignancies remains dismal due to limited treatment options, resulting in high mortality rates and long-term morbidities. Immunotherapies, including checkpoint inhibition, cancer vaccines, engineered T cell therapies, and oncolytic viruses, have promising results in some hematological and solid malignancies, and are being investigated in clinical trials for various high-grade CNS malignancies. However, the role of the tumor immune microenvironment (TIME) in CNS malignancies is mostly unknown for pediatric cases. In order to successfully implement immunotherapies and to eventually predict which patients would benefit from such treatments, in-depth characterization of the TIME at diagnosis and throughout treatment is essential. In this review, we provide an overview of techniques for immune profiling of CNS malignancies, and detail how they can be utilized for different tissue types and studies. These techniques include immunohistochemistry and flow cytometry for quantifying and phenotyping the infiltrating immune cells, bulk and single-cell transcriptomics for describing the implicated immunological pathways, as well as functional assays. Finally, we aim to describe the potential benefits of evaluating other compartments of the immune system implicated by cancer therapies, such as cerebrospinal fluid and blood, and how such liquid biopsies are informative when designing immune monitoring studies. Understanding and uniformly evaluating the TIME and immune landscape of pediatric CNS malignancies will be essential to eventually integrate immunotherapy into clinical practice.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=616&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"93c317893a71752704b24953cea4696507a975db\",\n",
      "        \"title\": \"Single-cell analysis of S\\u00e9zary syndrome reveals novel markers and shifting gene profiles associated with treatment\",\n",
      "        \"abstract\": \"Key Points \\u2022 Integrated clonotype and mRNA expression levels in 6 patients, identifying common and divergent transcriptional states.\\u2022 Characterization of transcriptional changes with progression on photopheresis and vorinostat.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=617&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"25c65fe2668639fd933afe61dbb01ebced2b42b2\",\n",
      "        \"title\": \"MIMAS: microfluidic platform in tandem with MALDI mass spectrometry for protein quantification from small cell ensembles\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=618&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2141ab045e99da496efea859993b162c16d0346c\",\n",
      "        \"title\": \"Glutathione-dependent redox balance characterizes the distinct metabolic properties of follicular and marginal zone B cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=619&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e17e78e81824d868664325815be2180ba5a13c7e\",\n",
      "        \"title\": \"Integrated single-cell (phospho-)protein and RNA detection uncovers phenotypic characteristics of human antibody secreting cells\",\n",
      "        \"abstract\": \"Antibody-secreting cells (ASCs) secrete IgM, IgA, or IgG antibodies and are key components of humoral immunity; however, little is known about unique characteristics of the Ig-classes due to limited availability of material and challenges to quantify many intracellular molecular modalities at a single-cell resolution. We combined a method to in vitro differentiate peripheral B-cells into ASCs with integrated multi-omic single-cell sequencing technologies to quantify subclass-specific hallmark surface markers, transcriptional profiles and signaling transduction pathway components. Our approach detected differential expression of plasmablast and plasma cell markers, homing receptors and IL-2, IL-6, JAK/STAT and mTOR signaling activity across Ig-subclasses. Taken together, our integrated multi-omics approach allowed high-resolution phenotypic characterization of single cells in a complex sample of in vitro differentiated human ASCs. Our strategy is expected to further our understanding of human ASCs in healthy and diseased samples and provide a valuable tool to identify novel biomarkers and potential drug targets. Teaser Integrated single-cell analysis allows tri-modal phenotypic analysis of in-vitro generated human antibody-secreting cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=620&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"618f633aa98ff5213878334702bd97a23286bfff\",\n",
      "        \"title\": \"Host cell maturation modulates parasite invasion and sexual differentiation in Plasmodium berghei\",\n",
      "        \"abstract\": \"Malaria remains a global health problem causing more than 400,000 deaths annually. Plasmodium parasites, the causative agents of malaria, replicate asexually in red blood cells (RBCs) of their vertebrate host, while a subset differentiates into sexual stages (gametocytes) for mosquito transmission. Parasite replication and gametocyte maturation in the erythropoietic niches of the bone marrow and spleen contribute to pathogenesis and drive transmission, but the mechanisms underlying this organ enrichment remain unknown. Here, we performed a comprehensive analysis of rodent P. berghei infection by flow cytometry and single-cell RNA sequencing. We identified CD71 as a host receptor for reticulocyte invasion and found that parasites metabolically adapt to the host cell environment. Transcriptional analysis and functional assays further revealed a nutrient-dependent tropism for gametocyte formation in reticulocytes. Together, we provide a thorough characterization of host-parasite interactions in erythropoietic niches and define host cell maturation state as the key driver of parasite adaptation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=621&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ba2dc451e998069a6bbd2fdddc45d374830956a0\",\n",
      "        \"title\": \"Proteomics approaches to characterize the immune responses in cancer.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=622&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3d98dc18a65484fd41e731987730f8350c69260a\",\n",
      "        \"title\": \"The Role of Osteopontin in Microglia Biology: Current Concepts and Future Perspectives\",\n",
      "        \"abstract\": \"The innate immune landscape of the central nervous system (CNS), including the brain and the retina, consists of different myeloid cell populations with distinct tasks to fulfill. Whereas the CNS borders harbor extraparenchymal CNS-associated macrophages whose main duty is to build up a defense against invading pathogens and other damaging factors from the periphery, the resident immune cells of the CNS parenchyma and the retina, microglia, are highly dynamic cells with a plethora of functions during homeostasis and disease. Therefore, microglia are constantly sensing their environment and closely interacting with surrounding cells, which is in part mediated by soluble factors. One of these factors is Osteopontin (OPN), a multifunctional protein that is produced by different cell types in the CNS, including microglia, and is upregulated in neurodegenerative and neuroinflammatory conditions. In this review, we discuss the current literature about the interaction between microglia and OPN in homeostasis and several disease entities, including multiple sclerosis (MS), Alzheimer\\u2019s and cerebrovascular diseases (AD, CVD), amyotrophic lateral sclerosis (ALS), age-related macular degeneration (AMD) and diabetic retinopathy (DR), in the context of the molecular pathways involved in OPN signaling shaping the function of microglia. As nearly all CNS diseases are characterized by pathological alterations in microglial cells, accompanied by the disturbance of the homeostatic microglia phenotype, the emergence of disease-associated microglia (DAM) states and their interplay with factors shaping the DAM-signature, such as OPN, is of great interest for therapeutical interventions in the future.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=623&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"00105f013232318a8bb90647558d516a5552cfb8\",\n",
      "        \"title\": \"cyCombine allows for robust integration of single-cell cytometry datasets within and across technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=624&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1dd20942e7224a5952ce4e2d18760b9c708d9170\",\n",
      "        \"title\": \"Dntt expression reveals developmental hierarchy and lineage specification of hematopoietic progenitors\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=625&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ef1d6f6601e7dfd7c6bbeb0aa2f43b93268fb2ea\",\n",
      "        \"title\": \"Maximizing the Anti-tumor Potential of Immune Checkpoint Blockade through Modulation of Myeloid-specific CXCL16 and STAT1 Signaling\",\n",
      "        \"abstract\": \"Sensitivity to immune checkpoint blockades (ICB) depends on the overall balance of immunogenic and immunosuppressive signals in the tumor immune microenvironment (TIME). Chemotherapy as an immunostimulatory strategy showed potential in improving ICB\\u2019s clinical efficacy. Yet, evolution of highly plastic tumor-associated myeloid cells hinders ICB\\u2019s potential to reach its full therapeutic potential. In this study, we leveraged single-cell transcriptomic and trajectory analyses to delineate TIME dynamics after chemotherapy priming. We found that metronomic chemotherapy (MCT) treatment led to an accelerated T cell exhaustion through CXCL16-mediated recruitment of peripheral immature myeloid cells and expansion of STAT1-driven PD-L1 expressing myeloid cells. Inhibiting STAT1 signaling in MCT-primed breast cancer relieved T cell exhaustion and significantly enhanced the efficacy of anti-PD-1 ICB treatment. Our study leveraged single-cell analyses to dissect the dynamics of breast cancer TIME and provides a pre-clinical rationale to translate the anti-STAT1 plus anti-PD-1 combinatorial immunotherapy regimen to maximize ICB\\u2019s efficacy. Manuscript Summary Single-cell analyses on low dose chemotherapy primed breast tumor-associated immune cells demonstrates a parallel coexistence of immunogenic and immunosuppressive myeloid cell subsets. Modulating STAT1 signaling in the tumor microenvironment fine-tunes immunogenic and immunosuppressive balance and maximizes the anti-PD-1 immunotherapy efficacy in chemotherapy-primed breast cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=626&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"70ced753d1fdb8fcf3c4f38a51d8f8fbe014ae72\",\n",
      "        \"title\": \"Rewiring mitochondrial metabolism to counteract exhaustion of CAR-T cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=627&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"91dfe1c867881d707f6230dbe0f08f3eb41d0b68\",\n",
      "        \"title\": \"Neoadjuvant clinical trials provide a window of opportunity for cancer drug discovery\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=628&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0f7c26e2b012ed3f8236191f31c0428ab68cae0e\",\n",
      "        \"title\": \"Probabilistic machine learning ensures accurate ambient denoising in droplet-based single-cell omics\",\n",
      "        \"abstract\": \"Droplet-based single-cell omics, including single-cell RNA sequencing (scRNAseq), single-cell CRISPR perturbations (e.g., CROP-seq), and single-cell protein and transcriptomic profiling (CITE-seq) hold great promise for comprehensive cell profiling and genetic screening at the single-cell resolution. However, these technologies suffer from substantial noise, among which ambient signals present in the cell suspension may be the predominant source. Current models to address this issue are highly technology-specific and relatively scRNAseq-centric. while a universal model to describe the noise across these technologies may reveal this common source, improving the denoising accuracy. To this end, we explicitly examined these unexpected signals in multiple datasets across droplet-based technologies, summarised a predictable pattern, and developed single-cell Ambient Remover (scAR) \\u2013 a hypothesis-driven machine learning model to predict and remove ambient signals (including mRNA counts, protein counts, and sgRNA counts) at the molecular level. We benchmarked scAR on three technologies \\u2013 single-cell CRISPR screens, CITE-seq, and scRNAseq along with the state-of-the-art single-technology-specific approaches. scAR showed high denoising accuracy for each type of dataset.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=629&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"756fa5ec1f04461ec67ea5928eadcd9afe51ecab\",\n",
      "        \"title\": \"New insights into Human Hematopoietic Stem and Progenitor Cells via Single-Cell Omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=630&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"131f5a4b82dedd747b59537180fb456a43b690bb\",\n",
      "        \"title\": \"Mass Cytometric Analysis of Early-Stage Mycosis Fungoides\",\n",
      "        \"abstract\": \"Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma. Early-stage disease is characterized by superficial infiltrates of small- to medium-sized atypical epidermotropic T lymphocytes that are clonal related. Nevertheless, the percentage of atypical T cells is low with many admixed reactive immune cells. Despite earlier studies, the composition and spatial characteristics of the cutaneous lymphocytic infiltrate has been incompletely characterized. Here, we applied mass cytometry to profile the immune system in skin biopsies of patients with early-stage MF and in normal skin from healthy individuals. Single-cell suspensions were prepared and labeled with a 43-antibody panel, and data were acquired on a Helios mass cytometer. Unbiased hierarchical clustering of the data identified the major immune lineages and heterogeneity therein. This revealed patient-unique cell clusters in both the CD4+ and myeloid cell compartments but also phenotypically distinct cell clusters that were shared by most patients. To characterize the immune compartment in the tissue context, we developed a 36-antibody panel and performed imaging mass cytometry on MF skin tissue. This visualized the structure of MF skin and the distribution of CD4+ T cells, regulatory T cells, CD8+ T cells, malignant T cells, and various myeloid cell subsets. We observed clusters of CD4+ T cells and multiple types of dendritic cells (DCs) identified through differential expression of CD11c, CD1a, and CD1c in the dermis. These results indicated substantial heterogeneity in the composition of the local immune infiltrate but suggest a prominent role for clustered CD4\\u2013DC interactions in disease pathogenesis. Probably, the local inhibition of such interactions may constitute an efficient treatment modality.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=631&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f60aad44dfc39b7da0ee0372abf1c5fa2090c70\",\n",
      "        \"title\": \"Linking the genotypes and phenotypes of cancer cells in heterogenous populations via real-time optical tagging and image analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=632&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"af5d5242c57202de9f12501d8a7a20677e9835ef\",\n",
      "        \"title\": \"Mapping the human kidney using single-cell genomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=633&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"619b88045131579be6c3b041133974062d060e5a\",\n",
      "        \"title\": \"A genome-scale screen for synthetic drivers of T cell proliferation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=634&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4aae1f1b4820a3fc0425a1093c924b9e26ce37a\",\n",
      "        \"title\": \"Circulating tumour cells in the -omics era: how far are we from achieving the \\u2018singularity\\u2019?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=635&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f6695c530378ec6bbcde138abd19ee8b5eeaef00\",\n",
      "        \"title\": \"Single-cell temporal analysis of natural dengue infection reveals skin-homing lymphocyte expansion one day before defervescence\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=636&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c2cdc09708307e75396530bb76625a0a3e88a32\",\n",
      "        \"title\": \"Single\\u2010cell RNA sequencing technologies and applications: A brief overview\",\n",
      "        \"abstract\": \"Abstract Single\\u2010cell RNA sequencing (scRNA\\u2010seq) technology has become the state\\u2010of\\u2010the\\u2010art approach for unravelling the heterogeneity and complexity of RNA transcripts within individual cells, as well as revealing the composition of different cell types and functions within highly organized tissues/organs/organisms. Since its first discovery in 2009, studies based on scRNA\\u2010seq provide massive information across different fields making exciting new discoveries in better understanding the composition and interaction of cells within humans, model animals and plants. In this review, we provide a concise overview about the scRNA\\u2010seq technology, experimental and computational procedures for transforming the biological and molecular processes into computational and statistical data. We also provide an explanation of the key technological steps in implementing the technology. We highlight a few examples on how scRNA\\u2010seq can provide unique information for better understanding health and diseases. One important application of the scRNA\\u2010seq technology is to build a better and high\\u2010resolution catalogue of cells in all living organism, commonly known as atlas, which is key resource to better understand and provide a solution in treating diseases. While great promises have been demonstrated with the technology in all areas, we further highlight a few remaining challenges to be overcome and its great potentials in transforming current protocols in disease diagnosis and treatment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=637&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1f522d7f7651af873837928ce8a56be91fe79a3a\",\n",
      "        \"title\": \"Molecular characterization of hypoxanthine guanine phosphoribosyltransferase mutant T cells in human blood: The concept of surrogate selection for immunologically relevant cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=638&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe16cdb647be52fbd0f6fee26995a7efcf57e0e1\",\n",
      "        \"title\": \"Secondary influenza challenge triggers resident memory B cell migration and rapid relocation to boost antibody secretion at infected sites\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=639&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"343901e04bb9ed3a35931333b5454d3c42c77d0c\",\n",
      "        \"title\": \"Single-cell immunology: Past, present, and future.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=640&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8bb748abfefbe4403dd184ebaab1f67b68af3acd\",\n",
      "        \"title\": \"Cell-Type Resolved Insights into the Cis-Regulatory Genome of NAFLD\",\n",
      "        \"abstract\": \"The prevalence of non-alcoholic fatty liver disease (NAFLD) is increasing rapidly, and unmet treatment can result in the development of hepatitis, fibrosis, and liver failure. There are difficulties involved in diagnosing NAFLD early and for this reason there are challenges involved in its treatment. Furthermore, no drugs are currently approved to alleviate complications, a fact which highlights the need for further insight into disease mechanisms. NAFLD pathogenesis is associated with complex cellular changes, including hepatocyte steatosis, immune cell infiltration, endothelial dysfunction, hepatic stellate cell activation, and epithelial ductular reaction. Many of these cellular changes are controlled by dramatic changes in gene expression orchestrated by the cis-regulatory genome and associated transcription factors. Thus, to understand disease mechanisms, we need extensive insights into the gene regulatory mechanisms associated with tissue remodeling. Mapping cis-regulatory regions genome-wide is a step towards this objective and several current and emerging technologies allow detection of accessible chromatin and specific histone modifications in enriched cell populations of the liver, as well as in single cells. Here, we discuss recent insights into the cis-regulatory genome in NAFLD both at the organ-level and in specific cell populations of the liver. Moreover, we highlight emerging technologies that enable single-cell resolved analysis of the cis-regulatory genome of the liver.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=641&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"19ef017217fe4bae989c446b08706c08cbe7fb39\",\n",
      "        \"title\": \"Precision medicine in rheumatoid arthritis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=642&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f1f547ded9555f5068a1842387247519b81f7166\",\n",
      "        \"title\": \"Are batch effects still relevant in the age of big data?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=643&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ed8e39a05af9a8ea95e9a129335cb794f07f616f\",\n",
      "        \"title\": \"Applications of Single-Cell Sequencing Technology to the Enteric Nervous System\",\n",
      "        \"abstract\": \"With recent technical advances and diminishing sequencing costs, single-cell sequencing modalities have become commonplace. These tools permit analysis of RNA expression, DNA sequence, chromatin structure, and cell surface antigens at single-cell resolution. Simultaneous measurement of numerous parameters can resolve populations including rare cells, thus revealing cellular diversity within organs and permitting lineage reconstruction in developing tissues. Application of these methods to the enteric nervous system has yielded a wealth of data and biological insights. We review recent papers applying single-cell sequencing tools to the nascent neural crest and to the developing and mature enteric nervous system. These studies have shown significant diversity of enteric neurons and glia, suggested paradigms for neuronal specification, and revealed signaling pathways active during development. As technology evolves and multiome techniques combining two or more of transcriptomic, genomic, epigenetic, and proteomic data become prominent, we anticipate these modalities will become commonplace in ENS research and may find a role in diagnostic testing and personalized therapeutics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=644&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a970bb8a8582c5749471e380e162150454eb51a\",\n",
      "        \"title\": \"Dissection of artifactual and confounding glial signatures by single-cell sequencing of mouse and human brain\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=645&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a22fdbc38c28904a3d0ed2f4acba597727d350d0\",\n",
      "        \"title\": \"Dictionary learning for integrative, multimodal and scalable single-cell analysis\",\n",
      "        \"abstract\": \"Mapping single-cell sequencing profiles to comprehensive reference datasets provides a powerful alternative to unsupervised analysis. However, most reference datasets are constructed from single-cell RNA-sequencing data and cannot be used to annotate datasets that do not measure gene expression. Here we introduce \\u2018bridge integration\\u2019, a method to integrate single-cell datasets across modalities using a multiomic dataset as a molecular bridge. Each cell in the multiomic dataset constitutes an element in a \\u2018dictionary\\u2019, which is used to reconstruct unimodal datasets and transform them into a shared space. Our procedure accurately integrates transcriptomic data with independent single-cell measurements of chromatin accessibility, histone modifications, DNA methylation and protein levels. Moreover, we demonstrate how dictionary learning can be combined with sketching techniques to improve computational scalability and harmonize 8.6 million human immune cell profiles from sequencing and mass cytometry experiments. Our approach, implemented in version 5 of our Seurat toolkit ( http://www.satijalab.org/seurat ), broadens the utility of single-cell reference datasets and facilitates comparisons across diverse molecular modalities. Reference mapping is extended beyond scRNA-seq to single-cell epigenetic and proteomic data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=646&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a3e168a0b63b9a66ebcfc0c2d00d371afc8ab9b5\",\n",
      "        \"title\": \"Single-cell atlases: shared and tissue-specific cell types across human organs\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=647&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ab7ed93cc665179b774a7f8acfe305e27d55bf4\",\n",
      "        \"title\": \"Single-Cell Analysis Reveals the Range of Transcriptional States of Circulating Human Neutrophils\",\n",
      "        \"abstract\": \"Neutrophils are the most abundant leukocytes in human blood and are essential components of innate immunity. Until recently, neutrophils were considered homogeneous and transcriptionally inactive cells, but both concepts are being challenged. Single-cell RNA sequencing (scRNA-seq) offers an unbiased view of cells along a continuum of transcriptional states. However, the use of scRNA-seq to characterize neutrophils has proven technically difficult, explaining in part the paucity of published single-cell data on neutrophils. We have found that modifications to the data analysis pipeline, rather than to the existing scRNA-seq chemistries, can significantly increase the detection of human neutrophils in scRNA-seq. We have then applied a modified pipeline to the study of human peripheral blood neutrophils. Our findings indicate that circulating human neutrophils are transcriptionally heterogeneous cells, which can be classified into one of four transcriptional clusters that are reproducible among healthy human subjects. We demonstrate that peripheral blood neutrophils shift from relatively immature (Nh0) cells, through a transitional phenotype (Nh1), into one of two end points defined by either relative transcriptional inactivity (Nh2) or high expression of type I IFN-inducible genes (Nh3). Transitions among states are characterized by the expression of specific transcription factors. By simultaneously measuring surface proteins and intracellular transcripts at the single-cell level, we show that these transcriptional subsets are independent of the canonical surface proteins that are commonly used to define and characterize human neutrophils. These findings provide a new view of human neutrophil heterogeneity, with potential implications for the characterization of neutrophils in health and disease. Key Points Human neutrophils can be classified into distinct subsets based on transcriptomics. A modified analysis pipeline provides enhanced detection of neutrophil transcripts. These subsets cannot be classified using common surface markers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=648&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0447fa504fb12aa69aadb3986b1bbff47dc3e4ce\",\n",
      "        \"title\": \"Quantifying the phenotypic information in mRNA abundance\",\n",
      "        \"abstract\": \"Quantifying the dependency between mRNA abundance and downstream cellular phenotypes is a fundamental open problem in biology. Advances in multimodal single cell measurement technologies provide an opportunity to apply new computational frameworks to dissect the contribution of individual genes and gene combinations to a given phenotype. Using an information theory approach, we analyzed multimodal data of the expression of 83 genes in the Ca2+ signaling network and the dynamic Ca2+ response in the same cell. We found that the overall expression levels of these 83 genes explain approximately 60% of Ca2+ signal entropy. The average contribution of each single gene was 16%, revealing a large degree of redundancy between genes. Using different heuristics we estimated the dependency between the size of a gene set and its information content, revealing that on average a set of 53 genes contains 90% of the information about Ca2+ signaling within the cellular transcriptional state. Our results provide the first direct quantification of information content about complex cellular phenotype that exists in mRNA abundance measurements.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=649&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4a127bd12f1d8c768edd91e90f9be71e57314612\",\n",
      "        \"title\": \"The Intriguing Landscape of Single\\u2010Cell Protein Analysis\",\n",
      "        \"abstract\": \"Profiling protein expression at single\\u2010cell resolution is essential for fundamental biological research (such as cell differentiation and tumor microenvironmental examination) and clinical precision medicine where only a limited number of primary cells are permitted. With the recent advances in engineering, chemistry, and biology, single\\u2010cell protein analysis methods are developed rapidly, which enable high\\u2010throughput and multiplexed protein measurements in thousands of individual cells. In combination with single cell RNA sequencing and mass spectrometry, single\\u2010cell multi\\u2010omics analysis can simultaneously measure multiple modalities including mRNAs, proteins, and metabolites in single cells, and obtain a more comprehensive exploration of cellular signaling processes, such as DNA modifications, chromatin accessibility, protein abundance, and gene perturbation. Here, the recent progress and applications of single\\u2010cell protein analysis technologies in the last decade are summarized. Current limitations, challenges, and possible future directions in this field are also discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=650&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"417dd90071135bb3f464bb932cbf478bbe98ef38\",\n",
      "        \"title\": \"Hematopoietic cell transplantation donor-derived memory-like NK cells functionally persist after transfer into patients with leukemia\",\n",
      "        \"abstract\": \"Natural killer (NK) cells are innate lymphoid cells that eliminate cancer cells, produce cytokines, and are being investigated as a nascent cellular immunotherapy. Impaired NK cell function, expansion, and persistence remain key challenges for optimal clinical translation. One promising strategy to overcome these challenges is cytokine-induced memory-like (ML) differentiation, whereby NK cells acquire enhanced antitumor function after stimulation with interleukin-12 (IL-12), IL-15, and IL-18. Here, reduced-intensity conditioning (RIC) for HLA-haploidentical hematopoietic cell transplantation (HCT) was augmented with same-donor ML NK cells on day +7 and 3 weeks of N-803 (IL-15 superagonist) to treat patients with relapsed/refractory acute myeloid leukemia (AML) in a clinical trial (NCT02782546). In 15 patients, donor ML NK cells were well tolerated, and 87% of patients achieved a composite complete response at day +28, which corresponded with clearing high-risk mutations, including TP53 variants. NK cells were the major blood lymphocytes for 2 months after HCT with 1104-fold expansion (over 1 to 2 weeks). Phenotypic and transcriptional analyses identified donor ML NK cells as distinct from conventional NK cells and showed that ML NK cells persisted for over 2 months. ML NK cells expressed CD16, CD57, and high granzyme B and perforin, along with a unique transcription factor profile. ML NK cells differentiated in patients had enhanced ex vivo function compared to conventional NK cells from both patients and healthy donors. Overall, same-donor ML NK cell therapy with 3 weeks of N-803 support safely augmented RIC haplo-HCT for AML. Description Same-donor memory-like NK cells persist for months and help drive remission in patients with leukemia after hematopoietic cell transplantation. Naturally killing leukemia Natural killer (NK) cells are innate immune cells that are under investigation as a cell therapy for multiple types of cancer. A challenge for NK cell\\u2013based therapies, however, is ensuring that the cells persist and remain functional long-term. Here, Berrien-Elliott et al. generated cytokine-induced memory-like NK (ML NK) cells as a therapeutic for leukemia and investigated their functional capacity and persistence in patients receiving HLA-haploidentical hematopoietic cell transplantation (HCT). The authors showed that ML NK cells derived from the same donor as the HCT persisted for at least 2 months after HCT and were highly functional ex vivo. Together, these findings support the use of ML NK cells as a component of HCT for leukemia.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=651&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f12d768e952fa4ab6c3bce5743b25376ffa4ea82\",\n",
      "        \"title\": \"Single\\u2010cell transcriptomics in human skin research: available technologies, technical considerations and disease applications\",\n",
      "        \"abstract\": \"Single\\u2010cell technologies have revolutionized research in the last decade, including for skin biology. Single\\u2010cell RNA sequencing has emerged as a powerful tool allowing the dissection of human disease pathophysiology at unprecedented resolution by assessing cell\\u2010to\\u2010cell variation, facilitating identification of rare cell populations and elucidating cellular heterogeneity. In dermatology, this technology has been widely applied to inflammatory skin disorders, fibrotic skin diseases, wound healing complications and cutaneous neoplasms. Here, we discuss the available technologies and technical considerations of single\\u2010cell RNA sequencing and describe its applications to a broad spectrum of dermatological diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=652&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"999a46d38cb6d637d268415c892cf22aa8bcdb6c\",\n",
      "        \"title\": \"Computational exploration of cellular communication in skin from emerging single-cell and spatial transcriptomic data\",\n",
      "        \"abstract\": \"Tissue development and homeostasis require coordinated cell\\u2013cell communication. Recent advances in single-cell sequencing technologies have emerged as a revolutionary method to reveal cellular heterogeneity with unprecedented resolution. This offers a great opportunity to explore cell\\u2013cell communication in tissues systematically and comprehensively, and to further identify signaling mechanisms driving cell fate decisions and shaping tissue phenotypes. Using gene expression information from single-cell transcriptomics, several computational tools have been developed for inferring cell\\u2013cell communication, greatly facilitating analysis and interpretation. However, in single-cell transcriptomics, spatial information of cells is inherently lost. Given that most cell signaling events occur within a limited distance in tissues, incorporating spatial information into cell\\u2013cell communication analysis is critical for understanding tissue organization and function. Spatial transcriptomics provides spatial location of cell subsets along with their gene expression, leading to new directions for leveraging spatial information to develop computational approaches for cell\\u2013cell communication inference and analysis. These computational approaches have been successfully applied to uncover previously unrecognized mechanisms of intercellular communication within various contexts and across organ systems, including the skin, a formidable model to study mechanisms of cell\\u2013cell communication due to the complex interactions between the different cell populations that comprise it. Here, we review emergent cell\\u2013cell communication inference tools using single-cell transcriptomics and spatial transcriptomics, and highlight the biological insights gained by applying these computational tools to exploring cellular communication in skin development, homeostasis, disease and aging, as well as discuss future potential research avenues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=653&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f04d4b18bfe37583b8cc849a830ee1efefbc12b7\",\n",
      "        \"title\": \"An IL-9\\u2013pulmonary macrophage axis defines the allergic lung inflammatory environment\",\n",
      "        \"abstract\": \"Despite IL-9 functioning as a pleiotropic cytokine in mucosal environments, the IL-9\\u2013responsive cell repertoire is still not well defined. Here, we found that IL-9 mediates proallergic activities in the lungs by targeting lung macrophages. IL-9 inhibits alveolar macrophage expansion and promotes recruitment of monocytes that develop into CD11c+ and CD11c\\u2212 interstitial macrophage populations. Interstitial macrophages were required for IL-9\\u2013dependent allergic responses. Mechanistically, IL-9 affected the function of lung macrophages by inducing Arg1 activity. Compared with Arg1-deficient lung macrophages, Arg1-expressing macrophages expressed greater amounts of CCL5. Adoptive transfer of Arg1+ lung macrophages but not Arg1\\u2212 lung macrophages promoted allergic inflammation that Il9r\\u2212/\\u2212 mice were protected against. In parallel, the elevated expression of IL-9, IL-9R, Arg1, and CCL5 was correlated with disease in patients with asthma. Thus, our study uncovers an IL-9/macrophage/Arg1 axis as a potential therapeutic target for allergic airway inflammation. Description IL-9 promotes allergic inflammation by altering the heterogeneity and function of lung macrophages. Macrophages amplify allergic airway inflammation Interleukin-9 (IL-9) can promote type 2 lung inflammation during allergic responses to inhaled allergens, but the direct cellular targets of IL-9 are not well characterized. Fu et al. demonstrate that pulmonary macrophages are a major population of immune cells responding to IL-9 produced in mouse models of allergic airway disease. Mice deficient in the IL-9 receptor had impaired expansion of monocyte-derived interstitial macrophages that promoted airway inflammation dependent on arginase expression. IL-9 signaling also promoted macrophage production of the eosinophil-attracting chemokine CCL5, which was elevated in the serum of patients with asthma and correlated with IL-9 levels. These results identify lung macrophages as key cellular targets of IL-9 during allergic disease that subsequently amplify type 2 inflammatory responses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=654&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e6c0565b1ae39d725d384f76c7093775ba870361\",\n",
      "        \"title\": \"Deciphering endothelial heterogeneity in health and disease at single-cell resolution: progress and perspectives\",\n",
      "        \"abstract\": \"Abstract Endothelial cells (ECs) constitute the inner lining of vascular beds in mammals and are crucial for homeostatic regulation of blood vessel physiology, but also play a key role in pathogenesis of many diseases, thereby representing realistic therapeutic targets. However, it has become evident that ECs are heterogeneous, encompassing several subtypes with distinct functions, which makes EC targeting and modulation in diseases challenging. The rise of the new single-cell era has led to an emergence of studies aimed at interrogating transcriptome diversity along the vascular tree, and has revolutionized our understanding of EC heterogeneity from both a physiological and pathophysiological context. Here, we discuss recent landmark studies aimed at teasing apart the heterogeneous nature of ECs. We cover driving (epi)genetic, transcriptomic, and metabolic forces underlying EC heterogeneity in health and disease, as well as current strategies used to combat disease-enriched EC phenotypes, and propose strategies to transcend largely descriptive heterogeneity towards prioritization and functional validation of therapeutically targetable drivers of EC diversity. Lastly, we provide an overview of the most recent advances and hurdles in single EC OMICs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=655&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8da0516e4ff5ee22c86bc7d95c9d79c6135f52b3\",\n",
      "        \"title\": \"Immunopathological signatures in multisystem inflammatory syndrome in children and pediatric COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=656&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5086889ebdfe56ff6f72f8a29cb39c5fa6ecc8c\",\n",
      "        \"title\": \"Optimising protein detection with fixable custom oligo\\u2010labelled antibodies for single\\u2010cell multi\\u2010omics approaches\",\n",
      "        \"abstract\": \"Single\\u2010cell RNA sequencing (scRNA\\u2010seq) is a powerful method utilising transcriptomic data for detailed characterisation of heterogeneous cell populations. The use of oligonucleotide\\u2010labelled antibodies for targeted proteomics addresses the shortcomings of the scRNA\\u2010seq\\u2010only based approach by improving detection of low expressing targets. However, optimisation of large antibody panels is challenging and depends on the availability of co\\u2010functioning oligonucleotide\\u2010labelled antibodies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=657&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"df907c913112ca930457ce1070a9c87eac6474bb\",\n",
      "        \"title\": \"Recent advances in spatially resolved transcriptomics: challenges and opportunities\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) has greatly advanced our understanding of cellular heterogeneity by profiling individual cell transcriptomes. However, cell dissociation from the tissue structure causes a loss of spatial information, which hinders the identification of intercellular communication networks and global transcriptional patterns present in the tissue architecture. To overcome this limitation, novel transcriptomic platforms that preserve spatial information have been actively developed. Significant achievements in imaging technologies have enabled in situ targeted transcriptomic profiling in single cells at single-molecule resolution. In addition, technologies based on mRNA capture followed by sequencing have made possible profiling of the genome-wide transcriptome at the 55-100 \\u03bcm resolution. Unfortunately, neither imaging-based technology nor capture-based method elucidates a complete picture of the spatial transcriptome in a tissue. Therefore, addressing specific biological questions requires balancing experimental throughput and spatial resolution, mandating the efforts to develop computational algorithms that are pivotal to circumvent technology-specific limitations. In this review, we focus on the current state-of-the-art spatially resolved transcriptomic technologies, describe their applications in a variety of biological domains, and explore recent discoveries demonstrating their enormous potential in biomedical research. We further highlight novel integrative computational methodologies with other data modalities that provide a framework to derive biological insight into heterogeneous and complex tissue organization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=658&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"06584b66a6ccc4cceab58c6d46b007b7f383c3c9\",\n",
      "        \"title\": \"A unified computational framework for single-cell data integration with optimal transport\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=659&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0d21f20829645f91f58e0306210ab7693ab4dc79\",\n",
      "        \"title\": \"Cell specific peripheral immune responses predict survival in critical COVID-19 patients\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=660&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4cc2737fa7dd1c16dafc976aa0e131dc54bcc629\",\n",
      "        \"title\": \"Proteomics technologies for cancer liquid biopsies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=661&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fde3bc9aeed4cb237d67016f715b1cd63a619fd3\",\n",
      "        \"title\": \"Cell specific peripheral immune responses predict survival in critical COVID-19 patients\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=662&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c068182e85c540843b9ec8189bde1b47771841b8\",\n",
      "        \"title\": \"STAT3 gain-of-function mutations connect leukemia with autoimmune disease by pathological dysregulation of NKG2Dhi CD8 killer T cells\",\n",
      "        \"abstract\": \"The association between cancer and autoimmune disease is unexplained, exemplified by T-cell large granular lymphocytic leukemia (T-LGL) where gain-of-function somatic mutations in STAT3 correlate with co-existing autoimmunity. To resolve whether these mutations are the cause or consequence of CD8 clonal expansions and autoimmunity, here we analyse patients with germline STAT3 GOF syndrome and mice with the T-LGL mutation STAT3K658Nor the most common germline mutation, STAT3T716M. STAT3 GOF mutations drove accumulation of effector CD8 T cell clones highly expressing the NKG2D receptor for MHC-I-related molecules expressed on stressed cells, and the genes for inflammatory/cytotoxic granzymes, perforin, interferon-\\u03b3 and Ccl5/Rantes. CD8 cells were essential to lethal disease in Stat3K658Nmice and their accumulation required NKG2D and the receptor for IL-15 and IL-2, IL2RB. These results demonstrate that STAT3 GOF mutations cause effector CD8 T cell oligoclonal accumulation and that these rogue T cells contribute to autoimmune pathology, supporting the hypothesis that somatic mutations in leukemia/lymphoma driver genes contribute to autoimmune disease. IN BRIEF Leukemia and autoimmune-associated STAT3 gain-of-function mutations dysregulate CD8 T cells to cause autoimmune pathology and oligoclonal expansion of cytotoxic killer CD8 T cells, that depend upon NKG2D and IL2RB receptors for signals displayed on stressed, damaged, infected, or mutated tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=663&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c44549982f8eb544e1a1faccb19f7d7ec8f4dea1\",\n",
      "        \"title\": \"SM-Omics is an automated platform for high-throughput spatial multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=664&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ff94f336b36c4d0e84c40e53686ebeb159f42cb\",\n",
      "        \"title\": \"A Final Frontier in Environment-Genome Interactions? Integrated, Multi-Omic Approaches to Predictions of Non-Communicable Disease Risk\",\n",
      "        \"abstract\": \"Epidemiological and associative research from humans and animals identifies correlations between the environment and health impacts. The environment\\u2014health inter-relationship is effected through an individual\\u2019s underlying genetic variation and mediated by mechanisms that include the changes to gene regulation that are associated with the diversity of phenotypes we exhibit. However, the causal relationships have yet to be established, in part because the associations are reduced to individual interactions and the combinatorial effects are rarely studied. This problem is exacerbated by the fact that our genomes are highly dynamic; they integrate information across multiple levels (from linear sequence, to structural organisation, to temporal variation) each of which is open to and responds to environmental influence. To unravel the complexities of the genomic basis of human disease, and in particular non-communicable diseases that are also influenced by the environment (e.g., obesity, type II diabetes, cancer, multiple sclerosis, some neurodegenerative diseases, inflammatory bowel disease, rheumatoid arthritis) it is imperative that we fully integrate multiple layers of genomic data. Here we review current progress in integrated genomic data analysis, and discuss cases where data integration would lead to significant advances in our ability to predict how the environment may impact on our health. We also outline limitations which should form the basis of future research questions. In so doing, this review will lay the foundations for future research into the impact of the environment on our health.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=665&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d0e41b4d90c9b4ad083d99c37e36e8314e897dc1\",\n",
      "        \"title\": \"Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=666&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0266a3c3d2f6bb10096212f8423b04a0cd29646f\",\n",
      "        \"title\": \"Biologically informed deep learning to infer gene program activity in single cells\",\n",
      "        \"abstract\": \"The increasing availability of large-scale single-cell datasets has enabled the detailed description of cell states across multiple biological conditions and perturbations. In parallel, recent advances in unsupervised machine learning, particularly in transfer learning, have enabled fast and scalable mapping of these new single-cell datasets onto reference atlases. The resulting large-scale machine learning models however often have millions of parameters, rendering interpretation of the newly mapped datasets challenging. Here, we propose expiMap, a deep learning model that enables interpretable reference mapping using biologically understandable entities, such as curated sets of genes and gene programs. The key concept is the substitution of the uninterpretable nodes in an autoencoder\\u2019s bottleneck by labeled nodes mapping to interpretable lists of genes, such as gene ontologies, biological pathways, or curated gene sets, for which activities are learned as constraints during reconstruction. This is enabled by the incorporation of predefined gene programs into the reference model, and at the same time allowing the model to learn de novo new programs and refine existing programs durin reference mapping. We show that the model retains similar integration performance as existing methods while providing a biologically interpretable framework for understanding cellular behavior. We demonstrate the capabilities of expiMap by applying it to 15 datasets encompassing five different tissues and species. The interpretable nature of the mapping revealed unreported associations between interferon signaling via the RIG-I/MDA5 and GPCRs pathways, with differential behavior in CD8+ T cells and CD14+ monocytes in severe COVID-19, as well as the role of annexins in the cellular communications between lymphoid and myeloid compartments for explaining patient response to the applied drugs. Finally, expiMap enabled the direct comparison of a diverse set of pancreatic beta cells from multiple studies where we observed a strong, previously unreported correlation between the unfolded protein response and asparagine N-linked glycosylation. Altogether, expiMap enables the interpretable mapping of single cell transcriptome data sets across cohorts, disease states and other perturbations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=667&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7466a7cb2eb4aa0081b3fc94fe5c5f232730e4bc\",\n",
      "        \"title\": \"Efficient combinatorial targeting of RNA transcripts in single cells with Cas13 RNA Perturb-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=668&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7e1b849349c6daf8f5e69e4ca15d5997bdbe7986\",\n",
      "        \"title\": \"Single-Cell Technologies to Decipher the Immune Microenvironment in Myeloid Neoplasms: Perspectives and Opportunities\",\n",
      "        \"abstract\": \"Myeloid neoplasms (MN) are heterogeneous clonal disorders arising from the expansion of hematopoietic stem and progenitor cells. In parallel with genetic and epigenetic dynamics, the immune system plays a critical role in modulating tumorigenesis, evolution and therapeutic resistance at the various stages of disease progression. Single-cell technologies represent powerful tools to assess the cellular composition of the complex tumor ecosystem and its immune environment, to dissect interactions between neoplastic and non-neoplastic components, and to decipher their functional heterogeneity and plasticity. In addition, recent progress in multi-omics approaches provide an unprecedented opportunity to study multiple molecular layers (DNA, RNA, proteins) at the level of single-cell or single cellular clones during disease evolution or in response to therapy. Applying single-cell technologies to MN holds the promise to uncover novel cell subsets or phenotypic states and highlight the connections between clonal evolution and immune escape, which is crucial to fully understand disease progression and therapeutic resistance. This review provides a perspective on the various opportunities and challenges in the field, focusing on key questions in MN research and discussing their translational value, particularly for the development of more efficient immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=669&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd7b49a392875037e65bef185230f2428778c8e7\",\n",
      "        \"title\": \"Single-cell multiomics reveals heterogeneous cell states linked to metastatic potential in liver cancer cell lines\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=670&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6c4ddff52a847f84506f7aa5ed6c85682932fb09\",\n",
      "        \"title\": \"DAISM-DNNXMBD: Highly accurate cell type proportion estimation with in silico data augmentation and deep neural networks\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=671&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f99fcf527de821a6788f4e9747d2105b64646be1\",\n",
      "        \"title\": \"HOXD13 is a direct EWS-FLI1 target and moderates fusion-dependent transcriptional states\",\n",
      "        \"abstract\": \"Oncogenic fusion proteins display exquisite tissue specificity, revealing that malignant transformation requires cooperation with cell-autonomous factors. Recent studies have also demonstrated that tumorigenicity of Ewing sarcoma requires precise regulation of the transcriptional activity of the EWS-FLI1 oncogenic driver. Here we show that the developmentally and anatomically restricted transcription factor HOXD13 is a direct target of EWS-FLI1. Transcriptomic and CUT&RUN studies revealed that HOXD13 binds active, fusion-bound enhancers, resulting in altered expression of EWS-FLI1-induced targets. More strikingly, HOXD13 was found to bind and activate cis-regulatory regions of genes that are normally repressed by EWS-FLI1. Single-cell sequencing demonstrated marked intra-tumoral heterogeneity of HOXD13 transcriptional activity and revealed that antagonism between HOXD13-mediated gene activation and EWS-FLI1-dependent gene repression confers a spectrum of transcriptional cell states along a mesenchymal axis. Thus, HOXD13 serves as an internal rheostat for EWS-FLI1 activity, providing a paradigm for tissue-specific transcription factors as critical partners in fusion-driven cancers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=672&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4972f1554856489677afeb17e60fce6c04bedfeb\",\n",
      "        \"title\": \"A Long Journey before Cycling: Regulation of Quiescence Exit in Adult Muscle Satellite Cells\",\n",
      "        \"abstract\": \"Skeletal muscle harbors a pool of stem cells called muscle satellite cells (MuSCs) that are mainly responsible for its robust regenerative capacities. Adult satellite cells are mitotically quiescent in uninjured muscles under homeostasis, but they exit quiescence upon injury to re-enter the cell cycle to proliferate. While most of the expanded satellites cells differentiate and fuse to form new myofibers, some undergo self-renewal to replenish the stem cell pool. Specifically, quiescence exit describes the initial transition of MuSCs from quiescence to the first cell cycle, which takes much longer than the time required for subsequent cell cycles and involves drastic changes in cell size, epigenetic and transcriptomic profiles, and metabolic status. It is, therefore, an essential period indispensable for the success of muscle regeneration. Diverse mechanisms exist in MuSCs to regulate quiescence exit. In this review, we summarize key events that occur during quiescence exit in MuSCs and discuss the molecular regulation of this process with an emphasis on multiple levels of intrinsic regulatory mechanisms. A comprehensive understanding of how quiescence exit is regulated will facilitate satellite cell-based muscle regenerative therapies and advance their applications in various disease and aging conditions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=673&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3f43bdc32b8e2434628f316ca2826ebfeb0e0750\",\n",
      "        \"title\": \"PD-1 blockade therapy promotes infiltration of tumor-attacking exhausted T\\u00a0cell clonotypes.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=674&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"be6982936e2e89605bd699ea2e66c599c28b4a1d\",\n",
      "        \"title\": \"Clonal lineage tracing reveals shared origin of conventional and plasmacytoid dendritic cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=675&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"35cd111517ef6af76549e935f76103d0e1c8e103\",\n",
      "        \"title\": \"ScRNA-seq expression of IFI27 and APOC2 identifies four alveolar macrophage superclusters in healthy BALF\",\n",
      "        \"abstract\": \"There are at least 14 AM subtypes; their frequency, along with other immune cells, is highly conserved across individuals, suggesting a specific niche exists for each leukocyte population. Alveolar macrophages (AMs) reside on the luminal surface of the airways and alveoli, ensuring proper gas exchange by ingesting cellular debris and pathogens, and regulating inflammatory responses. Therefore, understanding the heterogeneity and diverse roles played by AMs, interstitial macrophages, and recruited monocytes is critical for treating airway diseases. We performed single-cell RNA sequencing on 113,213 bronchoalveolar lavage cells from four healthy and three uninflamed cystic fibrosis subjects and identified two MARCKS+LGMN+IMs, FOLR2+SELENOP+ and SPP1+PLA2G7+ IMs, monocyte subtypes, DC1, DC2, migDCs, plasmacytoid DCs, lymphocytes, epithelial cells, and four AM superclusters (families) based on the gene expression of IFI27 and APOC2. These four AM families have at least eight distinct functional members (subclusters) named after their differentially expressed gene(s): IGF1, CCL18, CXCL5, cholesterol, chemokine, metallothionein, interferon, and small-cluster AMs. Interestingly, the chemokine cluster further divides with each subcluster selectively expressing a unique combination of chemokines. One of the most striking observations, besides the heterogeneity, is the conservation of AM family members in relatively equal ratio across all AM superclusters and individuals. Transcriptional data and TotalSeq technology were used to investigate cell surface markers that distinguish resident AMs from recruited monocytes. Last, other AM datasets were projected onto our dataset. Similar AM superclusters and functional subclusters were observed, along with a significant increase in chemokine and IFN AM subclusters in individuals infected with COVID-19. Overall, functional specializations of the AM subclusters suggest that there are highly regulated AM niches with defined programming states, highlighting a clear division of labor.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=676&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aed926d10854d4901813603be38d1a5d23f62477\",\n",
      "        \"title\": \"Single-cell sequencing and its applications in bladder cancer\",\n",
      "        \"abstract\": \"Abstract Bladder cancer is the most common malignant tumour of the urinary system that is characterised by significant intra-tumoural heterogeneity. While large-scale sequencing projects have provided a preliminary understanding of tumour heterogeneity, these findings are based on the average signals obtained from the pooled populations of diverse cells. Recent advances in single-cell sequencing (SCS) technologies have been critical in this regard, opening up new ways of understanding the nuanced tumour biology by identifying distinct cellular subpopulations, dissecting the tumour microenvironment, and characterizing cellular genomic mutations. By integrating these novel insights, SCS technologies are expected to make powerful and meaningful changes to the current diagnosis and treatment of bladder cancer through the identification and usage of novel biomarkers as well as targeted therapeutics. SCS can discriminate complex heterogeneity in a large population of tumour cells and determine the key molecular properties that influence clinical outcomes. Here, we review the advances in single-cell technologies and discuss their applications in cancer research and clinical practice, with a specific focus on bladder cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=677&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50051400965f65cb5dddc92766b55e6e6c2b7621\",\n",
      "        \"title\": \"Dual-layered hydrogels allow complete genome recovery with nucleic acid cytometry\",\n",
      "        \"abstract\": \"Targeting specific cells for sequencing is important for applications in cancer, microbiology, and infectious disease. Nucleic acid cytometry (NAC) is a powerful approach for accomplishing this because it allows specific cells to be isolated based on sequence biomarkers that are otherwise impossible to detect. However, existing methods require specialized microfluidic devices, limiting adoption. Here, a modified workflow is described that uses particle\\u2010templated emulsification (PTE) and flow cytometry to conduct the essential steps of cell detection and sorting normally accomplished by microfluidics. Our microfluidic\\u2010free workflow allows facile isolation and sequencing of cells, viruses, and nucleic acids and thus provides a powerful enrichment approach for targeted sequencing applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=678&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1f6c8b0bed0da47921c359c9301aeab30a80f516\",\n",
      "        \"title\": \"Leveraging CD39 To Identify Tumor-Reactive CD8 T cells In Human Lung Cancer\",\n",
      "        \"abstract\": \"CD39 has emerged as a marker of tumor-reactive CD8 T cells. Using single-cell CITE/RNA/TCRseq, we show that CD39+ CD8 T cells in human lung cancers demonstrate transcriptional and proteomic features of exhaustion, tumor reactivity, and clonal expansion. Flow cytometry of 440 lung cancer specimens revealed that CD39 level on CD8 T cells is only weakly correlated with tumoral features, such as histology, driver mutation, PD-L1 and tumor mutation burden. CD8 T cells reactive against EGFR driver mutations preferentially expressed CD39, suggesting that CD39 can be leveraged to identify neoantigen-reactive CD8 T cells. PD-1 axis blockade, but not cytotoxic chemotherapy, increased intratumoral CD39+ CD8 T cells. CD39 correlated with PD-1 expression on CD8 T cells and high pre-treatment/early-on-treatment levels were associated with improved clinical outcomes from immune checkpoint blockade. This extensive and comprehensive profiling of the clinical, pathological and molecular features highlights the utility of CD39 as a proxy for tumor-reactive CD8 T cells in human lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=679&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b69ed561809ac61a95c62e9e81820613efadbe8f\",\n",
      "        \"title\": \"High-speed fluorescence image\\u2013enabled cell sorting\",\n",
      "        \"abstract\": \"Fast and selective isolation of single cells with unique spatial and morphological traits remains a technical challenge. Here, we address this by establishing high-speed image-enabled cell sorting (ICS), which records multicolor fluorescence images and sorts cells based on measurements from image data at speeds up to 15,000 events per second. We show that ICS quantifies cell morphology and localization of labeled proteins and increases the resolution of cell cycle analyses by separating mitotic stages. We combine ICS with CRISPR-pooled screens to identify regulators of the nuclear factor \\u03baB (NF-\\u03baB) pathway, enabling the completion of genome-wide image-based screens in about 9 hours of run time. By assessing complex cellular phenotypes, ICS substantially expands the phenotypic space accessible to cell-sorting applications and pooled genetic screening. Description Sorting cells by intracellular features Fluorescence-activated cell sorting, reported in Science 52 years ago, has revolutionized biomedical research, giving us the ability to isolate cells according to the expression of labeled proteins. So far, however, flow cytometric cell sorting has been blind to spatial processes such as intracellular protein localization, which is traditionally measured using microscopy. Schraivogel et al. combined ultrafast microscopy and image analysis with a flow cytometric cell sorter to unlock spatial phenotypes for high-throughput sorting applications. The authors show how this technology can be used to rapidly isolate cells with complex cellular phenotypes and how it can accelerate genome-wide microscopy-based CRISPR screening. \\u2014DJ A high-speed cell sorter uses fluorescence imaging to enable genome-scale studies of complex phenotypes of cultured human HeLa cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=680&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0e4e6619741403dda5e793d131333c038cf143e5\",\n",
      "        \"title\": \"Single-cell multi-omics reveals dyssynchrony of the innate and adaptive immune system in progressive COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=681&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1fb1023b851d761ee127aeb0261147f98b5e45bc\",\n",
      "        \"title\": \"scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=682&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a30ddd7452acd3fa4b59972567bf70b2912f0d6a\",\n",
      "        \"title\": \"Computational Methods for Single-cell Multi-omics Integration and Alignment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=683&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f7d6f778a9ff7c1eb57287d2b33e2cb83f723b98\",\n",
      "        \"title\": \"Quantifying protein abundance on single cells using split-pool sequencing on DNA-barcoded antibodies for diagnostic applications\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=684&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3beff30d0eab8c239d1c2e5a79f5da74e851dcf9\",\n",
      "        \"title\": \"Single Cell RNA Sequencing in Autoimmune Inflammatory Rheumatic Diseases: Current Applications, Challenges and a Step Toward Precision Medicine\",\n",
      "        \"abstract\": \"Single cell RNA sequencing (scRNA-seq) represents a new large scale and high throughput technique allowing analysis of the whole transcriptome at the resolution of an individual cell. It has emerged as an imperative method in life science research, uncovering complex cellular networks and providing indices that will eventually lead to the development of more targeted and personalized therapies. The importance of scRNA-seq has been particularly highlighted through the analysis of complex biological systems, in which cellular heterogeneity is a key aspect, such as the immune system. Autoimmune inflammatory rheumatic diseases represent a group of disorders, associated with a dysregulated immune system and high patient heterogeneity in both pathophysiological and clinical aspects. This complicates the complete understanding of underlying pathological mechanisms, associated with limited therapeutic options available and their long-term inefficiency and even toxicity. There is an unmet need to investigate, in depth, the cellular and molecular mechanisms driving the pathogenesis of rheumatic diseases and drug resistance, identify novel therapeutic targets, as well as make a step forward in using stratified and informed therapeutic decisions, which could now be achieved with the use of single cell approaches. This review summarizes the current use of scRNA-seq in studying different rheumatic diseases, based on recent findings from published in vitro, in vivo, and clinical studies, as well as discusses the potential implementation of scRNA-seq in the development of precision medicine in rheumatology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=685&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2d6e261da20cfd614c80d3434f440dc3bde5dc47\",\n",
      "        \"title\": \"ISSAAC-seq enables sensitive and flexible multimodal profiling of chromatin accessibility and gene expression in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=686&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f4d3e5102b38245dbfa49c22d42bdbf5d8392122\",\n",
      "        \"title\": \"Computational Methods for Single-Cell Imaging and Omics Data Integration\",\n",
      "        \"abstract\": \"Integrating single cell omics and single cell imaging allows for a more effective characterisation of the underlying mechanisms that drive a phenotype at the tissue level, creating a comprehensive profile at the cellular level. Although the use of imaging data is well established in biomedical research, its primary application has been to observe phenotypes at the tissue or organ level, often using medical imaging techniques such as MRI, CT, and PET. These imaging technologies complement omics-based data in biomedical research because they are helpful for identifying associations between genotype and phenotype, along with functional changes occurring at the tissue level. Single cell imaging can act as an intermediary between these levels. Meanwhile new technologies continue to arrive that can be used to interrogate the genome of single cells and its related omics datasets. As these two areas, single cell imaging and single cell omics, each advance independently with the development of novel techniques, the opportunity to integrate these data types becomes more and more attractive. This review outlines some of the technologies and methods currently available for generating, processing, and analysing single-cell omics- and imaging data, and how they could be integrated to further our understanding of complex biological phenomena like ageing. We include an emphasis on machine learning algorithms because of their ability to identify complex patterns in large multidimensional data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=687&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2822334e4a74daa734c7e2925a29cb78e0eb32d8\",\n",
      "        \"title\": \"Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A R882 mutations perturb early progenitor states through selective hypomethylation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=688&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"389ff00301ee339c6dec2d9bbf45cbaaf07cecc9\",\n",
      "        \"title\": \"Chemokine Receptor Activation Enhances Memory B Cell Class Switching Linked to IgE Sensitization to Alpha Gal and Cardiovascular Disease\",\n",
      "        \"abstract\": \"Background: Recent studies have suggested that IgE sensitization to \\u03b1-gal is associated with coronary artery disease (CAD). However, the B cell subtype(s) responsible for production of IgE to \\u03b1-gal and mechanisms mediating this production remain elusive. Methods: Single cell multi-omics sequencing, was utilized to phenotype B cells obtained from 60 subjects that had undergone coronary angiography in whom serum IgE was evaluated by ImmunoCAP. Bioinformatics approaches were used to identify B cell subtype(s) and transcriptomic signatures associated with \\u03b1-gal sensitization. In vitro characterization of chemokine/chemokine receptor pairs on switched memory B cells associated with IgE to \\u03b1-gal was performed. Results: Of the 60 patients, 17 (28%) were positive for IgE to \\u03b1-gal. CITESeq identified CCR6+ class-switched memory (SWM) B cells and CXCR4 expresssion on these CCR6+ SWM B cells as significantly associated with IgE sensitization to \\u03b1-gal but not to other common allergens (peanut or inhalants). In vitro studies of enriched human B cells revealed significantly greater IgE on SWM B cells with high CCR6 and CXCR4 expression 10 days after cells were treated with IL-4 and CD40 to stimulate class switch recombination. Both CCL20 (CCR6 ligand) and CXCL12 (ligand for CXCR4) increased the expression of IgE on SWM B cells expressing their receptors. However, they appeared to have unique pathways mediating this effect as only CCL20 increased activation-induced cytidine deaminase (AID), while CXCL12 drove proliferation of CXCR4+ SWM B cells. Lastly, correlation analysis indicated an association between CAD severity and the frequency of both CCR6+ SWM and CXCR4+ SWM B cells. Conclusions: CCR6+ SWM B cells were identified as potential producers of IgE to \\u03b1-gal in CAD patients. Additionally, our findings highlighted non-chemotaxis roles of CCL20/CCR6 and CXCL12/CXCR4 signaling in mediating IgE class switching and cell proliferation of SWM B cells respectively. Results may have important implications for a better understanding and better therapeutic approaches for subjects with IgE sensitization to \\u03b1-gal.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=689&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"82166a381d5cd5f389db4f94d7f6f4e8d0858b4d\",\n",
      "        \"title\": \"Zero-preserving imputation of single-cell RNA-seq data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=690&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6adb932e029a7e0e640e5c6ffa39ec62d2a13552\",\n",
      "        \"title\": \"Supervised dimensionality reduction for exploration of single-cell data by Hybrid Subset Selection - Linear Discriminant Analysis\",\n",
      "        \"abstract\": \"Single-cell technologies generate large, high-dimensional datasets encompassing a diversity of omics. Dimensionality reduction enables visualization of data by representing cells in two-dimensional plots that capture the structure and heterogeneity of the original dataset. Visualizations contribute to human understanding of data and are useful for guiding both quantitative and qualitative analysis of cellular relationships. Existing algorithms are typically unsupervised, utilizing only measured features to generate manifolds, disregarding known biological labels such as cell type or experimental timepoint. Here, we repurpose the classification algorithm, linear discriminant analysis (LDA), for supervised dimensionality reduction of single-cell data. LDA identifies linear combinations of predictors that optimally separate a priori classes, enabling users to tailor visualizations to separate specific aspects of cellular heterogeneity. We implement feature selection by hybrid subset selection (HSS) and demonstrate that this flexible, computationally-efficient approach generates non-stochastic, interpretable axes amenable to diverse biological processes, such as differentiation over time and cell cycle. We benchmark HSS-LDA against several popular dimensionality reduction algorithms and illustrate its utility and versatility for exploration of single-cell mass cytometry, transcriptomics and chromatin accessibility data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=691&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"89cf27dbe12660f55a95b5b9c9abde396087cc4f\",\n",
      "        \"title\": \"Granulocyte Macrophage-Colony Stimulating Factor Produces a Splenic Subset of Monocyte-Derived Dendritic Cells That Efficiently Polarize T Helper Type 2 Cells in Response to Blood-Borne Antigen\",\n",
      "        \"abstract\": \"Dendritic cells (DCs) are key antigen-presenting cells that prime naive T cells and initiate adaptive immunity. Although the genetic deficiency and transgenic overexpression of granulocyte macrophage-colony stimulating factor (GM-CSF) signaling were reported to influence the homeostasis of DCs, the in vivo development of DC subsets following injection of GM-CSF has not been analyzed in detail. Among the treatment of mice with different hematopoietic cytokines, only GM-CSF generates a distinct subset of XCR1-33D1- DCs which make up the majority of DCs in the spleen after three daily injections. These GM-CSF-induced DCs (GMiDCs) are distinguished from classical DCs (cDCs) in the spleen by their expression of CD115 and CD301b and by their superior ability to present blood-borne antigen and thus to stimulate CD4+ T cells. Unlike cDCs in the spleen, GMiDCs are exceptionally effective to polarize and expand T helper type 2 (Th2) cells and able to induce allergic sensitization in response to blood-borne antigen. Single-cell RNA sequencing analysis and adoptive cell transfer assay reveal the sequential differentiation of classical monocytes into pre-GMiDCs and GMiDCs. Interestingly, mixed bone marrow chimeric mice of Csf2rb +/+ and Csf2rb -/- demonstrate that the generation of GMiDCs necessitates the cis expression of GM-CSF receptor. Besides the spleen, GMiDCs are generated in the CCR7-independent resident DCs of the LNs and in some peripheral tissues with GM-CSF treatment. Also, small but significant numbers of GMiDCs are generated in the spleen and other tissues during chronic allergic inflammation. Collectively, our present study identifies a splenic subset of CD115hiCD301b+ GMiDCs that possess a strong capacity to promote Th2 polarization and allergic sensitization against blood-borne antigen.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=692&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a65bc83d8ea58983fe644503a473e02a406fd71\",\n",
      "        \"title\": \"Understanding the effect of carrier proteomes in single cell proteomic studies - key lessons\",\n",
      "        \"abstract\": \"ABSTRACT Introduction Mass spectrometry-based single-cell proteomics (scMS) is experiencing rapid evolution due to the increased sensitivity of mass spectrometers as well as advances in multiplexing and sample preparation. To date, researchers have focused on two general approaches to scMS: label-free and isobaric label-based multiplexing. While label-free analysis provides straightforward sample preparation and a clear path to automation, it currently lacks the throughput necessary to practically analyze thousands of single cells. Multiplexed analysis utilizing isobaric labels requires additional sample manipulation but increases throughput such that analyzing thousands of cells is currently achievable. A key feature of multiplexed scMS experiments is a \\u2018carrier proteome\\u2019 \\u2013 a sample added at 25x-500x, the single-cell sample that increases the number of proteins that can be identified in an MS analysis. Areas covered Here, we review early examples of carrier proteomes in quantitative proteomics before summarizing advantages and challenges of using a carrier proteome in scMS experiments. Expert opinion We conclude that the addition of carrier proteomes improves depth of identification for scMS, but high levels of carrier proteomes can have adverse effects on quantitative accuracy and precision.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=693&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f498975bced7bf7e39ec0bbbd51fc7a8ee28b0ef\",\n",
      "        \"title\": \"CITEMOXMBD: A flexible single-cell multimodal omics analysis framework to reveal the heterogeneity of immune cells\",\n",
      "        \"abstract\": \"ABSTRACT Simultaneous measurement of multiple modalities in single-cell analysis, represented by CITE-seq, is a promising approach to link transcriptional changes to cellular phenotype and function, requiring new computational methods to define cellular subtypes and states based on multiple data types. Here, we design a flexible single-cell multimodal analysis framework, called CITEMO, to integrate the transcriptome and antibody-derived tags (ADT) data to capture cell heterogeneity from the multi omics perspective. CITEMO uses Principal Component Analysis (PCA) to obtain a low-dimensional representation of the transcriptome and ADT, respectively, and then employs PCA again to integrate these low-dimensional multimodal data for downstream analysis. To investigate the effectiveness of the CITEMO framework, we apply CITEMO to analyse the cell subtypes of Cord Blood Mononuclear Cells (CBMC) samples. Results show that the CITEMO framework can comprehensively analyse single-cell multimodal samples and accurately identify cell subtypes. Besides, we find some specific immune cells that co-express multiple ADT markers. To better describe the co-expression phenomenon, we introduce the co-expression entropy to measure the heterogeneous distribution of the ADT combinations. To further validate the robustness of the CITEMO framework, we analyse Human Bone Marrow Cell (HBMC) samples and identify different states of the same cell type. CITEMO has an excellent performance in identifying cell subtypes and states for multimodal omics data. We suggest that the flexible design idea of CITEMO can be an inspiration for other single-cell multimodal tasks. The complete source code and dataset of the CITEMO framework can be obtained from https://github.com/studentiz/CITEMO.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=694&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa20d3cfefe98944148e85214cb844459265da9f\",\n",
      "        \"title\": \"Spatial proteogenomics reveals distinct and evolutionarily conserved hepatic macrophage niches\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=695&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"74722c31a40944dfa0489d015dfee86322e6ebf9\",\n",
      "        \"title\": \"Reproducibility of 10x Genomics single cell RNA sequencing method in the immune cell environment.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=696&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"38d42b76731fe5f46ddcc10dfb4c600665dabb8b\",\n",
      "        \"title\": \"SPRITE: a genome-wide method for mapping higher-order 3D interactions in the nucleus using combinatorial split-and-pool barcoding\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=697&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f1b4aa2c334530050fa9fc27ea3c864f0a928631\",\n",
      "        \"title\": \"New horizons in the stormy sea of multimodal single-cell data integration.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=698&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"acc84b908911ef2f2d19cdcc4ee17e0ed3723f23\",\n",
      "        \"title\": \"ATACing single cells with phages.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=699&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"760644ac0c85f73b68a9dbfb6baa5eadfa7966c5\",\n",
      "        \"title\": \"EBV+ tumors exploit tumor cell-intrinsic and -extrinsic mechanisms to produce regulatory T cell-recruiting chemokines CCL17 and CCL22\",\n",
      "        \"abstract\": \"The Epstein-Barr Virus (EBV) is involved in the etiology of multiple hematologic and epithelial human cancers. EBV+ tumors employ multiple immune escape mechanisms, including the recruitment of immunosuppressive regulatory T cells (Treg). Here, we show some EBV+ tumor cells express high levels of the chemokines CCL17 and CCL22 both in vitro and in vivo and that this expression mirrors the expression levels of expression of the EBV LMP1 gene in vitro. Patient samples from lymphoblastic (Hodgkin lymphoma) and epithelial (nasopharyngeal carcinoma; NPC) EBV+ tumors revealed CCL17 and CCL22 expression of both tumor cell-intrinsic and -extrinsic origin, depending on tumor type. NPCs grown as mouse xenografts likewise showed both mechanisms of chemokine production. Single cell RNA-sequencing revealed in vivo tumor cell-intrinsic CCL17 and CCL22 expression combined with expression from infiltrating classical resident and migratory dendritic cells in a CT26 colon cancer mouse tumor engineered to express LMP1. These data suggest that EBV-driven tumors employ dual mechanisms for CCL17 and CCL22 production. Importantly, both in vitro and in vivo Treg migration was effectively blocked by a novel, small molecule antagonist of CCR4, CCR4-351. Antagonism of the CCR4 receptor may thus be an effective means of activating the immune response against a wide spectrum of EBV+ tumors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=700&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"33ff7320db14e7df5aa41a8cb97211a60a128073\",\n",
      "        \"title\": \"Fate-mapping mice: new tools and technology for immune discovery.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=701&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"23045b926017fef13a405b04d0eb387ffb5092e8\",\n",
      "        \"title\": \"Towards Tabula Gallus\",\n",
      "        \"abstract\": \"The Tabula Gallus is a proposed project that aims to create a map of every cell type in the chicken body and chick embryos. Chickens (Gallus gallus) are one of the most recognized model animals that recapitulate the development and physiology of mammals. The Tabula Gallus will generate a compendium of single-cell transcriptome data from Gallus gallus, characterize each cell type, and provide tools for the study of the biology of this species, similar to other ongoing cell atlas projects (Tabula Muris and Tabula Sapiens/Human Cell Atlas for mice and humans, respectively). The Tabula Gallus will potentially become an international collaboration between many researchers. This project will be useful for the basic scientific study of Gallus gallus and other birds (e.g., cell biology, molecular biology, developmental biology, neuroscience, physiology, oncology, virology, behavior, ecology, and evolution). It will eventually be beneficial for a better understanding of human health and diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=702&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a1404b881868335d3f841e5a07da515181de2a93\",\n",
      "        \"title\": \"Challenges and Opportunities for the Translation of Single-Cell RNA Sequencing Technologies to Dermatology\",\n",
      "        \"abstract\": \"Skin is a complex and heterogeneous organ at the cellular level. This complexity is beginning to be understood through the application of single-cell genomics and computational tools. A large number of datasets that shed light on how the different human skin cell types interact in homeostasis\\u2014and what ceases to work in diverse dermatological diseases\\u2014have been generated and are publicly available. However, translation of these novel aspects to the clinic is lacking. This review aims to summarize the state-of-the-art of skin biology using single-cell technologies, with a special focus on skin pathologies and the translation of mechanistic findings to the clinic. The main implications of this review are to summarize the benefits and limitations of single-cell analysis and thus help translate the emerging insights from these novel techniques to the bedside.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=703&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3efec54a3cf2a3845b2953a2dd0aee418fd6b192\",\n",
      "        \"title\": \"TRAF6 functions as a tumor suppressor in myeloid malignancies by directly targeting MYC oncogenic activity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=704&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9894aadf1c4075092c0aadf763964bb64663f1eb\",\n",
      "        \"title\": \"Optical Cell Tagging for Spatially Resolved Single-Cell RNA Sequencing.\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) has emerged as a powerful tool for profiling gene expression of distinct cell populations at the single-cell level. However, the information of the positions of cells within the multicellular samples is missing in scRNA-seq datasets. To overcome this limitation, we herein develop OpTAG (optical cell tagging) as a new chemical platform for attaching functional tags onto cell surfaces in a spatially resolved manner. With OpTAG, we establish OpTAG-seq, which enables spatially resolved scRNA-seq. We apply OpTAG-seq to investigate the spatially defined transcriptional program in migrating cancer cells and identified a list of genes that are potential regulators for cancer cell migration and invasion. OpTAG-seq provides a convenient method for mapping cellular heterogeneity with spatial information within multicellular biological systems.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=705&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"37cf115d2b3248deb54e51e3bc86af4fccec6b50\",\n",
      "        \"title\": \"FACS and immunomagnetic isolation of early erythroid progenitor cells from mouse fetal liver\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=706&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d7ab72c89cd38ace5957fdba7d889b6628170d1f\",\n",
      "        \"title\": \"A Combined Immunophenotypic And Transcriptional Single-Cell Map Of First Trimester Human Fetal Liver Hematopoiesis\",\n",
      "        \"abstract\": \"Knowledge of human fetal blood development and how it differs from adult is highly relevant for our understanding of congenital blood and immune disorders as well as childhood leukemia, the latter known to originate in utero. Blood production during development occurs in waves that overlap in time and space adding to heterogeneity, which necessitates single cell approaches. Here, a combined single cell immunophenotypic and transcriptional map of first trimester primitive blood development is presented. Using CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) the molecular profile of established immunophenotypic gated progenitors was analyzed in the fetal liver (FL). Classical markers for hematopoietic stem cells (HSCs) such as CD90 and CD49F were largely preserved, whereas CD135 (FLT3) and CD123 (IL3R) had a ubiquitous expression pattern capturing heterogenous populations. Direct molecular comparison with an adult bone marrow (BM) data set revealed that HSC-like cells were less frequent in FL, whereas cells with a lympho-myeloid signature were more abundant. Furthermore, an erythro-myeloid primed multipotent progenitor cluster was identified, potentially representing a transient, FL-specific progenitor. Based on the projection performed, up- and downregulated genes between fetal and adult cells were analyzed. In general, cell cycle pathways, including MYC targets were shown to be upregulated in fetal cells, whereas gene sets involved in inflammation and human leukocyte antigen (HLA) complex were downregulated. Importantly, a fetal core molecular signature was identified that could discriminate certain types of infant and childhood leukemia from adult counterparts. Our detailed single cell map presented herein emphasizes molecular as well as immunophenotypic differences between fetal and adult primitive blood cells, of significance for future studies of pediatric leukemia and blood development in general.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=707&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0bf97297f94d2ca36ca9fa6479e8337428c6cadc\",\n",
      "        \"title\": \"Proteomics in support of immunotherapy: contribution to model-based precision medicine\",\n",
      "        \"abstract\": \"ABSTRACT Introduction Proteomics encompasses a wide and expanding range of methods to identify, characterize, and quantify thousands of proteins from a variety of biological samples, including blood samples, tumors, and tissues. Such methods are supportive of various forms of immunotherapy applied to chronic conditions such as allergies, autoimmune diseases, cancers, and infectious diseases. Areas covered In support of immunotherapy, proteomics based on mass spectrometry has multiple specific applications related to (i) disease modeling and patient stratification, (ii) antigen/ autoantigen/neoantigen/ allergen identification, (iii) characterization of proteins and monoclonal antibodies used for immunotherapeutic or diagnostic purposes, (iv) identification of biomarkers and companion diagnostics and (v) monitoring by immunoproteomics of immune responses elicited in the course of the disease or following immunotherapy. Expert opinion Proteomics contributes as an enabling technology to an evolution of immunotherapy toward a precision medicine approach aiming to better tailor treatments to patients\\u2019 specificities in multiple disease areas. This trend is favored by a better understanding through multi-omics profiling of both the patient\\u2019s characteristics, his/her immune status as well as of the features of the immunotherapeutic drug.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=708&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"baba50ecdca2c5d36cb38e414e774520644f5eae\",\n",
      "        \"title\": \"Immunotherapy of cancer in single-cell RNA sequencing era: A precision medicine perspective.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=709&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"93c7d506fe0169a0a37da89a4c7fe84f04ba43f6\",\n",
      "        \"title\": \"MIAAIM: Multi-omics image integration and tissue state mapping using topological data analysis and cobordism learning\",\n",
      "        \"abstract\": \"High-parameter tissue imaging enables detailed molecular analysis of single cells in their spatial environment. However, the comprehensive characterization and mapping of tissue states through multimodal imaging across different physiological and pathological conditions requires data integration across multiple imaging systems. Here, we introduce MIAAIM (Multi-omics Image Alignment and Analysis by Information Manifolds) a modular, reproducible computational framework for aligning data across bioimaging technologies, modeling continuities in tissue states, and translating multimodal measures across tissue types. We demonstrate MIAAIM\\u2019s workflows across diverse imaging platforms, including histological stains, imaging mass cytometry, and mass spectrometry imaging, to link cellular phenotypic states with molecular microenvironments in clinical biopsies from multiple tissue types with high cellular complexity. MIAAIM provides a robust foundation for the development of computational methods to integrate multimodal, high-parameter tissue imaging data and enable downstream computational and statistical interrogation of tissue states.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=710&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc14f2ea6297eb851c958ef0e1fa6ebc59ae18dd\",\n",
      "        \"title\": \"Time-resolved assessment of single-cell protein secretion by sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=711&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"995681657aa3d3d4a4ff005007c6d3657a1b4b9b\",\n",
      "        \"title\": \"Comprehensive benchmarking of CITE-seq versus DOGMA-seq single cell multimodal omics\",\n",
      "        \"abstract\": \"The recently developed method TEA-seq and similar DOGMA-seq single cell trimodal omics assays provide unprecedented opportunities for understanding cell biology, but independent evaluation is lacking. We explore the utility of DOGMA-seq compared to the bimodal CITE-seq assay in activated and stimulated human peripheral blood T cells. We find that single cell trimodal omics measurements after digitonin (DIG) permeabilization were generally better than after an alternative \\u201clow-loss lysis\\u201d (LLL) permeabilization condition. Next, we find that DOGMA-seq with optimized DIG permeabilization and its ATAC library provides more information, although its mRNA and cell surface protein libraries have slightly inferior quality, compared to CITE-seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=712&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d21c9cb1dd3166cc78ba44b65c3499ec5e65e346\",\n",
      "        \"title\": \"Understanding immunity in a tissue\\u2010centric context: Combining novel imaging methods and mathematics to extract new insights into function and dysfunction *\",\n",
      "        \"abstract\": \"A central question in immunology is what features allow the immune system to respond in a timely manner to a variety of pathogens encountered at unanticipated times and diverse body sites. Two decades of advanced and static dynamic imaging methods have now revealed several major principles facilitating host defense. Suborgan spatial prepositioning of distinct cells promotes time\\u2010efficient interactions upon pathogen sensing. Such pre\\u2010organization also provides an effective barrier to movement of pathogens from parenchymal tissues into the blood circulation. Various molecular mechanisms maintain effective intercellular communication among otherwise rapidly moving cells. These and related discoveries have benefited from recent increases in the number of parameters that can be measured simultaneously in a single tissue section and the extension of such multiplex analyses to 3D tissue volumes. The application of new computational methods to such imaging data has provided a quantitative, in vivo context for cell trafficking and signaling pathways traditionally explored in vitro or with dissociated cell preparations. Here, we summarize our efforts to devise and employ diverse imaging tools to probe immune system organization and function, concluding with a commentary on future developments, which we believe will reveal even more about how the immune system operates in health and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=713&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f2f54f3b698441c74fc559aab85cec6403b43cf\",\n",
      "        \"title\": \"Systematic discovery of receptor-ligand biology by engineered cell entry and single-cell genomics\",\n",
      "        \"abstract\": \"Cells communicate with each other via receptor-ligand interactions on the cell surface. Here we describe a technology for lentiviral-mediated cell entry by engineered receptor-ligand interaction (ENTER) to decode receptor specificity. Engineered lentiviral particles displaying specific ligands deliver fluorescent proteins into target cells upon cognate receptor-ligand interaction, without genome integration or transgene transcription. We optimize ENTER to decode interactions between T cell receptor (TCR)-MHC peptides, antibody-antigen, and other receptor-ligand pairs. We develop an effective presentation strategy to capture interactions between B cell receptor (BCR) and intracellular antigen epitopes. Single-cell readout of ENTER by RNA sequencing (ENTER-seq) enables multiplexed enumeration of TCR-antigen specificities, clonality, cell type, and cell states of individual T cells. ENTER-seq of patient blood samples after CMV infection reveals the viral epitopes that drive human effector memory T cell differentiation and inter-clonal phenotypic diversity that targets the same epitope. ENTER enables systematic discovery of receptor specificity, linkage to cell fates, and cell-specific delivery of gene or protein payloads. HIGHLIGHTS ENTER displays ligands, deliver cargos, and records receptor specificity. ENTER deorphanizes antigen recognition of TCR and BCR. ENTER-seq maps TCR specificity, clonality and cell state in single cells. ENTER-seq of patient sample decodes antiviral T cell memory.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=714&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"daba1d189046a079795f707176d9aa8e823257e9\",\n",
      "        \"title\": \"The chromatin repressors EZH2 and Suv4\\u201020h coregulate cell fate specification during hippocampal development\",\n",
      "        \"abstract\": \"The cell fate transition from radial glial\\u2010like (RGL) cells to neurons and astrocytes is crucial for development and pathological conditions. Two chromatin repressors\\u2014the enhancer of zeste homolog 2 and suppressor of variegation 4\\u201020 homolog\\u2014are expressed in RGL cells in the hippocampus, implicating these epigenetic regulators in hippocampal cell fate commitment. Using a double knockout mouse model, we demonstrated that loss of both chromatin repressors in the RGL population leads to deficits in hippocampal development. Single\\u2010nuclei RNA\\u2010Seq revealed differential gene expression and provided mechanistic insight into how the two chromatin repressors are critical for the maintenance of cycling cells in the dentate gyrus as well as the balance of cell trajectories between neuronal and astroglial lineages.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=715&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1a27f17dd684b468d37a04805736f79882a65096\",\n",
      "        \"title\": \"Robust Single-cell Matching and Multi-modal Analysis Using Shared and Distinct Features Reveals Orchestrated Immune Responses\",\n",
      "        \"abstract\": \"The ability to align individual cellular information from multiple experimental sources, techniques and systems is fundamental for a true systems-level understanding of biological processes. While single-cell transcriptomic studies have transformed our appreciation for the complexities and contributions of diverse cell types to disease, they can be limited in their ability to assess protein-level phenotypic information and beyond. Therefore, matching and integrating single-cell datasets which utilize robust protein measurements across multiple modalities is critical for a deeper understanding of cell states, and signaling pathways particularly within their native tissue context. Current available tools are mainly designed for single-cell transcriptomics matching and integration, and generally rely upon a large number of shared features across datasets for mutual Nearest Neighbor (mNN) matching. This approach is unsuitable when applied to single-cell proteomic datasets, due to the limited number of parameters simultaneously accessed, and lack of shared markers across these experiments. Here, we introduce a novel cell matching algorithm, Matching with pARtIal Overlap (MARIO), that takes into account both shared and distinct features, while consisting of vital filtering steps to avoid sub-optimal matching. MARIO accurately matches and integrates data from different single-cell proteomic and multi-modal methods, including spatial techniques, and has cross-species capabilities. MARIO robustly matched tissue macrophages identified from COVID-19 lung autopsies via CODEX imaging to macrophages recovered from COVID-19 bronchoalveolar lavage fluid via CITE-seq. This cross-platform integrative analysis enabled the identification of unique orchestrated immune responses within the lung of complement-expressing macrophages and their impact on the local tissue microenvironment. MARIO thus provides an analytical framework for unified analysis of single-cell data for a comprehensive understanding of the underlying biological system.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=716&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21e3ae5bc1c40be162d29242bb093b8f193dbe1b\",\n",
      "        \"title\": \"A single-cell transcriptomic landscape of innate and adaptive intratumoral immunity in triple negative breast cancer during chemo- and immunotherapies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=717&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b08c76b8c712377261d1114c1ade392dac56379e\",\n",
      "        \"title\": \"How a cell decides its own fate: a single-cell view of molecular mechanisms and dynamics of cell-type specification\",\n",
      "        \"abstract\": \"On its path from a fertilized egg to one of the many cell types in a multicellular organism, a cell turns the blank canvas of its early embryonic state into a molecular profile fine-tuned to achieve a vital organismal function. This remarkable transformation emerges from the interplay between dynamically changing external signals, the cell's internal, variable state, and tremendously complex molecular machinery; we are only beginning to understand. Recently developed single-cell omics techniques have started to provide an unprecedented, comprehensive view of the molecular changes during cell-type specification and promise to reveal the underlying gene regulatory mechanism. The exponentially increasing amount of quantitative molecular data being created at the moment is slated to inform predictive, mathematical models. Such models can suggest novel ways to manipulate cell types experimentally, which has important biomedical applications. This review is meant to give the reader a starting point to participate in this exciting phase of molecular developmental biology. We first introduce some of the principal molecular players involved in cell-type specification and discuss the important organizing ability of biomolecular condensates, which has been discovered recently. We then review some of the most important single-cell omics methods and relevant findings they produced. We devote special attention to the dynamics of the molecular changes and discuss methods to measure them, most importantly lineage tracing. Finally, we introduce a conceptual framework that connects all molecular agents in a mathematical model and helps us make sense of the experimental data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=718&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0afa22dbf68d4c959aac7ce51f233c24decf8c92\",\n",
      "        \"title\": \"Why Single-Cell Sequencing Has Promise in MDS\",\n",
      "        \"abstract\": \"Myelodysplastic syndromes (MDS) are a heterogeneous group of diseases characterized by ineffective hematopoiesis. The risk of MDS is associated with aging and the accumulation of somatic mutations in hematopoietic stem cells and progenitors (HSPC). While advances in DNA sequencing in the past decade unveiled clonal selection driven by mutations in MDS, it is unclear at which stage the HSPCs are trapped or what prevents mature cells output. Single-cell-sequencing techniques in recent years have revolutionized our understanding of normal hematopoiesis by identifying the transitional cell states between classical hematopoietic hierarchy stages, and most importantly the biological activities behind cell differentiation and lineage commitment. Emerging studies have adapted these powerful tools to investigate normal hematopoiesis as well as the clonal heterogeneity in myeloid malignancies and provide a progressive description of disease pathogenesis. This review summarizes the potential of growing single-cell-sequencing techniques, the evolving efforts to elucidate hematopoiesis in physiological conditions and MDS at single-cell resolution, and discuss how they may fill the gaps in our current understanding of MDS biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=719&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0eee5a0e15b1defe60770bc08c698acb2fbc37e7\",\n",
      "        \"title\": \"Evaluating microglial phenotypes using single-cell technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=720&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"adfc5c1f27ec710c913d7b7430318db948d570b2\",\n",
      "        \"title\": \"Cobolt: integrative analysis of multimodal single-cell sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=721&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f08b4436f8816fd107f02861f144b692b5784ced\",\n",
      "        \"title\": \"Innovation in culture systems to study muscle complexity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=722&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fddf51b4d1d8f2458c517ff41b827d9253c85fea\",\n",
      "        \"title\": \"Single Cell Technologies to Dissect Heterogenous Immune Cell Therapy Products.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=723&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b0d907515d63aaf16b3e3b6fd6661c8b7c06d001\",\n",
      "        \"title\": \"Multianalyte lateral flow immunoassay for simultaneous detection of protein-based inflammation biomarkers and pathogen DNA\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=724&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"afb1117ebd151c9765c9277bff63ee8b74578fb9\",\n",
      "        \"title\": \"Intravital and high-content multiplex imaging of the immune system.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=725&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc396060d79c56024e25a01936e95e0650b52199\",\n",
      "        \"title\": \"Completing the cancer jigsaw puzzle with single-cell multiomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=726&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75b37a95ec925fecd3d6d69e5b538baa186ccbc4\",\n",
      "        \"title\": \"Charting the tumor antigen maps drawn by single-cell genomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=727&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cb8daa61c80c26893ca4b6106e390a04eecf1a3d\",\n",
      "        \"title\": \"nSeP: immune and metabolic biomarkers for early detection of neonatal sepsis\\u2014protocol for a prospective multicohort study\",\n",
      "        \"abstract\": \"Introduction Diagnosing neonatal sepsis is heavily dependent on clinical phenotyping as culture-positive body fluid has poor sensitivity, and existing blood biomarkers have poor specificity. A combination of machine learning, statistical and deep pathway biology analyses led to the identification of a tripartite panel of biologically connected immune and metabolic markers that showed greater than 99% accuracy for detecting bacterial infection with 100% sensitivity. The cohort study described here is designed as a large-scale clinical validation of this previous work. Methods and analysis This multicentre observational study will prospectively recruit a total of 1445 newborn infants (all gestations)\\u20141084 with suspected early\\u2014or late-onset sepsis, and 361 controls\\u2014over 4\\u2009years. A small volume of whole blood will be collected from infants with suspected sepsis at the time of presentation. This sample will be used for integrated transcriptomic, lipidomic and targeted proteomics profiling. In addition, a subset of samples will be subjected to cellular phenotype and proteomic analyses. A second sample from the same patient will be collected at 24 hours, with an opportunistic sampling for stool culture. For control infants, only one set of blood and stool sample will be collected to coincide with clinical blood sampling. Along with detailed clinical information, blood and stool samples will be analysed and the information will be used to identify and validate the efficacy of immune-metabolic networks in the diagnosis of bacterial neonatal sepsis and to identify new host biomarkers for viral sepsis. Ethics and dissemination The study has received research ethics committee approval from the Wales Research Ethics Committee 2 (reference 19/WA/0008) and operational approval from Health and Care Research Wales. Submission of study results for publication will involve making available all anonymised primary and processed data on public repository sites. Trial registration number NCT03777670\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=728&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"20edae40df77682764a0124403ab39bca5a9e53e\",\n",
      "        \"title\": \"Redefining intestinal immunity with single-cell transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=729&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"98646e314f1ac840e09384f7ba8f58de209c50c2\",\n",
      "        \"title\": \"Imaging of Glioblastoma Tumor-Associated Myeloid Cells Using Nanobodies Targeting Signal Regulatory Protein Alpha\",\n",
      "        \"abstract\": \"Glioblastoma (GBM) is the most common malignant primary brain tumor. Glioblastomas contain a large non-cancerous stromal compartment including various populations of tumor-associated macrophages and other myeloid cells, of which the presence was documented to correlate with malignancy and reduced survival. Via single-cell RNA sequencing of human GBM samples, only very low expression of PD-1, PD-L1 or PD-L2 could be detected, whereas the tumor micro-environment featured a marked expression of signal regulatory protein alpha (SIRP\\u03b1), an inhibitory receptor present on myeloid cells, as well as its widely distributed counter-receptor CD47. CITE-Seq revealed that both SIRP\\u03b1 RNA and protein are prominently expressed on various populations of myeloid cells in GBM tumors, including both microglia- and monocyte-derived tumor-associated macrophages (TAMs). Similar findings were obtained in the mouse orthotopic GL261 GBM model, indicating that SIRP\\u03b1 is a potential target on GBM TAMs in mouse and human. A set of nanobodies, single-domain antibody fragments derived from camelid heavy chain-only antibodies, was generated against recombinant SIRP\\u03b1 and characterized in terms of affinity for the recombinant antigen and binding specificity on cells. Three selected nanobodies binding to mouse SIRP\\u03b1 were radiolabeled with 99mTc, injected in GL261 tumor-bearing mice and their biodistribution was evaluated using SPECT/CT imaging and radioactivity detection in dissected organs. Among these, Nb15 showed clear accumulation in peripheral organs such as spleen and liver, as well as a clear tumor uptake in comparison to a control non-targeting nanobody. A bivalent construct of Nb15 exhibited an increased accumulation in highly vascularized organs that express the target, such as spleen and liver, as compared to the monovalent format. However, penetration into the GL261 brain tumor fell back to levels detected with a non-targeting control nanobody. These results highlight the tumor penetration advantages of the small monovalent nanobody format and provide a qualitative proof-of-concept for using SIRP\\u03b1-targeting nanobodies to noninvasively image myeloid cells in intracranial GBM tumors with high signal-to-noise ratios, even without blood-brain barrier permeabilization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=730&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"008fb3609e1235994f93361da300eaba2f3c3262\",\n",
      "        \"title\": \"Single-cell analysis identifies dynamic gene expression networks that govern B cell development and transformation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=731&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b323b333686749b942da78b8d2ecbb845abca79e\",\n",
      "        \"title\": \"Emerging Single-cell Approaches to Understand HIV in the Central Nervous System\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=732&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f00a1aea001259e0f1eadfc6551d5bd0165ff3c8\",\n",
      "        \"title\": \"Advanced technologies for single-cell in situ protein profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=733&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e7b830ff33471c51eb2608bab38048db6db1ad73\",\n",
      "        \"title\": \"Identifying disease-critical cell types and cellular processes across the human body by integration of single-cell profiles and human genetics\",\n",
      "        \"abstract\": \"Genome-wide association studies (GWAS) provide a powerful means to identify loci and genes contributing to disease, but in many cases the related cell types/states through which genes confer disease risk remain unknown. Deciphering such relationships is important for identifying pathogenic processes and developing therapeutics. Here, we introduce sc-linker, a framework for integrating single-cell RNA-seq (scRNA-seq), epigenomic maps and GWAS summary statistics to infer the underlying cell types and processes by which genetic variants influence disease. We analyzed 1.6 million scRNA-seq profiles from 209 individuals spanning 11 tissue types and 6 disease conditions, and constructed gene programs capturing cell types, disease progression, and cellular processes both within and across cell types. We evaluated these gene programs for disease enrichment by transforming them to SNP annotations with tissue-specific epigenomic maps and computing enrichment scores across 60 diseases and complex traits (average N=297K). Cell type, disease progression, and cellular process programs captured distinct heritability signals even within the same cell type, as we show in multiple complex diseases that affect the brain (Alzheimer\\u2019s disease, multiple sclerosis), colon (ulcerative colitis) and lung (asthma, idiopathic pulmonary fibrosis, severe COVID-19). The inferred disease enrichments recapitulated known biology and highlighted novel cell-disease relationships, including GABAergic neurons in major depressive disorder (MDD), a disease progression M cell program in ulcerative colitis, and a disease-specific complement cascade process in multiple sclerosis. In autoimmune disease, both healthy and disease progression immune cell type programs were associated, whereas for epithelial cells, disease progression programs were most prominent, perhaps suggesting a role in disease progression over initiation. Our framework provides a powerful approach for identifying the cell types and cellular processes by which genetic variants influence disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=734&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8326dd063d32132e4efd05f3fb99340023ca9918\",\n",
      "        \"title\": \"Immune ageing at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=735&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ee8bead70f6a9e9485f6e862812f2eb5ef258751\",\n",
      "        \"title\": \"Antigen presentation by type 3 innate lymphoid cells instructs the differentiation of gut microbiota-specific regulatory T cells\",\n",
      "        \"abstract\": \"The mutualistic relationship of gut-resident microbiota and cells of the host immune system promotes homeostasis that ensures maintenance of the microbial community and of a poised, but largely non-aggressive, immune cell compartment1,2. Consequences of disturbing this balance, by environmental or genetic factors, include proximal inflammatory conditions, like Crohn's disease, and systemic illnesses, both metabolic and autoimmune. One of the means by which this equilibrium is achieved is through induction of both effector and suppressor or regulatory arms of the adaptive immune system. In mice, Helicobacter species induce regulatory (iTreg) and follicular helper (Tfh) T cells in the colon-draining mesenteric lymph nodes under homeostatic conditions, but can instead induce inflammatory Th17 cells and colitis when iTreg cells are compromised3,4. How Helicobacter hepaticus and other gut bacteria direct T cells to adopt distinct functions remains poorly understood. Here, we investigated which cells and molecular components are required to convey the microbial instruction for the iTreg differentiation program. We found that antigen presentation by cells expressing ROR\\u03b3t, rather than by classical dendritic cells, was both required and sufficient for iTreg induction. These ROR\\u03b3t+ cells, likely to be type 3 innate lymphoid cells (ILC3), require the MHC class II antigen presentation machinery, the chemokine receptor CCR7, and \\u03b1v integrin, which activates TGF-\\u03b2, for iTreg cell differentiation. In the absence of any of these, instead of iTreg cells there was expansion of microbiota-specific pathogenic Th17 cells, which were induced by other antigen presenting cells (APCs) that did not require CCR7. Thus, intestinal commensal microbes and their products target multiple APCs with pre-determined features suited to directing appropriate T cell differentiation programs, rather than a common APC that they endow with appropriate functions. Our results illustrate the ability of microbiota to exploit specialized functions of distinct innate immune system cells, targeting them to achieve the desired composition of equipoised T cells, thus maintaining tolerance.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=736&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"941bee1136acd7fad126f6392b8802aa4aa0daa5\",\n",
      "        \"title\": \"Mosaic loss of Chromosome Y in aged human microglia\",\n",
      "        \"abstract\": \"Mosaic loss of chromosome Y (LOY) is a particularly common acquired structural mutation in the leukocytes of aging men and it has been shown to correlate with several age-related diseases including Alzheimer\\u2019s disease (AD). To derive the molecular basis of LOY in brain cells, we create an integrated resource by aggregating data from 21 single-cell and single-nuclei RNA brain studies, yielding 763,410 cells to investigate the presence and cell-type specific burden of LOY. We created robust quantification metrics for assessing LOY, which were validated using a multi-modal dataset. Using this new resource and LOY-quantification approach, we found that LOY frequencies differed widely between CNS cell-types and individual donors. Among five common neural cell types, microglia were most affected by LOY (7.79%, n=41,949), while LOY in neurons was rare (0.48%, n=220,010). Differential gene expression analysis in microglia found 188 autosomal genes, 6 X-linked genes, and 11 pseudoautosomal genes, pointing to broad dysregulation in lipoprotein metabolism, inflammatory response, and antigen processing that coincides with loss of Y. To our knowledge, we provide the first evidence of LOY in the microglia, and highlight its potential roles in aging and the pathogenesis of neurodegenerative disorders such as AD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=737&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a7c398841ac48be1905329fb4c2786581823e99b\",\n",
      "        \"title\": \"The Origins of Phenotypic Heterogeneity in Cancer\",\n",
      "        \"abstract\": \"Heterogeneity is a pervasive feature of cancer, and understanding the sources and regulatory mechanisms underlying heterogeneity could provide key insights to help improve the diagnosis and treatment of cancer. In this review, we discuss the origin of heterogeneity in the phenotype of individual cancer cells. Genotype\\u2013phenotype (G\\u2013P) maps are widely used in evolutionary biology to represent the complex interactions of genes and the environment that lead to phenotypes that impact fitness. Here, we present the rationale of an extended G\\u2013P (eG\\u2013P) map with a cone structure in cancer. The eG\\u2013P cone is formed by cells that are similar at the genome layer but gradually increase variability in the epigenome, transcriptome, proteome, metabolome, and signalome layers to produce large variability at the phenome layer. Experimental evidence from single-cell-omics analyses supporting the cancer eG\\u2013P cone concept is presented, and the impact of epimutations and the interaction of cancer and tumor microenvironmental eG\\u2013P cones are integrated with the current understanding of cancer biology. The eG\\u2013P cone concept uncovers potential therapeutic strategies to reduce cancer evolution and improve cancer treatment. More methods to study phenotypes in single cells will be the key to better understand cancer cell fitness in tumor biology and therapeutics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=738&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a72989f83f02e96cf6960b8b07e784d49c0ad927\",\n",
      "        \"title\": \"\\u201cMultiomics\\u201d Approaches to Understand and Treat COVID-19: Mass Spectrometry and Next-Generation Sequencing\",\n",
      "        \"abstract\": \"In the race against COVID-19 for timely therapeutic developments, mass spectrometry-based high-throughput methods have been valuable. COVID-19 manifests an extremely diverse spectrum of phenotypes from asymptomatic to life-threatening, drastic elevations in immune response or cytokine storm, multiple organ failure and death. These observations warrant a detailed understanding of associated molecular mechanisms to develop therapies. In this direction, high-throughput methods that generate large datasets focusing on changes in protein interactions, lipid metabolism, transcription, and epigenetic regulation of gene expression are extremely beneficial sources of information. Hence, mass spectrometry-based methods have been employed in several studies to detect changes in interactions among host proteins, and between host and viral proteins in COVID-19 patients. The methods have also been used to characterize host and viral proteins, and analyze lipid metabolism in COVID-19 patients. Information obtained using the above methods are complemented by high-throughput analysis of transcriptomic and epigenomic changes associated with COVID-19, coupled with next-generation sequencing. Hence, this review discusses the most recent studies focusing on the methods described above. The results establish the importance of mass spectrometry-based studies towards understanding the infection process, immune imbalance, disease mechanism, and indicate the potential of the methods\\u2019 therapeutic developments and biomarker screening against COVID-19 and future outbreaks.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=739&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18dd1d0cbc561c35a751017a0751c3ce05924dc9\",\n",
      "        \"title\": \"Combined Measurement of RNA and Protein Expression on a Single-Cell Level.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=740&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bb13862b1ea838a9aad27bb487e7e848a97b66bb\",\n",
      "        \"title\": \"Single-Cell Protein Profiling by Microdroplet Barcoding and Next-Generation Sequencing.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=741&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17bc622a42d8a51d701001bcefba60360e8a6ad3\",\n",
      "        \"title\": \"Simultaneous Analysis of Single-Cell Transcriptomes and Cell Surface Protein Expression of Human Hematopoietic Stem Cells and Progenitors Using the 10x Genomics Platform.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=742&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"585aae2ee77df042f38e6eaadc02ef2cf4ab97e5\",\n",
      "        \"title\": \"Generation of Centered Log-Ratio Normalized Antibody-Derived Tag Counts from Large Single-Cell Sequencing Datasets.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=743&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7074ddfe1590ed8c9013192c4fb25d38e25e44b0\",\n",
      "        \"title\": \"Aging of the Hematopoietic Stem Cell Niche: New Tools to Answer an Old Question\",\n",
      "        \"abstract\": \"The hematopoietic stem cell (HSC) niche is a specialized microenvironment, where a complex and dynamic network of interactions across multiple cell types regulates HSC function. During the last years, it became progressively clearer that changes in the HSC niche are responsible for specific alterations of HSC behavior. The aging of the bone marrow (BM) microenvironment has been shown to critically contribute to the decline in HSC function over time. Interestingly, while upon aging some niche structures within the BM are degenerated and negatively affect HSC functionality, other niche cells and specific signals are preserved and essential to retaining HSC function and regenerative capacity. These new findings on the role of the aging BM niche critically depend on the implementation of new technical tools, developed thanks to transdisciplinary approaches, which bring together different scientific fields. For example, the development of specific mouse models in addition to coculture systems, new 3D-imaging tools, ossicles, and ex-vivo BM mimicking systems is highlighting the importance of new technologies to unravel the complexity of the BM niche on aging. Of note, an exponential impact in the understanding of this biological system has been recently brought by single-cell sequencing techniques, spatial transcriptomics, and implementation of artificial intelligence and deep learning approaches to data analysis and integration. This review focuses on how the aging of the BM niche affects HSCs and on the new tools to investigate the specific alterations occurring in the BM upon aging. All these new advances in the understanding of the BM niche and its regulatory function on HSCs have the potential to lead to novel therapeutical approaches to preserve HSC function upon aging and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=744&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"30eea6750b3f6b3b4e11ca4127b02df90feb135b\",\n",
      "        \"title\": \"Single-cell immunophenotyping of the fetal immune response to maternal SARS-CoV-2 infection in late gestation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=745&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bcd1cd718a88b2c927c8fd36374e4ff740c61fae\",\n",
      "        \"title\": \"Whole-animal multiplexed single-cell RNA-seq reveals transcriptional shifts across Clytia medusa cell types\",\n",
      "        \"abstract\": \"Whole-organism perturbations reveal single-cell responses across the Clytia medusa.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=746&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3d2eac3fad55c13a03a4b44e044e0fe40642d8fd\",\n",
      "        \"title\": \"GM-CSF: Master regulator of the T cell-phagocyte interface during inflammation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=747&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8a4b71e0368c852b9034bc17f4c0f5685a15b507\",\n",
      "        \"title\": \"Delayed antiretroviral therapy in HIV-infected individuals leads to irreversible depletion of skin- and mucosa-resident memory T\\u00a0cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=748&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3353f0d5d557d9f5a039fb01567e5687be0ab6f5\",\n",
      "        \"title\": \"Single-cell analysis of human non-small cell lung cancer lesions refines tumor classification and patient stratification.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=749&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cf700b1b5f50d7d15d2ed935674cd3835a42e6c9\",\n",
      "        \"title\": \"Automation enables high-throughput and reproducible single-cell transcriptomics library preparation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=750&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"29c799b258a81bddbfaccf089cca8d7db4bb7227\",\n",
      "        \"title\": \"Anatomical structures, cell types and biomarkers of the Human Reference Atlas\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=751&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"81e025dc769824605a438ebae0e18b255349055d\",\n",
      "        \"title\": \"Special Issue \\u201cMicroglia Heterogeneity and Its Relevance for Translational Research\\u201d\",\n",
      "        \"abstract\": \"Microglia are necessary for the development and function of the central nervous system (CNS) [...].\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=752&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d0df073efff86022e2e767b3974674a23a9afdb\",\n",
      "        \"title\": \"A comparative study of cell type annotation methods for immune cells using single-cell sequencing technology\",\n",
      "        \"abstract\": \"Abstract: Single-cell sequencing is an emerging technique that allows high-throughput data analysis at an individual cell resolution and is applied in diverse fields of biology. Due to the large amount of data, downstream analysis is very complicated, and cell type annotation is a critical step; however, it is currently difficult to obtain good results. Single-cell sequencing has resulted in new breakthroughs in multi-omics, such as CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by sequencing), which allows the measurement of surface marker proteins simultaneously with the sequencing of mRNA at the single-cell level. In this study, a CITE-seq dataset of human PBMCs (peripheral blood mononuclear cells) was annotated using the most popular reference-based annotation methods, including SingleR, Seurat with the RNA-seq dataset, and Seurat with both the RNA and protein datasets; the results were then compared with RNA and protein expression levels to determine the role of proteins in cell annotation. The results indicate that protein expression can supplement datasets with some mRNAs with low expression to improve accuracy. With the verification of single-cell biomarkers, the multi-omics annotation method Seurat with both the RNA and protein databases showed the best performance, especially in the differentiation of NK cells and T cells and of dendritic cells and monocytes. This study shows the significance of multi-omics information for improving cell annotation and has great potential for perfecting these annotations with more data support.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=753&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e33c50f23aa064b9d370cd9e12e1a54021fccf6f\",\n",
      "        \"title\": \"Analytical Technology for Single-Cancer-Cell Analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=754&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"db5cf22e24f6b65abc1c09cf72dda9cef53c9b9c\",\n",
      "        \"title\": \"Single-Cell Transcriptome Sequencing Using Microfluidics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=755&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a3ead0d59a807aa4fdeda67b1ab9044524211e0d\",\n",
      "        \"title\": \"Curated single cell multimodal landmark datasets for R/Bioconductor\",\n",
      "        \"abstract\": \"Background The majority of high-throughput single-cell molecular profiling methods quantify RNA expression; however, recent multimodal profiling methods add simultaneous measurement of genomic, proteomic, epigenetic, and/or spatial information on the same cells. The development of new statistical and computational methods in Bioconductor for such data will be facilitated by easy availability of landmark datasets using standard data classes. Results We collected, processed, and packaged publicly available landmark datasets from important single-cell multimodal protocols, including CITE-Seq, ECCITE-Seq, SCoPE2, scNMT, 10X Multiome, seqFISH, and G&T. We integrate data modalities via the MultiAssayExperiment Bioconductor class, document and re-distribute datasets as the SingleCellMultiModal package in Bioconductor\\u2019s Cloud-based ExperimentHub. The result is single-command actualization of landmark datasets from seven single-cell multimodal data generation technologies, without need for further data processing or wrangling in order to analyze and develop methods within Bioconductor\\u2019s ecosystem of hundreds of packages for single-cell and multimodal data. Conclusions We provide two examples of integrative analyses that are greatly simplified by SingleCellMultiModal. The package will facilitate development of bioinformatic and statistical methods in Bioconductor to meet the challenges of integrating molecular layers and analyzing phenotypic outputs including cell differentiation, activity, and disease. Author Summary Experimental data packages that provide landmark datasets have historically played an important role in the development of new statistical methods in Bioconductor by lowering the barrier of access to relevant data, providing a common testing ground for software development and benchmarking, and encouraging interoperability around common data structures. In this manuscript, we review major classes of technologies for collecting multimodal data including genomics, transcriptomics, epigenetics, proteomics, and spatial information at the level of single cells. We present the SingleCellMultiModal R/Bioconductor package that provides single-command access to landmark datasets from seven different technologies, storing datasets using HDF5 and sparse arrays for memory efficiency and integrating data modalities via the MultiAssayExperiment class. We demonstrate two integrative analyses that are greatly simplified by SingleCellMultiModal. The package facilitates development and benchmarking of bioinformatic and statistical methods to integrate molecular layers at the level of single cells with phenotypic outputs including cell differentiation, activity, and disease, within Bioconductor\\u2019s ecosystem of hundreds of packages for single-cell and multimodal data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=756&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5152c6254aed0bb3c5fca2c9cfaf1fb3d52971bb\",\n",
      "        \"title\": \"Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=757&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"482d200a69b0fdf716257f726c25cd5dc8c767b2\",\n",
      "        \"title\": \"Die Vermessung von Einzelzellen\",\n",
      "        \"abstract\": \"Rudolf Virchow stellte 1858 mit seiner Ver\\u00f6ffentlichung der Cellularpathologie das Konzept auf, dass Krankheiten auf der St\\u00f6rung von Zellen bzw. derer Funktionen beruhen. Viele Erkenntnisse in der Immunologie basieren auf der Untersuchung einzelner Zellen, angefangen mit der ph\\u00e4notypischen und funktionellen Beschreibung verschiedener Immunzellen durch Paul Ehrlich und Elias Metchnikov mittels der Mikroskopie bis zu der detaillierten molekularen Analyse des gesamten Transkriptoms einzelner Zellen. Heutzutage erm\\u00f6glichen es uns eine Vielzahl verschiedener Technologien zur Einzelzellanalyse, Immunprozesse zu verfolgen und besser zu verstehen. Im Folgenden werden verschiedene Technologien vorgestellt, mit ihren jeweiligen Facetten und zuk\\u00fcnftigen Herausforderungen.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=758&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4619911a8be7eb98e03edf2f7ccdb1b199714c20\",\n",
      "        \"title\": \"Feature selection revisited in the single-cell era\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=759&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"12883898a0b674ede44ec3c99424dc914ded7b9a\",\n",
      "        \"title\": \"Disentangling single-cell omics representation with a power spectral density-based feature extraction\",\n",
      "        \"abstract\": \"Emerging single-cell technologies provide high-resolution measurements of distinct cellular modalities opening new avenues for generating detailed cellular atlases of many and diverse tissues. The high dimensionality, sparsity, and inaccuracy of single cell sequencing measurements, however, can obscure discriminatory information, mask cellular subtype variations and complicate downstream analyses which can limit our understanding of cell function and tissue heterogeneity. Here, we present a novel pre-processing method (scPSD) inspired by power spectral density analysis that enhances the accuracy for cell subtype separation from large-scale single-cell omics data. We comprehensively benchmarked our method on a wide range of single-cell RNA-sequencing datasets and showed that scPSD pre-processing, while being fast and scalable, significantly reduces data complexity, enhances cell-type separation, and enables rare cell identification. Additionally, we applied scPSD to transcriptomics and chromatin accessibility cell atlases and demonstrated its capacity to discriminate over 100 cell types across the whole organism and across different modalities of single-cell omics data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=760&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bb5f4a67aa55fa9a41963cfac712d877461e8ef8\",\n",
      "        \"title\": \"Single-cell profiling of proteins and chromatin accessibility using PHAGE-ATAC\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=761&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f308f4e70dde9f276392785274720a801cb6b60\",\n",
      "        \"title\": \"Co-staining human PBMCs with fluorescent antibodies and antibody-oligonucleotide conjugates for cell sorting prior to single-cell CITE-Seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=762&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"76a60907276a11b297f90986728967352c08b1c1\",\n",
      "        \"title\": \"Single-cell genomics in plants: current state, future directions, and hurdles to overcome\",\n",
      "        \"abstract\": \"Abstract Single-cell genomics has the potential to revolutionize the study of plant development and tissue-specific responses to environmental stimuli by revealing heretofore unknown players and gene regulatory processes. Here, I focus on the current state of single-cell genomics in plants, emerging technologies and applications, in addition to outlining possible future directions for experiments. I describe approaches to enable cheaper and larger experiments and technologies to measure multiple types of molecules to better model and understand cell types and their different states and trajectories throughout development. Lastly, I discuss the inherent limitations of single-cell studies and the technological hurdles that need to be overcome to widely apply single-cell genomics in crops to generate the greatest possible knowledge gain.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=763&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fc45597cf045209db0e6ebb9446cb858257d39f8\",\n",
      "        \"title\": \"Single-Cell Genomics: Catalyst for Cell Fate Engineering\",\n",
      "        \"abstract\": \"As we near a complete catalog of mammalian cell types, the capability to engineer specific cell types on demand would transform biomedical research and regenerative medicine. However, the current pace of discovering new cell types far outstrips our ability to engineer them. One attractive strategy for cellular engineering is direct reprogramming, where induction of specific transcription factor (TF) cocktails orchestrates cell state transitions. Here, we review the foundational studies of TF-mediated reprogramming in the context of a general framework for cell fate engineering, which consists of: discovering new reprogramming cocktails, assessing engineered cells, and revealing molecular mechanisms. Traditional bulk reprogramming methods established a strong foundation for TF-mediated reprogramming, but were limited by their small scale and difficulty resolving cellular heterogeneity. Recently, single-cell technologies have overcome these challenges to rapidly accelerate progress in cell fate engineering. In the next decade, we anticipate that these tools will enable unprecedented control of cell state.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=764&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ccd4ae18417c937b7b6440e755b7ce07f3711d7\",\n",
      "        \"title\": \"\\u03b3\\u03b4 T cells are effectors of immune checkpoint blockade in mismatch repair-deficient colon cancers with antigen presentation defects\",\n",
      "        \"abstract\": \"DNA mismatch repair deficient (MMR-d) cancers present an abundance of neoantigens that likely underlies their exceptional responsiveness to immune checkpoint blockade (ICB)1,2. However, MMR-d colon cancers that evade CD8+ T cells through loss of Human Leukocyte Antigen (HLA) class I-mediated antigen presentation3\\u20136, frequently remain responsive to ICB7 suggesting the involvement of other immune effector cells. Here, we demonstrate that HLA class I-negative MMR-d cancers are highly infiltrated by \\u03b3\\u03b4 T cells. These \\u03b3\\u03b4 T cells are mainly composed of V\\u03b41 and V\\u03b43 subsets, and express high levels of PD-1, activation markers including cytotoxic molecules, and a broad repertoire of killer-cell immunoglobulin-like receptors (KIRs). In vitro, PD-1+ \\u03b3\\u03b4 T cells, isolated from MMR-d colon cancers, exhibited a cytolytic response towards HLA class I-negative MMR-d colon cancer cell lines and \\u03b22-microglobulin (B2M)-knockout patient-derived tumor organoids (PDTOs), which was enhanced as compared to antigen presentation-proficient cells. This response was diminished after blocking the interaction between NKG2D and its ligands. By comparing paired tumor samples of MMR-d colorectal cancer patients obtained before and after dual PD-1 and CTLA-4 blockade, we found that ICB profoundly increased the intratumoral frequency of \\u03b3\\u03b4 T cells in HLA class I-negative cancers. Taken together, these data indicate that \\u03b3\\u03b4 T cells contribute to the response to ICB therapy in patients with HLA class I-negative, MMR-d colon cancers, and illustrate the potential of \\u03b3\\u03b4 T cells in cancer immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=765&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a2c3a18dc820143343d68f59207b3b836de54fcd\",\n",
      "        \"title\": \"Cardiac Resident Macrophages Prevent Fibrosis and Stimulate Angiogenesis\",\n",
      "        \"abstract\": \"Supplemental Digital Content is available in the text. Rationale: The initial hypertrophy response to cardiac pressure overload is considered compensatory, but with sustained stress, it eventually leads to heart failure. Recently, a role for recruited macrophages in determining the transition from compensated to decompensated hypertrophy has been established. However, whether cardiac resident immune cells influence the early phase of hypertrophy development has not been established. Objective: To assess the role of cardiac immune cells in the early hypertrophy response to cardiac pressure overload induced by transverse aortic constriction (TAC). Methods and Results: We performed cytometry by time-of-flight to determine the identity and abundance of immune cells in the heart at 1 and 4 weeks after TAC. We observed a substantial increase in cardiac macrophages 1 week after TAC. We then conducted Cite-Seq single-cell RNA sequencing of cardiac immune cells isolated from 4 sham and 6 TAC hearts. We identified 12 clusters of monocytes and macrophages, categorized as either resident or recruited macrophages, that showed remarkable changes in their abundance between sham and TAC conditions. To determine the role of cardiac resident macrophages early in the response to a hypertrophic stimulus, we used a blocking antibody against macrophage colony-stimulating factor 1 receptor (CD115). As blocking CD115 initially depletes all macrophages, we allowed the replenishment of recruited macrophages by monocytes before performing TAC. This preferential depletion of resident macrophages resulted in enhanced fibrosis and a blunted angiogenesis response to TAC. Macrophage depletion in CCR2 (C-C chemokine receptor type 2) knockout mice showed that aggravated fibrosis was primarily caused by the recruitment of monocyte-derived macrophages. Finally, 6 weeks after TAC these early events lead to depressed cardiac function and enhanced fibrosis, despite complete restoration of cardiac immune cells. Conclusions: Cardiac resident macrophages are a heterogeneous population of immune cells with key roles in stimulating angiogenesis and inhibiting fibrosis in response to cardiac pressure overload.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=766&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e82565d279ab6de22168c3757957e1ce21b35e6d\",\n",
      "        \"title\": \"Scalable dual-omics profiling with single-nucleus chromatin accessibility and mRNA expression sequencing 2 (SNARE-seq2)\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=767&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e108f23b669bf6f0e4b251c014d5650541ff20ab\",\n",
      "        \"title\": \"Advanced Genomics-Based Approaches for Defining Allograft Rejection With Single Cell Resolution\",\n",
      "        \"abstract\": \"Solid organ transplant recipients require long-term immunosuppression for prevention of rejection. Calcineurin inhibitor (CNI)-based immunosuppressive regimens have remained the primary means for immunosuppression for four decades now, yet little is known about their effects on graft resident and infiltrating immune cell populations. Similarly, the understanding of rejection biology under specific types of immunosuppression remains to be defined. Furthermore, development of innovative, rationally designed targeted therapeutics for mitigating or preventing rejection requires a fundamental understanding of the immunobiology that underlies the rejection process. The established use of microarray technologies in transplantation has provided great insight into gene transcripts associated with allograft rejection but does not characterize rejection on a single cell level. Therefore, the development of novel genomics tools, such as single cell sequencing techniques, combined with powerful bioinformatics approaches, has enabled characterization of immune processes at the single cell level. This can provide profound insights into the rejection process, including identification of resident and infiltrating cell transcriptomes, cell-cell interactions, and T cell receptor \\u03b1/\\u03b2 repertoires. In this review, we discuss genomic analysis techniques, including microarray, bulk RNAseq (bulkSeq), single-cell RNAseq (scRNAseq), and spatial transcriptomic (ST) techniques, including considerations of their benefits and limitations. Further, other techniques, such as chromatin analysis via assay for transposase-accessible chromatin sequencing (ATACseq), bioinformatic regulatory network analyses, and protein-based approaches are also examined. Application of these tools will play a crucial role in redefining transplant rejection with single cell resolution and likely aid in the development of future immunomodulatory therapies in solid organ transplantation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=768&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8e3cdb1cf2d31bc647bd7ee82fcd41a1428d0169\",\n",
      "        \"title\": \"Functional single cell selection and annotated profiling of dynamically changing cancer cells\",\n",
      "        \"abstract\": \"A method connecting single cell genomic or transcriptomic profiles to functional cellular characteristics, in particular time-varying phenotypic changes, would be transformative for single cell and cancer biology. Here, we present fSCS: functional single cell selection. This technology combines a custom-built ultrawide field-of-view optical screening microscope, fast automated image analysis and a new photolabeling method, phototagging, using a newly synthesized visible-light-photoactivatable dye. Using fSCS, we screen, selectively photolabel and isolate cells of interest from large heterogeneous populations based on functional dynamics like fast migration, morphological variation, small molecule uptake or cell division. We combined fSCS with single cell RNA sequencing for functionally annotated transcriptomic profiling of fast migrating and spindle-shaped MCF10A cells with or without TGF\\u03b2 induction. We identified critical genes and pathways driving aggressive migration as well as mesenchymal-like morphology that could not be detected with state-of-the-art single cell transcriptomic analysis. fSCS provides a crucial upstream selection paradigm for single cell sequencing independent of biomarkers, allows enrichment of rare cells and can facilitate the identification and understanding of molecular mechanisms underlying functional phenotypes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=769&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2d1636b3e46b649717e243b2962733728309b4b7\",\n",
      "        \"title\": \"Early treatment with inhaled GM-CSF improves oxygenation and anti-viral immunity in COVID-19 induced lung injury \\u2013 a randomized clinical trial\",\n",
      "        \"abstract\": \"\\n Granulocyte-macrophage colony-stimulating factor (GM-CSF) instructs monocytes to differentiate into alveolar macrophages (AM) that preserve lung homeostasis. By comparing AM development in mouse and human, we discovered that COVID-19 patients showed marked defects in GM-CSF-dependent AM instruction. The multi-center, open-label, randomized, controlled SARPAC-trial evaluated the efficacy and safety of 5 days of inhalation of rhu-GM-CSF (sargramostim, Leukine\\u00ae) in 81 non-ventilated patients with COVID-19 and hypoxemic respiratory failure identified by PaO2/FiO2 ratio < 350mmHg. At day 6, more patients in the sargramostim group experienced at least 25% improvement in oxygenation compared with the standard of care group. Higher numbers of circulating class-switched B cells and effector virus-specific CD8 lymphocytes were found in the sargramostim group. Treatment adverse events, including signs of cytokine storm, were not different between active and control group. This proof-of-concept study demonstrates the feasibility and safety of inhaled GM-CSF in restoring alveolar gas exchange, while simultaneously boosting anti-COVID-19 immunity. ClinicalTrials.gov (NCT04326920).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=770&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c9ae1471ad77cb77210133f62cd322a15500b805\",\n",
      "        \"title\": \"The Application of Single-Cell Technologies in Cardiovascular Research\",\n",
      "        \"abstract\": \"Cardiovascular diseases (CVDs) are the leading cause of deaths in the world. The intricacies of the cellular composition and tissue microenvironment in heart and vasculature complicate the dissection of molecular mechanisms of CVDs. Over the past decade, the rapid development of single-cell omics technologies generated vast quantities of information at various biological levels, which have shed light on the cellular and molecular dynamics in cardiovascular development, homeostasis and diseases. Here, we summarize the latest single-cell omics techniques, and show how they have facilitated our understanding of cardiovascular biology. We also briefly discuss the clinical value and future outlook of single-cell applications in the field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=771&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3be1dd73f04421743568dfcc9edd143ca511a8d0\",\n",
      "        \"title\": \"Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=772&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"746e485417500e700917387c030eabe747fb31be\",\n",
      "        \"title\": \"scONE-seq: A one-tube single-cell multi-omics method enables simultaneous dissection of molecular phenotype and genotype heterogeneity from frozen tumors\",\n",
      "        \"abstract\": \"\\n Genomic and transcriptomic heterogeneity both play important roles in normal cellular function as well as in disease development. To be able to characterize these different forms of cellular heterogeneity in diverse sample types, we developed scONE-seq, which enables simultaneous transcriptome and genome profiling in a one-tube reaction. Previous single-cell-whole-genome-RNA-sequencing (scWGS-RNA-seq) methods require physical separation of DNA and RNA, often by physical separation of the nucleus from the cytoplasm. These methods are labor-intensive and technically demanding, time-consuming, or require special devices, and they are not applicable to frozen samples that cannot generate intact single-cell suspensions. scONE-seq is a one-tube reaction, thus is highly scalable and is the first scWGS-RNA-seq method compatible with frozen biobanked tissue. We benchmarked scONE-seq against existing methods using cell lines and lymphocytes from a healthy donor, and we applied it to a 2-year-frozen astrocytoma sample profiling over 1,200 nuclei, subsequently identifying a unique transcriptionally normal-like tumor clone. scONE-seq makes it possible to perform large-scale single-cell multi-omics interrogation with ease on the vast quantities of biobanked tissue, which could transform the scale of future multi-omics single-cell cancer profiling studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=773&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7c8698ad69d17ddac2c903e48809d88ff60f4957\",\n",
      "        \"title\": \"Single cell biology\\u2014a Keystone Symposia report\",\n",
      "        \"abstract\": \"Single cell biology has the potential to elucidate many critical biological processes and diseases, from development and regeneration to cancer. Single cell analyses are uncovering the molecular diversity of cells, revealing a clearer picture of the variation among and between different cell types. New techniques are beginning to unravel how differences in cell state\\u2014transcriptional, epigenetic, and other characteristics\\u2014can lead to different cell fates among genetically identical cells, which underlies complex processes such as embryonic development, drug resistance, response to injury, and cellular reprogramming. Single cell technologies also pose significant challenges relating to processing and analyzing vast amounts of data collected. To realize the potential of single cell technologies, new computational approaches are needed. On March 17\\u201319, 2021, experts in single cell biology met virtually for the Keystone eSymposium \\u201cSingle Cell Biology\\u201d to discuss advances both in single cell applications and technologies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=774&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d7a51d4753994d16f2de69e75e93d3433361e8f5\",\n",
      "        \"title\": \"Multiplex protein analysis for the study of glaucoma\",\n",
      "        \"abstract\": \"ABSTRACT Introduction Glaucoma, a leading cause of irreversible blindness in the world, is a chronic neurodegenerative disease of multifactorial origin. Extensive research is ongoing to better understand, prevent, and treat progressive degeneration of retinal ganglion cells in glaucoma. While experimental models of glaucoma and postmortem tissues of human donors are analyzed for pathophysiological comprehension and improved treatment of this blinding disease, clinical samples of intraocular biofluids and blood collected from glaucoma patients are analyzed to identify predictive, diagnostic, and prognostic biomarkers. Multiplexing techniques for protein analysis offer a valuable approach for translational glaucoma research. Areas covered This review provides an overview of the increasing applications of multiplex protein analysis for glaucoma research and also highlights current research challenges in the field and expected solutions from emerging technological advances. Expert opinion Analytical techniques for multiplex analysis of proteins can help uncover neurodegenerative processes for enhanced treatment of glaucoma and can help identify molecular biomarkers for improved clinical testing and monitoring of this complex disease. This evolving field and continuously growing availability of new technologies are expected to broaden the comprehension of this complex neurodegenerative disease and speed up the progress toward new therapeutics and personalized patient care to prevent blindness from glaucoma.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=775&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d7dbcc2b8f493cf38ac85ca1038e1dbb368e4cd\",\n",
      "        \"title\": \"Identification of fetal liver stroma in spectral cytometry using the parameter autofluorescence\",\n",
      "        \"abstract\": \"The fetal liver is the main hematopoietic organ during embryonic development. The fetal liver is also the unique anatomical site where hematopoietic stem cells expand before colonizing the bone marrow, where they ensure life-long blood cell production and become mostly resting. The identification of the different cell types that comprise the hematopoietic stroma in the fetal liver is essential to understand the signals required for the expansion and differentiation of the hematopoietic stem cells. We used a panel of monoclonal antibodies to identify fetal liver stromal cells in a 5-laser equipped spectral flow cytometry analyzer. The \\u201cAutofluorescence Finder\\u201d of SONY ID7000 software identified two distinct autofluorescence emission spectra. Using autofluorescence as a fluorescence parameter we could assign the two autofluorescent signals to three distinct cell types and identified surface markers that characterize these populations. We found that one autofluorescent population corresponds to hepatoblasts and cholangiocytes whereas the other expresses mesenchymal transcripts and was identified as stellate cells. Importantly, after birth, autofluorescence becomes the unique identifying property of hepatoblasts because mature cholangiocytes are no longer autofluorescent. These results show that autofluorescence used as a parameter in spectral flow cytometry is a useful tool to identify new cell subsets that are difficult to analyze in conventional flow cytometry.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=776&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd1ff60d8293addf6ea1df3dc026b5d54efeeedd\",\n",
      "        \"title\": \"Single-cell transcriptomics reveals the identity and regulators of human mast cell progenitors\",\n",
      "        \"abstract\": \"Mast cell accumulation is a hallmark of a number of diseases including allergic asthma and systemic mastocytosis. IgE-mediated crosslinking of the Fc\\u03b5RI receptors causes mast cell activation and contributes to disease pathogenesis. The mast cell lineage is one of the least studied among the hematopoietic cell lineages and there are still controversies about the identity of the mast cell progenitor, i.e., whether Fc\\u03b5RI expression appears during the hematopoietic progenitor stage or in maturing mast cells. Here, we used single-cell transcriptomics to reveal a temporal association between the appearance of Fc\\u03b5RI and the mast cell gene signature in CD34+ hematopoietic progenitors. In agreement with these data, the Fc\\u03b5RI+ hematopoietic progenitors formed morphologically, phenotypically and functionally mature mast cells in long-term culture assays. Single-cell transcriptomics analysis further revealed the expression patterns of prospective cytokine receptors regulating mast cell progenitor development. Culture assays showed that IL-3 and IL-5 promoted disparate effects on progenitor cell proliferation and survival, respectively, whereas IL-33 caused robust Fc\\u03b5RI downregulation. Taken together, we have demonstrated that Fc\\u03b5RI appears during the hematopoietic progenitor stage of mast cell differentiation and that external stimuli may regulate the Fc\\u03b5RI expression. Thus, the results resolve the controversy regarding the appearance of Fc\\u03b5RI during mast cell development. One-sentence summary Single-cell analysis of human hematopoiesis uncovers the stage at which Fc\\u03b5RI appears during mast cell differentiation and reveals disparate effects of IL-3, IL-5 and IL-33 on mast cell progenitor proliferation, survival, and suppression of Fc\\u03b5RI expression.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=777&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fb2e024eb06fb5b888a03fd09ed0e9301f3ed1ac\",\n",
      "        \"title\": \"\\u2018A picture is worth a thousand words\\u2019: The use of microscopy for imaging neuroinflammation\",\n",
      "        \"abstract\": \"Since the first studies of the nervous system by the Nobel laureates Camillo Golgi and Santiago Ramon y Cajal using simple dyes and conventional light microscopes, microscopy has come a long way to the most recent techniques that make it possible to perform images in live cells and animals in health and disease. Many pathological conditions of the central nervous system have already been linked to inflammatory responses. In this scenario, several available markers and techniques can help imaging and unveil the neuroinflammatory process. Moreover, microscopy imaging techniques have become even more necessary to validate the large quantity of data generated in the era of \\u2018omics\\u2019. This review aims to highlight how to assess neuroinflammation by using microscopy as a tool to provide specific details about the cell\\u2019s architecture during neuroinflammatory conditions. First, we describe specific markers that have been used in light microscopy studies and that are widely applied to unravel and describe neuroinflammatory mechanisms in distinct conditions. Then, we discuss some important methodologies that facilitate the imaging of these markers, such as immunohistochemistry and immunofluorescence techniques. Emphasis will be given to studies using two\\u2010photon microscopy, an approach that revolutionized the real\\u2010time assessment of neuroinflammatory processes. Finally, some studies integrating omics with microscopy will be presented. The fusion of these techniques is developing, but the high amount of data generated from these applications will certainly improve comprehension of the molecular mechanisms involved in neuroinflammation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=778&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2de532ecc56d125c5f235a6b587c3fc3c091326e\",\n",
      "        \"title\": \"The immune niche of the liver.\",\n",
      "        \"abstract\": \"The liver is an essential organ that is critical for the removal of toxins, the production of proteins, and the maintenance of metabolic homeostasis. Behind each liver functional unit, termed lobules, hides a heterogeneous, complex, and well-orchestrated system. Despite parenchymal cells being most commonly associated with the liver's primary functionality, it has become clear that it is the immune niche of the liver that plays a central role in maintaining both local and systemic homeostasis by propagating hepatic inflammation and orchestrating its resolution. As such, the immunological processes that are at play in healthy and diseased livers are being investigated thoroughly in order to understand the underpinnings of inflammation and the potential avenues for restoring homeostasis. This review highlights recent advances in our understanding of the immune niche of the liver and provides perspectives for how the implementation of new transcriptomic, multimodal, and spatial technologies can uncover the heterogeneity, plasticity, and location of hepatic immune populations. Findings from these technologies will further our understanding of liver biology and create a new framework for the identification of therapeutic targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=779&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"625acdb410174062459e4bef41e53dd85c53611b\",\n",
      "        \"title\": \"CellexalVR: A virtual reality platform to visualize and analyze single-cell omics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=780&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b30e2512943798902d0335bcc0bbdfd79362820\",\n",
      "        \"title\": \"Applications of single-cell genomics and computational strategies to study common disease and population-level variation\",\n",
      "        \"abstract\": \"The advent and rapid development of single-cell technologies have made it possible to study cellular heterogeneity at an unprecedented resolution and scale. Cellular heterogeneity underlies phenotypic differences among individuals, and studying cellular heterogeneity is an important step toward our understanding of the disease molecular mechanism. Single-cell technologies offer opportunities to characterize cellular heterogeneity from different angles, but how to link cellular heterogeneity with disease phenotypes requires careful computational analysis. In this article, we will review the current applications of single-cell methods in human disease studies and describe what we have learned so far from existing studies about human genetic variation. As single-cell technologies are becoming widely applicable in human disease studies, population-level studies have become a reality. We will describe how we should go about pursuing and designing these studies, particularly how to select study subjects, how to determine the number of cells to sequence per subject, and the needed sequencing depth per cell. We also discuss computational strategies for the analysis of single-cell data and describe how single-cell data can be integrated with bulk tissue data and data generated from genome-wide association studies. Finally, we point out open problems and future research directions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=781&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"80c07f3640cfb5bb77761d2c91bc5728e3b41a53\",\n",
      "        \"title\": \"Transcriptomics paving the way for improved diagnostics and precision medicine of acute leukemia.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=782&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a8c7eccf21aa80a97be61b58cbe0643591454ee7\",\n",
      "        \"title\": \"Joint single-cell measurements of nuclear proteins and RNA in vivo\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=783&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"80909ff334db3e052046894dfae4ddd3af318bd0\",\n",
      "        \"title\": \"Complex biological questions being addressed using single cell sequencing technologies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=784&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f2ed404f2539de2e126cb89f1fd7ffa4293e9736\",\n",
      "        \"title\": \"Staggered starts in the race to T cell activation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=785&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"62d94f8858a309557b62b381c1cf2f7768a02355\",\n",
      "        \"title\": \"A Survey of Autoencoder Algorithms to Pave the Diagnosis of Rare Diseases\",\n",
      "        \"abstract\": \"Rare diseases (RDs) concern a broad range of disorders and can result from various origins. For a long time, the scientific community was unaware of RDs. Impressive progress has already been made for certain RDs; however, due to the lack of sufficient knowledge, many patients are not diagnosed. Nowadays, the advances in high-throughput sequencing technologies such as whole genome sequencing, single-cell and others, have boosted the understanding of RDs. To extract biological meaning using the data generated by these methods, different analysis techniques have been proposed, including machine learning algorithms. These methods have recently proven to be valuable in the medical field. Among such approaches, unsupervised learning methods via neural networks including autoencoders (AEs) or variational autoencoders (VAEs) have shown promising performances with applications on various type of data and in different contexts, from cancer to healthy patient tissues. In this review, we discuss how AEs and VAEs have been used in biomedical settings. Specifically, we discuss their current applications and the improvements achieved in diagnostic and survival of patients. We focus on the applications in the field of RDs, and we discuss how the employment of AEs and VAEs would enhance RD understanding and diagnosis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=786&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b92a967f5186b8b0dc3f0df26aec6c96b6cf1659\",\n",
      "        \"title\": \"Ectopic cervical thymi and no thymic involution until midlife in naked mole rats\",\n",
      "        \"abstract\": \"Immunosenescence is a hallmark of aging and manifests as increased susceptibility to infection, autoimmunity, and cancer in the elderly. One component of immunosenescence is thymic involution, age\\u2010associated shrinkage of the thymus, observed in all vertebrates studied to date. The naked mole rat (Heterocephalus glaber) has become an attractive animal model in aging research due to its extreme longevity and resistance to disease. Here, we show that naked mole rats display no thymic involution up to 11 years of age. Furthermore, we found large ectopic cervical thymi in addition to the canonical thoracic thymus, both being identical in their cell composition. The developmental landscape in naked mole rat thymi revealed overt differences from the murine T\\u2010cell compartment, most notably a decrease of CD4+/CD8+ double\\u2010positive cells and lower abundance of cytotoxic effector T cells. Our observations suggest that naked mole rats display a delayed immunosenescence. Therapeutic interventions aimed at reversing thymic aging remain limited, underscoring the importance of understanding the cellular and molecular mechanisms behind a sustained immune function in the naked mole rat.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=787&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8ec438059055824d97a6e0d68ad91a284f526b8c\",\n",
      "        \"title\": \"Differential abundance testing on single-cell data using k-nearest neighbor graphs\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=788&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"28aacc689bab499129e82bc530ca1c9bd09af36a\",\n",
      "        \"title\": \"Sparsely Connected Autoencoders: A Multi-Purpose Tool for Single Cell omics Analysis\",\n",
      "        \"abstract\": \"Background: Biological processes are based on complex networks of cells and molecules. Single cell multi-omics is a new tool aiming to provide new incites in the complex network of events controlling the functionality of the cell. Methods: Since single cell technologies provide many sample measurements, they are the ideal environment for the application of Deep Learning and Machine Learning approaches. An autoencoder is composed of an encoder and a decoder sub-model. An autoencoder is a very powerful tool in data compression and noise removal. However, the decoder model remains a black box from which is impossible to depict the contribution of the single input elements. We have recently developed a new class of autoencoders, called Sparsely Connected Autoencoders (SCA), which have the advantage of providing a controlled association among the input layer and the decoder module. This new architecture has the benefit that the decoder model is not a black box anymore and can be used to depict new biologically interesting features from single cell data. Results: Here, we show that SCA hidden layer can grab new information usually hidden in single cell data, like providing clustering on meta-features difficult, i.e. transcription factors expression, or not technically not possible, i.e. miRNA expression, to depict in single cell RNAseq data. Furthermore, SCA representation of cell clusters has the advantage of simulating a conventional bulk RNAseq, which is a data transformation allowing the identification of similarity among independent experiments. Conclusions: In our opinion, SCA represents the bioinformatics version of a universal \\u201cSwiss-knife\\u201d for the extraction of hidden knowledgeable features from single cell omics data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=789&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5c66e55c60c0bccb3fee16abd67bc1ce75841365\",\n",
      "        \"title\": \"Multi-omics approach identifies novel age-, time- and treatment-related immunopathological signatures in MIS-C and pediatric COVID-19\",\n",
      "        \"abstract\": \"Pediatric COVID-19 (pCOVID-19) is rarely severe, however a minority of SARS-CoV-2-infected children may develop MIS-C, a multisystem inflammatory syndrome with significant morbidity. In this longitudinal multi-institutional study, we used multi-omics to identify novel time- and treatment-related immunopathological signatures in children with COVID-19 (n=105) and MIS-C (n=76). pCOVID-19 was characterized by enhanced type I IFN responses, and MIS-C by type II IFN- and NF-{kappa}B dependent responses, matrisome activation, and increased levels of Spike protein. Reduced levels of IL-33 in pCOVID-19, and of CCL22 in MIS-C suggested suppression of Th2 responses. Expansion of TRBV11-2 T-cell clonotypes in MIS-C was associated with inflammation and signatures of T-cell activation, and was reversed by glucocorticoids. The association of MIS-C with the combination of HLA A*02, B*35, C*04 alleles suggests genetic susceptibility. MIS-C B cells showed higher mutation load. Use of IVIG was identified as a confounding factor in the interpretation of autoantibody levels.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=790&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4653ca376a80258b8404cc8a9dce7db4716815dc\",\n",
      "        \"title\": \"Deep learning tackles single-cell analysis - a survey of deep learning for scRNA-seq analysis\",\n",
      "        \"abstract\": \"Since its selection as the method of the year in 2013, single-cell technologies have become mature enough to provide answers to complex research questions. With the growth of single-cell profiling technologies, there has also been a significant increase in data collected from single-cell profilings, resulting in computational challenges to process these massive and complicated datasets. To address these challenges, deep learning (DL) is positioned as a competitive alternative for single-cell analyses besides the traditional machine learning approaches. Here, we survey a total of 25 DL algorithms and their applicability for a specific step in the single cell RNA-seq processing pipeline. Specifically, we establish a unified mathematical representation of variational autoencoder, autoencoder, generative adversarial network and supervised DL models, compare the training strategies and loss functions for these models, and relate the loss functions of these models to specific objectives of the data processing step. Such a presentation will allow readers to choose suitable algorithms for their particular objective at each step in the pipeline. We envision that this survey will serve as an important information portal for learning the application of DL for scRNA-seq analysis and inspire innovative uses of DL to address a broader range of new challenges in emerging multi-omics and spatial single-cell sequencing.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=791&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d1dceb299d3e7b3e53b5d0c1d3ce122a894109a9\",\n",
      "        \"title\": \"Harnessing the Potential of Multiomics Studies for Precision Medicine in Infectious Disease\",\n",
      "        \"abstract\": \"Abstract The field of infectious diseases currently takes a reactive approach and treats infections as they present in patients. Although certain populations are known to be at greater risk of developing infection (eg, immunocompromised), we lack a systems approach to define the true risk of future infection for a patient. Guided by impressive gains in \\u201comics\\u201d technologies, future strategies to infectious diseases should take a precision approach to infection through identification of patients at intermediate and high-risk of infection and deploy targeted preventative measures (ie, prophylaxis). The advances of high-throughput immune profiling by multiomics approaches (ie, transcriptomics, epigenomics, metabolomics, proteomics) hold the promise to identify patients at increased risk of infection and enable risk-stratifying approaches to be applied in the clinic. Integration of patient-specific data using machine learning improves the effectiveness of prediction, providing the necessary technologies needed to propel the field of infectious diseases medicine into the era of personalized medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=792&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6c217c6a3529cfc655c758a2e061f010e6b9e449\",\n",
      "        \"title\": \"Advancing root developmental research through single-cell technologies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=793&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6cf24647ac99614882b9b2f0f423a32130d71233\",\n",
      "        \"title\": \"CombiFlow: combinatorial AML-specific plasma membrane expression profiles allow longitudinal tracking of clones\",\n",
      "        \"abstract\": \"Key Points MRD detection with AML-specific PM markers predicts shorter relapse-free survival. CombiFlow integrates expression data of multiple PM markers that allows longitudinal tracking of clonal evolution.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=794&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ba5448e13d26d03cbde387f8453f147583ffadfc\",\n",
      "        \"title\": \"A new era in functional genomics screens\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=795&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"813b9689a5dd3a0ad22bc54f6d3954b0252536e2\",\n",
      "        \"title\": \"Microfluidic single-cell transcriptomics: moving towards multimodal and spatiotemporal omics.\",\n",
      "        \"abstract\": \"Cells are the basic units of life with vast heterogeneity. Single-cell transcriptomics unveils cell-to-cell gene expression variabilities, discovers novel cell types, and uncovers the critical roles of cellular heterogeneity in biological processes. The recent advances in microfluidic technologies have greatly accelerated the development of single-cell transcriptomics with regard to throughput, sensitivity, cost, and automation. In this article, we review state-of-the-art microfluidic single-cell transcriptomics, with a focus on the methodologies. We first summarize six typical microfluidic platforms for isolation and transcriptomic analysis of single cells. Then the on-going trend of microfluidic transcriptomics towards multimodal omics, which integrates transcriptomics with other omics to provide more comprehensive pictures of gene expression networks, is discussed. We also highlight single-cell spatial transcriptomics and single-cell temporal transcriptomics that provide unprecedented spatiotemporal resolution to reveal transcriptomic dynamics in space and time, respectively. The emerging applications of microfluidic single-cell transcriptomics are also discussed. Finally, we discuss the current challenges to be tackled and provide perspectives on the future development of microfluidic single-cell transcriptomics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=796&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a6ec6dcd4a1d28122e846789657adde2a40649f\",\n",
      "        \"title\": \"Non-linear archetypal analysis of single-cell RNA-seq data by deep autoencoders\",\n",
      "        \"abstract\": \"Advances in single-cell RNA sequencing (scRNA-seq) have led to successes in discovering novel cell types and understanding cellular heterogeneity among complex cell populations through cluster analysis. However, cluster analysis is not able to reveal continuous spectrum of states and underlying gene expression programs (GEPs) shared across cell types. We introduce scAAnet, an autoencoder for single-cell non-linear archetypal analysis, to identify GEPs and infer the relative activity of each GEP across cells. We use a count distribution-based loss term to account for the sparsity and overdispersion of the raw count data and add an archetypal constraint to the loss function of scAAnet. We first show that scAAnet outperforms existing methods for archetypal analysis across different metrics through simulations. We then demonstrate the ability of scAAnet to extract biologically meaningful GEPs using publicly available scRNA-seq datasets including a pancreatic islet dataset, a lung idiopathic pulmonary fibrosis dataset and a prefrontal cortex dataset.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=797&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6496077be7aba5694249f511f5258a28b98e2ccf\",\n",
      "        \"title\": \"Interpretable deep generative models for genomics\",\n",
      "        \"abstract\": \"Deep neural networks implementing generative models for dimensionality reduction have been extensively used for the visualization and analysis of genomic data. One of their key limitations is lack of interpretability: it is challenging to quantitatively identify which input features are used to construct the embedding dimensions, thus preventing insight into why cells are organized in a particular data visualization, for example. Here we present a scalable, interpretable variational autoencoder (siVAE) that is interpretable by design: it learns feature embeddings that guide the interpretation of the cell embeddings in a manner analogous to factor loadings of factor analysis. siVAE is as powerful and nearly as fast to train as the standard VAE but achieves full interpretability of the embedding dimensions. Using siVAE, we exploit a number of connections between dimensionality reduction and gene network inference to identify gene neighborhoods and gene hubs, without the explicit need for gene network inference. We observe a systematic difference in the gene neighborhoods identified by dimensionality reduction methods and gene network inference algorithms in general, suggesting they provide complementary information about the underlying structure of the gene co-expression network. Finally, we apply siVAE to implicitly learn gene networks for individual iPSC lines and uncover a correlation between neuronal differentiation efficiency and loss of co-expression of several mitochondrial complexes, including NADH dehydrogenase, cytochrome C oxidase, and cytochrome b.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=798&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1194c3f6d8b912fb5f6a0281a6157d3a3688a088\",\n",
      "        \"title\": \"Monocyte-derived transcriptome signature indicates antibody-dependent cellular phagocytosis as a potential mechanism of vaccine-induced protection against HIV-1\",\n",
      "        \"abstract\": \"A gene signature was previously found to be correlated with mosaic adenovirus 26 vaccine protection in simian immunodeficiency virus and simian-human immunodeficiency virus challenge models in non-human primates. In this report, we investigated the presence of this signature as a correlate of reduced risk in human clinical trials and potential mechanisms of protection. The absence of this gene signature in the DNA/rAd5 human vaccine trial, which did not show efficacy, strengthens our hypothesis that this signature is only enriched in studies that demonstrated protection. This gene signature was enriched in the partially effective RV144 human trial that administered the ALVAC/protein vaccine, and we find that the signature associates with both decreased risk of HIV-1 acquisition and increased vaccine efficacy (VE). Total RNA-seq in a clinical trial that used the same vaccine regimen as the RV144 HIV vaccine implicated antibody-dependent cellular phagocytosis (ADCP) as a potential mechanism of vaccine protection. CITE-seq profiling of 53 surface markers and transcriptomes of 53,777 single cells from the same trial showed that genes in this signature were primarily expressed in cells belonging to the myeloid lineage, including monocytes, which are major effector cells for ADCP. The consistent association of this transcriptome signature with VE represents a tool both to identify potential mechanisms, as with ADCP here, and to screen novel approaches to accelerate the development of new vaccine candidates.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=799&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8439775fc58f642a7ea99b05fe176dd601ac2615\",\n",
      "        \"title\": \"Cellular heterogeneity of circulating CD4+CD8+ double-positive T cells characterized by single-cell RNA sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=800&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e2f5cf0f51616e63f52d05e873da0cce49293452\",\n",
      "        \"title\": \"A structured evaluation of cryopreservation in generating single cell transcriptomes from cerebrospinal fluid\",\n",
      "        \"abstract\": \"Importance A robust cerebrospinal fluid (CSF) cell cryopreservation protocol using high resolution single-cell (sc) transcriptomic data would enable the deployment of this important modality in multi-center translational research studies and clinical trials in which many sites do not have the expertise or resources to produce data from fresh samples. It would also serve to reduce technical variability in larger projects. Objective To test a reliable cryopreservation protocol adapted for CSF cells, facilitating the characterization of these rare, fragile cells in moderate to large scale studies. Design Diagnostic lumbar punctures were performed on twenty-one patients at two independent sites. Excess CSF was collected and cells were isolated. Each cell sample was split into two fractions for single cell analysis using one of two possible chemistries: 3\\u2019 sc-RNA-Sequencing or 5\\u2019sc-RNA-Sequencing. One cell fraction was processed fresh while the second sample was cryopreserved and profiled at a later time after thawing. Setting The research protocol was deployed at two academic medical centers taking care of multiple sclerosis and other neurological conditions. Participants 21 subjects (age 24 \\u2013 72) were recruited from individuals undergoing a diagnostic lumbar puncture for suspected neuroinflammatory disease or another neurologic illness; they donated excess CSF. Findings Our comparison of fresh and cryopreserved data from the same individuals demonstrates highly efficient recovery of all known CSF cell types. The proportion of all cell types was similar between the fresh and the cryopreserved cells processed, and RNA expression was not significantly different. Results were comparable at both performance sites, and with different single cell sequencing chemistries. Cryopreservation also did not affect recovery of T and B cell clonotype diversity. Conclusion and relevance Our cryopreservation protocol for CSF-cells provides an important alternative to fresh processing of fragile CSF cells: cryopreservation enables the involvement of sites with limited capacity for experimental manipulation and reduces technical variation by enabling batch processing and pooling of samples. Key points Question How efficient is CSF cryopreservation for single-cell transcriptome analysis and can it be implemented in large multi-center translational and clinical trial settings? Findings We compared single-cell transcriptomes of paired fresh and cryopreserved CSF from 21 patients at two independent sites. We validate the efficacy of a simple and cost effective CSF cryopreservation method that preserves the composition and the transcriptomes of CSF cells stored for weeks-months. The protocol is deployed in a large multicenter Phase 4 MS clinical trial. Meaning A validated CSF cryopreservation method that would significantly advance basic science and biomarker research in neurological disorders by implementing single-cell transcriptome analyses in multi-center research and clinical trials.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=801&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"55c017396b5c788bd33d36bbac3182eb97aa6c6d\",\n",
      "        \"title\": \"Characterizing cellular heterogeneity in chromatin state with scCUT&Tag-pro\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=802&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"85c3fccaf0b0e6686550123b9805b528fa3aa154\",\n",
      "        \"title\": \"ProtAnno, an Automated Cell Type Annotation Tool for Single Cell Proteomics Data that integrates information from Multiple Reference Sources\",\n",
      "        \"abstract\": \"Compared with sequencing-based global genomic profiling, cytometry labels targeted surface markers on millions of cells in parallel either by conjugated rare earth metal particles or Unique Molecular Identifier (UMI) barcodes. Correct annotation of these cells to specific cell types is a key step in the analysis of these data. However, there is no computational tool that automatically annotates single cell proteomics data for cell type inference. In this manuscript, we propose an automated single cell proteomics data annotation approach called ProtAnno to facilitate cell type assignments without laborious manual gating. ProtAnno is designed to incorporate information from annotated single cell RNA-seq (scRNA-seq), CITE-seq, and prior data knowledge (which can be imprecise) on biomarkers for different cell types. We have performed extensive simulations to demonstrate the accuracy and robustness of ProtAnno. For several single cell proteomics datasets that have been manually labeled, ProtAnno was able to correctly label most single cells. In summary, ProtAnno offers an accurate and robust tool to automate cell type annotations for large single cell proteomics datasets, and the analysis of such annotated cell types can offer valuable biological insights.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=803&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"507f15da1cf488ce9dbae5d8513007a193ef3f6f\",\n",
      "        \"title\": \"Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose-response study designs\",\n",
      "        \"abstract\": \"The application of single-cell RNA sequencing (scRNAseq) for the evaluation of chemicals, drugs, and food contaminants presents the opportunity to consider cellular heterogeneity in pharmacological and toxicological responses. Current differential gene expression analysis (DGEA) methods focus primarily on two group comparisons, not multi-group dose-response study designs used in safety assessments. To benchmark DGEA methods for dose-response scRNAseq experiments, we proposed a multiplicity corrected Bayesian testing approach and compare it against 8 other methods including two frequentist fit-for-purpose tests using simulated and experimental data. Our Bayesian test method outperformed all other tests for a broad range of accuracy metrics including control of false positive error rates. Most notable, the fit-for-purpose and standard multiple group DGEA methods were superior to the two group scRNAseq methods for dose-response study designs. Collectively, our benchmarking of DGEA methods demonstrates the importance in considering study design when determining the most appropriate test methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=804&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f2ec7b62c4469ff8858530efe05875a5865d38e6\",\n",
      "        \"title\": \"Clustering single cell CITE-seq data with a canonical correlation based deep learning method\",\n",
      "        \"abstract\": \"Single cell sequencing examines the sequence information from individual cells with optimized next generation sequencing (NGS) technologies. It provides researchers a higher resolution of cellular differences and a better understanding of the function of an individual cell in the context of its microenvironment. CITE-seq, or Cellular Indexing of Transcriptomes and Epitopes by sequencing, is one of the latest innovations in the domain of single cell sequencing. It enables researchers to simultaneously capture RNA and surface protein expression on the same cells so that we can correlate the two data types, identify biomarkers and better characterize cell phenotypes. Although multi-omics sequencing technologies developed rapidly, data analyzing methods tailored for multiomics sequencing data are lacking. Several serious problems have to be faced. An important one is how to integrate the information from different modalities, namely scRNA and protein data, efficiently. In this paper, we introduce a canonical correlation based deep learning method called scCTClust for clustering analysis over CITE-seq data. We impute and extract the characteristics of the high dimensional RNA part of data with a ZINB model-based autoencoder. A t-kernel distance is introduced to measure the similarity between query cell and cluster centroids. And the protein data rectifies the feature extraction of scRNA data in a canonical correlation analysis(CCA) way. Extensive data experiments shows that scCTClust can precisely recover the dropout values for RNA sequencing data and extract authentic information from both modalities, getting a much better clustering result than state-of-the-art methods, no matter single-omic or multi-omics clustering algorithms.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=805&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0976e7d80d2b001ddcd1dfd4669bbf2ded8d4699\",\n",
      "        \"title\": \"Ascertaining cells\\u2019 synaptic connections and RNA expression simultaneously with massively barcoded rabies virus libraries\",\n",
      "        \"abstract\": \"Brain function depends on forming and maintaining connections between neurons of specific types, ensuring neural function while allowing the plasticity necessary for cellular and behavioral dynamics. However, systematic descriptions of how brain cell types organize into synaptic networks and which molecules instruct these relationships are not readily available. Here, we introduce SBARRO (Synaptic Barcode Analysis by Retrograde Rabies ReadOut), a method that uses single-cell RNA sequencing to reveal directional, monosynaptic relationships based on the paths of a barcoded rabies virus from its \\u201cstarter\\u201d postsynaptic cell to that cell\\u2019s presynaptic partners1. Thousands of these partner relationships can be ascertained in a single experiment, alongside genome-wide RNA profiles \\u2013 and thus cell identities and molecular states \\u2013 of each host cell. We used SBARRO to describe synaptic networks formed by diverse mouse brain cell types in vitro, leveraging a system similar to those used to identify synaptogenic molecules. We found that the molecular identity (cell type/subtype) of the starter cell predicted the number and types of cells that had synapsed onto it. Rabies transmission tended to occur into cells with RNA-expression signatures related to developmental maturation and synaptic transmission. The estimated size of a cell\\u2019s presynaptic network, relative to that of other cells of the same type, associated with increased expression of Arpp21 and Cdh13. By tracking individual virions and their clonal progeny as they travel among host cells, single-cell, single-virion genomic technologies offer new opportunities to map the synaptic organization of neural circuits in health and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=806&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c2617300cc7481a194c4a5ef93ffb60416335166\",\n",
      "        \"title\": \"MOSCATO: a supervised approach for analyzing multi-Omic single-Cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=807&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6fad350dd0c87bbf61e4bd25b16eb4620f30daba\",\n",
      "        \"title\": \"Single-cell proteo-genomic reference maps of the hematopoietic system enable the purification and massive profiling of precisely defined cell states\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=808&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1ba00597a0b1f7c0ac315e54c19875bd62b26d24\",\n",
      "        \"title\": \"Myeloid Heterogeneity in Kidney Disease as Revealed through Single-Cell RNA Sequencing\",\n",
      "        \"abstract\": \"Kidney disease represents a global health burden of increasing prevalence and is an independent risk factor for cardiovascular disease. Myeloid cells are a major cellular compartment of the immune system; they are found in the healthy kidney and in increased numbers in the damaged and/or diseased kidney, where they act as key players in the progression of injury, inflammation, and fibrosis. They possess enormous plasticity and heterogeneity, adopting different phenotypic and functional characteristics in response to stimuli in the local milieu. Although this inherent complexity remains to be fully understood in the kidney, advances in single-cell genomics promise to change this. Specifically, single-cell RNA sequencing (scRNA-seq) has had a transformative effect on kidney research, enabling the profiling and analysis of the transcriptomes of single cells at unprecedented resolution and throughput, and subsequent generation of cell atlases. Moving forward, combining scRNA- and single-nuclear RNA-seq with greater-resolution spatial transcriptomics will allow spatial mapping of kidney disease of varying etiology to further reveal the patterning of immune cells and nonimmune renal cells. This review summarizes the roles of myeloid cells in kidney health and disease, the experimental workflow in currently available scRNA-seq technologies, and published findings using scRNA-seq in the context of myeloid cells and the kidney.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=809&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e14eb84f65e198bfc33c335e51bd8682e8d2823e\",\n",
      "        \"title\": \"Covarying neighborhood analysis identifies cell populations associated with phenotypes of interest from single-cell transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=810&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a284348773017b042e08ed8a98424e845cbf9b8d\",\n",
      "        \"title\": \"An overview of multiplexed analyses of CAR T-cell therapies: insights and potential\",\n",
      "        \"abstract\": \"ABSTRACT Introduction Cancer immunotherapy is a rapidly growing field with exponential advancement in engineered immune cell-based therapies. For instance, an engineered chimeric antigen receptor (CAR) can be introduced in T-cells or other immune cells and adoptively transferred to target and kill cancer cells in hematologic malignancies or solid tumors. The first CAR-T-cell (CAR-T) therapy has been developed against CD19, a B-cell marker expressed on lymphoma and lymphoblastic leukemia. To allow for personalized treatment, proteomics approaches could provide insights into biomarkers for CAR-T therapy efficacy and toxicity. Areas Covered We researched the most recent technology methods of biomarker evaluation used in the laboratory and clinical setting. Publications of CAR-T biomarkers were then systematically reviewed to provide a narrative of the most validated biomarkers of CAR-T efficacy and toxicity. Examples of biomarkers include CAR-T functionality and phenotype as well as interleukin-6 and other cytokines. Expert Commentary Biomarkers of CAR-T efficacy and toxicity have been identified, but still need to be validated and standardized across institutions. Moreover, few are used in the clinical setting due to limitations in real-time technology. Expansion of biomarker research could provide better understanding of patient response and risk of life-threatening side effects with potential for improved precision medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=811&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5455167c28d1093bd45b48e59e0045dc8c9c746\",\n",
      "        \"title\": \"Aberrant Extrafollicular B Cells, Immune Dysfunction, Myeloid Inflammation, and MyD88-Mutant Progenitors Precede Waldenstrom Macroglobulinemia.\",\n",
      "        \"abstract\": \"Waldenstrom macroglobulinemia (WM) and its precursor IgM gammopathy are distinct disorders characterized by clonal mature IgM-expressing B-cell outgrowth in the bone marrow. Here, we show by high-dimensional single-cell immunogenomic profiling of patient samples that these disorders originate in the setting of global B-cell compartment alterations, characterized by expansion of genomically aberrant extrafollicular B cells of the nonmalignant clonotype. Alterations in the immune microenvironment preceding malignant clonal expansion include myeloid inflammation and na\\u00efve B- and T-cell depletion. Host response to these early lesions involves clone-specific T-cell immunity that may include MYD88 mutation-specific responses. Hematopoietic progenitors carry the oncogenic MYD88 mutations characteristic of the malignant WM clone. These data support a model for WM pathogenesis wherein oncogenic alterations and signaling in progenitors, myeloid inflammation, and global alterations in extrafollicular B cells create the milieu promoting extranodal pattern of growth in differentiated malignant cells.\\n\\n\\nSignificance\\nThese data provide evidence that growth of the malignant clone in WM is preceded by expansion of extrafollicular B cells, myeloid inflammation, and immune dysfunction in the preneoplastic phase. These changes may be related in part to MYD88 oncogenic signaling in pre-B progenitor cells and suggest a novel model for WM pathogenesis. This article is highlighted in the In This Issue feature, p. 549.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=812&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"41aac74dcf7e2c02fec101ae0ad7329254b4b920\",\n",
      "        \"title\": \"A single-cell and spatially resolved atlas of human breast cancers\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=813&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18862c269a01e56567adf1330714824d35d4cee2\",\n",
      "        \"title\": \"High-throughput single-cell quantification of hundreds of proteins using conventional flow cytometry and machine learning\",\n",
      "        \"abstract\": \"This study presents a novel method for low-cost cell surface proteomics using flow cytometry and machine learning.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=814&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd3ba4a4b78112ca0e8edcd494b985cfece512ff\",\n",
      "        \"title\": \"Single-cell intracellular epitope and transcript detection reveals signal transduction dynamics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=815&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"80bba54f1c914ff4f6c45379fb6227683509dbc1\",\n",
      "        \"title\": \"Towards a Systems Immunology Approach to Understanding Correlates of Protective Immunity against HCV\",\n",
      "        \"abstract\": \"Over the past decade, tremendous progress has been made in systems biology-based approaches to studying immunity to viral infections and responses to vaccines. These approaches that integrate multiple facets of the immune response, including transcriptomics, serology and immune functions, are now being applied to understand correlates of protective immunity against hepatitis C virus (HCV) infection and to inform vaccine development. This review focuses on recent progress in understanding immunity to HCV using systems biology, specifically transcriptomic and epigenetic studies. It also examines proposed strategies moving forward towards an integrated systems immunology approach for predicting and evaluating the efficacy of the next generation of HCV vaccines.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=816&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f710ddcce2fa27cd172fbd6937b11af636ab8ab9\",\n",
      "        \"title\": \"Mapping interindividual dynamics of innate immune response at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=817&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9ff3c957c88e67b48154942cae4870b674987577\",\n",
      "        \"title\": \"Single-Cell Omics in Dissecting Immune Microenvironment of Malignant Gliomas\\u2014Challenges and Perspectives\",\n",
      "        \"abstract\": \"Single-cell technologies allow precise identification of tumor composition at the single-cell level, providing high-resolution insights into the intratumoral heterogeneity and transcriptional activity of cells in the tumor microenvironment (TME) that previous approaches failed to capture. Malignant gliomas, the most common primary brain tumors in adults, are genetically heterogeneous and their TME consists of various stromal and immune cells playing an important role in tumor progression and responses to therapies. Previous gene expression or immunocytochemical studies of immune cells infiltrating TME of malignant gliomas failed to dissect their functional phenotypes. Single-cell RNA sequencing (scRNA-seq) and cytometry by time-of-flight (CyTOF) are powerful techniques allowing quantification of whole transcriptomes or >30 protein targets in individual cells. Both methods provide unprecedented resolution of TME. We summarize the findings from these studies and the current state of knowledge of a functional diversity of immune infiltrates in malignant gliomas with different genetic alterations. A precise definition of functional phenotypes of myeloid and lymphoid cells might be essential for designing effective immunotherapies. Single-cell omics studies have identified crucial cell subpopulations and signaling pathways that promote tumor progression, influence patient survival or make tumors vulnerable to immunotherapy. We anticipate that the widespread usage of single-cell omics would allow rational design of oncoimmunotherapeutics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=818&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4015ccdede517bdb649c1f90670fe31e5f14b485\",\n",
      "        \"title\": \"In vivo Pooled Screening: A Scalable Tool to Study the Complexity of Aging and Age-Related Disease\",\n",
      "        \"abstract\": \"Biological aging, and the diseases of aging, occur in a complex in vivo environment, driven by multiple interacting processes. A convergence of recently developed technologies has enabled in vivo pooled screening: direct administration of a library of different perturbations to a living animal, with a subsequent readout that distinguishes the identity of each perturbation and its effect on individual cells within the animal. Such screens hold promise for efficiently applying functional genomics to aging processes in the full richness of the in vivo setting. In this review, we describe the technologies behind in vivo pooled screening, including a range of options for delivery, perturbation and readout methods, and outline their potential application to aging and age-related disease. We then suggest how in vivo pooled screening, together with emerging innovations in each of its technological underpinnings, could be extended to shed light on key open questions in aging biology, including the mechanisms and limits of epigenetic reprogramming and identifying cellular mediators of systemic signals in aging.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=819&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cec0cf40c589ecaa6d779b8433d96393fdcd9fdc\",\n",
      "        \"title\": \"Mapping single-cell data to reference atlases by transfer learning\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=820&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"900f95e7f075f559eea4557af2611fc13d67bdcb\",\n",
      "        \"title\": \"Helios represses megakaryocyte priming in hematopoietic stem and progenitor cells\",\n",
      "        \"abstract\": \"The mechanism controlling lineage commitment in early hematopoiesis is unclear. Cova et al. show that the Helios transcription factor is required to actively suppress megakaryocyte priming in hematopoietic stem and progenitor cells, which contributes to increased stem cell pluripotency and lymphoid commitment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=821&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"10aee48025f19121b79b449fa5a36fb4ff3d4d2c\",\n",
      "        \"title\": \"Single\\u2010cell omics: Overview, analysis, and application in biomedical science\",\n",
      "        \"abstract\": \"Single\\u2010cell sequencing methods provide the highest resolution insight into cellular heterogeneity. Owing to their rapid growth and decreasing cost, they are now widely accessible to scientists worldwide. Single\\u2010cell technologies enable analysis of a large number of cells, making them powerful tools to characterise rare cell types and refine our understanding of diverse cell states. Moreover, single\\u2010cell application in biomedical sciences helps to unravel mechanisms related to disease pathogenesis and outcome. In this Viewpoint, we briefly describe existing single\\u2010cell methods (genomics, transcriptomics, epigenomics, proteomics, and mulitomics), comment on available analysis tools, and give examples of method applications in the biomedical field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=822&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"07ca0007193424e3fe94960d9fafdb5049faa182\",\n",
      "        \"title\": \"SampleQC: robust multivariate, multi-cell type, multi-sample quality control for single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=823&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e5a1745ec401bf602021d4f221a6f53005593ee7\",\n",
      "        \"title\": \"An in\\u00a0vitro platform supports generation of human innate lymphoid cells from CD34+ hematopoietic progenitors that recapitulate ex\\u00a0vivo identity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=824&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e318ffeb5116f94420c34f3a99ff64ff0779e73a\",\n",
      "        \"title\": \"Multi-omics integration in the age of million single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=825&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"284652df432642e7029d23def7c99ba0165b208f\",\n",
      "        \"title\": \"Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and subsequent weight regain\",\n",
      "        \"abstract\": \"Most individuals do not maintain weight loss, and weight regain increases cardio-metabolic risk beyond that of obesity. Adipose inflammation directly contributes to insulin resistance; however, immune-related changes that occur with weight loss and weight regain are not well understood. Single cell RNA-sequencing was completed with CITE-sequencing and biological replicates to profile changes in murine immune subpopulations following obesity, weight loss, and weight cycling. Weight loss normalized glucose tolerance, however, type 2 immune cells did not repopulate adipose following weight loss. Many inflammatory populations persisted with weight loss and increased further following weight regain. Obesity drove T cell exhaustion and broad increases in antigen presentation, lipid handing, and inflammation that persisted with weight loss and weight cycling. This work provides critical groundwork for understanding the immunological causes of weight cycling-accelerated metabolic disease. Thus, we have created an open-access interactive portal for our processed data to improve accessibility for the research community.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=826&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5be57cb75a515bb75db93431d94eb27b5e9e365f\",\n",
      "        \"title\": \"Multiomic analysis reveals decidual\\u2010specific transcriptional programing of MAIT cells\",\n",
      "        \"abstract\": \"Mucosal\\u2010Associated Invariant T (MAIT) cells have been recently identified at the maternal\\u2010fetal interface. However, transcriptional programming of decidual MAIT cells in pregnancy remains poorly understood.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=827&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5450c58077701cfb95ab294ab8a5c7c00cbea650\",\n",
      "        \"title\": \"CD11b+ lung dendritic cells at different stages of maturation induce Th17 or Th2 differentiation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=828&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50e2efed55e835e34cbf520f41eebbf73dc7a3fa\",\n",
      "        \"title\": \"New Views of Old Proteins: Clarifying the Enigmatic Proteome\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=829&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6e76a62e882740ed12d29a7dc1fca99a9fcae30f\",\n",
      "        \"title\": \"POLYseq: A poly(\\u03b2-amino ester)-based vector for multifunctional cellular barcoding\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=830&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17397cb3e6f76b35afdfc239ad60877187d6d710\",\n",
      "        \"title\": \"Over 1000 tools reveal trends in the single-cell RNA-seq analysis landscape\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=831&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc19199a52dc7cef36312bed1965b2a3078b95a9\",\n",
      "        \"title\": \"The Evolution of Single-Cell Analysis and Utility in Drug Development\",\n",
      "        \"abstract\": \"This review provides a brief history of the advances of cellular analysis tools focusing on instrumentation, detection probes, and data analysis tools. The interplay of technological advancement and a deeper understanding of cellular biology are emphasized. The relevance of this topic to drug development is that the evaluation of cellular biomarkers has become a critical component of the development strategy for novel immune therapies, cell therapies, gene therapies, antiviral therapies, and vaccines. Moreover, recent technological advances in single-cell analysis are providing more robust cellular measurements and thus accelerating the advancement of novel therapies. Graphical abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=832&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1efb781438c6156e647de93e7a031df0836218ff\",\n",
      "        \"title\": \"Plant single-cell solutions for energy and the environment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=833&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ee31f1e588132b32138747e907f1cf4585b39172\",\n",
      "        \"title\": \"A Meta-Analytic Single-Cell Atlas of Mouse Bone Marrow Hematopoietic Development\",\n",
      "        \"abstract\": \"The clinical importance of the hematopoietic system makes it one of the most heavily studied lineages in all of biology. A clear understanding of the cell types and functional programs during hematopoietic development is central to research in aging, cancer, and infectious diseases. Known cell types are traditionally identified by the expression of proteins on the surface of the cells. Stem and progenitor cells defined based on these markers are assigned functions based on their lineage potential. The rapid growth of single cell RNA sequencing technologies (scRNAseq) provides a new modality for evaluating the cellular and functional landscape of hematopoietic stem and progenitor cells. The popularity of this technology among hematopoiesis researchers enables us to conduct a robust meta-analysis of mouse bone marrow scRNAseq data. Using over 300,000 cells across 12 datasets, we evaluate the classification and function of cell types based on discrete clustering, in silico FACS sorting, and a continuous trajectory. We identify replicable signatures that define cell types based on genes and known cellular functions. Additionally, we evaluate the conservation of signatures associated with erythroid and monocyte lineage development across species using co-expression networks. The co-expression networks predict the effectiveness of the signature at identifying erythroid and monocyte cells in zebrafish and human scRNAseq data. Together, this analysis provides a robust reference, particularly marker genes and functional annotations, for future experiments in hematopoietic development. Key Points Meta-analysis of 9 mouse bone marrow scRNAseq identifies markers for cell types and hematopoietic development Characterize a replicable functional landscape of cell types by exploiting co-expression\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=834&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7da1e7bf39222b745b0afb97d475f6425d098eb5\",\n",
      "        \"title\": \"LRcell: detecting the source of differential expression at the sub-cell type level from bulk RNA-seq data\",\n",
      "        \"abstract\": \"Given most tissues are consist of abundant and diverse sub cell-types, an important yet unaddressed problem in bulk RNA-seq analysis is to identify at which sub cell-type(s) the differential expression occur. Single-cell RNA-sequencing (scRNA-seq) technologies can answer the question, but they are often labor-intensive and cost-prohibitive. Here, we present LRcell, a computational method aiming to identify specific sub-cell type(s) that drives the changes observed in a bulk RNA-seq experiment. In addition, LRcell provides pre-embedded marker genes computed from putative single-cell RNA-seq experiments as options to execute the analyses. Using three different real datasets, we show that LRcell successfully identifies known cell types involved in psychiatric disorders and LRcell is more sensitive than even the leading deconvolution methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=835&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a115288accce07524ae263e293951df6e10a47a4\",\n",
      "        \"title\": \"Effector differentiation downstream of lineage commitment in ILC1 is driven by Hobit across tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=836&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e6756ea5f8da195d6ff0c04d9463cc637a9d9bd8\",\n",
      "        \"title\": \"Community-wide hackathons to identify central themes in single-cell multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=837&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ca49361eb5c29d0e920837b0ddb3701120c36ba1\",\n",
      "        \"title\": \"Current tools to interrogate microglial biology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=838&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59ffb028dcd706841bef118924ebd926bed445b4\",\n",
      "        \"title\": \"TAS-Seq is a robust and sensitive amplification method for bead-based scRNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=839&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d56f1237d5b88096942551e1c78273aedc89479d\",\n",
      "        \"title\": \"Single-cell multimodal analysis in a case with reduced penetrance of Progranulin-Frontotemporal Dementia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=840&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9bf61818a623139a6c07877b58783ff30501d6e\",\n",
      "        \"title\": \"Ectopic cervical thymi and no thymic involution until midlife in naked mole-rats\",\n",
      "        \"abstract\": \"Immunosenescence is a hallmark of aging and manifests as increased susceptibility to infection, autoimmunity, and cancer in the elderly. One component of immunosenescence is thymic involution, age-associated shrinkage of the thymus, observed in all vertebrates studied to date. The naked mole-rat (Heterocephalus glaber) has become an attractive animal model in aging research due to its extreme longevity and resistance to disease. Here we show that naked mole rats display no thymic involution up to 11 years of age. Furthermore, we found large ectopic cervical thymi in addition to the canonical thoracic thymus, both being identical in their cell composition. The developmental landscape in naked mole-rat thymi revealed overt differences from the murine T cell compartment, most notably a decrease of CD4+/CD8+ double-positive cells and lower abundance of cytotoxic effector T cells. Our observations suggest that naked mole rats display a delayed immunosenescence. Therapeutic interventions aimed at reversing thymic aging remain limited, underscoring the importance of understanding the cellular and molecular mechanisms behind a sustained immune function in the naked mole rat.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=841&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6e66c218d6855be0afd147fb1518329c13d7bbb5\",\n",
      "        \"title\": \"An abundant myeloid progenitor shapes neonatal hematopoiesis of naked mole-rats\",\n",
      "        \"abstract\": \"The naked mole-rat became a significant animal model for its exceptional longevity and resistance mechanisms to cancers, however the developmental properties of their native stem and progenitor compartments are poorly understood. Here we report a high frequency bone marrow progenitor exclusively present in neonatal animals. The heterotypic developmental stage was marked by an immunophenotype resembling adult naked mole-rat CMP-like progenitors with mature neutrophilic Thy1.1-antigen levels and showed a clear differentiation bias and transcriptome signature towards myelopoiesis. Importantly, myeloid progenitors overexpressed both CXCL12 and its alternative receptor CXCR7, thereby abrogating their homing capacity and produced multiple secreted matrix components related to neonatal bone marrow niche formation. These results indicate a transient ontogenic cell stage during postnatal hematopoiesis unique to naked mole-rats and further advances our understanding of the developmental steps involved in creating the regenerative cell pool and its environment in a long-lived species.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=842&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dc390bde5ffa2123a96a75cbc2860ee7acf1ef7f\",\n",
      "        \"title\": \"Multi-modal profiling of human fetal liver hematopoietic stem cells reveals the molecular signature of engraftment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=843&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d89835d4803ed24b0a78873ce7b888244ad267f\",\n",
      "        \"title\": \"Naked Mole-Rat Hematopoietic Stem and Progenitors are Highly Quiescent with an Inherent Myeloid Bias\",\n",
      "        \"abstract\": \"Naked mole-rats (NMRs) are the longest-lived rodents yet their stem cell characteristics remain enigmatic. Here we comprehensively mapped the NMR hematopoietic landscape and identified unique features likely contributing to longevity. Adult NMRs form red blood cells in spleen and marrow, which is a neotenic trait. A myeloid bias towards granulopoiesis in concert with decreased B-lymphopoiesis defines the marrow composition, resembling fetal leukopoiesis. Very similar to primates, the primitive stem cell compartment is marked by CD34 and THY1. Remarkably, stem and progenitor respiration rates are as low as in human cells, while NMR cells show a strong expression signature for fatty acid metabolism. The pool of quiescent stem cells is higher than in mice, and the cell cycle of hematopoietic cells is prolonged. Our work provides a platform to study immunology and stem cell biology in an animal model of exceptional longevity. Teaser Juvenile features of hematopoiesis shape the blood system of the longest-lived rodent.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=844&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd5ff32f92840ed92e4f40eef7c3fa5b99131f83\",\n",
      "        \"title\": \"Exploring tissue architecture using spatial transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=845&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3be99f66fc2ec2c80993820eae975b793f264a6f\",\n",
      "        \"title\": \"Clinical Perspectives of Single-Cell RNA Sequencing\",\n",
      "        \"abstract\": \"The ability of single-cell genomics to resolve cellular heterogeneity is highly appreciated in cancer and is being exploited for precision medicine. In the recent decade, we have witnessed the incorporation of cancer genomics into the clinical decision-making process for molecular-targeted therapies. Compared with conventional genomics, which primarily focuses on the specific and sensitive detection of the molecular targets, single-cell genomics addresses intratumoral heterogeneity and the microenvironmental components impacting the treatment response and resistance. As an exploratory tool, single-cell genomics provides an unprecedented opportunity to improve the diagnosis, monitoring, and treatment of cancer. The results obtained upon employing bulk cancer genomics indicate that single-cell genomics is at an early stage with respect to exploration of clinical relevance and requires further innovations to become a widely utilized technology in the clinic.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=846&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"73e303076cc8024f5302e4577533ffda1c41e713\",\n",
      "        \"title\": \"Progress in the production of haematopoietic stem and progenitor cells from human pluripotent stem cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=847&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eb381b263ca20a3cb5aed48ee44dd0331fb26d6d\",\n",
      "        \"title\": \"NEAT-seq: simultaneous profiling of intra-nuclear proteins, chromatin accessibility and gene expression in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=848&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7149fe93a008dadbf6848da861cd0baf27a919b2\",\n",
      "        \"title\": \"Modeling memory T cell states at single-cell resolution identifies in vivo state-dependence of eQTLs influencing disease\",\n",
      "        \"abstract\": \"Many non-coding genetic variants cause disease by modulating gene expression. However, identifying these expression quantitative trait loci (eQTLs) is complicated by gene-regulation differences between cell states. T cells, for example, have fluid, multifaceted functional states in vivo that cannot be modeled in eQTL studies that aggregate cells. Here, we modeled T cell states and eQTLs at single-cell resolution. Using >500,000 resting memory T cells from 259 Peruvians, we found over one-third of the 6,511 cis-eQTLs had state-dependent effects. By integrating single-cell RNA and surface protein measurements, we defined continuous cell states that explained more eQTL variation than discrete states like CD4+ or CD8+ T cells and could have opposing effects on independent eQTL variants in a locus. Autoimmune variants were enriched in cell-state-dependent eQTLs, such as a rheumatoid-arthritis variant near ORMDL3 strongest in cytotoxic CD8+ T cells. These results argue that fine-grained cell state context is crucial to understanding disease-associated eQTLs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=849&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d3cacb60051002cf9e1c2a189f165b9383cb6fa\",\n",
      "        \"title\": \"Single cell analysis of endometriosis reveals a coordinated transcriptional program driving immunotolerance and angiogenesis across eutopic and ectopic tissues\",\n",
      "        \"abstract\": \"Endometriosis is characterized by growth of endometrial-like tissue outside of the uterus affecting many women in their reproductive age, causing years of pelvic pain and potential infertility. Its pathophysiology remains largely unknown, limiting diagnosis and treatment. We characterized peritoneal and ovarian lesions at single-cell transcriptome resolution and compared to matched eutopic endometrium, control endometrium, and organoids derived from these tissues, generating data on over 100,000 cells across 12 individuals. We spatially localized many of the cell types using imaging mass cytometry. We identify a perivascular mural cell unique to the peritoneal lesions with dual roles in angiogenesis promotion and immune cell trafficking. We define an immunotolerant peritoneal niche, fundamental differences in eutopic endometrium and between lesions microenvironments, and a novel progenitor-like epithelial cell subpopulation. Altogether, this study provides a holistic view of the endometriosis microenvironment representing the first comprehensive cell atlas of the disease, essential information for advancing therapeutics and diagnostics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=850&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"daf606e7cfcb1b3929331bbed05139bd45cebf49\",\n",
      "        \"title\": \"Host cell maturation modulates parasite invasion and sexual differentiation in Plasmodium\",\n",
      "        \"abstract\": \"Malaria remains a global health problem with over 400,000 deaths annually1. Plasmodium parasites, the causative agents of malaria, replicate asexually in red blood cells (RBCs) of their vertebrate host, while a subset differentiates into sexual stages (gametocytes) for mosquito transmission. Parasite replication and gametocyte maturation in the erythropoietic niches of the bone marrow and spleen contribute to pathogenesis and drive transmission2, but the mechanisms underlying this organ enrichment remain unknown. We performed a comprehensive single cell analysis of rodent P. berghei in spleen, bone marrow and blood to define parasite phenotypes specific to those niches. Single cell RNA-seq analysis of host and parasite cells reveals an interferon-driven host response to infection as well as transcriptional adaptations of Plasmodium to RBC maturation status. We show that P. berghei exhibits a bimodal invasion pattern into either normocytes or early reticulocytes and, using functional assays, identify CD71 as a host receptor for reticulocyte invasion. Importantly, we observe an increased rate of gametocyte formation in reticulocytes that is nutrient-dependent and triggered post invasion (i.e., same cycle sexual commitment). Our data provides a thorough characterisation of host-parasite interactions in erythropoietic niches and defines host cell maturation state as the key driver of parasite adaptation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=851&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"10164c6305c377875d952d929373f0d9489d7dbc\",\n",
      "        \"title\": \"RCA2: a scalable supervised clustering algorithm that reduces batch effects in scRNA-seq data\",\n",
      "        \"abstract\": \"Abstract The transcriptomic diversity of cell types in the human body can be analysed in unprecedented detail using single cell (SC) technologies. Unsupervised clustering of SC transcriptomes, which is the default technique for defining cell types, is prone to group cells by technical, rather than biological, variation. Compared to de-novo (unsupervised) clustering, we demonstrate using multiple benchmarks that supervised clustering, which uses reference transcriptomes as a guide, is robust to batch effects and data quality artifacts. Here, we present RCA2, the first algorithm to combine reference projection (batch effect robustness) with graph-based clustering (scalability). In addition, RCA2 provides a user-friendly framework incorporating multiple commonly used downstream analysis modules. RCA2 also provides new reference panels for human and mouse and supports generation of custom panels. Furthermore, RCA2 facilitates cell type-specific QC, which is essential for accurate clustering of data from heterogeneous tissues. We demonstrate the advantages of RCA2 on SC data from human bone marrow, healthy PBMCs and PBMCs from COVID-19 patients. Scalable supervised clustering methods such as RCA2 will facilitate unified analysis of cohort-scale SC datasets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=852&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0e1956fa488ec6baa91ea39cc5932c1d18381820\",\n",
      "        \"title\": \"Comprehensive multi-omics single-cell data integration reveals greater heterogeneity in the human immune system\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=853&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ddd551c572aa4f1b8dcad2c7236ea37f8c285c76\",\n",
      "        \"title\": \"Leukocytes with chromosome Y loss have reduced abundance of the cell surface immunoprotein CD99\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=854&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"78859c1b1629416f74a12ea18889b69aeaa90f5b\",\n",
      "        \"title\": \"Improved integration of single-cell transcriptome and surface protein expression by LinQ-View\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=855&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa8e70ff822012fe41de20585a5c112868aeb294\",\n",
      "        \"title\": \"Single cell analysis of M. tuberculosis phenotype and macrophage lineages in the infected lung\",\n",
      "        \"abstract\": \"Through scRNA-seq and ATAC-seq, Pisu et al. characterize specific lung macrophage subsets that either restrict or promote M. tuberculosis growth. Comparable macrophage populations are present in the human airways, and cells show evidence of epigenetic imprinting.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=856&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"12b0b3a908fd4801fe52b285953fc7c8bb1b19b9\",\n",
      "        \"title\": \"Direct cardiac reprogramming comes of age: Recent advance and remaining challenges.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=857&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9236ad8d58a1330845fd9d73955690335ee76525\",\n",
      "        \"title\": \"ELAVL4, splicing, and glutamatergic dysfunction precede neuron loss in MAPT mutation cerebral organoids\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=858&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d44b3572fb0f9416d9b1fdc0e04e820a5d994936\",\n",
      "        \"title\": \"Phenotype, specificity and avidity of antitumour CD8+ T cells in melanoma\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=859&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7ed5c12002145173492f65f1e0137c7ba947bc71\",\n",
      "        \"title\": \"Single-cell analysis of human skin identifies CD14+ type 3 dendritic cells co-producing IL1B and IL23A in psoriasis\",\n",
      "        \"abstract\": \"Skin index-sorted single-cell RNA sequencing reveals the transcriptional landscape of macrophages and dendritic cells in atopic and psoriatic skin and identifies CD14+ type 3 dendritic cells co-producing IL1B and IL23A in psoriatic skin.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=860&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"471eba65c98c464da909d699c05a3998033e204f\",\n",
      "        \"title\": \"High-throughput and single-cell T cell receptor sequencing technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=861&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e381667e72241480976b5a7a281cf6632f1927fe\",\n",
      "        \"title\": \"Inhibition of MET Signaling with Ficlatuzumab in Combination with Chemotherapy in Refractory AML: Clinical Outcomes and High-Dimensional Analysis.\",\n",
      "        \"abstract\": \"Acute myeloid leukemia patients refractory to induction therapy or relapsed within one year have poor outcomes. Autocrine production of hepatocyte growth factor by myeloid blasts drives leukemogenesis in pre-clinical models. A phase Ib trial evaluated ficlatuzumab, a first-in-class anti-HGF antibody, in combination with cytarabine in this high-risk population. Dose-limiting toxicities were not observed, and 20 mg/kg was established as the recommended phase II dose. The most frequent treatment-related adverse event was febrile neutropenia. Among 17 evaluable patients, the overall response rate was 53%, all complete remissions. Phospho-proteomic mass cytometry showed potent on-target suppression of p-MET after ficlatuzumab treatment and that attenuation of p-S6 was associated with clinical response. Multiplexed single cell RNA sequencing using prospectively acquired patient specimens identified interferon response genes as adverse predictive factors. The ficlatuzumab and cytarabine combination is well-tolerated with favorable efficacy. High-dimensional analyses at single-cell resolution represent promising approaches for identifying biomarkers of response and mechanisms of resistance in prospective clinical studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=862&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1c9ec5485ec57f4e8c3b75c09c3f4ff582144793\",\n",
      "        \"title\": \"From bench to bedside: single-cell analysis for cancer immunotherapy.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=863&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d2afdb3bec8fb22818cbe47c9289a51c1b8e7f75\",\n",
      "        \"title\": \"Spatial mapping of protein composition and tissue organization: a primer for multiplexed antibody-based imaging\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=864&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f50e7e85be4bd48f0da904b1ffffb07d9a86b06\",\n",
      "        \"title\": \"Immune cell profiling in atherosclerosis: role in research and precision medicine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=865&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff701532495876a1b2846aad0f293c6a9cac8d86\",\n",
      "        \"title\": \"Machine learning methods to model multicellular complexity and tissue specificity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=866&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e836bd470569effe0f6fd0e423bda886c150e469\",\n",
      "        \"title\": \"Single-Cell RNA Sequencing Approaches for Tracing T Cell Development\",\n",
      "        \"abstract\": \"T cell development occurs in the thymus, where uncommitted progenitors are directed into a range of sublineages with distinct functions. The goal is to generate a TCR repertoire diverse enough to recognize potential pathogens while remaining tolerant of self. Decades of intensive research have characterized the transcriptional programs controlling critical differentiation checkpoints at the population level. However, greater precision regarding how and when these programs orchestrate differentiation at the single-cell level is required. Single-cell RNA sequencing approaches are now being brought to bear on this question, to track the identity of cells and analyze their gene expression programs at a resolution not previously possible. In this review, we discuss recent advances in the application of these technologies that have the potential to yield unprecedented insight to T cell development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=867&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"11ee3199f2c264a30b19413d541f3d6cc87106cb\",\n",
      "        \"title\": \"Breast cancer heterogeneity through the lens of single-cell analysis and spatial pathologies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=868&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7ba00b62adf7e02e97a687578f71755d10b56efd\",\n",
      "        \"title\": \"Deconvolution of Focal Segmental Glomerulosclerosis Pathophysiology Using Transcriptomics Techniques\",\n",
      "        \"abstract\": \"Background: Focal segmental glomerulosclerosis is a histopathological pattern of renal injury and comprises a heterogeneous group of clinical conditions with different pathophysiology, clinical course, prognosis, and treatment. Nevertheless, subtype differentiation in clinical practice often remains challenging, and we currently lack reliable diagnostic, prognostic, and therapeutic biomarkers. The advent of new transcriptomics techniques in kidney research poses great potential in the identification of gene expression biomarkers that can be applied in clinical practice. Summary: Transcriptomics techniques have been completely revolutionized in the last 2 decades, with the evolution from low-throughput reverse-transcription polymerase chain reaction and in situ hybridization techniques to microarrays and next-generation sequencing techniques, including RNA-sequencing and single-cell transcriptomics. The integration of human gene expression profiles with functional in vitro and in vivo experiments provides a deeper mechanistic insight into the candidate genes, which enable the development of novel-targeted therapies. The correlation of gene expression profiles with clinical outcomes of large patient cohorts allows for the development of clinically applicable biomarkers that can aid in diagnosis and predict prognosis and therapy response. Finally, the integration of transcriptomics with other \\u201comics\\u201d modalities creates a holistic view on disease pathophysiology. Key Messages: New transcriptomics techniques allow high-throughput gene expression profiling of patients with focal segmental glomerulosclerosis (FSGS). The integration with clinical outcomes and fundamental mechanistic studies enables the discovery of new clinically useful biomarkers that will finally improve the clinical outcome of patients with FSGS.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=869&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"96200968e76b0e26abc100615f130561b56166b3\",\n",
      "        \"title\": \"Deconvolution of FSGS pathophysiology using transcriptomics techniques\",\n",
      "        \"abstract\": \"Background: Focal segmental glomerulosclerosis (FSGS) is a histopathological pattern of renal injury and comprises of a heterogeneous group of clinical conditions with different pathophysiology, clinical course, prognosis and treatment. Nevertheless, subtype differentiation in clinical practice often remains challenging, and we currently lack reliable diagnostic, prognostic and therapeutic biomarkers. The advent of new transcriptomics techniques in kidney research poses great potential in the identification of gene-expression biomarkers that can be applied in clinical practice. Summary: Transcriptomics techniques have been completely revolutionized in the last two decades, with the evolution from low-throughput reverse transcription polymerase chain reaction (RT-PCR) and in situ hybridization (ISH) techniques to microarrays and next-generation sequencing techniques including RNA-sequencing (RNA-seq) and single-cell transcriptomics. The integration of human gene expression profiles with functional in vitro and in vivo experiments provides a deeper mechanistic insight into the candidate genes, which enables the development of novel targeted therapies. The correlation of gene expression profiles with clinical outcomes of large patient cohorts, allows for the development of clinically applicable biomarkers that can aid in diagnosis and predict prognosis and therapy response. Finally, the integration of transcriptomics with other \\u2018omics\\u2019 modalities creates a holistic view on disease pathophysiology. New transcriptomics techniques allow high-throughput gene expression profiling of patients with FSGS. The integration with clinical outcomes and fundamental mechanistic studies, enables the discovery of new clinically useful biomarkers that will finally improve the clinical outcome of patients with FSGS.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=870&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"67f15179615bfd69f961655105a8fd048a46537c\",\n",
      "        \"title\": \"Single-cell multi-omic analysis of thymocyte development reveals drivers of CD4/CD8 lineage commitment\",\n",
      "        \"abstract\": \"The development of CD4 and CD8 T cells in the thymus is critical to adaptive immunity and is widely studied as a model of lineage commitment. Recognition of self-MHCI/II by the T cell antigen receptor (TCR) determines the lineage choice, but how distinct TCR signals drive transcriptional programs of lineage commitment remains largely unknown. We applied CITE-seq to measure RNA and surface proteins in thymocytes from wild-type and lineage-restricted mice to generate a comprehensive timeline of cell state for each lineage. These analyses revealed a sequential process whereby all thymocytes initiate CD4 lineage differentiation during an initial wave of TCR signaling, followed by a second TCR signaling wave that coincides with CD8 lineage specification. CITE-seq and pharmaceutical inhibition experiments implicate a TCR/calcineurin/NFAT/GATA3 axis in driving the CD4 fate. Overall, our data suggest that multiple redundant mechanisms contribute to the accuracy and efficiency of the lineage choice.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=871&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c41c6c944a9c395038c124ea88523597567df8bd\",\n",
      "        \"title\": \"Single-Cell Analysis of the Multicellular Ecosystem in Viral Carcinogenesis by HTLV-1\",\n",
      "        \"abstract\": \"High-dimensional single-cell landscape of immune alterations during HTLV-1 infection and leukemogenesis identifies hallmarks of premalignant and malignant T-cell states and the accompanying shift of systemic immune state toward myeloid and immunosuppressive.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=872&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a7b7492197c45e46865cedebac37e140259a28f6\",\n",
      "        \"title\": \"Patterns of T\\u2010Cell Phenotypes in Rheumatic Diseases From Single\\u2010Cell Studies of Tissue\",\n",
      "        \"abstract\": \"High\\u2010dimensional analyses of tissue samples from patients with rheumatic diseases are providing increasingly detailed descriptions of the immune cell populations that infiltrate tissues in different rheumatic diseases. Here we review key observations emerging from high\\u2010dimensional analyses of T cells within tissues in different rheumatic diseases, highlighting common themes across diseases as well as distinguishing features. Single\\u2010cell RNA sequencing analyses capture several dimensions of T\\u2010cell states, yet surprisingly, these analyses generally have not demonstrated distinct clusters of paradigmatic T\\u2010cell effector subsets, such as T helper (Th) 1, Th2, and Th17 cells. Rather, global transcriptomics robustly identify both proliferating T cells and regulatory T cells and have also helped to reveal new effector subsets in inflamed tissues, including T peripheral helper cells and granzyme K+ T cells. Further characterization of the T\\u2010cell populations that accumulate within target tissues should enable more precise targeting of biologic therapies and accelerate development of more specific biomarkers to track activity of relevant immune pathways in patients with rheumatic diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=873&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4fa0658877eb9031d7425fd2b13cb307c5b262f2\",\n",
      "        \"title\": \"Exploiting Single-Cell Tools in Gene and Cell Therapy\",\n",
      "        \"abstract\": \"Single-cell molecular tools have been developed at an incredible pace over the last five years as sequencing costs continue to drop and numerous molecular assays have been coupled to sequencing readouts. This rapid period of technological development has facilitated the delineation of individual molecular characteristics including the genome, transcriptome, epigenome, and proteome of individual cells, leading to an unprecedented resolution of the molecular networks governing complex biological systems. The immense power of single-cell molecular screens has been particularly highlighted through work in systems where cellular heterogeneity is a key feature, such as stem cell biology, immunology, and tumor cell biology. Single-cell-omics technologies have already contributed to the identification of novel disease biomarkers, cellular subsets, therapeutic targets and diagnostics, many of which would have been undetectable by bulk sequencing approaches. More recently, efforts to integrate single-cell multi-omics with single cell functional output and/or physical location have been challenging but have led to substantial advances. Perhaps most excitingly, there are emerging opportunities to reach beyond the description of static cellular states with recent advances in modulation of cells through CRISPR technology, in particular with the development of base editors which greatly raises the prospect of cell and gene therapies. In this review, we provide a brief overview of emerging single-cell technologies and discuss current developments in integrating single-cell molecular screens and performing single-cell multi-omics for clinical applications. We also discuss how single-cell molecular assays can be usefully combined with functional data to unpick the mechanism of cellular decision-making. Finally, we reflect upon the introduction of spatial transcriptomics and proteomics, its complementary role with single-cell RNA sequencing (scRNA-seq) and potential application in cellular and gene therapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=874&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"529cdf58dbce4683e8e5bc5ff5a97e8da8459f59\",\n",
      "        \"title\": \"Pooled screening of CAR T cells identifies non-native signaling domains for next-generation immunotherapies\",\n",
      "        \"abstract\": \"Chimeric antigen receptors (CARs) repurpose natural signaling components to retarget T cells to refractory cancers, but have shown limited efficacy against solid tumors. Here, we introduce \\u2018CAR Pooling\\u2019, a multiplexed approach to rapidly identify CAR designs with clinical potential. Forty CARs with diverse immune costimulatory domains were assessed in pooled assays for their ability to stimulate critical T cell effector functions during repetitive stimulation that mimics long-term tumor antigen exposure. Several non-native domains from the TNF receptor family exhibited enhanced proliferation (CD40) or cytotoxicity (BAFF-R and TACI) relative to clinical benchmarks, and fell into distinct states of memory, cytotoxicity, and metabolism. BAFF-R CAR T cells were enriched for a highly cytotoxic and NK-cell-like innate phenotype previously associated with positive clinical outcomes. \\u2018CAR Pooling\\u2019 enables efficient exploration of how CAR design affects cell activity and can be applied to optimize receptors across a range of applications and cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=875&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c98c668869da3c86e4613d6fe13b783a96dd96cb\",\n",
      "        \"title\": \"Comparison and evaluation of statistical error models for scRNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=876&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"79b0591b67d1fd24b9ffd442cbf429667f8b394c\",\n",
      "        \"title\": \"Multifactorial chromatin regulatory landscapes at single cell resolution\",\n",
      "        \"abstract\": \"Chromatin profiling at locus resolution uncovers gene regulatory features that define cell types and developmental trajectories, but it remains challenging to map and compare distinct chromatin-associated proteins within the same sample. Here we describe a scalable antibody barcoding approach for profiling multiple chromatin features simultaneously in the same individual cells, Multiple Target Identification by Tagmentation (MulTI-Tag). MulTI-Tag is optimized to retain high sensitivity and specificity of enrichment for multiple chromatin targets in the same assay. We use MulTI-Tag in a combinatorial barcoding approach to resolve distinct cell types and developmental trajectories using multiple chromatin features, and to distinguish unique, coordinated patterns of active and repressive element regulatory usage in the same individual cells that are associated with distinct differentiation outcomes. Multifactorial profiling allows us to detect novel associations between histone marks in single cells and holds promise for comprehensively characterizing cell-specific gene regulatory landscapes in development and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=877&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a5a2aab2c1b7cbac5133ffc7cbd1614fd7e266c\",\n",
      "        \"title\": \"Multiplexing Methods for Simultaneous Large\\u2010Scale Transcriptomic Profiling of Samples at Single\\u2010Cell Resolution\",\n",
      "        \"abstract\": \"Barcoding technology has greatly improved the throughput of cells and genes detected in single\\u2010cell RNA sequencing (scRNA\\u2010seq) studies. Recently, increasing studies have paid more attention to the use of this technology to increase the throughput of samples, as it has greatly reduced the processing time, technical batch effects, and library preparation costs, and lowered the per\\u2010sample cost. In this review, the various DNA\\u2010based barcoding methods for sample multiplexing are focused on, specifically, on the four major barcoding strategies. A detailed comparison of the barcoding methods is also presented, focusing on aspects such as sample/cell throughput and gene detection, and guidelines for choosing the most appropriate barcoding technique according to the personalized requirements are developed. Finally, the critical applications of sample multiplexing and technical challenges in combinatorial labeling, barcoding in vivo, and multimodal tagging at the spatially resolved resolution, as well as, the future prospects of multiplexed scRNA\\u2010seq, for example, prioritizing and predicting the severity of coronavirus disease 2019 (COVID\\u201019) in patients of different gender and age are highlighted.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=878&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f8a551bb1615cf8390d6212e107219dd0636c8f8\",\n",
      "        \"title\": \"Cellular Heterogeneity of Mesenchymal Stem/Stromal Cells in the Bone Marrow\",\n",
      "        \"abstract\": \"Mesenchymal stem/stromal cells (MSCs) are present in various body tissues and help in maintaining homeostasis. The stemness of MSCs has been evaluated in vitro. In addition, analyses of cell surface antigens and gene expression patterns have shown that MSCs comprise a heterogeneous population, and the diverse and complex nature of MSCs makes it difficult to identify the specific roles in diseases. There is a lack of understanding regarding the classification of MSC properties. In this review, we explore the characteristics of heterogeneous MSC populations based on their markers and gene expression profiles. We integrated the contents of previously reported single-cell analysis data to better understand the properties of mesenchymal cell populations. In addition, the cell populations involved in the development of myeloproliferative neoplasms (MPNs) are outlined. Owing to the diversity of terms used to describe MSCs, we used the text mining technology to extract topics from MSC research articles. Recent advances in technology could improve our understanding of the diversity of MSCs and help us evaluate cell populations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=879&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a2be09dbf4e6f455191a75350b26a7e4e0853fef\",\n",
      "        \"title\": \"Shortening of 3\\u2032 UTRs in most cell types composing tumor tissues implicates alternative polyadenylation in protein metabolism\",\n",
      "        \"abstract\": \"During pre-mRNA maturation 3\\u2019 end processing can occur at different polyadenylation sites in the 3\\u2019 untranslated region (3\\u2019 UTR) to give rise to transcript isoforms that differ in the length of their 3\\u2019UTRs. Longer 3\\u2019 UTRs contain additional cis-regulatory elements that impact the fate of the transcript and/or of the resulting protein. Extensive alternative polyadenylation (APA) has been observed in cancers, but the mechanisms and roles remain elusive. In particular, it is unclear whether the APA occurs in the malignant cells or in other cell types that infiltrate the tumor. To resolve this, we developed a computational method, called SCUREL, that quantifies changes in 3\\u2019UTR length between groups of cells, including cells of the same type originating from tumor and control tissue. We used this method to study APA in human lung adenocarcinoma (LUAD). SCUREL relies solely on annotated 3\\u2019UTRs and on control systems, such as T cell activation and spermatogenesis gives qualitatively similar results at much greater sensitivity compared to the previously published scAPA method. In the LUAD samples, we find a general trend towards 3\\u2019UTR shortening not only in cancer cells compared to the cell type of origin, but also when comparing other cell types from the tumor vs. the control tissue environment. However, we also find high variability in the individual targets between patients. The findings help to understand the extent and impact of APA in LUAD, which may support improvements in diagnosis and treatment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=880&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c0e3182aa46751462383dc45ba293f93e0a809db\",\n",
      "        \"title\": \"Optimal balancing of clinical factors in large scale clinical RNA-Seq studies\",\n",
      "        \"abstract\": \"Omics technologies are ubiquitous in biomedical research. However, improper sample selection is an often-overlooked complication with large omics studies, resulting in confounding effects that can disrupt the internal validity of a study and lead to false conclusions. Here, we present a method called BalanceIT, which uses a genetic algorithm to identify an optimal set of samples with balanced clinical factors for large-scale omics experiments. We apply our approach to two large RNA-Seq studies in autism (1) to find a post-hoc balanced sample set among an imbalanced study, and (2) to design an optimal study that allows for efficient batch correction. Our approach leads to near-perfect estimates of differential gene expression, superior performance of pathway-level enrichment analysis, and consistent network dysregulation patterns of autism symptom severity. These results provide empirical support for the importance of balanced experimental design, and BalanceIT will be invaluable for large-scale study design and batch effect correction.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=881&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"002706db676789b5131e1e99048db9ce89dfa741\",\n",
      "        \"title\": \"Bridging the B Cell Gap: Novel Technologies to Study Antigen-Specific Human B Cell Responses\",\n",
      "        \"abstract\": \"The generation of high affinity antibodies is a crucial aspect of immunity induced by vaccination or infection. Investigation into the B cells that produce these antibodies grants key insights into the effectiveness of novel immunogens to induce a lasting protective response against endemic or pandemic pathogens, such as influenza viruses, human immunodeficiency virus, or severe acute respiratory syndrome coronavirus-2. However, humoral immunity has largely been studied at the serological level, limiting our knowledge on the specificity and function of B cells recruited to respond to pathogens. In this review, we cover a number of recent innovations in the field that have increased our ability to connect B cell function to the B cell repertoire and antigen specificity. Moreover, we will highlight recent advances in the development of both ex vivo and in vivo models to study human B cell responses. Together, the technologies highlighted in this review can be used to help design and validate new vaccine designs and platforms.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=882&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fda269075376b4a8be35753fc73f8b8e993cbefa\",\n",
      "        \"title\": \"Single-cell transcriptomic profiling reveals distinct mechanical responses between normal and diseased tendon progenitor cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=883&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aad6eb30bce00bd4aac962c72507fd6e0625027f\",\n",
      "        \"title\": \"Analytical solutions of the chemical master equation with bursty production and isomerization reactions\",\n",
      "        \"abstract\": \"Splicing cascades that alter gene products post-transcriptionally also affect expression dynamics. We study a class of processes and associated distributions that emerge from a bursty promoter model coupled to a path graph of downstream mRNA splicing, and more generally examine the behavior of finite-activity jump drivers coupled to a directed acyclic graph of splicing with one or more roots. These solutions provide full time-dependent joint distributions for an arbitrary number of species, offering qualitative and quantitative insights about how splicing can regulate expression dynamics. Finally, we derive a set of quantitative constraints on the minimum complexity necessary to reproduce gene co-expression patterns using synchronized burst models. We validate these findings by analyzing long-read sequencing data, where we find evidence of expression patterns largely consistent with these constraints.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=884&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0016f55349d813c600b04f9f1b18d72ae8f69d49\",\n",
      "        \"title\": \"Mechanistic models of blood cell fate decisions in the era of single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=885&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4aff60277155c5529c0855505573f32538a46c90\",\n",
      "        \"title\": \"The contribution of single-cell analysis of acute leukemia in the therapeutic strategy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=886&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e6dd692190e8c197c8033352e40fad9163f563ce\",\n",
      "        \"title\": \"Genomic cytometry is maturing but has single\\u2010cell multiomics hit puberty yet?\",\n",
      "        \"abstract\": \"Since the inception of high throughput single-cell RNASeq approaches in 2015 [1, 2], the field of Genomic Cytometry has been maturing at a rapid pace. As part of this maturation, Genomic Cytometry is no longer restricted to RNA analysis but can be used to allow deep interrogation of RNA [3], DNA [4, 5], protein [6, 7], and epigenetic states [8]. Moreover, the ability to combine these modalities onto a single cell is ushering in a new era of single-cell multiomics [9\\u201311]. The recent paper from Corselli et al. [12] combined a targeted scRNAseq approach with oligo AbSeq to simultaneously follow T-cells during in vitro stimulation. This allowed the authors to track a panel of 399 immune-related transcripts and a panel of 38 cell surface proteins over the course of a 14-day stimulation period. Importantly, results from the multiomic comparison were compared with traditional fluorescence flow cytometry. This helps understand how a multiomics approach compares to traditional gold standard methods. The comparison here would suggest that while genomic cytometry is maturing, it is still in its infancy and requires careful attention to see it mature into a robust reliable method with results identical to the current gold standard.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=887&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ad12682e00dd428abc02919551d8346936741151\",\n",
      "        \"title\": \"Prioritization of cell types responsive to biological perturbations in single-cell data with Augur\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=888&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6d4fc31751f57ad5bc7954fae4716c7beb53edb\",\n",
      "        \"title\": \"A Systems Approach to Brain Tumor Treatment\",\n",
      "        \"abstract\": \"Simple Summary Pronounced differences across individuals (interpatient variability) and cell\\u2013cell heterogeneity within a tumor (intratumoral heterogeneity) severely hinder effective brain tumor treatment. To overcome these challenges, a personalized precision medicine approach that considers the uniqueness of an individual patient\\u2019s tumor and its cellular composition is required. A systems biology approach is needed to develop a multiscale understanding of the mechanistic drivers of disease etiology and progression to realize this vision. A systems-level understanding of disease characteristics can facilitate precise patient stratification into clinically meaningful subtypes and inform on potential druggable targets that can enhance treatment. Here, we synthesize and review various methodologies that can be integrated into a framework designed to achieve a personalized precision medicine approach for treating brain tumors. Finally, we provide a practical example in the context of analyzing an individual glioblastoma (GBM) patient at various stages of disease progression. Abstract Brain tumors are among the most lethal tumors. Glioblastoma, the most frequent primary brain tumor in adults, has a median survival time of approximately 15 months after diagnosis or a five-year survival rate of 10%; the recurrence rate is nearly 90%. Unfortunately, this prognosis has not improved for several decades. The lack of progress in the treatment of brain tumors has been attributed to their high rate of primary therapy resistance. Challenges such as pronounced inter-patient variability, intratumoral heterogeneity, and drug delivery across the blood\\u2013brain barrier hinder progress. A comprehensive, multiscale understanding of the disease, from the molecular to the whole tumor level, is needed to address the intratumor heterogeneity resulting from the coexistence of a diversity of neoplastic and non-neoplastic cell types in the tumor tissue. By contrast, inter-patient variability must be addressed by subtyping brain tumors to stratify patients and identify the best-matched drug(s) and therapies for a particular patient or cohort of patients. Accomplishing these diverse tasks will require a new framework, one involving a systems perspective in assessing the immense complexity of brain tumors. This would in turn entail a shift in how clinical medicine interfaces with the rapidly advancing high-throughput (HTP) technologies that have enabled the omics-scale profiling of molecular features of brain tumors from the single-cell to the tissue level. However, several gaps must be closed before such a framework can fulfill the promise of precision and personalized medicine for brain tumors. Ultimately, the goal is to integrate seamlessly multiscale systems analyses of patient tumors and clinical medicine. Accomplishing this goal would facilitate the rational design of therapeutic strategies matched to the characteristics of patients and their tumors. Here, we discuss some of the technologies, methodologies, and computational tools that will facilitate the realization of this vision to practice.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=889&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"002e602b0427d7b2cd124420258d0a257e06bf40\",\n",
      "        \"title\": \"Future Directions of Fibrosis in Medicine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=890&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d1aaa584a8d2ea32d5a3bf3f4420accf7e4e232e\",\n",
      "        \"title\": \"Leveraging Novel Integrated Single-Cell Analyses to Define HIV-1 Latency Reversal\",\n",
      "        \"abstract\": \"While suppressive antiretroviral therapy can effectively limit HIV-1 replication and evolution, it leaves behind a residual pool of integrated viral genomes that persist in a state of reversible nonproductive infection, referred to as the HIV-1 reservoir. HIV-1 infection models were established to investigate HIV-1 latency and its reversal; recent work began to probe the dynamics of HIV-1 latency reversal at single-cell resolution. Signals that establish HIV-1 latency and govern its reactivation are complex and may not be completely resolved at the cellular and regulatory levels by the aggregated measurements of bulk cellular-sequencing methods. High-throughput single-cell technologies that characterize and quantify changes to the epigenome, transcriptome, and proteome continue to rapidly evolve. Combinations of single-cell techniques, in conjunction with novel computational approaches to analyze these data, were developed and provide an opportunity to improve the resolution of the heterogeneity that may exist in HIV-1 reactivation. In this review, we summarize the published single-cell HIV-1 transcriptomic work and explore how cutting-edge advances in single-cell techniques and integrative data-analysis tools may be leveraged to define the mechanisms that control the reversal of HIV-1 latency.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=891&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"02e1a5c41cd0d74d5d704fad93ec87ec858c7bb3\",\n",
      "        \"title\": \"The triumphs and limitations of computational methods for scRNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=892&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0b99ec2c49546f552ba8c1e8eaf718ea8e151e7a\",\n",
      "        \"title\": \"Visual Genomics Analysis Studio as a Tool to Analyze Multiomic Data\",\n",
      "        \"abstract\": \"Type B adverse drug reactions (ADRs) are iatrogenic immune-mediated syndromes with mechanistic etiologies that remain incompletely understood. Some of the most severe ADRs, including delayed drug hypersensitivity reactions, are T-cell mediated, restricted by specific human leukocyte antigen risk alleles and sometimes by public or oligoclonal T-cell receptors (TCRs), central to the immunopathogenesis of tissue-damaging response. However, the specific cellular signatures of effector, regulatory, and accessory immune populations that mediate disease, define reaction phenotype, and determine severity have not been defined. Recent development of single-cell platforms bringing together advances in genomics and immunology provides the tools to simultaneously examine the full transcriptome, TCRs, and surface protein markers of highly heterogeneous immune cell populations at the site of the pathological response at a single-cell level. However, the requirement for advanced bioinformatics expertise and computational hardware and software has often limited the ability of investigators with the understanding of diseases and biological models to exploit these new approaches. Here we describe the features and use of a state-of-the-art, fully integrated application for analysis and visualization of multiomic single-cell data called Visual Genomics Analysis Studio (VGAS). This unique user-friendly, Windows-based graphical user interface is specifically designed to enable investigators to interrogate their own data. While VGAS also includes tools for sequence alignment and identification of associations with host or organism genetic polymorphisms, in this review we focus on its application for analysis of single-cell TCR\\u2013RNA\\u2013Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE)-seq, enabling holistic cellular characterization by unbiased transcriptome and select surface proteome. Critically, VGAS does not require user-directed coding or access to high-performance computers, instead incorporating performance-optimized hidden code to provide application-based fast and intuitive tools for data analyses and production of high-resolution publication-ready graphics on standard specification laptops. Specifically, it allows analyses of comprehensive single-cell TCR sequencing (scTCR-seq) data, detailing (i) functional pairings of \\u03b1\\u2013\\u03b2 heterodimer TCRs, (ii) one-click histograms to display entropy and gene rearrangements, and (iii) Circos and Sankey plots to visualize clonality and dominance. For unbiased single-cell RNA sequencing (scRNA-seq) analyses, users extract cell transcriptome signatures according to global structure via principal component analysis, t-distributed stochastic neighborhood embedding, or uniform manifold approximation and projection plots, with overlay of scTCR-seq enabling identification and selection of the immunodominant TCR-expressing populations. Further integration with similar sequence-based detection of surface protein markers using oligo-labeled antibodies (CITE-seq) provides comparative understanding of surface protein expression, with differential gene or protein analyses visualized using volcano plot or heatmap functions. These data can be compared to reference cell atlases or suitable controls to reveal discrete disease-specific subsets, from epithelial to tissue-resident memory T-cells, and activation status, from senescence through exhaustion, with more finite transcript expression displayed as violin and box plots. Importantly, guided tutorial videos are available, as are regular application updates based on the latest advances in bioinformatics and user feedback.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=893&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"067d2dc62084890ef36224141ccd82a28782a4a7\",\n",
      "        \"title\": \"Current Methodological Challenges of Single-Cell and Single-Nucleus RNA-Sequencing in Glomerular Diseases\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) and single-nucleus RNA-seq (snRNA-seq) allow transcriptomic profiling of thousands of cells from a renal biopsy specimen at a single-cell resolution. Both methods are promising tools to unravel the underlying pathophysiology of glomerular diseases. This review provides an overview of the technical challenges that should be addressed when designing single-cell transcriptomics experiments that focus on glomerulopathies. The isolation of glomerular cells from core needle biopsy specimens for single-cell transcriptomics remains difficult and depends upon five major factors. First, core needle biopsies generate little tissue material, and several samples are required to identify glomerular cells. Second, both fresh and frozen tissue samples may yield glomerular cells, although every experimental pipeline has different (dis)advantages. Third, enrichment for glomerular cells in human tissue before single-cell analysis is challenging because no effective standardized pipelines are available. Fourth, the current warm cell-dissociation protocols may damage glomerular cells and induce transcriptional artifacts, which can be minimized by using cold dissociation techniques at the cost of less efficient cell dissociation. Finally, snRNA-seq methods may be superior to scRNA-seq in isolating glomerular cells; however, the efficacy of snRNA-seq on core needle biopsy specimens remains to be proven. The field of single-cell omics is rapidly evolving, and the integration of these techniques in multiomics assays will undoubtedly create new insights in the complex pathophysiology of glomerular diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=894&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"09e9ad5f945dcb7de208988a1c7a1a2ceb784ad2\",\n",
      "        \"title\": \"Single-Cell Mapping of GLP-1 and GIP Receptor Expression in the Dorsal Vagal Complex\",\n",
      "        \"abstract\": \"The dorsal vagal complex (DVC) in the hindbrain, composed of the area postrema, nucleus of the solitary tract, and dorsal motor nucleus of the vagus, plays a critical role in modulating satiety. The incretins glucagon-like peptide 1 (GLP-1) and glucose-dependent insulinotropic polypeptide (GIP) act directly in the brain to modulate feeding, and receptors for both are expressed in the DVC. Given the impressive clinical responses to pharmacologic manipulation of incretin signaling, understanding the central mechanisms by which incretins alter metabolism and energy balance is of critical importance. Here, we review recent single-cell approaches used to detect molecular signatures of GLP-1 and GIP receptor\\u2013expressing cells in the DVC. In addition, we discuss how current advancements in single-cell transcriptomics, epigenetics, spatial transcriptomics, and circuit mapping techniques have the potential to further characterize incretin receptor circuits in the hindbrain.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=895&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fc7ec7b2bc1578a7996ac4bf6d3f0f649273cd08\",\n",
      "        \"title\": \"A single-cell guide to retinal development: Cell fate decisions of multipotent retinal progenitors in scRNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=896&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59e7060735f720ccc35b302463dda10baaa79ff7\",\n",
      "        \"title\": \"Impaired humoral immunity to SARS-CoV-2 BNT162b2 vaccine in kidney transplant recipients and dialysis patients.\",\n",
      "        \"abstract\": \"Patients with kidney failure are at increased risk for SARS-CoV-2 infection making effective vaccinations a critical need. It is not known how well mRNA vaccines induce B and plasma cell responses in dialysis patients (DP) or kidney transplant recipients (KTR) compared to healthy controls (HC). We studied humoral and B cell responses of 35 HC, 44 DP and 40 KTR. Markedly impaired anti-BNT162b2 responses were identified among KTR and DP compared to HC. In DP, the response was delayed (3-4 weeks after boost) and reduced with anti-S1 IgG and IgA positivity in 70.5% and 68.2%, respectively. In contrast, KTR did not develop IgG responses except one patient who had a prior unrecognized infection and developed anti-S1 IgG. The majority of antigen-specific B cells (RBD+) were identified in the plasmablast or post-switch memory B cell compartments in HC, whereas RBD+ B cells were enriched among pre-switch and na\\u00efve B cells from DP and KTR. The frequency and absolute number of antigen-specific circulating plasmablasts in the cohort correlated with the Ig response, a characteristic not reported for other vaccinations. In conclusion, these data indicated that immunosuppression resulted in impaired protective immunity after mRNA vaccination, including Ig induction with corresponding generation of plasmablasts and memory B cells. Thus, there is an urgent need to improve vaccination protocols in patients after kidney transplantation or on chronic dialysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=897&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5237d77e04a05e3821c640abeb7d0e73e4ddab44\",\n",
      "        \"title\": \"Analyzing signaling activity and function in hematopoietic cells\",\n",
      "        \"abstract\": \"Cells constantly sense their environment to adapt their behavior to changing needs. Kull and Schroeder discuss the requirements, potential, and limitations of current technologies for quantification and manipulation of cellular signaling in the context of inflammatory signaling in hematopoietic cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=898&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"66c804ba7a6add9f1c8db882f1b64e56f920e1ae\",\n",
      "        \"title\": \"Multiplex protein profiling method for extracellular vesicle protein detection\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=899&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e132aa484ac3eb4f8f99839f97b02138f26d455d\",\n",
      "        \"title\": \"Single-cell biology to decode the immune cellular composition of kidney inflammation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=900&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"749b69b8d61e83c90781f53e143dfdd3083efc81\",\n",
      "        \"title\": \"Multi-Omics Approaches in Immunological Research\",\n",
      "        \"abstract\": \"The immune system plays a vital role in health and disease, and is regulated through a complex interactive network of many different immune cells and mediators. To understand the complexity of the immune system, we propose to apply a multi-omics approach in immunological research. This review provides a complete overview of available methodological approaches for the different omics data layers relevant for immunological research, including genetics, epigenetics, transcriptomics, proteomics, metabolomics, and cellomics. Thereafter, we describe the various methods for data analysis as well as how to integrate different layers of omics data. Finally, we discuss the possible applications of multi-omics studies and opportunities they provide for understanding the complex regulatory networks as well as immune variation in various immune-related diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=901&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a2943ca71f7ea6ca3250bdfbc26fe53a9de87f9\",\n",
      "        \"title\": \"Single-cell multi-omics sequencing: application trends, COVID-19, data analysis issues and prospects\",\n",
      "        \"abstract\": \"Abstract Single-cell sequencing is a biotechnology to sequence one layer of genomic information for individual cells in a tissue sample. For example, single-cell DNA sequencing is to sequence the DNA from every single cell. Increasing in complexity, single-cell multi-omics sequencing, or single-cell multimodal omics sequencing, is to profile in parallel multiple layers of omics information from a single cell. In practice, single-cell multi-omics sequencing actually detects multiple traits such as DNA, RNA, methylation information and/or protein profiles from the same cell for many individuals in a tissue sample. Multi-omics sequencing has been widely applied to systematically unravel interplay mechanisms of key components and pathways in cell. This survey overviews recent developments in single-cell multi-omics sequencing, and their applications to understand complex diseases in particular the COVID-19 pandemic. We also summarize machine learning and bioinformatics techniques used in the analysis of the intercorrelated multilayer heterogeneous data. We observed that variational inference and graph-based learning are popular approaches, and Seurat V3 is a commonly used tool to transfer the missing variables and labels. We also discussed two intensively studied issues relating to data consistency and diversity and commented on currently cared issues surrounding the error correction of data pairs and data imputation methods. The survey is concluded with some open questions and opportunities for this extraordinary field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=902&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"01cec81e59074c6133fc8778c4b9c43b3a295da8\",\n",
      "        \"title\": \"Applications of single-cell sequencing in cancer research: progress and perspectives\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=903&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"65e57e8cc95a024f76afe5d9461da58153eada00\",\n",
      "        \"title\": \"VISTA is an activating receptor in human monocytes\",\n",
      "        \"abstract\": \"VISTA biology has yet to be fully resolved despite much enthusiasm to define it as an immune checkpoint protein. Here, we show that VISTA antibodies can singly activate human monocytes and provide evidence that Syndecan-2 can regulate VISTA interactions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=904&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8ed33d2ca98d301fd7cd57161214dbf3b8158cad\",\n",
      "        \"title\": \"Applications of Single-Cell Omics in Tumor Immunology\",\n",
      "        \"abstract\": \"The tumor microenvironment (TME) is an ecosystem that contains various cell types, including cancer cells, immune cells, stromal cells, and many others. In the TME, cancer cells aggressively proliferate, evolve, transmigrate to the circulation system and other organs, and frequently communicate with adjacent immune cells to suppress local tumor immunity. It is essential to delineate this ecosystem\\u2019s complex cellular compositions and their dynamic intercellular interactions to understand cancer biology and tumor immunology and to benefit tumor immunotherapy. But technically, this is extremely challenging due to the high complexities of the TME. The rapid developments of single-cell techniques provide us powerful means to systemically profile the multiple omics status of the TME at a single-cell resolution, shedding light on the pathogenic mechanisms of cancers and dysfunctions of tumor immunity in an unprecedently resolution. Furthermore, more advanced techniques have been developed to simultaneously characterize multi-omics and even spatial information at the single-cell level, helping us reveal the phenotypes and functionalities of disease-specific cell populations more comprehensively. Meanwhile, the connections between single-cell data and clinical characteristics are also intensively interrogated to achieve better clinical diagnosis and prognosis. In this review, we summarize recent progress in single-cell techniques, discuss their technical advantages, limitations, and applications, particularly in tumor biology and immunology, aiming to promote the research of cancer pathogenesis, clinically relevant cancer diagnosis, prognosis, and immunotherapy design with the help of single-cell techniques.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=905&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75f2338c413782e6fbdc1923ad17d98f6a559542\",\n",
      "        \"title\": \"Recent Advances in Single-Cell Profiling and Multispecific Therapeutics: Paving the Way for a New Era of Precision Medicine Targeting Cardiac Fibroblasts\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=906&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d36c9c82dd51df085b4917bfb62adadfef0ca92\",\n",
      "        \"title\": \"Scalable, multimodal profiling of chromatin accessibility, gene expression and protein levels in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=907&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abeb144b11df3504025263b619dd863785bc7a68\",\n",
      "        \"title\": \"Chronic Opioid Treatment Arrests Neurodevelopment and Alters Synaptic Activity in Human Midbrain Organoids\",\n",
      "        \"abstract\": \"The impact of long-term opioid exposure on the embryonic brain is crucial to healthcare due to the surging number of pregnant mothers with an opioid dependency. Current studies on the neuronal effects are limited due to human brain inaccessibility and cross-species differences among animal models. Here, we report a model to assess cell-type specific responses to acute and chronic fentanyl treatment, as well as fentanyl withdrawal, using human induced pluripotent stem cell (hiPSC)-derived midbrain organoids. Single cell mRNA sequencing (25,510 single cells in total) results suggest that chronic fentanyl treatment arrests neuronal subtype specification during early midbrain development and alters the pathways associated with synaptic activities and neuron projection. Acute fentanyl treatment, however, increases dopamine release but does not induce significant changes in gene expressions of cell lineage development. To date, our study is the first unbiased examination of midbrain transcriptomics with synthetic opioid treatment at the single cell level.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=908&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2699d91a02a04f750ff7f55da7c24da0ff95652c\",\n",
      "        \"title\": \"MUON: multimodal omics analysis framework\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=909&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1bb46a652498614c2b33f6149bed9204b2044f32\",\n",
      "        \"title\": \"PIKE-R2P: Protein\\u2013protein interaction network-based knowledge embedding with graph neural network for single-cell RNA to protein prediction\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=910&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"146c1e0f2a3b187c7eeb5f7640efc52eefdfa66e\",\n",
      "        \"title\": \"Undisclosed, unmet and neglected challenges in multi-omics studies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=911&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b0bcdc51a71669912b55b431d2cd23ad5112f38\",\n",
      "        \"title\": \"Diversity and intratumoral heterogeneity in human gallbladder cancer progression revealed by single\\u2010cell RNA sequencing\",\n",
      "        \"abstract\": \"Abstract Background Gallbladder cancer (GC) is a malignant disease characterized with highly cellular heterogeneity and poor prognosis. Determining the intratumoral heterogeneity and microenvironment (TME) can provide novel therapeutic strategies for GC. Methods We performed the single\\u2010cell RNA sequencing on the primary and lymph node metastatic gallbladder tumors and the adjacent normal tissues of five patients. The transcriptomic atlas and ligand\\u2013receptor\\u2010based intercellular communication networks of the single cells were characterized. Results The transcriptomic landscape of 24,887 single cells was obtained and characterized as 10 cellular clusters, including epithelial, neuroendocrine tumor cells, T&NK cells, B cells, RGS5+ fibroblasts, POSTN+ fibroblasts, PDGFRA+ fibroblasts, endothelial, myeloid cells, and mast cells. Different types of GC harbored distinct epithelial tumor subpopulations, and squamous cell carcinoma could be differentiated from adenocarcinoma cells. Abundant immune cells infiltrated into adenocarcinoma and squamous cell carcinoma, rather than neuroendocrine neoplasms, which showed significant enrichment of stromal cells. CD4+/FOXP3+ T\\u2010reg and CD4+/CXCL13+ T helper cells with higher exhausting biomarkers, as well as a dynamic lineage transition of tumor\\u2010associated macrophages from CCL20hi/CD163lo, CCL20lo/CD163hi to APOE+, were identified in GC tissues, suggesting the immunosuppressive and tumor\\u2010promoting status of immune cells in TME. Two distinct endothelial cells (KDR+ and ACKR1+), which were involved in angiogenesis and lymphangiogenesis, showed remarkable ligand\\u2013receptor interactions with primary GC cells and macrophages in gallbladder tumors. Conclusions This study reveals a widespread reprogramming across multiple cell populations in GC progression, dissects the cellular heterogeneity and interactions in gallbladder TME, and provides potential therapeutic targets for GC.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=912&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcb43e7861d9c02618f008c3eabe8d7ea0112dc6\",\n",
      "        \"title\": \"New applications of advanced instrumental techniques for the characterization of food allergenic proteins\",\n",
      "        \"abstract\": \"Abstract Current approaches based on electrophoretic, chromatographic or immunochemical principles have allowed characterizing multiple allergens, mapping their epitopes, studying their mechanisms of action, developing detection and diagnostic methods and therapeutic strategies for the food and pharmaceutical industry. However, some of the common structural features related to the allergenic potential of food proteins remain unknown, or the pathological mechanism of food allergy is not yet fully understood. In addition, it is also necessary to evaluate new allergens from novel protein sources that may pose a new risk for consumers. Technological development has allowed the expansion of advanced technologies for which their whole potential has not been entirely exploited and could provide novel contributions to still unexplored molecular traits underlying both the structure of food allergens and the mechanisms through which they sensitize or elicit adverse responses in human subjects, as well as improving analytical techniques for their detection. This review presents cutting-edge instrumental techniques recently applied when studying structural and functional aspects of proteins, mechanism of action and interaction between biomolecules. We also exemplify their role in the food allergy research and discuss their new possible applications in several areas of the food allergy field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=913&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4868005f6a7a96d2a1f65cfe642acbebd26da4a6\",\n",
      "        \"title\": \"Spatial proteomics for understanding the tissue microenvironment.\",\n",
      "        \"abstract\": \"The human body comprises rich populations of cells, which are arranged into tissues and organs with diverse functionalities. These cells exhibit a broad spectrum of phenotypes and are often organized as a heterogeneous but sophisticatedly regulated ecosystem - tissue microenvironment, inside which every cell interacts with and is reciprocally influenced by its surroundings through its life span. Therefore, it is critical to comprehensively explore the cellular machinery and biological processes in the tissue microenvironment, which is best exemplified by the tumor microenvironment (TME). The past decade has seen increasing advances in the field of spatial proteomics, the main purpose of which is to characterize the abundance and spatial distribution of proteins and their post-translational modifications in the microenvironment of diseased tissues. Herein, we outline the achievements and remaining challenges of mass spectrometry-based tissue spatial proteomics. Exciting technology developments along with important biomedical applications of spatial proteomics are highlighted. In detail, we focus on high-quality resources built by scalpel macrodissection-based region-resolved proteomics, method development of sensitive sample preparation for laser microdissection-based spatial proteomics, and antibody recognition-based multiplexed tissue imaging. In the end, critical issues and potential future directions for spatial proteomics are also discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=914&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"91ddc5d7e8a01b8706791d40176303cc93c4a247\",\n",
      "        \"title\": \"The role of single-cell sequencing in studying tumour evolution\",\n",
      "        \"abstract\": \"Tumour evolution is a complex interplay between the acquisition of somatic (epi)genomic changes in tumour cells and the phenotypic consequences they cause, all in the context of a changing microenvironment. Single-cell sequencing offers a window into this dynamic process at the ultimate resolution of individual cells. In this review, we discuss the transformative insight offered by single-cell sequencing technologies for understanding tumour evolution.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=915&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e2e8982fa20ccdba735d7c73f7a06c518b77fe4e\",\n",
      "        \"title\": \"Immunotherapy in the Treatment of Urothelial Bladder Cancer: Insights From Single-Cell Analysis\",\n",
      "        \"abstract\": \"Urothelial bladder cancer (UBC) is a global challenge of public health with limited therapeutic options. Although the emergence of cancer immunotherapy, most notably immune checkpoint inhibitors, represents a major breakthrough in the past decade, many patients still suffer from unsatisfactory clinical outcome. A thorough understanding of the fundamental cellular and molecular mechanisms responsible for antitumor immunity may lead to optimized treatment guidelines and new immunotherapeutic strategies. With technological developments and protocol refinements, single-cell approaches have become powerful tools that provide unprecedented insights into the kaleidoscopic tumor microenvironment and intricate cell-cell communications. In this review, we summarize recent applications of single-cell analysis in characterizing the UBC multicellular ecosystem, and discuss how to leverage the high-resolution information for more effective immune-based therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=916&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8ed554c66ad3feacee5b4ce88cc2e2a13c11c8f7\",\n",
      "        \"title\": \"Single\\u2010cell advances in stromal\\u2010leukocyte interactions in cancer\",\n",
      "        \"abstract\": \"Mesenchymal stromal cells in solid tumors have emerged as important mediators of immune function and response to immunotherapies. As such, comprehensive insights into their biology may reveal new predictors of drug response and new drug targets. While our understanding of mesenchymal biology in cancer is nascent, it is rapidly evolving, driven by advances in single\\u2010cell technologies. These studies reveal distinct subclasses of cancer\\u2010associated fibroblasts (CAFs) with unique properties for immune regulation and control of leukocyte activity. While these studies have revealed several similarities across distinct types of cancer, they still face key challenges in nomenclature. Single\\u2010cell analysis of tumors has also revealed an abundance of perivascular cells with unique biology and associations with immune infiltration. They are often misclassified, likely confounding previous bulk studies, revealing a distinct lineage of cells that remain to be fully characterized. These studies have also shed light on the discrete cell types or transient cell states that shape mesenchymal heterogeneity in tumors, offering insights into new therapeutic strategies to modulate stromal cell differentiation. In this review, we will address how recent advances in single\\u2010cell technologies have shaped our understanding of stromal heterogeneity and their coordination of immune responses in cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=917&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b9d348797a444f78f4cc96bbb717da308099bcaa\",\n",
      "        \"title\": \"A combined approach for single-cell mRNA and intracellular protein expression analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=918&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"162b7c1a9177b2b47d72d3a8dfa8f48ead633600\",\n",
      "        \"title\": \"High-Throughput Detection of Autoantigen-Specific B Cells Among Distinct Functional Subsets in Autoimmune Donors\",\n",
      "        \"abstract\": \"Antigen-specific B cells (ASBCs) can drive autoimmune disease by presenting autoantigen to cognate T cells to drive their activation, proliferation, and effector cell differentiation and/or by differentiating into autoantibody-secreting cells. Autoantibodies are frequently used to predict risk and diagnose several autoimmune diseases. ASBCs can drive type 1 diabetes even when immune tolerance mechanisms block their differentiation into antibody-secreting cells. Furthermore, anti-histidyl tRNA synthetase syndrome patients have expanded IgM+ Jo-1-binding B cells, which clinically diagnostic IgG Jo-1 autoantibodies may not fully reflect. Given the potential disconnect between the pathologic function of ASBCs and autoantibody secretion, direct study of ASBCs is a necessary step towards developing better therapies for autoimmune diseases, which often have no available cure. We therefore developed a high-throughput screening pipeline to 1) phenotypically identify specific B cell subsets, 2) expand them in vitro, 3) drive them to secrete BCRs as antibody, and 4) identify wells enriched for ASBCs through ELISA detection of antibody. We tested the capacity of several B cell subset(s) to differentiate into antibody-secreting cells following this robust stimulation. IgM+ and/or IgD+, CD27- memory, memory, switched memory, and BND B cells secreted B cell receptor (BCR) as antibody following in vitro stimulation, whereas few plasmablasts responded. Bimodal responses were observed across autoimmune donors for IgM+ CD21lo and IgM- CD21lo B cells, consistent with documented heterogeneity within the CD21lo subset. Using this approach, we detected insulin-binding B cell bias towards CD27- memory and CD27+ memory subsets in pre-symptomatic type 1 diabetes donors. We took advantage of routine detection of Jo-1-binding B cells in Jo-1+ anti-histidyl tRNA synthetase syndrome patients to show that Jo-1-binding B cells and total B cells expanded 20-30-fold using this culture system. Overall, these studies highlight technology that is amenable to small numbers of cryopreserved peripheral blood mononuclear cells that enables interrogation of phenotypic and repertoire attributes of ASBCs derived from autoimmune patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=919&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6d82bbd9b23f76cdc0e73f17a0cbcd6865cc53b7\",\n",
      "        \"title\": \"Immature acute leukaemias: lessons from the haematopoietic roadmap\",\n",
      "        \"abstract\": \"It is essential to relate the biology of acute leukaemia to normal blood cell development. In this review, we discuss how modern models of haematopoiesis might inform approaches to diagnosis and management of immature leukaemias, with a specific focus on T\\u2010lymphoid and myeloid cases. In particular, we consider whether next\\u2010generation analytical tools could provide new perspectives that could improve our understanding of immature blood cancer biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=920&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0cd5f710009179143a2c9ffd6b66125cde669301\",\n",
      "        \"title\": \"Tutorial: guidelines for annotating single-cell transcriptomic maps using automated and manual methods\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=921&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"332643d0167f4f708a0fd929472b5bdd2147da11\",\n",
      "        \"title\": \"Perspectives in systems nephrology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=922&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c3cc2a7bad100a056bdd8077722245ecde533e72\",\n",
      "        \"title\": \"Mitigating myopia in tuberculosis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=923&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"68e4811ee442505b5c2a2f8a6718f76853338d9a\",\n",
      "        \"title\": \"Comparison of Resources and Methods to infer Cell-Cell Communication from Single-cell RNA Data\",\n",
      "        \"abstract\": \"The growing availability of single-cell data has sparked an increased interest in the inference of cell-cell communication from this data. Many tools have been developed for this purpose. Each of them consists of a resource of intercellular interactions prior knowledge and a method to predict potential cell-cell communication events. Yet the impact of the choice of resource and method on the resulting predictions is largely unknown. To shed light on this, we created a framework, available at https://github.com/saezlab/ligrec_decoupler, to facilitate a comparative assessment of methods for inferring cell-cell communication from single cell transcriptomics data and then compared 15 resources and 6 methods. We found few unique interactions and a varying degree of overlap among the resources, and observed uneven coverage in terms of pathways and biological categories. We analysed a colorectal cancer single cell RNA-Seq dataset using all possible combinations of methods and resources. We found major differences among the highest ranked intercellular interactions inferred by each method even when using the same resources. The varying predictions lead to fundamentally different biological interpretations, highlighting the need to benchmark resources and methods. Findings Built a framework to systematically combine 15 resources and 6 methods to estimate cell-cell communication from single-cell RNA data Cell-cell communication resources are often built from the same original databases and very few interactions are unique to a single resource. Yet overlap varies among resources and certain biological terms are unevenly represented Different methods and resources provided notably different results The observed disagreement among the methods could have a considerable impact on the interpretation of results\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=924&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c535e6b9357d4b456079c0047a7bd1f631f52251\",\n",
      "        \"title\": \"SC3s - efficient scaling of single cell consensus clustering to millions of cells\",\n",
      "        \"abstract\": \"Technological advances have paved the way for single cell RNAseq (scRNAseq) datasets containing several million cells 1. Such large datasets require highly efficient algorithms to enable analyses at reasonable times and hardware requirements 2. A crucial step in single cell workflows is unsupervised clustering, which aims to delineate putative cell types or cell states based on transcriptional similarity 3. Here, we present a highly efficient k-means based approach, and we demonstrate that it scales linearly with the number of cells with regards to time and memory.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=925&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd1a5198501f50c080943ceecaf983cbab5938b4\",\n",
      "        \"title\": \"Systematic, Protein Activity-based Characterization of Single Cell State\",\n",
      "        \"abstract\": \"While single-cell RNA sequencing provides a remarkable window on pathophysiologic tissue biology and heterogeneity, its high gene-dropout rate and low signal-to-noise ratio challenge quantitative analyses and mechanistic understanding. To address this issue, we developed PISCES, a platform for the network-based, single-cell analysis of mammalian tissue. PISCES accurately estimates the mechanistic contribution of regulatory and signaling proteins to cell state implementation and maintenance, based on the expression of their lineage-specific transcriptional targets, thus supporting discovery and visualization of Master Regulators of cell state and cell state transitions. Experimental validation assays, including by assessing concordance with antibody and CITE-Seq-based measurements, show significant improvement in the ability to identify rare subpopulations and to elucidate key lineage markers, compared to gene expression analysis. Systematic analysis of single cell profiles in the Human Protein Atlas (HPA) produced a comprehensive resource for human tissue studies, supporting fine-grain stratification of distinct cell states, molecular determinants, and surface markers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=926&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bbea6abc15bbf12457097ecf12d1ba91611cf8c6\",\n",
      "        \"title\": \"AutoSpill is a principled framework that simplifies the analysis of multichromatic flow cytometry data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=927&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cf3d46bab1213541d19025528e5e179f0fd374e5\",\n",
      "        \"title\": \"FBA: feature barcoding analysis for single cell RNA-Seq\",\n",
      "        \"abstract\": \"MOTIVATION\\nSingle cell RNA-Seq (scRNA-Seq) has broadened our understanding of cellular heterogeneity and provided valuable insights into cellular functions. Recent experimental strategies extend scRNA-Seq readouts to include additional features, including cell surface proteins and genomic perturbations. These \\\"feature barcoding\\\" strategies rely on converting molecular and cellular features to unique sequence barcodes, which are then detected with the transcriptome.\\n\\n\\nRESULTS\\nHere, we introduce FBA, a flexible and streamlined package to perform quality control, quantification, demultiplexing, multiplet detection, clustering, and visualization of feature barcoding assays.\\n\\n\\nAVAILABILITY\\nFBA is available on PyPi at https://pypi.org/project/fba and on GitHub at https://github.com/jlduan/fba.\\n\\n\\nSUPPLEMENTARY INFORMATION\\nSupplementary data are available at Bioinformatics online.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=928&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6659a2ab9efce1f2105585b974a501b5d49cb550\",\n",
      "        \"title\": \"SeqStain is an efficient method for multiplexed, spatialomic profiling of human and murine tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=929&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0efb888639f7a071022ce06f50752857d586ff37\",\n",
      "        \"title\": \"Single Cell Technologies: Beyond Microfluidics.\",\n",
      "        \"abstract\": \"Single-cell RNA-sequencing (scRNA-seq) has been widely adopted in recent years due to standardized protocols and automation, reliability, and standardized bioinformatic pipelines. The most widely adopted platform is the 10\\u00d7 Genomics solution. Although powerful, this system is limited by its high cost, moderate throughput, and the inability to customize due to fixed kit components. This study will cover new approaches that do not rely on microfluidics and thus have low entry costs, are highly customizable, and are within the reach of any laboratory possessing molecular biology expertise.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=930&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"691206502218221f1d1e82fdc276cba5cdf295d6\",\n",
      "        \"title\": \"A method for rapid flow-cytometric isolation of endothelial nuclei and RNA from archived frozen brain tissue\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=931&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6df43ed50e67083dfad231400cbceafa88e017c6\",\n",
      "        \"title\": \"Monocyte-derived transcriptome signature indicates antibody-dependent cellular phagocytosis as the primary mechanism of vaccine-induced protection against HIV-1\",\n",
      "        \"abstract\": \"A gene signature previously correlated with mosaic adenovirus 26 vaccine protection in simian immunodeficiency virus (SIV) and SHIV challenge models in non-human primates (NHP). In this report we investigated presence of this signature as a correlate of reduced risk in human clinical trials and potential mechanism for protection. The absence of this gene signature in the DNA/rAd5 human vaccine trial which did not show efficacy, strengthens our hypothesis that this signature is only enriched in studies that demonstrated protection. This gene signature was enriched in the partially effective RV144 human trial that administered the ALVAC/protein vaccine, and we find that the signature associates with both decreased risk of HIV-1 acquisition and increased vaccine efficacy. Total RNA-seq in a clinical trial that used the same vaccine regimen as the RV144 HIV vaccine implicated antibody-dependent cellular phagocytosis (ADCP) as a potential mechanism of vaccine protection. CITE-seq profiling of 53 surface markers and transcriptomes of 53,777 single cells from the same trial, showed that genes in this signature were primarily expressed in cells belonging to the myeloid lineage including monocytes, which are major effector cells for ADCP. The consistent association of this transcriptome signature with vaccine efficacy represents a tool both to identify potential mechanisms, as with ADCP here, and to screen novel approaches to accelerate development of new vaccine candidates.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=932&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e0527571259bea49ee56734ad4fe53a01c195522\",\n",
      "        \"title\": \"Isolation of mouse brain-infiltrating leukocytes for single cell profiling of epitopes and transcriptomes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=933&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f81398b3095e9501ff83fd7e919580ec7f0b05ee\",\n",
      "        \"title\": \"Single-cell protein activity analysis identifies recurrence-associated renal tumor macrophages\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=934&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"44aae3d94d64ed72709ffd2039acd1728e2c33c3\",\n",
      "        \"title\": \"Defining the Emerging Blood System During Development at Single-Cell Resolution\",\n",
      "        \"abstract\": \"Developmental hematopoiesis differs from adult and is far less described. In the developing embryo, waves of lineage-restricted blood precede the ultimate emergence of definitive hematopoietic stem cells (dHSCs) capable of maintaining hematopoiesis throughout life. During the last two decades, the advent of single-cell genomics has provided tools to circumvent previously impeding characteristics of embryonic hematopoiesis, such as cell heterogeneity and rare cell states, allowing for definition of lineage trajectories, cellular hierarchies, and cell-type specification. The field has rapidly advanced from microfluidic platforms and targeted gene expression analysis, to high throughput unbiased single-cell transcriptomic profiling, single-cell chromatin analysis, and cell tracing\\u2014offering a plethora of tools to resolve important questions within hematopoietic development. Here, we describe how these technologies have been implemented to address a wide range of aspects of embryonic hematopoiesis ranging from the gene regulatory network of dHSC formation via endothelial to hematopoietic transition (EHT) and how EHT can be recapitulated in vitro, to hematopoietic trajectories and cell fate decisions. Together, these studies have important relevance for regenerative medicine and for our understanding of genetic blood disorders and childhood leukemias.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=935&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fb084c8dd68d8caef62b43edd722b3bc41b59435\",\n",
      "        \"title\": \"What are the applications of single-cell RNA sequencing in cancer research: a systematic review\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=936&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3915a28026cccb13f5d1a130b009e87aa080c4fb\",\n",
      "        \"title\": \"Multimodal detection of protein isoforms and nucleic acids from low starting cell numbers.\",\n",
      "        \"abstract\": \"Protein isoforms play a key role in disease progression and arise from mechanisms involving multiple molecular subtypes, including DNA, mRNA and protein. Recently introduced multimodal assays successfully link genomes and transcriptomes to protein expression landscapes. However, the specificity of the protein measurement relies on antibodies alone, leading to major challenges when measuring different isoforms of the same protein. Here we utilize microfluidic design to perform same-cell profiling of DNA, mRNA and protein isoforms (triBlot) on low starting cell numbers (1-100 s of cells). After fractionation lysis, cytoplasmic proteins are resolved by molecular mass during polyacrylamide gel electrophoresis (PAGE), adding a degree of specificity to the protein measurement, while nuclei are excised from the device in sections termed \\\"gel pallets\\\" for subsequent off-chip nucleic acid analysis. By assaying TurboGFP-transduced glioblastoma cells, we observe a strong correlation between protein expression prior to lysis and immunoprobed protein. We measure both mRNA and DNA from retrieved nuclei, and find that mRNA levels correlate with protein abundance in TurboGFP-expressing cells. Furthermore, we detect the presence of TurboGFP isoforms differing by an estimated <1 kDa in molecular mass, demonstrating the ability to discern different proteoforms with the same antibody probe. By directly relating nucleic acid modifications to protein isoform expression in 1-100 s of cells, the triBlot assay holds potential as a screening tool for novel biomarkers in diseases driven by protein isoform expression.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=937&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3beaf385923e9961198697e7e38b64aaa987b647\",\n",
      "        \"title\": \"The Transcriptome of SH-SY5Y at Single-Cell Resolution: A CITE-Seq Data Analysis Workflow\",\n",
      "        \"abstract\": \"Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) is a recently established multimodal single cell analysis technique combining the immunophenotyping capabilities of antibody labeling and cell sorting with the resolution of single-cell RNA sequencing (scRNA-seq). By simply adding a 12-bp nucleotide barcode to antibodies (cell hashing), CITE-seq can be used to sequence antibody-bound tags alongside the cellular mRNA, thus reducing costs of scRNA-seq by performing it at the same time on multiple barcoded samples in a single run. Here, we illustrate an ideal CITE-seq data analysis workflow by characterizing the transcriptome of SH-SY5Y neuroblastoma cell line, a widely used model to study neuronal function and differentiation. We obtained transcriptomes from a total of 2879 single cells, measuring an average of 1600 genes/cell. Along with standard scRNA-seq data handling procedures, such as quality checks and cell filtering procedures, we performed exploratory analyses to identify most stable genes to be possibly used as reference housekeeping genes in qPCR experiments. We also illustrate how to use some popular R packages to investigate cell heterogeneity in scRNA-seq data, namely Seurat, Monocle, and slalom. Both the CITE-seq dataset and the code used to analyze it are freely shared and fully reusable for future research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=938&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b5bb0303397a14799ece5114c3940a235752454f\",\n",
      "        \"title\": \"Constrained chromatin accessibility in PU.1-mutated agammaglobulinemia patients\",\n",
      "        \"abstract\": \"PU.1 haploinsufficiency reduces the access of key transcription factors to open chromatin regions, undermines essential stage-specific transcriptional programs, and arrests human B cell development at the pro\\u2013 to pre\\u2013B cell transition.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=939&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eedf57fe620636aee26df9444acace8e9eca1416\",\n",
      "        \"title\": \"STARsolo: accurate, fast and versatile mapping/quantification of single-cell and single-nucleus RNA-seq data\",\n",
      "        \"abstract\": \"We present STARsolo, a comprehensive turnkey solution for quantifying gene expression in single-cell/nucleus RNA-seq data, built into RNA-seq aligner STAR. Using simulated data that closely resembles realistic scRNA-seq, we demonstrate that STARsolo is highly accurate and significantly outperforms pseudoalignment-to-transcriptome tools. STARsolo can replicate the results of, but is considerably faster than CellRanger, currently the most widely used tool for pre-processing scRNA-seq data. In addition to uniquely mapped reads, STARsolo takes account of multi-gene reads, necessary to detect certain classes of biologically important genes. It has a flexible cell barcode processing scheme, compatible with many established scRNA-seq protocols, and extendable to emerging technologies. STARsolo can quantify transcriptomic features beyond gene expression, which we illustrate by analyzing cell-type-specific alternative splicing in the Tabula Muris project.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=940&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"becdf202f96f02c7a2bdc6d77afdf7d1a40652f2\",\n",
      "        \"title\": \"Unraveling Root Development Through Single-Cell Omics and Reconstruction of Gene Regulatory Networks\",\n",
      "        \"abstract\": \"Over the last decades, research on postembryonic root development has been facilitated by \\u201comics\\u201d technologies. Among these technologies, microarrays first, and RNA sequencing (RNA-seq) later, have provided transcriptional information on the underlying molecular processes establishing the basis of System Biology studies in roots. Cell fate specification and development have been widely studied in the primary root, which involved the identification of many cell type transcriptomes and the reconstruction of gene regulatory networks (GRN). The study of lateral root (LR) development has not been an exception. However, the molecular mechanisms regulating cell fate specification during LR formation remain largely unexplored. Recently, single-cell RNA-seq (scRNA-seq) studies have addressed the specification of tissues from stem cells in the primary root. scRNA-seq studies are anticipated to be a useful approach to decipher cell fate specification and patterning during LR formation. In this review, we address the different scRNA-seq strategies used both in plants and animals and how we could take advantage of scRNA-seq to unravel new regulatory mechanisms and reconstruct GRN. In addition, we discuss how to integrate scRNA-seq results with previous RNA-seq datasets and GRN. We also address relevant findings obtained through single-cell based studies and how LR developmental studies could be facilitated by scRNA-seq approaches and subsequent GRN inference. The use of single-cell approaches to investigate LR formation could help to decipher fundamental biological mechanisms such as cell memory, synchronization, polarization, or pluripotency.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=941&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcfc20fad97736540e48e09877fe2b12880df70c\",\n",
      "        \"title\": \"Computational principles and challenges in single-cell data integration\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=942&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3668a54177372de46e9d47fc4cdbbddf248bb802\",\n",
      "        \"title\": \"Schema: metric learning enables interpretable synthesis of heterogeneous single-cell modalities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=943&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e1bc95f4306f3d2737327b3187997cba81f26fcf\",\n",
      "        \"title\": \"Scarf: A toolkit for memory efficient analysis of large-scale single-cell genomics data\",\n",
      "        \"abstract\": \"The increasing capacity to perform large-scale single-cell genomic experiments continues to outpace the computational requirements to efficiently handle growing datasets. Herein we present Scarf, a modularly designed Python package that seamlessly interoperates with other single-cell toolkits and allows for memory-efficient single-cell analysis of millions of cells on a laptop or low-cost devices like single board computers. We demonstrate Scarf\\u2019s memory and compute-time efficiency by applying it to the largest existing single-cell RNA-Seq and ATAC-Seq datasets. Scarf wraps memory-efficient implementations of a graph-based t-stochastic neighbour embedding and hierarchical clustering algorithm. Moreover, Scarf performs accurate reference-anchored mapping of datasets while maintaining memory efficiency. By implementing a novel data downsampling algorithm, Scarf additionally can generate representative sampling of cells from a given dataset wherein rare cell populations and lineage differentiation trajectories are conserved. Together, Scarf provides a framework wherein any researcher can perform advanced processing, downsampling, reanalysis, and integration of atlas-scale datasets on standard laptop computers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=944&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"96812dc84e4c8312b470a67881c0eb7630314b29\",\n",
      "        \"title\": \"Immune profiling of COVID-19: preliminary findings and implications for the pandemic\",\n",
      "        \"abstract\": \"SARS-CoV-2 infection can have widely diverse clinical outcomes, from asymptomatic infection to death, with many possible clinical symptoms and syndromes. It is thus essential to understand how the virus interacts with the host immune system to bring about these varied outcomes and to inform vaccine development. We now know that both antibody and T cell responses are induced in the majority of infected individuals, and that cross-reactive responses from other coronaviruses also exist in the uninfected population. Innate immune responses are a key focus of research and may influence the course of disease and the character of subsequent adaptive responses. Finally, baseline immune profiles and changes during early acute infection may be key to predicting the course of disease. Understanding all these aspects can help to create better immune monitoring tools for COVID-19, including tools for predicting disease severity or specific sequelae, perhaps even prior to infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=945&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a49d4d874c3afe619e0691d69d961108935af49\",\n",
      "        \"title\": \"The Known Unknowns of the Immune Response to Coccidioides\",\n",
      "        \"abstract\": \"Coccidioidomycosis, otherwise known as Valley Fever, is caused by the dimorphic fungi Coccidioides immitis and C. posadasii. While most clinical cases present with self-limiting pulmonary infection, dissemination of Coccidioides spp. results in prolonged treatment and portends higher mortality rates. While the structure, genome, and niches for Coccidioides have provided some insight into the pathogenesis of disease, the underlying immunological mechanisms of clearance or inability to contain the infection in the lung are poorly understood. This review focuses on the known innate and adaptive immune responses to Coccidioides and highlights three important areas of uncertainty and potential approaches to address them. Closing these gaps in knowledge may enable new preventative and therapeutic strategies to be pursued.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=946&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1295d88d45ac872122a838255e7228e8e6b2995f\",\n",
      "        \"title\": \"Cell-level metadata are indispensable for documenting single-cell sequencing datasets\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) provides an unprecedented view of cellular diversity of biological systems. However, across the thousands of publications and datasets generated using this technology, we estimate that only a minority (<25%) of studies provide cell-level metadata information containing identified cell types and related findings of the published dataset. Metadata omission hinders reproduction, exploration, validation, and knowledge transfer and is a common problem across journals, data repositories, and publication dates. We encourage investigators, reviewers, journals, and data repositories to improve their standards and ensure proper documentation of these valuable datasets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=947&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b77dbd68f467e4541f47218538276a2616ce5b8a\",\n",
      "        \"title\": \"Single-cell multiomics dissection of basal and antigen-specific activation states of CD19-targeted CAR T cells\",\n",
      "        \"abstract\": \"Background Autologous T cells engineered to express a chimeric antigen receptor (CAR) specific for CD19 molecule have transformed the therapeutic landscape in patients with highly refractory leukemia and lymphoma, and the use of donor-generated allogeneic CAR T is paving the way for further breakthroughs in the treatment of cancer. However, it remains unknown how the intrinsic heterogeneities of these engineered cells mediate therapeutic efficacy and whether allogeneic products match the effectiveness of autologous therapies. Methods Using single-cell mRNA sequencing in conjunction with CITE-seq, we performed multiomics characterization of CAR T cells generated from healthy donor and patients with acute lymphoblastic leukemia. CAR T cells used in this study were manufactured at the University of Pennsylvania through lentiviral transduction with a CD19-4-1BB-CD3\\u03b6 construct. Besides the baseline condition, we engineered NIH-3T3 cells with human CD19 or mesothelin expression to conduct ex vivo antigen-specific or non-antigen stimulation of CAR T cells through 6-hour coculture at a 1:1 ratio. Results We delineated the global cellular and molecular CAR T landscape and identified that transcriptional CAR tonic signaling was regulated by a mixture of early activation, exhaustion signatures, and cytotoxic activities. On CD19 stimulation, we illuminated the disparities of CAR T cells derived from different origins and found that donor CAR T had more pronounced activation level in correlation with the upregulation of major histocompatibility complex class II genes compared with patient CAR T cells. This finding was independently validated in additional datasets from literature. Furthermore, GM-CSF(CSF2) expression was found to be associated with functional gene productions, but it induced little impact on the CAR T activation. Conclusions Through integrated multiomics profiling and unbiased canonical pathway analyses, our results unveil heterogeneities in the transcriptional, phenotypic, functional, and metabolic profiles of donor and patient CAR T cells, providing mechanistic basis for ameliorating clinical outcomes and developing next-generation \\u2018off- the-shelf\\u2019 allogeneic products.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=948&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17bda6d5b9cbc92818ce67b2a985b8f74b7e9827\",\n",
      "        \"title\": \"A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=949&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3a052246a9457bf22a834401a25b2b7c64eb7ead\",\n",
      "        \"title\": \"Single-cell manifold-preserving feature selection for detecting rare cell populations\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=950&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0f20c882394937e1f9526d2c7c57abb189900f8f\",\n",
      "        \"title\": \"Current and Prospective Methods for Assessing Anti-Tumor Immunity in Colorectal Cancer\",\n",
      "        \"abstract\": \"Colorectal cancer (CRC) remains one of the deadliest malignancies worldwide despite recent progress in treatment strategies. Though immune checkpoint inhibition has proven effective for a number of other tumors, it offers benefits in only a small group of CRC patients with high microsatellite instability. In general, heterogenous cell groups in the tumor microenvironment are considered as the major barrier for unveiling the causes of low immune response. Therefore, deconvolution of cellular components in highly heterogeneous microenvironments is crucial for understanding the immune contexture of cancer. In this review, we assimilate current knowledge and recent studies examining anti-tumor immunity in CRC. We also discuss the utilization of novel immune contexture assessment methods that have not been used in CRC research to date.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=951&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"52320a6fc658634454434abeb74c74d23bc67f5a\",\n",
      "        \"title\": \"Taming Cell-to-Cell Heterogeneity in Acute Myeloid Leukaemia With Machine Learning\",\n",
      "        \"abstract\": \"Acute Myeloid Leukaemia (AML) is a phenotypically and genetically heterogenous blood cancer characterised by very poor prognosis, with disease relapse being the primary cause of treatment failure. AML heterogeneity arise from different genetic and non-genetic sources, including its proposed hierarchical structure, with leukemic stem cells (LSCs) and progenitors giving origin to a variety of more mature leukemic subsets. Recent advances in single-cell molecular and phenotypic profiling have highlighted the intra and inter-patient heterogeneous nature of AML, which has so far limited the success of cell-based immunotherapy approaches against single targets. Machine Learning (ML) can be uniquely used to find non-trivial patterns from high-dimensional datasets and identify rare sub-populations. Here we review some recent ML tools that applied to single-cell data could help disentangle cell heterogeneity in AML by identifying distinct core molecular signatures of leukemic cell subsets. We discuss the advantages and limitations of unsupervised and supervised ML approaches to cluster and classify cell populations in AML, for the identification of biomarkers and the design of personalised therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=952&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b5da48f9dd2a4283af06dfb7ed480c111f00de04\",\n",
      "        \"title\": \"Single-cell multi-omics analysis reveals IFN-driven alterations in T lymphocytes and natural killer cells in systemic lupus erythematosus\",\n",
      "        \"abstract\": \"Background The characterisation of the peripheral immune system in the autoimmune disease systemic lupus erythematosus (SLE) at the single-cell level has been hampered by the reduced sensitivity of current whole-transcriptomic technologies. Here we employ a targeted single-cell multi-omics approach, combining protein and mRNA quantification, to generate a high-resolution map of the T lymphocyte and Natural Killer (NK) cell populations in blood from SLE patients.\\n\\nMethods We designed a custom panel to quantify the transcription of 534 genes in parallel with the expression of 51 surface protein targets using the BD Rhapsody single-cell system. We applied this technology to profile 20,656 T and NK cells isolated from peripheral blood from an SLE patient with a type I interferon (IFN) signature (IFNhi), and an age- and sex- matched IFNlow SLE patient and healthy donor.\\n\\nResults We confirmed the presence of a rare cytotoxic CD4+ T cell (CTL) subset, which was exclusively present in the IFNhi patient. Furthermore, we identified additional alterations consistent with increased immune activation in this patient, most notably a shift towards terminally differentiated CD57+ CD8+ T cell and CD16+ NKdim phenotypes, and the presence of a subset of naive CD4+ T cells recently activated by cytokines.\\n\\nConclusions Our results identify IFN-driven changes in the composition and phenotype of T and NK cells that are consistent with a systemic immune activation within the IFNhi patient, and underscore the power of this multi-omics approach to identify rare immune subsets, which could be missed using lower parametric or less sensitive single-cell technologies. Consequently, we were able to find evidence for novel cellular peripheral biomarkers of SLE disease activity, including a subset of CD57+ CD4+ CTLs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=953&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e8c0f2d479b673b163cac2016f782708019c1b64\",\n",
      "        \"title\": \"Hotspot identifies informative gene modules across modalities of single-cell genomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=954&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3172bbb8adf533dc0df0d875ea13e0a512443f80\",\n",
      "        \"title\": \"Subcellular proteomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=955&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ef4afb99e575445d27d541a283bbb356f0a1e5cb\",\n",
      "        \"title\": \"Gene signature extraction and cell identity recognition at the single-cell level with Cell-ID\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=956&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b824fda4a28aac54d845432a387db0d8eb47b20c\",\n",
      "        \"title\": \"scvi-tools: a library for deep probabilistic analysis of single-cell omics data\",\n",
      "        \"abstract\": \"Probabilistic models have provided the underpinnings for state-of-the-art performance in many single-cell omics data analysis tasks, including dimensionality reduction, clustering, differential expression, annotation, removal of unwanted variation, and integration across modalities. Many of the models being deployed are amenable to scalable stochastic inference techniques, and accordingly they are able to process single-cell datasets of realistic and growing sizes. However, the community-wide adoption of probabilistic approaches is hindered by a fractured software ecosystem resulting in an array of packages with distinct, and often complex interfaces. To address this issue, we developed scvi-tools (https://scvi-tools.org), a Python package that implements a variety of leading probabilistic methods. These methods, which cover many fundamental analysis tasks, are accessible through a standardized, easy-to-use interface with direct links to Scanpy, Seurat, and Bioconductor workflows. By standardizing the implementations, we were able to develop and reuse novel functionalities across different models, such as support for complex study designs through nonlinear removal of unwanted variation due to multiple covariates and reference-query integration via scArches. The extensible software building blocks that underlie scvi-tools also enable a developer environment in which new probabilistic models for single cell omics can be efficiently developed, benchmarked, and deployed. We demonstrate this through a code-efficient reimplementation of Stereoscope for deconvolution of spatial transcriptomics profiles. By catering to both the end user and developer audiences, we expect scvi-tools to become an essential software dependency and serve to formulate a community standard for probabilistic modeling of single cell omics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=957&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3aa8055fa19f175b4ffccc3313e12c5ca88609e3\",\n",
      "        \"title\": \"Computational strategies for single-cell multi-omics integration\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=958&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"74341138918a4203c27276c8c06482ed4e1e77bf\",\n",
      "        \"title\": \"Sequential single cell transcriptional and protein marker profiling reveals TIGIT as a marker of CD19 CAR-T cell dysfunction in patients with non-Hodgkin\\u2019s lymphoma\",\n",
      "        \"abstract\": \"Chimeric antigen receptor T cell (CAR-T cell) therapy is known to produce durable remissions in the treatment of CD19+ relapsed/refractory B cell malignancies. Nonetheless, a significant portion of patients receiving the therapy experience poor outcomes in the acute response for unknown reasons. Given the decreased expansion and persistence of CD8 CAR-T cells in poor outcome groups, this failure may be attributed to CAR-T cell dysfunction. However, a comparison of the post-infusion transcriptional profiles and phenotypes between CAR-T cells of poor and favorable response groups has not been performed. Here, we employed single cell RNA sequencing and protein surface marker profiling of serial CAR-T cell blood samples from patients with CD19+ relapsed/refractory non-Hodgkin\\u2019s lymphoma (NHL) to reveal CAR-T cell evolution, identify biomarkers of response, and test for evidence of exhaustion in CAR-T cells of poor responders. At the transcriptional and protein levels, we note the evolution of a majority of CAR-T cells toward a non-proliferative and highly-differentiated state. In poor outcome patients, we observed a more marked enrichment of an exhaustion profile as compared to favorable outcome patients. Lastly, we identified the checkpoint receptor TIGIT (T cell immunoreceptor with Ig and ITIM domains) as a novel prognostic biomarker and potential driver of CAR-T cell exhaustion. Altogether, we provide evidence of CAR-T cell dysfunction marked by TIGIT expression driving poor response in NHL patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=959&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa805b0346712af8be33e5f68ce8d0ccf890522d\",\n",
      "        \"title\": \"Theoretical Considerations for Next-Generation Proteomics\",\n",
      "        \"abstract\": \"While mass spectrometry still dominates proteomics research, alternative and potentially disruptive, next-generation technologies are receiving increased investment and attention. Most of these technologies aim at the sequencing of single peptide or protein molecules, typically labeling or otherwise distinguishing a subset of the proteinogenic amino acids. This note considers some theoretical aspects of these future technologies from a bottom-up proteomics viewpoint, including the ability to uniquely identify human proteins as a function of which and how many amino acids can be read, enzymatic efficiency, and the maximum read length. This is done through simulations under ideal and non-ideal conditions to set benchmarks for what may be achievable with future single-molecule sequencing technology. The simulations reveal, among other observations, that the best choice of reading N amino acids performs similarly to the average choice of N+1 amino acids, and that the discrimination power of the amino acids scales with their frequency in the proteome. The simulations are agnostic with respect to the next-generation proteomics platform, and the results and conclusions should therefore be applicable to any single-molecule partial peptide sequencing technology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=960&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c71536e0712efe96906301254c06ab5f1732563b\",\n",
      "        \"title\": \"Gene regulatory network inference in single-cell biology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=961&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ad76ea16f078db33683ee98dd7a137683c86492\",\n",
      "        \"title\": \"Robust single-cell discovery of RNA targets of RNA binding proteins and ribosomes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=962&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c91385154ce488fef5594e7ee87e7ebf64798871\",\n",
      "        \"title\": \"Barcoded viral tracing of single-cell interactions in central nervous system inflammation\",\n",
      "        \"abstract\": \"Single-cell analysis of CNS interactions Despite their importance in the physiology and pathology of the central nervous system (CNS), few methods are available for the unbiased, systematic investigation of cell-to-cell interactions at single-cell resolution. Clark et al. developed RABID-seq, a method that combines barcoded viral tracing with single-cell RNA sequencing (see the Perspective by Silvin and Ginhoux). RABID-seq identified the axon guidance molecules Sema4D-PlexinB2 and EphrinB3-EphB3 as mediators of microglia-astrocyte interactions that promote CNS pathology in experimental autoimmune encephalomyelitis and, potentially, multiple sclerosis. These studies also identified candidate therapeutic molecules for the modulation of microglia-astrocyte interactions in multiple sclerosis. Science, this issue p. eabf1230; see also p. 342 A high-throughput method characterizing cell interactions elucidates microglia-astrocyte communication in the brain. INTRODUCTION Glial cells of the central nervous system (CNS), including astrocytes and microglia, play critical roles in development, tissue repair, and homeostasis. However, dysregulated astrocyte and microglia responses contribute to the pathogenesis of neurologic diseases. Indeed, environmental chemicals, microbial metabolites, and cell-cell interactions have been shown to modulate disease-promoting responses in astrocytes and microglia in the context of multiple sclerosis (MS) and its model, experimental autoimmune encephalomyelitis (EAE). In particular, although astrocyte interactions with microglia are known to play important roles in the pathology of MS and other neurologic diseases, the pathways that facilitate astrocyte-microglia cross-talk are poorly understood, and consequently, few therapeutic inventions are available to target them. RATIONALE Understanding the complexity of astrocyte-microglia cross-talk in CNS inflammation requires the study of precise neuroimmune interactions in vivo, but methodologies for defining the specific cell types, pathways, and molecules that mediate these interactions are limited. We developed a virus-based barcoding method for the identification of thousands of CNS cell interactions in vivo and the simultaneous analysis of the transcriptome of interacting cells with single-cell resolution. We applied this technique, named rabies barcode interaction detection followed by sequencing (RABID-seq), to the study of microglia-astrocyte communication in the context of CNS inflammation in EAE and MS. RESULTS To develop RABID-seq, we engineered an mRNA-barcoded library in glycoprotein G\\u2013deficient pseudorabies virus (Rab\\u0394G-BC), which spreads between interacting cells but can only replicate in cells that transgenically express viral glycoprotein G. We pseudotyped the Rab\\u0394G-BC plasmid library using envelope protein of subgroup A (EnvA) packaging. Thus, the pseudotyped virus only infects cells that transgenically express the EnvA receptor, TVA. After its replication in cells that express TVA and viral glycoprotein G, Rab\\u0394G-BC infects interacting cells, labelling them with the virus-encoded barcode. To study RABID-seq astrocyte interactions in vivo during CNS inflammation in the EAE model of MS, we used transgenic mice expressing glycoprotein G and TVA in astrocytes under the control of the Gfap promoter. These studies identified several axon guidance molecules as critical mediators of microglia-astrocyte interactions in the context of inflammation. By combining RABID-seq with genetic perturbation studies in vivo, validation with primary mouse and human cells in vitro, and the analysis of MS patient samples by immunostaining and single-cell RNA-seq, we established that microglia-astrocyte interactions mediated by Sema4D-PlexinB1, Sema4D-PlexinB2, and Ephrin-B3\\u2013EphB3 promote CNS pathology in EAE\\u2014and potentially MS. Notably, Ephrin-B\\u2013EphB3 participated in forward and reverse signaling, which boosted both microglia and astrocyte pathogenic activities via the regulation of nuclear factor \\u03baB and mammalian target of rapamycin, respectively. Finally, we demonstrated that a CNS-penetrant small-molecule inhibitor of the kinase activity of the EphB3 intracellular domain ameliorates EAE in both acute and chronic progressive models. CONCLUSION We developed RABID-seq, a novel approach for the simultaneous investigation of cell interactions and the transcriptome of interacting cells in vivo with single-cell resolution. RABID-seq identified signaling pathways controlled by the axon guidance molecules Sema4D-PlexinB1, Sema4D-PlexinB2, and Ephrin-B3/EphB3 as mediators of microglia-astrocyte interactions that promote CNS pathogenesis and also as candidate targets for therapeutic intervention in neurologic disorders. Elucidation of microglia-astrocyte interactions by rabies barcode interaction detection followed by sequencing (RABID-seq). Pseudotyped rabies virus expressing barcoded mRNA targets Gfap+ astrocytes, where it replicates before infecting neighboring cells, leaving a barcoded trace. Single-cell RNA sequencing reads both cellular mRNAs and viral barcodes, allowing for the reconstruction of in vivo cell interactions and the transcriptional analysis of interacting cells with single-cell resolution. Cell-cell interactions control the physiology and pathology of the central nervous system (CNS). To study astrocyte cell interactions in vivo, we developed rabies barcode interaction detection followed by sequencing (RABID-seq), which combines barcoded viral tracing and single-cell RNA sequencing (scRNA-seq). Using RABID-seq, we identified axon guidance molecules as candidate mediators of microglia-astrocyte interactions that promote CNS pathology in experimental autoimmune encephalomyelitis (EAE) and, potentially, multiple sclerosis (MS). In vivo cell-specific genetic perturbation EAE studies, in vitro systems, and the analysis of MS scRNA-seq datasets and CNS tissue established that Sema4D and Ephrin-B3 expressed in microglia control astrocyte responses via PlexinB2 and EphB3, respectively. Furthermore, a CNS-penetrant EphB3 inhibitor suppressed astrocyte and microglia proinflammatory responses and ameliorated EAE. In summary, RABID-seq identified microglia-astrocyte interactions and candidate therapeutic targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=963&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f7967304174a83fa6306c526d7fa437b5ec459a\",\n",
      "        \"title\": \"Deciphering transcriptional and functional heterogeneity in hematopoiesis with single-cell genomics.\",\n",
      "        \"abstract\": \"PURPOSE OF REVIEW\\nSingle-cell genomic approaches have uncovered cell fate biases and heterogeneity within hematopoietic subpopulations. However, standard single-cell transcriptomics suffers from high sampling noise, which particularly skews the distribution of lowly expressed genes, such as transcription factors (TFs). This might preclude the identification of rare transcripts that define cell identity and demarcate cell fate biases. Moreover, these studies need to go hand in hand with relevant functional assays to ensure that observed gene expression changes represent biologically meaningful alterations.\\n\\n\\nRECENT FINDINGS\\nSingle-cell lineage tracing and functional validation studies have uncovered cell fate bias within transcriptionally distinct hematopoietic stem and progenitor subpopulations. Novel markers identified using these strategies have been proposed to prospectively isolate functionally distinct subpopulations, including long-term hematopoietic stem cells for ex vivo applications. Furthermore, the continuous nature of hematopoiesis has prompted the study of the relationship between stochastic transcriptional noise in hematopoietic TFs and cell fate determination.\\n\\n\\nSUMMARY\\nAn understanding of the limitations of single-cell genomic approaches and follow-up functional assays is critical to discern the technical and biological contribution of noise in hematopoietic heterogeneity, to identify rare gene expression states, and to uncover functionally distinct subpopulations within hematopoiesis.\\n\\n\\nSUPPLEMENTARY VIDEO\\nhttp://links.lww.com/COH/A23.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=964&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b9da75b635ad5d9c9317a3095279b9faea879e8\",\n",
      "        \"title\": \"Discrete immune response signature to SARS-CoV-2 mRNA vaccination versus infection\",\n",
      "        \"abstract\": \"Both SARS-CoV-2 infection and vaccination elicit potent immune responses. A number of studies have described immune responses to SARS-CoV-2 infection. However, beyond antibody production, immune responses to COVID-19 vaccines remain largely uncharacterized. Here, we performed multimodal single-cell sequencing on peripheral blood of patients with acute COVID- 19 and healthy volunteers before and after receiving the SARS-CoV-2 BNT162b2 mRNA vaccine to compare the immune responses elicited by the virus and by this vaccine. Phenotypic and transcriptional profiling of immune cells, coupled with reconstruction of the B and T cell antigen receptor rearrangement of individual lymphocytes, enabled us to characterize and compare the host responses to the virus and to defined viral antigens. While both infection and vaccination induced robust innate and adaptive immune responses, our analysis revealed significant qualitative differences between the two types of immune challenges. In COVID-19 patients, immune responses were characterized by a highly augmented interferon response which was largely absent in vaccine recipients. Increased interferon signaling likely contributed to the observed dramatic upregulation of cytotoxic genes in the peripheral T cells and innate-like lymphocytes in patients but not in immunized subjects. Analysis of B and T cell receptor repertoires revealed that while the majority of clonal B and T cells in COVID-19 patients were effector cells, in vaccine recipients clonally expanded cells were primarily circulating memory cells. Importantly, the divergence in immune subsets engaged, the transcriptional differences in key immune populations, and the differences in maturation of adaptive immune cells revealed by our analysis have far-ranging implications for immunity to this novel pathogen.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=965&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"250017a7084c9fe5db29b8e06d09e86d8c90042c\",\n",
      "        \"title\": \"IFN-\\u03b3 and TNF-\\u03b1 drive a CXCL10+ CCL2+ macrophage phenotype expanded in severe COVID-19 lungs and inflammatory diseases with tissue inflammation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=966&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8083d39d09c2aa01514f8736eebb77d1cee70f2e\",\n",
      "        \"title\": \"Impaired antigen-specific memory B cell and plasma cell responses including lack of specific IgG upon SARS-CoV-2 BNT162b2 vaccination among Kidney Transplant and Dialysis patients\",\n",
      "        \"abstract\": \"Patients with kidney failure are at increased risk during the COVID-19 pandemic and effective vaccinations are needed. It is not known how efficient mRNA vaccines mount B and plasma cell responses in dialysis patients (DP) or kidney transplant recipients (KTR) compared to healthy controls (HC). We studied humoral and B cell responses of 25 HC, 44 DP and 40 KTR. Markedly impaired anti-BNT162b2 responses were identified among KTR and DP compared to 100% seroconversion in HC. In DP, the response was delayed (3-4 weeks after boost) and reduced with anti-S1 IgG positivity in 31 (70.5%) and anti-S1 IgA in 30 (68.2%) of 44, respectively. In contrast, KTR did not develop IgG response except one patient who had prior unrecognized infection and developed anti-S1 IgG. The majority of antigen-specific B cells (RBD+) were identified in the plasmablast or post-switch memory B cell compartments in HC, whereas these RBD+ B cells were enriched among pre-switch and naive B cells from DP and KTR. Single cell transcriptome and CITE-seq analyses found reduced frequencies of plasmablasts, TCF7+CD27+GZMK+ T cells and proliferating MKI67-expressing lymphocytes among KTR non-responders. Importantly, the frequency and absolute number of antigen-specific circulating plasmablasts in the whole cohort correlated with the Ig response, a characteristic not reported for other vaccinations. In conclusion, this data indicate that lack of T cell help related to immunosuppression results in impaired germinal center differentiation of B and plasma cell memory. There is an urgent need to improve vaccination protocols in patients after kidney transplantation or on chronic dialysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=967&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fda87aa21463f92a16ecd493be995c67cffec755\",\n",
      "        \"title\": \"Axes of inter-sample variability among transcriptional neighborhoods reveal disease-associated cell states in single-cell data\",\n",
      "        \"abstract\": \"As single-cell datasets grow in sample size, there is a critical need to characterize cell states that vary across samples and associate with sample attributes like clinical phenotypes. Current statistical approaches typically map cells to cell-type clusters and examine sample differences through that lens alone. Here we present covarying neighborhood analysis (CNA), an unbiased method to identify cell populations of interest with greater flexibility and granularity. CNA characterizes dominant axes of variation across samples by identifying groups of very small regions in transcriptional space\\u2014termed neighborhoods\\u2014that covary in abundance across samples, suggesting shared function or regulation. CNA can then rigorously test for associations between any sample-level attribute and the abundances of these covarying neighborhood groups. We show in simulation that CNA enables more powerful and accurate identification of disease-associated cell states than a cluster-based approach. When applied to published datasets, CNA captures a Notch activation signature in rheumatoid arthritis, redefines monocyte populations expanded in sepsis, and identifies a previously undiscovered T-cell population associated with progression to active tuberculosis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=968&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4429c7e3398cbea4c6d254dd4202653d7959e391\",\n",
      "        \"title\": \"Multimodally profiling memory T cells from a tuberculosis cohort identifies cell state associations with demographics, environment, and disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=969&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d70847f54cd25bf07c0859d955bf69ebc11af6e9\",\n",
      "        \"title\": \"Harnessing pluripotent stem cells as models to decipher human evolution\",\n",
      "        \"abstract\": \"The study of human evolution, long constrained by a lack of experimental model systems, has been transformed by the emergence of the induced pluripotent stem cell (iPSC) field. iPSCs can be readily established from noninvasive tissue sources, from both humans and other primates; they can be maintained in the laboratory indefinitely, and they can be differentiated into other tissue types. These qualities mean that iPSCs are rapidly becoming established as viable and powerful model systems with which it is possible to address questions in human evolution that were until now logistically and ethically intractable, especially in the quest to understand humans' place among the great apes, and the genetic basis of human uniqueness. In this review, we discuss the key lessons and takeaways of this nascent field; from the types of research, iPSCs make possible to lingering challenges and likely future directions. We provide a comprehensive overview of how the seemingly unlikely combination of iPSCs and explicit evolutionary frameworks is transforming what is possible in our understanding of humanity's past and present.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=970&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bce06dec9655f7c8ba471f0641bd0ced19d654dd\",\n",
      "        \"title\": \"MarkerCount: A stable, count-based cell type identifier for single-cell RNA-seq experiments\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=971&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"162292ac6ee9a4f4e35300403aeaaa382246c3ad\",\n",
      "        \"title\": \"Brown Adipose Tissue Heterogeneity, Energy Metabolism, and Beyond\",\n",
      "        \"abstract\": \"Brown adipocyte in brown adipose tissue (BAT) specializes in expending energy through non-shivering thermogenesis, a process that produces heat either by uncoupling protein 1 (UCP1) dependent uncoupling of mitochondrial respiration or by UCP1 independent mechanisms. Apart from this, there is ample evidence suggesting that BAT has an endocrine function. Studies in rodents point toward its vital roles in glucose and lipid homeostasis, making it an important therapeutic target for treating metabolic disorders related to morbidities such as obesity and type 2 diabetes. The rediscovery of thermogenically active BAT depots in humans by several independent research groups in the last decade has revitalized interest in BAT as an even more promising therapeutic intervention. Over the last few years, there has been overwhelming interest in understanding brown adipocyte\\u2019s developmental lineages and how brown adipocyte uniquely utilizes energy beyond UCP1 mediated uncoupling respiration. These new discoveries would be leveraged for designing novel therapeutic interventions for metabolic disorders.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=972&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a7b3348e47c299ab52564d6f685a4f00722bdacf\",\n",
      "        \"title\": \"Single-Cell Sequencing Methodologies: From Transcriptome to Multi-Dimensional Measurement.\",\n",
      "        \"abstract\": \"Cells are the basic building blocks of biological systems, with inherent unique molecular features and development trajectories. The study of single cells facilitates in-depth understanding of cellular diversity, disease processes, and organization of multicellular organisms. Single-cell RNA sequencing (scRNA-seq) technologies have become essential tools for the interrogation of gene expression patterns and the dynamics of single cells, allowing cellular heterogeneity to be dissected at unprecedented resolution. Nevertheless, measuring at only transcriptome level or 1D is incomplete; the cellular heterogeneity reflects in multiple dimensions, including the genome, epigenome, transcriptome, spatial, and even temporal dimensions. Hence, integrative single cell analysis is highly desired. In addition, the way to interpret sequencing data by virtue of bioinformatic tools also exerts critical roles in revealing differential gene expression. Here, a comprehensive review that summarizes the cutting-edge single-cell transcriptome sequencing methodologies, including scRNA-seq, spatial and temporal transcriptome profiling, multi-omics sequencing and computational methods developed for scRNA-seq data analysis is provided. Finally, the challenges and perspectives of this field are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=973&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0add6518b127ea203ac8e96325fee2f574568635\",\n",
      "        \"title\": \"Single cell analyses to understand the immune continuum in atherosclerosis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=974&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ded299379ce978746af3f425bebff12b2a544226\",\n",
      "        \"title\": \"Maximizing statistical power to detect differentially abundant cell states with scPOST\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=975&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50677703db5f43c37d66b3da4520d052da115ef6\",\n",
      "        \"title\": \"Single-cell antigen-specific activation landscape of CAR T infusion product identifies determinants of CD19 positive relapse in patients with ALL\",\n",
      "        \"abstract\": \"Chimeric antigen receptor-modified (CAR) T cells targeting CD19 have mediated dramatic responses in relapsed/refractory acute lymphoblastic leukemia (ALL), yet a notable number of patients have CD19-positive relapse within one year of treatment. It remains unclear if the long-term response is associated with the characteristics of CAR T cells in infusion products, hindering the identification of biomarkers to predict therapeutic outcomes prior to treatment. Herein we present 101,326 single cell transcriptomes and surface protein landscape from the CAR T infusion products of 12 pediatric ALL patients upon CAR antigen-specific stimulation in comparison with TCR-mediated activation and controls. We observed substantial heterogeneity in the antigen-specific activation states, among which a deficiency of Th2 function was associated with CD19-positive relapsed patients (median remission 9.6 months) compared with very durable responders (remission>54 months). Proteomic profiles also revealed that the frequency of early memory T cell subsets, rather than activation or co-inhibitory signatures could distinguish CD19-positive relapse. Additionally, a deficit of type 1 helper and cytotoxic effector function and an enrichment for terminally differentiated CD8+ T cells exhibiting low cytokine polyfunctionality was associated with initial non-responders. By contrast, the single-cell transcriptomic data of unstimulated or TCR-activated CAR T cells failed to predict clinical responses. In aggregate, our results dissect the landscape of CAR-specific activation states in infusion products that can identify patients who do not develop a durable response to the therapy, and unveil the molecular mechanisms that may inform strategies to boost specific T cell function to maintain long term remission.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=976&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"09e6ae1f5222dea683b4ecc53f6c6d3e82cfa9d8\",\n",
      "        \"title\": \"Absence of Batf3 reveals a new dimension of cell state heterogeneity within conventional dendritic cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=977&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c9559b662ac557bbab278726e82ce59510555b57\",\n",
      "        \"title\": \"Applications of single-cell and bulk RNA sequencing in onco-immunology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=978&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5a6bdf088980a6d3fa89337114699bf04b812500\",\n",
      "        \"title\": \"Molecular tracking devices quantify antigen distribution and archiving in the murine lymph node\",\n",
      "        \"abstract\": \"The detection of foreign antigens in vivo has relied on fluorescent conjugation or indirect read-outs such as antigen presentation. In our studies, we found that these widely used techniques had several technical limitations that have precluded a complete picture of antigen trafficking or retention across lymph node cell types. To address these limitations, we developed a \\u2018molecular tracking device\\u2019 to follow the distribution, acquisition, and retention of antigen in the lymph node. Utilizing an antigen conjugated to a nuclease-resistant DNA tag, acting as a combined antigen-adjuvant conjugate, and single-cell mRNA sequencing, we quantified antigen abundance in the lymph node. Variable antigen levels enabled the identification of caveolar endocytosis as a mechanism of antigen acquisition or retention in lymphatic endothelial cells. Thus, these molecular tracking devices enable new approaches to study dynamic tissue dissemination of antigen-adjuvant conjugates and identify new mechanisms of antigen acquisition and retention at cellular resolution in vivo.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=979&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ba4f991a03550a165de05a5e51b5162c3ace81e\",\n",
      "        \"title\": \"The Human Islet: Mini-organ with Mega-impact.\",\n",
      "        \"abstract\": \"This review focuses on the human pancreatic islet - including its structure, cell composition, development, function, and dysfunction. After providing a historical timeline of key discoveries about human islets over the past century, we describe new research approaches and technologies that are being used to study human islets and how these are providing insight into human islet physiology and pathophysiology. We also describe changes or adaptations in human islets in response to physiologic challenges such as pregnancy, aging, and insulin resistance and discuss islet changes in human diabetes of many forms. We outline current and future interventions being developed to protect, restore, or replace human islets. The review also highlights unresolved questions about human islets and proposes areas where additional research on human islets is needed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=980&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cca92c80e0fcee841637b3eec99510d42de31f69\",\n",
      "        \"title\": \"Simultaneous trimodal single-cell measurement of transcripts, epitopes, and chromatin accessibility using TEA-seq\",\n",
      "        \"abstract\": \"Single-cell measurements of cellular characteristics have been instrumental in understanding the heterogeneous pathways that drive differentiation, cellular responses to signals, and human disease. Recent advances have allowed paired capture of protein abundance and transcriptomic state, but a lack of epigenetic information in these assays has left a missing link to gene regulation. Using the heterogeneous mixture of cells in human peripheral blood as a test case, we developed a novel scATAC-seq workflow that increases signal-to-noise and allows paired measurement of cell surface markers and chromatin accessibility: integrated cellular indexing of chromatin landscape and epitopes, called ICICLE-seq. We extended this approach using a droplet-based multiomics platform to develop a trimodal assay that simultaneously measures transcriptomics (scRNA-seq), epitopes, and chromatin accessibility (scATAC-seq) from thousands of single cells, which we term TEA-seq. Together, these multimodal single-cell assays provide a novel toolkit to identify type-specific gene regulation and expression grounded in phenotypically defined cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=981&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3a3d7146582b1f101bddf9f1444aa90d20f18fbe\",\n",
      "        \"title\": \"Phase I Trial of N-803, an IL15 Receptor Agonist, with Rituximab in Patients with Indolent Non-Hodgkin Lymphoma\",\n",
      "        \"abstract\": \"Purpose: N-803 is an IL15 receptor superagonist complex, designed to optimize in vivo persistence and trans-presentation, thereby activating and expanding natural killer (NK) cells and CD8+ T cells. Monoclonal antibodies (mAbs) direct Fc receptor\\u2013bearing immune cells, including NK cells, to recognize and eliminate cancer targets. The ability of IL15R agonists to enhance tumor-targeting mAbs in patients has not been reported previously. Patients and Methods: Relapsed/refractory patients with indolent non-Hodgkin lymphoma were treated with rituximab and intravenous or subcutaneous N-803 on an open-label, dose-escalation phase I study using a 3+3 design (NCT02384954). Primary endpoint was maximum tolerated dose. Immune correlates were performed using multidimensional analysis via mass cytometry and cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) which simultaneously measures protein and single-cell RNA expression. Results: This immunotherapy combination was safe and well tolerated and resulted in durable clinical responses including in rituximab-refractory patients. Subcutaneous N-803 plus rituximab induced sustained proliferation, expansion, and activation of peripheral blood NK cells and CD8 T cells, with increased NK cell and T cells present 8 weeks following last N-803 treatment. CITE-seq revealed a therapy-altered NK cell molecular program, including enhancement of AP-1 transcription factor. Furthermore, the monocyte transcriptional program was remodeled with enhanced MHC expression and antigen-presentation genes. Conclusions: N-803 combines with mAbs to enhance tumor targeting in patients, and warrants further investigation in combination with immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=982&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"effec8b598c1d4d2f69716e4f900d5bfd4f124f2\",\n",
      "        \"title\": \"Prenatal Development and Function of Human Mononuclear Phagocytes\",\n",
      "        \"abstract\": \"The human mononuclear phagocyte (MP) system, which includes dendritic cells, monocytes, and macrophages, is a critical regulator of innate and adaptive immune responses. During embryonic development, MPs derive sequentially in yolk sac progenitors, fetal liver, and bone marrow haematopoietic stem cells. MPs maintain tissue homeostasis and confer protective immunity in post-natal life. Recent evidence \\u2013 primarily in animal models \\u2013 highlight their critical role in coordinating the remodeling, maturation, and repair of target organs during embryonic and fetal development. However, the molecular regulation governing chemotaxis, homeostasis, and functional diversification of resident MP cells in their respective organ systems during development remains elusive. In this review, we summarize the current understanding of the development and functional contribution of tissue MPs during human organ development and morphogenesis and its relevance to regenerative medicine. We outline how single-cell multi-omic approaches and next-generation ex-vivo organ-on-chip models provide new experimental platforms to study the role of human MPs during development and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=983&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4ba089148c3114d8377544a405c134d029afeb6a\",\n",
      "        \"title\": \"Discovery of target genes and pathways of blood trait loci using pooled CRISPR screens and single cell RNA sequencing\",\n",
      "        \"abstract\": \"The majority of variants associated with complex traits and common diseases identified by genome-wide association studies (GWAS) map to noncoding regions of the genome with unknown regulatory effects in cis and trans. By leveraging biobank-scale GWAS data, massively parallel CRISPR screens and single cell transcriptome sequencing, we discovered target genes of noncoding variants for blood trait loci. The closest gene was often the target gene, but this was not always the case. We also identified trans-effects networks of noncoding variants when cis target genes encoded transcription factors, such as GFI1B and NFE2. We observed that GFI1B trans-target genes were enriched for GFI1B binding sites and fine-mapped GWAS variants, and expressed in human bone marrow progenitor cells, suggesting that GFI1B acts as a master regulator of blood traits. This platform will enable massively parallel assays to catalog the target genes of human noncoding variants in both cis and trans.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=984&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bdd19aeff011dc175ee5583399f3d24257493edc\",\n",
      "        \"title\": \"Single-cell sequencing unveils distinct immune microenvironments with CCR6-CCL20 crosstalk in human chronic pancreatitis\",\n",
      "        \"abstract\": \"Objective Chronic pancreatitis (CP) is a potentially fatal disease of the exocrine pancreas, with no specific or effective approved therapies. Due to difficulty in accessing pancreas tissues, little is known about local immune responses or pathogenesis in human CP. We sought to characterise pancreatic immune responses using tissues derived from patients with different aetiologies of CP and non-CP organ donors in order to identify key signalling molecules associated with human CP. Design We performed single-cell level cellular indexing of transcriptomes and epitopes by sequencing and T-cell receptor (TCR) sequencing of pancreatic immune cells isolated from organ donors, hereditary and idiopathic patients with CP who underwent total pancreatectomy. We validated gene expression data by performing flow cytometry and functional assays in a second patient with CP cohort. Results Deep single-cell sequencing revealed distinct immune characteristics and significantly enriched CCR6+ CD4+ T cells in hereditary compared with idiopathic CP. In hereditary CP, a reduction in T-cell clonality was observed due to the increased CD4+ T (Th) cells that replaced tissue-resident CD8+ T cells. Shared TCR clonotype analysis among T-cell lineages also unveiled unique interactions between CCR6+ Th and Th1 subsets, and TCR clustering analysis showed unique common antigen binding motifs in hereditary CP. In addition, we observed a significant upregulation of the CCR6 ligand (CCL20) expression among monocytes in hereditary CP as compared with those in idiopathic CP. The functional significance of CCR6 expression in CD4+ T cells was confirmed by flow cytometry and chemotaxis assay. Conclusion Single-cell sequencing with pancreatic immune cells in human CP highlights pancreas-specific immune crosstalk through the CCR6-CCL20 axis, a signalling pathway that might be leveraged as a potential future target in human hereditary CP.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=985&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d3370e28c6060833b62711f0b1a977559feace7\",\n",
      "        \"title\": \"Integrative bulk and single-cell profiling of pre-manufacture T-cell populations reveals factors mediating long-term persistence of CAR T-cell therapy.\",\n",
      "        \"abstract\": \"The adoptive transfer of Chimeric Antigen Receptor (CAR) T-cells represents a breakthrough in clinical oncology, yet both between- and within-patient differences in autologously-derived T cells are a major contributor to therapy failure. In order to interrogate the molecular determinants of clinical CAR T-cell persistence, we extensively characterized the pre manufacture T-cells of 71 patients with B-cell malignancies on trial to receive anti-CD19 CAR T-cell therapy. We performed RNA-Seq on sorted T-cell subsets from all 71 patients, followed by paired CITE-Seq and single-cell ATAC-Seq on T-cells from 6 of these patients. We found that chronic interferon signaling regulated by IRF7 was associated with poor CAR T-cell persistence across T-cell subsets, and that the TCF7 regulon not only associates with the favorable naive T-cell state, but is maintained in effector T-cells among patients with long term CAR T-cell persistence. These findings provide key insights into the underlying molecular determinants of clinical CAR T-cell function.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=986&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ae6fc623f785fc46805a538c60334fbcaa6c9c37\",\n",
      "        \"title\": \"Leveraging Single-Cell Sequencing for Chimeric Antigen Receptor T Cell Therapies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=987&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"154a3c1efd290d3b1af064403cb3c5dce906e3de\",\n",
      "        \"title\": \"Critical downstream analysis steps for single-cell RNA sequencing data\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) has enabled us to study biological questions at the single-cell level. Currently, many analysis tools are available to better utilize these relatively noisy data. In this review, we summarize the most widely used methods for critical downstream analysis steps (i.e. clustering, trajectory inference, cell-type annotation and integrating datasets). The advantages and limitations are comprehensively discussed, and we provide suggestions for choosing proper methods in different situations. We hope this paper will be useful for scRNA-seq data analysts and bioinformatics tool developers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=988&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c6d0b2d579254b8e8cb16e36370116172afcd673\",\n",
      "        \"title\": \"Cobolt: Joint analysis of multimodal single-cell sequencing data\",\n",
      "        \"abstract\": \"A growing number of single-cell sequencing platforms enable joint profiling of multiple omics from the same cells. We present Cobolt, a novel method that not only allows for analyzing the data from joint-modality platforms, but provides a coherent framework for the integration of multiple datasets measured on different modalities. We demonstrate its performance on multi-modality data of gene expression and chromatin accessibility and illustrate the integration abilities of Cobolt by jointly analyzing this multi-modality data with single-cell RNA-seq and ATAC-seq datasets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=989&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5c8e1ee49a5e4f442d65b9aa4d51c7c5b6c29286\",\n",
      "        \"title\": \"Single-Cell Multiomics Reveals Distinct Cell States at the Top of the Human Hematopoietic Hierarchy\",\n",
      "        \"abstract\": \"The advent of single cell (Sc) genomics has challenged the dogma of haematopoiesis as a tree-like structure of stepwise lineage commitment through distinct and increasingly restricted progenitor populations. Instead, analysis of ScRNA-seq has proposed that the earliest events in human hematopoietic stem cell (HSC) differentiation are characterized by only subtle molecular changes, with hematopoietic stem and progenitor cells (HSPCs) existing as a continuum of low-primed cell-states that gradually transition into a specific lineage (CLOUD-HSPCs). Here, we combine ScRNA-seq, ScATAC-seq and cell surface proteomics to dissect the heterogeneity of CLOUD-HSPCs at different stages of human life. Within CLOUD-HSPCs, pseudotime ordering of both mRNA and chromatin data revealed a bifurcation of megakaryocyte/erythroid and lympho/myeloid trajectories immediately downstream a subpopulation with an HSC-specific enhancer signature. Importantly, both HSCs and lineage-restricted progenitor populations could be prospectively isolated based on correlation of their molecular signatures with CD35 and CD11A expression, respectively. Moreover, we describe the changes that occur in this heterogeneity as hematopoiesis develops from neonatal to aged bone marrow, including an increase of HSCs and depletion of lympho-myeloid biased MPPs. Thus, this study dissects the heterogeneity of human CLOUD-HSPCs revealing distinct HSPC-states of relevance in homeostatic settings such as ageing.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=990&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3d3b76f56107b79added62fbc4ab236326e5ae5f\",\n",
      "        \"title\": \"CASB: a concanavalin A\\u2010based sample barcoding strategy for single\\u2010cell sequencing\",\n",
      "        \"abstract\": \"Sample multiplexing facilitates single\\u2010cell sequencing by reducing costs, revealing subtle difference between similar samples, and identifying artifacts such as cell doublets. However, universal and cost\\u2010effective strategies are rather limited. Here, we reported a concanavalin A\\u2010based sample barcoding strategy (CASB), which could be followed by both single\\u2010cell mRNA and ATAC (assay for transposase\\u2010accessible chromatin) sequencing techniques. The method involves minimal sample processing, thereby preserving intact transcriptomic or epigenomic patterns. We demonstrated its high labeling efficiency, high accuracy in assigning cells/nuclei to samples regardless of cell type and genetic background, and high sensitivity in detecting doublets by three applications: 1) CASB followed by scRNA\\u2010seq to track the transcriptomic dynamics of a cancer cell line perturbed by multiple drugs, which revealed compound\\u2010specific heterogeneous response; 2) CASB together with both snATAC\\u2010seq and scRNA\\u2010seq to illustrate the IFN\\u2010\\u03b3\\u2010mediated dynamic changes on epigenome and transcriptome profile, which identified the transcription factor underlying heterogeneous IFN\\u2010\\u03b3 response; and 3) combinatorial indexing by CASB, which demonstrated its high scalability.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=991&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1299bf04f1df296be53594f522823a5618c73527\",\n",
      "        \"title\": \"Application of Single-Cell Approaches to Study Myeloproliferative Neoplasm Biology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=992&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bde77f346c13da1b1ecd6b0db36dd1a4617b9252\",\n",
      "        \"title\": \"Research Techniques Made Simple: Use of Imaging Mass Cytometry for Dermatological Research and Clinical Applications.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=993&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5f8708a6a7529cd27f092a5a452431dba5a31952\",\n",
      "        \"title\": \"Single-Cell Toolkits Opening a New Era for Cell Engineering\",\n",
      "        \"abstract\": \"Since the introduction of RNA sequencing (RNA-seq) as a high-throughput mRNA expression analysis tool, this procedure has been increasingly implemented to identify cell-level transcriptome changes in a myriad of model systems. However, early methods processed cell samples in bulk, and therefore the unique transcriptomic patterns of individual cells would be lost due to data averaging. Nonetheless, the recent and continuous development of new single-cell RNA sequencing (scRNA-seq) toolkits has enabled researchers to compare transcriptomes at a single-cell resolution, thus facilitating the analysis of individual cellular features and a deeper understanding of cellular functions. Nonetheless, the rapid evolution of high throughput single-cell \\u201comics\\u201d tools has created the need for effective hypothesis verification strategies. Particularly, this issue could be addressed by coupling cell engineering techniques with single-cell sequencing. This approach has been successfully employed to gain further insights into disease pathogenesis and the dynamics of differentiation trajectories. Therefore, this review will discuss the current status of cell engineering toolkits and their contributions to single-cell and genome-wide data collection and analyses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=994&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7756f51eea7c4a8c1bc20e34a70da8ea53065f92\",\n",
      "        \"title\": \"High Throughput Multi-Omics Approaches for Clinical Trial Evaluation and Drug Discovery\",\n",
      "        \"abstract\": \"High throughput single cell multi-omics platforms, such as mass cytometry (cytometry by time-of-flight; CyTOF), high dimensional imaging (>6 marker; Hyperion, MIBIscope, CODEX, MACSima) and the recently evolved genomic cytometry (Citeseq or REAPseq) have enabled unprecedented insights into many biological and clinical questions, such as hematopoiesis, transplantation, cancer, and autoimmunity. In synergy with constantly adapting new single-cell analysis approaches and subsequent accumulating big data collections from these platforms, whole atlases of cell types and cellular and sub-cellular interaction networks are created. These atlases build an ideal scientific discovery environment for reference and data mining approaches, which often times reveals new cellular disease networks. In this review we will discuss how combinations and fusions of different -omic workflows on a single cell level can be used to examine cellular phenotypes, immune effector functions, and even dynamic changes, such as metabolomic state of different cells in a sample or even in a defined tissue location. We will touch on how pre-print platforms help in optimization and reproducibility of workflows, as well as community outreach. We will also shortly discuss how leveraging single cell multi-omic approaches can be used to accelerate cellular biomarker discovery during clinical trials to predict response to therapy, follow responsive cell types, and define novel druggable target pathways. Single cell proteome approaches already have changed how we explore cellular mechanism in disease and during therapy. Current challenges in the field are how we share these disruptive technologies to the scientific communities while still including new approaches, such as genomic cytometry and single cell metabolomics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=995&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e1aada85ad05f6c59de1fd4ebce2647064b9e6ef\",\n",
      "        \"title\": \"Mammary gland development from a single cell \\u2018omics view\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=996&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"53bce8823e5821db95553d059dd6a254aac1f3b8\",\n",
      "        \"title\": \"Single-cell profiling of myeloid cells in glioblastoma across species and disease stage reveals macrophage competition and specialization\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=997&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c914e19e85f8edfb00f7b86899dfb7fa7c1cb4f4\",\n",
      "        \"title\": \"Multiplexed Immunohistochemistry and Digital Pathology as the Foundation for Next-Generation Pathology in Melanoma: Methodological Comparison and Future Clinical Applications\",\n",
      "        \"abstract\": \"The state-of-the-art for melanoma treatment has recently witnessed an enormous revolution, evolving from a chemotherapeutic, \\u201cone-drug-for-all\\u201d approach, to a tailored molecular- and immunological-based approach with the potential to make personalized therapy a reality. Nevertheless, methods still have to improve a lot before these can reliably characterize all the tumoral features that make each patient unique. While the clinical introduction of next-generation sequencing has made it possible to match mutational profiles to specific targeted therapies, improving response rates to immunotherapy will similarly require a deep understanding of the immune microenvironment and the specific contribution of each component in a patient-specific way. Recent advancements in artificial intelligence and single-cell profiling of resected tumor samples are paving the way for this challenging task. In this review, we provide an overview of the state-of-the-art in artificial intelligence and multiplexed immunohistochemistry in pathology, and how these bear the potential to improve diagnostics and therapy matching in melanoma. A major asset of in-situ single-cell profiling methods is that these preserve the spatial distribution of the cells in the tissue, allowing researchers to not only determine the cellular composition of the tumoral microenvironment, but also study tissue sociology, making inferences about specific cell-cell interactions and visualizing distinctive cellular architectures - all features that have an impact on anti-tumoral response rates. Despite the many advantages, the introduction of these approaches requires the digitization of tissue slides and the development of standardized analysis pipelines which pose substantial challenges that need to be addressed before these can enter clinical routine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=998&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"320b4b74ed4db2b5d5e36276a5ee71086564c89d\",\n",
      "        \"title\": \"Advances in bulk and single-cell multi-omics approaches for systems biology and precision medicine.\",\n",
      "        \"abstract\": \"Multi-omics allows the systematic understanding of the information flow across different omics layers, while single omics can mainly reflect one aspect of the biological system. The advancement of bulk and single-cell sequencing technologies and related computational methods for multi-omics largely facilitated the development of system biology and precision medicine. Single-cell approaches have the advantage of dissecting cellular dynamics and heterogeneity, whereas traditional bulk technologies are limited to individual/population-level investigation. In this review, we first summarize the technologies for producing bulk and single-cell multi-omics data. Then, we survey the computational approaches for integrative analysis of bulk and single-cell multimodal data, respectively. Moreover, the databases and data storage for multi-omics, as well as the tools for visualizing multimodal data are summarized. We also outline the integration between bulk and single-cell data, and discuss the applications of multi-omics in precision medicine. Finally, we present the challenges and perspectives for multi-omics development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=999&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"22d46ba1d36c3a92c9e677609b78e1400d9d7d15\",\n",
      "        \"title\": \"Time to Move to the Single-Cell Level: Applications of Single-Cell Multi-Omics to Hematological Malignancies and Waldenstr\\u00f6m\\u2019s Macroglobulinemia\\u2014A Particularly Heterogeneous Lymphoma\",\n",
      "        \"abstract\": \"Simple Summary Intra-tumor heterogeneity is inherent to all cancers and makes direct mapping of genotype\\u2013phenotype relationships challenging. The advent of single-cell multi-omics techniques has allowed us to begin to comprehensively dissect cellular heterogeneity and access biological information unobtainable from bulk analysis. Applications cover many fields and are increasingly numerous. This review aims to summarize the most important applications of single-cell technologies in hematological tumors, providing a translational view. Data show the power of single-cell multi-omics to resolve the complex biology of heterogeneous populations and to derive information that can be used to improve treatment strategies. We discuss, with a practical example, how to make use of these techniques to study the heterogeneity of a specific type of monoclonal gammopathy called Waldenstr\\u00f6m\\u2019s macroglobulinemia. Abstract Single-cell sequencing techniques have become a powerful tool for characterizing intra-tumor heterogeneity, which has been reflected in the increasing number of studies carried out and reported. We have rigorously reviewed and compiled the information about these techniques inasmuch as they are relative to the area of hematology to provide a practical view of their potential applications. Studies show how single-cell multi-omics can overcome the limitations of bulk sequencing and be applied at all stages of tumor development, giving insights into the origin and pathogenesis of the tumors, the clonal architecture and evolution, or the mechanisms of therapy resistance. Information at the single-cell level may help resolve questions related to intra-tumor heterogeneity that have not been previously explained by other techniques. With that in mind, we review the existing knowledge about a heterogeneous lymphoma called Waldenstr\\u00f6m\\u2019s macroglobulinemia and discuss how single-cell studies may help elucidate the underlying causes of this heterogeneity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1000&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"97fc13bc7028ad17c5cf8ceee687bff6d45b5309\",\n",
      "        \"title\": \"Model-based deep embedding for constrained clustering analysis of single cell RNA-seq data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1001&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5f9d0979e4fb4e9134e9e6af96f8dd6ede5618c3\",\n",
      "        \"title\": \"Macrophage phenotypes in tissue repair and the foreign body response: implications for biomaterial-based regenerative medicine strategies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1002&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"05f76401a46221a5943fb5dfa33835f3f6652cfd\",\n",
      "        \"title\": \"Importance of lymph node immune responses in MSI-H/dMMR colorectal cancer\",\n",
      "        \"abstract\": \"Patients with colorectal cancers (CRCs) generally exhibit improved survival through intensive lymph node (LN) dissection. However, recent progress in cancer immunotherapy revisits the potential importance of regional LNs, where T cells are primed to attack tumor cells. To elucidate the role of regional LN, we investigated the immunological status of nonmetastatic regional LN lymphocytes (LNLs) in comparison with those of the tumor microenvironment (tumor-infiltrating lymphocytes; TILs) using flow cytometry and next-generation sequencing. LNLs comprised an intermediate level of the effector T cell population between peripheral blood lymphocytes (PBLs) and TILs. Significant overlap of the T cell receptor (TCR) repertoire was observed in microsatellite instability\\u2013high/mismatch repair\\u2013deficient (MSI-H/dMMR) CRCs with high tumor mutation burden (TMB), although limited TCRs were shared between nonmetastatic LNs and primary tumors in microsatellite stable/MMR proficient (MSS/pMMR) CRC patients with low TMB. In line with the overlap of the TCR repertoire, an excessive LN dissection did not provide a positive impact on long-term prognosis in our MSI-H/dMMR CRC cohort (n = 130). We propose that regional LNs play an important role in antitumor immunity, particularly in MSI-H/dMMR CRCs with high TMB, requiring care to be taken regarding excessive nonmetastatic LN dissection in MSI-H/dMMR CRC patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1003&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4b77fe47634d68ad0eaf087a531baf72eef13da\",\n",
      "        \"title\": \"Forest Fire Clustering for single-cell sequencing combines iterative label propagation with parallelized Monte Carlo simulations\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1004&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bc2cd8067399d9ef1bb8b79d0fcc891dfd35959f\",\n",
      "        \"title\": \"Multifaceted involvement of microglia in gray matter pathology in multiple sclerosis\",\n",
      "        \"abstract\": \"In the inflammatory demyelinating neurodegenerative disease multiple sclerosis (MS), there is increasing interest in gray matter pathology, as neuronal loss and cortical atrophy correlate with disability and disease progression, and MS therapeutics fail to significantly slow or stop neurodegeneration. Microglia, the central nervous system (CNS)\\u2010resident macrophages, are extensively involved in white matter MS pathology, but are also implicated in gray matter pathology, similar to other neurodegenerative diseases, for which there is synaptic, axonal, and neuronal degeneration. Microglia display regional heterogeneity within the CNS, which reflects their highly plastic nature and their ability to deliver context\\u2010dependent responses tailored to the demands of their microenvironment. Therefore, microglial roles in the MS gray matter in part reflect and in part diverge from those in the white matter. The present review summarizes current knowledge of microglial involvement in gray matter changes in MS, in demyelination, synaptic damage, and neurodegeneration, with evidence implicating microglia in pathology, neuroprotection, and repair. As our understanding of microglial physiology and pathophysiology increases, we describe how we are moving toward potential therapeutic applications in MS, harnessing microglia to protect and regenerate the CNS.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1005&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"63368440f02b21838008c79a4284c50a56371a35\",\n",
      "        \"title\": \"Characterisation of tumour immune microenvironment remodelling following oncogene inhibition in preclinical studies using an optimised imaging mass cytometry workflow\",\n",
      "        \"abstract\": \"\\n Mouse models are critical in pre-clinical studies of cancer therapy, allowing dissection of mechanisms through chemical and genetic manipulations that are not feasible in the clinical setting. In studies of the tumour microenvironment (TME), novel highly multiplexed imaging methods can provide a rich source of information. However, the application of such technologies in mouse tissues is still in its infancy. Here we present a workflow for studying the TME using imaging mass cytometry with a panel of 27 antibodies on frozen mouse tissues. We optimised and validated image segmentation strategies and automated the process in a Nextflow-based pipeline (imcyto) that is scalable and portable, allowing for parallelised segmentation of large multi-image datasets. Incorporating user-specific plugins, imcyto can be flexibly tailored to a wide range of segmentation needs. With these methods we interrogated the dramatic remodelling of the TME induced by a KRAS G12C inhibitor in an immune competent mouse orthotopic lung cancer model, showcasing their potential as key discovery tools to enhance understanding of the interplay between tumour, stroma, and immune cells in the spatial context of the tissue.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1006&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50b00f5121a6acefb20e386897dcb662ee572eb5\",\n",
      "        \"title\": \"Single Cell Analysis of Blood Mononuclear Cells Stimulated Through Either LPS or Anti-CD3 and Anti-CD28\",\n",
      "        \"abstract\": \"Immune cell activation assays have been widely used for immune monitoring and for understanding disease mechanisms. However, these assays are typically limited in scope. A holistic study of circulating immune cell responses to different activators is lacking. Here we developed a cost-effective high-throughput multiplexed single-cell RNA-seq combined with epitope tagging (CITE-seq) to determine how classic activators of T cells (anti-CD3 coupled with anti-CD28) or monocytes (LPS) alter the cell composition and transcriptional profiles of peripheral blood mononuclear cells (PBMCs) from healthy human donors. Anti-CD3/CD28 treatment activated all classes of lymphocytes either directly (T cells) or indirectly (B and NK cells) but reduced monocyte numbers. Activated T and NK cells expressed senescence and effector molecules, whereas activated B cells transcriptionally resembled autoimmune disease- or age-associated B cells (e.g., CD11c, T-bet). In contrast, LPS specifically targeted monocytes and induced two main states: early activation characterized by the expression of chemoattractants and a later pro-inflammatory state characterized by expression of effector molecules. These data provide a foundation for future immune activation studies with single cell technologies (https://czi-pbmc-cite-seq.jax.org/).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1007&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a4b066c7be81f1689d96e463550e370e4b8fa5bd\",\n",
      "        \"title\": \"G\\u00f6ttingen Minipigs as a Model to Evaluate Longevity, Functionality, and Memory of Immune Response Induced by Pertussis Vaccines\",\n",
      "        \"abstract\": \"Evaluation of the short-term and long-term immunological responses in a preclinical model that simulates the targeted age population with a relevant vaccination schedule is essential for human vaccine development. A G\\u00f6ttingen minipig model was assessed, using pertussis vaccines, to demonstrate that vaccine antigen-specific humoral and cellular responses, including IgG titers, functional antibodies, Th polarization and memory B cells can be assessed in a longitudinal study. A vaccination schedule of priming with a whole cell (DTwP) or an acellular (DTaP) pertussis vaccine was applied in neonatal and infant minipigs followed by boosting with a Tdap acellular vaccine. Single cell RNAsequencing was used to explore the long-term maintenance of immune memory cells and their functionality for the first time in this animal model. DTaP but not DTwP vaccination induced pertussis toxin (PT) neutralizing antibodies. The cellular immune response was also characterized by a distinct Th polarization, with a Th-2-biased response for DTaP and a Th-1/Th-17-biased response for DTwP. No difference in the maintenance of pertussis-specific memory B cells was observed in DTaP- or DTwP-primed animals 6 months post Tdap boost. However, an increase in pertussis-specific T cells was still observed in DTaP primed minipigs, together with up-regulation of genes involved in antigen presentation and interferon pathways. Overall, the minipig model reproduced the humoral and cellular immune responses induced in humans by DTwP vs. DTaP priming, followed by Tdap boosting. Our data suggest that the G\\u00f6ttingen minipig is an attractive preclinical model to predict the long-term immunogenicity of human vaccines against Bordetella pertussis and potentially also vaccines against other pathogens.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1008&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"faf442c1dba9d320f1b1f3be33e8634d907c29bc\",\n",
      "        \"title\": \"Continuous Polony Gels for Tissue Mapping with High Resolution and RNA Capture Efficiency\",\n",
      "        \"abstract\": \"Current technologies for acquiring spatial transcript information from tissue sections rely on either RNA probes or spatial barcodes. The former methods require a priori knowledge for probeset formulation; the latter have yet to achieve single cell resolution and/or transcript capture efficiencies approaching dissociative, single-cell methods. Here, we describe a novel spatial transcriptome assay called polony (or DNA cluster)-indexed library-sequencing (PIXEL-seq). It improves upon other spatial barcoding methods by employing \\u201ccontinuous\\u201d polony oligos arrayed across a customized gel surface. In terms of assay performance, PIXEL-seq attains \\u2264 1 \\u00b5m resolution and captures >1,000 unique molecular identifiers/10\\u00d710 \\u00b5m2. In other words, this global, naive platform achieves subcellular spatial transcriptome mapping while maintaining high transcript capture efficiencies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1009&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5bbc17e89096446a22c5e4cf752621403fb72412\",\n",
      "        \"title\": \"Advances and Opportunities of Single-Cell Transcriptomics for Plant Research.\",\n",
      "        \"abstract\": \"Single-cell approaches are quickly changing our view on biological systems by increasing the spatiotemporal resolution of our analyses to the level of the individual cell. The field of plant biology has fully embraced single-cell transcriptomics and is rapidly expanding the portfolio of available technologies and applications. In this review, we give an overview of the main advances in plant single-cell transcriptomics over the past few years and provide the reader with an accessible guideline covering all steps, from sample preparation to data analysis. We end by offering a glimpse of how these technologies will shape and accelerate plant-specific research in the near future. Expected final online publication date for the Annual Review of Plant Biology, Volume 72 is May 2021. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1010&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"08cc85d4fc152e80c1fe940878cea8f61da0d969\",\n",
      "        \"title\": \"Single cell transcriptomic landscape of diabetic foot ulcers\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1011&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6237d43931d1c965b96942eff45ab18f4d775ff0\",\n",
      "        \"title\": \"Identifying CNS-colonizing T cells as potential therapeutic targets to prevent progression of multiple sclerosis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1012&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"28bc53940773c7f5fea8aa6e3130a8e27aec6e8a\",\n",
      "        \"title\": \"Multiplexed Analysis of Retinal Gene Expression and Chromatin Accessibility using scRNA-Seq and scATAC-Seq.\",\n",
      "        \"abstract\": \"Powerful next generation sequencing techniques offer robust and comprehensive analysis to investigate how retinal gene regulatory networks function during development and in disease states. Single-cell RNA sequencing allows us to comprehensively profile gene expression changes observed in retinal development and disease at a cellular level, while single-cell ATAC-Seq allows analysis of chromatin accessibility and transcription factor binding to be profiled at similar resolution. Here the use of these techniques in the developing retina is described, and MULTI-Seq is demonstrated, where individual samples are labeled with a modified oligonucleotide-lipid complex, enabling researchers to both increase the scope of individual experiments and substantially reduce costs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1013&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7fea0beaa5a1884aeb86fa479e35e10a71cd9a3d\",\n",
      "        \"title\": \"Single-cell RNA sequencing to study vascular diversity and function\",\n",
      "        \"abstract\": \"Purpose of review Single-cell RNA sequencing (scRNA-seq) can capture the transcriptional profile of thousands of individual cells concurrently from complex tissues and with remarkable resolution. Either with the goal of seeking information about distinct cell subtypes or responses to a stimulus, the approach has provided robust information and promoted impressive advances in cardiovascular research. The goal of this review is to highlight strategies and approaches to leverage this technology and bypass potential caveats related to evaluation of the vascular cells. Recent findings As the most recent technological development, details associated with experimental strategies, analysis, and interpretation of scRNA-seq data are still being discussed and scrutinized by investigators across the vascular field. Compilation of this information is valuable for those using the technology but particularly important to those about to start utilizing scRNA-seq to seek transcriptome information of vascular cells. Summary As our field progresses to catalog transcriptomes from distinct vascular beds, it is undeniable that scRNA-seq technology is here to stay. Sharing approaches to improve the quality of cell dissociation procedures, analysis, and a consensus of best practices is critical as information from this powerful experimental platform continues to emerge.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1014&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"053474cf57cdfe169e8c81e76d0df5ca5334a1d5\",\n",
      "        \"title\": \"High-throughput and high-dimensional single-cell analysis of antigen-specific CD8+ T cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1015&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8e02807d9a06ebd1e4584513667763d6d0e20b4\",\n",
      "        \"title\": \"SARS-CoV-2 mutations in MHC-I-restricted epitopes evade CD8+ T cell responses\",\n",
      "        \"abstract\": \"SARS-CoV-2 mutations in MHC-I epitopes identified by deep viral sequencing evade CTL responses through decreased peptide-MHC-I binding. CD8+ T cell immunity to SARS-CoV-2 has been implicated in COVID-19 severity and virus control. Here, we identified nonsynonymous mutations in MHC-I-restricted CD8+ T cell epitopes after deep sequencing of 747 SARS-CoV-2 virus isolates. Mutant peptides exhibited diminished or abrogated MHC-I binding in a cell-free in vitro assay. Reduced MHC-I binding of mutant peptides was associated with decreased proliferation, IFN-\\u03b3 production and cytotoxic activity of CD8+ T cells isolated from HLA-matched COVID-19 patients. Single cell RNA sequencing of ex vivo expanded, tetramer-sorted CD8+ T cells from COVID-19 patients further revealed qualitative differences in the transcriptional response to mutant peptides. Our findings highlight the capacity of SARS-CoV-2 to subvert CD8+ T cell surveillance through point mutations in MHC-I-restricted viral epitopes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1016&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd4daac2f0b59fe5bbe4d63b43d1647b10043a30\",\n",
      "        \"title\": \"Neutralizing IFNL3 Autoantibodies in Severe COVID-19 Identified Using Molecular Indexing of Proteins by Self-Assembly\",\n",
      "        \"abstract\": \"Unbiased antibody profiling can identify the targets of an immune reaction. A number of likely pathogenic autoreactive antibodies have been associated with life-threatening SARS-CoV-2 infection; yet, many additional autoantibodies likely remain unknown. Here we present Molecular Indexing of Proteins by Self Assembly (MIPSA), a technique that produces ORFeome-scale libraries of proteins covalently coupled to uniquely identifying DNA barcodes for analysis by sequencing. We used MIPSA to profile circulating autoantibodies from 55 patients with severe COVID-19 against 11,076 DNA-barcoded proteins of the human ORFeome library. MIPSA identified previously known autoreactivities, and also detected undescribed neutralizing interferon lambda 3 (IFN-\\u03bb3) autoantibodies. At-risk individuals with anti-IFN-\\u03bb3 antibodies may benefit from interferon supplementation therapies, such as those currently undergoing clinical evaluation. One-Sentence Summary Molecular Indexing of Proteins by Self Assembly (MIPSA) identifies neutralizing IFNL3 autoantibodies in patients with severe COVID-19. Graphical Abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1017&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6c994c6dc9b6fd44d60a8ea5cf5c665d40f8c025\",\n",
      "        \"title\": \"Knowledge-based classification of fine-grained immune cell types in single-cell RNA-Seq data\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-Seq) is an emerging strategy for characterizing immune cell populations. Compared to flow or mass cytometry, scRNA-Seq could potentially identify cell types and activation states that lack precise cell surface markers. However, scRNA-Seq is currently limited due to the need to manually classify each immune cell from its transcriptional profile. While recently developed algorithms accurately annotate coarse cell types (e.g. T cells versus macrophages), making fine distinctions (e.g. CD8+ effector memory T cells) remains a difficult challenge. To address this, we developed a machine learning classifier called ImmClassifier that leverages a hierarchical ontology of cell type. We demonstrate that its predictions are highly concordant with flow-based markers from CITE-seq and outperforms other tools (+15% recall, +14% precision) in distinguishing fine-grained cell types with comparable performance on coarse ones. Thus, ImmClassifier can be used to explore more deeply the heterogeneity of the immune system in scRNA-Seq experiments.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1018&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3076014d24e660c0e5fc1f4363936ee843a02119\",\n",
      "        \"title\": \"Vaccine genetics of IGHV1-2 VRC01-class broadly neutralizing antibody precursor na\\u00efve human B cells\",\n",
      "        \"abstract\": \"A successful HIV vaccine eliciting broadly neutralizing antibodies (bnAbs) must overcome the hurdle of being able to activate naive precursor B cells encoding features within their germline B cell receptors (BCR) that allow recognition of broadly neutralizing epitopes. Knowledge of whether bnAb precursor B cells are circulating at sufficient frequencies within individuals in communities heavily impacted by HIV may be important. Using a germline-targeting eOD-GT8 immunogen and high-throughput droplet-based single-cell BCR sequencing, we demonstrate that large numbers of paired BCR sequences from multiple donors can be efficiently screened to elucidate precursor frequencies of rare, naive VRC01-class B cells. Further, we analyzed IGHV1-2 allelic usage among three different cohorts; we find that IGHV1-2 alleles traditionally thought to be incompatible with VRC01-class responses are relatively common in various human populations and that germline variation within IGHV1-2 associates with gene usage frequencies in the naive BCR repertoire.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1019&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"555cc72ff2083b3b4869726880689e554b321f4e\",\n",
      "        \"title\": \"Single\\u2010cell RNA sequencing in cancer research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1020&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"79c9d4fef3fe786e07c166ce15a51de4cd5957d7\",\n",
      "        \"title\": \"Dive into Single, Seek Out Multiple: Probing Cancer Metastases via Single-Cell Sequencing and Imaging Techniques\",\n",
      "        \"abstract\": \"Simple Summary Treating cancer metastasis is the biggest challenge in clinical practice. It is largely due to our limited understanding of the complex process, which involves not only the evolution and/or selection of heterogeneous primary tumor cells to metastatic tumors, but also interaction and/or adaption with various types of cells in different microenvironments, including temporally in the circulation system. These limitations are currently resolved by single-cell technologies. This review summarizes recent applications of single-cell technologies in metastatic studies, highlights the unique findings, and discusses the future directions. Abstract Metastasis is the cause of most cancer deaths and continues to be the biggest challenge in clinical practice and laboratory investigation. The challenge is largely due to the intrinsic heterogeneity of primary and metastatic tumor populations and the complex interactions among cancer cells and cells in the tumor microenvironment. Therefore, it is important to determine the genotype and phenotype of individual cells so that the metastasis-driving events can be precisely identified, understood, and targeted in future therapies. Single-cell sequencing techniques have allowed the direct comparison of the genomic and transcriptomic changes among different stages of metastatic samples. Single-cell imaging approaches have enabled the live visualization of the heterogeneous behaviors of malignant and non-malignant cells in the tumor microenvironment. By applying these technologies, we are achieving a spatiotemporal precision understanding of cancer metastases and clinical therapeutic translations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1021&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3597bebae860de49ffbcd14e192cee3da8d48f1b\",\n",
      "        \"title\": \"Multimodal pooled Perturb-CITE-seq screens in patient models define mechanisms of cancer immune evasion\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1022&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c3aacee9de5c9dd9416bdb2f0c5fc6912cec387b\",\n",
      "        \"title\": \"The landscape of cell-cell communication through single-cell transcriptomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1023&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"49fdc4bf2a81774bea22ea20aa44362a33449082\",\n",
      "        \"title\": \"Micro-scale technologies propel biology and medicine.\",\n",
      "        \"abstract\": \"Historically, technology has been central to new discoveries in biology and progress in medicine. Among various technologies, microtechnologies, in particular, have had a prominent role in the revolution experienced by the life sciences in the last few decades, which will surely continue in the years to come. In this Perspective, we illustrate how microtechnologies, with a focus on microfluidics, have evolved in trends/waves to tackle the boundary of knowledge in the life sciences. We provide illustrative examples of technology-enabled biological breakthroughs and their current and future use in clinics. Finally, we take a closer look at the translational process to understand why the incorporation of new micro-scale technologies in medicine has been comparatively slow so far.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1024&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"62e0f789b7e7d83c1ff94d7547c691c316de3889\",\n",
      "        \"title\": \"Single-Cell Proteomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1025&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c09e1cfb21baa9e5b9d914db5e69f846ec88f375\",\n",
      "        \"title\": \"Human airway mast cells proliferate and acquire distinct inflammation-driven phenotypes during type 2 inflammation\",\n",
      "        \"abstract\": \"Mast cells in type 2 airway inflammation exhibit heterogeneous inflammation-associated phenotypes and proliferate in situ. Mast cell spectrum Mast cells (MCs), key players in type 2 (T2) allergic airway diseases such as chronic rhinosinusitis with nasal polyposis (CRSwNP), have mainly been characterized into different subsets based on protease content. Dwyer et al. have used flow cytometry and single-cell RNA-sequencing (scRNA-seq) to carry out a detailed study of human MC hyperplasia in CRSwNP. Relative to control tissue from patients with CRS lacking polyps, mature epithelial MCs that express tryptase (MCT), subepithelial MCs that express tryptase and chymase (MCTC), and CD38highCD117high MCs with an intermediate phenotype between MCT and MCTC all expanded in polyps of patients with CRSwNP. These results suggest that local proliferation of CD38highCD117high MCs is linked to both mature MC subsets and contributes to T2 disease. Mast cells (MCs) play a pathobiologic role in type 2 (T2) allergic inflammatory diseases of the airway, including asthma and chronic rhinosinusitis with nasal polyposis (CRSwNP). Distinct MC subsets infiltrate the airway mucosa in T2 disease, including subepithelial MCs expressing the proteases tryptase and chymase (MCTC) and epithelial MCs expressing tryptase without chymase (MCT). However, mechanisms underlying MC expansion and the transcriptional programs underlying their heterogeneity are poorly understood. Here, we use flow cytometry and single-cell RNA-sequencing (scRNA-seq) to conduct a comprehensive analysis of human MC hyperplasia in CRSwNP, a T2 cytokine\\u2013mediated inflammatory disease. We link discrete cell surface phenotypes to the distinct transcriptomes of CRSwNP MCT and MCTC, which represent polarized ends of a transcriptional gradient of nasal polyp MCs. We find a subepithelial population of CD38highCD117high MCs that is markedly expanded during T2 inflammation. These CD38highCD117high MCs exhibit an intermediate phenotype relative to the expanded MCT and MCTC subsets. CD38highCD117high MCs are distinct from circulating MC progenitors and are enriched for proliferation, which is markedly increased in CRSwNP patients with aspirin-exacerbated respiratory disease, a severe disease subset characterized by increased MC burden and elevated MC activation. We observe that MCs expressing a polyp MCT\\u2013like effector program are also found within the lung during fibrotic diseases and asthma, and further identify marked differences between MCTC in nasal polyps and skin. These results indicate that MCs display distinct inflammation-associated effector programs and suggest that in situ MC proliferation is a major component of MC hyperplasia in human T2 inflammation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1026&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e941139b6e47367e4e9cf2c1a0b2a396e01c8df3\",\n",
      "        \"title\": \"Lymphoma Heterogeneity Unraveled by Single-Cell Transcriptomics\",\n",
      "        \"abstract\": \"High-definition transcriptomic studies through single-cell RNA sequencing (scRNA-Seq) have revealed the heterogeneity and functionality of the various microenvironments across numerous solid tumors. Those pioneer studies have highlighted different cellular signatures correlated with clinical response to immune checkpoint inhibitors. scRNA-Seq offers also a unique opportunity to unravel the intimate heterogeneity of the ecosystems across different lymphoma entities. In this review, we will first cover the basics and future developments of the technology, and we will discuss its input in the field of translational lymphoma research, from determination of cell-of-origin and functional diversity, to monitoring of anti-cancer targeted drugs response and toxicities, and how new improvements in both data collection and interpretation will further foster precision medicine in the upcoming years.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1027&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d5c437c56aaa22025a31c2b89c00c52330243810\",\n",
      "        \"title\": \"Integrated single-cell analysis unveils diverging immune features of COVID-19, influenza, and other community-acquired pneumonia\",\n",
      "        \"abstract\": \"The exact immunopathophysiology of community-acquired pneumonia (CAP) caused by SARS-CoV-2 (COVID-19) remains clouded by a general lack of relevant disease controls. The scarcity of single-cell investigations in the broader population of patients with CAP renders it difficult to distinguish immune features unique to COVID-19 from the common characteristics of a dysregulated host response to pneumonia. We performed integrated single-cell transcriptomic and proteomic analyses in peripheral blood mononuclear cells from a matched cohort of eight patients with COVID-19, eight patients with CAP caused by Influenza A or other pathogens, and four non-infectious control subjects. Using this balanced, multi-omics approach, we describe shared and diverging transcriptional and phenotypic patterns\\u2014including increased levels of type I interferon-stimulated natural killer cells in COVID-19, cytotoxic CD8 T EMRA cells in both COVID-19 and influenza, and distinctive monocyte compositions between all groups\\u2014and thereby expand our understanding of the peripheral immune response in different etiologies of pneumonia.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1028&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4190e0541b291cde93442ec526430350a347605c\",\n",
      "        \"title\": \"scSorter: assigning cells to known cell types according to marker genes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1029&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3f502b0f7e8852e7827e8c048ac11c78aec21e99\",\n",
      "        \"title\": \"Single\\u2010cell sequencing in translational cancer research and challenges to meet clinical diagnostic needs\",\n",
      "        \"abstract\": \"The ability to capture alterations in the genome or transcriptome by next\\u2010generation sequencing has provided critical insight into molecular changes and programs underlying cancer biology. With the rapid technological development in single\\u2010cell sequencing, it has become possible to study individual cells at the transcriptional, genetic, epigenetic, and protein level. Using single\\u2010cell analysis, an increased resolution of fundamental processes underlying cancer development is obtained, providing comprehensive insights otherwise lost by sequencing of entire (bulk) samples, in which molecular signatures of individual cells are averaged across the entire cell population. Here, we provide a concise overview on the application of single\\u2010cell analysis of different modalities within cancer research by highlighting key articles of their respective fields. We furthermore examine the potential of existing technologies to meet clinical diagnostic needs and discuss current challenges associated with this translation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1030&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6cb332ee659b7223f3158bc81493edebfef14ede\",\n",
      "        \"title\": \"MultiMAP: dimensionality reduction and integration of multimodal data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1031&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"53223efcb9e38ded34927a6c218855023b4dcc78\",\n",
      "        \"title\": \"Robust clustering and interpretation of scRNA-seq data using reference component analysis\",\n",
      "        \"abstract\": \"Motivation The transcriptomic diversity of the hundreds of cell types in the human body can be analysed in unprecedented detail using single cell (SC) technologies. Though clustering of cellular transcriptomes is the default technique for defining cell types and subtypes, single cell clustering can be strongly influenced by technical variation. In fact, the prevalent unsupervised clustering algorithms can cluster cells by technical, rather than biological, variation. Results Compared to de novo (unsupervised) clustering methods, we demonstrate using multiple benchmarks that supervised clustering, which uses reference transcriptomes as a guide, is robust to batch effects. To leverage the advantages of supervised clustering, we present RCA2, a new, scalable, and broadly applicable version of our RCA algorithm. RCA2 provides a user-friendly framework for supervised clustering and downstream analysis of large scRNA-seq data sets. RCA2 can be seamlessly incorporated into existing algorithmic pipelines. It incorporates various new reference panels for human and mouse, supports generation of custom panels and uses efficient graph-based clustering and sparse data structures to ensure scalability. We demonstrate the applicability of RCA2 on SC data from human bone marrow, healthy PBMCs and PBMCs from COVID-19 patients. Importantly, RCA2 facilitates cell-type-specific QC, which we show is essential for accurate clustering of SC data from heterogeneous tissues. In the era of cohort-scale SC analysis, supervised clustering methods such as RCA2 will facilitate unified analysis of diverse SC datasets. Availability RCA2 is implemented in R and is available at github.com/prabhakarlab/RCAv2\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1032&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fb388954e0f5daef6a1d8e2198c90ebd43584cd4\",\n",
      "        \"title\": \"A Single-Cell Atlas of Lymphocyte Adaptive Immune Repertoires and Transcriptomes Reveals Age-Related Differences in Convalescent COVID-19 Patients\",\n",
      "        \"abstract\": \"COVID-19 disease outcome is highly dependent on adaptive immunity from T and B lymphocytes, which play a critical role in the control, clearance and long-term protection against SARS-CoV-2. To date, there is limited knowledge on the composition of the T and B cell immune receptor repertoires [T cell receptors (TCRs) and B cell receptors (BCRs)] and transcriptomes in convalescent COVID-19 patients of different age groups. Here, we utilize single-cell sequencing (scSeq) of lymphocyte immune repertoires and transcriptomes to quantitatively profile the adaptive immune response in COVID-19 patients of varying age. We discovered highly expanded T and B cells in multiple patients, with the most expanded clonotypes coming from the effector CD8+ T cell population. Highly expanded CD8+ and CD4+ T cell clones show elevated markers of cytotoxicity (CD8: PRF1, GZMH, GNLY; CD4: GZMA), whereas clonally expanded B cells show markers of transition into the plasma cell state and activation across patients. By comparing young and old convalescent COVID-19 patients (mean ages = 31 and 66.8 years, respectively), we found that clonally expanded B cells in young patients were predominantly of the IgA isotype and their BCRs had incurred higher levels of somatic hypermutation than elderly patients. In conclusion, our scSeq analysis defines the adaptive immune repertoire and transcriptome in convalescent COVID-19 patients and shows important age-related differences implicated in immunity against SARS-CoV-2.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1033&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"da6e624c3b450bcf0399eecf0479f8336f6fa55a\",\n",
      "        \"title\": \"Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1034&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a4d9fc0ea515d09dbd0115604826162a5835efa6\",\n",
      "        \"title\": \"Multiomics uncovers developing immunological lineages in human\",\n",
      "        \"abstract\": \"The development of the human immune system during embryonic and fetal life has historically been difficult to research due to limited access to human tissue. Experimental animal models have been widely used to study development but cellular and molecular programmes may not be conserved across species. The advent of multiomic single\\u2010cell technologies and an increase in human developmental tissue biobank resources have facilitated single\\u2010cell multiomic studies focused on human immune development. A critical question in the near future is \\\"How do we best reconcile scientific findings across multiple omic modalities, developmental time, and organismic space?\\\" In this review, we discuss the application of single\\u2010cell multiomic technologies to unravel the major cellular lineages in the prenatal human immune system. We also identify key areas where the combined power of multiomics technologies can be leveraged to address specific immunological gaps in our current knowledge and explore new research horizons in human development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1035&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75e16de73d9aabad4678a0f2a7d3fe78031bdee9\",\n",
      "        \"title\": \"Determinants, mechanisms, and functional outcomes of myeloid cell diversity in cancer\",\n",
      "        \"abstract\": \"Most, if not all, aspects of carcinogenesis are influenced by the tumor microenvironment (TME), a complex architecture of cells, matrix components, soluble signals, and their dynamic interactions in the context of physical traits of the tissue. Expanding application of technologies for high\\u2010dimensional analyses with single\\u2010cell resolution has begun to decipher the contributions of the immune system to cancer progression and its implications for therapy. In this review, we will discuss the multifaceted roles of tumor\\u2010associated macrophages and neutrophils, focusing on factors that subvert tissue immune homeostasis and offer therapeutic opportunities for TME reprogramming. By performing a critical analysis of available datasets, we elaborate on diversification mechanisms and unifying principles of myeloid cell heterogeneity in human tumors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1036&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"898ab64b23405005abbfaddfdffc90676c8a541b\",\n",
      "        \"title\": \"Leveraging Single-Cell Approaches in Cancer Precision Medicine.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1037&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0dc9813adb63d98e1791c4bc10ae23f24b7c3d03\",\n",
      "        \"title\": \"Immuno-metabolic interfaces in cardiac disease and failure.\",\n",
      "        \"abstract\": \"The interplay between the cardiovascular system, metabolism, and inflammation plays a central role in the pathophysiology of a wide spectrum of cardiovascular diseases, including heart failure. Here, we provide an overview of the fundamental aspects of the interrelation between inflammation and metabolism, ranging from the role of metabolism in immune cell function to the processes how inflammation modulates systemic and cardiac metabolism. Furthermore, we discuss how disruption of this immuno-metabolic interface is involved in the development and progression of cardiovascular disease, with a special focus on heart failure. Finally, we present new technologies and therapeutic approaches that have recently emerged and hold promise for the future of cardiovascular medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1038&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"de0a0ba2a650f1c090fe19acccf4d4ddfe98d3d2\",\n",
      "        \"title\": \"Glutamatergic dysfunction precedes neuron loss in cerebral organoids with MAPT mutation\",\n",
      "        \"abstract\": \"Frontotemporal dementia (FTD) due to MAPT mutation causes pathological accumulation of tau and glutamatergic cortical neuronal death by unknown mechanisms. We used human induced pluripotent stem cell (iPSC)-derived cerebral organoids expressing tau-V337M and isogenic corrected controls to discover early alterations due to the mutation that precede neurodegeneration. At 2 months, mutant organoids show upregulated expression of MAPT, and glutamatergic signaling pathways and regulators including the RNA-binding protein ELAVL4. Over the following 4 months, mutant organoids accumulate splicing changes, disruption of autophagy function and build-up of tau and P-tau S396. By 6 months, tau-V337M organoids show specific loss of glutamatergic neurons of layers affected in patients. Mutant neurons are susceptible to glutamate toxicity which was rescued pharmacologically by treatment with the PIKFYVE kinase inhibitor apilimod. Our results demonstrate a sequence of events that precede cell death, revealing molecular pathways associated with glutamate signaling as potential targets for therapeutic intervention in FTD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1039&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"202f45b54d355cee00b236330c37a1134336c91b\",\n",
      "        \"title\": \"Characterisation of tumour microenvironment remodelling following oncogene inhibition in preclinical studies with imaging mass cytometry\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1040&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a00ace2e8f6512e185eaaf5c43f406e8baed3496\",\n",
      "        \"title\": \"Understanding cell\\u2010cell communication and signaling in the colorectal cancer microenvironment\",\n",
      "        \"abstract\": \"Abstract Carcinomas are complex heterocellular systems containing epithelial cancer cells, stromal fibroblasts, and multiple immune cell\\u2010types. Cell\\u2010cell communication between these tumor microenvironments (TME) and cells drives cancer progression and influences response to existing therapies. In order to provide better treatments for patients, we must understand how various cell\\u2010types collaborate within the TME to drive cancer and consider the multiple signals present between and within different cancer types. To investigate how tissues function, we need a model to measure both how signals are transferred between cells and how that information is processed within cells. The interplay of collaboration between different cell\\u2010types requires cell\\u2010cell communication. This article aims to review the current in vitro and in vivo mono\\u2010cellular and multi\\u2010cellular cultures models of colorectal cancer (CRC), and to explore how they can be used for single\\u2010cell multi\\u2010omics approaches for isolating multiple types of molecules from a single\\u2010cell required for cell\\u2010cell communication to distinguish cancer cells from normal cells. Integrating the existing single\\u2010cell signaling measurements and models, and through understanding the cell identity and how different cell types communicate, will help predict drug sensitivities in tumor cells and between\\u2010 and within\\u2010patients responses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1041&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ca4900494659398bbdc04aa73a8207efd60432e\",\n",
      "        \"title\": \"Ex uno, plures\\u2013From One Tissue to Many Cells: A Review of Single-Cell Transcriptomics in Cardiovascular Biology\",\n",
      "        \"abstract\": \"Recent technological advances have revolutionized the study of tissue biology and garnered a greater appreciation for tissue complexity. In order to understand cardiac development, heart tissue homeostasis, and the effects of stress and injury on the cardiovascular system, it is essential to characterize the heart at high cellular resolution. Single-cell profiling provides a more precise definition of tissue composition, cell differentiation trajectories, and intercellular communication, compared to classical bulk approaches. Here, we aim to review how recent single-cell multi-omic studies have changed our understanding of cell dynamics during cardiac development, and in the healthy and diseased adult myocardium.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1042&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ea5bd0920aeff659cdd25dd8fbf8f702b7cd8a71\",\n",
      "        \"title\": \"Stem cell quiescence: the challenging path to activation\",\n",
      "        \"abstract\": \"ABSTRACT Quiescence is a cellular state in which a cell remains out of the cell cycle but retains the capacity to divide. The unique ability of adult stem cells to maintain quiescence is crucial for life-long tissue homeostasis and regenerative capacity. Quiescence has long been viewed as an inactive state but recent studies have shown that it is in fact an actively regulated process and that adult stem cells are highly reactive to extrinsic stimuli. This has fuelled hopes of boosting the reactivation potential of adult stem cells to improve tissue function during ageing. In this Review, we provide a perspective of the quiescent state and discuss how quiescent adult stem cells transition into the cell cycle. We also discuss current challenges in the field, highlighting recent technical advances that could help overcome some of these challenges. Summary: This Review provides a perspective of the quiescent state of adult stem cells and discusses how recent technical advances could help overcome some of the challenges in the field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1043&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"20b01b4130346379ede82762d7d24a8720e5cbf6\",\n",
      "        \"title\": \"Atypical B cells are part of an alternative lineage of B cells that participates in responses to vaccination and infection in humans\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1044&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6beed6bb60e2036552215a3a886195a25bbb7d83\",\n",
      "        \"title\": \"Full Spectrum Flow Cytometry as a Powerful Technology for Cancer Immunotherapy Research\",\n",
      "        \"abstract\": \"The Nobel Prize-deserving concept of blocking inhibitory pathways in T cells, to unleash their anti-tumoral capacity, became one of the pillars of cancer treatment in the last decade and has resulted in durable clinical responses for multiple cancer types. Currently, two of the most important goals in cancer immunotherapy are to understand the mechanisms resulting in failure to checkpoint blockade and to identify predictive immunological biomarkers that correlate to treatment response, disease progression or adverse effects. The identification and validation of biomarkers for routine clinical use is not only critical to monitor disease or treatment progression, but also to personalize and develop new therapies. To achieve these goals, powerful research tools are needed. Flow cytometry stands as one of the most successful single-cell analytical tools used to characterize immune cell phenotypes to monitor solid tumors, hematological malignancies, minimal residual disease or metastatic progression. This technology has been fundamental in diagnosis, treatment and translational research in cancer clinical trials. Most recently, the need to evaluate simultaneously more features in each cell has pushed the field to implement more powerful adaptations beyond conventional flow cytometry, including Full Spectrum Flow Cytometry (FSFC). FSFC captures the full emission spectrum of fluorescent molecules using arrays of highly sensitive light detectors, and to date has enabled characterization of 40 parameters in a single sample. We will summarize the contributions of this technology to the advancement of research in immunotherapy studies and discuss best practices to obtain reliable, robust and reproducible FSFC results.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1045&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f25bbf3645bf575adaa79b80ec84872edd1956ac\",\n",
      "        \"title\": \"Seq-Scope: Submicrometer-resolution spatial transcriptomics for single cell and subcellular studies\",\n",
      "        \"abstract\": \"Spatial barcoding technologies have the potential to reveal histological details of transcriptomic profiles; however, they are currently limited by their low resolution. Here we report Seq-Scope, a spatial barcoding technology with a resolution almost comparable to an optical microscope. Seq-Scope is based on a solid-phase amplification of randomly barcoded single-molecule oligonucleotides using an Illumina sequencing-by-synthesis platform. The resulting clusters annotated with spatial coordinates are processed to expose RNA-capture moiety. These RNA-capturing barcoded clusters define the pixels of Seq-Scope that are approximately 0.5-1 \\u03bcm apart from each other. From tissue sections, Seq-Scope visualizes spatial transcriptome heterogeneity at multiple histological scales, including tissue zonation according to the portal-central (liver), crypt-surface (colon) and inflammation-fibrosis (injured liver) axes, cellular components including single cell types and subtypes, and subcellular architectures of nucleus, cytoplasm and mitochondria. Seq-scope is quick, straightforward and easy-to-implement, and makes spatial single cell analysis accessible to a wide group of biomedical researchers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1046&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e81fa8c45cf62b2c59c9462c44c29335ca712793\",\n",
      "        \"title\": \"Elucidating the cellular dynamics of the brain with single-cell RNA sequencing\",\n",
      "        \"abstract\": \"ABSTRACT Single-cell RNA-sequencing (scRNA-seq) has emerged in recent years as a breakthrough technology to understand RNA metabolism at cellular resolution. In addition to allowing new cell types and states to be identified, scRNA-seq can permit cell-type specific differential gene expression changes, pre-mRNA processing events, gene regulatory networks and single-cell developmental trajectories to be uncovered. More recently, a new wave of multi-omic adaptations and complementary spatial transcriptomics workflows have been developed that facilitate the collection of even more holistic information from individual cells. These developments have unprecedented potential to provide penetrating new insights into the basic neural cell dynamics and molecular mechanisms relevant to the nervous system in both health and disease. In this review we discuss this maturation of single-cell RNA-sequencing over the past decade, and review the different adaptations of the technology that can now be applied both at different scales and for different purposes. We conclude by highlighting how these methods have already led to many exciting discoveries across neuroscience that have furthered our cellular understanding of the neurological disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1047&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bb1f88e3ef724676927e1f50a32219a8828694b2\",\n",
      "        \"title\": \"Next Generation Imaging Techniques to Define Immune Topographies in Solid Tumors\",\n",
      "        \"abstract\": \"In recent years, cancer immunotherapy experienced remarkable developments and it is nowadays considered a promising therapeutic frontier against many types of cancer, especially hematological malignancies. However, in most types of solid tumors, immunotherapy efficacy is modest, partly because of the limited accessibility of lymphocytes to the tumor core. This immune exclusion is mediated by a variety of physical, functional and dynamic barriers, which play a role in shaping the immune infiltrate in the tumor microenvironment. At present there is no unified and integrated understanding about the role played by different postulated models of immune exclusion in human solid tumors. Systematically mapping immune landscapes or \\u201ctopographies\\u201d in cancers of different histology is of pivotal importance to characterize spatial and temporal distribution of lymphocytes in the tumor microenvironment, providing insights into mechanisms of immune exclusion. Spatially mapping immune cells also provides quantitative information, which could be informative in clinical settings, for example for the discovery of new biomarkers that could guide the design of patient-specific immunotherapies. In this review, we aim to summarize current standard and next generation approaches to define Cancer Immune Topographies based on published studies and propose future perspectives.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1048&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c5636c4b105298f14916840f2ba0052f3cd39b2f\",\n",
      "        \"title\": \"Building a high-quality Human Cell Atlas\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1049&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d4a56f73697886273d7422d89af8b7eaadabc924\",\n",
      "        \"title\": \"No detectable alloreactive transcriptional responses under standard sample preparation conditions during donor-multiplexed single-cell RNA sequencing of peripheral blood mononuclear cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1050&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5dcc6b0d51668115a3382468382e074d053801a\",\n",
      "        \"title\": \"Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1051&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"302197ba6fd0a08a87d4f560649b44f5109bb377\",\n",
      "        \"title\": \"Simultaneous single cell measurements of intranuclear proteins and gene expression\",\n",
      "        \"abstract\": \"Identifying gene regulatory targets of nuclear proteins in tissues remains a challenge. Here we describe intranuclear Cellular Indexing of Transcriptomes and Epitopes (inCITE-seq), a scalable method for measuring multiplexed intranuclear protein levels and the transcriptome in parallel in thousands of cells, enabling joint analysis of TF levels and gene expression in vivo. We apply inCITE-seq to characterize cell state-related changes upon pharmacological induction of neuronal activity in the mouse brain. Modeling gene expression as a linear combination of quantitative protein levels revealed the genome-wide effect of each TF and recovered known targets. Cell type-specific genes associated with each TF were co-expressed as distinct modules that each corresponded to positive or negative TF levels, showing that our approach can disentangle relative contributions of TFs to gene expression and add interpretability to gene networks. InCITE-seq can illuminate how combinations of nuclear proteins shape gene expression in native tissue contexts, with direct applications to solid or frozen tissues and clinical specimens.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1052&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"602cd2d54b0fcbff07bcbef0b19303632df04fe1\",\n",
      "        \"title\": \"Automated methods for cell type annotation on scRNA-seq data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1053&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ea9674393b5296921c2b9eb945a5cf23e770c0fc\",\n",
      "        \"title\": \"Gut CD4+ T cell phenotypes are a continuum molded by microbes, not by TH archetypes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1054&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"001ccd1e6bbada2b1c6b4d606d318367f05077e0\",\n",
      "        \"title\": \"Multimodal detection of protein isoforms and nucleic acids from mouse pre-implantation embryos\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1055&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a14dc32389cf7ca6719a379204e67e9cfd0af0b\",\n",
      "        \"title\": \"Connecting Timescales in Biology: Can Early Dynamical Measurements Predict Long-Term Outcomes?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1056&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a19b5e0d6070eba171626e3007496bb190ac395\",\n",
      "        \"title\": \"A generalization of t-SNE and UMAP to single-cell multimodal omics\",\n",
      "        \"abstract\": \"Emerging single-cell technologies profile multiple types of molecules within individual cells. A fundamental step in the analysis of the produced high-dimensional data is their visualization using dimensionality reduction techniques such as t-SNE and UMAP. We introduce j-SNE and j-UMAP as their natural generalizations to the joint visualization of multimodal omics data. Our approach automatically learns the relative contribution of each modality to a concise representation of cellular identity that promotes discriminative features but suppresses noise. On eight datasets, j-SNE and j-UMAP produce unified embeddings that better agree with known cell types and that harmonize RNA and protein velocity landscapes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1057&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7966c714c8bd4a0efc5107b937bac293793d32e9\",\n",
      "        \"title\": \"Spatially resolved transcriptomics and its applications in cancer.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1058&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b135e330cc1473c8c24fa63bb9a5b64f51993f9e\",\n",
      "        \"title\": \"Joint probabilistic modeling of single-cell multi-omic data with totalVI\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1059&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"134e52f1b3cb362acab8c0bbaed1642a339182b5\",\n",
      "        \"title\": \"Molecular underpinnings of ssDNA specificity by Rep HUH-endonucleases and implications for HUH-tag multiplexing and engineering\",\n",
      "        \"abstract\": \"Abstract Replication initiator proteins (Reps) from the HUH-endonuclease superfamily process specific single-stranded DNA (ssDNA) sequences to initiate rolling circle/hairpin replication in viruses, such as crop ravaging geminiviruses and human disease causing parvoviruses. In biotechnology contexts, Reps are the basis for HUH-tag bioconjugation and a critical adeno-associated virus genome integration tool. We solved the first co-crystal structures of Reps complexed to ssDNA, revealing a key motif for conferring sequence specificity and for anchoring a bent DNA architecture. In combination, we developed a deep sequencing cleavage assay, termed HUH-seq, to interrogate subtleties in Rep specificity and demonstrate how differences can be exploited for multiplexed HUH-tagging. Together, our insights allowed engineering of only four amino acids in a Rep chimera to predictably alter sequence specificity. These results have important implications for modulating viral infections, developing Rep-based genomic integration tools, and enabling massively parallel HUH-tag barcoding and bioconjugation applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1060&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff971387ef4197ff5d120816a04560446fb30ec3\",\n",
      "        \"title\": \"Finding needles in a haystack: dissecting tumor heterogeneity with single-cell transcriptomic and chromatin accessibility profiling.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1061&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"00946f80e86b0463f99000292508c80bfd43da97\",\n",
      "        \"title\": \"scJoint: transfer learning for data integration of single-cell RNA-seq and ATAC-seq\",\n",
      "        \"abstract\": \"Single-cell multi-omics data continues to grow at an unprecedented pace, and while integrating different modalities holds the promise for better characterization of cell identities, it remains a significant computational challenge. In particular, extreme sparsity is a hallmark in many modalities such as scATAC-seq data and often limits their power in cell type identification. Here we present scJoint, a transfer learning method to integrate heterogeneous collections of scRNA-seq and scATAC-seq data. scJoint uses a neural network to simultaneously train labeled and unlabeled data and embed cells from both modalities in a common lower dimensional space, enabling label transfer and joint visualization in an integrative framework. We demonstrate scJoint consistently provides meaningful joint visualizations and achieves significantly higher label transfer accuracy than existing methods using a complex cell atlas data and a biologically varying multi-modal data. This suggests scJoint is effective in overcoming the heterogeneity in different modalities towards a more comprehensive understanding of cellular phenotypes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1062&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"25cf125985ec114e3610e509908486f82c10a40e\",\n",
      "        \"title\": \"Single\\u2010cell transcriptomics analysis of mild cognitive impairment in World Trade Center disaster responders\",\n",
      "        \"abstract\": \"Recent research has found that World Trade Center (WTC) responders in their mid\\u201050s have an elevated prevalence of mild cognitive impairment (MCI) that is associated with neural degeneration and subcortical thinning. This article extends our understanding of the molecular complexity of MCI through gene expression profiling of blood.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1063&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e908a0f1daef825e23982c427ccff3b04863cd76\",\n",
      "        \"title\": \"Phased differentiation of \\u03b3\\u03b4 T and T CD8 tumor-infiltrating lymphocytes revealed by single-cell transcriptomics of human cancers\",\n",
      "        \"abstract\": \"ABSTRACT \\u03b3\\u03b4 T lymphocytes diverge from conventional T CD8 lymphocytes for ontogeny, homing, and antigen specificity, but whether their differentiation in tumors also deviates was unknown. Using innovative analyses of our original and ~150 published single-cell RNA sequencing datasets validated by phenotyping of human tumors and murine models, here we present the first high-resolution view of human \\u03b3\\u03b4 T cell differentiation in cancer. While \\u03b3\\u03b4 T lymphocytes prominently encompass TCRV\\u03b39 cells more differentiated than T CD8 in healthy donor\\u2019s blood, a different scenario is unveiled in tumors. Solid tumors and lymphomas are infiltrated by a majority of TCRV\\u03b3non9 \\u03b3\\u03b4 T cells which are quantitatively correlated and remarkably aligned with T CD8 for differentiation, exhaustion, gene expression profile, and response to immune checkpoint therapy. This cancer-wide association is critical for developing cancer immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1064&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"152cf18e5890918832f1db46e9f2b3a812e15d4c\",\n",
      "        \"title\": \"Single cell analysis of blood mononuclear cells stimulated through CD3 and CD28 shows collateral activation of B and NK cells and demise of monocytes\",\n",
      "        \"abstract\": \"Immune cell activation assays have been widely used for immune monitoring and for understanding disease mechanisms. However, these assays are typically limited in scope. A holistic study of circulating immune cell responses to different activators is lacking. Here we developed a cost-effective high-throughput multiplexed single-cell RNA-seq combined with epitope tagging (CITE-seq) to determine how classic activators of T cells (anti-CD3 coupled with anti-CD28) or monocytes (LPS) alter the cell composition and transcriptional profiles of peripheral blood mononuclear cells (PBMCs) from healthy human donors. Anti-CD3/CD28 treatment activated all classes of lymphocytes either directly (T cells) or indirectly (B and NK cells) but reduced monocyte numbers. Activated T and NK cells expressed senescence and effector molecules, whereas activated B cells transcriptionally resembled autoimmune disease- or age-associated B cells (e.g., CD11c, T-bet). In contrast, LPS specifically targeted monocytes and induced two main states: early activation characterized by the expression of chemoattractants and a later pro-inflammatory state characterized by expression of effector molecules. These data provide a foundation for future immune activation studies with single cell technologies (https://czi-pbmc-cite-seq.jax.org/). Graphical abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1065&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"69d22ba9a2fe16f0bc6db2a71ba37cd28a4cb78b\",\n",
      "        \"title\": \"SPaRTAN, a computational framework for linking cell-surface receptors to transcriptional regulators\",\n",
      "        \"abstract\": \"The identity and functions of specialized cell types are dependent on the complex interplay between signaling and transcriptional networks. We present SPaRTAN (Single-cell Proteomic and RNA based Transcription factor Activity Network), a computational method to link cell-surface receptors to transcription factors (TFs) by exploiting cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) datasets with cis-regulatory information. SPaRTAN is applied to immune cell types in the blood to predict the coupling of signaling receptors with cell context-specific TFs. The predictions are validated by prior knowledge and flow cytometry analyses. SPaRTAN is then used to predict the signaling coupled TF states of tumor infiltrating CD8+ T cells in malignant peritoneal and pleural mesotheliomas. SPaRTAN greatly enhances the utility of CITE-seq datasets to uncover TF and cell-surface receptor relationships in diverse cellular states. Significance Recently single-cell technologies such as CITE-seq have been developed that enable simultaneous quantitative analysis of cell-surface receptor expression with transcriptional states. To date, these datasets have not been used to systematically develop cell-context-specific maps of the interface between signaling and transcriptional regulators orchestrating cellular identity and function. We developed a computational framework, SPaRTAN (Single-cell Proteomic and RNA based Transcription factor Activity Network) that integrates single-cell proteomic and transcriptomic data based on CITE-seq with cis-regulatory information. We applied our method to publicly available (PBMCs) and new CITE-seq datasets (mesothelioma). Our predicted TF activity and cell-surface receptor relationships are validated by prior knowledge as well as experimental testing. SPaRTAN reveals many previously unidentified surface-receptor associated TFs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1066&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a8240735e89391eddc0d4bcd847f1a09552f3b4f\",\n",
      "        \"title\": \"Ultra\\u2010high sensitivity mass spectrometry quantifies single\\u2010cell proteome changes upon perturbation\",\n",
      "        \"abstract\": \"Single-cell technologies are revolutionizing biology but are today mainly limited to imaging and deep sequencing1\\u20133. However, proteins are the main drivers of cellular function and in-depth characterization of individual cells by mass spectrometry (MS)-based proteomics would thus be highly valuable and complementary4,5. Chemical labeling-based single-cell approaches introduce hundreds of cells into the MS, but direct analysis of single cells has not yet reached the necessary sensitivity, robustness and quantitative accuracy to answer biological questions6,7. Here, we develop a robust workflow combining miniaturized sample preparation, very low flow-rate chromatography and a novel trapped ion mobility mass spectrometer, resulting in a more than ten-fold improved sensitivity. We accurately and robustly quantify proteomes and their changes in single, FACS-isolated cells. Arresting cells at defined stages of the cell cycle by drug treatment retrieves expected key regulators such as CDK2NA, the E2 ubiquitin ligase UBE2S, DNA topoisomerases TOP2A/B and the chromatin regulator HMGA1. Furthermore, it highlights potential novel ones and allows cell phase prediction. Comparing the variability in more than 430 single-cell proteomes to transcriptome data revealed a stable core proteome despite perturbation, while the transcriptome appears volatile. This emphasizes substantial regulation of translation and sets the stage for its elucidation at the single cell level. Our technology can readily be applied to ultra-high sensitivity analyses of tissue material8, posttranslational modifications and small molecule studies to gain unprecedented insights into cellular heterogeneity in health and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1067&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"503ab4ca37d4454121b9535c01bc099cefe7e1fd\",\n",
      "        \"title\": \"Tumor-infiltrating dendritic cell states are conserved across solid human cancers\",\n",
      "        \"abstract\": \"Gerhard et al. reveal the conservation of human tumor-infiltrating dendritic cell states via meta-analysis of scRNA-seq datasets. They further explore dendritic cell state conservation in mice and the origins, behavior, and functions of these cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1068&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d3b093d209107f5579f107a091f7dc50f02842ea\",\n",
      "        \"title\": \"Micro size exclusion chromatography combined with a multiplex protein profiling method for extracellular vesicle protein detection from small sample volumes\",\n",
      "        \"abstract\": \"\\n Extracellular vesicles (EVs) are small nanometer-sized membrane sacs secreted into biological fluids by all cells. EVs encapsulate proteins, RNAs and metabolites from its origin cell and play important roles in intercellular communication events. Over the past decade, EVs have become a new emerging source for cancer diagnostics. One of the challenges in the study of EVs and there utility as diagnostic biomarkers is the amount of EVs needed for traditional protein analysis methods. Here, we present a new immuno-PCR method that takes advantage of commercially available TotalSeq\\u2122 antibodies containing DNA conjugated oligos to identify immobilized protein analysts using real-time qPCR. Using this method, we demonstrate that multiple EV surface proteins can be profiled simultaneously with high sensitivity and specificity. This approach was also successfully applied to similar protocol using live cell and serum samples. We further described the development of a micro-size exclusion chromatography method, where we were able to detect EV surface proteins with as little as 10 \\u03bcL of human serum when combined with immuno-PCR. Overall, these results show that the immuno-PCR method results in rapid detection of multiple EV markers from small sample volumes in a single tube.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1069&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1b6f097f60ea964ad65817a5855c841fca3fdd46\",\n",
      "        \"title\": \"Harnessing Single-Cell RNA Sequencing to Better Understand How Diseased Cells Behave the Way They Do in Cardiovascular Disease.\",\n",
      "        \"abstract\": \"The transition of healthy arteries and cardiac valves into dense, cell-rich, calcified, and fibrotic tissues is driven by a complex interplay of both cellular and molecular mechanisms. Specific cell types in these cardiovascular tissues become activated following the exposure to systemic stimuli including circulating lipoproteins or inflammatory mediators. This activation induces multiple cascades of events where changes in cell phenotypes and activation of certain receptors may trigger multiple pathways and specific alterations to the transcriptome. Modifications to the transcriptome and proteome can give rise to pathological cell phenotypes and trigger mechanisms that exacerbate inflammation, proliferation, calcification, and recruitment of resident or distant cells. Accumulating evidence suggests that each cell type involved in vascular and valvular diseases is heterogeneous. Single-cell RNA sequencing is a transforming medical research tool that enables the profiling of the unique fingerprints at single-cell levels. Its applications have allowed the construction of cell atlases including the mammalian heart and tissue vasculature and the discovery of new cell types implicated in cardiovascular disease. Recent advances in single-cell RNA sequencing have facilitated the identification of novel resident cell populations that become activated during disease and has allowed tracing the transition of healthy cells into pathological phenotypes. Furthermore, single-cell RNA sequencing has permitted the characterization of heterogeneous cell subpopulations with unique genetic profiles in healthy and pathological cardiovascular tissues. In this review, we highlight the latest groundbreaking research that has improved our understanding of the pathological mechanisms of atherosclerosis and future directions for calcific aortic valve disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1070&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58f212571cd5f71efb86ec66fc378bef728358bf\",\n",
      "        \"title\": \"PD-1 blockade restores helper activity of tumor-infiltrating, exhausted PD-1hiCD39+ CD4 T cells\",\n",
      "        \"abstract\": \"Tumor antigen\\u2013specific CD4 T cells accumulate at tumor sites, evoking their involvement in antitumor effector functions in situ. Contrary to CD8 cytotoxic T lymphocyte exhaustion, that of CD4 T cells remains poorly appreciated. Here, using phenotypic, transcriptomic, and functional approaches, we characterized CD4 T cell exhaustion in patients with head and neck, cervical, and ovarian cancer. We identified a CD4 tumor-infiltrating lymphocyte (TIL) population, defined by high PD-1 and CD39 expression, which contained high proportions of cytokine-producing cells, although the quantity of cytokines produced by these cells was low, evoking an exhausted state. Terminal exhaustion of CD4 TILs was instated regardless of TIM-3 expression, suggesting divergence with CD8 T cell exhaustion. scRNA-Seq and further phenotypic analyses uncovered similarities with the CD8 T cell exhaustion program. In particular, PD-1hiCD39+ CD4 TILs expressed the exhaustion transcription factor TOX and the chemokine CXCL13 and were tumor antigen specific. In vitro, PD-1 blockade enhanced CD4 TIL activation, as evidenced by increased CD154 expression and cytokine secretion, leading to improved dendritic cell maturation and consequently higher tumor-specific CD8 T cell proliferation. Our data identify exhausted CD4 TILs as players in responsiveness to immune checkpoint blockade.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1071&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"155a2e40d6475a4f05d3c54d2dd79db393440068\",\n",
      "        \"title\": \"Experimental and computational technologies to dissect the kidney at the single-cell level.\",\n",
      "        \"abstract\": \"The field of single-cell technologies, in particular single-cell genomics with transcriptomics and epigenomics, and most recently single-cell proteomics, is rapidly growing and holds promise to advance our understanding of organ homoeostasis and disease, and facilitate the identification of novel therapeutic targets and biomarkers. This review offers an introduction to these technologies. In addition, as the size and complexity of the data require sophisticated computational methods for analysis and interpretation, we will also provide an overview of these methods and summarize the single-cell literature specifically pertaining to the kidney.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1072&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1cc337ca86b52d3ecd3ffc19c23dde8d2fd05e56\",\n",
      "        \"title\": \"Sfaira accelerates data and model reuse in single cell genomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1073&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1480d800afbf3900837fb8e71c13b5f5550d62ab\",\n",
      "        \"title\": \"Recent advances in single\\u2010cell multimodal analysis to study immune cells\",\n",
      "        \"abstract\": \"Recent advances in single\\u2010cell technologies have enabled the profiling of the genome, epigenome, transcriptome and proteome, along with temporal and spatial information of individual cells. These technologies have provided unique opportunities to understand mechanisms underpinning the immune system, such as characterizations of the molecular cell state, how the cell state evolves along its lineage and the impact of spatial location on cell state. In this review, we discuss how these mechanisms have been studied through recent advances in single\\u2010cell multimodal technologies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1074&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4428116808ccf9c003bbac145e667bbf33151bef\",\n",
      "        \"title\": \"Assigning immunoglobulin class from single-cell transcriptomes in IgA1-secreting versus membrane subpopulations\",\n",
      "        \"abstract\": \"IgA nephropathy (IgAN) is an autoimmune disease characterized by renal glomerular immunodeposits enriched for galactose-deficient IgA1 (Gd-IgA1; autoantigen) with the corresponding IgG autoantibodies. Despite the known contribution of Gd-IgA1 to IgAN, little is known concerning IgA1-secreting subpopulations responsible for autoantigen production. The goal of this study is to identify IgA1-secreting and membrane subpopulations from single-cell transcriptomic analysis. We developed a novel single-cell analytics workflow to discern cells expressing IgA1 secreted isoform or membrane-bound isoform. Multiple approaches were compared to assess immunoglobulin-isotype identity in single cells, and multiple immunoglobulin heavy-chain genes expressed in the same cells were found. To better identify specific immunoglobulin heavy-chain transcripts, we merged a software platform called Alteryx with the existing single-cell R toolkit program Seurat. This process allowed for improved calls on IgA1-secreting subpopulations based on secreting versus membrane splice-variant expression levels.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1075&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"014df485e731b8af31ef0609aee387c3be7886a1\",\n",
      "        \"title\": \"Unbiased integration of single cell multi-omics data\",\n",
      "        \"abstract\": \"Acquiring accurate single-cell multiomics profiles often requires performing unbiased in silico integration of data matrices generated by different single-cell technologies from the same biological sample. However, both the rows and the columns can represent different entities in different data matrices, making such integration a computational challenge that has only been solved approximately by existing approaches. Here, we present bindSC, a single-cell data integration tool that realizes simultaneous alignment of the rows and the columns between data matrices without making approximations. Using datasets produced by multiomics technologies as gold standard, we show that bindSC generates accurate multimodal co-embeddings that are substantially more accurate than those generated by existing approaches. Particularly, bindSC effectively integrated single cell RNA sequencing (scRNA-seq) and single cell chromatin accessibility sequencing (scATAC-seq) data towards discovering key regulatory elements in cancer cell-lines and mouse cells. It achieved accurate integration of both common and rare cell types (<0.25% abundance) in a novel mouse retina cell atlas generated using the 10x Genomics Multiome ATAC+RNA kit. Further, it achieves unbiased integration of scRNA-seq and 10x Visium spatial transcriptomics data derived from mouse brain cortex samples. Lastly, it demonstrated efficacy in delineating immune cell types via integrating single-cell RNA and protein data. Thus, bindSC, available at https://github.com/KChen-lab/bindSC, can be applied in a broad variety of context to accelerate discovery of complex cellular and biological identities and associated molecular underpinnings in diseases and developing organisms.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1076&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"66d79bafce806798b72b4333854b8761073221cc\",\n",
      "        \"title\": \"Single-cell RNA sequencing reveals distinct tumor microenvironmental patterns in lung adenocarcinoma\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1077&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8c09654fd58d9a0a03a774215331e1508a450666\",\n",
      "        \"title\": \"Innate Lymphoid Cells and Celiac Disease: Current Perspective\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1078&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d68e32fc34dd20541b7c985197c979783a9862c7\",\n",
      "        \"title\": \"Longitudinal CITE-Seq profiling of chronic lymphocytic leukemia during ibrutinib treatment: evolution of leukemic and immune cells at relapse\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1079&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7a7a6092361b897b98977639a40a9eb192c3e205\",\n",
      "        \"title\": \"Quantification of spatial tumor heterogeneity in immunohistochemistry staining images\",\n",
      "        \"abstract\": \"MOTIVATION\\nQuantitative Immunofluorescence (QIF) is often used for immunohistochemistry (IHC) quantification of proteins that serve as cancer biomarkers. Advanced image analysis systems for pathology allow capturing expression levels in each individual cell or subcellular compartment. However, only the Mean Signal Intensity (MSI) within the cancer tissue region of interest is usually considered as biomarker completely ignoring the issue of tumor heterogeneity.\\n\\n\\nRESULTS\\nWe propose using IHC image-derived information on the spatial distribution of cellular signal intensity (CSI) of protein expression within the cancer cell population to quantify both mean expression level and tumor heterogeneity of CSI levels. We view CSI levels as marks in a marked point process of cancer cells in the tissue and define spatial indices based on conditional mean and conditional variance of the marked point process. The proposed methodology provides objective metrics of cell-to-cell heterogeneity in protein expressions that allow discriminating between different patterns of heterogeneity. The prognostic utility of new spatial indices is investigated and compared to the standard MSI biomarkers using the protein expressions in tissue microarrays (TMAs) incorporating tumor tissues from1000+ breast cancer patients.\\n\\n\\nSUPPLEMENTARY INFORMATION\\nSupplementary data are available at Bioinformatics online.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1080&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e23efcb846af9367a7a43f1af7d79d0194ab6dba\",\n",
      "        \"title\": \"Applying high-dimensional single-cell technologies to the analysis of cancer immunotherapy\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1081&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4d6d5aeea1e841fc00653fd8769f3966afdd5a83\",\n",
      "        \"title\": \"Heterogeneity of T Cells in Atherosclerosis Defined by Single-Cell RNA-Sequencing and Cytometry by Time of Flight\",\n",
      "        \"abstract\": \"The infiltration and accumulation of pro- and anti-inflammatory leukocytes within the intimal layer of the arterial wall is a hallmark of developing and progressing atherosclerosis. While traditionally perceived as macrophage- and foam cell-dominated disease, it is now established that atherosclerosis is a partial autoimmune disease that involves the recognition of peptides from ApoB (apolipoprotein B), the core protein of LDL (low-density lipoprotein) cholesterol particles, by CD4+ T-helper cells and autoantibodies against LDL and ApoB. Autoimmunity in the atherosclerotic plaque has long been understood as a pathogenic T-helper type-1 driven response with proinflammatory cytokine secretion. Recent developments in high-parametric cell immunophenotyping by mass cytometry, single-cell RNA-sequencing, and in tools exploring antigen-specificity have established the existence of several unforeseen layers of T-cell diversity with mixed TH1 and T regulatory cells transcriptional programs and unpredicted fates. These findings suggest that pathogenic ApoB-reactive T cells evolve from atheroprotective and immunosuppressive CD4+ T regulatory cells that lose their protective properties over time. Here, we discuss T-cell heterogeneity in atherosclerosis with a focus on plasticity, antigen-specificity, exhaustion, maturation, tissue residency, and its potential use in clinical prediction.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1082&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f711f2ca7b8c83309917b223c060116110aa8ba9\",\n",
      "        \"title\": \"Single Cell Sequencing Reveals Glial Specific Responses to Tissue Processing & Enzymatic Dissociation in Mice and Humans\",\n",
      "        \"abstract\": \"A key aspect of nearly all single cell experiments is the necessity to dissociate intact tissues into single cell suspensions for processing. While many protocols have been optimized for optimal cell yield, they have often overlooked the effects that dissociation can have on ex vivo gene expression changes during this process. Microglia, the brain\\u2019s resident macrophages, are a highly dynamic population that are extremely sensitive to their microenvironment and have been shown to dramatically alter their transcriptome upon stimulation. We demonstrate that use of enzymatic dissociation methods on mouse central nervous system (CNS) tissue induces an aberrant gene expression signature in microglia that can significantly confound downstream analysis. To minimize this issue, we developed a flexible protocol, that can be used with existing enzymatic protocols for fresh tissue, to eliminate artifactual gene expression while allowing for increased cell type diversity and yield. We demonstrate efficacy of this protocol in analysis of diverse CNS cell types and sorted myeloid populations while using enzymatic dissociation. Generation of new and reanalysis of previously published human brain single nucleus RNAseq (snRNA-seq) datasets reveal that a similar signature is also present in post-mortem tissue. Through novel snRNA-seq analysis of acutely-resected neurosurgical tissue we demonstrate that this signature can be induced in human tissue due to technical differences in sample processing. These results provide key insight into the potential confounds of enzymatic digestion and provide a solution to allow for enzymatic digestion for scRNA-seq while avoiding ex vivo transcriptional artifacts. Analysis of human tissue reveals potential for artifacts in current and future snRNA-seq datasets that will require deeper analysis and careful consideration to separate true biology from artifacts related to post-mortem processes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1083&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bce315179c94d5f70a0ea2e0c1b312d1f50f3f21\",\n",
      "        \"title\": \"Global and Individualized Community Detection in Inhomogeneous Multilayer Networks\",\n",
      "        \"abstract\": \"In network applications, it has become increasingly common to obtain datasets in the form of multiple networks observed on the same set of subjects, where each network is obtained in a related but different experiment condition or application scenario. Such datasets can be modeled by multilayer networks where each layer is a separate network itself while different layers are associated and share some common information. The present paper studies community detection in a stylized yet informative inhomogeneous multilayer network model. In our model, layers are generated by different stochastic block models, the community structures of which are (random) perturbations of a common global structure while the connecting probabilities in different layers are not related. Focusing on the symmetric two block case, we establish minimax rates for both \\\\emph{global estimation} of the common structure and \\\\emph{individualized estimation} of layer-wise community structures. Both minimax rates have sharp exponents. In addition, we provide an efficient algorithm that is simultaneously asymptotic minimax optimal for both estimation tasks under mild conditions. The optimal rates depend on the \\\\emph{parity} of the number of most informative layers, a phenomenon that is caused by inhomogeneity across layers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1084&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0b26f7d06842f30bfd513a2a9735cdb89b912094\",\n",
      "        \"title\": \"Single-cell intracellular epitope and transcript detection revealing signal transduction dynamics\",\n",
      "        \"abstract\": \"Current high-throughput single-cell multi-omics methods cannot concurrently map changes in (phospho)protein levels and the associated gene expression profiles. We present QuRIE-seq (Quantification of RNA and Intracellular Epitopes by sequencing) and use multi-factor omics analysis (MOFA+) to map signal transduction over multiple timescales. We demonstrate that QuRIE-seq can trace the activation of the B-cell receptor pathway at the minute and hour time-scale and provide insight into the mechanism of action of an inhibitory drug, Ibrutinib.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1085&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3defe73dacf01c212d155cefa9e412e3c647173e\",\n",
      "        \"title\": \"Single-Cell Manifold Preserving Feature Selection (SCMER)\",\n",
      "        \"abstract\": \"A key challenge in studying organisms and diseases is to detect rare molecular programs and rare cell populations (RCPs) that drive development, differentiation, and transformation. Molecular features such as genes and proteins defining RCPs are often unknown and difficult to detect from unenriched single-cell data, using conventional dimensionality reduction and clustering-based approaches. Here, we propose a novel unsupervised approach, named SCMER, which performs UMAP style dimensionality reduction via selecting a compact set of molecular features with definitive meanings. We applied SCMER in the context of hematopoiesis, lymphogenesis, tumorigenesis, and drug resistance and response. We found that SCMER can identify non-redundant features that sensitively delineate both common cell lineages and rare cellular states ignored by current approaches. SCMER can be widely used for discovering novel molecular features in a high dimensional dataset, designing targeted, cost-effective assays for clinical applications, and facilitating multi-modality integration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1086&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3e6f421a7791ff970f1a76133176076f22e057a5\",\n",
      "        \"title\": \"Deciphering Organoids: High-Dimensional Analysis of Biomimetic Cultures.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1087&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4e758791ca405efbfbb6186eba2894e01e5e0bb\",\n",
      "        \"title\": \"To NMD or Not To NMD: Nonsense-Mediated mRNA Decay in Cancer and Other Genetic Diseases.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1088&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"476aab7de92e4ed1d273fbd12ac2f6a4bef3fbf2\",\n",
      "        \"title\": \"Analysis of Antigen\\u2010Specific Human Memory B Cell Populations Based on In Vitro Polyclonal Stimulation\",\n",
      "        \"abstract\": \"Antigen\\u2010specific memory B cell (MBC) populations mediate the rapid, strong, and high\\u2010affinity secondary antibody responses that play a key role in combating infection and generating protective responses to vaccination. Recently, cell staining with fluorochrome\\u2010labeled antigens together with sequencing methods such as Drop\\u2010seq and CITE\\u2010seq have provided information on the specificity, phenotype, and transcriptome of single MBCs. However, characterization of MBCs at the level of antigen\\u2010reactive populations remains an important tool for assessing an individual's B cell immunity and responses to antigen exposure. This is readily performed using a long\\u2010established method based on in vitro polyclonal stimulation of MBCs to induce division and differentiation into antibody\\u2010secreting cells (ASCs). Post\\u2010stimulation antigen\\u2010specific measurement of the MBC\\u2010derived ASCs (or the secreted antibodies) indicates the size of precursor MBC populations. Additional information about the character of antigen\\u2010reactive MBC populations is provided by analysis of MBC\\u2010derived antibodies of particular specificities for binding avidity and functionality. This article outlines a simple and reliable strategy for efficient in vitro MBC stimulation and use of the ELISpot assay as a post\\u2010stimulation readout to determine the size of antigen\\u2010specific MBC populations. Other applications of the in vitro stimulation technique for MBC analysis are discussed. The following protocols are included. \\u00a9 2020 Wiley Periodicals LLC\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1089&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5e33dfdcef5c320ad744cde4ea4e6ed908593aef\",\n",
      "        \"title\": \"Reliable and accurate diagnostics from highly multiplexed sequencing assays\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1090&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd57f80ffe5784374a73354a19a2b88a43e336db\",\n",
      "        \"title\": \"New perspectives of the cardiac cellular landscape: mapping cellular mediators of cardiac fibrosis using single-cell transcriptomics.\",\n",
      "        \"abstract\": \"Single-cell transcriptomics enables inference of context-dependent phenotypes of individual cells and determination of cellular diversity of complex tissues. Cardiac fibrosis is a leading factor in the development of heart failure and a major cause of morbidity and mortality worldwide with no effective treatment. Single-cell RNA-sequencing (scRNA-seq) offers a promising new platform to identify new cellular and molecular protagonists that may drive cardiac fibrosis and development of heart failure. This review will summarize the application scRNA-seq for understanding cardiac fibrosis and development of heart failure. We will also discuss some key considerations in interpreting scRNA-seq data and some of its limitations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1091&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5beddf50ed7baa1e30cb37fbb0e4e1fae600a8d4\",\n",
      "        \"title\": \"Development of a sequencing system for spatial decoding of DNA barcode molecules at single-molecule resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1092&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abe0718f303495a9660123041a8586e84cc4fd43\",\n",
      "        \"title\": \"SITC cancer immunotherapy resource document: a compass in the land of biomarker discovery\",\n",
      "        \"abstract\": \"Since the publication of the Society for Immunotherapy of Cancer\\u2019s (SITC) original cancer immunotherapy biomarkers resource document, there have been remarkable breakthroughs in cancer immunotherapy, in particular the development and approval of immune checkpoint inhibitors, engineered cellular therapies, and tumor vaccines to unleash antitumor immune activity. The most notable feature of these breakthroughs is the achievement of durable clinical responses in some patients, enabling long-term survival. These durable responses have been noted in tumor types that were not previously considered immunotherapy-sensitive, suggesting that all patients with cancer may have the potential to benefit from immunotherapy. However, a persistent challenge in the field is the fact that only a minority of patients respond to immunotherapy, especially those therapies that rely on endogenous immune activation such as checkpoint inhibitors and vaccination due to the complex and heterogeneous immune escape mechanisms which can develop in each patient. Therefore, the development of robust biomarkers for each immunotherapy strategy, enabling rational patient selection and the design of precise combination therapies, is key for the continued success and improvement of immunotherapy. In this document, we summarize and update established biomarkers, guidelines, and regulatory considerations for clinical immune biomarker development, discuss well-known and novel technologies for biomarker discovery and validation, and provide tools and resources that can be used by the biomarker research community to facilitate the continued development of immuno-oncology and aid in the goal of durable responses in all patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1093&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2de30bc7293535d04f4733bd264a89342b30fa6c\",\n",
      "        \"title\": \"Single\\u2010Cell Immune Profiling in Coronary Artery Disease: The Role of State\\u2010of\\u2010the\\u2010Art Immunophenotyping With Mass Cytometry in the Diagnosis of Atherosclerosis\",\n",
      "        \"abstract\": \"Abstract Coronary artery disease remains the leading cause of death globally and is a major burden to every health system in the world. There have been significant improvements in risk modification, treatments, and mortality; however, our ability to detect asymptomatic disease for early intervention remains limited. Recent discoveries regarding the inflammatory nature of atherosclerosis have prompted investigation into new methods of diagnosis and treatment of coronary artery disease. This article reviews some of the highlights of the important developments in cardioimmunology and summarizes the clinical evidence linking the immune system and atherosclerosis. It provides an overview of the major serological biomarkers that have been associated with atherosclerosis, noting the limitations of these markers attributable to low specificity, and then contrasts these serological markers with the circulating immune cell subtypes that have been found to be altered in coronary artery disease. This review then outlines the technique of mass cytometry and its ability to provide high\\u2010dimensional single\\u2010cell data and explores how this high\\u2010resolution quantification of specific immune cell subpopulations may assist in the diagnosis of early atherosclerosis in combination with other complimentary techniques such as single\\u2010cell RNA sequencing. We propose that this improved specificity has the potential to transform the detection of coronary artery disease in its early phases, facilitating targeted preventative approaches in the precision medicine era.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1094&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"35397b03d0265b8809c2143227515e204620a9eb\",\n",
      "        \"title\": \"MRBLES 2.0: High-throughput generation of chemically functionalized spectrally and magnetically encoded hydrogel beads using a simple single-layer microfluidic device\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1095&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0d00f5fdabe2606cbaec00612901d71f8ffdef9f\",\n",
      "        \"title\": \"Applications of Single-Cell Omics to Dissect Tumor Microenvironment\",\n",
      "        \"abstract\": \"The recent technical and computational advances in single-cell sequencing technologies have significantly broaden our toolkit to study tumor microenvironment (TME) directly from human specimens. The TME is the complex and dynamic ecosystem composed of multiple cell types, including tumor cells, immune cells, stromal cells, endothelial cells, and other non-cellular components such as the extracellular matrix and secreted signaling molecules. The great success on immune checkpoint blockade therapy has highlighted the importance of TME on anti-tumor immunity and has made it a prime target for further immunotherapy strategies. Applications of single-cell transcriptomics on studying TME has yielded unprecedented resolution of the cellular and molecular complexity of the TME, accelerating our understanding of the heterogeneity, plasticity, and complex cross-interaction between different cell types within the TME. In this review, we discuss the recent advances by single-cell sequencing on understanding the diversity of TME and its functional impact on tumor progression and immunotherapy response driven by single-cell sequencing. We primarily focus on the major immune cell types infiltrated in the human TME, including T cells, dendritic cells, and macrophages. We further discuss the limitations of the existing methodologies and the prospects on future studies utilizing single-cell multi-omics technologies. Since immune cells undergo continuous activation and differentiation within the TME in response to various environmental cues, we highlight the importance of integrating multimodal datasets to enable retrospective lineage tracing and epigenetic profiling of the tumor infiltrating immune cells. These novel technologies enable better characterization of the developmental lineages and differentiation states that are critical for the understanding of the underlying mechanisms driving the functional diversity of immune cells within the TME. We envision that with the continued accumulation of single-cell omics datasets, single-cell sequencing will become an indispensable aspect of the immune-oncology experimental toolkit. It will continue to drive the scientific innovations in precision immunotherapy and will be ultimately adopted by routine clinical practice in the foreseeable future.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1096&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c56c3ef87c2395505424299fc45d432706b9caf3\",\n",
      "        \"title\": \"Phenotyping of Adaptive Immune Responses in Inflammatory Diseases\",\n",
      "        \"abstract\": \"Immunophenotyping on the molecular and cellular level is a central aspect for characterization of patients with inflammatory diseases, both to better understand disease etiopathogenesis and based on this to develop diagnostic and prognostic biomarkers which allow patient stratification and tailor-made treatment strategies. Technology-driven developments have considerably expanded the range of analysis tools. Especially the analysis of adaptive immune responses, often regarded as central though mostly poorly characterized disease drivers, is a major focus of personalized medicine. The identification of the disease-relevant antigens and characterization of corresponding antigen-specific lymphocytes in individual patients benefits significantly from recent developments in cytometry by sequencing and proteomics. The aim of this workshop was to identify the important developments for state-of-the-art immunophenotyping for clinical application and precision medicine. We focused here on recent key developments in analysis of antigen-specific lymphocytes, sequencing, and proteomics approaches, their relevance in precision medicine and the discussion of the major challenges and opportunities for the future.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1097&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b43a0ea340a0451f67366a8d2480b0808740ce22\",\n",
      "        \"title\": \"Milo: differential abundance testing on single-cell data using k-NN graphs\",\n",
      "        \"abstract\": \"Single-cell omic protocols applied to disease, development or mechanistic studies can reveal the emergence of aberrant cell states or changes in differentiation. These perturbations can manifest as a shift in the abundance of cells associated with a biological condition. Current computational workflows for comparative analyses typically use discrete clusters as input when testing for differential abundance between experimental conditions. However, clusters are not always an optimal representation of the biological manifold on which cells lie, especially in the context of continuous differentiation trajectories. To overcome these barriers to discovery, we present Milo, a flexible and scalable statistical framework that performs differential abundance testing by assigning cells to partially overlapping neighbourhoods on a k-nearest neighbour graph. Our method samples and refines neighbourhoods across the graph and leverages the flexibility of generalized linear models, making it applicable to a wide range of experimental settings. Using simulations, we show that Milo is both robust and sensitive, and can reveal subtle but important cell state perturbations that are obscured by discretizing cells into clusters. We illustrate the power of Milo by identifying the perturbed differentiation during ageing of a lineage-biased thymic epithelial precursor state and by uncovering extensive perturbation to multiple lineages in human cirrhotic liver. Milo is provided as an open-source R software package with documentation and tutorials at https://github.com/MarioniLab/miloR.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1098&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e190ce4c720171700dca5e61425d9600c733c63e\",\n",
      "        \"title\": \"Maximizing statistical power to detect clinically associated cell states with scPOST\",\n",
      "        \"abstract\": \"As advances in single-cell technologies enable the unbiased assay of thousands of cells simultaneously, human disease studies are able to identify clinically associated cell states using case-control study designs. These studies require precious clinical samples and costly technologies; therefore, it is critical to employ study design principles that maximize power to detect cell state frequency shifts between conditions, such as disease versus healthy. Here, we present single-cell Power Simulation Tool (scPOST), a method that enables users to estimate power under different study designs. To approximate the specific experimental and clinical scenarios being investigated, scPOST takes prototype (public or pilot) single-cell data as input and generates large numbers of single-cell datasets in silico. We use scPOST to perform power analyses on three independent single-cell datasets that span diverse experimental conditions: a batch-corrected 21-sample rheumatoid arthritis dataset (5,265 cells) from synovial tissue, a 259-sample tuberculosis progression dataset (496,517 memory T cells) from peripheral blood mononuclear cells (PBMCs), and a 30-sample ulcerative colitis dataset (235,229 cells) from intestinal biopsies. Over thousands of simulations, we consistently observe that power to detect frequency shifts in cell states is maximized by larger numbers of independent clinical samples, reduced batch effects, and smaller variation in a cell state\\u2019s frequency across samples.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1099&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"42ca6d6a9b8c7a93d230e08aa42a10f8c9aec86e\",\n",
      "        \"title\": \"Single cell profiling of COVID-19 patients: an international data resource from multiple tissues\",\n",
      "        \"abstract\": \"In late 2019 and through 2020, the COVID-19 pandemic swept the world, presenting both scientific and medical challenges associated with understanding and treating a previously unknown disease. To help address the need for great understanding of COVID-19, the scientific community mobilized and banded together rapidly to characterize SARS-CoV-2 infection, pathogenesis and its distinct disease trajectories. The urgency of COVID-19 provided a pressing use-case for leveraging relatively new tools, technologies, and nascent collaborative networks. Single-cell biology is one such example that has emerged over the last decade as a powerful approach that provides unprecedented resolution to the cellular and molecular underpinnings of biological processes. Early foundational work within the single-cell community, including the Human Cell Atlas, utilized published and unpublished data to characterize the putative target cells of SARS-CoV-2 sampled from diverse organs based on expression of the viral receptor ACE2 and associated entry factors TMPRSS2 and CTSL (Muus et al., 2020; Sungnak et al., 2020; Ziegler et al., 2020). This initial characterization of reference data provided an important foundation for framing infection and pathology in the airway as well as other organs. However, initial community analysis was limited to samples derived from uninfected donors and other previously-sampled disease indications. This report provides an overview of a single-cell data resource derived from samples from COVID-19 patients along with initial observations and guidance on data reuse and exploration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1100&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d2dce7d206bba41403e14c5c6d2f9dd0955bedd\",\n",
      "        \"title\": \"Efficient and precise single-cell reference atlas mapping with Symphony\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1101&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2e3059236c1797aa272730c1249ecf910083ebaf\",\n",
      "        \"title\": \"PCQC: Selecting optimal principal components for identifying clusters with highly imbalanced class sizes in single-cell RNA-seq data\",\n",
      "        \"abstract\": \"Summary Prior to identifying clusters in single cell gene expression experiments, selecting the top principal components is a critical step for filtering out noise in the data set. Identifying these top principal components typically focuses on the total variance explained, and principal components that explain small clusters from rare populations will not necessarily capture a large percentage of variance in the data. We present a computationally efficient alternative for identifying the optimal principal components based on the tails of the distribution of variance explained for each observation. We then evaluate the efficacy of our approach in three different single cell RNA-sequencing data sets and find that our method matches, or outperforms, other selection criteria that are typically employed in the literature. Availability and implementation pcqc is written in Python and available at github.com/RoussosLab/pcqc\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1102&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"163907ba6a09c343adbce6ec742ebbd57134bae7\",\n",
      "        \"title\": \"IBEX: A versatile multiplex optical imaging approach for deep phenotyping and spatial analysis of cells in complex tissues\",\n",
      "        \"abstract\": \"Significance Single-cell flow cytometry and genomic methods are rapidly increasing our knowledge of the diversity of cell types in metazoan tissues. However, suitably robust methods for placing these cells in a spatial context that reveals how their localization and putative interactions contribute to tissue physiology and pathology are still lacking. Here we provide a readily accessible pipeline (IBEX) for highly multiplex immunofluorescent imaging that enables a fine-grained analysis of cells in their tissue context. Additionally, we describe extensions of the IBEX workflow to handle hard-to-image tissue preparations and a method to facilitate direct integration of the imaging data with flow cytometry and sequencing technologies. The diverse composition of mammalian tissues poses challenges for understanding the cell\\u2013cell interactions required for organ homeostasis and how spatial relationships are perturbed during disease. Existing methods such as single-cell genomics, lacking a spatial context, and traditional immunofluorescence, capturing only two to six molecular features, cannot resolve these issues. Imaging technologies have been developed to address these problems, but each possesses limitations that constrain widespread use. Here we report a method that overcomes major impediments to highly multiplex tissue imaging. \\u201cIterative bleaching extends multiplexity\\u201d (IBEX) uses an iterative staining and chemical bleaching method to enable high-resolution imaging of >65 parameters in the same tissue section without physical degradation. IBEX can be employed with various types of conventional microscopes and permits use of both commercially available and user-generated antibodies in an \\u201copen\\u201d system to allow easy adjustment of staining panels based on ongoing marker discovery efforts. We show how IBEX can also be used with amplified staining methods for imaging strongly fixed tissues with limited epitope retention and with oligonucleotide-based staining, allowing potential cross-referencing between flow cytometry, cellular indexing of transcriptomes and epitopes by sequencing, and IBEX analysis of the same tissue. To facilitate data processing, we provide an open-source platform for automated registration of iterative images. IBEX thus represents a technology that can be rapidly integrated into most current laboratory workflows to achieve high-content imaging to reveal the complex cellular landscape of diverse organs and tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1103&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"41f4b52b4dd5f41bbd793c600f389e3c7457ec90\",\n",
      "        \"title\": \"From ArrayExpress to BioStudies\",\n",
      "        \"abstract\": \"Abstract ArrayExpress (https://www.ebi.ac.uk/arrayexpress) is an archive of functional genomics data at EMBL-EBI, established in 2002, initially as an archive for publication-related microarray data and was later extended to accept sequencing-based data. Over the last decade an increasing share of biological experiments involve multiple technologies assaying different biological modalities, such as epigenetics, and RNA and protein expression, and thus the BioStudies database (https://www.ebi.ac.uk/biostudies) was established to deal with such multimodal data. Its central concept is a study, which typically is associated with a publication. BioStudies stores metadata describing the study, provides links to the relevant databases, such as European Nucleotide Archive (ENA), as well as hosts the types of data for which specialized databases do not exist. With BioStudies now fully functional, we are able to further harmonize the archival data infrastructure at EMBL-EBI, and ArrayExpress is being migrated to BioStudies. In future, all functional genomics data will be archived at BioStudies. The process will be seamless for the users, who will continue to submit data using the online tool Annotare and will be able to query and download data largely in the same manner as before. Nevertheless, some technical aspects, particularly programmatic access, will change. This update guides the users through these changes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1104&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9ad5a662f4bd8df08c3dd778071456125d7fe025\",\n",
      "        \"title\": \"Looking at the Developing Lung in Single-cell Resolution.\",\n",
      "        \"abstract\": \"Lung development is a complicated and delicate process, facilitated by spatially and temporarily coordinated crosstalk of up to 40 cell types. Developmental origin and heterogeneity of lung cell lineages in context of lung development have been a focus of research efforts for decades. Bulk RNA and protein measurements, RNA and protein labelling, and lineage tracing techniques have been traditionally employed. However, the complex and heterogeneous nature of lung tissue presents a particular challenge when identifying subtle changes in gene expression in individual cell types. Rapidly developing single-cell RNA sequencing (scRNA-seq) techniques allow for unbiased and robust assessment of complex cellular dynamics during biological processes in unprecedented ways. Discovered a decade ago, scRNA-seq has been applied in respiratory research to understand lung cellular composition and to identify novel cell types. Still, very few studies to date have addressed the single-cell transcriptome in healthy or aberrantly developing lung. In this mini-review, we discuss principal discoveries with scRNA-seq in the field of prenatal and postnatal lung development. In addition, we examine challenges and expectations, and propose future steps associated with the use of scRNA-seq to study developmental lung diseases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1105&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ec4743a75d27cec3c5e1487222797e26c068f3b9\",\n",
      "        \"title\": \"Comparative analysis of antibody- and lipid-based multiplexing methods for single-cell RNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1106&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7a79754f919262420994a277dfa55909eb251f3d\",\n",
      "        \"title\": \"SeqStain using fluorescent-DNA conjugated antibodies allows efficient, multiplexed, spatialomic profiling of human and murine tissues\",\n",
      "        \"abstract\": \"Spatial organization of molecules and cells in complex tissue microenvironments provides essential cues during healthy growth and in disease. Novel techniques are needed for elucidation of their spatial relationships and architecture. Although a few multiplex immunofluorescence based techniques have been developed for visualization of the spatial relationships of various molecules in cells and tissues, there remains a significant need for newer methods that are rapid, easy to adapt and are gentle during the cyclic steps of fluorescence staining and de-staining. Here, we describe a novel, multiplex immunofluorescence imaging method, termed SeqStain, that uses fluorescent-DNA labelled antibodies for immunofluorescence staining of cells and tissues, and nuclease treatment for de-staining that allows selective enzymatic removal of the fluorescent signal. SeqStain can be used with primary antibodies, secondary antibodies and antibody fragments, such as Fabs, to efficiently analyse complex cells and tissues in multiple rounds of staining and de-staining. Additionally, incorporation of specific endonuclease restriction sites in antibody labels allows for selective removal of fluorescent signals, while retaining other signals that can serve as marks for subsequent analyses. The application of SeqStain on human kidney tissue provided spatialomic profile of the organization of >25 markers in the kidney, highlighting it as a versatile, easy to use and gentle new technique for spatialomic analyses of complex microenvironments.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1107&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"be2d5d43e42201f324e50ae7d999c9c8951f6abe\",\n",
      "        \"title\": \"Gut CD4+ T cell phenotypes are a continuum molded by microbes, not by TH archetypes.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1108&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"23321d0ff2530925a67f1340e98b9f6e449eb6d7\",\n",
      "        \"title\": \"Characterisation of CD4+\\u2009T-cell subtypes using single cell RNA sequencing and the impact of cell number and sequencing depth\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1109&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b61e7f58f0c18144e8b3936d4542e15e39f03b6b\",\n",
      "        \"title\": \"Multi-Modal Profiling of Human Fetal Liver-Derived Hematopoietic Stem Cells Reveals the Molecular Signature of Engraftment Potential\",\n",
      "        \"abstract\": \"The human hematopoietic stem cell (HSC) harbors remarkable regenerative potential that can be harnessed therapeutically. During early development, HSCs in the fetal liver (FL) undergo active expansion while simultaneously retaining robust engraftment capacity, yet the underlying molecular program responsible for their efficient engraftment remains unclear. We profiled 26,407 FL cells at both transcriptional and protein levels including over 7,000 highly enriched and functional FL HSCs to establish a detailed molecular signature of engraftment potential. Integration of transcript and linked cell surface marker expression revealed a generalizable signature defining functional FL HSCs and allowed for the stratification of enrichment strategies with high translational potential. This comprehensive, multi-modal profiling of engraftment capacity connects a critical biological function at a key developmental timepoint with its underlying molecular drivers, serving as a useful resource for the field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1110&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c91f8ae00958df9f1fba3cab3963a7dac34bfa7a\",\n",
      "        \"title\": \"A primer on single-cell genomics in myeloid biology\",\n",
      "        \"abstract\": \"Purpose of review Understanding the fast-moving field of single-cell technologies, as applied to myeloid biology, requires an appreciation of basic molecular, informatics, and biological concepts. Here, we highlight both key and recent articles to illustrate basic concepts for those new to molecular single-cell analyses in myeloid hematology. Recent findings Recent studies apply single-cell omics to discover novel cell populations, construct relationships between cell populations, reconfigure the organization of hematopoiesis, and study hematopoietic lineage tree and fate choices. Accompanying development of technologies, new informatic tools have emerged, providing exciting new insights. Summary Hematopoietic stem and progenitor cells are regulated by complex intrinsic and extrinsic factors to produce blood cell types. In this review, we discuss recent advances in single-cell omics to profile these cells, methods to infer cell type identify, and trajectories from molecular omics data to ultimately derive new insights into hematopoietic stem and progenitor cell biology. We further discuss future applications of these technologies to understand hematopoietic cell interactions, function, and development. The goal is to offer a comprehensive overview of current single-cell technologies and their impact on our understanding of myeloid cell development for those new to single-cell analyses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1111&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"eadb79ea4b6c2a27e060dcd71adedc284ea3f01e\",\n",
      "        \"title\": \"Digital microfluidic isolation of single cells for -Omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1112&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7e0b12557d5e362657c010d6e4650b796db27d48\",\n",
      "        \"title\": \"Censcyt: censored covariates in differential abundance analysis in cytometry\",\n",
      "        \"abstract\": \"Innovations in single cell technologies have lead to a flurry of datasets and computational tools to process and interpret them, including analyses of cell composition changes and transition in cell states. The diffcyt workflow for differential discovery in cytometry data consist of several steps, including preprocessing, cell population identification and differential testing for an association with a binary or continuous covariate. However, the commonly measured quantity of survival time in clinical studies often results in a censored covariate where classical differential testing is inapplicable. To overcome this limitation, multiple methods to directly include censored covariates in differential abundance analysis were examined with the use of simulation studies and a case study. Results show that multiple imputation based methods offer on-par performance with the Cox proportional hazards model in terms of sensitivity and error control, while offering flexibility to account for covariates. The tested methods are implemented in the R package censcyt as an extension of diffcyt and are available at https://bioconductor.org/packages/censcyt. Methods for the direct inclusion of a censored variable as a predictor in GLMMs are a valid alternative to classical survival analysis methods, such as the Cox proportional hazard model, while allowing for more flexibility in the differential analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1113&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d89f109485c5d3d3934467223af668f72f4bdda\",\n",
      "        \"title\": \"BABEL enables cross-modality translation between multiomic profiles at single-cell resolution\",\n",
      "        \"abstract\": \"Significance Simultaneous measurement of the DNA, RNA, and proteins of single cells can lead to important new insights but is experimentally challenging. This work introduces a deep learning algorithm that flexibly translates between chromatin, RNA, and protein profiles in single cells. This makes it possible to computationally synthesize matched multiomic measurements when only one modality is experimentally available. This algorithm complements experimental advances to efficiently achieve single-cell multiomic insight. Simultaneous profiling of multiomic modalities within a single cell is a grand challenge for single-cell biology. While there have been impressive technical innovations demonstrating feasibility\\u2014for example, generating paired measurements of single-cell transcriptome (single-cell RNA sequencing [scRNA-seq]) and chromatin accessibility (single-cell assay for transposase-accessible chromatin using sequencing [scATAC-seq])\\u2014widespread application of joint profiling is challenging due to its experimental complexity, noise, and cost. Here, we introduce BABEL, a deep learning method that translates between the transcriptome and chromatin profiles of a single cell. Leveraging an interoperable neural network model, BABEL can predict single-cell expression directly from a cell\\u2019s scATAC-seq and vice versa after training on relevant data. This makes it possible to computationally synthesize paired multiomic measurements when only one modality is experimentally available. Across several paired single-cell ATAC and gene expression datasets in human and mouse, we validate that BABEL accurately translates between these modalities for individual cells. BABEL also generalizes well to cell types within new biological contexts not seen during training. Starting from scATAC-seq of patient-derived basal cell carcinoma (BCC), BABEL generated single-cell expression that enabled fine-grained classification of complex cell states, despite having never seen BCC data. These predictions are comparable to analyses of experimental BCC scRNA-seq data for diverse cell types related to BABEL\\u2019s training data. We further show that BABEL can incorporate additional single-cell data modalities, such as protein epitope profiling, thus enabling translation across chromatin, RNA, and protein. BABEL offers a powerful approach for data exploration and hypothesis generation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1114&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"51dd774ef5692678861b3a5f004494f7fec2da92\",\n",
      "        \"title\": \"SECANT: a biology-guided semi-supervised method for clustering, classification, and annotation of single-cell multi-omics\",\n",
      "        \"abstract\": \"The recent advance of single cell sequencing (scRNA-seq) technology such as Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) allows researchers to quantify cell surface protein abundance and RNA expression simultaneously at single cell resolution. Although CITE-seq and other similar technologies have quickly gained enormous popularity, novel methods for analyzing this new type of single cell multi-omics data are still in urgent need. A limited number of available tools utilize data-driven approach, which may undermine the biological importance of surface protein data. In this study, we developed SECANT, a biology-guided SEmi-supervised method for Clustering, classification, and ANnoTation of single-cell multi-omics. SECANT can be used to analyze CITE-seq data, or jointly analyze CITE-seq and scRNA-seq data. The novelties of SECANT include 1) using confident cell type labels identified from surface protein data as guidance for cell clustering, 2) providing general annotation of confident cell types for each cell cluster, 3) fully utilizing cells with uncertain or missing cell type labels to increase performance, and 4) accurate prediction of confident cell types identified from surface protein data for scRNA-seq data. Besides, as a model-based approach, SECANT can quantify the uncertainty of the results, and our framework can be easily extended to handle other types of multi-omics data. We successfully demonstrated the validity and advantages of SECANT via simulation studies and analysis of public and in-house real datasets. We believe this new method will greatly help researchers characterize novel cell types and make new biological discoveries using single cell multi-omics data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1115&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8c2e57eb19446b9d2fcc20b56a92cbf59f528f3d\",\n",
      "        \"title\": \"High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1116&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ca31a40f800d8b226cd51bea33211dc0db524b72\",\n",
      "        \"title\": \"Microglia Diversity in Health and Multiple Sclerosis\",\n",
      "        \"abstract\": \"Multiple Sclerosis (MS) is a neurodegenerative disease characterized by multiple focal lesions, ongoing demyelination and, for most people, a lack of remyelination. MS lesions are enriched with monocyte-derived macrophages and brain-resident microglia that, together, are likely responsible for much of the immune-mediated neurotoxicity. However, microglia and macrophage also have documented neuroprotective and regenerative roles, suggesting a potential diversity in their functions. Linked with microglial functional diversity, they take on diverse phenotypes developmentally, regionally and across disease conditions. Advances in technologies such as single-cell RNA sequencing and mass cytometry of immune cells has led to dramatic developments in understanding the phenotypic changes of microglia and macrophages. This review highlights the origins of microglia, their heterogeneity throughout normal ageing and their contribution to pathology and repair, with a specific focus on autoimmunity and MS. As phenotype dictates function, the emerging heterogeneity of microglia and macrophage populations in MS offers new insights into the potential immune mechanisms that result in inflammation and regeneration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1117&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a3b1c61807fc5ba0cd6089453d47ad5b4830ee9b\",\n",
      "        \"title\": \"ILoReg: a tool for high-resolution cell population identification from single-cell RNA-seq data\",\n",
      "        \"abstract\": \"Abstract Motivation Single-cell RNA-seq allows researchers to identify cell populations based on unsupervised clustering of the transcriptome. However, subpopulations can have only subtle transcriptomic differences and the high dimensionality of the data makes their identification challenging. Results We introduce ILoReg, an R package implementing a new cell population identification method that improves identification of cell populations with subtle differences through a probabilistic feature extraction step that is applied before clustering and visualization. The feature extraction is performed using a novel machine learning algorithm, called iterative clustering projection (ICP), that uses logistic regression and clustering similarity comparison to iteratively cluster data. Remarkably, ICP also manages to integrate feature selection with the clustering through L1-regularization, enabling the identification of genes that are differentially expressed between cell populations. By combining solutions of multiple ICP runs into a single consensus solution, ILoReg creates a representation that enables investigating cell populations with a high resolution. In particular, we show that the visualization of ILoReg allows segregation of immune and pancreatic cell populations in a more pronounced manner compared with current state-of-the-art methods. Availability and implementation ILoReg is available as an R package at https://bioconductor.org/packages/ILoReg. Supplementary information Supplementary data are available at Bioinformatics online.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1118&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5121eca9c1c2a50fa9f0952dd66511be639622e2\",\n",
      "        \"title\": \"Single-cell Multi-omics reveal heterogeneity and metastasis potential in different liver cancer cell lines\",\n",
      "        \"abstract\": \"Hepatocellular carcinoma (HCC) is a malignant neo-plasm with a high recurrence and metastatic rate, accounted for poor prognosis. Commonly existed heterogeneity is concerned with neoplasia, cancer progression, therapeutic resistance and metastasis is the principal cause of cancer lethality. As development of multi-omics methods in single-cell technology provides multi-faceted insight into disease processes in the era of precision medicine. Here, we interrogated single-cell transcriptomes, proteomes and epigenetic information, revealing metastasis potential heterogeneity in 5 HCC cell lines across different metastasis capacity. We confirmed that higher mesenchymal (M) status but not proliferation rate was associated with stronger metastasis ability of cell lines. Besides, we identified a subgroup being common in several cell lines, showing a higher hypoxic signature. A gene set involving 14 genes were chosen to represent the hypoxia state, much consistent than previous reported gene set, and showed worse prognosis association in TCGA data. This hypoxic subgroup prefers glycolysis metabolism than OXPO, and showed non-cycling, quiescent state which could be resistant to many proliferation-targeting drugs. Our results provide a comprehensive understanding of characteristic associated with metastasis capacity of HCC cell line, which will guide the metastasis mechanism study of HCC.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1119&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a85b045bb7ef910582c9250f55cf65fbdf85f762\",\n",
      "        \"title\": \"The single-cell landscape of immunological responses of CD4+ T cells in HIV versus severe acute respiratory syndrome coronavirus 2\",\n",
      "        \"abstract\": \"Purpose of review CD4+ T cell loss is the hallmark of uncontrolled HIV-1 infection. Strikingly, CD4+ T cell depletion is a strong indicator for disease severity in the recently emerged coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection. We reviewed recent single-cell immune profiling studies in HIV-1 infection and COVID-19 to provide critical insight in virus-induced immunopathogenesis. Recent findings Cytokine dysregulation in HIV-1 leads to chronic inflammation, while severe SARS-CoV-2 infection induces cytokine release syndrome and increased mortality. HIV-1-specific CD4+ T cells are dysfunctional, while SARS-CoV-2-specific CD4+ T cells exhibit robust Th1 function and correlate with protective antibody responses. In HIV-1 infection, follicular helper T cells (TFH) are susceptible to HIV-1 infection and persist in immune-sanctuary sites in lymphoid tissues as an HIV-1 reservoir. In severe SARS-CoV-2 infection, TFH are absent in lymphoid tissues and are associated with diminished protective immunity. Advancement in HIV-1 DNA, RNA, and protein-based single-cell capture methods can overcome the rarity and heterogeneity of HIV-1-infected cells and identify mechanisms of HIV-1 persistence and clonal expansion dynamics. Summary Single-cell immune profiling identifies a high-resolution picture of immune dysregulation in HIV-1 and SARS-CoV-2 infection and informs outcome prediction and therapeutic interventions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1120&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"13028455708d7b9242cb440c4d39dd0b213a7798\",\n",
      "        \"title\": \"Fibrosis: from mechanisms to medicines\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1121&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1ef8ea87d7f4100e04a0a2cccb6c7793684b8ffc\",\n",
      "        \"title\": \"Emerging Concepts of Hybrid Epithelial-to-Mesenchymal Transition in Cancer Progression\",\n",
      "        \"abstract\": \"Epithelial mesenchymal transition (EMT) is a complex process through which epithelial (E) cells lose their adherens junctions, transform into mesenchymal (M) cells and attain motility, leading to metastasis at distant organs. Nowadays, the concept of EMT has shifted from a binary phase of interconversion of pure E to M cells and vice versa to a spectrum of E/M transition states preferably coined as hybrid/partial/intermediate EMT. Hybrid EMT, being a plastic transient state, harbours cells which co-express both E and M markers and exhibit high tumourigenic properties, leading to stemness, metastasis, and therapy resistance. Several preclinical and clinical studies provided the evidence of co-existence of E/M phenotypes. Regulators including transcription factors, epigenetic regulators and phenotypic stability factors (PSFs) help in maintaining the hybrid state. Computational and bioinformatics approaches may be excellent for identifying new factors or combinations of regulatory elements that govern the different EMT transition states. Therapeutic intervention against hybrid E/M cells, though few, may evolve as a rational strategy against metastasis and drug resistance. This review has attempted to present the recent advancements on the concept and regulation of the process of hybrid EMT which generates hybrid E/M phenotypes, evidence of intermediate EMT in both preclinical and clinical setup, impact of partial EMT on promoting tumourigenesis, and future strategies which might be adapted to tackle this phenomenon.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1122&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f2a1dbb20c28868a100f0ce7174c96a9b315a86\",\n",
      "        \"title\": \"Molecular Biomarkers for Celiac Disease: Past, Present and Future\",\n",
      "        \"abstract\": \"Celiac disease (CeD) is a complex immune-mediated disorder that is triggered by dietary gluten in genetically predisposed individuals. CeD is characterized by inflammation and villous atrophy of the small intestine, which can lead to gastrointestinal complaints, malnutrition, and malignancies. Currently, diagnosis of CeD relies on serology (antibodies against transglutaminase and endomysium) and small-intestinal biopsies. Since small-intestinal biopsies require invasive upper-endoscopy, and serology cannot predict CeD in an early stage or be used for monitoring disease after initiation of a gluten-free diet, the search for non-invasive biomarkers is ongoing. Here, we summarize current and up-and-coming non-invasive biomarkers that may be able to predict, diagnose, and monitor the progression of CeD. We further discuss how current and emerging techniques, such as (single-cell) transcriptomics and genomics, can be used to uncover the pathophysiology of CeD and identify non-invasive biomarkers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1123&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c21c1221fdf0642aa621490822dcf0ba27a47a5f\",\n",
      "        \"title\": \"Predicting single-cell gene expression profiles of imaging flow cytometry data with machine learning\",\n",
      "        \"abstract\": \"Abstract High-content imaging and single-cell genomics are two of the most prominent high-throughput technologies for studying cellular properties and functions at scale. Recent studies have demonstrated that information in large imaging datasets can be used to estimate gene mutations and to predict the cell-cycle state and the cellular decision making directly from cellular morphology. Thus, high-throughput imaging methodologies, such as imaging flow cytometry can potentially aim beyond simple sorting of cell-populations. We introduce IFC-seq, a machine learning methodology for predicting the expression profile of every cell in an imaging flow cytometry experiment. Since it is to-date unfeasible to observe single-cell gene expression and morphology in flow, we integrate uncoupled imaging data with an independent transcriptomics dataset by leveraging common surface markers. We demonstrate that IFC-seq successfully models gene expression of a moderate number of key gene-markers for two independent imaging flow cytometry datasets: (i) human blood mononuclear cells and (ii) mouse myeloid progenitor cells. In the case of mouse myeloid progenitor cells IFC-seq can predict gene expression directly from brightfield images in a label-free manner, using a convolutional neural network. The proposed method promises to add gene expression information to existing and new imaging flow cytometry datasets, at no additional cost.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1124&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1c2eb565c84e3b19fa8ae862038eaba936677952\",\n",
      "        \"title\": \"Single-cell RNA-seq analysis of human coronary arteries using an enhanced workflow reveals SMC transitions and candidate drug targets\",\n",
      "        \"abstract\": \"Background and Aims The atherosclerotic plaque microenvironment is highly complex, and selective agents that modulate plaque stability or other plaque phenotypes are not yet available. We sought to investigate the human atherosclerotic cellular environment using scRNA-seq to uncover potential therapeutic approaches. We aimed to make our workflow user-friendly, reproducible, and applicable to other disease-specific scRNA-seq datasets. Methods Here we incorporate automated cell labeling, pseudotemporal ordering, ligand-receptor evaluation, and drug-gene interaction analysis into an enhanced and reproducible scRNA-seq analysis workflow. Notably, we also developed an R Shiny based interactive web application to enable further exploration and analysis of the scRNA dataset. Results We applied this analysis workflow to a human coronary artery scRNA dataset and revealed distinct derivations of chondrocyte-like and fibroblast-like cells from smooth muscle cells (SMCs), and show the key changes in gene expression along their de-differentiation path. We highlighted several key ligand-receptor interactions within the atherosclerotic environment through functional expression profiling and revealed several attractive avenues for future pharmacological repurposing in precision medicine. Further, our interactive web application, PlaqView (www.plaqview.com), allows other researchers to easily explore this dataset and benchmark applicable scRNA-seq analysis tools without prior coding knowledge. Conclusions These results suggest novel effects of chemotherapeutics on the atherosclerotic cellular environment and provide future avenues of studies in precision medicine. This publicly available workflow will also allow for more systematic and user-friendly analysis of scRNA datasets in other disease and developmental systems. PlaqView allows for rapid visualization and analysis of atherosclerosis scRNA-seq datasets without the need of prior coding experience. Future releases of PlaqView will feature additional larger scRNA-seq and scATAC-seq atherosclerosis-related datasets, thus providing a critical resource for the field by promoting data harmonization and biological interpretation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1125&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1068ac9d2f4c32e9b9c9e2ad274d7b6ad6102db3\",\n",
      "        \"title\": \"FGF9 promotes mouse spermatogonial stem cell proliferation mediated by p38 MAPK signalling\",\n",
      "        \"abstract\": \"Fibroblast growth factor 9 (FGF9) is expressed by somatic cells in the seminiferous tubules, yet little information exists about its role in regulating spermatogonial stem cells (SSCs).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1126&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f37718e57f6b34a993fe8b4e7a6c6508c5a1ab75\",\n",
      "        \"title\": \"Decoding the Heterogeneity of Human Dendritic Cell Subsets.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1127&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dca7fd81b00215ca307b8b83fa0640084a7e258a\",\n",
      "        \"title\": \"High-Resolution Transcriptomic and Proteomic Profiling of Heterogeneity of Brain-Derived Microglia in Multiple Sclerosis\",\n",
      "        \"abstract\": \"Microglia are important for central nervous system (CNS) homeostasis and first to respond to tissue damage and perturbations. Microglia are heterogeneous cells; in case of pathology, microglia adopt a range of phenotypes with altered functions. However, how these different microglia subtypes are implicated in CNS disease is largely unresolved. Multiple sclerosis (MS) is a chronic demyelinating disease of the CNS, characterized by inflammation and axonal degeneration, ultimately leading to neurological decline. One way microglia are implicated in MS is through stimulation of remyelination. They facilitate efficient remyelination by phagocytosis of myelin debris. In addition, microglia recruit oligodendrocyte precursor cells (OPCs) to demyelinated areas and stimulate remyelination. The development of high-resolution technologies to profile individual cells has greatly contributed to our understanding of microglia heterogeneity and function under normal and pathological conditions. Gene expression profiling technologies have evolved from whole tissue RNA sequencing toward single-cell or nucleus sequencing. Single microglia proteomic profiles are also increasingly generated, offering another layer of high-resolution data. Here, we will review recent studies that have employed these technologies in the context of MS and their respective advantages and disadvantages. Moreover, recent developments that allow for (single) cell profiling while retaining spatial information and tissue context will be discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1128&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bdf75cb08c31ffdee950a22ab8e59b3b34d76f4a\",\n",
      "        \"title\": \"Predicting Cross-Reactivity and Antigen Specificity of T Cell Receptors\",\n",
      "        \"abstract\": \"Adaptive immune recognition is mediated by specific interactions between heterodimeric T cell receptors (TCRs) and their cognate peptide-MHC (pMHC) ligands, and the methods to accurately predict TCR:pMHC interaction would have profound clinical, therapeutic and pharmaceutical applications. Herein, we review recent developments in predicting cross-reactivity and antigen specificity of TCR recognition. We discuss current experimental and computational approaches to investigate cross-reactivity and antigen-specificity of TCRs and highlight how integrating kinetic, biophysical and structural features may offer valuable insights in modeling immunogenicity. We further underscore the close inter-relationship of these two interconnected notions and the need to investigate each in the light of the other for a better understanding of T cell responsiveness for the effective clinical applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1129&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"81a1d88c3ab82c18a473ec57863c484b2e9ab7e0\",\n",
      "        \"title\": \"CNS-Native Myeloid Cells Drive Immune Suppression in the Brain Metastatic Niche through Cxcl10\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1130&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a32015c79bd160189ddb13fa99f034c432b94c59\",\n",
      "        \"title\": \"Probing infectious disease by single-cell RNA sequencing: Progresses and perspectives\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1131&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd858391c733080bd99c3532c07bbab760127042\",\n",
      "        \"title\": \"What has single\\u2010cell RNA sequencing revealed about microglial neuroimmunology?\",\n",
      "        \"abstract\": \"The use of single\\u2010cell RNA sequencing (scRNA\\u2010seq) in microglial research is increasing rapidly. The basic workflow of this approach consists of isolating single cells, followed by sequencing. scRNA\\u2010seq is capable of examining microglial heterogeneity on a cellular level. However, the results gained from applying this technique suffer from discrepancies due to differences between applied methods characteristics such as the number of cells sequenced and the depth of sequencing. This review aims to shed more light on the recent developments that happened in this field and how they are related to the methods used. To do that, we track the progress and limitations of various scRNA\\u2010seq methods currently available. The review then summarizes the current knowledge gained using scRNA\\u2010seq in the field of microglia, including novel subpopulations associated with function and development under homeostasis as well during several pathological conditions such as Alzheimer, lipopolysaccharide response, and HIV in relation to the methods employed. Our review points out that despite major developments found using this technique, current scRNA\\u2010seq methods suffer from high cost, low yields, and nonstandardization of generated data. Additional development of scRNA\\u2010seq methods will raise our awareness of microglia's heterogeneity and plasticity under healthy and pathological conditions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1132&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fdcc58f506fed5d60d4c83e9b05a087ee0217960\",\n",
      "        \"title\": \"Double-jeopardy: scRNA-seq doublet/multiplet detection using multi-omic profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1133&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fc5507b9c350d304fc0dedc4dfe9a0e8196bf8ab\",\n",
      "        \"title\": \"CASB: A concanavalin A-based sample barcoding strategy for single-cell sequencing\",\n",
      "        \"abstract\": \"Sample multiplexing facilitates single cell sequencing by reducing costs, revealing subtle difference between similar samples, and identifying artifacts such as cell doublets. However, universal and cost-effective strategies are rather limited. Here, we reported a Concanavalin A-based Sample Barcoding strategy (CASB), which could be followed by both single-cell mRNA and ATAC (assay for transposase accessible chromatin) sequencing techniques. The method involves minimal sample processing, thereby preserving intact transcriptomic or epigenomic patterns. We demonstrated its high labeling efficiency, high accuracy in assigning cells/nuclei to samples regardless of cell type and genetic background, as well as high sensitivity in detecting doublets by two applications: 1) CASB followed by scRNA-seq to track the transcriptomic dynamics of a cancer cell line perturbed by multiple drugs, which revealed compound-specific heterogeneous response; 2) CASB together with both snATAC-seq and scRNA-seq to illustrate the IFN-\\u03b3-mediated dynamic changes on epigenome and transcriptome profile, which identified the transcription factor underlying heterogeneous IFN-\\u03b3 response.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1134&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"82fa819bce732cdbdc601e7daae8b109d6972082\",\n",
      "        \"title\": \"SM-Omics: An automated platform for high-throughput spatial multi-omics\",\n",
      "        \"abstract\": \"The spatial organization of cells and molecules plays a key role in tissue function in homeostasis and disease. Spatial Transcriptomics (ST) has recently emerged as a key technique to capture and positionally barcode RNAs directly in tissues. Here, we advance the application of ST at scale, by presenting Spatial Multiomics (SM-Omics) as a fully automated high-throughput platform for combined and spatially resolved transcriptomics and antibody-based proteomics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1135&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c101cde821b04bfce96897ff44f08cc775ec4c53\",\n",
      "        \"title\": \"Detecting cancer metastasis and accompanying protein biomarkers at single cell levels using a 3D-printed microfluidic immunoarray.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1136&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9e11be8c7071f16f40b43515c57c867ed871434b\",\n",
      "        \"title\": \"Single-cell transcriptomics: a novel precision medicine technique in nephrology\",\n",
      "        \"abstract\": \"Due to the complex structure and function of the kidneys, the mechanism of kidney disease is unclear. In particular, transcriptomics approaches at the bulk level are unable to differentiate primary autonomous responses, which lead to disease development, from secondary cell non-autonomous responses. Single-cell analysis techniques can overcome the limitations inherent in the measurement of heterogeneous cell populations and clarify the central issues in kidney biology and disease pathogenesis. Single-cell sequencing helps in identifying disease-related biomarkers and pathways, stratifying patients, and deciding on appropriate treatment methods. Here we review a variety of single-cell analysis techniques and single-cell transcriptomics studies performed in the field of nephrology. Moreover, we discuss the future prospects of single-cell analysis-based precision medicine in nephrology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1137&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"089d05864f34b3ba3dc27496e198755f60598aab\",\n",
      "        \"title\": \"CD155 on Tumor Cells Drives Resistance to Immunotherapy by Inducing the Degradation of the Activating Receptor CD226 in CD8+ T\\u00a0Cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1138&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c7a475175e5b2d5ad6ea55cf674cf1721e98899c\",\n",
      "        \"title\": \"Targeted Single-Cell RNA and DNA Sequencing With Fluorescence-Activated Droplet Merger.\",\n",
      "        \"abstract\": \"Analyzing every cell in a diverse sample provides insight into population-level heterogeneity, but abundant cell types dominate the analysis and rarer populations are scarcely represented in the data. To focus on specific cell types, the current paradigm is to physically isolate subsets of interest prior to analysis; however, it remains difficult to isolate and then single-cell sequence such populations because of compounding losses. Here, we describe an alternative approach that selectively merges cells with reagents to achieve enzymatic reactions without having to physically isolate cells. We apply this technique to perform single-cell transcriptome and genome sequencing of specific cell subsets. Our method for analyzing heterogeneous populations obviates the need for pre- or post-enrichment and simplifies single-cell workflows, making it useful for other applications in single-cell biology, combinatorial chemical synthesis, and drug screening.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1139&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ce080083a7a3d3177732dff0da508e0da152837d\",\n",
      "        \"title\": \"Integrated analysis of multimodal single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1140&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2918951b4b7094cd5b8bf93916f62aba4101f3a3\",\n",
      "        \"title\": \"Human Developmental Cell Atlas: milestones achieved and the roadmap ahead\",\n",
      "        \"abstract\": \"\\n The Human Developmental Cell Atlas (HDCA), as part of the Human Cell Atlas, aims to generate a comprehensive reference map of cells during development. This detailed study of development will be critical for understanding normal organogenesis, the impact of mutations, environmental factors and infectious agents on congenital and childhood disorders, and the molecular cellular basis of ageing, cancer and regenerative medicine. In this perspective, we outline the challenges of mapping and modelling human development using state of the art technologies to create a reference atlas across gestation for scientific and clinical benefit. We discuss the potential value of HDCA to enhance human pluripotent stem cell-derived organoid model systems and, in turn, the use of organoids and animal models to inform HDCA. Finally, we provide a roadmap towards a complete atlas of human development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1141&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bf213026897b1702fd0442b71089ab0b1b5a43ce\",\n",
      "        \"title\": \"T cell fate mapping and lineage tracing technologies probing clonal aspects underlying the generation of CD8 T cell subsets\",\n",
      "        \"abstract\": \"T cells responding to acute infections generally provide two key functions to protect the host: (1) active contribution to pathogen elimination and (2) providing long\\u2010lived cells that are poised to rapidly respond to renewed infection, thus ensuring long\\u2010lasting protection against the particular pathogen. Extensive work has established an astonishing amount of additional diversity among T cells actively contributing to pathogen elimination, as well as among resting, long\\u2010lived antigen\\u2010experienced T cells. This led to the description of a variety of functionally distinct T cell \\u2018subsets\\u2019. Understanding how this heterogeneity develops among T cells responding to the same antigen is currently an active area of research, since knowledge of such mechanisms may have implications for the development of vaccines and immunotherapy. The number of na\\u00efve T cells specific to a given antigen span a great range. Considering this, one mechanistic angle focusses on how individual na\\u00efve T cells contribute to the development of the distinct T cell subsets. In this review, we highlight the current technologies that enable one to address the contributions of individual na\\u00efve T cells to different T cell subsets, with a focus on CD8 T cell subsets generated in the context of acute infections. Moreover, we discuss the requirements of new technologies to further our understanding of the mechanisms that help generate long\\u2010lasting immunity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1142&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d0f8aeec70615a225a412e60021ff7556490c5c\",\n",
      "        \"title\": \"Ensemble learning models that predict surface protein abundance from single-cell multimodal omics data.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1143&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4330b61811e7f90051f0dbc25df9d6f8bc107d61\",\n",
      "        \"title\": \"How to bridle a neutrophil.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1144&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"71b3b96a5bce3602570aac5df711e67819031b7c\",\n",
      "        \"title\": \"Single-cell multimodal profiling of proteins and chromatin accessibility using PHAGE-ATAC\",\n",
      "        \"abstract\": \"Multi-modal measurements of single cell profiles are a powerful tool for characterizing cell states and regulatory mechanisms. While current methods allow profiling of RNA along with either chromatin or protein levels, connecting chromatin state to protein levels remains a barrier. Here, we developed PHAGE-ATAC, a method that uses engineered camelid single-domain antibody (\\u2018nanobody\\u2019)-displaying phages for simultaneous single-cell measurement of surface proteins, chromatin accessibility profiles, and mtDNA-based clonal tracing through a massively parallel droplet-based assay of single-cell transposase-accessible chromatin with sequencing (ATAC-seq). We demonstrate PHAGE-ATAC for multimodal analysis in primary human immune cells and for sample multiplexing. Finally, we construct a synthetic high-complexity phage library for selection of novel antigen-specific nanobodies that bind cells of particular molecular profiles, opening a new avenue for protein detection, cell characterization and screening with single-cell genomics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1145&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9b95a4d6991f62036ffb0497d3a0ffa56b4515cb\",\n",
      "        \"title\": \"Aged Breast Extracellular Matrix Drives Mammary Epithelial Cells to an Invasive and Cancer\\u2010Like Phenotype\",\n",
      "        \"abstract\": \"Age is a major risk factor for cancer. While the importance of age related genetic alterations in cells on cancer progression is well documented, the effect of aging extracellular matrix (ECM) has been overlooked. Here, we show for the first time that the aging breast ECM is sufficient to drive normal mammary epithelial cells (KTB21) to a more invasive and cancer-like phenotype, while promoting motility and invasiveness in MDA-MB-231 cells. E-cadherin membrane localization was lost in KTB21 cells cultured on the decellularized breast matrix from aged mice. Cell motility, cell invasion, and inflammatory cytokine and cancer-related protein production were increased significantly on the aged matrix, and many genes related to invasion were upregulated. Strikingly, we showed using single cell RNA sequencing that the aged matrix led to enrichment of a subpopulation of KTB21 cells that highly expressed epithelial-mesenchymal transition (EMT) and invasion-related genes. Lysyl oxidase (LOX) knockdown reverted the aged matrix-induced changes to the young levels; LOX siRNA treatment prevented the loss of E-cadherin membrane localization, and reduced cell motility, cell invasion, and cytokine and cancer-related protein production. Finally, we showed that the biophysical, mechanical and biochemical properties of the breast ECM were altered dramatically upon aging. Analyzing these factors and studying the differential response of the epithelial cells to young and aged ECMs could lead to identification of new targets for cancer treatment and could pave the way for the discovery of new therapeutic options.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1146&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e309721ad12014ab045f9b994e2248b81aabee2d\",\n",
      "        \"title\": \"Single-cell analyses identify circulating anti-tumor CD8 T cells and markers for their enrichment\",\n",
      "        \"abstract\": \"The ability to monitor anti-tumor CD8+ T cell responses in the blood has tremendous therapeutic potential. Here, we used paired single-cell RNA sequencing and T cell receptor (TCR) sequencing to detect and characterize \\u201ctumor matching\\u201d (TM) CD8+ T cells in the blood of mice with MC38 tumors and melanoma patients using the TCR as a molecular barcode. TM cells showed increased activation compared to non-matching T cells in blood, and appeared less exhausted than matching counterparts in tumor. Importantly, PD-1, which has been used to identify putative circulating anti-tumor CD8+ T cells, showed poor sensitivity for identifying TM cells. By leveraging the transcriptome we identified candidate cell surface marker panels for TM cells in mice and melanoma patients, and validated NKG2D, CD39, and CX3CR1 in mice. These data demonstrate that the TCR can be used to identify tumor-relevant populations for comprehensive characterization, reveal unique transcriptional properties of TM cells, and develop marker panels for tracking and analysis of these cells. Summary Using single-cell RNA-sequencing coupled with TCR sequencing, we detected CD8+ T cell clones shared between blood and tumor in mice and melanoma patients, characterized these matching clones in blood and tumor, and identified potential biomarkers for their isolation in blood.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1147&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3b2cd586f3e25246f9a4d02eb5c90d34b407f30d\",\n",
      "        \"title\": \"Pathway dynamics can delineate the sources of transcriptional noise in gene expression\",\n",
      "        \"abstract\": \"Single-cell expression profiling is destructive, giving rise to only static snapshots of cellular states. This loss of temporal information presents significant challenges in inferring dynamics from population data. Here we provide a formal analysis of the extent to which dynamic variability from within individual systems (\\u201cintrinsic noise\\u201d) is distinguishable from variability across the population (\\u201cextrinsic noise\\u201d). Our results mathematically formalise observations that it is impossible to identify these sources of variability from the transcript abundance distribution alone. Notably, we find that systems subject to population variation invariably inflate the apparent degree of burstiness of the underlying process. Such identifiability problems can be remedied by the dual-reporter method, which separates the total gene expression noise into intrinsic and extrinsic contributions. This noise decomposition, however, requires strictly independent and identical gene reporters integrated into the same cell, which can be difficult to implement experimentally in many systems. Here we demonstrate mathematically that, in some cases, the same noise decomposition can be achieved at the transcriptional level with non-identical and not-necessarily independent reporters. We use our result to show that generic reporters lying in the same biochemical pathways (e.g. mRNA and protein) can replace dual reporters, enabling the noise decomposition to be obtained from only a single gene. Stochastic simulations are used to support our theory, and show that our \\u201cpathway-reporter\\u201d method compares favourably to the dual-reporter method.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1148&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4118f9cf41d18a34cb1e1181db3ded50802e0103\",\n",
      "        \"title\": \"Nebulosa recovers single cell gene expression signals by kernel density estimation\",\n",
      "        \"abstract\": \"Summary Data sparsity in single-cell experiments prevents an accurate assessment of gene expression when visualised in a low-dimensional space. Here, we introduce Nebulosa, an R package that uses weighted kernel density estimation to recover signals lost through drop-out or low expression. Availability and implementation Nebulosa can be easily installed from www.github.com/powellgenomicslab/Nebulosa\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1149&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b388f54df2c1ee625c1d99557db468c22bc5c3db\",\n",
      "        \"title\": \"Immunometabolism in the Single-Cell Era.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1150&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ab10c2adcee32fee94e408cd9bfc4156dc1e1982\",\n",
      "        \"title\": \"The functional and phenotypic diversity of single T-cell infiltrates in human colorectal cancer as correlated with clinical outcome\",\n",
      "        \"abstract\": \"Although degree of T-cell infiltration in CRC was shown to correlate with a positive prognosis, the contribution of phenotypically and functionally distinct T cell subtypes within tumors remains unclear. We analyzed 37,931 single T cells with respect to transcriptome, TCR sequence and 23 cell surface proteins, from tumors and adjacent normal colon of 16 patients. Our comprehensive analysis revealed two phenotypically distinct cytotoxic T cell populations within tumors, including positively prognostic effector memory cells and non-prognostic resident memory cells. These cytotoxic T cell infiltrates transitioned from effector memory to resident memory in a stage-dependent manner. We further defined several Treg subpopulations within tumors. While Tregs overall were associated with positive clinical outcomes, CD38+ peripherally-derived Tregs, phenotypically related to Th17 cells, correlated with poor outcomes independent of cancer stage. Thus, our data highlight the diversity of T cells in CRC and demonstrate the prognostic significance of distinct T cell subtypes, which could inform therapeutic strategies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1151&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"30b4de96343875bec587c20efeb7c90dacc3f2ee\",\n",
      "        \"title\": \"Towards High-Throughput Single-Cell Proteomics Using Droplet Microfluidics\",\n",
      "        \"abstract\": \"Single-cell proteomics is emerging as an effective method to characterize cell populations. To accommodate alternative protein labeling and quantification technologies, it may be beneficial to incorporate cell encapsulation and droplet coalescence in a standard workflow. In this work, a microfluidic device was designed and tested towards the goal of high-throughput single-cell analysis. The device incorporated a cell-encapsulated droplet generator, an on-chip DNA-barcode droplet separator, and a droplet merging module. The microfluidic droplets that can contain cells are merged with the barcode-containing droplets. A merging efficiency of 68% was achieved.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1152&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d7cee8db1162743ed0972bd03e6106eb91ea4b45\",\n",
      "        \"title\": \"Profiling the Non-genetic Origins of Cancer Drug Resistance with a Single-Cell Functional Genomics Approach Using Predictive Cell Dynamics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1153&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b256fd6333503790f39d3a1402f302eeec612bc3\",\n",
      "        \"title\": \"Resolving Neurodevelopmental and Vision Disorders Using Organoid Single-Cell Multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1154&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2bef51ddd017bad5410f2c67dee6ae658339f8d2\",\n",
      "        \"title\": \"MarkerCapsule: Explainable Single Cell Typing using Capsule Networks\",\n",
      "        \"abstract\": \"Many single cell typing methods require manual annotation which casts problems with respect to resolution of (sub-)types, manpower resources and bias towards existing human knowledge. The integration of heterogeneous data and biologically meaningful interpretation of results are further current key challenges. We introduce MarkerCapsule, which leverages the landmark advantages of capsule networks achieved in their original applications in single cell typing. Thereby, the small amount of labeled data required and the naturally arising, biologically meaningful interpretation of cell types in terms of characteristic gene activity patterns are exemplary strengths, beyond outperforming the state of the art in terms of basic typing accuracy. MarkerCapsule is available at: https://github.com/sumantaray/MarkerCapsule.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1155&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"09e669f97ce1ac51b2fb5f5fda69a0fb0429ba13\",\n",
      "        \"title\": \"Single-cell multimodal omics and directly reprogrammed neurons to probe reduced penetrance in Frontotemporal Dementia\",\n",
      "        \"abstract\": \"We identified an individual with reduced penetrance (RedPenMC) associated with an autosomal dominant progranulin (GRN) pathogenic variant (p.Tyr294*) causing frontotemporal dementia. The RedPenMC carrier had no signs or symptoms of frontotemporal lobar degeneration in life or at autopsy. This resistance to develop expected pathology gives a unique opportunity to interrogate neurodegenerative mechanisms, identify protective disease modifiers, and gain new therapeutic insights. We performed multimodal droplet-based single nuclei analyses of the frontal cortex from the RedPenMC post-mortem, including unbiased transcriptomics and profiling of global levels of twelve chromatin marks. Further, we analyzed neurons directly reprogrammed from skin fibroblasts from RedPenMC. Single nuclei analyses revealed that the RedPenMC carrier showed a higher expression of GRN in microglia than in any other cell types as well as compared to an affected mutation carrier and a non-carrier. Global levels of heterochromatin marks H3k9me1 and H3k9me3 were significantly lower in the RedPenMC carrier, whereas global levels of H2A2 were significantly upregulated in the RedPenMC carrier compared to an affected mutation carrier. Analyses of directly reprogrammed neurons from the RedPenMC carrier suggested a dendritic loss of GRN and spillage of TAR DNA-binding protein 43 (TDP-43) from the nucleus into the cytoplasm which was not observed in directly reprogrammed neurons from non-carriers. Taken together, our data suggest that higher expression of GRN in microglia and reduced levels of repressive marks may provide endogenous protection against the disease process/neurodegeneration in the RedPenMC carrier. Our study uses unique patient material and provides novel insights into disease mechanisms that can be explored for designing new therapeutic approaches for GRN associated frontotemporal dementia. Graphical abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1156&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f410f1d479eb05bad5948549ae503163c9e31e9\",\n",
      "        \"title\": \"Joint Grid Discretization for Biological Pattern Discovery\",\n",
      "        \"abstract\": \"The complexity, dynamics, and scale of data acquired by modern biotechnology increasingly favor model-free computational methods that make minimal assumptions about underlying biological mechanisms. For example, single-cell transcriptome and proteome data have a throughput several orders more than bulk methods. Many model-free statistical methods for pattern discovery such as mutual information and chi-squared tests, however, require discrete data. Most discretization methods minimize squared errors for each variable independently, not necessarily retaining joint patterns. To address this issue, we present a joint grid discretization algorithm that preserves clusters in the original data. We evaluated this algorithm on simulated data to show its advantage over other methods in maintaining clusters as measured by the adjusted Rand index. We also show it promotes global functional patterns over independent patterns. On single-cell proteome and transcriptome of leukemia and healthy blood, joint grid discretization captured known protein-to-RNA regulatory relationships, while revealing previously unknown interactions. As such, the joint grid discretization is applicable as a data transformation step in associative, functional, and causal inference of molecular interactions fundamental to systems biology. The developed software is publicly available at https://cran.r-project.org/package=GridOnClusters\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1157&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5519fb7c4d72f102084ac484ba122661a2e31de\",\n",
      "        \"title\": \"Novel Protein-oligonucleotide Conjugation Method Involving a High-affinity Capture HaloTag.\",\n",
      "        \"abstract\": \"Highly sensitive quantitative protein profiling can play a key role in the early diagnosis of diseases, such as autoimmune diseases and cancer. We developed a modified protein-oligonucleotide conjugation method termed HaloTag-mediated barcoding, for quantifying protein molecules at a higher sensitivity than conventional protein quantification methods. This novel and efficient conjugation method can be used to prepare HaloTag-barcoded proteins using a click chemistry-based labeling technique. Here, we describe the preparation of protein-DNA complexes and detection of protein-protein interactions which can be used in a HaloTag protein barcode assay to detect an antibody. The protocol includes procedures for preparing the ligand-oligonucleotide complex, plasmid DNA preparation for protein expression, and preparation of the protein-oligonucleotide complex. The described click reaction-based protocols simplify the conventional amine-ester reaction methods which require additional steps for chromatography purification.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1158&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ba8866380e3cad704bbabcb0873065f5be6682c6\",\n",
      "        \"title\": \"Single-Cell Profiling Reveals Divergent, Globally Patterned Immune Responses in Murine Skin Inflammation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1159&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e02ae00322897f29b59c49196e9154addef64a37\",\n",
      "        \"title\": \"Single-cell genomics to understand disease pathogenesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1160&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3c794aab6e4093c1ca5cf965160c388a078b0693\",\n",
      "        \"title\": \"Combined protein and transcript single-cell RNA sequencing in human peripheral blood mononuclear cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1161&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0359c88b3604f858e11992153ca7cb925f81e760\",\n",
      "        \"title\": \"Combined protein and transcript single cell RNA sequencing reveals cardiovascular disease and HIV signatures\",\n",
      "        \"abstract\": \"Background HIV-infected people have an increased risk of atherosclerosis-based cardiovascular disease (CVD), even when the HIV virus is fully controlled. Both chronic HIV infection and CVD are chronic inflammatory diseases. The interaction between these two diseases is not well understood. Methods The Women\\u2019s Interagency HIV Study (WIHS) collected peripheral blood mononuclear cells (PBMCs) and data on subclinical CVD defined by carotid artery ultrasound from HIV-infected women. We interrogated 32 PBMC samples using combined protein and transcript panel single cell (sc) RNA sequencing of women without HIV or CVD, with HIV only, with HIV and CVD, and with HIV and CVD treated with cholesterol-lowering drugs. Expression of 40 surface markers enabled detailed analysis of all major cell types, resolving 58 clusters in almost 42,000 single cells. Results Many clusters including 5 of 8 classical monocyte clusters showed significantly different gene expression between the groups of participants, revealing the inflammatory signatures of HIV, CVD and their interactions. Genes highly upregulated by CVD included CCL3, CCL4 and IL-32, whereas CXCL2 and 3 were more highly upregulated by HIV. Many genes were synergistically upregulated by HIV and CVD, but others were antagonistically regulated, revealing that the gene signature in people with HIV and CVD is not simply the sum of the HIV and CVD signatures. Elevated expression of most inflammatory genes was reversed by cholesterol control (statin treatment). The cell numbers in 3 of 5 intermediate monocyte subsets, 1 of 14 CD8 T cell subsets, 1 of 6 B cell subsets and 1 of 6 NK cell subsets showed significant changes with HIV or CVD. Conclusions We conclude that HIV and CVD show interactive inflammatory signatures including chemokines and cytokines that are improved by cholesterol-lowering drugs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1162&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7ea93af5ac121ab8baca0a02cb9d98feb906d3e0\",\n",
      "        \"title\": \"Robust clustering and interpretation of scRNA-seq data using reference component analysis\",\n",
      "        \"abstract\": \"Motivation The transcriptomic diversity of the hundreds of cell types in the human body can be analysed in unprecedented detail using single cell (SC) technologies. Though clustering of cellular transcriptomes is the default technique for defining cell types and subtypes, single cell clustering can be strongly influenced by technical variation. In fact, the prevalent unsupervised clustering algorithms can cluster cells by technical, rather than biological, variation. Results Compared to de novo (unsupervised) clustering methods, we demonstrate using multiple benchmarks that supervised clustering, which uses reference transcriptomes as a guide, is robust to batch effects. To leverage the advantages of supervised clustering, we present RCA2, a new, scalable, and broadly applicable version of our RCA algorithm. RCA2 provides a user-friendly framework for supervised clustering and downstream analysis of large scRNA-seq data sets. RCA2 can be seamlessly incorporated into existing algorithmic pipelines. It incorporates various new reference panels for human and mouse, supports generation of custom panels and uses efficient graph-based clustering and sparse data structures to ensure scalability. We demonstrate the applicability of RCA2 on SC data from human bone marrow, healthy PBMCs and PBMCs from COVID-19 patients. Importantly, RCA2 facilitates cell-type-specific QC, which we show is essential for accurate clustering of SC data from heterogeneous tissues. In the era of cohort-scale SC analysis, supervised clustering methods such as RCA2 will facilitate unified analysis of diverse SC datasets. Availability RCA2 is implemented in R and is available at github.com/prabhakarlab/RCAv2\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1163&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a413c7b06ca7461451e4953432a48a55190259b2\",\n",
      "        \"title\": \"Application of Single-Cell Sequencing to Immunotherapy.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1164&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9dcf53cc1ce9bb69ce24f875e10a183d86c9be2b\",\n",
      "        \"title\": \"Scalable, multimodal profiling of chromatin accessibility and protein levels in single cells\",\n",
      "        \"abstract\": \"Recent technological advances have enabled massively parallel chromatin profiling with single-cell Assay for Transposase Accessible Chromatin by sequencing (scATAC-seq) in thousands of individual cells. Here, we extend these approaches and present ATAC with Select Antigen Profiling by sequencing, ASAP-seq, a tool to simultaneously profile accessible chromatin and protein levels in thousands of single cells. Our approach pairs sparse scATAC-seq data with robust detection of hundreds of cell surface and intracellular protein markers and optional capture of mitochondrial DNA (mtDNA) for clonal tracking, thus concomitantly capturing three distinct modalities in single cells. Importantly, ASAP-seq uses a novel bridging approach that repurposes antibody:oligo conjugates designed for existing technologies that pair protein measurements with single cell RNA-seq. We demonstrate the utility of ASAP-seq by revealing coordinated and distinct changes in chromatin, RNA, and surface proteins during native hematopoietic differentiation, peripheral blood mononuclear cell stimulation, and as a combinatorial decoder and reporter of multiplexed perturbations in primary T cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1165&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a9a4af418f0b7dff04e64661d0a726f28828ba7c\",\n",
      "        \"title\": \"TEA-seq: a trimodal assay for integrated single cell measurement of transcription, epitopes, and chromatin accessibility\",\n",
      "        \"abstract\": \"Single-cell measurements of cellular characteristics have been instrumental in understanding the heterogeneous pathways that drive differentiation, cellular responses to extracellular signals, and human disease states. scATAC-seq has been particularly challenging due to the large size of the human genome and processing artefacts resulting from DNA damage that are an inherent source of background signal. Downstream analysis and integration of scATAC-seq with other single-cell assays is complicated by the lack of clear phenotypic information linking chromatin state and cell type. Using the heterogeneous mixture of cells in human peripheral blood as a test case, we developed a novel scATAC-seq workflow that increases the signal-to-noise ratio and allows simultaneous measurement of cell surface markers: Integrated Cellular Indexing of Chromatin Landscape and Epitopes (ICICLE-seq). We extended this approach using a droplet-based multiomics platform to develop a trimodal assay to simultaneously measure Transcriptomic state (scRNA-seq), cell surface Epitopes, and chromatin Accessibility (scATAC-seq) from thousands of single cells, which we term TEA-seq. Together, these multimodal single-cell assays provide a novel toolkit to identify type-specific gene regulation and expression grounded in phenotypically defined cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1166&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"60379c30f39e4c0513a94546364bfe9e223de66a\",\n",
      "        \"title\": \"Multi-omic single cell analysis resolves novel stromal cell populations in healthy and diseased human tendon\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1167&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"65d54979bf80bfae86bb85920ed427e6dabd325a\",\n",
      "        \"title\": \"Applying Single-Cell Analysis to Gonadogenesis and DSDs (Disorders/Differences of Sex Development)\",\n",
      "        \"abstract\": \"The gonads are unique among the body\\u2019s organs in having a developmental choice: testis or ovary formation. Gonadal sex differentiation involves common progenitor cells that form either Sertoli and Leydig cells in the testis or granulosa and thecal cells in the ovary. Single-cell analysis is now shedding new light on how these cell lineages are specified and how they interact with the germline. Such studies are also providing new information on gonadal maturation, ageing and the somatic-germ cell niche. Furthermore, they have the potential to improve our understanding and diagnosis of Disorders/Differences of Sex Development (DSDs). DSDs occur when chromosomal, gonadal or anatomical sex are atypical. Despite major advances in recent years, most cases of DSD still cannot be explained at the molecular level. This presents a major pediatric concern. The emergence of single-cell genomics and transcriptomics now presents a novel avenue for DSD analysis, for both diagnosis and for understanding the molecular genetic etiology. Such -omics datasets have the potential to enhance our understanding of the cellular origins and pathogenesis of DSDs, as well as infertility and gonadal diseases such as cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1168&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e60a1f002f7bb237ea68d19c00cd8bda71f3f4c3\",\n",
      "        \"title\": \"Immune Literacy: Reading, Writing, and Editing Adaptive Immunity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1169&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"31c560a7dfd344ff5b5f62e3b7a0184e0742ed71\",\n",
      "        \"title\": \"Systemic Immunometabolism: Challenges and Opportunities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1170&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f5ca7154be485680eb823ae8d23bdbf38bf341d4\",\n",
      "        \"title\": \"Molecular underpinnings of ssDNA specificity by Rep HUH endonucleases and implications for HUH-tag multiplexing and engineering\",\n",
      "        \"abstract\": \"Replication initiator proteins (Reps) from the HUH-endonuclease superfamily process specific single-stranded DNA (ssDNA) sequences to initiate rolling circle/hairpin replication in viruses, such as crop ravaging geminiviruses and human disease causing parvoviruses. In biotechnology contexts, Reps are the basis for HUH-tag bioconjugation and a critical adeno-associated virus genome integration tool. We solved the first co-crystal structures of Reps complexed to ssDNA, revealing a key motif for conferring sequence specificity and anchoring a bent DNA architecture. In combination, we developed a deep sequencing cleavage assay termed HUH-seq to interrogate subtleties in Rep specificity, and demonstrate how differences can be exploited for multiplexed HUH-tagging. Together, our insights allowed us to engineer a Rep chimera to predictably alter sequence specificity. These results have important implications for modulating viral infections, developing Rep-based genomic integration tools, and enabling massively parallel HUH-tag barcoding and bioconjugation applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1171&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8565afdc7c5a2fdae7b792dc9bb995d7f95a85f3\",\n",
      "        \"title\": \"Single-cell transcriptomics in cancer: computational challenges and opportunities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1172&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"92cae17ca0d64b5f095ab3c9acf36845f1540f54\",\n",
      "        \"title\": \"Single-cell sequencing techniques from individual to multiomics analyses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1173&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"05f5bdfff61003794842ef08241bfa9997937871\",\n",
      "        \"title\": \"Single-cell multiomics: technologies and data analysis methods\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1174&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fc6c2296466d979680f59251edaac83afed0f3ef\",\n",
      "        \"title\": \"An era of single-cell genomics consortia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1175&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6ef5c2404be3f7fab62533ccb98a58b89a0228e6\",\n",
      "        \"title\": \"Single-cell genomics technology: perspectives\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1176&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0f282dc7128d656ef8a96c55ae065e6557278a85\",\n",
      "        \"title\": \"Multi-modal pooled Perturb-CITE-Seq screens in patient models define novel mechanisms of cancer immune evasion\",\n",
      "        \"abstract\": \"Resistance to immune checkpoint inhibitors (ICI) that activate T cell mediated anti-tumor immunity is a key challenge in cancer therapy, yet the underlying mechanisms remain poorly understood. To further elucidate those, we developed a new approach, Perturb-CITE-seq, for pooled CRISPR perturbation screens with multi-modal RNA and protein single-cell profiling readout and applied it to screen patient-derived autologous melanoma and tumor infiltrating lymphocyte (TIL) co-cultures. We profiled RNA and 20 surface proteins in over 218,000 cells under ~750 perturbations, chosen by their membership in an immune evasion program that is associated with immunotherapy resistance in patients. Our screen recovered clinically-relevant resistance mechanisms concordantly reflected in RNA, protein and perturbation effects on susceptibility to T cell mediated killing. These were organized in eight co-functional modules whose perturbation distinctly affect four co-regulated programs associated with immune evasion. Among these were defects in the IFN\\u03b3-JAK/STAT pathway and in antigen presentation, and several novel mechanisms, including loss or downregulation of CD58, a surface protein without known mouse homolog. Leveraging the rich profiles in our screen, we found that loss of CD58 did not compromise MHC protein expression and that CD58 was not transcriptionally induced by the IFN\\u03b3 pathway, allowing us to distinguish it as a novel mechanism of immune resistance. We further show that loss of CD58 on cancer cells conferred immune evasion across multiple T cell and Natural Killer cell patient co-culture models. Notably, CD58 is downregulated in tumors with resistance to immunotherapy in melanoma patients. Our work identifies novel mechanisms at the nexus of immune evasion and drug resistance and provides a general framework for deciphering complex mechanisms by large-scale perturbation screens with multi-modal singlecell profiles, including in systems consisting of multiple cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1177&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9493261475472b1288b88a81d2f3ac6a0956744\",\n",
      "        \"title\": \"The prospects of single\\u2010cell analysis in autoimmunity\",\n",
      "        \"abstract\": \"In the last decade, there has been a tremendous development of technologies focused on analysing various molecular attributes in single cells, with an ever\\u2010increasing number of parameters becoming available at the DNA, RNA and protein levels. Much of this progress has involved cells in suspension, but also in situ analysis of tissues has taken great leaps. Paralleling the development in the laboratory, and because of increasing complexity, the analysis of single\\u2010cell data is also constantly being updated with new algorithms and analysis platforms. Our immune system shares this complexity, and immunologists have therefore been in the forefront of this technological development. These technologies clearly open new avenues for immunology research, maybe particularly within autoimmunity where the interaction between the faulty immune system and the thymus or the target organ is important. However, the technologies currently available can seem overwhelming and daunting. The aim of this review is to remedy this by giving a balanced overview of the prospects of using single\\u2010cell analysis in basal and clinical autoimmunity research, with an emphasis on endocrine autoimmunity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1178&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff45da97a30288ca41a9c18bec05588e34b76548\",\n",
      "        \"title\": \"Integrative Methods and Practical Challenges for Single-Cell Multi-omics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1179&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5c34252733cbf479498ab1c5292e21a160ecbc6\",\n",
      "        \"title\": \"Dissecting the landscape of activated CMV-stimulated CD4+ T cells in human by linking single-cell RNA-seq with T-cell receptor sequencing\",\n",
      "        \"abstract\": \"CD4 T cell is crucial in CMV infection, but its role is still unclear during this process. Here, we present a single-cell RNA-seq together with T cell receptor (TCR) sequencing to screen the heterogenicity and potential function of CMV pp65 reactivated CD4+ T cell subsets from human peripheral blood, and unveil their potential interactions. Notably, Treg composed the major part of these reactivated cells. Treg gene expression data revealed multiple transcripts of both inflammatory and inhibitory functions. Additionally, we describe the detailed phenotypes of CMV-reactivated effector-memory (Tem), cytotoxic T (CTL), and na\\u00efve T cells at the single-cell resolution, and implied the direct derivation of CTL from na\\u00efve CD4+ T cells. By analyzing the TCR repertoire, we identified a clonality in stimulated Tem and CTLs, and a tight relationship of Tem and CTL showing a large share in TCR. This study provides clues for understanding the function of CD4+ T cells subsets and unveils their interaction in CMV infection, and may promote the development of CMV immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1180&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6460582a385df7eb6f29892bb653e5737657cdba\",\n",
      "        \"title\": \"Single-cell metabolic profiling of human cytotoxic T cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1181&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aaf85152b102d9e20ebb6d7106cb163e4aa616ed\",\n",
      "        \"title\": \"Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram\",\n",
      "        \"abstract\": \"Charting a biological atlas of an organ, such as the brain, requires us to spatially-resolve whole transcriptomes of single cells, and to relate such cellular features to the histological and anatomical scales. Single-cell and single-nucleus RNA-Seq (sc/snRNA-seq) can map cells comprehensively5,6, but relating those to their histological and anatomical positions in the context of an organ\\u2019s common coordinate framework remains a major challenge and barrier to the construction of a cell atlas7\\u201310. Conversely, Spatial Transcriptomics allows for in-situ measurements11\\u201313 at the histological level, but at lower spatial resolution and with limited sensitivity. Targeted in situ technologies1\\u20133 solve both issues, but are limited in gene throughput which impedes profiling of the entire transcriptome. Finally, as samples are collected for profiling, their registration to anatomical atlases often require human supervision, which is a major obstacle to build pipelines at scale. Here, we demonstrate spatial mapping of cells, histology, and anatomy in the somatomotor area and the visual area of the healthy adult mouse brain. We devise Tangram, a method that aligns snRNA-seq data to various forms of spatial data collected from the same brain region, including MERFISH1, STARmap2, smFISH3, and Spatial Transcriptomics4 (Visium), as well as histological images and public atlases. Tangram can map any type of sc/snRNA-seq data, including multi-modal data such as SHARE-seq data5, which we used to reveal spatial patterns of chromatin accessibility. We equipped Tangram with a deep learning computer vision pipeline, which allows for automatic identification of anatomical annotations on histological images of mouse brain. By doing so, Tangram reconstructs a genome-wide, anatomically-integrated, spatial map of the visual and somatomotor area with \\u223c30,000 genes at single-cell resolution, revealing spatial gene expression and chromatin accessibility patterning beyond current limitation of in-situ technologies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1182&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1badfa570afe24993ffa666cfb35acde3b1b1cd6\",\n",
      "        \"title\": \"The Whole Body as the System in Systems Immunology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1183&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2bd0a92398a9cef3d3e76a4e88ae598a471bbe61\",\n",
      "        \"title\": \"A pathogenic and clonally expanded B cell transcriptome in active multiple sclerosis\",\n",
      "        \"abstract\": \"Significance B cells serve as a key weapon against infectious diseases. They also contribute to multiple autoimmune diseases, including multiple sclerosis (MS) where depletion of B cells is a highly effective therapy. We describe a comprehensive profile of central nervous system (CNS)-specific transcriptional B cell phenotypes in MS at single-cell resolution with paired immune repertoires. We reveal a polyclonal immunoglobulin M (IgM) and IgG1 cerebrospinal fluid B cell expansion polarized toward an inflammatory, memory and plasmablast/plasma cell phenotype, with differential up-regulation of specific proinflammatory pathways. We did not find evidence that CNS B cells harbor a neurotropic virus. These data support the targeting of activated resident B cells in the CNS as a potentially effective strategy for control of treatment-resistant chronic disease. Central nervous system B cells have several potential roles in multiple sclerosis (MS): secretors of proinflammatory cytokines and chemokines, presenters of autoantigens to T cells, producers of pathogenic antibodies, and reservoirs for viruses that trigger demyelination. To interrogate these roles, single-cell RNA sequencing (scRNA-Seq) was performed on paired cerebrospinal fluid (CSF) and blood from subjects with relapsing-remitting MS (RRMS; n = 12), other neurologic diseases (ONDs; n = 1), and healthy controls (HCs; n = 3). Single-cell immunoglobulin sequencing (scIg-Seq) was performed on a subset of these subjects and additional RRMS (n = 4), clinically isolated syndrome (n = 2), and OND (n = 2) subjects. Further, paired CSF and blood B cell subsets (RRMS; n = 7) were isolated using fluorescence activated cell sorting for bulk RNA sequencing (RNA-Seq). Independent analyses across technologies demonstrated that nuclear factor kappa B (NF-\\u03baB) and cholesterol biosynthesis pathways were activated, and specific cytokine and chemokine receptors were up-regulated in CSF memory B cells. Further, SMAD/TGF-\\u03b21 signaling was down-regulated in CSF plasmablasts/plasma cells. Clonally expanded, somatically hypermutated IgM+ and IgG1+ CSF B cells were associated with inflammation, blood\\u2013brain barrier breakdown, and intrathecal Ig synthesis. While we identified memory B cells and plasmablast/plasma cells with highly similar Ig heavy-chain sequences across MS subjects, similarities were also identified with ONDs and HCs. No viral transcripts, including from Epstein\\u2013Barr virus, were detected. Our findings support the hypothesis that in MS, CSF B cells are driven to an inflammatory and clonally expanded memory and plasmablast/plasma cell phenotype.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1184&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3f19abe85bb5006bce5ff29bda7c3e9da088f7a3\",\n",
      "        \"title\": \"Phenotypic analysis of the unstimulated in vivo HIV CD4 T cell reservoir\",\n",
      "        \"abstract\": \"The latent reservoir is a major barrier to HIV cure. As latently infected cells cannot be phenotyped directly, the features of the in vivo reservoir have remained elusive. Here, we describe a method that leverages high-dimensional phenotyping using CyTOF to trace latently infected cells reactivated ex vivo to their original pre-activation states. Our results suggest that, contrary to common assumptions, the reservoir is not randomly distributed among cell subsets, and is remarkably conserved between individuals. However, reservoir composition differs between tissues and blood, as do cells successfully reactivated by different latency reversing agents. By selecting 8\\u201310 of our 39 original CyTOF markers, we were able to isolate highly purified populations of unstimulated in vivo latent cells. These purified populations were highly enriched for replication-competent and intact provirus, transcribed HIV, and displayed clonal expansion. The ability to isolate unstimulated latent cells from infected individuals enables previously impossible studies on HIV persistence.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1185&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc76ead97c25c12cd302af11abd1045c7627cf1b\",\n",
      "        \"title\": \"Osteopontin Expression Identifies a Subset of Recruited Macrophages Distinct from Kupffer Cells in the Fatty Liver\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1186&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"91d6d632dec25b2280466d0f01cede51eabc2663\",\n",
      "        \"title\": \"Holding our breath: the promise of tissue-resident memory T cells in lung cancer\",\n",
      "        \"abstract\": \"T cell memory is critical in controlling infection and plays an important role in anti-tumor responses in solid cancers. While effector memory and central memory T cells circulate and patrol non-lymphoid and lymphoid organs respectively, tissue resident memory T cells (TRM) permanently reside in tissues and provide local protective immune responses. In a number of solid tumors, tumor-specific T cell memory responses likely play an important role in keeping tumors in check, limiting cancer cell dissemination and reducing risk of relapse. In non-small cell lung cancer (NSCLC), a subset of tumor infiltrating lymphocytes (TILs) display phenotypic and functional characteristics associated with lung TRM (TRM-like TILs), including the expression of tissue-specific homing molecules and immune exhaustion markers. High infiltration of TRM-like TILs correlates with better survival outcomes for lung cancer patients, indicating that TRM-like TILs may contribute to anti-tumor responses. However, a number of TRM-like TILs do not display tumor specificity and the exact role of TRM-like TILs in mediating anti-tumor response in lung cancer is unclear. Here we review the characteristics of TRM-like TILs in lung cancer, the role these cells play in mediating anti-tumor immunity and the therapeutic implications of TRM-like TILs in the use and development of immunotherapy for lung cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1187&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0feb1ddef24db84dc776068181fc6660762b6c95\",\n",
      "        \"title\": \"AbSeq Protocol Using the Nano-Well Cartridge-Based Rhapsody Platform to Generate Protein and Transcript Expression Data on the Single-Cell Level\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1188&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"43eec04b643eb760adb3c1755271aa32311cf5c9\",\n",
      "        \"title\": \"Epigenetic reprogramming of plasmacytoid dendritic cells drives type I interferon-dependent differentiation of acute myeloid leukemias for therapeutic benefit\",\n",
      "        \"abstract\": \"Pharmacological inhibition of epigenetic enzymes can have therapeutic benefit, particularly against hematological malignancies. While these agents can affect tumor cell growth and proliferation, recent studies have demonstrated that pharmacological de-regulation of epigenetic modifiers may additionally mediate anti-tumor immune responses. Here we discovered a novel mechanism of immune regulation through the inhibition of histone deacetylases (HDACs). In a genetically engineered model of t(8;21) AML, leukemia cell differentiation and therapeutic benefit mediated by the HDAC inhibitor panobinostat required activation of the type I interferon (IFN) signaling pathway. Plasmacytoid dendritic cells (pDCs) were identified as the cells producing type I IFN in response to panobinostat, through transcriptional activation of IFN genes concomitant with increased H3K27 acetylation at these loci. Depletion of pDCs abrogated panobinostat-mediated activation of type I IFN signaling in leukemia cells and impaired therapeutic efficacy, while combined treatment of panobinostat and recombinant IFN\\u03b1 improved therapeutic outcomes. These discoveries offer a new therapeutic approach for t(8;21) AML and demonstrate that epigenetic rewiring of pDCs enhances anti-tumor immunity, opening the possibility of exploiting this cell type as a new target for immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1189&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7491d2a4fb18a180719a700aeb01b595dfb7cd5a\",\n",
      "        \"title\": \"Genomic Cytometry Editorial\",\n",
      "        \"abstract\": \"GENOMIC Cytometry is a newly emerging field of single cell analysis that utilizes highly sensitive genomic methods to deeply characterize individual cells. Just as the field of fluorescent cytometry uses fluorochromes and mass cytometry utilizes metal isotope tagged antibodies as reporters; the field of genomic cytometry uses nucleotide sequences to characterize individual cells. Since the inception of cytometry in the late 1960s, it has been clear that the ability to analyze biology at single-cell resolution is critical to the understanding of development, disease, and homeostasis. While Conventional cytometry techniques have given us the ability to interrogate individual cells, they have generally been limited to a handful of parameters. Recent expansions in technical capabilities in the high dimensional fluorescence and mass cytometry space have increased our capability to probe multiple analytes simultaneously from each cell. However, even in ultra-high dimensional studies, this capability usually tops out around the 40-parameter mark. Conversely with Genomic cytometry, it is possible to:\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1190&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e58a7f28f86b6689128368a6312ccad03aa79a71\",\n",
      "        \"title\": \"Molecular tracking devices quantify antigen distribution and archiving in the lymph node\",\n",
      "        \"abstract\": \"Live, attenuated vaccines generate humoral and cellular immune memory, increasing the duration of protective immune memory. We previously found that antigens derived from vaccination or viral infection persist within lymphatic endothelial cells (LECs) beyond the clearance of the infection, a process we termed \\u201cantigen archiving\\u201d. Technical limitations of fluorescent labeling have precluded a complete picture of antigen archiving across cell types in the lymph node. We developed a \\u201cmolecular tracking device\\u201d to follow the distribution, acquisition, and retention of antigen in the lymph node. We immunized mice with an antigen conjugated to a nuclease-resistant DNA tag and used single-cell mRNA sequencing to quantify its abundance in lymph node hematopoietic and non-hematopoietic cell types. At early and late time points after vaccination we found antigen acquisition by dendritic cell populations (DCs), associated expression of genes involved in DC activation and antigen processing, and antigen acquisition and archiving by LECs as well as unexpected stromal cell types. Variable antigen levels in LECs enabled the identification of caveolar endocytosis as a mechanism of antigen acquisition or retention. Molecular tracking devices enable new approaches to study dynamic tissue dissemination of antigens and identify new mechanisms of antigen acquisition and retention at cellular resolution in vivo.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1191&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"19e180531c29e35e950af4cabad69c0bcc5b8fff\",\n",
      "        \"title\": \"Dynamics of Cardiac Neutrophil Diversity in Murine Myocardial Infarction\",\n",
      "        \"abstract\": \"Supplemental Digital Content is available in the text. Rationale: After myocardial infarction, neutrophils rapidly and massively infiltrate the heart, where they promote both tissue healing and damage. Objective: To characterize the dynamics of circulating and cardiac neutrophil diversity after infarction. Methods and results: We employed single-cell transcriptomics combined with cell surface epitope detection by sequencing to investigate temporal neutrophil diversity in the blood and heart after murine myocardial infarction. At day 1, 3, and 5 after infarction, cardiac Ly6G+ (lymphocyte antigen 6G) neutrophils could be delineated into 6 distinct clusters with specific time-dependent patterning and proportions. At day 1, neutrophils were characterized by a gene expression profile proximal to bone marrow neutrophils (Cd177, Lcn2, Fpr1), and putative activity of transcriptional regulators involved in hypoxic response (Hif1a) and emergency granulopoiesis (Cebpb). At 3 and 5 days, 2 major subsets of Siglecfhi (enriched for eg, Icam1 and Tnf) and Siglecflow (Slpi, Ifitm1) neutrophils were found. Cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) analysis in blood and heart revealed that while circulating neutrophils undergo a process of aging characterized by loss of surface CD62L and upregulation of Cxcr4, heart infiltrating neutrophils acquired a unique SiglecFhi signature. SiglecFhi neutrophils were absent from the bone marrow and spleen, indicating local acquisition of the SiglecFhi signature. Reducing the influx of blood neutrophils by anti-Ly6G treatment increased proportions of cardiac SiglecFhi neutrophils, suggesting accumulation of locally aged neutrophils. Computational analysis of ligand/receptor interactions revealed putative pathways mediating neutrophil to macrophage communication in the myocardium. Finally, SiglecFhi neutrophils were also found in atherosclerotic vessels, revealing that they arise across distinct contexts of cardiovascular inflammation. Conclusions: Altogether, our data provide a time-resolved census of neutrophil diversity and gene expression dynamics in the mouse blood and ischemic heart at the single-cell level, and reveal a process of local tissue specification of neutrophils in the ischemic heart characterized by the acquisition of a SiglecFhi signature.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1192&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fb623a4f0e08cdd65ca547fca9b31673a826d428\",\n",
      "        \"title\": \"Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1193&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f6d6c4d476a9c7dc268b0629098ba23c433de544\",\n",
      "        \"title\": \"Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1194&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4f76ef2ce0c9aa58f53c23941f92d128b188be37\",\n",
      "        \"title\": \"Evaluating spectral cytometry for immune profiling in viral disease\",\n",
      "        \"abstract\": \"In conventional fluorescence cytometry, each fluorophore present in a panel is measured in a target detector, through the use of wide band\\u2010pass optical filters. In contrast, spectral cytometry uses a large number of detectors with narrow band\\u2010pass filters to measure a fluorophore's signal across the spectrum, creating a more detailed fluorescent signature for each fluorophore. The spectral approach shows promise in adding flexibility to panel design and improving the measurement of fluorescent signal. However, few comparisons between conventional and spectral systems have been reported to date. We therefore sought to compare a modern conventional cytometry system with a modern spectral system, and to assess the quality of resulting datasets from the point of view of a flow cytometry user. Signal intensity, spread, and resolution were compared between the systems. Subsequently, the different methods of separating fluorophore signals were compared, where compensation mathematically separates multiple overlapping fluorophores and unmixing relies on creating a detailed fluorescent signature across the spectrum to separate the fluorophores. Within the spectral data set, signal spread and resolution were comparable between compensation and unmixing. However, for some highly overlapping fluorophores, unmixing resolved the two fluorescence signals where compensation did not. Finally, data from mid\\u2010 to large\\u2010size panels were acquired and were found to have comparable resolution for many fluorophores on both instruments, but reduced levels of spreading error on our spectral system improved signal resolution for a number of fluorophores, compared with our conventional system. Furthermore, autofluorescence extraction on the spectral system allowed for greater population resolution in highly autofluorescent samples. Overall, the implementation of a spectral cytometry approach resulted in data that are comparable to that generated on conventional systems, with a number of potential advantages afforded by the larger number of detectors, and the integration of the spectral unmixing approach. \\u00a9 2020 International Society for Advancement of Cytometry\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1195&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f932c93a2c32e7eea1cd114c6ff07208e3d5af43\",\n",
      "        \"title\": \"Pinpointing Cell Identity in Time and Space\",\n",
      "        \"abstract\": \"Mammalian cells display a broad spectrum of phenotypes, morphologies, and functional niches within biological systems. Our understanding of mechanisms at the individual cellular level, and how cells function in concert to form tissues, organs and systems, has been greatly facilitated by centuries of extensive work to classify and characterize cell types. Classic histological approaches are now complemented with advanced single-cell sequencing and spatial transcriptomics for cell identity studies. Emerging data suggests that additional levels of information should be considered, including the subcellular spatial distribution of molecules such as RNA and protein, when classifying cells. In this Perspective piece we describe the importance of integrating cell transcriptional state with tissue and subcellular spatial and temporal information for thorough characterization of cell type and state. We refer to recent studies making use of single cell RNA-seq and/or image-based cell characterization, which highlight a need for such in-depth characterization of cell populations. We also describe the advances required in experimental, imaging and analytical methods to address these questions. This Perspective concludes by framing this argument in the context of projects such as the Human Cell Atlas, and related fields of cancer research and developmental biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1196&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21aaa73d0d481c69722e510dc2bc3b14cb7fe0d2\",\n",
      "        \"title\": \"Genomic Cytometry and New Modalities for Deep Single\\u2010Cell Interrogation\",\n",
      "        \"abstract\": \"In the past few years, the rapid development of single\\u2010cell analysis techniques has allowed for increasingly in\\u2010depth analysis of DNA, RNA, protein, and epigenetic states, at the level of the individual cell. This unprecedented characterization ability has been enabled through the combination of cytometry, microfluidics, genomics, and informatics. Although traditionally discrete, when properly integrated, these fields create the synergistic field of Genomic Cytometry. In this review, we look at the individual methods that together gave rise to the broad field of Genomic Cytometry. We further outline the basic concepts that drive the field and provide a framework to understand this increasingly complex, technology\\u2010intensive space. Thus, we introduce Genomic Cytometry as an emerging field and propose that synergistic rationalization of disparate modalities of cytometry, microfluidics, genomics, and informatics under one banner will enable massive leaps forward in the understanding of complex biology. \\u00a9 2020 International Society for Advancement of Cytometry\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1197&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"071b4c289f757d21d6039b40532824f3ee9399e6\",\n",
      "        \"title\": \"Single cell analysis of signaling proteins provides insights into pro-apoptotic properties of anti-cancer drugs.\",\n",
      "        \"abstract\": \"Single cell DNA analysis technology has provided unprecedented insights into many physiological and pathological processes. In contrast, technologies that allow protein analysis in single cells have lagged behind. Herein, a method called single cell Plasmonic ImmunoSandwich Assay (scPISA) that is capable of measuring signaling proteins and protein complexes in single living cells is described. scPISA is straightforward, comprising specific in-cell extraction and ultrasensitive plasmonic detection. It is applied to evaluate the efficacy and kinetics of cytotoxic drugs. It reveals that different drugs exhibit distinct pro-apoptotic properties at the single cell level. A set of new parameters is thus proposed for comprehensive and quantitative evaluation of the efficacy of anti-cancer drugs. It discloses that metformin can dramatically enhance the overall anti-cancer efficacy when combined with actinomycin D, though it itself is significantly less effective. Furthermore, scPISA reveals that survivin interacts with cytochrome C and caspase-3 in a dynamic fashion in single cells during continuous drug treatment. As compared with conventional assays, scPISA exhibits several significant advantages, such as ultra-high sensitivity, single-cell resolution, fast speed, and so on. Therefore, this approach may provide a powerful tool for wide important applications from basic research to clinical applications, particularly precision medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1198&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c3d1426106411b5f54c67c289efccfa9827d0d5d\",\n",
      "        \"title\": \"Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) revolutionised our understanding of disease biology and presented the promise of transforming translational research. We developed Besca, a toolkit that streamlines scRNA-seq analyses according to current best practices. A standard workflow covers quality control, filtering, and clustering. Two complementary Besca modules, utilizing hierarchical cell signatures or supervised machine learning, automate cell annotation and provide harmonised nomenclatures across studies. Subsequently, Besca enables estimation of cell type proportions in bulk transcriptomics studies. Using multiple heterogeneous scRNA-seq datasets we show how Besca aids acceleration, interoperability, reusability, and interpretability of scRNA-seq data analysis, crucial aspects in translational research and beyond.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1199&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c57d316fcba254c83352b14d3cc03415247b9ba9\",\n",
      "        \"title\": \"Dynamic single-cell phenotyping of immune cells using the microfluidic platform DropMap\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1200&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"465bf6c4b06951f74fede358542b39b34c9ba7f4\",\n",
      "        \"title\": \"Fluorescent Droplet Cytometry for On-Cell Phenotype Tracking.\",\n",
      "        \"abstract\": \"Profiling the heterogeneous phenotypes of live cancer cells is a key capability that requires single-cell analysis. However, acquiring information at the single cell level for live cancer cells is challenging due to low abundance of cells available for study. Here, we report single-cell analysis enabled by fluorescent-droplet cytometry (FDC), an approach that uses a biomarker-specific enzymatic fluorescent assay carried out using a droplet microfluidic platform. FDC utilizes DNA-functionalized antibodies in droplets to achieve specific on-cell target detection and enables the single-cell resolution char-acterization and profiling of live cancer cells based on their surface phenotype expressions. Using this approach, we achieve live-cell phenotype profiling of single and multi-surface markers acquired with small (< 40 cells) collections of cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1201&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f75788b9a0afc09cf3856bc6d23b36c34de065f\",\n",
      "        \"title\": \"Pathogen-induced tissue-resident memory TH17 (TRM17) cells amplify autoimmune kidney disease\",\n",
      "        \"abstract\": \"CD4+ TRM17 cells become activated by inflammatory cytokines to produce IL-17A and thereby promote autoimmunity. Autoimmunity promoter Tissue-resident memory T (TRM) cells are involved in peripheral immunity against reinfection, but their role in autoimmunity is unclear. Krebs et al. examine the contribution of TRM cells in patients with antineutrophil cytoplasmic antibody (ANCA)\\u2013dependent glomerulonephritis (GN). They identified multiple T cell subsets in healthy kidney tissue biopsies, but a marked increase in CD4+ TRM cells was seen in kidney biopsies from patients with ANCA-GN. They infected mice with Staphylococcus aureus, which induced renal TH17 cells that had a TRM cell phenotype and persisted in kidney tissue. In a mouse model of crescentic GN, S. aureus infection aggravated kidney pathology and appeared to drive localized renal autoimmune responses. These findings provide critical insight into the role of CD4+ TRM cells in contributing to autoimmune disease. Although it is well established that microbial infections predispose to autoimmune diseases, the underlying mechanisms remain poorly understood. After infection, tissue-resident memory T (TRM) cells persist in peripheral organs and provide immune protection against reinfection. However, whether TRM cells participate in responses unrelated to the primary infection, such as autoimmune inflammation, is unknown. By using high-dimensional single-cell analysis, we identified CD4+ TRM cells with a TH17 signature (termed TRM17 cells) in kidneys of patients with ANCA-associated glomerulonephritis. Experimental models demonstrated that renal TRM17 cells were induced by pathogens infecting the kidney, such as Staphylococcus aureus, Candida albicans, and uropathogenic Escherichia coli, and persisted after the clearance of infections. Upon induction of experimental glomerulonephritis, these kidney TRM17 cells rapidly responded to local proinflammatory cytokines by producing IL-17A and thereby exacerbate renal pathology. Thus, our data show that pathogen-induced TRM17 cells have a previously unrecognized function in aggravating autoimmune disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1202&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"733d50d10da149c8f832e511b5fef9d8c886cf40\",\n",
      "        \"title\": \"Integrated Skin Transcriptomics and Serum Multiplex Assays Reveal Novel Mechanisms of Wound Healing in Diabetic Foot Ulcers\",\n",
      "        \"abstract\": \"Nonhealing diabetic foot ulcers (DFUs) are characterized by low-grade chronic inflammation, both locally and systemically. We prospectively followed a group of patients who either healed or developed nonhealing chronic DFUs. Serum and forearm skin analysis, both at the protein expression and the transcriptomic level, indicated that increased expression of factors such as interferon-\\u03b3 (IFN-\\u03b3), vascular endothelial growth factor, and soluble vascular cell adhesion molecule-1 were associated with DFU healing. Furthermore, foot skin single-cell RNA sequencing analysis showed multiple fibroblast cell clusters and increased inflammation in the dorsal skin of patients with diabetes mellitus (DM) and DFU specimens compared with control subjects. In addition, in myeloid cell DM and DFU upstream regulator analysis, we observed inhibition of interleukin-13 and IFN-\\u03b3 and dysregulation of biological processes that included cell movement of monocytes, migration of dendritic cells, and chemotaxis of antigen-presenting cells pointing to an impaired migratory profile of immune cells in DM skin. The SLCO2A1 and CYP1A1 genes, which were upregulated at the forearm of nonhealers, were mainly expressed by the vascular endothelial cell cluster almost exclusively in DFU, indicating a potential important role in wound healing. These results from integrated protein and transcriptome analyses identified individual genes and pathways that can potentially be targeted for enhancing DFU healing.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1203&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"050b52bc707cd5e3a3d3c5c0738fbbb303943e93\",\n",
      "        \"title\": \"The Application of Single-Cell RNA Sequencing in Vaccinology\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing allows highly detailed profiling of cellular immune responses from limited-volume samples, advancing prospects of a new era of systems immunology. The power of single-cell RNA sequencing offers various opportunities to decipher the immune response to infectious diseases and vaccines. Here, we describe the potential uses of single-cell RNA sequencing methods in prophylactic vaccine development, concentrating on infectious diseases including COVID-19. Using examples from several diseases, we review how single-cell RNA sequencing has been used to evaluate the immunological response to different vaccine platforms and regimens. By highlighting published and unpublished single-cell RNA sequencing studies relevant to vaccinology, we discuss some general considerations how the field could be enriched with the widespread adoption of this technology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1204&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"87f93f6fedf881ed6c58967eb5389b9d9b29cc64\",\n",
      "        \"title\": \"Single-cell metabolic profiling of human cytotoxic T cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1205&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dc8686715a5a7da8ef9ecb993c5422d37391c448\",\n",
      "        \"title\": \"Single\\u2010cell proteomic analysis\",\n",
      "        \"abstract\": \"The ability to comprehensively profile proteins in every individual cell of complex biological systems is crucial to advance our understanding of normal physiology and disease pathogenesis. Conventional bulk cell experiments mask the cell heterogeneity in the population, while the single\\u2010cell imaging methods suffer from the limited multiplexing capacities. Recent advances in microchip\\u2010, mass spectrometry\\u2010, and reiterative staining\\u2010based technologies have enabled comprehensive protein profiling in single cells. These approaches will bring new insights into a variety of biological and biomedical fields, such as signaling network regulation, cell heterogeneity, tissue architecture, disease diagnosis, and treatment monitoring. In this article, we will review the recent advances in the development of single\\u2010cell proteomic technologies, describe their advantages, discuss the current limitations and challenges, and propose potential solutions. We will also highlight the wide applications of these technologies in biology and medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1206&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d700c251e3f22d30e91c112f47985345a62b0e5b\",\n",
      "        \"title\": \"The Power of Single\\u2010Cell Analysis for the Study of Liver Pathobiology\",\n",
      "        \"abstract\": \"Single cell transcriptomics has emerged as a powerful lens through which to study the molecular diversity of complex tissues such as the liver, during health and disease, both in animal models and in humans. The earliest gene expression methods measured bulk tissue RNA, but the results were often confusing because they derived from the combined transcriptomes of many different cell types in unknown proportions. To better delineate cell\\u2010type\\u2010specific expression, investigators developed cell isolation, purification, and sorting protocols, yet still, the RNA derived from ensembles of cells obscured recognition of cellular heterogeneity. Profiling transcriptomes at the single\\u2010cell level has opened the door to analyses that were not possible in the past. In this review, we discuss the evolution of single cell transcriptomics and how it has been applied for the study of liver physiology and pathobiology to date.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1207&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b5b401f7254bc9add8bd8484ab305444bd3cdccb\",\n",
      "        \"title\": \"Coupled scRNA-Seq and Intracellular Protein Activity Reveal an Immunosuppressive Role of TREM2 in Cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1208&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"65186f1e480cb0357f08dd80e64e7b98584c891e\",\n",
      "        \"title\": \"Myeloid Cells TREM Down Anti-tumor Responses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1209&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9873f7955a651dcf27bcd485814982fa31dd0c71\",\n",
      "        \"title\": \"Mass cytometry and type 1 diabetes research in the age of single-cell data science.\",\n",
      "        \"abstract\": \"PURPOSE OF REVIEW\\nNew single-cell tec. hnologies developed over the past decade have considerably reshaped the biomedical research landscape, and more recently have found their way into studies probing the pathogenesis of type 1 diabetes (T1D). In this context, the emergence of mass cytometry in 2009 revolutionized immunological research in two fundamental ways that also affect the T1D world: first, its ready embrace by the community and rapid dissemination across academic and private science centers alike established a new standard of analytical complexity for the high-dimensional proteomic stratification of single-cell populations; and second, the somewhat unexpected arrival of mass cytometry awoke the flow cytometry field from its seeming sleeping beauty stupor and precipitated substantial technological advances that by now approach a degree of analytical dimensionality comparable to mass cytometry.\\n\\n\\nRECENT FINDINGS\\nHere, we summarize in detail how mass cytometry has thus far been harnessed for the pursuit of discovery studies in T1D science; we provide a succinct overview of other single-cell analysis platforms that already have been or soon will be integrated into various T1D investigations; and we briefly consider how effective adoption of these technologies requires an adjusted model for expense allocation, prioritization of experimental questions, division of labor, and recognition of scientific contributions.\\n\\n\\nSUMMARY\\nThe introduction of contemporary single-cell technologies in general, and of mass cytometry, in particular, provides important new opportunities for current and future T1D research; the necessary reconfiguration of research strategies to accommodate implementation of these technologies, however, may both broaden research endeavors by fostering genuine team science, and constrain their actual practice because of the need for considerable investments into infrastructure and technical expertise.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1210&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dc1c43ba1d7061ef49d6525c51e326a25452176d\",\n",
      "        \"title\": \"GMM-Demux: sample demultiplexing, multiplet detection, experiment planning, and novel cell-type verification in single cell sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1211&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd0b227d124e40ddb5f1266c4762ed86b2dfe9b1\",\n",
      "        \"title\": \"Status and Potential of Single\\u2010Cell Transcriptomics for Understanding Plant Development and Functional Biology\",\n",
      "        \"abstract\": \"The advent of modern \\u201comics\\u201d technologies (genomics, transcriptomics, proteomics, and metabolomics) are attributed to innovative breakthroughs in genome sequencing, bioinformatics, and analytic tools. An organism's biological structure and function is the result of the concerted action of single cells in different tissues. Single cell genomics has emerged as a ground\\u2010breaking technology that has greatly enhanced our understanding of the complexity of gene expression at a microscopic resolution and holds the potential to revolutionize the way we characterize complex cell assemblies and study their spatial organization, dynamics, clonal distribution, pathways, function, and networking. Mammalian systems have benefitted immensely from these approaches to dissect complex systems such as cancer, immunological disorders, epigenetic controls of diseases, and understanding of developmental biology. However, the applications of single\\u2010cell omics in plant research are just starting. The potential to decipher the fundamentals of developmental and functional biology of large and complex plant species at the single\\u2010cell resolution are now becoming important drivers of research. In this review, we present the status, challenges and potential of one important and most commonly used single\\u2010cell omics technique in plants, namely single cell transcriptomics. \\u00a9 2020 International Society for Advancement of Cytometry\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1212&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"678270592111e9f780b1b08add1ed1c8ce633c06\",\n",
      "        \"title\": \"mRNAs, proteins and the emerging principles of gene expression control\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1213&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"70a880b0a679fdee688a45884b0f6a1e785fdb64\",\n",
      "        \"title\": \"Cell-ID: gene signature extraction and cell identity recognition at individual cell level\",\n",
      "        \"abstract\": \"The exhaustive exploration of human cell heterogeneity requires the unbiased identification of molecular signatures that can serve as unique cell identity cards for every cell in the body. However, the stochasticity associated with high-throughput single-cell sequencing has made it necessary to use clustering-based computational approaches in which the characterization of cell-type heterogeneity is performed at cell-subpopulation level rather than at full single-cell resolution. We present here Cell-ID, a clustering-free multivariate statistical method for the robust extraction of per-cell gene signatures from single-cell sequencing data. Cell-ID signatures allow unbiased cell identity recognition across different donors, tissues-of-origin, model organisms and single-cell omics technologies. Cell-ID is distributed as an open-source R software package: https://github.com/RausellLab/CelliD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1214&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e5520ae1a596de6ec02f5e297f04010ee5ae1341\",\n",
      "        \"title\": \"Unraveling the Complexity of the Cancer Microenvironment With Multidimensional Genomic and Cytometric Technologies\",\n",
      "        \"abstract\": \"Cancers are characterized by extensive heterogeneity that occurs intratumorally, between lesions, and across patients. To study cancer as a complex biological system, multidimensional analyses of the tumor microenvironment are paramount. Single-cell technologies such as flow cytometry, mass cytometry, or single-cell RNA-sequencing have revolutionized our ability to characterize individual cells in great detail and, with that, shed light on the complexity of cancer microenvironments. However, a key limitation of these single-cell technologies is the lack of information on spatial context and multicellular interactions. Investigating spatial contexts of cells requires the incorporation of tissue-based techniques such as multiparameter immunofluorescence, imaging mass cytometry, or in situ detection of transcripts. In this Review, we describe the rise of multidimensional single-cell technologies and provide an overview of their strengths and weaknesses. In addition, we discuss the integration of transcriptomic, genomic, epigenomic, proteomic, and spatially-resolved data in the context of human cancers. Lastly, we will deliberate on how the integration of multi-omics data will help to shed light on the complex role of cell types present within the human tumor microenvironment, and how such system-wide approaches may pave the way toward more effective therapies for the treatment of cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1215&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"24d9cf39f218b3857ece4d7867c93e888d1d414e\",\n",
      "        \"title\": \"Single-cell multi-omics reveals dyssynchrony of the innate and adaptive immune system in progressive COVID-19\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1216&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"37a0ca4508e47358fee44b54a1d0cd31100107cd\",\n",
      "        \"title\": \"CITEseq analysis of non-small-cell lung cancer lesions reveals an axis of immune cell activation associated with tumor antigen load and TP53 mutations\",\n",
      "        \"abstract\": \"Immunotherapy is becoming a mainstay in the treatment of NSCLC. While tumor mutational burden (TMB) has been shown to correlate with response to immunotherapy, little is known about the relation of the baseline immune response with the tumor genotype. Here, we profiled 35 early stage NSCLC lesions using multiscale single cell sequencing. Unsupervised clustering identified in a subset of patients a key cellular module consisting of PDCD1+ CXCL13+ activated T cells, IgG+ plasma cells, and SPP1+ macrophages, referred to as the lung cancer activation module (LCAMhi). Transcriptional data from two NSCLC cohorts confirmed a subset of patients with LCAMhi enrichment, which was independent of overall immune cell content. The LCAMhi module strongly correlated with TMB, expression of cancer testis antigens, and with TP53 mutations in smokers and non-smokers. These data establish LCAM as a key mode of immune cell activation associated with high tumor antigen load and driver mutations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1217&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c52ae883344ba9311b2fd44924c08a912027a158\",\n",
      "        \"title\": \"Unraveling Heterogeneity in Transcriptome and Its Regulation Through Single-Cell Multi-Omics Technologies\",\n",
      "        \"abstract\": \"Cellular heterogeneity plays a pivotal role in tissue homeostasis and the disease development of multicellular organisms. To deconstruct the heterogeneity, a multitude of single-cell toolkits measuring various cellular contents, including genome, transcriptome, epigenome, and proteome, have been developed. More recently, multi-omics single-cell techniques enable the capture of molecular footprints with a higher resolution by simultaneously profiling various cellular contents within an individual cell. Integrative analysis of multi-omics datasets unravels the relationships between cellular modalities, builds sophisticated regulatory networks, and provides a holistic view of the cell state. In this review, we summarize the major developments in the single-cell field and review the current state-of-the-art single-cell multi-omic techniques and the bioinformatic tools for integrative analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1218&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"83355da31e5dbab879a69a5f303f4fe585f474b6\",\n",
      "        \"title\": \"Meta-Analysis of Leukocyte Diversity in Atherosclerotic Mouse Aortas\",\n",
      "        \"abstract\": \"Supplemental Digital Content is available in the text. The diverse leukocyte infiltrate in atherosclerotic mouse aortas was recently analyzed in 9 single-cell RNA sequencing and 2 mass cytometry studies. In a comprehensive meta-analysis, we confirm 4 known macrophage subsets\\u2014resident, inflammatory, interferon-inducible cell, and Trem2 (triggering receptor expressed on myeloid cells-2) foamy macrophages\\u2014and identify a new macrophage subset resembling cavity macrophages. We also find that monocytes, neutrophils, dendritic cells, natural killer cells, innate lymphoid cells-2, and CD (cluster of differentiation)-8 T cells form prominent and separate immune cell populations in atherosclerotic aortas. Many CD4 T cells express IL (interleukin)-17 and the chemokine receptor CXCR (C-X-C chemokine receptor)-6. A small number of regulatory T cells and T helper 1 cells is also identified. Immature and naive T cells are present in both healthy and atherosclerotic aortas. Our meta-analysis overcomes limitations of individual studies that, because of their experimental approach, over- or underrepresent certain cell populations. Mass cytometry studies demonstrate that cell surface phenotype provides valuable information beyond the cell transcriptomes. The present analysis helps resolve some long-standing controversies in the field. First, Trem2+ foamy macrophages are not proinflammatory but interferon-inducible cell and inflammatory macrophages are. Second, about half of all foam cells are smooth muscle cell-derived, retaining smooth muscle cell transcripts rather than transdifferentiating to macrophages. Third, Pf4, which had been considered specific for platelets and megakaryocytes, is also prominently expressed in the main population of resident vascular macrophages. Fourth, a new type of resident macrophage shares transcripts with cavity macrophages. Finally, the discovery of a prominent innate lymphoid cell-2 cluster links the single-cell RNA sequencing work to recent flow cytometry data suggesting a strong atheroprotective role of innate lymphoid cells-2. This resolves apparent discrepancies regarding the role of T helper 2 cells in atherosclerosis based on studies that predated the discovery of innate lymphoid cells-2 cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1219&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"817ebb2f01d01895a7ee59e6d1353e19db1e964d\",\n",
      "        \"title\": \"Multi-modal single cell analysis reveals brain immune landscape plasticity during aging and gut microbiota dysbiosis\",\n",
      "        \"abstract\": \"The brain contains a diverse array of immune cell types. The phenotypic and functional plasticity of brain immune cells collectively contribute to brain tissue homeostasis and disease progression. Immune cell plasticity is profoundly influenced by local tissue microenvironment cues and systemic factors. Yet, the transcriptional mechanism by which systemic stimuli, such as aging and gut microbiota dysbiosis, reshape brain immune cell plasticity and homeostasis has not been fully delineated. Using Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq), we analyzed compositional and transcriptional changes of the brain immune landscape in response to aging and gut dysbiosis. We first examined the discordance between canonical surface marker-defined immune cell types (Cell-ID) and their transcriptome signatures, which suggested transcriptional plasticity among immune cells despite sharing the same cell surface markers. Specifically, inflammatory and patrolling Ly6C+ monocytes were shifted predominantly to a pro-inflammatory transcriptional program in the aged brain, while brain ILCs shifted toward an ILC2 transcriptional profile. Finally, aging led to an increase of ILC-like cells expressing a T memory stemness (Tscm) signature in the brain. Antibiotics (ABX)-induced gut dysbiosis reduced the frequency of ILCs exhibiting Tscm-like properties in the aged mice, but not in the young mice. Enabled by high-resolution single-cell molecular phenotyping, our study revealed that systemic changes due to aging and gut dysbiosis prime the brain environment for an increased propensity for neuroinflammation, which provided insights into gut dysbiosis in age-related neurological diseases. Manuscript Summary Golomb et al. performed Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) on immune cells from the brains of young and aged mice with and without antibiotics-induced gut dysbiosis. High resolution, single cell immunophenotyping enabled the dissection of extensive transcriptional plasticity of canonically identified monocytes and innate lymphoid cells (ILCs) in the aged brain. Through differential gene expression and trajectory inference analyses, the authors revealed tissue microenvironment-dependent cellular responses influenced by aging and gut dysbiosis that may potentiate neuroinflammatory diseases. Graphical Abstract\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1220&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7f90939656b4291602db13f90dcb96f3401ed97a\",\n",
      "        \"title\": \"Query to reference single-cell integration with transfer learning\",\n",
      "        \"abstract\": \"Large single-cell atlases are now routinely generated with the aim of serving as reference to analyse future smaller-scale studies. Yet, learning from reference data is complicated by batch effects between datasets, limited availability of computational resources, and sharing restrictions on raw data. Leveraging advances in machine learning, we propose a deep learning strategy to map query datasets on top of a reference called single-cell architectural surgery (scArches, https://github.com/theislab/scarches). It uses transfer learning and parameter optimization to enable efficient, decentralized, iterative reference building, and the contextualization of new datasets with existing references without sharing raw data. Using examples from mouse brain, pancreas, and whole organism atlases, we showcase that scArches preserves nuanced biological state information while removing batch effects in the data, despite using four orders of magnitude fewer parameters compared to de novo integration. To demonstrate mapping disease variation, we show that scArches preserves detailed COVID-19 disease variation upon reference mapping, enabling discovery of new cell identities that are unseen during training. We envision our method to facilitate collaborative projects by enabling the iterative construction, updating, sharing, and efficient use of reference atlases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1221&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8ee66f9f993e503d5db0b7dfd37c59a05af7ab5\",\n",
      "        \"title\": \"Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1222&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b758d753edff0ae4ea275b4977af1787758fc541\",\n",
      "        \"title\": \"DISC: a highly scalable and accurate inference of gene expression and structure for single-cell transcriptomes using semi-supervised deep learning\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1223&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d6aa7c0dcb1b31c8093026645531caabb0437f8\",\n",
      "        \"title\": \"Identification and characterization of adipose surface epitopes\",\n",
      "        \"abstract\": \"Adipose tissue is a central regulator of metabolism and an important pharmacological target to treat the metabolic consequences of obesity, such as insulin resistance and dyslipidemia. Among the various cellular compartments, the adipocyte cell surface is especially appealing as a drug target as it contains various proteins that when activated or inhibited promote adipocyte health, change its endocrine function and eventually maintain or restore whole-body insulin sensitivity. In addition, cell surface proteins are readily accessible by various drug classes. However, targeting individual cell surface proteins in adipocytes has been difficult due to important functions of these proteins outside adipose tissue, raising various safety concerns. Thus, one of the biggest challenges is the lack of adipose selective surface proteins and/or targeting reagents. Here, we discuss several receptor families with an important function in adipogenesis and mature adipocytes to highlight the complexity at the cell surface and illustrate the problems with identifying adipose selective proteins. We then discuss that, while no unique adipocyte surface protein might exist, how splicing, posttranslational modifications as well as protein/protein interactions can create enormous diversity at the cell surface that vastly expands the space of potentially unique epitopes and how these selective epitopes can be identified and targeted.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1224&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bb188d88b8acb45f48e8e53ea7267381c0b29a26\",\n",
      "        \"title\": \"Network Approaches for Dissecting the Immune System\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1225&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a6480f6c03dce65f503691d050269c217283589\",\n",
      "        \"title\": \"Hematopoietic Stem Cells in Health and Disease\\u2014Insights from Single-Cell Multi-omic Approaches\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1226&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dc14abf36a1203b7637ee9d8ac39e09cc4dca149\",\n",
      "        \"title\": \"Dissecting genetic determinants of variation in human immune responses.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1227&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b0ecf3c5b1664e9bb39f0e8bc7ca0b083ed8276f\",\n",
      "        \"title\": \"Multiplex profiling of serum proteins in solution using barcoded antibody fragments and next generation sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1228&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"50b7e4007a7eb6badcb26b01dd333d4c8f825a91\",\n",
      "        \"title\": \"Multiplex profiling of serum proteins in solution using barcoded antibody fragments and next generation sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1229&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0fdc00ed3faa61461bc49d48c23dac7a79a4e584\",\n",
      "        \"title\": \"Single cell analyses and machine learning define hematopoietic progenitor and HSC-like cells derived from human PSCs.\",\n",
      "        \"abstract\": \"Haematopoietic stem and progenitor cells (HSPCs) develop through distinct waves at various anatomical sites during embryonic development. The in vitro differentiation of human pluripotent stem cells (hPSCs) is able to recapitulate some of these processes, however, it has proven difficult to generate functional haematopoietic stem cells (HSCs). To define the dynamics and heterogeneity of HSPCs that can be generated in vitro from hPSCs, we exploited single cell RNA sequencing (scRNAseq) in combination with single cell protein expression analysis. Bioinformatics analyses and functional validation defined the transcriptomes of na\\u00efve progenitors as well as erythroid, megakaryocyte and leukocyte-committed progenitors and we identified CD44, CD326, ICAM2/CD9 and CD18 as markers of these progenitors, respectively. Using an artificial neural network (ANN), that we trained on a scRNAseq derived from human fetal liver, we were able to identify a wide range of hPSCs-derived HPSC phenotypes, including a small group classified as HSCs. This transient HSC-like population decreased as differentiation proceeded and was completely missing in the dataset that had been generated using cells selected on the basis of CD43expression. By comparing the single cell transcriptome of in vitro-generated HSC-like cells with those generated within the fetal liver we identified transcription factors and molecular pathways that can be exploited in the future to improve the in vitro production of HSCs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1230&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ac119e4480710580588b0a70dbd8b628c8e369e8\",\n",
      "        \"title\": \"Deciphering Cell Fate Decision by Integrated Single-Cell Sequencing Analysis.\",\n",
      "        \"abstract\": \"Cellular differentiation is a common underlying feature of all multicellular organisms through which na\\u00efve cells progressively become fate restricted and develop into mature cells with specialized functions. A comprehensive understanding of the regulatory mechanisms of cell fate choices during de- velopment, regeneration, homeostasis, and disease is a central goal of mod- ern biology. Ongoing rapid advances in single-cell biology are enabling the exploration of cell fate specification at unprecedented resolution. Here, we review single-cell RNA sequencing and sequencing of other modalities as methods to elucidate the molecular underpinnings of lineage specification. We specifically discuss how the computational tools available to reconstruct lineage trajectories, quantify cell fate bias, and perform dimensionality re- duction for data visualization are providing new mechanistic insights into the process of cell fate decision. Studying cellular differentiation using single- cell genomic tools is paving the way for a detailed understanding of cellular behavior in health and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1231&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1c82559c419737ae7e333089e7e65b5d8e322e82\",\n",
      "        \"title\": \"Single-cell analysis of developing B cells reveals dynamic gene expression networks that govern B cell development and transformation\",\n",
      "        \"abstract\": \"Integration of external signals and B-lymphoid transcription factor activities orchestrate B cell lineage commitment through alternating cycles of proliferation and differentiation, producing a diverse repertoire of mature B cells. We used single-cell transcriptomics and proteomics to characterize B cell development. Our analysis revealed unique transcriptional signatures that refine the pre-B cell expansion stages into novel pre-BCR-dependent and pre-BCR-independent proliferative phases. These changes correlate with unexpected dynamic and reciprocal changes in expression of the transcription factor EBF1 and the RNA binding protein YBX3, that are defining features of the pre-BCR-dependent stage. Using pseudotime analysis, we further characterize the expression kinetics of different biological modalities across B cell development, including transcription factors, cytokines, chemokines, and their associated receptors. Our findings reveal the underlying heterogeneity of developing B cells and point to key developmental nodes linked to B cell transformation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1232&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7c1d86823efb0a492a112b6bc7f30224fc031b3c\",\n",
      "        \"title\": \"The Impact of Single-Cell Genomics on Adipose Tissue Research\",\n",
      "        \"abstract\": \"Adipose tissue is an important regulator of whole-body metabolism and energy homeostasis. The unprecedented growth of obesity and metabolic disease worldwide has required paralleled advancements in research on this dynamic endocrine organ system. Single-cell RNA sequencing (scRNA-seq), a highly meticulous methodology used to dissect tissue heterogeneity through the transcriptional characterization of individual cells, is responsible for facilitating critical advancements in this area. The unique investigative capabilities achieved by the combination of nanotechnology, molecular biology, and informatics are expanding our understanding of adipose tissue\\u2019s composition and compartmentalized functional specialization, which underlie physiologic and pathogenic states, including adaptive thermogenesis, adipose tissue aging, and obesity. In this review, we will summarize the use of scRNA-seq and single-nuclei RNA-seq (snRNA-seq) in adipocyte biology and their applications to obesity and diabetes research in the hopes of increasing awareness of the capabilities of this technology and acting as a catalyst for its expanded use in further investigation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1233&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ccdc05d28aac416e2fc0e660bdb98a2f24f3d33d\",\n",
      "        \"title\": \"A Cancer Biologist's Primer on Machine Learning Applications in High-Dimensional Cytometry.\",\n",
      "        \"abstract\": \"The application of machine learning and artificial intelligence to high-dimensional cytometry data sets has increasingly become a staple of bioinformatic data analysis over the past decade. This is especially true in the field of cancer biology, where protocols for collecting multiparameter single-cell data in a high-throughput fashion are rapidly developed. As the use of machine learning methodology in cytometry becomes increasingly common, there is a need for cancer biologists to understand the basic theory and applications of a variety of algorithmic tools for analyzing and interpreting cytometry data. We introduce the reader to several keystone machine learning-based analytic approaches with an emphasis on defining key terms and introducing a conceptual framework for making translational or clinically relevant discoveries. The target audience consists of cancer cell biologists and physician-scientists interested in applying these tools to their own data, but who may have limited training in bioinformatics. \\u00a9 2020 International Society for Advancement of Cytometry.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1234&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"56f2af5de0f180b664f53b63d85e658322b7e79e\",\n",
      "        \"title\": \"Single-cell analysis of germinal-center B cells informs on lymphoma cell of origin and outcome\",\n",
      "        \"abstract\": \"Single-cell transcriptomic analysis of germinal center B cells identifies multiple linked populations, most of which represent cell of origin of lymphomas. The newly identified cell of origin of diffuse large B cell lymphoma informs on novel prognostic subgroups.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1235&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1e96c8e1208d29210b6b7435e490d62ed9f398c1\",\n",
      "        \"title\": \"Breathing fresh air into respiratory research with single-cell RNA sequencing\",\n",
      "        \"abstract\": \"The complex cellular heterogeneity of the lung poses a unique challenge to researchers in the field. While the use of bulk RNA sequencing has become a ubiquitous technology in systems biology, the technique necessarily averages out individual contributions to the overall transcriptional landscape of a tissue. Single-cell RNA sequencing (scRNA-seq) provides a robust, unbiased survey of the transcriptome comparable to bulk RNA sequencing while preserving information on cellular heterogeneity. In just a few years since this technology was developed, scRNA-seq has already been adopted widely in respiratory research and has contributed to impressive advancements such as the discoveries of the pulmonary ionocyte and of a profibrotic macrophage population in pulmonary fibrosis. In this review, we discuss general technical considerations when considering the use of scRNA-seq and examine how leading investigators have applied the technology to gain novel insights into respiratory biology, from development to disease. In addition, we discuss the evolution of single-cell technologies with a focus on spatial and multi-omics approaches that promise to drive continued innovation in respiratory research. Single-cell RNA sequencing is being used more and more in respiratory research. This technology can be leveraged in different ways in studies of lung development, structure and function, and lung diseases from pulmonary fibrosis to asthma to lung cancer. https://bit.ly/36TdvJT\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1236&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"46519bcd55a9f0adbf04ab68633e3baff978fe5c\",\n",
      "        \"title\": \"AutoSpill: A method for calculating spillover coefficients to compensate or unmix high-parameter flow cytometry data\",\n",
      "        \"abstract\": \"Compensating in classical flow cytometry or unmixing in spectral systems is an unavoidable challenge in the data analysis of fluorescence-based flow cytometry. In both cases, spillover coefficients are estimated for each fluorophore using single-color controls. This approach has remained essentially unchanged since its inception, and is increasingly limited in its ability to deal with high-parameter flow cytometry. Here, we present AutoSpill, a novel approach for calculating spillover coefficients or spectral signatures of fluorophores. The approach combines automated gating of cells, calculation of an initial spillover matrix based on robust linear regression, and iterative refinement to reduce error. Moreover, autofluorescence can be compensated out, by processing it as an endogenous dye in an unstained control. AutoSpill uses single-color controls and is compatible with common flow cytometry software, but it differs in two key aspects from current methods: (1) it is much less demanding in the preparation of controls, as it does not require the presence of well-defined positive and negative populations, and (2) it does not require manual tuning of the spillover matrix, as the algorithm iteratively computes the tuning, producing an optimal compensation matrix. Another algorithm, AutoSpread, complements this approach, providing a robust estimate of the Spillover Spreading Matrix (SSM), while avoiding the need for well-defined positive and negative populations. Together, AutoSpill and AutoSpread provide a superior solution to the problem of fluorophore spillover, allowing simpler and more robust workflows in high-parameter flow cytometry.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1237&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd23515152e7a0592d9feb02634a3dad810e0bd6\",\n",
      "        \"title\": \"Characterizing the molecular regulation of inhibitory immune checkpoints with multi-modal single-cell screens.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1238&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18de256aec607a83bec682e8eb891b03dbab413d\",\n",
      "        \"title\": \"The Atlas of the In Vivo HIV CD4 T Cell Reservoir\",\n",
      "        \"abstract\": \"The latent reservoir is a main barrier for curing HIV. But because latently-infected cells cannot be phenotyped directly, the features of the in vivo reservoir have remained elusive. Here, we describe a method that leverages high-dimensional phenotyping using CyTOF to trace latently-infected cells reactivated ex vivo to their original pre-activation states. Our results suggest that contrary to common assumptions, the reservoir is not randomly distributed among cell subsets, and is remarkably conserved between individuals. However, reservoir composition differs between tissues and blood, as do cells successfully reactivated by different latency reversing agents. Most importantly, by selecting 8-10 of our 39 original CyTOF markers, we were able to isolate highly purified populations of unstimulated in vivo latent cells, thereby validating the PP-SLIDE approach for reservoir characterization. These purified populations were highly enriched for replication-competent and intact provirus, transcribed HIV, and displayed clonal expansion. The ability to isolate unstimulated latent cells from infected individuals enables previously impossible studies of HIV persistence.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1239&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2f8b10adf2da758660b1782b8880b1345effd483\",\n",
      "        \"title\": \"Rapid single cell evaluation of human disease and disorder targets using REVEAL: SingleCell\\u2122\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1240&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"da6fd56fd35141ab24c986fd4282bf37eeeadf10\",\n",
      "        \"title\": \"The Comparison of Two Single-cell Sequencing Platforms: BD Rhapsody and 10x Genomics Chromium\",\n",
      "        \"abstract\": \"The cell is the unit of life for all organisms, and all cells are certainly not the same. So the technology to generate transcription expression or genomic DNA profiles from single cells is crucial. Since its establishment in 2009, single-cell RNA sequencing (scRNA-seq) has emerged as a major driver of progress in biomedical research. During the last three years, several new single-cell sequencing platforms have emerged. Yet there are only a few systematic comparisons of the advantages and limitations of these commonly used platforms. Here we compare two single-cell sequencing platforms: BD Rhapsody and 10x Genomics Chromium, including their different mechanisms and some scRNA-seq results obtained with them.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1241&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0c5ad704cd3bbd87ec106aba0276e38895afb043\",\n",
      "        \"title\": \"MRBLES 2.0: High-throughput generation of chemically functionalized spectrally and magnetically encoded hydrogel beads using a simple single-layer microfluidic device\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1242&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6e016b4868d7354dbaf6540e1e1df9f275e028b\",\n",
      "        \"title\": \"Infinity Flow: High-throughput single-cell quantification of 100s of proteins using conventional flow cytometry and machine learning\",\n",
      "        \"abstract\": \"Modern immunologic research increasingly requires high-dimensional analyses in order to understand the complex milieu of cell-types that comprise the tissue microenvironments of disease. To achieve this, we developed Infinity Flow combining hundreds of overlapping flow cytometry panels using machine learning to enable the simultaneous analysis of the co-expression patterns of 100s of surface-expressed proteins across millions of individual cells. In this study, we demonstrate that this approach allows the comprehensive analysis of the cellular constituency of the steady-state murine lung and to identify novel cellular heterogeneity in the lungs of melanoma metastasis bearing mice. We show that by using supervised machine learning, Infinity Flow enhances the accuracy and depth of clustering or dimensionality reduction algorithms. Infinity Flow is a highly scalable, low-cost and accessible solution to single cell proteomics in complex tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1243&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"23352958aee5d95805568db528b6de3eb1ed939c\",\n",
      "        \"title\": \"A pan-cancer blueprint of the heterogeneous tumor microenvironment revealed by single-cell profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1244&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"361b957deda67cfe496a9d5a3abc8546915864d6\",\n",
      "        \"title\": \"Simple Method to Quantify Protein Abundances from 1000 Cells\",\n",
      "        \"abstract\": \"The rise of single-cell transcriptomics has created an urgent need for similar approaches that use a minimal number of cells to quantify expression levels of proteins. We integrated and optimized multiple recent developments to establish a proteomics workflow to quantify proteins from as few as 1000 mammalian stem cells. The method uses chemical peptide labeling, does not require specific equipment other than cell lysis tools, and quantifies >2500 proteins with high reproducibility. We validated the method by comparing mouse embryonic stem cells and in vitro differentiated motor neurons. We identify differentially expressed proteins with small fold changes and a dynamic range in abundance similar to that of standard methods. Protein abundance measurements obtained with our protocol compared well to corresponding transcript abundance and to measurements using standard inputs. The protocol is also applicable to other systems, such as fluorescence-activated cell sorting (FACS)-purified cells from the tunicate Ciona. Therefore, we offer a straightforward and accurate method to acquire proteomics data from minimal input samples.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1245&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"112d13c2eadfe0ccbd4bb8ea00259f1a082b4766\",\n",
      "        \"title\": \"Sparse bottleneck neural networks for exploratory non-linear visualization of Patch-seq data\",\n",
      "        \"abstract\": \"Patch-seq, a recently developed experimental technique, allows neuroscientists to obtain transcriptomic and electrophysiological information from the same neurons. Efficiently analyzing and visualizing such paired multivariate data in order to extract biologically meaningful interpretations has, however, remained a challenge. Here, we use sparse deep neural networks with and without a two-dimensional bottleneck to predict electrophysiological features from the transcriptomic ones using a group lasso penalty, yielding concise and biologically interpretable two-dimensional visualizations. In two large example data sets, this visualization reveals known neural classes and their marker genes without biological prior knowledge. We also demonstrate that our method is applicable to other kinds of multimodal data, such as paired transcriptomic and proteomic measurements provided by CITE-seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1246&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c948d49a33e9c7a56af2cdd164cd821dd8c8ad44\",\n",
      "        \"title\": \"Sequencing of RNA in single cells reveals a distinct transcriptome signature of hematopoiesis in GATA2 deficiency.\",\n",
      "        \"abstract\": \"Constitutional GATA2 deficiency caused by heterozygous germline GATA2 mutations has a broad spectrum of clinical phenotypes, including systemic infections, lymphedema, cytopenias, and myeloid neoplasms. Genotype-phenotype correlation is not well understood mechanistically in GATA2 deficiency. We performed whole transcriptome sequencing of single hematopoietic stem and progenitor cells from 8 patients, who had pathogenic GATA2 mutations and myelodysplasia. Mapping patients' cells onto normal hematopoiesis, we observed deficiency in lymphoid/myeloid progenitors, also evident from highly constrained gene correlations. HSPCs of patients exhibited distinct patterns of gene expression and coexpression compared with counterparts from healthy donors. Distinct lineages showed differently altered transcriptional profiles. Stem cells in patients had dysregulated gene expression related to apoptosis, cell cycle, and quiescence; increased expression of erythroid/megakaryocytic priming genes; and decreased lymphoid priming genes. The prominent deficiency in lympho-myeloid lineages in GATA2 deficiency appeared at least partly due to the expression of aberrant gene programs in stem cells prior to lineage commitment. We computationally imputed cells with chromosomal abnormalities and determined their gene expression; DNA repair genes were downregulated in trisomy 8 cells, potentially rendering these cells vulnerable to second-hit somatic mutations and additional chromosomal abnormalities. Cells with complex cytogenetic abnormalities showed defects in genes related to multilineage differentiation and cell cycle. Single-cell RNA sequencing is powerful in resolving transcriptomes of cell subpopulations despite a paucity of cells in marrow failure. Our study discloses previously uncharacterized transcriptome signatures of stem cells and progenitors in GATA2 deficiency, providing a broad perspective of potential mechanisms by which germline mutations modulate early hematopoiesis in a human disease. This trial was registered at www.clinicaltrials.gov as NCT01905826, NCT01861106, and NCT00001620.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1247&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"118c0d1fd2186be29eff704a9678d823870f8190\",\n",
      "        \"title\": \"Chromatin Potential Identified by Shared Single-Cell Profiling of RNA and Chromatin\",\n",
      "        \"abstract\": \"Cell differentiation and function are regulated across multiple layers of gene regulation, including the modulation of gene expression by changes in chromatin accessibility. However, differentiation is an asynchronous process precluding a temporal understanding of the regulatory events leading to cell fate commitment. Here, we developed SHARE-seq, a highly scalable approach for measurement of chromatin accessibility and gene expression within the same single cell. Using 34,774 joint profiles from mouse skin, we develop a computational strategy to identify cis-regulatory interactions and define Domains of Regulatory Chromatin (DORCs), which significantly overlap with super-enhancers. We show that during lineage commitment, chromatin accessibility at DORCs precedes gene expression, suggesting changes in chromatin accessibility may prime cells for lineage commitment. We therefore develop a computational strategy (chromatin potential) to quantify chromatin lineage-priming and predict cell fate outcomes. Together, SHARE-seq provides an extensible platform to study regulatory circuitry across diverse cells within tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1248&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d4301eb39131ba37cb5ca94e07f28a462940966\",\n",
      "        \"title\": \"Integrated analysis of glycan and RNA in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1249&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ded260913f49e7245d5dd69b939f77dcd8e83bff\",\n",
      "        \"title\": \"Improving oligo-conjugated antibody signal in multimodal single-cell analysis\",\n",
      "        \"abstract\": \"Simultaneous measurement of surface proteins and gene expression within single cells using oligo-conjugated antibodies offers high resolution snapshots of complex cell populations. Signal from oligo-conjugated antibodies is quantified by high-throughput sequencing and is highly scalable and sensitive. In this study, we investigated the response of oligo-conjugated antibodies towards four variables: Concentration, staining volume, cell number at staining, and tissue. We find that staining with recommended antibody concentrations cause unnecessarily high background and that concentrations can be drastically reduced without loss of biological information. Reducing volume only affects antibodies targeting abundant epitopes used at low concentrations and is counteracted by reducing cell numbers. Adjusting concentrations increases signal, lowers background and reduces costs. Background signal can account for a major fraction of the total sequencing and is primarily derived from antibodies used at high concentrations. Together, this study provides new insight into the titration response and background of oligo-conjugated antibodies and offers concrete guidelines on how such panels can be improved. Impact statement Oligo-conjugated antibodies are a powerful tool but require thorough optimization to reduce background signal, increase sensitivity, and reduce sequencing costs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1250&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"27a99766b0f8d7853b871f53b99e6bd2f94adbc9\",\n",
      "        \"title\": \"Platelet \\\"-omics\\\" in health and cardiovascular disease.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1251&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3e240e5defc2ecd286aa3b2dff914c827c5a3b52\",\n",
      "        \"title\": \"Linear-time cluster ensembles of large-scale single-cell RNA-seq and multimodal data\",\n",
      "        \"abstract\": \"A fundamental task in single-cell RNA-seq (scRNA-seq) analysis is the identification of transcriptionally distinct groups of cells. Numerous methods have been proposed for this problem, with a recent focus on methods for the cluster analysis of ultra-large scRNA-seq datasets produced by droplet-based sequencing technologies. Most existing methods rely on a sampling step to bridge the gap between algorithm scalability and volume of the data. Ignoring large parts of the data, however, often yields inaccurate groupings of cells and risks overlooking rare cell types. We propose method Specter that adopts and extends recent algorithmic advances in (fast) spectral clustering. In contrast to methods that cluster a (random) subsample of the data, we adopt the idea of landmarks that are used to create a sparse representation of the full data from which a spectral embedding can then be computed in linear time. We exploit Specter\\u2019s speed in a cluster ensemble scheme that achieves a substantial improvement in accuracy across 16 scRNA-seq datasets and that is sensitive to rare cell types. Its linear time complexity allows Specter to cluster a dataset comprising 2 million cells in just 26 minutes. Furthermore, on CITE-seq data that simultaneously measures gene and protein marker expression we demonstrate that Specter is able to utilize multimodal omics measurements to resolve subtle transcriptomic differences between subpopulations of cells. Specter is open source and available at https://github.com/canzarlab/Specter.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1252&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"257d4bf489c84416cf6827f17c63e89b4fc49bc0\",\n",
      "        \"title\": \"SCIM: universal single-cell matching with unpaired feature sets\",\n",
      "        \"abstract\": \"Motivation Recent technological advances have led to an increase in the production and availability of single-cell data. The ability to integrate a set of multi-technology measurements would allow the identification of biologically or clinically meaningful observations through the unification of the perspectives afforded by each technology. In most cases, however, profiling technologies consume the used cells and thus pairwise correspondences between datasets are lost. Due to the sheer size single-cell datasets can acquire, scalable algorithms that are able to universally match single-cell measurements carried out in one cell to its corresponding sibling in another technology are needed. Results We propose Single-Cell data Integration via Matching (SCIM), a scalable approach to recover such correspondences in two or more technologies. SCIM assumes that cells share a common (low-dimensional) underlying structure and that the underlying cell distribution is approximately constant across technologies. It constructs a technology-invariant latent space using an auto-encoder framework with an adversarial objective. Multi-modal datasets are integrated by pairing cells across technologies using a bipartite matching scheme that operates on the low-dimensional latent representations. We evaluate SCIM on a simulated cellular branching process and show that the cell-to-cell matches derived by SCIM reflect the same pseudotime on the simulated dataset. Moreover, we apply our method to two real-world scenarios, a melanoma tumor sample and a human bone marrow sample, where we pair cells from a scRNA dataset to their sibling cells in a CyTOF dataset achieving 93% and 84% cell-matching accuracy for each one of the samples respectively. Availability https://github.com/ratschlab/scim\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1253&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"89e95eb2e30cf354aadff41f653b4cf537d94631\",\n",
      "        \"title\": \"Functional crosstalk between T cells and monocytes in cancer and atherosclerosis\",\n",
      "        \"abstract\": \"Monocytes and monocyte\\u2010derived cells, including M\\u03d5s and dendritic cells, exhibit a diverse array of phenotypic states that are dictated by their surrounding microenvironment. These cells direct T cell activation and function via cues that range from being immunosuppressive to immunostimulatory. Solid tumors and atherosclerotic plaques represent two pathological niches with distinct immune microenvironments. While monocytes and their progeny possess a phenotypic spectrum found within both disease contexts, most within tumors are pro\\u2010tumoral and support evasion of host immune responses by tumor cells. In contrast, monocyte\\u2010derived cells within atherosclerotic plaques are usually pro\\u2010atherogenic, pro\\u2010inflammatory, and predominantly directed against self\\u2010antigens. Consequently, cancer immunotherapies strive to enhance the immune response against tumor antigens, whereas atherosclerosis treatments seek to dampen the immune response against lipid antigens. Insights into monocyte\\u2010T cell interactions within these niches could thus inform therapeutic strategies for two immunologically distinct diseases. Here, we review monocyte diversity, interactions between monocytes and T cells within tumor and plaque microenvironments, how certain therapies have leveraged these interactions, and novel strategies to assay such associations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1254&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aa78f599e593c2045a84f301086240a643cf2481\",\n",
      "        \"title\": \"Isolation of Adipose Tissue Nuclei for Single-Cell Genomic Applications.\",\n",
      "        \"abstract\": \"Brown and beige fat are specialized adipose tissues that dissipate energy for thermogenesis by UCP1 (Uncoupling Protein-1)-dependent and independent pathways. Until recently, thermogenic adipocytes were considered a homogeneous population. However, recent studies have indicated that there are multiple subtypes or subpopulations that are distinct in developmental origin, substrate use, and transcriptome. Despite advances in single-cell genomics, unbiased decomposition of adipose tissues into cellular subtypes has been challenging because of the fragile nature of lipid-filled adipocytes. The protocol presented was developed to circumvent these obstacles by effective isolation of single nuclei from adipose tissue for downstream applications, including RNA sequencing. Cellular heterogeneity can then be analyzed by RNA sequencing and bioinformatic analyses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1255&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e7a35e62c917f37f898cdeee558536f945f73963\",\n",
      "        \"title\": \"Single-cell biophysical study reveals deformability and internal ordering relationship in T cells.\",\n",
      "        \"abstract\": \"Deformability and internal ordering are key features related to cell function, particularly critical for cells that routinely undergo large deformations, like T cells during extravasation and migration. In the measurement of cell deformability, a considerable variability is typically obtained, masking the identification of possible interrelationships between deformability, internal ordering and cell function. We report the development of a single-cell methodology that combines measurements of living-cell deformability, using micropipette aspiration, and three-dimensional confocal analysis of the nucleus and cytoskeleton. We show that this single-cell approach can serve as a powerful tool to identify appropriate parameters that characterize deformability within a population of cells, not readably discernable in population-averaged data. By applying this single-cell methodology to mouse CD4+ T cells, our results demonstrate that the relative size of the nucleus, better than other geometrical or cytoskeletal features, effectively determines the overall deformability of the cells within the population.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1256&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"323494180ebfd7bda4cb01b1db572b856d054c90\",\n",
      "        \"title\": \"Warp Speed Ahead! Technology-driven Breakthroughs in Skin Immunity and Inflammatory Disease.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1257&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"39cdee1496a240f9d554107e18804bc4bc1cbac1\",\n",
      "        \"title\": \"Unravelling the complex genetics of common kidney diseases: from variants to mechanisms\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1258&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"65f5f21ff75c776e8cd32aefba0689f3794bc564\",\n",
      "        \"title\": \"Double Emulsion Picoreactors for High-Throughput Single-Cell Encapsulation and Phenotyping via FACS\",\n",
      "        \"abstract\": \"In the past five years, droplet microfluidic techniques have unlocked new opportunities for the high-throughput genome-wide analysis of single cells, transforming our understanding of cellular diversity and function. However, the field lacks an accessible method to screen and sort droplets based on cellular phenotype upstream of genetic analysis, particularly for large and complex cells. To meet this need, we developed Dropception, a robust, easy-to-use workflow for precise single-cell encapsulation into picoliter-scale double emulsion droplets compatible with high-throughput phenotyping via fluorescence-activated cell sorting (FACS). We demonstrate the capabilities of this method by encapsulating five standardized mammalian cell lines of varying size and morphology as well as a heterogeneous cell mixture of a whole dissociated flatworm (5 - 25 \\u03bcm in diameter) within highly monodisperse double emulsions (35 \\u03bcm in diameter). We optimize for preferential encapsulation of single cells with extremely low multiple-cell loading events (<2% of cell-containing droplets), thereby allowing direct linkage of cellular phenotype to genotype. Across all cell lines, cell loading efficiency approaches the theoretical limit with no observable bias by cell size. FACS measurements reveal the ability to discriminate empty droplets from those containing cells with good agreement to single-cell occupancies quantified via microscopy, establishing robust droplet screening at single-cell resolution. High-throughput FACS phenotyping of cellular picoreactors has the potential to shift the landscape of single-cell droplet microfluidics by expanding the repertoire of current nucleic acid droplet assays to include functional screening. ABSTRACT FIGURE\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1259&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d306475c1cc6126639a913289fdcad0c9b3d6c8\",\n",
      "        \"title\": \"SCHNAPPs - Single Cell sHiNy APPlication(s)\",\n",
      "        \"abstract\": \"Motivation Single-cell RNA-sequencing (scRNAseq) experiments are becoming a standard tool for bench-scientists to explore the cellular diversity present in all tissues. On one hand, the data produced by scRNASeq is technically complex, with analytical workflows that are still very much an active field of bioinformatics research, and on the other hand, a wealth of biological background knowledge is often needed to guide the investigation. Therefore, there is an increasing need to develop applications geared towards bench-scientists to help them abstract the technical challenges of the analysis, so that they can focus on the Science at play. It is also expected that such applications should support closer collaboration between bioinformaticians and bench-scientists by providing reproducible science tools. Results We present SCHNAPPs, a computer program designed to enable bench-scientists to autonomously explore and interpret single cell RNA-seq expression data and associated annotations. The Shiny-based application allows selecting genes and cells of interest, performing quality control, normalization, clustering, and differential expression analyses, applying standard workflows from Seurat (Stuart et al., 2019) or Scran (Lun et al., 2016) packages, and most of the common visualizations. An R-markdown report can be generated that tracks the modifications, and selected visualizations facilitating communication and reproducibility between bench-scientist and bioinformatician. The modular design of the tool allows to easily integrate new visualizations and analyses by bioinformaticians. We still recommend that a data analysis specialist oversees the analysis and interpretation. Availability The SCHNAPPs application, docker file, and documentation are available on GitHub: https://c3bi-pasteur-fr.github.io/UTechSCB-SCHNAPPs; Example contribution are available at the following GitHub site: https://github.com/baj12/SCHNAPPsContributions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1260&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"40a9a948bd19182b5326ff8572cab7f257271ec0\",\n",
      "        \"title\": \"Cryopreservation of human cancers conserves tumour heterogeneity for single-cell multi-omics analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1261&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6642e7878bf634a4b0038f19a910184261525ed0\",\n",
      "        \"title\": \"Per-sample standardization and asymmetric winsorization lead to accurate clustering of RNA-seq expression profiles\",\n",
      "        \"abstract\": \"Motivation Data transformations are an important step in the analysis of RNA-seq data. Nonetheless, the impact of transformations on the outcome of unsupervised clustering procedures is still unclear. Results Here, we present an Asymmetric Winsorization per Sample Transformation (AWST), which is robust to data perturbations and removes the need for selecting the most informative genes prior to sample clustering. Our procedure leads to robust and biologically meaningful clusters both in bulk and in single-cell applications. Availability The AWST method is available at https://github.com/drisso/awst. The code to reproduce the analyses is available at https://github.com/drisso/awst_analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1262&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"be6d739b9cf1330dbf03d818bde8893a3d5d6768\",\n",
      "        \"title\": \"Single-cell technologies in hepatology: new insights into liver biology and disease pathogenesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1263&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d0a0cd90ce665e8fa508a7e3b2346fee56034c21\",\n",
      "        \"title\": \"Dynamics of the 4D genome during in vivo lineage specification and differentiation\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1264&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dfd5a96423fb6feae9315ff8c6c78a707bae87ba\",\n",
      "        \"title\": \"Identification of non-cancer cells from cancer transcriptomic data\\u2606\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1265&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59b803e0b5750ccd053c2c075220f126d03d89b1\",\n",
      "        \"title\": \"Atypical B cells are a normal component of immune responses to vaccination and infection in humans\",\n",
      "        \"abstract\": \"The full diversity of the circulating human B cell compartment is unknown. Flow cytometry analysis suggests that in addition to na\\u00efve and memory B cells, there exists a population of CD11c+, CD27\\u2212 CD21\\u2212 \\u201catypical\\u201d B cells, that are associated with chronic or recurrent infection and autoimmunity. We used single cell RNA-seq approaches to examine the diversity of both antigen-specific B cells and total B cells in healthy subjects and individuals naturally-exposed to recurrent malaria infections. This analysis revealed two B cell lineages: a classical lineage of activated and resting memory B cells, and an atypical-like lineage. Surprisingly, the atypical lineage was common in both malaria exposed individuals and non-exposed healthy controls. Using barcoded antibodies in conjunction with our transcriptomic data, we found that atypical lineage cells in healthy individuals lack many atypical B markers and thus represent an undercounted cryptic population. We further determined using antigen specific probes that atypical cells can be induced by primary vaccination in humans and can be recalled upon boosting. Collectively these data suggest that atypical cells are not necessarily pathogenic but can be a normal component of B responses to antigen.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1266&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0fee888668acfa2e78521e9ae98d271a8eaadb39\",\n",
      "        \"title\": \"Leukocyte Heterogeneity in Adipose Tissue, Including in Obesity.\",\n",
      "        \"abstract\": \"Adipose tissue (AT) plays a central role in both metabolic health and pathophysiology. Its expansion in obesity results in increased mortality and morbidity, with contributions to cardiovascular disease, diabetes mellitus, fatty liver disease, and cancer. Obesity prevalence is at an all-time high and is projected to be 50% in the United States by 2030. AT is home to a large variety of immune cells, which are critical to maintain normal tissue functions. For example, \\u03b3\\u03b4 T cells are fundamental for AT innervation and thermogenesis, and macrophages are required for recycling of lipids released by adipocytes. The expansion of visceral white AT promotes dysregulation of its immune cell composition and likely promotes low-grade chronic inflammation, which has been proposed to be the underlying cause for the complications of obesity. Interestingly, weight loss after obesity alters the AT immune compartment, which may account for the decreased risk of developing these complications. Recent technological advancements that allow molecular investigation on a single-cell level have led to the discovery of previously unappreciated heterogeneity in many organs and tissues. In this review, we will explore the heterogeneity of immune cells within the visceral white AT and their contributions to homeostasis and pathology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1267&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6cb97277eb132ca8a1bbd11a39f583fc9e107651\",\n",
      "        \"title\": \"New avenues for systematically inferring cell-cell communication: through single-cell transcriptomics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1268&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e05f8d9946301e57db37e6df9c95d57508ac1add\",\n",
      "        \"title\": \"A Multiplexed Barcodelet Single-Cell RNA-Seq Approach Elucidates Combinatorial Signaling Pathways that Drive ESC Differentiation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1269&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"062655253ec57342f95a87154541da98e2e59a24\",\n",
      "        \"title\": \"Antigen presentation by dendritic cells and their instruction of CD4+ T helper cell responses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1270&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"82284dcfe204b2d9a37782c81c372ec3f3f83aba\",\n",
      "        \"title\": \"Fast and accurate diagnostics from highly multiplexed sequencing assays\",\n",
      "        \"abstract\": \"Scalable, inexpensive, accurate, and secure testing for SARS-CoV-2 infection is crucial for control of the novel coronavirus pandemic. Recently developed highly multiplexed sequencing assays that rely on high-throughput sequencing (HMSAs) can, in principle, meet these demands, and present promising alternatives to currently used RT-qPCR-based tests. However, the analysis and interpretation of HMSAs requires overcoming several computational and statistical challenges. Using recently acquired experimental data, we present and validate an accurate and fast computational testing workflow based on kallisto and bustools, that utilize robust statistical methods and fast, memory efficient algorithms for processing high-throughput sequencing data. We show that our workflow is effective at processing data from all recently proposed SARS-CoV-2 sequencing based diagnostic tests, and is generally applicable to any diagnostic HMSAs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1271&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"881638eb5d0858a1def8f020e2293737057931a6\",\n",
      "        \"title\": \"Quantification of proteins, protein complexes and mRNA in single cells by proximity-sequencing\",\n",
      "        \"abstract\": \"Multiplexed analysis of single-cells enables accurate modeling of cellular behaviors, classification of new cell types, and characterization of their functional states. Here we present proximity-sequencing (Prox-seq), a method for simultaneous measurement of an individual cell\\u2019s proteins, protein complexes and mRNA. Prox-seq utilizes deep sequencing and barcoded proximity assays to measure proteins and their complexes from all pairwise combinations of targeted proteins, in thousands of single-cells. The number of measured protein complexes scales quadratically with the number of targeted proteins, providing unparalleled multiplexing capacity. We developed a high-throughput experimental and computational pipeline and demonstrated the potential of Prox-Seq for multi-omic analysis with a panel of 13 barcoded proximity probes, enabling the measurement of 91 protein complexes, along with thousands of mRNA molecules in single T-cells and B-cells. Prox-seq provides access to an untapped yet powerful measurement modality for single-cell phenotyping and can discover new protein interactions in signaling and drug studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1272&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"80a42595c014abde1fb389f5afba35dede418a81\",\n",
      "        \"title\": \"Extensive Remodeling of the Immune Microenvironment in B Cell Acute Lymphoblastic Leukemia.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1273&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"92ee051f7678221434ffc19ae342e06487c6ff80\",\n",
      "        \"title\": \"Spatial-omics: Novel Approaches to Probe Cell Heterogeneity and Extracellular Matrix Biology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1274&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"83c9b16ad8542507e83b156404787d151096dc11\",\n",
      "        \"title\": \"MOFA+: a statistical framework for comprehensive integration of multi-modal single-cell data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1275&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0b4ad4c677f164cb19a0a7ab644172c80acc44d8\",\n",
      "        \"title\": \"T cell-inducing vaccine durably prevents mucosal SHIV infection even with lower neutralizing antibody titers\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1276&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"673d2998bb11ea56ee4616af73684741f4736ed7\",\n",
      "        \"title\": \"Joint probabilistic modeling of paired transcriptome and proteome measurements in single cells\",\n",
      "        \"abstract\": \"The paired measurement of RNA and surface protein abundance in single cells with CITE-seq is a promising approach to connect transcriptional variation with cell phenotypes and functions. However, each data modality exhibits unique technical biases, making it challenging to conduct a joint analysis and combine these two views into a unified representation of cell state. Here we present Total Variational Inference (totalVI), a framework for the joint probabilistic analysis of paired RNA and protein data from single cells. totalVI probabilistically represents the data as a composite of biological and technical factors such as limited sensitivity of the RNA data, background in the protein data, and batch effects. To evaluate totalVI, we performed CITE-seq on immune cells from murine spleen and lymph nodes with biological replicates and with different antibody panels measuring over 100 surface proteins. With this dataset, we demonstrate that totalVI provides a cohesive solution for common analysis tasks like the integration of datasets with matched or unmatched protein panels, dimensionality reduction, clustering, evaluation of correlations between molecules, and differential expression testing. totalVI enables scalable, end-to-end analysis of paired RNA and protein data from single cells and is available as open-source software.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1277&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"baa72332a09cef7b4e577067c205803c614f7195\",\n",
      "        \"title\": \"Inflammatory Bowel Disease Through the Lens of Single-cell RNA-seq Technologies\",\n",
      "        \"abstract\": \"Abstract The intestinal mucosa represents a unique environment where the coordinated function of diverse epithelial, mesenchymal, and immune cells maintains a physiologically balanced environment in the presence of gut microbiota. The intestinal mucosa plays a central role in the pathogenesis of inflammatory bowel disease (IBD), yet the molecular and cellular composition of this diverse environment is poorly understood. However, the recent advent of multimodal single-cell technologies, including single-cell RNA sequencing (scRNA-seq), now provides an opportunity to accurately map the tissue architecture, characterize rare cell types that were previously overlooked, and define function at a single-cell level. In this review, we summarize key advances in single-cell technology and provide an overview of important aspects of computational analysis. We describe emerging data in the field of IBD and discuss how the characterization of novel intestinal mucosa cell populations is reshaping our understanding of this complex disease. We conclude by considering the potential clinical applications, including the definition of novel drug targets and the opportunity for personalization of care in this exciting new era of precision medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1278&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0d701f0a34510ad70c808ffeed24fceef02b4b58\",\n",
      "        \"title\": \"Tree! I am no Tree! I am a Low Dimensional Hyperbolic Embedding\",\n",
      "        \"abstract\": \"Given data, finding a faithful low-dimensional hyperbolic embedding of the data is a key method by which we can extract hierarchical information or learn representative geometric features of the data. In this paper, we explore a new method for learning hyperbolic representations that takes a metric-first approach. Rather than determining the low-dimensional hyperbolic embedding directly, we learn a tree structure on the data as an intermediate step. This tree structure can then be used directly to extract hierarchical information, embedded into a hyperbolic manifold using Sarkar's construction (Sarkar, 2012), or used as a tree approximation of the original metric. To this end, we present a novel fast algorithm TreeRep such that, given a $\\\\delta$-hyperbolic metric (for any $\\\\delta \\\\geq 0$), the algorithm learns a tree structure that approximates the original metric. In the case when $\\\\delta = 0$, we show analytically that TreeRep exactly recovers the original tree structure. We show empirically that TreeRep is not only many orders of magnitude faster than previous known algorithms, but also produces metrics with lower average distortion and higher mean average precision than most previous algorithms for learning hyperbolic embeddings, extracting hierarchical information, and approximating metrics via tree metrics.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1279&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0488f61b55bad90800abbad45ecc4b3031962eee\",\n",
      "        \"title\": \"Ultra-high throughput single-cell analysis of proteins and RNAs by split-pool synthesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1280&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a6e70b8d3c7b2460640d890f212ca8a885928d70\",\n",
      "        \"title\": \"Ultra-high throughput single-cell analysis of proteins and RNAs by split-pool synthesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1281&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c5f7779ac3e05a1e5c417917f5d7b7e7f1c6359e\",\n",
      "        \"title\": \"Improved mouse models and advanced genetic and genomic technologies for the study of neutrophils.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1282&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcd111de892d484e9213001db38428aafdf2a00f\",\n",
      "        \"title\": \"Sparsity-Penalized Stacked Denoising Autoencoders for Imputing Single-Cell RNA-seq Data\",\n",
      "        \"abstract\": \"Single-cell RNA-seq (scRNA-seq) is quite prevalent in studying transcriptomes, but it suffers from excessive zeros, some of which are true, but others are false. False zeros, which can be seen as missing data, obstruct the downstream analysis of single-cell RNA-seq data. How to distinguish true zeros from false ones is the key point of this problem. Here, we propose sparsity-penalized stacked denoising autoencoders (scSDAEs) to impute scRNA-seq data. scSDAEs adopt stacked denoising autoencoders with a sparsity penalty, as well as a layer-wise pretraining procedure to improve model fitting. scSDAEs can capture nonlinear relationships among the data and incorporate information about the observed zeros. We tested the imputation efficiency of scSDAEs on recovering the true values of gene expression and helping downstream analysis. First, we show that scSDAE can recover the true values and the sample\\u2013sample correlations of bulk sequencing data with simulated noise. Next, we demonstrate that scSDAEs accurately impute RNA mixture dataset with different dilutions, spike-in RNA concentrations affected by technical zeros, and improves the consistency of RNA and protein levels in CITE-seq data. Finally, we show that scSDAEs can help downstream clustering analysis. In this study, we develop a deep learning-based method, scSDAE, to impute single-cell RNA-seq affected by technical zeros. Furthermore, we show that scSDAEs can recover the true values, to some extent, and help downstream analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1283&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e95970849ce3b30ce56912638d9407fdc3c0f3d3\",\n",
      "        \"title\": \"Single-Cell RNA Sequencing and Its Combination with Protein and DNA Analyses\",\n",
      "        \"abstract\": \"Heterogeneity in cell populations poses a significant challenge for understanding complex cell biological processes. The analysis of cells at the single-cell level, especially single-cell RNA sequencing (scRNA-seq), has made it possible to comprehensively dissect cellular heterogeneity and access unobtainable biological information from bulk analysis. Recent efforts have combined scRNA-seq profiles with genomic or proteomic data, and show added value in describing complex cellular heterogeneity than transcriptome measurements alone. With the rising demand for scRNA-seq for biomedical and clinical applications, there is a strong need for a timely and comprehensive review on the scRNA-seq technologies and their potential biomedical applications. In this review, we first discuss the latest state of development by detailing each scRNA-seq technology, including both conventional and microfluidic technologies. We then summarize their advantages and limitations along with their biomedical applications. The efforts of integrating the transcriptome profile with highly multiplexed proteomic and genomic data are thoroughly reviewed with results showing the integrated data being more informative than transcriptome data alone. Lastly, the latest progress toward commercialization, the remaining challenges, and future perspectives on the development of scRNA-seq technologies are briefly discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1284&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6984a3e215286d09979229867379780a159998d2\",\n",
      "        \"title\": \"Cell-type specific gene expression profiling in heterogeneous in vitro cultures using epitope-tagged RPL22\",\n",
      "        \"abstract\": \"Human induced pluripotent stem cells (hiPSCs) allow the production of defined, disease relevant cell types for transcriptomic and functional studies. A major challenge for hiPSCs disease modeling is to more closely mimic heterocellular networks observed in vivo. As hiPSC cultures become more complex, measuring cell-type specific transcriptomics will become more challenging. Here we report an extension of the RiboTag system, which uses cell-type restricted expression of epitope-tagged RPL22 in mouse tissue, applied to immortalized cell lines, primary mouse astrocytes, and hiPSC-derived neural cells. We constructed lentiviral vectors expressing epitope-tagged RPL22 using constitutive and inducible promoters fused to HA, Flag, or V5 affinity tags in combination with fluorescent reporters; differential expression of uniquely tagged RPL22 in specific cell types allows mRNA from each population to be biochemically purified from mixed cultures. RiboTag expression enables efficient depletion of off-target RNA in mixed species co-cultures of immortalized cell lines using HA and V5 RiboTags. In monocultures of hiPSC-derived neural progenitor cells, motor neurons, and GABAergic neurons expressing RiboTags, we observed an enrichment of protein coding genes and concomitant decrease in mitochondrial and non-coding genes in immunoprecipitated RNA. Depletion efficiency varied across independent replicates of mixed-species co-cultures of hiPSC-derived motor neurons and primary mouse astrocytes. The challenges and potential of implementing RiboTags in complex in vitro cultures are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1285&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0d3624f2e4e537e74c156130eecbc94184562722\",\n",
      "        \"title\": \"Cell Type-Specific In Vitro Gene Expression Profiling of Stem Cell-Derived Neural Models\",\n",
      "        \"abstract\": \"Genetic and genomic studies of brain disease increasingly demonstrate disease-associated interactions between the cell types of the brain. Increasingly complex and more physiologically relevant human induced pluripotent stem cell (hiPSC)-based models better explore the molecular mechanisms underlying disease, but also challenge our ability to resolve cell-type specific perturbations. Here we report an extension of the RiboTag system, first developed to achieve cell-type restricted expression of epitope-tagged ribosomal protein (RPL22) in mouse tissue, to a variety of in vitro applications, including immortalized cell lines, primary mouse astrocytes, and hiPSC-derived neurons. RiboTag expression enables efficient depletion of off-target RNA in mixed species primary co-cultures and in hiPSC-derived neural progenitor cells, motor neurons, and GABAergic neurons. Nonetheless, depletion efficiency varies across independent experimental replicates. The challenges and potential of implementing RiboTags in complex in vitro cultures are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1286&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4b3862e0e64a3e158e5f24a840a4dde925df319\",\n",
      "        \"title\": \"Multiplex digital spatial profiling of proteins and RNA in fixed tissue\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1287&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"254e213e67aca0c5323194b728d59b591f393086\",\n",
      "        \"title\": \"Advances of Single-Cell Protein Analysis\",\n",
      "        \"abstract\": \"Proteins play a significant role in the key activities of cells. Single-cell protein analysis provides crucial insights in studying cellular heterogeneities. However, the low abundance and enormous complexity of the proteome posit challenges in analyzing protein expressions at the single-cell level. This review summarizes recent advances of various approaches to single-cell protein analysis. We begin by discussing conventional characterization approaches, including fluorescence flow cytometry, mass cytometry, enzyme-linked immunospot assay, and capillary electrophoresis. We then detail the landmark advances of microfluidic approaches for analyzing single-cell protein expressions, including microfluidic fluorescent flow cytometry, droplet-based microfluidics, microwell-based assay (microengraving), microchamber-based assay (barcoding microchips), and single-cell Western blotting, among which the advantages and limitations are compared. Looking forward, we discuss future research opportunities and challenges for multiplexity, analyte, throughput, and sensitivity of the microfluidic approaches, which we believe will prompt the research of single-cell proteins such as the molecular mechanism of cell biology, as well as the clinical applications for tumor treatment and drug development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1288&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2988a26f21a00b5aa5bfd281560e8b21b1c07f1b\",\n",
      "        \"title\": \"Tools and Concepts for Interrogating and Defining Cellular Identity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1289&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b2a5b9d02ff32e0da7996bdc9bf53f34cd0ebc71\",\n",
      "        \"title\": \"Riemannian geometry and statistical modeling correct for batch effects and control false discoveries in single-cell surface protein count data from CITE-seq\",\n",
      "        \"abstract\": \"Recent advances in next generation sequencing-based single-cell technologies have allowed high-throughput quantitative detection of cell-surface proteins along with the transcriptome in individual cells, extending our understanding of the heterogeneity of cell populations in diverse tissues that are in different diseased states or under different experimental conditions. Count data of surface proteins from the cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) technology pose new computational challenges, and there is currently a dearth of rigorous mathematical tools for analyzing the data. This work utilizes concepts and ideas from Riemannian geometry to remove batch effects between samples and develops a statistical framework for distinguishing positive signals from background noise. The strengths of these approaches are demonstrated on two independent CITE-seq data sets in mouse and human. Python source code implementing the algorithms is available at https://github.com/jssong-lab/SAGACITE.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1290&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2e3430a375eb5755c86e480c3b76c466e18f581e\",\n",
      "        \"title\": \"Spatial Transcriptional Mapping of the Human Nephrogenic Program\",\n",
      "        \"abstract\": \"Congenital abnormalities of the kidney and urinary tract are amongst the most common birth defects affecting 3% of newborns. The human kidney develops over a 30-week period in which a nephron progenitor pool gives rise to around a million nephrons. To establish a framework for human nephrogenesis, we spatially resolved a stereotypical process by which equipotent nephron progenitors generate a nephron anlagen, then applied data-driven approaches to construct three-dimensional protein maps on anatomical models of the nephrogenic program. Single cell RNA sequencing identified novel progenitor states which were spatially mapped to the nephron anatomy enabling the generation of functional gene-networks predicting interactions within and between nephron cell-types. Network mining identified known developmental disease genes and predicts new targets of interest. The spatially resolved nephrogenic program made available through the Human Nephrogenesis Atlas (https://sckidney.flatironinstitute.org/) will facilitate an understanding of kidney development and disease, and enhance efforts to generate new kidney structures.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1291&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe212ba698bdba0453f21b51be886fc73c22c2eb\",\n",
      "        \"title\": \"Cell Atlas technologies and insights into tissue architecture\",\n",
      "        \"abstract\": \"Since Robert Hooke first described the existence of \\u2018cells\\u2019 in 1665, scientists have sought to identify and further characterise these fundamental units of life. While our understanding of cell location, morphology and function has expanded greatly; our understanding of cell types and states at the molecular level, and how these function within tissue architecture, is still limited. A greater understanding of our cells could revolutionise basic biology and medicine. Atlasing initiatives like the Human Cell Atlas aim to identify all cell types at the molecular level, including their physical locations, and to make this reference data openly available to the scientific community. This is made possible by a recent technology revolution: both in single-cell molecular profiling, particularly single-cell RNA sequencing, and in spatially resolved methods for assessing gene and protein expression. Here, we review available and upcoming atlasing technologies, the biological insights gained to date and the promise of this field for the future.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1292&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"86cd0965c54332769458e034d095550651cf1d34\",\n",
      "        \"title\": \"Multimodal memory T cell profiling identifies a reduction in a polyfunctional Th17 state associated with tuberculosis progression\",\n",
      "        \"abstract\": \"Mycobacterium tuberculosis (M.tb) results in 10 million active tuberculosis (TB) cases and 1.5 million deaths each year1, making it the world\\u2019s leading infectious cause of death2. Infection leads to either an asymptomatic latent state or TB disease. Memory T cells have been implicated in TB disease progression, but the specific cell states involved have not yet been delineated because of the limited scope of traditional profiling strategies. Furthermore, immune activation during infection confounds underlying differences in T cell state distributions that influence risk of progression. Here, we used a multimodal single-cell approach to integrate measurements of transcripts and 30 functionally relevant surface proteins to comprehensively define the memory T cell landscape at steady state (i.e., outside of active infection). We profiled 500,000 memory T cells from 259 Peruvians > 4.7 years after they had either latent M.tb infection or active disease and defined 31 distinct memory T cell states, including a CD4+CD26+CD161+CCR6+ effector memory state that was significantly reduced in patients who had developed active TB (OR = 0.80, 95% CI: 0.73\\u20130.87, p = 1.21 \\u00d7 10\\u22126). This state was also polyfunctional; in ex vivo stimulation, it was enriched for IL-17 and IL-22 production, consistent with a Th17-skewed phenotype, but also had more capacity to produce IFN\\u03b3 than other CD161+CCR6+ Th17 cells. Additionally, in progressors, IL-17 and IL-22 production in this cell state was significantly lower than in non-progressors. Reduced abundance and function of this state may be an important factor in failure to control M.tb infection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1293&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b06b2ee98723cb5375e5db54c82a15c6872316fb\",\n",
      "        \"title\": \"Vector Copy Distribution at a Single-Cell Level Enhances Analytical Characterization of Gene-Modified Cell Therapies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1294&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"570bd7c28f99ce549ab0c7a446cf9d1738d291d9\",\n",
      "        \"title\": \"scConsensus: combining supervised and unsupervised clustering for cell type identification in single-cell RNA sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1295&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"053fb443fd193194a1b14877d7ceb47f84dee591\",\n",
      "        \"title\": \"Technologies for assessing vaccine responses in the very young.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1296&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"736b77cd77826ec1dea940f4a2078c1ca6944a09\",\n",
      "        \"title\": \"Single Cell RNA Sequencing in Atherosclerosis Research\",\n",
      "        \"abstract\": \"Technological advances in characterizing molecular heterogeneity at the single cell level have ushered in a deeper understanding of the biological diversity of cells present in tissues including atherosclerotic plaques. New subsets of cells have been discovered among cell types previously considered homogenous. The commercial availability of systems to obtain transcriptomes and matching surface phenotypes from thousands of single cells is rapidly changing our understanding of cell types and lineage identity. Emerging methods to infer cellular functions are beginning to shed new light on the interplay of components involved in multifaceted disease responses, like atherosclerosis. Here, we provide a technical guide for design, implementation, assembly, and interpretations of current single cell transcriptomics approaches from the perspective of employing these tools for advancing cardiovascular disease research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1297&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"29cab7ccd8a160ca178628b924ca2c029d04ef6b\",\n",
      "        \"title\": \"Dual Relief of T-lymphocyte Proliferation and Effector Function Underlies Response to PD-1 Blockade in Epithelial Malignancies\",\n",
      "        \"abstract\": \"Tumor-infiltrating CD8+ T cells from patients with three cancer types exhibit a memory phenotype, a sequential pattern of immune checkpoint expression with additional CD39 expression, and CD28 loss. These cells' effector potential can be unleashed with anti\\u2013PD-1. Although understanding of T-cell exhaustion is widely based on mouse models, its analysis in patients with cancer could provide clues indicating tumor sensitivity to immune checkpoint blockade (ICB). Data suggest a role for costimulatory pathways, particularly CD28, in exhausted T-cell responsiveness to PD-1/PD-L1 blockade. Here, we used single-cell transcriptomic, phenotypic, and functional approaches to dissect the relation between CD8+ T-cell exhaustion, CD28 costimulation, and tumor specificity in head and neck, cervical, and ovarian cancers. We found that memory tumor\\u2013specific CD8+ T cells, but not bystander cells, sequentially express immune checkpoints once they infiltrate tumors, leading, in situ, to a functionally exhausted population. Exhausted T cells were nonetheless endowed with effector and tumor residency potential but exhibited loss of the costimulatory receptor CD28 in comparison with their circulating memory counterparts. Accordingly, PD-1 inhibition improved proliferation of circulating tumor\\u2013specific CD8+ T cells and reversed functional exhaustion of specific T cells at tumor sites. In agreement with their tumor specificity, high infiltration of tumors by exhausted cells was predictive of response to therapy and survival in ICB-treated patients with head and neck cancer. Our results showed that PD-1 blockade\\u2013mediated proliferation/reinvigoration of circulating memory T cells and local reversion of exhaustion occur concurrently to control tumors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1298&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7cca53ca7adf0caf78f0c0b4e85c42c4efcfbf8b\",\n",
      "        \"title\": \"Advances and challenges in epigenomic single-cell sequencing applications.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1299&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b76813626391542486e58e5977955dc3cb0e90d\",\n",
      "        \"title\": \"Expanding the scope of antibody rebridging with new pyridazinedione-TCO constructs.\",\n",
      "        \"abstract\": \"Disulfide rebridging methods have recently emerged as a route to hinge region-specific antibody modification, and there exist numerous examples of successful rebridging chemistry applied to clinically relevant human IgG1 antibodies. Here, dibromopyridazinedione disulfide rebridging is adapted to fast trans-cyclooctene/tetrazine (TCO/Tz) bioorthogonal ligations and extended beyond therapeutic human IgG1 antibodies for the first time to include mouse and rat monoclonal antibodies integral to multiplexed analytical diagnostics. In spite of a common architecture, only a subset of antibody host species and IgG isotype subclasses can be rebridged, highlighting the intricate relationship between hinge region sequence, structure, biological activity, and the conjugation chemistry of IgG antibodies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1300&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"56a034e68e3cad0a119b5a7b8168487f377addfc\",\n",
      "        \"title\": \"Single-cell transcriptomic profiling maps monocyte/macrophage transitions after myocardial infarction in mice\",\n",
      "        \"abstract\": \"Rationale Monocytes and macrophages have a critical and dual role in post-ischemic cardiac repair, as they can foster both tissue healing and damage. To decipher how monocytes/macrophages acquire heterogeneous functional phenotypes in the ischemic myocardium, we profiled the gene expression dynamics at the single-cell level in circulating and cardiac monocytes/macrophages following experimental myocardial infarction (MI) in mice. Methods and results Using time-series single-cell transcriptome and cell surface epitope analysis of blood and cardiac monocytes/macrophages, as well as the integration of publicly available and independently generated single-cell RNA-seq data, we tracked the transitions in circulating and cardiac monocyte/macrophage states from homeostatic conditions up to 11 days after MI in mice. We show that MI induces marked and rapid transitions in the cardiac mononuclear phagocyte population, with almost complete disappearance of tissue resident macrophages 1 day after ischemia, and rapid infiltration of monocytes that locally acquire discrete and time-dependent transcriptional states within 3 to 7 days. Ischemic injury induced a shift of circulating monocytes towards granulocyte-like transcriptional features (Chil3, Lcn2, Prtn3). Trajectory inference analysis indicated that while conversion to Ly6Clow monocytes appears as the default fate of Ly6Chi monocytes in the blood, infiltrated monocytes acquired diverse gene expression signatures in the injured heart, notably transitioning to two main MI-associated macrophage populations characterized by MHCIIhi and Trem2hiIgf1hi gene expression signatures. Minor ischemia-associated macrophage populations with discrete gene expression signature suggesting specialized functions in e.g. iron handling or lipid metabolism were also observed. We further identified putative transcriptional regulators and new cell surface markers of cardiac monocyte/macrophage states. Conclusions Altogether, our work provides a comprehensive landscape of circulating and cardiac monocyte/macrophage states and their regulators after MI, and will help to further understand their contribution to post-myocardial infarction heart repair.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1301&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"600e0ea58b199ecfb1e260f9e08091307d574be6\",\n",
      "        \"title\": \"Single-cell RNA sequencing deconvolutes the in vivo heterogeneity of human bone marrow-derived mesenchymal stem cells\",\n",
      "        \"abstract\": \"Bone marrow-derived mesenchymal stem cells (BM-MSCs) are multipotent stromal cells, which have a critical role in the maintenance of skeletal tissues such as bone, cartilage, and the fat found in bone marrow. In addition to providing microenvironmental support for hematopoietic processes, BM-MSCs can differentiate into various mesodermal lineages including osteoblast/osteocyte, chondrocyte, and adipocyte cells that are crucial for bone metabolism. While BM-MSCs have high cell-to-cell heterogeneity in gene expression, the cell subtypes that contribute to this heterogeneity in vivo in humans have not been characterized. To investigate the transcriptional diversity of BM-MSCs, we applied single-cell RNA sequencing (scRNA-seq) on freshly isolated CD271+ BM-derived mononuclear cells (BM-MNCs) from two human subjects. We successfully identified LEPRhiCD45low BM-MSCs within the CD271+ BM-MNC population, and further codified the BM-MSCs into distinct subpopulations corresponding to the osteogenic, chondrogenic, and adipogenic differentiation trajectories, as well as terminal-stage quiescent cells. Biological functional annotations of transcriptomes suggest that osteoblast precursors may induce angiogenesis coupled with osteogenesis, and chondrocyte precursors may have the potential to differentiate into myocytes. We discovered transcripts for several cluster of differentiation (CD) markers that were highly expressed (e.g., CD167b, CD91, CD130 and CD118) or absent (e.g., CD74, CD217, CD148 and CD68) in BM-MSCs and could be novel markers for human BM-MSC purification. This study is the first systematic in vivo dissection of human BM-MSCs cell subtypes at the single-cell resolution, revealing insight into the extent of their cellular heterogeneity and bone homeostasis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1302&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0e7bdc84b8c812f0ec6a1eff5cebef107a5cf7fb\",\n",
      "        \"title\": \"FlexDotPlot: a universal and modular dot plot visualization tool for complex multifaceted data\",\n",
      "        \"abstract\": \"Motivation Dot plots are heatmap-like charts that provide a compact way to simultaneously display two quantitative information by means of dots of different sizes and colours. Despite the popularity of this visualization method, particularly in single-cell RNA-seq studies, existing tools used to make dot plots are limited in terms of functionality and usability. Results We developed FlexDotPlot, an R package for generating dot plots from any type of multifaceted data, including single-cell RNA-seq data. FlexDotPlot provides a universal and easy-to-use solution with a high versatility. An interactive R Shiny application is also available in the FlexDotPlot package allowing non-R users to easily generate dot plots with several tunable parameters. Availability and implementation Source code and detailed manual are available at https://github.com/Simon-Leonard/FlexDotPlot. The Shiny app is available as a stand-alone application within the package.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1303&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b557352665befc038fb212c3482dcb5ca7bbbad9\",\n",
      "        \"title\": \"Spectrally Combined Encoding for Profiling Heterogeneous Circulating Tumor Cells Using a Multifunctional Nanosphere-Mediated Micro\\ufb02uidic Platform.\",\n",
      "        \"abstract\": \"Comprehensive phenotypic profiling of heterogeneous circulating tumor cells (CTCs) at single-cell resolution has great importance for cancer management. However, multiplex biomarker profiling of individual CTCs is greatly hampered by the limitations of current single-cell protein assay techniques. Herein, a novel spectrally combined encoding (SCE) strategy was proposed for multiplex biomarker profiling of single CTCs using a multifunctional nanosphere-mediated micro\\ufb02uidic platform. Different cellular biomarkers uniquely labeled by multifunctional nanosphere barcodes, possessing identical magnetic tags and distinct optical signatures, enabled isolation of heterogeneous CTCs with over 91.6% efficiency and in situ SCE of phenotypes. By further trapping individual CTCs in ordered microstructures on chip, composite single-cell spectral signatures were conveniently and efficiently obtained, allowing reliable spectra-readout for multiplex biomarker profiling. This SCE strategy exhibited great potential in multiplex profiling of heterogeneous CTC phenotypes, offering new avenues for cancer study and precise medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1304&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"861b46d082d1075351e7765921b5ca07bfaa71e8\",\n",
      "        \"title\": \"A pan-cancer blueprint of the heterogeneous tumor microenvironment revealed by single-cell profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1305&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"005182a0494b3aa29271bdfb9a3c99225a721179\",\n",
      "        \"title\": \"Single-cell omics in ageing: a young and growing field\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1306&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"84f04dba995353104e68770be0b4b8b53893415d\",\n",
      "        \"title\": \"CD4+ teff cell heterogeneity: the perspective from single-cell transcriptomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1307&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"15d2bee545e416cdb823c08fab61d5d74bcd0788\",\n",
      "        \"title\": \"The promise of single-cell genomics in plants.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1308&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"010950eaa2a73e3d3223b194bc4db2ea2c94d233\",\n",
      "        \"title\": \"Single-cell omics in ageing: a young and growing field\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1309&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6f1c308d6b71d733a3d1b4ad4ae79570307dbe0f\",\n",
      "        \"title\": \"Lineage tracing meets single-cell omics: opportunities and challenges\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1310&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d692a3163f83fe140f637cad576f4fae878b1e9\",\n",
      "        \"title\": \"Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1311&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4d8e85c918327bfd42cc67875a473b6ecf2555d8\",\n",
      "        \"title\": \"Single-cell RNA sequencing in cardiovascular development, disease and medicine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1312&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a714382fd98dd1acd2a1d5f75ac9878087bc888a\",\n",
      "        \"title\": \"SCITO-seq: single-cell combinatorial indexed cytometry sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1313&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a64b0e63b17e0b2010db440c53dd020e1c287e47\",\n",
      "        \"title\": \"Special function methods for bursty models of transcription.\",\n",
      "        \"abstract\": \"We explore a Markov model used in the analysis of gene expression, involving the bursty production of pre-mRNA, its conversion to mature mRNA, and its consequent degradation. We demonstrate that the integration used to compute the solution of the stochastic system can be approximated by the evaluation of special functions. Furthermore, the form of the special function solution generalizes to a broader class of burst distributions. In light of the broader goal of biophysical parameter inference from transcriptomics data, we apply the method to simulated data, demonstrating effective control of precision and runtime. Finally, we propose and validate a non-Bayesian approach for parameter estimation based on the characteristic function of the target joint distribution of pre-mRNA and mRNA.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1314&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"24609bc512eea90d2981780630f6cbb3150b8485\",\n",
      "        \"title\": \"Single\\u2010Cell Multi\\u2010Omics and Its Prospective Application in Cancer Biology\",\n",
      "        \"abstract\": \"In recent years, the emergence of single\\u2010cell omics technologies, which can profile genomics, transcriptomics, epigenomics, and proteomics, has provided unprecedented insights into characteristics of cancer, enabling higher resolution and accuracy to decipher the cellular and molecular mechanisms relating to tumorigenesis, evolution, metastasis, and immune responses. Single\\u2010cell multi\\u2010omics technologies, which are developed based on the combination of multiple single\\u2010cell mono\\u2010omics technologies, can simultaneously analyze RNA expression, single nucleotide polymorphism, epigenetic modification, or protein abundance, enabling the in\\u2010depth understanding of gene expression regulatory mechanisms. In this review, the state\\u2010of\\u2010the\\u2010art single\\u2010cell multi\\u2010omics technologies are summarized and the prospects of their application in cancer biology are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1315&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"185a6f884841626f589fb0c9a5c728d68ef47088\",\n",
      "        \"title\": \"DAISM-DNNXMBD: Highly accurate cell type proportion estimation with in silico data augmentation and deep neural networks\",\n",
      "        \"abstract\": \"Understanding the immune cell abundance of cancer and other disease-related tissues has an important role in guiding disease treatments. Computational cell type proportion estimation methods have been previously developed to derive such information from bulk RNA sequencing (RNA-seq) data. Unfortunately, our results show that the performance of these methods can be seriously plagued by the mismatch between training data and real-world data. To tackle this issue, we propose the DAISM-DNNXMBD1 pipeline that trains a deep neural network (DNN) with dataset-specific training data populated from a small number of calibrated samples using DAISM, a novel Data Augmentation method with an In Silico Mixing strategy. The evaluation results demonstrate that the DAISM-DNN pipeline outperforms other existing methods consistently and substantially for all the cell types under evaluation on real-world datasets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1316&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd1dddb21ff99eec6590d342878628abf0a30855\",\n",
      "        \"title\": \"A conserved dendritic-cell regulatory program limits antitumour immunity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1317&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"83e57e03b6737259b4e5eccd82f5995896d0436b\",\n",
      "        \"title\": \"Knowledge-based classification of fine-grained immune cell types in single-cell RNA-Seq data with ImmClassifier\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing is an emerging strategy for characterizing the immune cell population in diverse environments including blood, tumor or healthy tissues. While this has traditionally been done with flow or mass cytometry targeting protein expression, scRNA-Seq has several established and potential advantages in that it can profile immune cells and non-immune cells (e.g. cancer cells) in the same sample, identify cell types that lack precise markers for flow cytometry, or identify a potentially larger number of immune cell types and activation states than is achievable in a single flow assay. However, scRNA-Seq is currently limited due to the need to identify the types of each immune cell from its transcriptional profile, which is not only time-consuming but also requires a significant knowledge of immunology. While recently developed algorithms accurately annotate coarse cell types (e.g. T cells vs macrophages), making fine distinctions has turned out to be a difficult challenge. To address this, we developed a machine learning classifier called ImmClassifier that leverages a hierarchical ontology of cell type. We demonstrate that ImmClassifier outperforms other tools (+20% recall, +14% precision) in distinguishing fine-grained cell types (e.g. CD8+ effector memory T cells) with comparable performance on coarse ones. Thus, ImmClassifier can be used to explore more deeply the heterogeneity of the immune system in scRNA-Seq experiments.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1318&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e7f110d97a9436d01d7e13b5e98a46623aba4b83\",\n",
      "        \"title\": \"Single-Cell Approaches to Profile the Response to Immune Checkpoint Inhibitors\",\n",
      "        \"abstract\": \"Novel treatments based upon the use of immune checkpoint inhibitors have an impressive efficacy in different types of cancer. Unfortunately, most patients do not derive benefit or lasting responses, and the reasons for the lack of therapeutic success are not known. Over the past two decades, a pressing need to deeply profile either the tumor microenvironment or cells responsible for the immune response has led investigators to integrate data obtained from traditional approaches with those obtained with new, more sophisticated, single-cell technologies, including high parameter flow cytometry, single-cell sequencing and high resolution imaging. The introduction and use of these technologies had, and still have a prominent impact in the field of cancer immunotherapy, allowing delving deeper into the molecular and cellular crosstalk between cancer and immune system, and fostering the identification of predictive biomarkers of response. In this review, besides the molecular and cellular cancer-immune system interactions, we are discussing how cutting-edge single-cell approaches are helping to point out the heterogeneity of immune cells in the tumor microenvironment and in blood.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1319&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b545c3b3e3db22fa46d1bc115b2d5ff5398e6f8f\",\n",
      "        \"title\": \"Single\\u2010Cell RNA Sequencing in Hematological Diseases\",\n",
      "        \"abstract\": \"Hematological diseases, including leukemia, lymphoma, and multiple myeloma, are characterized by high heterogeneity with diverse cellular subpopulations. Single\\u2010cell RNA sequencing (scRNA\\u2010seq), a transformational technology, provides deep insights into cell\\u2010to\\u2010cell variation in tumor and microenvironment, allows high\\u2010resolution dissection of the pathogenic mechanisms of diseases, and affords potential clinical utilities. Recent developments in single\\u2010cell transcriptomics and associated technologies and their applications in hematological disorders for unraveling cellular subpopulations, disease pathogenesis, patient stratification, and therapeutic responses are summarized.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1320&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0f507e7563d47032c256c371c8be9d9360dac924\",\n",
      "        \"title\": \"The single-cell eQTLGen consortium\",\n",
      "        \"abstract\": \"In recent years, functional genomics approaches combining genetic information with bulk RNA-sequencing data have identified the downstream expression effects of disease-associated genetic risk factors through so-called expression quantitative trait locus (eQTL) analysis. Single-cell RNA-sequencing creates enormous opportunities for mapping eQTLs across different cell types and in dynamic processes, many of which are obscured when using bulk methods. Rapid increase in throughput and reduction in cost per cell now allow this technology to be applied to large-scale population genetics studies. To fully leverage these emerging data resources, we have founded the single-cell eQTLGen consortium (sc-eQTLGen), aimed at pinpointing the cellular contexts in which disease-causing genetic variants affect gene expression. Here, we outline the goals, approach and potential utility of the sc-eQTLGen consortium. We also provide a set of study design considerations for future single-cell eQTL studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1321&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1e07adc414039e4d2eb7d2b219a70cd38283dc2f\",\n",
      "        \"title\": \"Mouse models of neutropenia reveal progenitor-stage-specific defects\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1322&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"18f52ae4503e26195c6d57389fb163eb6fd74b6c\",\n",
      "        \"title\": \"Next-Generation Neuroimmunology: New Technologies to Understand Central Nervous System Autoimmunity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1323&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d827cd67a6d6f44171d1b1903414d8a8f867e0a8\",\n",
      "        \"title\": \"Single-Cell Analysis of Foot-and-Mouth Disease Virus\",\n",
      "        \"abstract\": \"With the rapid development of single-cell technologies, the mechanisms underlying viral infections and the interactions between hosts and viruses are starting to be explored at the single-cell level. The foot-and-mouth-disease (FMD) virus (FMDV) causes an acute and persistent infection that can result in the break-out of FMD, which can have serious effects on animal husbandry. Single-cell techniques have emerged as powerful approaches to analyze virus infection at the resolution of individual cells. In this review, the existing single-cell studies examining FMDV will be systematically summarized, and the central themes of these studies will be presented.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1324&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"60a57678d8c2ca100491abb88d9a332d1b9e22a7\",\n",
      "        \"title\": \"Profiling Cell Signaling Networks at Single-cell Resolution\",\n",
      "        \"abstract\": \"Signaling network responses can be highly heterogeneous across cells in a tissue because of many sources of genetic and non-genetic variance. The emergence of multiplexed single-cell technologies has made it possible to evaluate this heterogeneity. In this review, we categorize currently established single-cell signaling network profiling approaches by their methodology, coverage, and application, and we discuss the advantages and limitations of these technologies. We describe the computational tools for network characterization using single-cell data and discuss potential confounding factors that may affect single-cell analyses. Graphical Abstract Highlights Signaling networks can be highly heterogeneous across cells in a tissue. Various technologies allow analyzing signaling networks at single-cell resolution. The advantages and limitations of each single-cell approach are summarized. Confounding factors in single-cell signaling network analysis are discussed. Signaling networks process intra- and extracellular information to modulate the functions of a cell. Deregulation of signaling networks results in abnormal cellular physiological states and often drives diseases. Network responses to a stimulus or a drug treatment can be highly heterogeneous across cells in a tissue because of many sources of cellular genetic and non-genetic variance. Signaling network heterogeneity is the key to many biological processes, such as cell differentiation and drug resistance. Only recently, the emergence of multiplexed single-cell measurement technologies has made it possible to evaluate this heterogeneity. In this review, we categorize currently established single-cell signaling network profiling approaches by their methodology, coverage, and application, and we discuss the advantages and limitations of each type of technology. We also describe the available computational tools for network characterization using single-cell data and discuss potential confounding factors that need to be considered in single-cell signaling network analyses.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1325&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ea81420e33fcdfe5d21859e5cb6c81984995c0dd\",\n",
      "        \"title\": \"Dissecting the Tumor\\u2013Immune Landscape in Chimeric Antigen Receptor T-cell Therapy: Key Challenges and Opportunities for a Systems Immunology Approach\",\n",
      "        \"abstract\": \"The adoptive transfer of genetically engineered chimeric antigen receptor (CAR) T cells has opened a new frontier in cancer therapy. Unlike the paradigm of targeted therapies, the efficacy of CAR T-cell therapy depends not only on the choice of target but also on a complex interplay of tumor, immune, and stromal cell communication. This presents both challenges and opportunities from a discovery standpoint. Whereas cancer consortia have traditionally focused on the genomic, transcriptomic, epigenomic, and proteomic landscape of cancer cells, there is an increasing need to expand studies to analyze the interactions between tumor, immune, and stromal cell populations in their relevant anatomical and functional compartments. Here, we focus on the promising application of systems biology to address key challenges in CAR T-cell therapy, from understanding the mechanisms of therapeutic resistance in hematologic and solid tumors to addressing important clinical challenges in biomarker discovery and therapeutic toxicity. We propose a systems biology view of key clinical objectives in CAR T-cell therapy and suggest a path forward for a biomedical discovery process that leverages modern technological approaches in systems biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1326&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21c79b83e1d11442f278da3b33ed9a704567346d\",\n",
      "        \"title\": \"Multiplexed single-cell morphometry for hematopathology diagnostics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1327&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"478773fda7817236af6392ac94f4b5b22e57d9c4\",\n",
      "        \"title\": \"Quantitative genetic analysis deciphers the impact of cis and trans regulation on cell-to-cell variability in protein expression levels\",\n",
      "        \"abstract\": \"Identifying the factors that shape protein expression variability in complex multi-cellular organisms has primarily focused on promoter architecture and regulation of single-cell expression in cis. However, this targeted approach has to date been unable to identify major regulators of cell-to-cell gene expression variability in humans. To address this, we have combined single-cell protein expression measurements in the human immune system using flow cytometry with a quantitative genetics analysis. For the majority of proteins whose variability in expression has a heritable component, we find that genetic variants act in trans, with notably fewer variants acting in cis. Furthermore, we highlight using Mendelian Randomization that these variability-Quantitative Trait Loci might be driven by the cis regulation of upstream genes. This indicates that natural selection may balance the impact of gene regulation in cis with downstream impacts on expression variability in trans.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1328&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b6e42ded36416bab1e2d0362bdad885e5c9333a\",\n",
      "        \"title\": \"Microfluidic Single-Cell Omics Analysis.\",\n",
      "        \"abstract\": \"The commonly existed cellular heterogeneity plays a critical role in biological processes such as embryonic development, cell differentiation, and disease progress. Single-cell omics-based heterogeneous study has great significance for identifying different cell populations, discovering new cell types, revealing informative cell features, and uncovering significant interrelationships between cells. Recently, microfluidics has evolved to be a powerful technology for single-cell omics analysis due to its merits of throughput, sensitivity, and accuracy. Herein, the recent advances of microfluidic single-cell omics analysis, including different microfluidic platform designs, lysis strategies, and omics analysis techniques, are reviewed. Representative applications of microfluidic single-cell omics analysis in complex biological studies are then summarized. Finally, a few perspectives on the future challenges and development trends of microfluidic-assisted single-cell omics analysis are discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1329&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7131a0e3d949584e268a04b9d84263803e7ada59\",\n",
      "        \"title\": \"The multiplex model of the genetics of Alzheimer\\u2019s disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1330&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"66d334d6562bf5362f88e9363837d6dfb8836021\",\n",
      "        \"title\": \"Normalizing and denoising protein expression data from droplet-based single cell profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1331&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"15b2b5a411df824647421c88b3b43def8ca58d75\",\n",
      "        \"title\": \"Broad immune activation underlies shared set point signatures for vaccine responsiveness in healthy individuals and disease activity in lupus patients\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1332&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"620174777a4d2c4683ff1f13f02f92e1f815a070\",\n",
      "        \"title\": \"Tracing tumorigenesis in a solid tumor model at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1333&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aded319ab15712075c030e459d9763fa38abcaed\",\n",
      "        \"title\": \"Dissolvable Polyacrylamide Beads for High\\u2010Throughput Droplet DNA Barcoding\",\n",
      "        \"abstract\": \"Droplet\\u2010based single cell sequencing technologies, such as inDrop, Drop\\u2010seq, and 10X Genomics, are catalyzing a revolution in the understanding of biology. Barcoding beads are key components for these technologies. What is limiting today are barcoding beads that are easy to fabricate, can efficiently deliver primers into drops, and thus achieve high detection efficiency. Here, this work reports an approach to fabricate dissolvable polyacrylamide beads, by crosslinking acrylamide with disulfide bridges that can be cleaved with dithiothreitol. The beads can be rapidly dissolved in drops and release DNA barcode primers. The dissolvable beads are easy to synthesize, and the primer cost for the beads is significantly lower than that for the previous barcoding beads. Furthermore, the dissolvable beads can be loaded into drops with >95% loading efficiency of a single bead per drop and the dissolution of beads does not influence reverse transcription or the polymerase chain reaction (PCR) in drops. Based on this approach, the dissolvable beads are used for single cell RNA and protein analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1334&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4fa6261640ce094dc68224171a562ae84ca79147\",\n",
      "        \"title\": \"Immunology in the Era of Single-Cell Technologies.\",\n",
      "        \"abstract\": \"Immune cells are characterized by diversity, specificity, plasticity, and adaptability-properties that enable them to contribute to homeostasis and respond specifically and dynamically to the many threats encountered by the body. Single-cell technologies, including the assessment of transcriptomics, genomics, and proteomics at the level of individual cells, are ideally suited to studying these properties of immune cells. In this review we discuss the benefits of adopting single-cell approaches in studying underappreciated qualities of immune cells and highlight examples where these technologies have been critical to advancing our understanding of the immune system in health and disease. Expected final online publication date for the Annual Review of Immunology, Volume 38 is April 26, 2020. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1335&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c324852e1af0b6d10ccc92ddd133e530a0ed2429\",\n",
      "        \"title\": \"Protein velocity and acceleration from single-cell multiomics experiments\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1336&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2ae1ccb8f9739e158d9cd87eae3acf207465e704\",\n",
      "        \"title\": \"Single-cell analysis targeting the proteome\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1337&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9608ade74f562a91b90d9ded281a795fda486302\",\n",
      "        \"title\": \"Extraembryonic endoderm cells induce neuroepithelial tissue in gastruloids\",\n",
      "        \"abstract\": \"Stem-cell derived in vitro systems, such as organoids or embryoids, hold great potential for modeling in vivo biology and engineering living systems with novel functions. To unlock that potential, we need new ways to elicit higher-level organization. Here we show that adding extraembryonic endoderm (XEN) cells to mouse gastruloids leads to the formation of neural epithelia. By single-cell RNA-seq, imaging and differentiation experiments, we demonstrate the neural characteristics and spatial patterning of the epithelial tissue. We further show that the XEN cells differentiate reciprocally to a visceral endoderm-like state. Finally, we demonstrate that local inhibition of WNT signaling and production of a basement membrane by the XEN cells underlie the formation of the neuroepithelial tissue. In summary, we establish \\u201cXEN Enhanced Gastruloids\\u201d (XEGs) to explore heterotypic cellular interactions as a means to achieve complex, tissue-level organization in vitro.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1338&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d20fad81db040b13029458198e6a7b04ad57649\",\n",
      "        \"title\": \"Integrating the \\\"Immunome\\\" in the Stratification of Myelodysplastic Syndromes and Future Clinical Trial Design.\",\n",
      "        \"abstract\": \"Myelodysplastic syndromes (MDS) are characterized by ineffective hematopoiesis and often include a dysregulation and dysfunction of the immune system. In the context of population aging, MDS incidence is set to increase substantially, with exponential increases in health care costs, given the limited and expensive treatment options for these patients. Treatment selection is mainly based on calculated risk categories according to a Revised International Prognostic Scoring System (IPSS-R). However, although IPSS-R is an excellent predictor of disease progression, it is an ineffective predictor of response to disease-modifying therapies. Redressing these unmet needs, the \\\"immunome\\\" is a key, multifaceted component in the initiation and overall response against malignant cells in MDS, and the current omission of immune status monitoring may in part explain the insufficiencies of current prognostic stratification methods. Nevertheless, integrating these and other recent molecular advances into clinical practice proves difficult. This review highlights the complexity of immune dysregulation in MDS pathophysiology and the fine balance between smoldering inflammation, adaptive immunity, and somatic mutations in promoting or suppressing malignant clones. We review the existing knowledge and discuss how state-of-the-art immune monitoring strategies could potentially permit novel patient substratification, thereby empowering practical predictions of response to treatment in MDS. We propose novel multicenter studies, which are needed to achieve this goal.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1339&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c2292055a47c3cba2fa07f17235ad68a3f7005d1\",\n",
      "        \"title\": \"Tumor cell heterogeneity profiling using single-cell analysis\",\n",
      "        \"abstract\": \"Cancer is a diverse disease with large variations between tumor types and patients regarding tumor progression and prognosis. Additionally, most individual tumors are heterogeneous, containing subpopulations of cells with various characteristics. Numerous factors affect the differences observed between tumor cells, such as variations in genetics, epigenetics, cellular states and the microenvironment surrounding the individual cells. One clinically relevant subpopulation, commonly referred to as cancer stem cells, consists of cells with stem cell characteristics. These are present in many tumor types and are known to be important for tumor development and treatment resistance. The tumor microenvironment is a key factor affecting the cellular phenotype, including the cancer stem cell subpopulation. Analysis at cell population level will not capture the true variations between individual cells. Instead, single-cell analysis offers new means to study and understand cellular and molecular differences between tumor subpopulations. The main objective of this thesis was to study tumor cell heterogeneity in myxoid liposarcoma and breast cancer with the help of single-cell gene expression analysis methods. We could generate a flexible workflow to measure gene expression, including the assessment of total mRNA amounts in each cell, using several diverse approaches developed from already existing protocols. Subsequently, we combined a number of functional cell culture methods to enrich for tumor cells with characteristic cellular properties together with single-cell gene expression profiling methods, to match phenotype with the corresponding transcription pattern. Single-cell analysis of myxoid liposarcoma cells, sorted based on the cell-cycle, identified a number of genes previously not reported as cell-cycle regulated and defined two subgroups of cells within the G1 phase. In the same tumor type, we identified a subpopulation of cells with cancer stem celland chemotherapy resistance properties associated with an active JAK-STAT signaling pathway. Here, a combination treatment of chemotherapy and JAK-STAT inhibition was in vitro shown to be more effective against tumor cells than chemotherapy alone. In breast cancer cells, we identified a number of potential biomarkers overexpressed in a subpopulation of cells with cancer stem cell characteristics. We also developed a new in vivo-like culture system based on decellularized human tumors to study the effect of the microenvironment on breast cancer cells. We demonstrated that the gene expression profiles of cells cultured in these patient-derived scaffolds closely mimic the profiles of in vivo cells. Furthermore, gene expression patterns changed differently depending on the patient-derived scaffold, which could be linked to patient recurrence. In conclusion, we developed single-cell analysis methods as well as a new in vivo-like model system. Furthermore, we identified genes and pathways connected to different subpopulations of myxoid liposarcoma or breast cancer cells that potentially can be used as biomarkers and future drug targets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1340&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"02b126681b52d2b9a55c17b26ba3253ed29cbebb\",\n",
      "        \"title\": \"Identifying Informative Gene Modules Across Modalities of Single Cell Genomics\",\n",
      "        \"abstract\": \"Two fundamental aims that emerge when analyzing single-cell RNA-seq data are that of identifying which genes vary in an informative manner and determining how these genes organize into modules. Here we propose a general approach to these problems that operates directly on a given metric of cell-cell similarity, allowing for its integration with any method (linear or non linear) for identifying the primary axes of transcriptional variation between cells. Additionally, we show that when using multimodal data, our procedure can be used to identify genes whose expression reflects alternative notions of similarity between cells, such as physical proximity in a tissue or clonal relatedness in a cell lineage tree. In this manner, we demonstrate that while our method, called Hotspot, is capable of identifying genes that reflect nuanced transcriptional variability between T helper cells, it can also identify spatially-dependent patterns of gene expression in the cerebellum as well as developmentally-heritable expression signatures during embryogenesis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1341&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4cad823f95a3a0f6a1be3331678c6204ca697b8d\",\n",
      "        \"title\": \"CD8+ T cell states in human cancer: insights from single-cell analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1342&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3dd81e22bfc6c2a34f8162f58e56344a659df58d\",\n",
      "        \"title\": \"ZipSeq: barcoding for real-time mapping of single cell transcriptomes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1343&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"99900312ee30fd59e89bb9f7b589ec851aaa2a5b\",\n",
      "        \"title\": \"Distribution and storage of inflammatory memory in barrier tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1344&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d1283675db5dc1d30a59e65576c493ccf969d0c7\",\n",
      "        \"title\": \"Artificial-cell-type aware cell-type classification in CITE-seq\",\n",
      "        \"abstract\": \"Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq), couples the measurement of surface marker proteins with simultaneous sequencing of mRNA at single cell level, which brings accurate cell surface phenotyping to single cell transcriptomics. Unfortunately, multiplets in CITE-seq datasets create artificial cell types and complicates the automation of cell surface phenotyping. We propose CITE-sort, an artificial-cell-type aware surface marker clustering method for CITE-seq. CITE-sort is aware of and is robust to multiplet-induced artificial cell types. We benchmarked CITE-sort with real and simulated CITE-seq datasets and compared CITE-sort against canonical clustering methods. We show that CITE-sort produces the best clustering performance across the board. CITE-sort not only accurately identifies real biological cell types but also consistently and reliably separates multiplet-induced artificial-cell-type droplet clusters from real biological-cell-type droplet clusters. In addition, CITE-sort organizes its clustering process with a binary tree, which facilitates easy interpretation and verification of its clustering result and simplifies cell type annotation with domain knowledge in CITE-seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1345&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d71e8fcd8172f564ca3ec847b03471598b6d1f47\",\n",
      "        \"title\": \"Single-cell genomic approaches for developing the next generation of immunotherapies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1346&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6f8549b7a2ecfab5926e02231bab91bb319a2534\",\n",
      "        \"title\": \"High-dimensional immune-profiling in cancer: implications for immunotherapy\",\n",
      "        \"abstract\": \"Immunotherapy is a rapidly growing field for cancer treatment. In contrast to conventional cancer therapies, immunotherapeutic strategies focus on reactivating the immune system to mount an antitumor response. Despite the encouraging outcome in clinical trials, a large proportion of patients still do not respond to treatment and many experience different degrees of immune-related adverse events. Furthermore, it is now increasingly appreciated that even many conventional cancer therapies such as radiotherapy could have a positive impact on the host immune system for better clinical response. Hence, there is a need to better understand tumor immunity in order to design immunotherapeutic strategies, especially evidence-based combination therapies, for improved clinical outcomes. With this aim, cancer research turned its attention to profiling the immune contexture of either the tumor microenvironment (TME) or peripheral blood to uncover mechanisms and biomarkers which might aid in precision immunotherapeutics. Conventional technologies used for this purpose were limited by the depth and dimensionality of the data. Advances in newer techniques have, however, greatly improved the breadth and depth, as well as the quantity and quality of data that can be obtained. The result of these advances is a wealth of new information and insights on how the TME could be affected by various immune cell-types, and how this might in turn impact the clinical outcome of cancer patients . We highlight herein some of the high-dimensional technologies currently employed in immune profiling in cancer and summarize the insights and potential benefits they could bring in designing better cancer immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1347&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dea55a7e38d1ec16c0c3d99d82a7b1db03b9b433\",\n",
      "        \"title\": \"Integration of Flow Cytometry and Single Cell Sequencing.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1348&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c09c2666f5dd046ec9dc2cf2b9dde03cb5e5e702\",\n",
      "        \"title\": \"Enrichment of melanoma-associated T cells in 6-thioguanine-resistant T cells from metastatic melanoma patients\",\n",
      "        \"abstract\": \"Supplemental Digital Content is available in the text. This study examines whether 6-thioguanine resistant T cells (mutant) from metastatic melanoma patients are enriched for melanoma-associated T cells compared to T cells obtained analogously without thioguanine selection (wild-type). Melanoma-associated antigen pentamer staining was performed on 5 tumour and 9 peripheral blood samples from metastatic melanoma patients. T cell receptor beta chain repertoire was examined via Sanger sequencing of mutant and wild-type in blood and tumour from metastatic melanoma patients at times of tumour progression (n = 8) and via Illumina sequencing in tumour derived T cells and in uncultured T cells (uncultured), wild-type and mutant from blood before and after immune checkpoint blockade (n = 1). Mutant from tumour (3 of 5; P < 0.001), but not blood (0 of 9), were enriched compared to wild-type for binding melanoma-associated antigen pentamers. T cell receptor beta analysis in patients with tumour progression (n = 8) detected increased melanoma associated T cells in mutant compared to wild-type from blood (Monte Carlo P = 10\\u22127). Comparison of blood samples before and after immune checkpoint blockade with prior tumor from one metastatic melanoma patient detected increased T cell receptor beta sharing between tumour and mutant compared to tumour and wild-type or tumour and uncultured: 11.0% (72/656), 1.5% (206/13 639) and 1.3% (381/29 807), respectively (Monte Carlo P = 10\\u22127 for mutant versus wild-type and mutant versus uncultured). These data demonstrate that mutant in metastatic melanoma patients are enriched for melanoma-associated T cells and are candidate probes to study in vivo melanoma-reactive T cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1349&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4616fa00e4fcbb14dc7bbfe3401b9e028d6b3177\",\n",
      "        \"title\": \"New Developments in Transcriptomic Analysis of Synovial Tissue\",\n",
      "        \"abstract\": \"Transcriptomic technologies are constantly changing and improving, resulting in an ever increasing understanding of gene expression in health and disease. These technologies have been used to investigate the pathological changes occurring in the joints of rheumatoid arthritis patients, leading to discoveries of disease mechanisms, and novel potential therapeutic targets. Microarrays were initially used on both whole tissue and cell subsets to investigate research questions, with bulk RNA sequencing allowing for further elaboration of these findings. A key example is the classification of pathotypes in rheumatoid arthritis using RNA sequencing that had previously been discovered using microarray and histology. Single-cell sequencing has now delivered a step change in understanding of the diversity and function of subpopulations of cells, in particular synovial fibroblasts. Future technologies, such as high resolution spatial transcriptomics, will enable step changes integrating single cell transcriptomic and geographic data to provide an integrated understanding of synovial pathology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1350&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bee8c7967299c4ee7fb0c4df575241e9d8840c91\",\n",
      "        \"title\": \"Preparation of single- and double-oligonucleotide antibody conjugates and their application for protein analytics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1351&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"053597232d0330bdb891ac318600a4765f39d6c7\",\n",
      "        \"title\": \"Applications of Next-Generation Sequencing in Neoantigen Prediction and Cancer Vaccine Development.\",\n",
      "        \"abstract\": \"Next-generation sequencing has changed the face of cancer immunotherapy research by making tumor-specific cancer vaccines a reality. Whole exome sequencing and RNA sequencing combined with bioinformatic pipelines allow the prediction of neoantigen targets for cancer vaccines. In this review, we discuss the preclinical and early clinical evidence for cancer vaccines; describe methods and challenges in neoantigen prediction; and summarize emerging new technologies that will improve neoantigen cancer vaccine development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1352&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b340be0e4a7ee67e9dbbdb30ba1a445947f1c66c\",\n",
      "        \"title\": \"Single-cell analyses uncover granularity of muscle stem cells\",\n",
      "        \"abstract\": \"Satellite cells are the main muscle-resident cells responsible for muscle regeneration. Much research has described this population as being heterogeneous, but little is known about the different roles each subpopulation plays. Recent advances in the field have utilized the power of single-cell analysis to better describe and functionally characterize subpopulations of satellite cells as well as other cell groups comprising the muscle tissue. Furthermore, emerging technologies are opening the door to answering as-yet-unresolved questions pertaining to satellite cell heterogeneity and cell fate decisions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1353&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"76e6c70ba24c5dd28c525443d22cc40ce2519f72\",\n",
      "        \"title\": \"ILoReg enables high-resolution cell population identification from single-cell RNA-seq data\",\n",
      "        \"abstract\": \"Single-cell RNA-seq allows researchers to identify cell populations based on unsupervised clustering of the transcriptome. However, subpopulations can have only subtle transcriptomic differences and the high dimensionality of the data makes their identification challenging. We introduce ILoReg (https://github.com/elolab/iloreg), an R package implementing a new cell population identification method that achieves high differentiation resolution through a probabilistic feature extraction step that is applied before clustering and visualization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1354&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"900be522a8dc479d2ffcac96c7841d21e3d2c3b3\",\n",
      "        \"title\": \"BREM-SC: a bayesian random effects mixture model for joint clustering single cell multi-omics data\",\n",
      "        \"abstract\": \"Droplet-based single cell transcriptome sequencing (scRNA-seq) technology, largely represented by the 10X Genomics Chromium system, is able to measure the gene expression from tens of thousands of single cells simultaneously. More recently, coupled with the cutting-edge Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq), the droplet-based system has allowed for immunophenotyping of single cells based on cell surface expression of specific proteins together with simultaneous transcriptome profiling in the same cell. Despite the rapid advances in technologies, novel statistical methods and computational tools for analyzing multi-modal CITE-Seq data are lacking. In this study, we developed BREM-SC, a novel Bayesian Random Effects Mixture model that jointly clusters paired single cell transcriptomic and proteomic data. Through simulation studies and analysis of public and in-house real data sets, we successfully demonstrated the validity and advantages of this method in fully utilizing both types of data to accurately identify cell clusters. In addition, as a probabilistic model-based approach, BREM-SC is able to quantify the clustering uncertainty for each single cell. This new method will greatly facilitate researchers to jointly study transcriptome and surface proteins at the single cell level to make new biological discoveries, particularly in the area of immunology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1355&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc8d0bd47b5482649e3348aad37e50db519ea26a\",\n",
      "        \"title\": \"Multiplexed Single-cell Metabolic Profiles Organize the Spectrum of Cytotoxic Human T Cells\",\n",
      "        \"abstract\": \"Cellular metabolism regulates immune cell activation, differentiation and effector functions to the extent that its perturbation can augment immune responses. However, the analytical technologies available to study cellular metabolism lack single-cell resolution, obscuring metabolic heterogeneity and its connection to immune phenotype and function. To that end, we utilized high-dimensional, antibody-based technologies to simultaneously quantify the single-cell metabolic regulome in combination with phenotypic identity. Mass cytometry (CyTOF)-based application of this approach to early human T cell activation enabled the comprehensive reconstruction of the coordinated metabolic remodeling of na\\u00efve CD8+ T cells and aligned with conventional bulk assays for glycolysis and oxidative phosphorylation. Extending this analysis to a variety of tissue-resident immune cells revealed tissue-restricted metabolic states of human cytotoxic T cells, including metabolically repressed subsets that expressed CD39 and PD1 and that were enriched in colorectal carcinoma versus healthy adjacent tissue. Finally, combining this approach with multiplexed ion beam imaging by time-of-flight (MIBI-TOF) demonstrated the existence of spatially enriched metabolic neighborhoods, independent of cell identity and additionally revealed exclusion of metabolically repressed cytotoxic T cell states from the tumor-immune boundary in human colorectal carcinoma. Overall, we provide an approach that permits the robust approximation of metabolic states in individual cells along with multimodal analysis of cell identity and functional characteristics that can be applied to human clinical samples to study cellular metabolism how it may be perturbed to affect immunological outcomes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1356&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6908a42b6736b314915538ada5eb6d8e9830c182\",\n",
      "        \"title\": \"Unraveling the Plastic Peripheral Neuroimmune Interactome\",\n",
      "        \"abstract\": \"Sensory neurons and immune cells share a common microenvironmental niche for surveying tissue integrity. The immune and nervous systems both sense deviations in homeostasis and initiate protective responses and, upon malfunction, also jointly contribute to disease. Barrier tissues are heavily innervated by nociceptors, the sensory neurons that detect noxious stimuli, leading to pain and itch. The same tissues are also home to diverse immune cells that respond to infections and injury. The physical proximity of nociceptors and immune cells allows for direct local interactions between the two, independent of the CNS. We discuss in this study their ligand\\u2013receptor\\u2013based interactions and propose the need to shift from studying individual neuroimmune interactions to exploring the reciprocal neuroimmune interaction network in its entirety: the \\u201cneuroimmune interactome.\\u201d Identification of the nature of the interactome in health and its plasticity in disease will unravel the functional consequences of interactions between nociceptors and immune cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1357&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5aa6006a67a683b950f343233c6b2ec9af42faf9\",\n",
      "        \"title\": \"Advancing Stem Cell Research through Multimodal Single-Cell Analysis.\",\n",
      "        \"abstract\": \"Technological advances play a key role in furthering our understanding of stem cell biology, and advancing the prospects of regenerative therapies. Highly parallelized methods, developed in the last decade, can profile DNA, RNA, or proteins in thousands of cells and even capture data across two or more modalities (multiomics). This allows unbiased and precise definition of molecular cell states, thus allowing classification of cell types, tracking of differentiation trajectories, and discovery of underlying mechanisms. Despite being based on destructive techniques, novel experimental and bioinformatic approaches enable embedding and extraction of temporal information, which is essential for deconvolution of complex data and establishing cause and effect relationships. Here, we provide an overview of recent studies pertinent to stem cell biology, followed by an outlook on how further advances in single-cell molecular profiling and computational analysis have the potential to shape the future of both basic and translational research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1358&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"706f6c74a120215e3ebc4443be66b46fdabf0677\",\n",
      "        \"title\": \"Are We meeting the Promise of Endotypes and Precision Medicine in Asthma.\",\n",
      "        \"abstract\": \"While the term \\\"asthma\\\" has long been known to describe heterogeneous groupings of patients, only recently have data evolved which enable a molecular understanding of the clinical differences. The evolution of transcriptomics (and other 'omics platforms) and improved statistical analyses in combination with large clinical cohorts opened the door for molecular characterization of pathobiologic processes associated with a range of asthma patients. When linked with data from animal models and clinical trials of targeted biologic therapies emerging distinctions arose between patients with and without elevations in Type-2 immune and inflammatory pathways, leading to the confirmation of a broad categorization of Type-2 Hi asthma. Differences in the ratios, sources and location of Type-2 cytokines and their relation to additional immune pathway activation, appear to distinguish several different (sub) molecular phenotypes, and perhaps endotypes of Type-2 HI asthma, which respond differently to broad and targeted anti-inflammatory therapies. Asthma in the absence of Type-2 inflammation is much less well defined, without clear biomarkers, but is generally linked with poor responses to corticosteroids. Integration of \\\"big data\\\" from large cohorts, over time, using machine learning approaches, combined with validation and iterative learning in animal (and human) model systems is needed to identify the biomarkers and tightly defined molecular phenotypes/endotypes required to fulfill the promise of precision medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1359&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7807a51af150778d17c2abc81fb3f896fe9820fa\",\n",
      "        \"title\": \"Advances in single-cell transcriptome sequencing technology and its application in hematopoietic system research\",\n",
      "        \"abstract\": \"Recent years have witnessed the rapid development of single cell sequencing technology, which could reveal the mystery of the heterogeneity of cell populations. Single-cell transcriptome sequencing technology offers us an alternative method to classify or characterize cells at the molecular level, which enables its broad application in various fields of research, such as developmental biology, neuroscience, hematology, immunity and cancer. In this paper, we will review the main development and trends of latest single cell transcriptome sequencing technology, and its applications in hematopoietic system is presented.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1360&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"392888bcc5d2e3d6f4a704d2adfefcae34d5e4f8\",\n",
      "        \"title\": \"Computational detection of doublets in single-cell RNA sequencing\",\n",
      "        \"abstract\": \"Questo lavoro si occupa di uno dei problemi nati con il sequenziamento single cell, il problema delle doublets. Esso e rappresentato dall'aggregazione o cattura insieme di due cellule durante l'analisi. Le doublets sono fattori di \\\"confusione\\\" \\u009dche possono portare a falsi risultati nell'analisi. Questa tesi studia due metodi computazionali, DoubletFinder e Doubletdecon, che classificano tali cellule.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1361&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5bb402fd4d109ed0e71924216485e7d648e88ac4\",\n",
      "        \"title\": \"A flexible microfluidic system for single-cell transcriptome profiling elucidates phased transcriptional regulators of cell cycle\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1362&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3dc14383adc1fcb47c1faead4bca8154f6d07af0\",\n",
      "        \"title\": \"The therapeutic landscape for cells engineered with chimeric antigen receptors\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1363&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b2d376c5cbaaef138b5dac1fe952272e2c134e08\",\n",
      "        \"title\": \"The emergence and promise of single-cell temporal-omics approaches.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1364&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a5ec35eacce13d5214346f1902b890cc04dd4f58\",\n",
      "        \"title\": \"Beyond mass spectrometry, the next step in proteomics\",\n",
      "        \"abstract\": \"This survey illuminates alternatives for sequencing proteins with the brightest prospects for displacing mass spectrometry. Proteins can be the root cause of a disease, and they can be used to cure it. The need to identify these critical actors was recognized early (1951) by Sanger; the first biopolymer sequenced was a peptide, insulin. With the advent of scalable, single-molecule DNA sequencing, genomics and transcriptomics have since propelled medicine through improved sensitivity and lower costs, but proteomics has lagged behind. Currently, proteomics relies mainly on mass spectrometry (MS), but instead of truly sequencing, it classifies a protein and typically requires about a billion copies of a protein to do it. Here, we offer a survey that illuminates a few alternatives with the brightest prospects for identifying whole proteins and displacing MS for sequencing them. These alternatives all boast sensitivity superior to MS and promise to be scalable and seem to be adaptable to bioinformatics tools for calling the sequence of amino acids that constitute a protein.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1365&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cfefae0a4e2f244e19ac83eb16c992014b58b365\",\n",
      "        \"title\": \"Single-cell multimodal omics: the power of many\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1366&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"767960c5e50bba05f7e0565245a288f72ca95c41\",\n",
      "        \"title\": \"The secret life of cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1367&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"95b3f268797216b9b18a6394017df9bb1620c7ac\",\n",
      "        \"title\": \"Computational methods for single-cell omics across modalities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1368&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f1140157d31a8a78df00293f706cae304b56d53a\",\n",
      "        \"title\": \"Designing signaling environments to steer transcriptional diversity in neural progenitor cell populations\",\n",
      "        \"abstract\": \"Stem and progenitor populations within developing embryos are diverse, composed of different subpopulations of precursor cells with varying developmental potential. How these different subpopulations are coordinately regulated by their signaling environments is not well understood. In this paper we develop a framework for controlling progenitor population structure in cell culture using high-throughput single cell mRNA-seq and computational analysis. We find that the natural transcriptional diversity of neural stem cell populations from the developing mouse brain collapses during in vitro culture. Cell populations are depleted of committed neuroblast progenitors and become dominated by a single pre-astrocytic cell population. By analyzing the response of neural stem cell populations to forty combinatorial signaling conditions, we demonstrate that signaling environments can restructure cell populations by modulating the relative abundance of pre-astrocytic and pre-neuronal subpopulations according to a simple log-linear model. Our work demonstrates that single-cell RNA-seq can be applied to learn how to modulate the diversity of stem cell populations, providing a new strategy for population-level stem cell control. Highlights Natural progenitor diversity in the brain collapses during in vitro culture to a single progenitor type Loss of progenitor diversity alters fate potential of cells during differentiation Large scale single-cell signaling screen identifies signals that reshape population structure towards neuronal cell types Signals regulate population structure according to a simple log-linear model GRAPHICAL ABSTRACT\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1369&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"867cc2b4655c0e4719a1d019d104d98d599564fb\",\n",
      "        \"title\": \"Single-cell sequencing in hematology.\",\n",
      "        \"abstract\": \"PURPOSE OF REVIEW\\nIn this review, we highlight key recent insights into hematopoiesis and hematological malignancies through the application of novel single-cell approaches. We particularly focus on biological insights made through the study of stem/progenitors cells in myeloid malignancy at single-cell resolution.\\n\\n\\nRECENT FINDINGS\\nBulk molecular profiling of hematological malignancies by next generation sequencing techniques has provided major insights into the molecular pathogenesis of blood cancers. This technology is now routinely implemented in advanced clinical diagnostics, leading to the development of novel targeted therapies. However, bulk genetic analysis can obscure key aspects of intratumoral heterogeneity which underlies critical disease events, such as treatment resistance and clonal evolution. The past few years have seen an explosion of novel techniques to analyze RNA, DNA, and protein expression at the single-cell level, providing unprecedented insight into cellular heterogeneity.\\n\\n\\nSUMMARY\\nGiven the ease of accessibility of liquid tumor biopsies, hematology is well positioned to move novel single-cell techniques towards routine application in the clinic. The present review sets out to discuss current and potential future applications for this technology in the management of patients with hematological cancers.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1370&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4e9fd89acb646a694bf0fb2208b009975ed97323\",\n",
      "        \"title\": \"Single\\u2010cell analysis of CD8 T lymphocyte diversity during adaptive immunity\",\n",
      "        \"abstract\": \"An effective adaptive immune response to microbial infection relies on the generation of heterogeneous T lymphocyte fates and functions. CD8 T lymphocytes play a pivotal role in mediating immediate and long\\u2010term protective immune responses to intracellular pathogen infection. Systems\\u2010based analysis of the immune response to infection has begun to identify cell fate determinants and the molecular mechanisms underpinning CD8 T lymphocyte diversity at single\\u2010cell resolution. Resolving CD8 T lymphocyte heterogeneity during adaptive immunity highlights the advantages of single\\u2010cell technologies and computational approaches to better understand the ontogeny of CD8 T cellular diversity following infection. Future directions of integrating single\\u2010cell multiplex approaches capitalize on the importance of systems biology in the understanding of immune CD8 T cell differentiation and functional diversity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1371&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"465b2e2ab17b15ddaae3a6050ab82564abe9c9e5\",\n",
      "        \"title\": \"Design and application of single-cell RNA sequencing to study kidney immune cells in lupus nephritis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1372&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c44f0e15edf077c5a456f461ea5962e34eb48404\",\n",
      "        \"title\": \"Identification of human tendon cell populations in healthy and diseased tissue using combined single cell transcriptomics and proteomics\",\n",
      "        \"abstract\": \"The long-term morbidity of tendon disease in an increasingly active and ageing population represents a growing area of unmet clinical need. Tendon disorders commonly affect the lower limb, ranging from isolated tendon rupture to degenerative joint deformity. In the absence of valid animal models of chronic tendinopathy, traditional methods to isolate and identify crucial sub types involved in disease are limited by the heterogeneity of tendon cells, by their relative paucity in tissue and by the density of the surrounding collagen matrix. To overcome this, we have used next generation CITE-sequencing to combine surface proteomics with in-depth, unbiased gene expression analysis of single cells derived ex vivo from healthy and diseased tendon. For the first time we have been able to show that human tendon consists of at least eight sub-populations of cells. In addition to endothelial cells, Tc cells, and macrophages, there are five distinct tenocyte populations expressing COL1A genes. These consist of a population of resident cells expressing microfibril associated genes (FBN1, VCAN, DCN, EMILIN1, MFAP5), a group of SCX+ cells co-expressing high levels of pro-inflammatory markers, a population of APOD+ fibro-adipogenic progenitors (FAPs), TPPP3/PRG4+ chondrogenic cells (COMP, CILP, PRG4) and ITGA7+ Smooth Muscle-Mesenchymal Cells, recently described in mouse muscle but not, as yet, in human tendon. Surface proteomic analysis identified markers by which these sub-classes could be isolated and targeted in future. In comparison to healthy tendon, diseased tendon harboured a greater proportion of SCX+ tendon cells and these expressed high levels of pro-inflammatory markers including CXCL1, CXCL6, CXCL8, PDPN and previously undescribed PTX3. We were also able to show that whereas disease associated genes such as CD248 and PDPN were expressed by COL1+ tenocytes, IL33 was restricted to endothelial cells of chronically diseased tendon.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1373&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0adc5be9a08907b595bc5bd56a337eba1bfc961a\",\n",
      "        \"title\": \"Tracking the immune response with single-cell genomics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1374&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd644cab29bebc4c0f7a5edd176e111501acc011\",\n",
      "        \"title\": \"Using single-cell technologies to map the human immune system \\u2014 implications for nephrology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1375&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8584ebafab70ae455823ebd915fcc6620e8fb266\",\n",
      "        \"title\": \"Functional diversity of small nucleolar RNAs\",\n",
      "        \"abstract\": \"Abstract Small nucleolar RNAs (snoRNAs) are short non-protein-coding RNAs with a long-recognized role in tuning ribosomal and spliceosomal function by guiding ribose methylation and pseudouridylation at targeted nucleotide residues of ribosomal and small nuclear RNAs, respectively. SnoRNAs are increasingly being implicated in regulation of new types of post-transcriptional processes, for example rRNA acetylation, modulation of splicing patterns, control of mRNA abundance and translational efficiency, or they themselves are processed to shorter stable RNA species that seem to be the principal or alternative bioactive isoform. Intriguingly, some display unusual cellular localization under exogenous stimuli, or tissue-specific distribution. Here, we discuss the new and unforeseen roles attributed to snoRNAs, focusing on the presumed mechanisms of action. Furthermore, we review the experimental approaches to study snoRNA function, including high resolution RNA:protein and RNA:RNA interaction mapping, techniques for analyzing modifications on targeted RNAs, and cellular and animal models used in snoRNA biology research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1376&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1274d84e2d14978b0b7cff2c08ce896649af66d1\",\n",
      "        \"title\": \"Understanding and Modulating Immunity With Cell Reprogramming\",\n",
      "        \"abstract\": \"Cell reprogramming concepts have been classically developed in the fields of developmental and stem cell biology and are currently being explored for regenerative medicine, given its potential to generate desired cell types for replacement therapy. Cell fate can be experimentally reversed or modified by enforced expression of lineage specific transcription factors leading to pluripotency or attainment of another somatic cell type identity. The possibility to reprogram fibroblasts into induced dendritic cells (DC) competent for antigen presentation creates a paradigm shift for understanding and modulating the immune system with direct cell reprogramming. PU.1, IRF8, and BATF3 were identified as sufficient and necessary to impose DC fate in unrelated cell types, taking advantage of Clec9a, a C-type lectin receptor with restricted expression in conventional DC type 1. The identification of such minimal gene regulatory networks helps to elucidate the molecular mechanisms governing development and lineage heterogeneity along the hematopoietic hierarchy. Furthermore, the generation of patient-tailored reprogrammed immune cells provides new and exciting tools for the expanding field of cancer immunotherapy. Here, we summarize cell reprogramming concepts and experimental approaches, review current knowledge at the intersection of cell reprogramming with hematopoiesis, and propose how cell fate engineering can be merged to immunology, opening new opportunities to understand the immune system in health and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1377&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"21e219c93afd763e8dd3210c84927d9835e4cc6a\",\n",
      "        \"title\": \"Dissecting the human immune system with single cell RNA sequencing technology\",\n",
      "        \"abstract\": \"Single\\u2010cell RNA sequencing (scRNA\\u2010seq) is a powerful new technology allowing the analysis of transcriptomes from individual cell and is ideally suited to dissect immune cell heterogeneity. ScRNA\\u2010seq has already been applied to identify novel immune cell subsets, elaborate cellular differentiation trajectories, and elucidate immunopathogenic mechanisms. Here, we briefly discuss the recent progresses and challenges in the scRNA\\u2010seq technology including the workflow, recent applications in immunology, and potential hurdles that need to be overcome. This review will highlight how single cell technology promotes our understanding of human immunology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1378&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c2f0fd061e38090f2895e0ce1689fb5eef32b3e\",\n",
      "        \"title\": \"Semisupervised Generative Autoencoder for Single-Cell Data\",\n",
      "        \"abstract\": \"Single-cell transcriptomics offers a tool to study the diversity of cell phenotypes through snapshots of the abundance of mRNA in individual cells. Often there is additional information available besides the single-cell gene expression counts, such as bulk transcriptome data from the same tissue, or quantification of surface protein levels from the same cells. In this study, we propose models based on the Bayesian deep learning approach, where protein quantification, available as CITE-seq counts, from the same cells is used to constrain the learning process, thus forming a SemI-SUpervised generative Autoencoder (SISUA) model. The generative model is based on the deep variational autoencoder (VAE) neural network architecture.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1379&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"72c2e78356e5863ca43bc1ba06cc99b1976dea62\",\n",
      "        \"title\": \"Emerging deep learning methods for single-cell RNA-seq data analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1380&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4dee629bf6960d1c1bd1ef0715ec120ec1812fdf\",\n",
      "        \"title\": \"Single-cell multiomic analysis identifies regulatory programs in mixed-phenotype acute leukemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1381&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d0675784bf5f6bbe194c4eeec79886eafca600e3\",\n",
      "        \"title\": \"Multimodal single-cell approaches shed light on T cell heterogeneity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1382&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2b24dd75f6683fa49fad9ba9516cd1ee78ce9ecb\",\n",
      "        \"title\": \"The hematopoietic landscape at single-cell resolution reveals unexpected stem cell features in naked mole-rats\",\n",
      "        \"abstract\": \"Naked mole-rats are the longest-lived rodents endowed with resistance to cancer and age-related diseases, yet their stem cell characteristics remain enigmatic. We profiled the naked mole-rat hematopoietic system down to single-cell resolution, and identified several unique features likely contributing to longevity. In adult naked mole-rats red blood cells are formed in spleen and marrow, a neotenic feature beneficial for hypoxic environments and to prevent anemia. Platelet numbers are lower compared to short-lived mice, which may preclude age-related platelet increase and thrombosis. T cells mature in thymus and lymph nodes, providing a supply of T cells after age-related thymus involution. The pool of quiescent stem cells is higher than in mice, and HSCs overexpress an oxidative phosphorylation signature, revealing a new paradigm of stem cell metabolism to benefit longevity and oppose oncogenesis. Our work provides a platform to study immunology and stem cell biology in an animal model of healthy aging. HIGHLIGHTS Flow cytometry labelling panel to purify viable naked mole-rat HSPCs The spleen as the major site of erythropoiesis in the naked mole-rat Naked mole-rats show extrathymic T-cell development under homeostatic conditions Naked mole-rat hematopoietic stem cells (HSCs) have high OXPHOS activity\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1383&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"287a5cc0f2a0f9d345c91b4ad20effc246a0b056\",\n",
      "        \"title\": \"Single\\u2010Cell Analysis Using Droplet Microfluidics\",\n",
      "        \"abstract\": \"Droplet microfluidics has revolutionized the study of single cells. The ability to compartmentalize cells within picoliter droplets in microfluidic devices has opened up a wide range of strategies to extract information at the genomic, transcriptomic, proteomic, or metabolomic level from large numbers of individual cells. Studying the different molecular landscapes at single\\u2010cell resolution has provided the authors with a detailed picture of intracellular heterogeneity and the resulting changes in cellular phenotypes. In addition, these technologies have aided in the discovery of rare cells in tumors or in the immune system, and left the authors with a deeper understanding of the fundamental biological processes that determine cell fate. This review aims to provide a detailed overview of the various droplet microfluidic strategies reported in the literature, taking into account the sometimes subtle differences in workflow or reagents that enable or improve certain protocols. Specifically, approaches to targeted\\u2010 and whole\\u2010genome analysis, as well as whole\\u2010transcriptome profiling techniques, are reviewed. In addition, an up\\u2010to\\u2010date overview of new methods to characterize and quantify single\\u2010cell protein levels, and of developments to screen secreted molecules such as antibodies, cytokines, or metabolites at the single\\u2010cell level, is provided.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1384&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff8d918b316e0f62e61508101caa2abe123331fa\",\n",
      "        \"title\": \"High-Throughput Mapping of B Cell Receptor Sequences to Antigen Specificity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1385&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8367031651cb30bf5780569fc68bafb5b4cd8c76\",\n",
      "        \"title\": \"clustifyr: an R package for automated single-cell RNA sequencing cluster classification\",\n",
      "        \"abstract\": \"Background In single-cell RNA sequencing (scRNA-seq) analysis, assignment of likely cell types remains a time-consuming, error-prone, and biased process. Current packages for identity assignment use limited types of reference data, and often have rigid data structure requirements. As such, a more flexible tool, capable of handling multiple types of reference data and data structures, would be beneficial. Findings To address difficulties in cluster identity assignment, we developed the clustifyr R package. The package leverages external datasets, including gene expression profiles from scRNA-seq, bulk RNA-seq, microarray expression data, and/or signature gene lists, to assign likely cell types. We benchmark various parameters of a correlation-based approach, and also implement a variety of gene list enrichment methods. By providing tools for exploratory data analysis, we demonstrate the feasibility of a simple and effective data-driven approach for cell type assignment in scRNA-seq cell clusters. Conclusions clustifyr is a lightweight and effective cell type assignment tool developed for compatibility with various scRNA-seq analysis workflows. clustifyr is publicly available at https://github.com/rnabioco/clustifyr\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1386&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"71e5ebdd6f93eeaa9dd34dab0d508274b531be49\",\n",
      "        \"title\": \"Deep generative model embedding of single-cell RNA-Seq profiles on hyperspheres and hyperbolic spaces\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1387&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aeea4c2443e95af34e474b55bd96e68fe4c2935f\",\n",
      "        \"title\": \"CiteFuse enables multi-modal analysis of CITE-seq data\",\n",
      "        \"abstract\": \"Multi-modal profiling of single cells represents one of the latest technological advancements in molecular biology. Among various single-cell multi-modal strategies, cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) allows simultaneous quantification of two distinct species: RNA and surface marker proteins (ADT). Here, we introduce CiteFuse, a streamlined package consisting of a suite of tools for pre-processing, modality integration, clustering, differential RNA and ADT expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualization of CITE-seq data. We show the capacity of CiteFuse to integrate the two data modalities and its relative advantage against data generated from single modality profiling. Furthermore, we illustrate the pre-processing steps in CiteFuse and in particular a novel doublet detection method based on a combined index of cell hashing and transcriptome data. Collectively, we demonstrate the utility and effectiveness of CiteFuse for the integrative analysis of transcriptome and epitope profiles from CITE-seq data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1388&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d940187ca0f19ca7c2407f31c1ae33b18faaaafd\",\n",
      "        \"title\": \"HaloTag-based conjugation of proteins to barcoding-oligonucleotides\",\n",
      "        \"abstract\": \"Abstract Highly sensitive protein quantification enables the detection of a small number of protein molecules that serve as markers/triggers for various biological phenomena, such as cancer. Here, we describe the development of a highly sensitive protein quantification system called HaloTag protein barcoding. The method involves covalent linking of a target protein to a unique molecule counting oligonucleotide at a 1:1 conjugation ratio based on an azido\\u2013cycloalkyne click reaction. The sensitivity of the HaloTag-based barcoding was remarkably higher than that of a conventional luciferase assay. The HaloTag system was successfully validated by analyzing a set of protein-protein interactions, with the identification rate of 44% protein interactions between positive reference pairs reported in the literature. Desmoglein 3, the target antigen of pemphigus vulgaris, an IgG-mediated autoimmune blistering disease, was used in a HaloTag protein barcode assay to detect the anti-DSG3 antibody. The dynamic range of the assay was over 104-times wider than that of a conventional enzyme-linked immunosorbent assay (ELISA). The technology was used to detect anti-DSG3 antibody in patient samples with much higher sensitivity compared to conventional ELISA. Our detection system, with its superior sensitivity, enables earlier detection of diseases possibly allowing the initiation of care/treatment at an early disease stage.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1389&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a7d5382bb16830edc0c7f04b4432df8b32b2197\",\n",
      "        \"title\": \"Microfluidics as an Enabling Technology for Personalized Cancer Therapy.\",\n",
      "        \"abstract\": \"Tailoring patient-specific treatments for cancer is necessary in order to achieve optimal results but requires new diagnostic approaches at affordable prices. Microfluidics has immense potential to provide solutions for this, as it enables the processing of samples that are not available in large quantities (e.g., cells from patient biopsies), is cost efficient, provides a high level of automation, and allows the set-up of complex models for cancer studies. In this review, individual solutions in the fields of genetics, circulating tumor cell monitoring, biomarker analysis, phenotypic drug sensitivity tests, and systems providing controlled environments for disease modeling are discussed. An overview on how these early stage achievements can be combined or developed further is showcased, and the required translational steps before microfluidics becomes a routine tool for clinical applications are critically discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1390&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4f700f6504c8ad397fdbf45a0e6f48b8e9da9e5\",\n",
      "        \"title\": \"29-Color Flow Cytometry: Unraveling Human Liver NK Cell Repertoire Diversity\",\n",
      "        \"abstract\": \"Recent studies have demonstrated extraordinary diversity in peripheral blood human natural killer (NK) cells and have suggested environmental control of receptor expression patterns on distinct subsets of NK cells. However, tissue localization may influence NK cell differentiation to an even higher extent and less is known about the receptor repertoire of human tissue-resident NK cells. Advances in single-cell technologies have allowed higher resolution studies of these cells. Here, the power of high-dimensional flow cytometry was harnessed to unravel the complexity of NK cell repertoire diversity in liver since recent studies had indicated high heterogeneity within liver NK cells. A 29-color flow cytometry panel allowing simultaneous measurement of surface tissue-residency markers, activating and inhibitory receptors, differentiation markers, chemokine receptors, and transcription factors was established. This panel was applied to lymphocytes across three tissues (liver, peripheral blood, and tonsil) with different distribution of distinct NK cell subsets. Dimensionality reduction of this data ordered events according to their lineage, rather than tissue of origin. Notably, narrowing the scope of the analysis to the NK cell lineage in liver and peripheral blood separated subsets according to tissue, enabling phenotypic characterization of NK cell subpopulations in individual tissues. Such dimensionality reduction, coupled with a clustering algorithm, identified CD49e as the preferred marker for future studies of liver-resident NK cell subsets. We present a robust approach for diversity profiling of tissue-resident NK cells that can be applied in various homeostatic and pathological conditions such as reproduction, infection, and cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1391&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c7c52eaa4b08a581d8c539d8a428feb83660015\",\n",
      "        \"title\": \"Single cell multiomic analysis of T cell exhaustion in vitro\",\n",
      "        \"abstract\": \"A key step in the clinical production of CAR-T cells is the expansion of engineered T cells. To generate enough cells for a therapeutic product, cells must be chronically stimulated, which raises the risk of inducing T-cell exhaustion and reducing therapeutic efficacy. As protocols for T-cell expansion are being developed to optimize CAR T cell yield, function and persistence, fundamental questions about the impact of in vitro manipulation on T-cell identity are important to answer. Namely: 1) what types of cells are generated during chronic stimulation? 2) how many unique cell states can be defined during chronic stimulation? We sought to answer these fundamental questions by performing single-cell multiomic analysis to simultaneously measure expression of 39 proteins and 399 genes in human T cells expanded in vitro. This approach allowed us to study \\u2013 with unprecedented depth - how T cells change over the course of chronic stimulation. Comprehensive immunophenotypic and transcriptomic analysis at day 0 enabled a refined characterization of T-cell maturational states (from na\\u00efve to TEMRA cells) and the identification of a donor-specific subset of terminally differentiated T-cells that would have been otherwise overlooked using canonical cell classification schema. As expected, T-cell activation induced downregulation of na\\u00efve-associated markers and upregulation of effector molecules, proliferation regulators, co-inhibitory and co-stimulatory receptors. Our deep kinetic analysis further revealed clusters of proteins and genes identifying unique states of activation defined by markers temporarily expressed upon 3 days of stimulation (PD-1, CD69, LTA), markers constitutively expressed throughout chronic activation (CD25, GITR, LGALS1), and markers uniquely up-regulated upon 14 days of stimulation (CD39, ENTPD1, TNFDF10). Notably, different ratios of cells expressing activation or exhaustion markers were measured at each time point. These data indicate high heterogeneity and plasticity of chronically stimulated T cells in response to different kinetics of activation. In this study, we demonstrate the power of a single-cell multiomic approach to comprehensively characterize T cells and to precisely monitor changes in differentiation, activation and exhaustion signatures in response to different activation protocols.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1392&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9447e5021ff7fdfdaeb7bc33f26ad1ac68284f86\",\n",
      "        \"title\": \"Single-Cell Virtual Cytometer allows user-friendly and versatile analysis and visualization of multimodal single cell RNAseq datasets\",\n",
      "        \"abstract\": \"The development of single cell transcriptomic technologies yields large datasets comprising multimodal informations such as transcriptomes and immunophenotypes. Currently however, there is no software to easily and simultaneously analyze both types of data. Here, we introduce Single-Cell Virtual Cytometer, an open-source software for flow cytometry-like visualization and exploration of multi-omics single cell datasets. Using an original CITE-seq dataset of PBMC from an healthy donor, we illustrate its use for the integrated analysis of transcriptomes and phenotypes of functional maturation in peripheral T lymphocytes from healthy donors. So this free and open-source algorithm constitutes a unique resource for biologists seeking for a user-friendly analytic tool for multimodal single cell datasets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1393&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"341b5ed5297d8e16204015f3252f8a826f5a659b\",\n",
      "        \"title\": \"MOFA+: a probabilistic framework for comprehensive integration of structured single-cell data\",\n",
      "        \"abstract\": \"Technological advances have enabled the joint analysis of multiple molecular layers at single cell resolution. At the same time, increased experimental throughput has facilitated the study of larger numbers of experimental conditions. While methods for analysing single-cell data that model the resulting structure of either of these dimensions are beginning to emerge, current methods do not account for complex experimental designs that include both multiple views (modalities or assays) and groups (conditions or experiments). Here we present Multi-Omics Factor Analysis v2 (MOFA+), a statistical framework for the comprehensive and scalable integration of structured single cell multi-modal data. MOFA+ builds upon a Bayesian Factor Analysis framework combined with fast GPU-accelerated stochastic variational inference. Similar to existing factor models, MOFA+ allows for interpreting variation in single-cell datasets by pooling information across cells and features to reconstruct a low-dimensional representation of the data. Uniquely, the model supports flexible group-level sparsity constraints that allow joint modelling of variation across multiple groups and views. To illustrate MOFA+, we applied it to single-cell data sets of different scales and designs, demonstrating practical advantages when analyzing datasets with complex group and/or view structure. In a multi-omics analysis of mouse gastrulation this joint modelling reveals coordinated changes between gene expression and epigenetic variation associated with cell fate commitment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1394&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9b329fa17400c9c462e4138b78e4a2ba5d12d2f5\",\n",
      "        \"title\": \"SCHEMA: A general framework for integrating heterogeneous single-cell modalities\",\n",
      "        \"abstract\": \"Advances in single-cell technologies now allow researchers to simultaneously measure diverse biological data modalities, including transcriptomic, proteomic, or spatial features. Sophisticated cross-modality analysis promises unprecedented and more complete insight into complex biology. Unfortunately, current methods for single-cell RNA-seq data integration focus on aligning patterns across multiple experiments and are not designed for the setting in which a single experiment simultaneously measures multiple modalities. We therefore propose Schema, a general, scalable, and flexible framework for integrating diverse modalities profiled within a single experimental setting. Schema has intuitive parameters that facilitate exploratory analysis and complements standard techniques like PCA and CCA (canonical correlation analysis). We first process each modality to compute a similarity (equivalently, distance) measure between cells. Schema is a framework for calibrating the agreement between these modality-specific distance measures. Given a primary dataset (i.e., modality) and an arbitrary number of secondary datasets, we search for an affine transformation of the primary dataset so that distances in the transformed space have the maximum possible agreement with the secondary datasets. Crucially, the user can calibrate this by specifying a limit on how much the primary dataset can be distorted; methods like CCA lack this control. Mathematically, we quantify the concept of agreement between datasets as the correlation between their respective sets of pairwise squared distances and offer some theoretical justification for this design choice. Our optimization problem can be formulated as a quadratic program (QP), allowing for a fast algorithm with intuitive free parameters and an appealing feature-weighting interpretation. Using Schema, we integrated spatial and transcriptomic modalities from the Slide-Seq experiment [1] and identified a gene-set in granule cells whose expression covaries with the spatial density of the cells. We investigated a multi-modal dataset from 10x Genomics [2], analyzing the selection pressure on residues in the CDR3 region of T-cell receptors. We also used Schema to improve UMAP & tSNE visualizations by infusing metadata into RNA-seq information. As the number and diversity of multi-modal datasets grows, Schema will be an essential component in the single-cell analyst\\u2019s arsenal. Availability Our implementation is freely available as the Python package schema_learn and code is available at http://schema.csail.mit.edu.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1395&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"99d71994dfa8e31b9708f4b168eaa128cdc610aa\",\n",
      "        \"title\": \"Studying early development using droplet single-cell RNA sequencing\",\n",
      "        \"abstract\": \"In complex organisms, each individual derives from a single fertilised ovum. This process of development is characterised by an increase in cell number and a diversification of the various cell types that are present. Development has been extensively studied not only to understand how cells commit to various lineages, but also with a view to reversing typical developmental pathways for regenerative medicine. However, studying development in vivo is challenging due to the very high degree of spatial complexity in an embryo, and the limited amount of biological material that is actually present in these systems. Recently developed single-cell RNA-sequencing technologies offer a powerful tool for studying development. They generate a rich phenotypic readout of a cell\\u2019s state, providing gene-level information about what cells are doing, and allowing reconstruction of the molecular bases of developmental pathways at the appropriate scale for which developmental decisions are made. This Thesis begins with a summary of how single-cell RNA-sequencing data are generated and analysed, and how it has been used to study cell fate decision-making. The second chapter covers my work in identifying the extent of a technical confounder of high throughput sequencing experiments (barcode swapping), and how its consequences may be excluded from a subset of single-cell RNA-sequencing experiments where thousands of single cells are captured in microfluidic droplets for library preparation. I then detail the compilation of a large single-cell RNA-sequencing timecourse atlas of mouse development between embryonic days 6.5 and 8.5. I also performed analyses into the formation of the embryonic gut, which surprisingly receives contributions from extraembryonic tissue. The fourth and fifth chapters describe my investigation into the role of the genes Tal1 and T (also known as Brachyury) using chimaeric embryos. These genes are essential for proper development, and embryos die when they are entirely absent. Chimaeric embryos are composed of both knockout and wild-type cells: the wild-type cells contribute to the lineages that are compromised by the knockout, and the embryo remains healthy. The cell-intrinsic roles of these critical genes can therefore by studied in a proper in vivo context, unconfounded by the global abnormalities that would be induced by their absence from the whole embryo. Finally, I discuss the future of the field of single-cell RNA-sequencing, with a particular focus on how we can move beyond the currently employed snapshot measurements of cell states, in order to properly identify how the state of a cell affects its future behaviour.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1396&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ed2a26734f4bcb8a4f7c560b5b3b26d274b0f138\",\n",
      "        \"title\": \"Sample demultiplexing, multiplet detection, experiment planning and novel cell type verification in single cell sequencing\",\n",
      "        \"abstract\": \"Identifying and removing multiplets from downstream analysis is essential to improve the scalability and reliability of single cell RNA sequencing (scRNA-seq). High multiplet rates create artificial cell types in the dataset. Sample barcoding, including the cell hashing technology and the MULTI-seq technology, enables analytical identification of a fraction of multiplets in a scRNA-seq dataset. We propose a Gaussian-mixture-model-based multiplet identification method, GMM-Demux. GMM-Demux accurately identifies and removes the sample-barcoding-detectable multiplets and estimates the percentage of sample-barcoding-undetectable multiplets in the remaining dataset. GMM-Demux describes the droplet formation process with an augmented binomial probabilistic model, and uses the model to authenticate cell types discovered from a scRNA-seq dataset. We conducted two cell-hashing experiments, collected a public cell-hashing dataset, and generated a simulated cellhashing dataset. We compared the classification result of GMM-Demux against a state-of-the-art heuristic-based classifier. We show that GMM-Demux is more accurate, more stable, reduces the error rate by up to 69\\u00d7, and is capable of reliably recognizing 9 multiplet-induced fake cell types and 8 real cell types in a PBMC scRNA-seq dataset.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1397&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a125b81ea11af071fe4a50d41445b483fb0ad36e\",\n",
      "        \"title\": \"Parallel bimodal single-cell sequencing of transcriptome and chromatin accessibility\",\n",
      "        \"abstract\": \"We developed ASTAR-Seq (Assay for Single-cell Transcriptome and Accessibility Regions) integrated with automated microfluidic chips, which allows for parallel sequencing of transcriptome and chromatin accessibility within the same single-cell. Using ASTAR-Seq, we profiled 192 mESCs cultured in serum+LIF and 2i medium, 424 human cell lines including BJ, K562, JK1, and Jurkat, and 480 primary cells undergoing erythroblast differentiation. Integrative analysis using Coupled NMF identified the distinct sub-populations and uncovered sets of regulatory regions and the respective target genes determining their distinctions. Analysis of epigenetic regulomes further unravelled the key transcription factors responsible for the heterogeneity observed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1398&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"561eb34fcec6a6bf7b297cdef46912fec38ce5c9\",\n",
      "        \"title\": \"Maturation of the Human Intestinal Immune System Occurs Early in Fetal Development.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1399&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f81f06a298821977993f3d50dac3823c1d85987a\",\n",
      "        \"title\": \"An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1400&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b97e4dd46bb9cf59731ffcd6a6fb5c965a86d677\",\n",
      "        \"title\": \"Cell-by-cell deciphering of T cells in allergic inflammation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1401&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f26a91b57e458c42d63bd171f73aee51b7684dc4\",\n",
      "        \"title\": \"Immunology Driven by Large-Scale Single-Cell Sequencing.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1402&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"66125ff1c89d218cbfeb579cad25d3bc7a42f81e\",\n",
      "        \"title\": \"Single-cell analysis to understand the diversity of immune cell types that drive disease pathogenesis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1403&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d4c4b6e5035c7fc397d7cc0a2c2b2622e307ed6\",\n",
      "        \"title\": \"Guidelines for reporting single-cell RNA-seq experiments\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1404&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d78095b557e2883b79cbf5bf79f18e61bc0d64b3\",\n",
      "        \"title\": \"Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq\",\n",
      "        \"abstract\": \"Massively parallel single-cell and single-nucleus RNA-seq (sc/snRNA-seq) have opened the way to systematic tissue atlases in health and disease, but as the scale of data generation is growing, so does the need for computational pipelines for scaled analysis. Here, we developed Cumulus, a cloud-based framework for analyzing large scale sc/snRNA-seq datasets. Cumulus combines the power of cloud computing with improvements in algorithm implementations to achieve high scalability, low cost, user-friendliness, and integrated support for a comprehensive set of features. We benchmark Cumulus on the Human Cell Atlas Census of Immune Cells dataset of bone marrow cells and show that it substantially improves efficiency over conventional frameworks, while maintaining or improving the quality of results, enabling large-scale studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1405&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e8019c3e628409cebddb13da98f1b97e86f44847\",\n",
      "        \"title\": \"Mapping the transcriptional landscape of haematopoietic stem and progenitor cells\",\n",
      "        \"abstract\": \"My funding for the CIMR 4 year programme was provided by the Medical Research Council (MRC).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1406&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"23f97c1957c7449db66575317c7d572b023bd0f7\",\n",
      "        \"title\": \"Sequencing the Human Brain at Single-Cell Resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1407&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1e2bbb24f986b0a9fbacf390bd28b2017dd3af5f\",\n",
      "        \"title\": \"Fast and efficient Fc-specific photoaffinity labelling to produce antibody-DNA-conjugates.\",\n",
      "        \"abstract\": \"Antibody-DNA-conjugates are powerful tools for DNA-assisted protein analysis. Growing usage of these methods demands efficient production of high-quality conjugates. We developed an easy and fast synthesis route yielding covalent antibody-DNA-conjugates with a defined conjugation site and low batch-to-batch variability. We utilize the Z domain from protein A, containing the unnatural amino acid 4 benzoyl\\u00acphenyl\\u00acalanine (BPA) for photoaffinity labelling of the antibodies' Fc region. Z(xBPA)-domains are C-terminally modified with triple-glycine (G3)-modified DNA-oligonucleotides via enzymatic Sortase A coupling. We show reliable modification of the most commonly used IgGs. To prove our conjugates' functionality, we detected antibody-antigen binding events in an assay called Droplet Barcode Sequencing for Protein analysis (DBS-Pro). It confirms not only retained functionality for both conjugate parts but also the potential of using DBS-Pro for quantifying protein abundances. As intermediates are easily storable and our approach is modular, it offers a convenient strategy for screening various antibody-DNA-conjugates using the same starting material.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1408&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc32b18de7a4487815c39a8581fe054bfea40c92\",\n",
      "        \"title\": \"The next frontier in single cell analysis: multimodal studies and clinical translation.\",\n",
      "        \"abstract\": \"Thought leaders Pratip Chattopadhyay and Daniel Chiu introduce the Lab on a Chip single cell analysis thematic collection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1409&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"46ae4fdec895997d65a6b09563094a0f0dd27069\",\n",
      "        \"title\": \"Systems genetics applications in metabolism research\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1410&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"92cb38264aaed0e3a35a1ab1379c426e493b45b6\",\n",
      "        \"title\": \"Automated optimized parameters for T-distributed stochastic neighbor embedding improve visualization and analysis of large datasets\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1411&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c053184fce2827b901efe054ad059e4172720e8\",\n",
      "        \"title\": \"Towards systems tissue engineering: Elucidating the dynamics, spatial coordination, and individual cells driving emergent behaviors.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1412&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b66c5a3531b24bd9641f0f7592aade827089ff98\",\n",
      "        \"title\": \"D\\u00e9veloppement du s\\u00e9quen\\u00e7age ARN cibl\\u00e9 sur cellules uniques en microfluidique de gouttes et applications\",\n",
      "        \"abstract\": \"Une large proportion de nouveaux-nes prematures sont sujets a des inflammations du cerveau pouvant entrainer des troubles neurologiques. Les memes mecanismes d'inflammation sont observes dans la progression de pathologies telles que Alzheimer ou Parkinson. Cette inflammation est due aux cellules microgliales, qui constitue le systeme immunitaire du cerveau. Durant cette these, nous allons developper des outils d'analyse a l'echelle de la cellule unique a tres haut debit (microfluidique en gouttes). Ils nous permettront d'etudier la microglie a l'echelle de la cellule unique et en particulier l'expression genique, la secretion de proteines d'interet ainsi que la reponse de ces cellules a des combinaisons de drogues. Les objectifs de la these sont de comprendre les mecanismes de regulation de l'inflammation dans le cerveau et de mettre au point de nouvelles strategies therapeutiques pour diminuer cette inflammation chez les nouveaux-nes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1413&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"935395d84a050e793120263de2f5a0318cd448ba\",\n",
      "        \"title\": \"Framing technology challenges associated with improving cancer immunotherapies.\",\n",
      "        \"abstract\": \"Thought leader Jim Heath introduces the Lab on a Chip Immunotherapy thematic collection.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1414&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b47e09e26af6ca6398f15707c54aa4b8371942cb\",\n",
      "        \"title\": \"A Joint Model of RNA Expression and Surface Protein Abundance in Single Cells\",\n",
      "        \"abstract\": \"Cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) combines unbiased single-cell transcriptome measurements with surface protein quantification comparable to flow cytometry, the gold standard for cell type identification. However, current analysis pipelines cannot address the two primary challenges of CITE-seq data: combining both modalities in a shared latent space that harnesses the power of the paired measurements, and handling the technical artifacts of the protein measurement, which is obscured by non-negligible background noise. Here we present Total Variational Inference (totalVI), a fully probabilistic end-to-end framework for normalizing and analyzing CITE-seq data, based on a hierarchical Bayesian model. In totalVI, the mRNA and protein measurements for each cell are generated from a low-dimensional latent random variable unique to that cell, representing its cellular state. totalVI uses deep neural networks to specify conditional distributions. By leveraging advances in stochastic variational inference, it scales easily to millions of cells. Explicit modeling of nuisance factors enables totalVI to produce denoised data in both domains, as well as a batch-corrected latent representation of cells for downstream analysis tasks.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1415&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0c74597a8fadcc6f10237d360a530096318a3351\",\n",
      "        \"title\": \"Mind the Map: Technology Shapes the Myeloid Cell Space\",\n",
      "        \"abstract\": \"The myeloid cell system shows very high plasticity, which is crucial to quickly adapt to changes during an immune response. From the beginning, this high plasticity has made cell type classification within the myeloid cell system difficult. Not surprising, naming schemes have been frequently changed. Recent advancements in multidimensional technologies, including mass cytometry and single-cell RNA sequencing, are challenging our current understanding of cell types, cell subsets, and functional states of cells. Despite the power of these technologies to create new reference maps for the myeloid cell system, it is essential to put these new results into context with previous knowledge that was established over decades. Here we report on earlier attempts of cell type classification in the myeloid cell system, discuss current approaches and their pros and cons, and propose future strategies for cell type classification within the myeloid cell system that can be easily extended to other cell types.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1416&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9dc5addb38f7834c1c8e4af3e100764b4d04d76\",\n",
      "        \"title\": \"Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1417&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c46bf0221f1a400d1041584333c9a1846413ed22\",\n",
      "        \"title\": \"Shared and distinct mechanisms of fibrosis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1418&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e81f25a39c1d3c2adba0b68bc887bbee4ad1885c\",\n",
      "        \"title\": \"Microglia Biology: One Century of Evolving Concepts\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1419&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9d226bec530384cfe1ce2b867356ca2ff32b7636\",\n",
      "        \"title\": \"Single-Cell Genomic Characterization Reveals the Cellular Reprogramming of the Gastric Tumor Microenvironment\",\n",
      "        \"abstract\": \"Purpose: The tumor microenvironment (TME) consists of a heterogenous cellular milieu that can influence cancer cell behavior. Its characteristics have an impact on treatments such as immunotherapy. These features can be revealed with single-cell RNA sequencing (scRNA-seq). We hypothesized that scRNA-seq analysis of gastric cancer together with paired normal tissue and peripheral blood mononuclear cells (PBMC) would identify critical elements of cellular deregulation not apparent with other approaches. Experimental Design: scRNA-seq was conducted on seven patients with gastric cancer and one patient with intestinal metaplasia. We sequenced 56,167 cells comprising gastric cancer (32,407 cells), paired normal tissue (18,657 cells), and PBMCs (5,103 cells). Protein expression was validated by multiplex immunofluorescence. Results: Tumor epithelium had copy number alterations, a distinct gene expression program from normal, with intratumor heterogeneity. Gastric cancer TME was significantly enriched for stromal cells, macrophages, dendritic cells (DC), and Tregs. TME-exclusive stromal cells expressed distinct extracellular matrix components than normal. Macrophages were transcriptionally heterogenous and did not conform to a binary M1/M2 paradigm. Tumor DCs had a unique gene expression program compared to PBMC DCs. TME-specific cytotoxic T cells were exhausted with two heterogenous subsets. Helper, cytotoxic T, Treg, and NK cells expressed multiple immune checkpoint or co-stimulatory molecules. Receptor\\u2013ligand analysis revealed TME-exclusive intercellular communication. Conclusions: Single-cell gene expression studies revealed widespread reprogramming across multiple cellular elements in the gastric cancer TME. Cellular remodeling was delineated by changes in cell numbers, transcriptional states, and intercellular interactions. This characterization facilitates understanding of tumor biology and enables identification of novel targets including for immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1420&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c992eb70a6620c1e3dbc4e9f1b6459f9fe6d4d3\",\n",
      "        \"title\": \"Single-cell immune landscape of human atherosclerotic plaques\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1421&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d1752108a47420ec316d14642cbc23c94576ad3\",\n",
      "        \"title\": \"Single-cell transcriptomics as a framework and roadmap for understanding the brain\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1422&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f2aa2f551b3d0c6f823cf4231943870bfe8f2d3f\",\n",
      "        \"title\": \"High-Spatial-Resolution Multi-Omics Atlas Sequencing of Mouse Embryos via Deterministic Barcoding in Tissue\",\n",
      "        \"abstract\": \"Spatial gene expression heterogeneity plays an essential role in a range of biological, physiological and pathological processes but it remains a scientific challenge to conduct high-spatial-resolution, genome-wide, unbiased biomolecular profiling over a large tissue area. Herein, we present a fundamentally new approach \\u2013 microfluidic Deterministic Barcoding in Tissue for spatial omics sequencing (DBiT-seq). Parallel microfluidic channels (10\\u03bcm, 25\\u03bcm, or 50\\u03bcm in width) are used to deliver molecular barcodes to the surface of a formaldehyde fixed tissue slide in a spatially confined manner. Crossflow of two sets of barcodes A1-A50 and B1-B50 followed by ligation in situ yields a 2D mosaic of tissue pixels, each containing a unique combination of full barcode AiBj (i=1-50, j=1-50). It permits simultaneous barcoding of mRNAs, proteins, or even other omics on a fixed tissue slide, enabling the construction of a high-spatial-resolution multi-omics atlas by NGS sequencing. Applying it to mouse embryo tissues revealed all major tissue types in early organogenesis, distinguished brain microvascular networks, discovered new developmental patterning in forebrain, and demonstrated the ability to detect a single-cell-layer of melanocytes lining an optical vesicle and asymmetric expression of Rorb and Aldh1a1 within it, presumably associated with the onset of retina and lens, respectively. Automated feature identification using spatial differential expression further identified dozens of developmental features. DBiT-seq is a highly versatile technology that may become a universal method for spatial barcoding and sequencing of a range of molecular information at a high resolution and the genome scale. It can be readily adopted by biologists with no experience in microfluidics or advanced imaging, and could be quickly disseminated for broader impacts in a variety of fields including developmental biology, cancer biology, neuroscience, and clinical pathology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1423&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d759eac1ebac27a0241eaf39c9c62af59f4f36d1\",\n",
      "        \"title\": \"Machine learning predicts putative hematopoietic stem cells within large single-cell transcriptomics data sets\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1424&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2c21457a9880a2b8c3dcedd22db6e3a35edf3d5c\",\n",
      "        \"title\": \"Learning transport cost from subset correspondence\",\n",
      "        \"abstract\": \"Learning to align multiple datasets is an important problem with many applications, and it is especially useful when we need to integrate multiple experiments or correct for confounding. Optimal transport (OT) is a principled approach to align datasets, but a key challenge in applying OT is that we need to specify a transport cost function that accurately captures how the two datasets are related. Reliable cost functions are typically not available and practitioners often resort to using hand-crafted or Euclidean cost even if it may not be appropriate. In this work, we investigate how to learn the cost function using a small amount of side information which is often available. The side information we consider captures subset correspondence---i.e. certain subsets of points in the two data sets are known to be related. For example, we may have some images labeled as cars in both datasets; or we may have a common annotated cell type in single-cell data from two batches. We develop an end-to-end optimizer (OT-SI) that differentiates through the Sinkhorn algorithm and effectively learns the suitable cost function from side information. On systematic experiments in images, marriage-matching and single-cell RNA-seq, our method substantially outperform state-of-the-art benchmarks.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1425&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0c0a479c4a30e3bef44aab97599f9fa1fdbe01ca\",\n",
      "        \"title\": \"Single-cell eQTLGen Consortium: a personalized understanding of disease.\",\n",
      "        \"abstract\": \"In recent years, functional genomics approaches combining genetic information with bulk RNA-sequencing data have identified the downstream expression effects of disease-associated genetic risk factors through so-called expression quantitative trait locus (eQTL) analysis. Single-cell RNA-sequencing creates enormous opportunities for mapping eQTLs across different cell types and in dynamic processes, many of which are obscured when using bulk methods. The enormous increase in throughput and reduction in cost per cell now allow this technology to be applied to large-scale population genetics studies. Therefore, we have founded the single-cell eQTLGen consortium (sc-eQTLGen), aimed at pinpointing disease-causing genetic variants and identifying the cellular contexts in which they affect gene expression. Ultimately, this information can enable development of personalized medicine. Here, we outline the goals, approach, potential utility and early proofs-of-concept of the sc-eQTLGen consortium. We also provide a set of study design considerations for future single-cell eQTL studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1426&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7cc0f9a318274123d2a6c66c05e8d0ab0e5622bf\",\n",
      "        \"title\": \"Mapping human cell phenotypes to genotypes with single-cell genomics\",\n",
      "        \"abstract\": \"The cumulative activity of all of the body\\u2019s cells, with their myriad interactions, life histories, and environmental experiences, gives rise to a condition that is distinctly human and specific to each individual. It is an enduring goal to catalog our human cell types, to understand how they develop, how they vary between individuals, and how they fail in disease. Single-cell genomics has revolutionized this endeavor because sequencing-based methods provide a means to quantitatively annotate cell states on the basis of high-information content and high-throughput measurements. Together with advances in stem cell biology and gene editing, we are in the midst of a fascinating journey to understand the cellular phenotypes that compose human bodies and how the human genome is used to build and maintain each cell. Here, we will review recent advances into how single-cell genomics is being used to develop personalized phenotyping strategies that cross subcellular, cellular, and tissue scales to link our genome to our cumulative cellular phenotypes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1427&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b0451b7a3b52638808b636a72f688bf965a5fbf\",\n",
      "        \"title\": \"Exploring and analysing single cell multi-omics data with VDJView\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1428&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3c66973b25f91ea49fee272583303220d50ff252\",\n",
      "        \"title\": \"High-throughput genome-wide phenotypic screening via immunomagnetic cell sorting\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1429&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6bf46613e4b9bf1cb32825ba22e1f90ed788222a\",\n",
      "        \"title\": \"Meta-Analysis of Human and Mouse Biliary Epithelial Cell Gene Profiles\",\n",
      "        \"abstract\": \"Background: Chronic liver diseases are frequently accompanied with activation of biliary epithelial cells (BECs) that can differentiate into hepatocytes and cholangiocytes, providing an endogenous back-up system. Functional studies on BECs often rely on isolations of an BEC cell population from healthy and/or injured livers. However, a consensus on the characterization of these cells has not yet been reached. The aim of this study was to compare the publicly available transcriptome profiles of human and mouse BECs and to establish gene signatures that can identify quiescent and activated human and mouse BECs. Methods: We used publicly available transcriptome data sets of human and mouse BECs, compared their profiles and analyzed co-expressed genes and pathways. By merging both human and mouse BEC-enriched genes, we obtained a quiescent and activation gene signature and tested them on BEC-like cells and different liver diseases using gene set enrichment analysis. In addition, we identified several genes from both gene signatures to identify BECs in a scRNA sequencing data set. Results: Comparison of mouse BEC transcriptome data sets showed that the isolation method and array platform strongly influences their general profile, still most populations are highly enriched in most genes currently associated with BECs. Pathway analysis on human and mouse BECs revealed the KRAS signaling as a new potential pathway in BEC activation. We established a quiescent and activated BEC gene signature that can be used to identify BEC-like cells and detect BEC enrichment in alcoholic hepatitis, non-alcoholic steatohepatitis (NASH) and peribiliary sclerotic livers. Finally, we identified a gene set that can distinguish BECs from other liver cells in mouse and human scRNAseq data. Conclusions: Through a meta-analysis of human and mouse BEC gene profiles we identified new potential pathways in BEC activation and created unique gene signatures for quiescent and activated BECs. These signatures and pathways will help in the further characterization of this progenitor cell type in mouse and human liver development and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1430&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59bae49b03c82a8331b844182ec84c9fb10025bb\",\n",
      "        \"title\": \"Single-cell profiling and SCOPE-seq reveal the lineage dynamics of adult neurogenesis and NOTUM as a key V-SVZ regulator\",\n",
      "        \"abstract\": \"Neural stem cells (NSCs) and their progeny reside in specialized niches in the adult mammalian brain where they generate new neurons and glia throughout life. Adult NSCs of the ventricular-subventricular zone (V-SVZ) are prone to rapid exhaustion; thus timely, context-dependent neurogenesis demands adaptive signaling among the vast number of neighboring progenitors nestled between the ventricular surface and nearby blood vessels. To dissect adult neuronal lineage progression and regulation, we profiled >56,000 V-SVZ and olfactory bulb (OB) cells by single-cell RNA-sequencing (scRNA-seq). Our analyses revealed the diversity of V-SVZ-derived OB neurons, the temporal dynamics of lineage progression, and a key intermediate NSC population enriched for expression of Notum, which encodes a secreted WNT antagonist. Single Cell Optical Phenotyping and Expression (SCOPE-seq), a technology linking live cell imaging with scRNA-seq, uncovered dynamic control of cell size concomitant with NSC differentiation with Notum+ NSCs at a critical size poised for cell division, and a preference of NOTUM surface binding to neuronal precursors with active WNT signaling. Finally, in vivo pharmacological inhibition of NOTUM significantly expanded neuronal precursor pools in the V-SVZ. Our findings highlight a critical regulatory state during NSC activation marked by NOTUM, a secreted enzyme that ensures efficient neurogenesis by preventing WNT signaling activation in NSC progeny.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1431&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7e304fed0bfed10b334c822994d896ca0ce45a15\",\n",
      "        \"title\": \"A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1432&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"84d33fe12e9cc59d63307df2af9018e7addc45ae\",\n",
      "        \"title\": \"Synthetic Physiology: Manipulating and Measuring Biological Pattern Formation With Light\",\n",
      "        \"abstract\": \"Multicellular tissues and organisms execute complex physiological functions through precise spatiotemporal coordination of their constituent cells. Cells achieve this coordination through longrange intercellular communication pathways \\u2013 for example, electrical signaling synchronizes the beating heart, and biochemical morphogen signals organize the developing embryo. It is an outstanding challenge in quantitative biology to develop mathematical models (borrowing from e.g. control theory or statistical physics) that explain how robust pattern formation emerges from these long-range interactions. It is generally challenging to connect theoretical models to experimental demonstrations due to the difficulty of manipulating and measuring physiological dynamics in complex tissues. The emergence of optogenetic tools has opened new opportunities in experimental biology. By placing signaling pathways under the control of light, one can leverage methods from optical physics in order to pattern biological signals with precise agility in space and time. By combining these tools with complementary optical methods for mapping resultant patterns with high spatial resolution, one can create high-dimensional biological interfaces in order to study pattern formation. Optogenetic tools have achieved widespread adoption in the study of the brain, but they have only begun to be extended to study biological pattern formation in other contexts like the developing embryo. The first portion of this thesis presents \\u2018synthetic electrophysiology\\u2019, a new framework for studying bioelectrical pattern formation. By deploying all-optical electrophysiology in engineered tissues with designed electrical components, we study electrical tissues as dynamical systems. In tissues comprised of synthetic excitable cells, we show that geometry is a fundamental determinant of stability and chaos in biological pacemakers, and we show that excitable cells can be composed into bioelectric circuits capable of primitive information processing and memory. We also show that electrically bistable tissues can form reaction-diffusion patterns of electrical domains which undergo phase transitions via spontaneous symmetry breaking. These previously unobserved classes of stable electrical patterns suggest a potential role for electrophysiology during embryonic development. This strategy of \\u2018bioelectrical engineering\\u2019 can articulate subtle aspects of electrophysiological patterning that may be obscured by the complexity of the embryo in vivo. The latter portion of this thesis describes progress towards developing tools for optical manipulation and measurement of spatial patterns of gene expression. The ability to spatially pattern morphogen signals in the embryo, and to map the complete transcriptional response of a system to a given signal, could enable the pressure-testing of mutually consistent morphogen models which\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1433&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d3c3490c7a4cefcc35c2ae1558ba2a499f165aa8\",\n",
      "        \"title\": \"Computational approaches for characterizing the tumor immune microenvironment\",\n",
      "        \"abstract\": \"Recent advances in high\\u2010throughput molecular profiling technologies and multiplexed imaging platforms have revolutionized our ability to characterize the tumor immune microenvironment. As a result, studies of tumor\\u2010associated immune cells increasingly involve complex data sets that require sophisticated methods of computational analysis. In this review, we present an overview of key assays and related bioinformatics tools for analyzing the tumor\\u2010associated immune system in bulk tissues and at the single\\u2010cell level. In parallel, we describe how data science strategies and novel technologies have advanced tumor immunology and opened the door for new opportunities to exploit host immunity to improve cancer clinical outcomes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1434&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c86a91466dc6ff586cc20cbf87bcaa1f78de216f\",\n",
      "        \"title\": \"Dissection of the 4D chromatin structure of the \\u03b1-globin locus through in vivo erythroid differentiation with extreme spatial and temporal resolution\",\n",
      "        \"abstract\": \"Precise gene expression patterns during mammalian development are controlled by regulatory elements in the non-coding genome. Active enhancer elements interact with gene promoters within Topologically Associating Domains (TADs)1\\u20133. However, the precise relationships between chromatin accessibility, nuclear architecture and gene activation are not completely understood. Here, we present Tiled-C, a new Chromosome Conformation Capture (3C) technology4, which allows for the generation of high-resolution contact matrices of loci of interest at unprecedented depth, and which can be optimized for as few as 2,000 cells of input material. We have used this approach to study the chromatin architecture of the mouse \\u03b1-globin locus through in vivo erythroid differentiation. Integrated analysis of matched chromatin accessibility and single-cell expression data shows that the \\u03b1-globin locus lies within a pre-existing TAD, which is established prior to activation of the domain. During differentiation, this TAD undergoes further sub-compartmentalization as regulatory elements gradually become accessible and specific interactions between enhancers and promoters are formed. As these chromatin changes develop, gene expression is progressively upregulated. Our findings demonstrate that chromatin architecture and gene activation are tightly linked during development and provide insights into the distinct mechanisms contributing to the establishment of tissue-specific chromatin structures.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1435&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2aeb6e1070ba7513a902b5e594f5fd3710467d68\",\n",
      "        \"title\": \"Dynamics of the 4D genome during lineage specification, differentiation and maturation in vivo\",\n",
      "        \"abstract\": \"Mammalian gene expression patterns are controlled by regulatory elements, which interact within Topologically Associating Domains (TADs). The relationship between activation of regulatory elements, formation of structural chromatin interactions and gene expression during development is unclear. We developed Tiled-C, a low-input Chromosome Conformation Capture (3C) approach, to study chromatin architecture at high spatial and temporal resolution through in vivo mouse erythroid differentiation. Integrated analysis of matched chromatin accessibility and single-cell expression data shows that regulatory elements gradually become accessible within pre-existing TADs during early differentiation. This is followed by structural re-organization within the TAD and formation of specific contacts between enhancers and promoters. In contrast to previous reports, our high-resolution data show that these enhancer-promoter interactions are not established prior to gene expression, but formed gradually during differentiation, concomitant with progressive upregulation of gene activity. Together, these results provide new insight into the close, interdependent relationship between chromatin architecture and gene regulation during development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1436&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"15af5beb3aa3cc26b804e9797653f3c46b7fd0d4\",\n",
      "        \"title\": \"scBFA: modeling detection patterns to mitigate technical noise in large-scale single-cell genomics data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1437&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"648d38c79a84315ec395d1f43d08e754c6667784\",\n",
      "        \"title\": \"Syndecan-1 promotes lung fibrosis by regulating epithelial reprogramming through extracellular vesicles.\",\n",
      "        \"abstract\": \"Idiopathic pulmonary fibrosis (IPF) is a chronic and fatal lung disease. A maladaptive epithelium due to chronic injury is a prominent feature and contributor to pathogenic cellular communication in IPF. Recent data highlight the concept of a \\\"reprogrammed\\\" lung epithelium as critical in the development of lung fibrosis. Extracellular vesicles (EVs) are potent mediator of cellular crosstalk, and recent evidence supports their role in lung pathologies such as IPF. Here, we demonstrate that syndecan-1 is overexpressed by the epithelium in the lungs of IPF patients and in murine models after bleomycin injury. Moreover, we find that syndecan-1 is a pro-fibrotic signal that alters alveolar type II (ATII) cell phenotypes by augmenting TGF\\u03b2 and Wnt signaling among other pro-fibrotic pathways. Importantly, we demonstrate that syndecan-1 controls the packaging of several anti-fibrotic microRNAs into EVs that have broad effects over several fibrogenic signaling networks as a mechanism of regulating epithelial plasticity and pulmonary fibrosis. Collectively, our work reveals new insight into how EVs orchestrate cellular signals that promote lung fibrosis and demonstrate the importance of syndecan-1 in coordinating these programs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1438&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6c50c30fdd8323886e3da969343f0f82fbbe30b3\",\n",
      "        \"title\": \"A simple method to quantify proteins from one thousand cells\",\n",
      "        \"abstract\": \"Recent developments in single-cell transcriptomics have created an urgent need for similar approaches to map the proteome in samples from a minimal number of cells. We optimized multiple steps in the mass spectrometry protocol to develop such a method, MinPut, with improved sensitivity to quantify proteins from as few as 1,000 mammalian cells. Min-Put uses chemical peptide labeling and does not require specific equipment, antibodies, or other materials. MinPut quantifies >2,500 proteins with high reproducibility. We established and validated the method by comparing mouse embryonic stem cells and in vitro differentiated motor neurons. MinPut correctly identifies differentially expressed proteins with small fold-changes, and a dynamic range in abundance similar to that of standard methods. Protein abundance measurements obtained with MinPut compare well to corresponding transcript abundance and to measurements using standard inputs. Therefore, MinPut offers a robust and accurate method to acquire proteomics data from minimal input samples.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1439&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"94cbb75771c9fe2e8594b850693ef1a88778bb85\",\n",
      "        \"title\": \"Single-cell multimodal transcriptomics to study neuronal diversity in human stem cell-derived brain tissue and organoid models\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1440&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a39c09ef3c0f28528df1523e12f6f47ef4b3a300\",\n",
      "        \"title\": \"Single\\u2010cell technologies in reproductive immunology\",\n",
      "        \"abstract\": \"The maternal\\u2010fetal interface represents a unique immune privileged site that maintains the ability to defend against pathogens while orchestrating the necessary tissue remodeling required for placentation. The recent discovery of novel cellular families (innate lymphoid cells, tissue\\u2010resident NK cells) suggests that our understanding of the decidual immunome is incomplete. To understand this complex milieu, new technological developments allow reproductive immunologists to collect increasingly complex data at a cellular resolution. Polychromatic flow cytometry allows for greater resolution in the identification of novel cell types by surface and intracellular protein. Single\\u2010cell RNA\\u2010seq coupled with microfluidics allows for efficient cellular transcriptomics. The extreme dimensionality and size of data sets generated, however, requires the application of novel computational approaches for unbiased analysis. There are now multiple dimensionality reduction (tSNE, SPADE) and visualization tools (SPICE) that allow researchers to efficiently analyze flow cytometry data. Development of computational tools has also been extended to RNA\\u2010seq data (including scRNA\\u2010seq), which requires specific analytical tools. Here, we provide an overview and a brief primer for the reproductive immunology community on data acquisition and computational tools for the analysis of complex flow cytometry and RNA\\u2010seq data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1441&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"39b68545a1f36a8316eb131121310901e2e2206a\",\n",
      "        \"title\": \"Data denoising with transfer learning in single-cell transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1442&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d553c705e4880935b8bf74c2a5d9e72b1033ad22\",\n",
      "        \"title\": \"Systematic identification of engineered methionines and oxaziridines for efficient, stable, and site-specific antibody bioconjugation\",\n",
      "        \"abstract\": \"Significance Site-specific chemical modification of proteins remains a critical need for bioconjugation. Here we explore the key parameters needed for efficient, selective, and stable modification of methionine using recently developed oxaziridine reagents, called ReACT. We systematically tested various oxaziridine compounds, and scanned accessible and buried sites in a therapeutic antibody for breast cancer to determine the chemical and structural parameters for most stable and efficient modification. We show these adducts are highly stable over days and can support the delivery of toxic payloads to regress tumors in animals. These studies on this important chemical modification expand our capability to site-specifically modify proteins and antibodies for many applications. The field of chemical modification of proteins has been dominated by random modification of lysines or more site-specific labeling of cysteines, each with attendant challenges. Recently, we have developed oxaziridine chemistry for highly selective modification of methionine called redox-activated chemical tagging (ReACT) but have not broadly tested the molecular parameters for efficient and stable protein modification. Here we systematically scanned methionines throughout one of the most popular antibody scaffolds, trastuzumab, used for antibody engineering and drug conjugation. We tested the expression, reactivities, and stabilities of 123 single engineered methionines distributed over the surface of the antibody when reacted with oxaziridine. We found uniformly high expression for these mutants and excellent reaction efficiencies with a panel of oxaziridines. Remarkably, the stability to hydrolysis of the sulfimide varied more than 10-fold depending on temperature and the site of the engineered methionine. Interestingly, the most stable and reactive sites were those that were partially buried, presumably because of their reduced access to water. There was also a 10-fold variation in stability depending on the nature of the oxaziridine, which was determined to be inversely correlated with the electrophilic nature of the sulfimide. Importantly, the stabilities of the best analogs were sufficient to support their use as antibody drug conjugates and potent in a breast cancer mouse xenograft model over a month. These studies provide key parameters for broad application of ReACT for efficient, stable, and site-specific antibody and protein bioconjugation to native or engineered methionines.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1443&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ad94b42d061d6ebf953b83390973850066d759fa\",\n",
      "        \"title\": \"Combined mRNA and protein single cell analysis in a dynamic cellular system using SPARC\",\n",
      "        \"abstract\": \"Combined measurements of mRNA and protein expression in single cells enables in-depth analysis of cellular states. We present single-cell protein and RNA co-profiling (SPARC), an approach to simultaneously measure global mRNA and large sets of intracellular protein in individual cells. Using SPARC, we show that mRNA expression fails to accurately reflect protein abundance at the time of measurement in human embryonic stem cells, although the direction of changes of mRNA and protein expression are in agreement during cellular differentiation. Moreover, protein levels of transcription factors better predict their downstream effects than do the corresponding transcripts. We further show that changes of the balance between protein and mRNA expression levels can be applied to infer expression kinetic trajectories, revealing future states of individual cells. Finally, we highlight that mRNA expression may be more varied among cells than levels of the corresponding proteins. Overall, our results demonstrate that mRNA and protein measurements in single cells provide different and complementary information regarding cell states. Accordingly, SPARC can offer valuable insights in gene expression programs of single cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1444&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"10147ef91caecf0382371915a05888fe0975960f\",\n",
      "        \"title\": \"Unravelling tumour heterogeneity by single-cell profiling of circulating tumour cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1445&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"237d57dff505a96b7bcbb5b000758cdd4837e6b2\",\n",
      "        \"title\": \"A single\\u2010cell transcriptome atlas of the adult human retina\",\n",
      "        \"abstract\": \"The retina is a specialized neural tissue that senses light and initiates image processing. Although the functional organization of specific retina cells has been well studied, the molecular profile of many cell types remains unclear in humans. To comprehensively profile the human retina, we performed single\\u2010cell RNA sequencing on 20,009 cells from three donors and compiled a reference transcriptome atlas. Using unsupervised clustering analysis, we identified 18 transcriptionally distinct cell populations representing all known neural retinal cells: rod photoreceptors, cone photoreceptors, M\\u00fcller glia, bipolar cells, amacrine cells, retinal ganglion cells, horizontal cells, astrocytes, and microglia. Our data captured molecular profiles for healthy and putative early degenerating rod photoreceptors, and revealed the loss of MALAT1 expression with longer post\\u2010mortem time, which potentially suggested a novel role of MALAT1 in rod photoreceptor degeneration. We have demonstrated the use of this retina transcriptome atlas to benchmark pluripotent stem cell\\u2010derived cone photoreceptors and an adult M\\u00fcller glia cell line. This work provides an important reference with unprecedented insights into the transcriptional landscape of human retinal cells, which is fundamental to understanding retinal biology and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1446&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8243c2ea85452dde15a37716fa8bface8a3e368e\",\n",
      "        \"title\": \"Time-resolved single-cell transcriptomics uncovers dynamics of cardiac neutrophil diversity in murine myocardial infarction\",\n",
      "        \"abstract\": \"Objective After myocardial infarction, neutrophils rapidly and massively infiltrate the heart, where they can promote both tissue healing and damage. Here, we investigated the dynamics of cardiac neutrophil heterogeneity after infarction. Methods and results We employed single-cell transcriptomics (scRNA-seq) to investigate temporal neutrophil heterogeneity in the heart after murine myocardial infarction. At day 1, 3, and 5 after infarction, neutrophils could be delineated into six distinct clusters with specific time-dependent patterning and proportions. While the majority of neutrophils at day 1 were characterized by high expression of chemokines (e.g. Cxcl3, Ccl6), and putative activity of transcriptional regulators involved in hypoxic response (Hif1a) and emergency granulopoiesis (Cebpb), two major subsets of Siglecfhi (enriched for e.g. Icam1 and Tnf) and Siglecflow (Slpi, Ifitm1) neutrophils were found at 3 and 5 days. Flow cytometry analysis confirmed the presence of LY6G+SIGLECFhi and LY6G+SIGLECFlow neutrophils in the heart from 3 days after infarction onwards. LY6G+SIGLECFhi neutrophils were absent from the bone marrow, blood and spleen, suggesting local acquisition of surface SIGLECF. Acquisition of the SIGLECFhi state was paralleled by features of neutrophil ageing and activation (ICAM1hiCXCR4hiCD49dhiCD62Llow). scRNA-seq of atherosclerotic aortas revealed two neutrophil subsets with gene expression patterns reminiscent of the major Siglecfhi and Siglecflow cardiac neutrophil subpopulations, revealing that these populations may be present across distinct contexts of cardiovascular inflammation. Conclusion Altogether, our data provide a time-resolved census of neutrophil diversity and gene expression dynamics in the mouse ischemic heart at the single-cell level, and suggests that temporal neutrophil heterogeneity is in part driven by local transition to a SIGLECFhi state.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1447&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b64d561059bec7d4c248f9d4628d964f436ba13f\",\n",
      "        \"title\": \"Ultra-sensitive digital quantification of proteins and mRNA in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1448&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"69c13c329179c19fbd8e675f9ea021594aaca7d4\",\n",
      "        \"title\": \"Immune Profiling of Atherosclerotic Plaques Identifies Innate and Adaptive Dysregulations Associated with Ischemic Cerebrovascular Events\",\n",
      "        \"abstract\": \"Atherosclerosis is driven by multifaceted contributions of the immune system within the circulation and at vascular focal sites. Yet the specific immune dysregulations within the atherosclerotic lesions that lead to clinical cerebro- and cardiovascular complications (i.e. ischemic stroke and myocardial infarction) are poorly understood. Here, using single-cell mass cytometry with Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) we found that atherosclerotic plaques were enriched in activated, differentiated, and exhausted subsets of T cells vs. blood. Next, using single-cell proteomic, transcriptomic, and cell-to-cell interaction analyses we found unique functional dysregulations of both T cells and macrophages in plaques of patients with clinically symptomatic (SYM; recent stroke of TIA) or asymptomatic (ASYM, no recent stroke) carotid artery disease. SYM plaques were enriched with a distinct CD4+ T cell subset, and T cells were activated, differentiated and presented subset specific exhaustion. SYM macrophages presented alternatively activated phenotypes including subsets associated with plaque vulnerability. In ASYM plaques, T cells and macrophages were activated and displayed a strong IL-1\\u03b2 signaling across cell types, that was absent in SYM plaques. The identification of plaque-specific innate and adaptive immune dysregulations associated with cerebrovascular events provides the basis for the design of precisely tailored cardiovascular immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1449&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"99097cc2ad29ea8c2c83303ddf334d24be07e41e\",\n",
      "        \"title\": \"Tumor heterogeneity of acute myeloid leukemia: insights from single-cell sequencing\",\n",
      "        \"abstract\": \"Abstract Individual tumors comprise genetically and epigenetically heterogeneous subclones, each of which is presumably associated with a distinct function, such as self-renewal or drug sensitivity. The dissection of such intratumoral heterogeneity is crucial to understand how tumors evolve during disease progression and under the selection of therapeutic intervention. As a paradigm of cancer intratumoral heterogeneity and clonal evolution, acute myeloid leukemia (AML) has been shown to possess complex clonal architecture based on karyotype studies, as well as deep sequencing of mixed cellular populations using next-generation sequencing (NGS) technologies. The recent development of single-cell sequencing (SCS) methods provides a powerful tool to allow analysis of genomes, transcriptomes, proteomes, and epigenomes at an individual cell level. The technologies applied in AML have broadened our understanding of AML heterogeneity and provided new insights for the development of novel therapeutic strategies. In this review, we summarize the progress in the research of AML heterogeneity using SCS technology and discuss the limitations and future direction regarding how SCS can contribute to AML prognosis and treatment.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1450&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5e78fab5edfd6e7f40fc8586ce205da4d5a0bcd2\",\n",
      "        \"title\": \"Recent progress in the analysis of \\u03b1\\u03b2T cell and B cell receptor repertoires.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1451&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"883accee04e309f7be480daa114808b76bac7c19\",\n",
      "        \"title\": \"CRISPR Tools for Systematic Studies of RNA Regulation.\",\n",
      "        \"abstract\": \"SUMMARYRNA molecules perform diverse functions in mammalian cells, including transferring genetic information from DNA to protein and playing diverse regulatory roles through interactions with other cellular components. Here, we discuss how clustered regularly interspaced short palindromic repeat (CRISPR)-based technologies for directed perturbations of DNA and RNA are revealing new insights into RNA regulation. First, we review the fundamentals of CRISPR-Cas enzymes and functional genomics tools that leverage these systems. Second, we explore how these new perturbation technologies are transforming the study of regulation of and by RNA, focusing on the functions of DNA regulatory elements and long noncoding RNAs (lncRNAs). Third, we highlight an emerging class of RNA-targeting CRISPR-Cas enzymes that have the potential to catalyze studies of RNA biology by providing tools to directly perturb or measure RNA modifications and functions. Together, these tools enable systematic studies of RNA function and regulation in mammalian cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1452&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7059ee2cbfa55f262ba882b1a6912518d23ea666\",\n",
      "        \"title\": \"Single Cell RNA-Sequencing for the Study of Atherosclerosis\",\n",
      "        \"abstract\": \"Atherosclerosis is a major cause of coronary artery disease and stroke. A massive and new type of data has finally arrived in the field of atherosclerosis: single cell RNA sequencing (scRNAseq). Recently, scRNAseq has been successfully applied to the study of atherosclerosis to identify previously uncharacterized cell populations. scRNAseq is an effective approach to evaluate heterogeneous cell populations by measuring the transcriptomic profiles at the single cell level. Besides the studies of atherosclerosis, scRNAseq is being employed in various areas of biology, including cancer research and organ development. In order to analyze these new massive datasets, various analytic approaches have been developed. This review aims to enhance the understanding of this new technology by exploring how the single cell transcriptome has been applied to the study of atherosclerosis and further discuss potential analysis of using scRNAseq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1453&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ab4d38328f7f7215bac2b3b1f50c243751b7516c\",\n",
      "        \"title\": \"Next-generation computational tools for interrogating cancer immunity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1454&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"86c092eafe2b4af8bfb40edf7cd2d62127223c81\",\n",
      "        \"title\": \"Heterogeneity and plasticity in healthy and atherosclerotic vasculature explored by single-cell sequencing\",\n",
      "        \"abstract\": \"Abstract Cellular characteristics and their adjustment to a state of disease have become more evident due to recent advances in imaging, fluorescent reporter mice, and whole genome RNA sequencing. The uncovered cellular heterogeneity and/or plasticity potentially complicates experimental studies and clinical applications, as markers derived from whole tissue \\u2018bulk\\u2019 sequencing is unable to yield a subtype transcriptome and specific markers. Here, we propose definitions on heterogeneity and plasticity, discuss current knowledge thereof in the vasculature and how this may be improved by single-cell sequencing (SCS). SCS is emerging as an emerging technique, enabling researchers to investigate different cell populations in more depth than ever before. Cell selection methods, e.g. flow assisted cell sorting, and the quantity of cells can influence the choice of SCS method. Smart-Seq2 offers sequencing of the complete mRNA molecule on a low quantity of cells, while Drop-seq is possible on large numbers of cells on a more superficial level. SCS has given more insight in heterogeneity in healthy vasculature, where it revealed that zonation is crucial in gene expression profiles among the anatomical axis. In diseased vasculature, this heterogeneity seems even more prominent with discovery of new immune subsets in atherosclerosis as proof. Vascular smooth muscle cells and mesenchymal cells also share these plastic characteristics with the ability to up-regulate markers linked to stem cells, such as Sca-1 or CD34. Current SCS studies show some limitations to the number of replicates, quantity of cells used, or the loss of spatial information. Bioinformatical tools could give some more insight in current datasets, making use of pseudo-time analysis or RNA velocity to investigate cell differentiation or polarization. In this review, we discuss the use of SCS in unravelling heterogeneity in the vasculature, its current limitations and promising future applications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1455&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cf9070b6bb84360565988f3733fc43f44116100d\",\n",
      "        \"title\": \"Bayesian correlation is a robust gene similarity measure for single-cell RNA-seq data\",\n",
      "        \"abstract\": \"Assessing similarity is highly important for bioinformatics algorithms to determine correlations between biological information. A common problem is that similarity can appear by chance, particularly for low expressed entities. This is especially relevant in single cell RNA-seq (scRNA-seq) data because read counts are much lower compared to bulk RNA-seq. Recently, a Bayesian correlation scheme, that assigns low similarity to genes that have low confidence expression estimates, has been proposed to assess similarity for bulk RNA-seq. Our goal is to extend the properties of the Bayesian correlation in scRNA-seq data by considering 3 ways to compute similarity. First, we compute the similarity of pairs of genes over all cells. Second, we identify specific cell populations and compute the correlation in those populations. Third, we compute the similarity of pairs of genes over all clusters, by considering the total mRNA expression. We demonstrate that Bayesian correlations are more reproducible than Pearson correlations. Compared to Pearson correlations, Bayesian correlations have a smaller dependence on the number of input cells. We show that the Bayesian correlation algorithm assigns high similarity values to genes with a biological relevance in a specific population. We conclude that Bayesian correlation is a robust similarity measure in scRNA-seq data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1456&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ee632fa425607e8ff91fc3730bc0782d43ce9c0c\",\n",
      "        \"title\": \"Techniques to Study Antigen-Specific B Cell Responses\",\n",
      "        \"abstract\": \"Antibodies against foreign antigens are a critical component of the overall immune response and can facilitate pathogen clearance during a primary infection and also protect against subsequent infections. Dysregulation of the antibody response can lead to an autoimmune disease, malignancy, or enhanced infection. Since the experimental delineation of a distinct B cell lineage in 1965, various methods have been developed to understand antigen-specific B cell responses in the context of autoimmune diseases, primary immunodeficiencies, infection, and vaccination. In this review, we summarize the established techniques and discuss new and emerging technologies for probing the B cell response in vitro and in vivo by taking advantage of the specificity of B cell receptor (BCR)-associated and secreted antibodies. These include ELISPOT, flow cytometry, mass cytometry, and fluorescence microscopy to identify and/or isolate primary antigen-specific B cells. We also present our approach to identify rare antigen-specific B cells using magnetic enrichment followed by flow cytometry. Once these cells are isolated, in vitro proliferation assays and adoptive transfer experiments in mice can be used to further characterize antigen-specific B cell activation, function, and fate. Transgenic mouse models of B cells targeting model antigens and of B cell signaling have also significantly advanced our understanding of antigen-specific B cell responses in vivo.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1457&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d7ce985df7fe5587601140df4745fb2637c5e7d5\",\n",
      "        \"title\": \"NASQAR: a web-based platform for high-throughput sequencing data analysis and visualization\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1458&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cdabf1485bba124b1301301a6b6c4ee8c9f24915\",\n",
      "        \"title\": \"Simultaneous mRNA and protein quantification at the single-cell level delineates trajectories of CD4+ T-cell differentiation\",\n",
      "        \"abstract\": \"Abstract The transcriptomic and proteomic characterisation of CD4+ T cells at the single-cell level has been performed traditionally by two largely exclusive types of technologies: single cell RNA-sequencing (scRNA-seq) technologies and antibody-based cytometry. Here we demonstrate that the simultaneous targeted quantification of mRNA and protein expression in single-cells provides a high-resolution map of human primary CD4+ T cells, and reveals precise trajectories of canonical T-cell lineage differentiation in blood and tissue. We report modest correlation between mRNA and protein in primary CD4+ T cells at the single-cell level, highlighting the importance of including quantitative protein expression data to characterise cell effector function. This approach provides a massively-parallel, cost-effective, solution to dissect the heterogeneity of immune cell populations and is ideally suited for the detailed immunophenotypic characterisation of rare and potentially pathogenic immune subsets. This transcriptomic and proteomic hybrid technology could have important clinical applications to re-define differentiation and activation of tissue-resident and blood human immune cells.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1459&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"93babfd45e4fc6b1b0c6d22c7144c7a1f06d3cfc\",\n",
      "        \"title\": \"Discovery of CD80 and CD86 as recent activation markers on regulatory T cells by protein-RNA single-cell analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1460&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2bdc46c158a65132bf44237a58aebd20d9c59159\",\n",
      "        \"title\": \"A Targeted Multi-omic Analysis Approach Measures Protein Expression and Low-Abundance Transcripts on the Single-Cell Level\",\n",
      "        \"abstract\": \"High throughput single-cell RNA sequencing (sc-RNAseq) has become a frequently used tool to assess immune cell function and heterogeneity. Recently, the combined measurement of RNA and protein expression by sequencing was developed, which is commonly known as CITE-Seq. Acquisition of protein expression data along with transcriptome data resolves some of the limitations inherent to only assessing transcript, but also nearly doubles the sequencing read depth required per single cell. Furthermore, there is still a paucity of analysis tools to visualize combined transcript-protein datasets. Here, we describe a novel targeted transcriptomics approach that combines analysis of over 400 genes with simultaneous measurement of over 40 proteins on more than 25,000 cells. This targeted approach requires only about 1/10 of the read depth compared to a whole transcriptome approach while retaining high sensitivity for low abundance transcripts. To analyze these multi-omic transcript-protein datasets, we adapted One-SENSE for intuitive visualization of the relationship of proteins and transcripts on a single-cell level.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1461&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ef4da81ef2e63c6c608b1ef459bc7a6a588080b\",\n",
      "        \"title\": \"Mapping Vector Field of Single Cells\",\n",
      "        \"abstract\": \"Understanding how gene expression in single cells progress over time is vital for revealing the mechanisms governing cell fate transitions. RNA velocity, which infers immediate changes in gene expression by comparing levels of new (unspliced) versus mature (spliced) transcripts (La Manno et al. 2018), represents an important advance to these efforts. A key question remaining is whether it is possible to predict the most probable cell state backward or forward over arbitrary time-scales. To this end, we introduce an inclusive model (termed Dynamo) capable of predicting cell states over extended time periods, that incorporates promoter state switching, transcription, splicing, translation and RNA/protein degradation by taking advantage of scRNA-seq and the co-assay of transcriptome and proteome. We also implement scSLAM-seq by extending SLAM-seq to plate-based scRNA-seq (Hendriks et al. 2018; Erhard et al. 2019; Cao, Zhou, et al. 2019) and augment the model by explicitly incorporating the metabolic labelling of nascent RNA. We show that through careful design of labelling experiments and an efficient mathematical framework, the entire kinetic behavior of a cell from this model can be robustly and accurately inferred. Aided by the improved framework, we show that it is possible to reconstruct the transcriptomic vector field from sparse and noisy vector samples generated by single cell experiments. The reconstructed vector field further enables global mapping of potential landscapes that reflects the relative stability of a given cell state, and the minimal transition time and most probable paths between any cell states in the state space. This work thus foreshadows the possibility of predicting long-term trajectories of cells during a dynamic process instead of short time velocity estimates. Our methods are implemented as an open source tool, dynamo (https://github.com/aristoteleo/dynamo-release).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1462&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5ae3aea987e923ddef16b762085b7b22241867de\",\n",
      "        \"title\": \"A single cell framework for multi-omic analysis of disease identifies malignant regulatory signatures in mixed phenotype acute leukemia\",\n",
      "        \"abstract\": \"In order to identify the molecular determinants of human diseases, such as cancer, that arise from a diverse range of tissue, it is necessary to accurately distinguish normal and pathogenic cellular programs.1\\u20133 Here we present a novel approach for single-cell multi-omic deconvolution of healthy and pathological molecular signatures within phenotypically heterogeneous malignant cells. By first creating immunophenotypic, transcriptomic and epigenetic single-cell maps of hematopoietic development from healthy peripheral blood and bone marrow mononuclear cells, we identify cancer-specific transcriptional and chromatin signatures from single cells in a cohort of mixed phenotype acute leukemia (MPAL) clinical samples. MPALs are a high-risk subtype of acute leukemia characterized by a heterogeneous malignant cell population expressing both myeloid and lymphoid lineage-specific markers.4, 5 Our results reveal widespread heterogeneity in the pathogenetic gene regulatory and expression programs across patients, yet relatively consistent changes within patients even across malignant cells occupying diverse portions of the hematopoietic lineage. An integrative analysis of transcriptomic and epigenetic maps identifies 91,601 putative gene-regulatory interactions and classifies a number of transcription factors that regulate leukemia specific genes, including RUNX1-linked regulatory elements proximal to CD69. This work provides a template for integrative, multi-omic analysis for the interpretation of pathogenic molecular signatures in the context of developmental origin.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1463&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f34b3123c91587062984ba9290faa17f81da831d\",\n",
      "        \"title\": \"Network modeling of single-cell omics data: challenges, opportunities, and progresses\",\n",
      "        \"abstract\": \"Abstract Single-cell multi-omics technologies are rapidly evolving, prompting both methodological advances and biological discoveries at an unprecedented speed. Gene regulatory network modeling has been used as a powerful approach to elucidate the complex molecular interactions underlying biological processes and systems, yet its application in single-cell omics data modeling has been met with unique challenges and opportunities. In this review, we discuss these challenges and opportunities, and offer an overview of the recent development of network modeling approaches designed to capture dynamic networks, within-cell networks, and cell\\u2013cell interaction or communication networks. Finally, we outline the remaining gaps in single-cell gene network modeling and the outlooks of the field moving forward.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1464&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"023269b13a9ab080d8005f20038b0fc1910b15d0\",\n",
      "        \"title\": \"Recharacterizing Tumor-Infiltrating Lymphocytes by Single-Cell RNA Sequencing\",\n",
      "        \"abstract\": \"T cells are central players in cancer immunotherapy. Despite much concentrated effort on the study of tumor-infiltrating lymphocytes (TIL), such as T cells, a series of fundamental properties that include heterogeneity, clonal expansion, migration, and functional state transition remain elusive. Advances of single-cell sequencing have enabled the detailed characterization of immune cells in tumors and have vastly improved our understanding of less-defined cell subsets. Here, we discuss the current strategies for uncovering the heterogeneity of TILs, and how the deep transcriptome coupled with T-cell receptor analysis enhances the understanding of detailed properties of T-cell subsets. We further discuss the identification of novel T-cell markers with therapeutic or prognosis potentials, and highlight distinct T-cell properties among different cancer indications.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1465&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f5a509e21663035a0d00e8a4db3fdc163da51a58\",\n",
      "        \"title\": \"The Human Lung Cell Atlas - A high-resolution reference map of the human lung in health and disease.\",\n",
      "        \"abstract\": \"Lung disease accounts for every sixth death globally. Profiling the molecular state of all lung cell types in health and disease is currently revolutionizing the identification of disease mechanisms and will aid the design of novel diagnostic and personalized therapeutic regimens. Recent progress in high-throughput techniques for single-cell genomic and transcriptomic analyses has opened up new possibilities to study individual cells within a tissue, classify these into cell types, and characterize variations in their molecular profiles as a function of genetics, environment, cell-cell interactions, developmental processes, ageing or disease. Integration of these cell state definitions with spatial information allows the in-depth molecular description of cellular neighborhoods and tissue microenvironments including the tissue resident structural and immune cells, the tissue matrix as well as the microbiome. The Human Cell Atlas consortium aims to characterize all cells in the healthy human body, and has prioritized lung tissue as one of the flagship projects. Here, we present the rationale, the approach and the expected impact of a Human Lung Cell Atlas.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1466&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3f29542388653b49afee3086a8942e205440c6df\",\n",
      "        \"title\": \"The Unmixing Problem: A Guide to Applying Single\\u2010Cell RNA Sequencing to Bone\",\n",
      "        \"abstract\": \"Bone is composed of a complex mixture of many dynamic cell types. Flow cytometry and in vivo lineage tracing have offered early progress toward deconvoluting this heterogeneous mixture of cells into functionally well\\u2010defined populations suitable for further studies. Single\\u2010cell sequencing is poised as a key complementary technique to better understand the cellular basis of bone metabolism and development. However, single\\u2010cell sequencing approaches still have important limitations, including transcriptional effects of cell isolation and sparse sampling of the transcriptome, that must be considered during experimental design and analysis to harness the power of this approach. Accounting for these limitations requires a deep knowledge of the tissue under study. Therefore, with the emergence of accessible tools for conducting and analyzing single\\u2010cell RNA sequencing (scRNA\\u2010seq) experiments, bone biologists will be ideal leaders in the application of scRNA\\u2010seq to the skeleton. Here we provide an overview of the steps involved with a single\\u2010cell sequencing analysis of bone, focusing on practical considerations needed for a successful study. \\u00a9 2019 American Society for Bone and Mineral Research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1467&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9cb3d2a8918d59be29dec97d1f91550bb5ede1b2\",\n",
      "        \"title\": \"Immuno-SABER enables highly multiplexed and amplified protein imaging in tissues\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1468&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9b85f6f1ca7350d69ee869f3a70c0ef4c874fa46\",\n",
      "        \"title\": \"MULTI-seq: sample multiplexing for single-cell RNA sequencing using lipid-tagged indices\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1469&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"007fab3ae2d888da9faedbd28080d9bf15b3887a\",\n",
      "        \"title\": \"Single-cell transcriptomic analysis of mIHC images via antigen mapping\",\n",
      "        \"abstract\": \"Mapping single-cell RNA-seq data onto IHC images enables the annotation of cell populations at high phenotypic resolution. Highly multiplexed immunohistochemistry (mIHC) enables the staining and quantification of dozens of antigens in a tissue section with single-cell resolution. However, annotating cell populations that differ little in the profiled antigens or for which the antibody panel does not include specific markers is challenging. To overcome this obstacle, we have developed an approach for enriching mIHC images with single-cell RNA sequencing data, building upon recent experimental procedures for augmenting single-cell transcriptomes with concurrent antigen measurements. Spatially-resolved Transcriptomics via Epitope Anchoring (STvEA) performs transcriptome-guided annotation of highly multiplexed cytometry datasets. It increases the level of detail in histological analyses by enabling the systematic annotation of nuanced cell populations, spatial patterns of transcription, and interactions between cell types. We demonstrate the utility of STvEA by uncovering the architecture of poorly characterized cell types in the murine spleen using published cytometry and mIHC data of this organ.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1470&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bf751285bb7a80abd116bc95ce2c406175e2cd87\",\n",
      "        \"title\": \"Concepts and limitations for learning developmental trajectories from single cell genomics\",\n",
      "        \"abstract\": \"ABSTRACT Single cell genomics has become a popular approach to uncover the cellular heterogeneity of progenitor and terminally differentiated cell types with great precision. This approach can also delineate lineage hierarchies and identify molecular programmes of cell-fate acquisition and segregation. Nowadays, tens of thousands of cells are routinely sequenced in single cell-based methods and even more are expected to be analysed in the future. However, interpretation of the resulting data is challenging and requires computational models at multiple levels of abstraction. In contrast to other applications of single cell sequencing, where clustering approaches dominate, developmental systems are generally modelled using continuous structures, trajectories and trees. These trajectory models carry the promise of elucidating mechanisms of development, disease and stimulation response at very high molecular resolution. However, their reliable analysis and biological interpretation requires an understanding of their underlying assumptions and limitations. Here, we review the basic concepts of such computational approaches and discuss the characteristics of developmental processes that can be learnt from trajectory models. Summary: This Review describes the concepts and use of computational approaches to infer cellular trajectories from single cell expression data and discusses the applications of these approaches to the study of development and disease.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1471&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d8a2abb4e8a709be5d7a49a4c66265c44a147e2c\",\n",
      "        \"title\": \"The evolving concept of cell identity in the single cell era\",\n",
      "        \"abstract\": \"ABSTRACT Fueled by recent advances in single cell biology, we are moving away from qualitative and undersampled assessments of cell identity, toward building quantitative, high-resolution cell atlases. However, it remains challenging to precisely define cell identity, leading to renewed debate surrounding this concept. Here, I present three pillars that I propose are central to the notion of cell identity: phenotype, lineage and state. I explore emerging technologies that are enabling the systematic and unbiased quantification of these properties, and outline how these efforts will enable the construction of a high-resolution, dynamic landscape of cell identity, potentially revealing its underlying molecular regulation to provide new opportunities for understanding and manipulating cell fate. Summary: This Spotlight explores emerging technologies that are enabling the systematic and unbiased quantification of cell identity, and how these efforts will enable the construction of high-resolution, dynamic cell atlases.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1472&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3716b018d20cc8e50a39521e3eb2a4d263ab3daf\",\n",
      "        \"title\": \"Surface protein imputation from single cell transcriptomes by deep neural networks\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1473&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5660bd87dcc6e95e6b38fbe656de5087292bf35e\",\n",
      "        \"title\": \"Single cell analysis reveals immune cell\\u2013adipocyte crosstalk regulating the transcription of thermogenic adipocytes\",\n",
      "        \"abstract\": \"Immune cells are vital constituents of the adipose microenvironment that influence both local and systemic lipid metabolism. Mice lacking IL10 have enhanced thermogenesis, but the roles of specific cell types in the metabolic response to IL10 remain to be defined. We demonstrate here that selective loss of IL10 receptor \\u03b1 in adipocytes recapitulates the beneficial effects of global IL10 deletion, and that local crosstalk between IL10-producing immune cells and adipocytes is a determinant of thermogenesis and systemic energy balance. Single Nuclei Adipocyte RNA-sequencing (SNAP-seq) of subcutaneous adipose tissue defined a metabolically-active mature adipocyte subtype characterized by robust expression of genes involved in thermogenesis whose transcriptome was selectively responsive to IL10R\\u03b1 deletion. Furthermore, single-cell transcriptomic analysis of adipose stromal populations identified lymphocytes as a key source of IL10 production in response to thermogenic stimuli. These findings implicate adaptive immune cell-adipocyte communication in the maintenance of adipose subtype identity and function.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1474&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9177987bfb5970ae122d743f6859b7ef54a93db4\",\n",
      "        \"title\": \"Ovarian cancer and the evolution of subtype classifications using transcriptional profiling\\u2020\",\n",
      "        \"abstract\": \"Abstract Ovarian cancer is a complex disease with multiple subtypes, each having distinct histopathologies and variable responses to treatment. This review highlights the technological milestones and the studies that have applied them to change our definitions of ovarian cancer. Over the past 50 years, technologies such as microarrays and next-generation sequencing have led to the discovery of molecular alterations that define each of the ovarian cancer subtypes and has enabled further subclassification of the most common subtype, high-grade serous ovarian cancer (HGSOC). Improvements in mutational profiling have provided valuable insight, such as the ubiquity of TP53 mutations in HGSOC tumors. However, the information derived from these technological advances has also revealed the immense heterogeneity of this disease, from variation between patients to compositional differences within single masses. In looking forward, the emerging technologies for single-cell and spatially resolved transcriptomics will allow us to better understand the cellular composition and structure of tumors and how these contribute to the molecular subtypes. Attempts to incorporate the complexities ovarian cancer has resulted in increasing sophistication of model systems, and the increased precision in molecular profiling of ovarian cancers has already led to the introduction of inhibitors of poly (ADP-ribose) polymerases as a new class of treatments for ovarian cancer with DNA repair deficiencies. Future endeavors to define increasingly accurate classification strategies for ovarian cancer subtypes will allow for confident prediction of disease progression and provide important insight into potentially targetable molecular mechanisms specific to each subtype. Summary Sentence Advances in technologies to define the transcriptional profile of ovarian cancers has resulted in continuing evolution of subtype classification, improvements in preclinical animal models and a greater appreciation for the complexity of these tumors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1475&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"38eb98cf7118c2be9b572eb2ec02c8fdf2426fd1\",\n",
      "        \"title\": \"Atheroprotective roles of smooth muscle cell phenotypic modulation and the TCF21 disease gene as revealed by single-cell analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1476&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ffb18dad970608411dd0d0ffb705346a46328b5e\",\n",
      "        \"title\": \"RNA velocity and protein acceleration from single-cell multiomics experiments\",\n",
      "        \"abstract\": \"The simultaneous quantification of protein and RNA makes possible the inference of past, present and future cell states from single experimental snapshots. To enable such temporal analysis from multimodal single-cell experiments, we introduce an extension of the RNA velocity method that leverages estimates of unprocessed transcript and protein abundances to extrapolate cell states. We apply the model to four datasets and demonstrate consistency among landscapes and phase portraits.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1477&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4832f3e33fd05027b83b8b9a9e84fe54a74eeadc\",\n",
      "        \"title\": \"Integration of Single-Cell Genomics Datasets\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1478&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d197473a348efadb23a6bb52de3476fdfdfe7dec\",\n",
      "        \"title\": \"Current best practices in single\\u2010cell RNA\\u2010seq analysis: a tutorial\",\n",
      "        \"abstract\": \"Single\\u2010cell RNA\\u2010seq has enabled gene expression to be studied at an unprecedented resolution. The promise of this technology is attracting a growing user base for single\\u2010cell analysis methods. As more analysis tools are becoming available, it is becoming increasingly difficult to navigate this landscape and produce an up\\u2010to\\u2010date workflow to analyse one's data. Here, we detail the steps of a typical single\\u2010cell RNA\\u2010seq analysis, including pre\\u2010processing (quality control, normalization, data correction, feature selection, and dimensionality reduction) and cell\\u2010 and gene\\u2010level downstream analysis. We formulate current best\\u2010practice recommendations for these steps based on independent comparison studies. We have integrated these best\\u2010practice recommendations into a workflow, which we apply to a public dataset to further illustrate how these steps work in practice. Our documented case study can be found at https://www.github.com/theislab/single-cell-tutorial. This review will serve as a workflow tutorial for new entrants into the field, and help established users update their analysis pipelines.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1479&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3de6efa6e95d2d315551d8ed5e8368b7adcb460b\",\n",
      "        \"title\": \"Single cell RNA-sequencing: replicability of cell types\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1480&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa07fe133713ba9250375d78d5fd6d29edfd5da7\",\n",
      "        \"title\": \"Single-cell RNA sequencing for the study of lupus nephritis\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing (scRNA-seq) has recently undergone rapid advances in the development of this technology, leading to high throughput and accelerating discovery in many biological systems and diseases. The single-cell resolution of the technique allows for the investigation of heterogeneity in cell populations, and the pinpointing of pathological populations contributing to disease. Here we review the development of scRNA-seq technology and the analysis that has evolved with the ever-increasing throughput. Finally, we highlight recent applications of scRNA-seq to understand the molecular pathogenesis of lupus and lupus nephritis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1481&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"614706be76a96a57b444c7482f32404cfe8a2da3\",\n",
      "        \"title\": \"Cyt\\u2010Geist: Current and Future Challenges in Cytometry: Reports of the CYTO 2018 Conference Workshops\",\n",
      "        \"abstract\": \"Kamila Czechowska,* Joanne Lannigan, Lili Wang, Judith Arcidiacono, Thomas M. Ashhurst, Ruth M. Barnard, Steven Bauer, Cl\\u00e1udia Bispo, Diana L. Bonilla, Ryan R. Brinkman, Maciej Cabanski, Hyun-Dong Chang, Lina Chakrabarti, Grace Chojnowski, Bunny Cotleur, Heba Degheidy, Gelo V. Dela Cruz, Steven Eck, John Elliott, Rachel Errington, Andy Filby, Dominic Gagnon, Rui Gardner, Cherie Green, Michael Gregory, Christopher J. Groves, Christopher Hall, Frederik Hammes, Michael Hedrick, Robert Hoffman, Jaroslav Icha, Johanna Ivaska, Dominic C. Jenner, Derek Jones, Frederiek-Maarten Kerckhof, Christian Kukat, David Lanham, Silas Leavesley, Michael Lee, Sheng Lin-Gibson, Virginia Litwin, Yanli Liu, Jenny Molloy, Jonni S. Moore, Susann M\\u00fcller, Jakub Nedbal, Raluca Niesner, Nao Nitta, Betsy Ohlsson-Wilhelm, Nicole E. Paul, Stephen Perfetto, Ziv Portat, Ruben Props, Stefan Radtke, Radhika Rayanki, Aja Rieger, Samson Rogers, Peter Rubbens, Robert Salomon, Matthias Schiemann, John Sharpe, Soren Ulrik Sonder, Jennifer J. Stewart, Yongliang Sun, Henning Ulrich, Gert Van Isterdael, Alessandra Vitaliti, Caryn van Vreden, Michael Weber, Jakob Zimmermann, Giacomo Vacca, Paul Wallace, Attila T\\u00e1rnok\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1482&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ea7b4dd0ba152eb8651150360371856e63487167\",\n",
      "        \"title\": \"Deep learning does not outperform classical machine learning for cell-type annotation\",\n",
      "        \"abstract\": \"Deep learning has revolutionized image analysis and natural language processing with remarkable accuracies in prediction tasks, such as image labeling and semantic segmentation or named-entity recognition and semantic role labeling. Specifically, the combination of algorithmic and hardware advances with the appearance of large and well-labeled datasets has led up to seminal contributions in these fields. The emergence of large amounts of data from single-cell RNA-seq and the recent global effort to chart all cell types in the Human Cell Atlas has attracted an interest in deep-learning applications. However, all current approaches are unsupervised, i.e., learning of latent spaces without using any cell labels, even though supervised learning approaches are often more powerful in feature learning and the most popular approach in the current AI revolution by far. Here, we ask why this is the case. In particular we ask whether supervised deep learning can be used for cell annotation, i.e. to predict cell-type labels from single-cell gene expression profiles. After evaluating 10 classification methods across 14 datasets, we notably find that deep learning does not outperform classical machine-learning methods in the task. Thus, cell-type prediction based on gene-signature derived cell-type labels is potentially too simplistic a task for complex non-linear methods, which demands better labels of functional single-cell readouts.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1483&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dd58b586fce821e1414de10853230f1530e283ae\",\n",
      "        \"title\": \"High throughput pMHC-I tetramer library production using chaperone-mediated peptide exchange\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1484&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c7687bee09f27dd6520bbf7a50721b8d5f936c8f\",\n",
      "        \"title\": \"Combinatorial prediction of marker panels from single\\u2010cell transcriptomic data\",\n",
      "        \"abstract\": \"Single-cell transcriptomic studies are identifying novel cell populations with exciting functional roles in various in vivo contexts, but identification of succinct gene-marker panels for such populations remains a challenge. In this work we introduce COMET, a computational framework for the identification of candidate marker panels consisting of one or more genes for cell populations of interest identified with single-cell RNA-seq data. We show that COMET outperforms other methods for the identification of single-gene panels, and enables, for the first time, prediction of multi-gene marker panels ranked by relevance. Staining by flow-cytometry assay confirmed the accuracy of COMET\\u2019s predictions in identifying marker-panels for cellular subtypes, at both the single- and multi-gene levels, validating COMET\\u2019s applicability and accuracy in predicting favorable marker-panels from transcriptomic input. COMET is a general non-parametric statistical framework and can be used as-is on various high-throughput datasets in addition to single-cell RNA-sequencing data. COMET is available for use via a web interface (http://www.cometsc.com) or a standalone software package (https://github.com/MSingerlab/COMETSC).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1485&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"63e131e07bd0f6a946aad1e91514865d303cd1a3\",\n",
      "        \"title\": \"Single-cell RNA sequencing unveils the shared and the distinct cytotoxic hallmarks of human TCRV\\u03b41 and TCRV\\u03b42 \\u03b3\\u03b4 T lymphocytes\",\n",
      "        \"abstract\": \"Significance None of the single-cell RNA sequencing (scRNA-seq) studies published so far convincingly identified human \\u03b3\\u03b4 T lymphocytes despite their anticancer functions. To address this, we here report scRNA-seq of \\u03b3\\u03b4 T lymphocytes purified from human blood, and a signature identifying \\u03b3\\u03b4 T cells. The single-cell transcriptomes of TCRV\\u03b41 and TCRV\\u03b42 lymphocytes are intermediates resembing those of NK cells and T CD8, respectively, they reflect their respective maturation stage and their pseudo-time maturation trajectory unveils terminally differentiated cells with mitotic capacity. In human cancers, the \\u03b3\\u03b4 TIL, mostly expressing TCRV\\u03b41, appear fewer in tumors than in blood and not correlated to \\u03b1\\u03b2 TIL abundance. This report is a landmark resource for future studies of \\u03b3\\u03b4 T lymphocytes at the single-cell level. \\u03b3\\u03b4 T lymphocytes represent \\u223c1% of human peripheral blood mononuclear cells and even more cells in most tissues of vertebrates. Although they have important anticancer functions, most current single-cell RNA sequencing (scRNA-seq) studies do not identify \\u03b3\\u03b4 T lymphocytes because their transcriptomes at the single-cell level are unknown. Here we show that high-resolution clustering of large scRNA-seq datasets and a combination of gene signatures allow the specific detection of human \\u03b3\\u03b4 T lymphocytes and identification of their T cell receptor (TCR)V\\u03b41 and TCRV\\u03b42 subsets in large datasets from complex cell mixtures. In t-distributed stochastic neighbor embedding plots from blood and tumor samples, the few \\u03b3\\u03b4 T lymphocytes appear collectively embedded between cytotoxic CD8 T and NK cells. Their TCRV\\u03b41 and TCRV\\u03b42 subsets form close yet distinct subclusters, respectively neighboring NK and CD8 T cells because of expression of shared and distinct cytotoxic maturation genes. Similar pseudotime maturation trajectories of TCRV\\u03b41 and TCRV\\u03b42 \\u03b3\\u03b4 T lymphocytes were discovered, unveiling in both subsets an unattended pool of terminally differentiated effector memory cells with preserved proliferative capacity, a finding confirmed by in vitro proliferation assays. Overall, the single-cell transcriptomes of thousands of individual \\u03b3\\u03b4 T lymphocytes from different CMV+ and CMV\\u2212 donors reflect cytotoxic maturation stages driven by the immunological history of donors. This landmark study establishes the rationale for identification, subtyping, and deep characterization of human \\u03b3\\u03b4 T lymphocytes in further scRNA-seq studies of complex tissues in physiological and disease conditions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1486&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"26399251362886b08b63ef3588cc1101950ea639\",\n",
      "        \"title\": \"Challenges in measuring and understanding biological noise\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1487&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1a6412f090cb6214128f1799227e6bc1ab725a5f\",\n",
      "        \"title\": \"MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1488&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"22b25abd4e0e28f60e411a3e7f140f760803c30d\",\n",
      "        \"title\": \"Targets for MAbs: innovative approaches for their discovery & validation, LabEx MAbImprove 6th antibody industrial symposium, June 25-26, 2018, Montpellier, France\",\n",
      "        \"abstract\": \"ABSTRACT Monoclonal antibodies (mAbs) have revolutionized the treatment landscape in many disciplines of human medicine. To continue this exciting trend, sustained identification of new, validated and preferably functional targets are needed. However, this is the precise bottleneck in today\\u2019s development of the next generation of therapeutic mAbs. Failures in translating a target into a successful therapeutic mAb are much more frequent than successes. Labex MAbImprove is a French-led consortium of academic laboratories jointly working on several aspects of the development of next-generation mAbs. The network organizes annual international meetings gathering academia and industry to discuss the different challenges faced in the therapeutic mAbs field. The 2018 symposium (also called AIS2018 and co-organized with MabDesign, the immunotherapy French industrial sector) focused on the discovery and validation of new targets for therapeutic mAbs. Key players from industry and academia outlined a number of exciting contributions, notably dealing with new innovations in the target discovery area, but also lessons learned from failures in the past. This report summarizes the talks presented at the AIS2018. We aim at broad dissemination of the most relevant, unpublished findings presented during the meeting, and hope to inspire all the contributors in this field to take new directions and bring about improvements.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1489&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f1400e4d57c61a9defcef6f4f4d839b7e9699038\",\n",
      "        \"title\": \"RNA-seq in Skeletal Biology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1490&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b0409a447337e4013e471968cbdf794aa8318b18\",\n",
      "        \"title\": \"Droplet-based single cell RNAseq tools: a practical guide.\",\n",
      "        \"abstract\": \"Droplet based scRNA-seq systems such as Drop-seq, inDrop and Chromium 10X have been the catalyst for the wide adoption of high-throughput scRNA-seq technologies in the research laboratory. In order to understand the capabilities of these systems to deeply interrogate biology; here we provide a practical guide through all the steps involved in a typical scRNA-seq experiment. Through comparing and contrasting these three main droplet based systems (and their derivatives), we provide an overview of all critical considerations in obtaining high quality and biologically relevant data. We also discuss the limitations of these systems and how they fit into the emerging field of Genomic Cytometry.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1491&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b183b05f8c7fe04c0c1c1b2247bc498af40e1dcb\",\n",
      "        \"title\": \"Single cell transcriptomics: moving towards multi-omics.\",\n",
      "        \"abstract\": \"As the basic units of life, cells present dramatic heterogeneity which, although crucial to an organism's behavior, is undetected by bulk analysis. Recently, much attention has been paid to reveal cellular types and states at the single-cell level including genome, transcriptome, epigenome or proteome-based on sequencing and immunological methods, etc. Among these approaches, transcriptomic analysis provides knowledge of the molecular linkages between genetic information and the proteome, leading to a comprehensive understanding of biological processes and diseases. Compared to single-dimensional inspection, multi-dimensional analysis combines the transcriptome with other \\\"omics\\\" to enable a comprehensive understanding of single-cell processes and functions. Moreover, compared to separate observations or single snapshots, spatial dimension and lineage time tracing can provide a more multifaceted understanding of the micro-environment and dynamic processes. In this review, we will introduce current transcriptomic analysis methods, as well as their combination with other omics methods including genomic, proteomic, and epigenetic approaches. Multi-dimensional analysis using these approaches for spatial positioning and lineage tracing applications will be reviewed. The future perspectives of single-cell multi-omics analysis based on the transcriptome will also be discussed.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1492&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8837ac6f6ce806086503fab677cea637a489423\",\n",
      "        \"title\": \"SISUA: Semi-Supervised Generative Autoencoder for Single Cell Data\",\n",
      "        \"abstract\": \"Single-cell transcriptomics offers a tool to study the diversity of cell phenotypes through snapshots of the abundance of mRNA in individual cells. Often there is additional information available besides the single cell gene expression counts, such as bulk transcriptome data from the same tissue, or quantification of surface protein levels from the same cells. In this study, we propose models based on the Bayesian generative approach, where protein quantification available as CITE-seq counts from the same cells are used to constrain the learning process, thus forming a semi-supervised model. The generative model is based on the deep variational autoencoder (VAE) neural network architecture.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1493&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d6556c5a4a4b7a61e80cfdd2b75c68e05b7a2acc\",\n",
      "        \"title\": \"Single-cell technologies \\u2014 studying rheumatic diseases one cell at a time\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1494&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a542cf8cc0fc47b199970c036a8bd202a3c3ccc\",\n",
      "        \"title\": \"Single cell transcriptomics reveals opioid usage evokes widespread suppression of antiviral gene program\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1495&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a2c8b611e64b8b4e5a8de686c1fe086b05653cd\",\n",
      "        \"title\": \"Single-cell analysis reveals fibroblast heterogeneity and myofibroblasts in systemic sclerosis-associated interstitial lung disease\",\n",
      "        \"abstract\": \"Objectives Myofibroblasts are key effector cells in the extracellular matrix remodelling of systemic sclerosis-associated interstitial lung disease (SSc-ILD); however, the diversity of fibroblast populations present in the healthy and SSc-ILD lung is unknown and has prevented the specific study of the myofibroblast transcriptome. We sought to identify and define the transcriptomes of myofibroblasts and other mesenchymal cell populations in human healthy and SSc-ILD lungs to understand how alterations in fibroblast phenotypes lead to SSc-ILD fibrosis. Methods We performed droplet-based, single-cell RNA-sequencing with integrated canonical correlation analysis of 13 explanted lung tissue specimens (56 196 cells) from four healthy control and four patients with SSc-ILD, with findings confirmed by cellular indexing of transcriptomes and epitopes by sequencing in additional samples. Results Examination of gene expression in mesenchymal cells identified two major, SPINT2hi and MFAP5hi, and one minor, WIF1hi, fibroblast populations in the healthy control lung. Combined analysis of control and SSc-ILD mesenchymal cells identified SPINT2hi, MFAP5hi, few WIF1hi fibroblasts and a new large myofibroblast population with evidence of actively proliferating myofibroblasts. We compared differential gene expression between all SSc-ILD and control mesenchymal cell populations, as well as among the fibroblast subpopulations, showing that myofibroblasts undergo the greatest phenotypic changes in SSc-ILD and strongly upregulate expression of collagens and other profibrotic genes. Conclusions Our results demonstrate previously unrecognised fibroblast heterogeneity in SSc-ILD and healthy lungs, and define multimodal transcriptome-phenotypes associated with these populations. Our data indicate that myofibroblast differentiation and proliferation are key pathological mechanisms driving fibrosis in SSc-ILD.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1496&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8a0a2c04cd66a3e4ec1aef38e48c11e44b0fb979\",\n",
      "        \"title\": \"Beyond Capture: Circulating Tumor Cell Release and Single\\u2010Cell Analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1497&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"12b9a2c7f37f06c435d834b940a648a07c0ef643\",\n",
      "        \"title\": \"Single-Cell Signature Explorer for comprehensive visualization of single cell signatures across scRNA-seq datasets\",\n",
      "        \"abstract\": \"The momentum of scRNA-seq data sets prompts for simple and powerful tools exploring their meaningful signatures. Here we present Single-Cell_Signature_Explorer (https://sites.google.com/site/fredsoftwares/products/single-cell-signature-explorer), the first method for high throughput scoring at single cell level of any gene set-based signature and visualization across t-SNE. By scanning data sets for single or combined signatures, it quantitatively and qualitatively maps any multi-gene feature, exemplified here with signatures of cell lineages, biological hallmarks and metabolic pathways in large scRNAseq datasets of human PBMC, lung cancer and adult testis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1498&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1972a19a5b3e552d65efc9f9d69ee07db27024ce\",\n",
      "        \"title\": \"OMIC Technologies and Vaccine Development: From the Identification of Vulnerable Individuals to the Formulation of Invulnerable Vaccines\",\n",
      "        \"abstract\": \"Routine vaccination is among the most effective clinical interventions to prevent diseases as it is estimated to save over 3 million lives every year. However, the full potential of global immunization programs is not realised because population coverage is still suboptimal. This is also due to the inadequate immune response and paucity of informative correlates of protection upon immunization of vulnerable individuals such as newborns, preterm infants, pregnant women, and elderly individuals as well as those patients affected by chronic and immune compromising medical conditions. In addition, these groups are undervaccinated for a number of reasons, including lack of awareness of vaccine-preventable diseases and uncertainty or misconceptions about the safety and efficacy of vaccination by parents and healthcare providers. The presence of these nonresponders/undervaccinated individuals represents a major health and economic burden to society, which will become particularly difficult to address in settings with limited public resources. This review describes innovative and experimental approaches that can help identify specific genomic profiles defining nonresponder individuals for whom specific interventions might be needed. We will provide examples that show how such information can be useful to identify novel biomarkers of safety and immunogenicity for future vaccine trials. Finally, we will discuss how system biology \\u201cOMICs\\u201d data can be used to design bioinformatic tools to predict the vaccination outcome providing genetic and molecular \\u201csignatures\\u201d of protective immune response. This strategy may soon enable identification of signatures highly predictive of vaccine safety, immunogenicity, and efficacy/protection thereby informing personalized vaccine interventions in vulnerable populations.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1499&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"28922fe5366d60a5a1bdc39a63b4877500c78d18\",\n",
      "        \"title\": \"What functional genomics has taught us about transcriptional regulation in malaria parasites\",\n",
      "        \"abstract\": \"Abstract Malaria parasites are characterized by a complex life cycle that is accompanied by dynamic gene expression patterns. The factors and mechanisms that regulate gene expression in these parasites have been searched for even before the advent of next generation sequencing technologies. Functional genomics approaches have substantially boosted this area of research and have yielded significant insights into the interplay between epigenetic, transcriptional and post-transcriptional mechanisms. Recently, considerable progress has been made in identifying sequence-specific transcription factors and DNA-encoded regulatory elements. Here, we review the insights obtained from these efforts including the characterization of core promoters, the involvement of sequence-specific transcription factors in life cycle progression and the mapping of gene regulatory elements. Furthermore, we discuss recent developments in the field of functional genomics and how they might contribute to further characterization of this complex gene regulatory network.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1500&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"171ad5db0b4e8b09aa8655c594d3d17216ca23a6\",\n",
      "        \"title\": \"Multiplexed detection of proteins, transcriptomes, clonotypes and CRISPR perturbations in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1501&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d4f7c7f23908f09d99ea4f349d0fc07d56ee4fa\",\n",
      "        \"title\": \"Exploring and analysing immune single cell multi-omics data with VDJView\",\n",
      "        \"abstract\": \"Background Single cell RNA sequencing provides unprecedented opportunity to simultaneously explore the transcriptomic and immune receptor diversity of T and B cells. However, there are limited tools available that simultaneously analyse large multi-omics datasets integrated with metadata such as patient and clinical information. Results We developed VDJView, which permits the simultaneous or independent analysis and visualisation of gene expression, immune receptors, and clinical metadata of both T and B cells. This tool is implemented as an easy-to-use R shiny web-application, which integrates numerous gene expression and TCR analysis tools, and accepts data from plate-based sorted or high-throughput single cell platforms. We utilised VDJView to analyse several 10X scRNA-seq datasets, including a recent dataset of 150,000 CD8+ T cells with available gene expression, TCR sequences, quantification of 15 surface proteins, and 44 antigen specificities (across viruses, cancer, and self-antigens). We performed quality control, filtering of tetramer non-specific cells, clustering, random sampling and hypothesis testing to discover antigen specific gene signatures which were associated with immune cell differentiation states and clonal expansion across the pathogen specific T cells. We also analysed 563 single cells (plate-based sorted) obtained from 11 subjects, revealing clonally expanded T and B cells across primary cancer tissues and metastatic lymph-node. These immune cells clustered with distinct gene signatures according to the breast cancer molecular subtype. VDJView has been tested in lab meetings and peer-to-peer discussions, showing effective data generation and discussion without the need to consult bioinformaticians. Conclusions VDJView enables researchers without profound bioinformatics skills to analyse immune scRNA-seq data, integrating and visualising this with clonality and metadata profiles, thus accelerating the process of hypothesis testing, data interpretation and discovery of cellular heterogeneity. VDJView is freely available at https://bitbucket.org/kirbyvisp/vdjview.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1502&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2da7e750eaf5d291ca353b409c2f5a0eaccf7128\",\n",
      "        \"title\": \"Quantifying Cancer Epithelial-Mesenchymal Plasticity and its Association with Stemness and Immune Response\",\n",
      "        \"abstract\": \"Cancer cells can acquire a spectrum of stable hybrid epithelial/mesenchymal (E/M) states during epithelial\\u2013mesenchymal transition (EMT). Cells in these hybrid E/M phenotypes often combine epithelial and mesenchymal features and tend to migrate collectively commonly as small clusters. Such collectively migrating cancer cells play a pivotal role in seeding metastases and their presence in cancer patients indicates an adverse prognostic factor. Moreover, cancer cells in hybrid E/M phenotypes tend to be more associated with stemness which endows them with tumor-initiation ability and therapy resistance. Most recently, cells undergoing EMT have been shown to promote immune suppression for better survival. A systematic understanding of the emergence of hybrid E/M phenotypes and the connection of EMT with stemness and immune suppression would contribute to more effective therapeutic strategies. In this review, we first discuss recent efforts combining theoretical and experimental approaches to elucidate mechanisms underlying EMT multi-stability (i.e., the existence of multiple stable phenotypes during EMT) and the properties of hybrid E/M phenotypes. Following we discuss non-cell-autonomous regulation of EMT by cell cooperation and extracellular matrix. Afterwards, we discuss various metrics that can be used to quantify EMT spectrum. We further describe possible mechanisms underlying the formation of clusters of circulating tumor cells. Last but not least, we summarize recent systems biology analysis of the role of EMT in the acquisition of stemness and immune suppression.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1503&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"541d743d53866b04e4a1a07bb04d57408ef92ee5\",\n",
      "        \"title\": \"Interleukin-12 elicits a non-canonical response in B16 melanoma cells to enhance survival\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1504&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d84ad40182fddcf4d07ada5b5baa6a9015ae22de\",\n",
      "        \"title\": \"Exploring dimension-reduced embeddings with Sleepwalk\",\n",
      "        \"abstract\": \"Dimension-reduction methods, such as t-SNE or UMAP, are widely used when exploring high-dimensional data describing many entities, e.g., RNA-seq data for many single cells. However, dimension reduction is commonly prone to introducing artefacts, and we hence need means to see where a dimension-reduced embedding is a faithful representation of the local neighbourhood and where it is not. We present Sleepwalk, a simple but powerful tool that allows the user to interactively explore an embedding, using colour to depict original or any other distances from all points to the cell under the mouse cursor. We show how this approach not only highlights distortions, but also reveals otherwise hidden characteristics of the data, and how Sleep-walk\\u2019s comparative modes help integrate multi-sample data and understand differences between embedding and preprocessing methods. Sleepwalk is a versatile and intuitive tool that unlocks the full power of dimension reduction and will be of value not only in single-cell RNA-seq but also in any other area with matrix-shaped big data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1505&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"666b8e4ad6df0488faac03f4231d3b19bd12c053\",\n",
      "        \"title\": \"Augmenting adaptive immunity: progress and challenges in the quantitative engineering and analysis of adaptive immune receptor repertoires\",\n",
      "        \"abstract\": \"The adaptive immune system is a natural diagnostic sensor and therapeutic.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1506&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c03244e20ee0e1b967a9adcdf440ba7ab0235524\",\n",
      "        \"title\": \"Optimal marker gene selection for cell type discrimination in single cell analyses\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1507&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f212e391b1182a4c81dba048ef8be779d9dca414\",\n",
      "        \"title\": \"The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1508&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"506b621f1d4e6ffc71619b285c4c921d4c1bb9d5\",\n",
      "        \"title\": \"CYTO Innovation: Inspiring and Supporting Innovators in ISAC\",\n",
      "        \"abstract\": \"PROGRESS in life sciences has always been driven by the development of new analytical tools and methods. Analytical cytometry has emerged as a linchpin enabling an understanding of the pathoetiology of diseases, the development of diagnostic methods, clinical monitoring methodologies, and supporting novel therapies. The International Society for Advancement of Cytometry (ISAC) has a diverse global community of members comprising: cytometry practitioners; biomedical scientists using cytometric technologies to solve specific problems; engineers and physical scientists developing future generations of cytometric technologies; data scientists who are developing cutting-edge algorithms to analyze and facilitate interpretation of complex multiparametric data\\u2014all contributing to the common mission of creating new knowledge using advanced cytometric techniques. This interplay among practitioners of different disciplines, enhancing the flow of knowledge, creativity, and innovation, has placed cytometry at the forefront of life sciences. Through its CYTO Innovation initiative, ISAC is inspiring, supporting, and promoting innovation, as an integral part of ISAC\\u2019s core mission. This program aims to support innovators in the society who contemplate conversion of their creative ideas into high impact commercial products, to help new entrepreneurs leverage inflection points to drive their success, and through initiatives to share creative solutions that do not yet address a large enough market to warrant commercialization. A detailed update of CYTO Innovation: An Initiative for ISAC\\u2019s Cytometry Product Innovators was published last year in this Journal by Diether Recktenwald (1). Innovation is about success. Innovation needs an environment that encourages inquiry and a forum for debate. Where aspiring current and future business leaders can navigate the rapidly changing world of new biotech discoveries. Where aspiring entrepreneurs can be alert to unmet needs and opportunities. In the words of business innovation guru Seth Kahan, \\u201cAs more advances roll out, there will be massive disruptions as industries struggle to build the infrastructure required to accommodate new technology.\\u201d (2) Innovators need access to a dynamic, agile, and diverse international community to be able to learn, share, collaborate, network, debate, and engage with one another and their potential customers. Through its CYTO Innovation program, ISAC provides such a forum for stimulating disruptive technologies and dissemination of authoritative knowledge related to traditional technologies. Now in its 7th year, CYTO Innovation continues to evolve adding significant value to both ISAC as an organization and to its membership\\u2014offering an environment for an open exchange of information, cross-fertilization of ideas, and stimulation of creative thinking. Its underlying goal is to increase the value to professionals working in academia and industry, particularly younger physical scientists and engineers enticing their direct interactions with biologists and clinicians/patients who become beneficiaries of cytometry innovations. The CYTO Innovation program consists of multilevel initiatives. Ongoing volunteer resources comprising cytometry experts and business professional are available through the ISAC website. It serves as a major pool of knowledge and feedback to entrepreneurs in need of professional expert opinion. Continuous education is encouraged and supported through online learning\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1509&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b4ebb13660377a989671cb175e2fca995ac951fe\",\n",
      "        \"title\": \"New insights into hematopoietic differentiation landscapes from single-cell RNA sequencing.\",\n",
      "        \"abstract\": \"Single-cell transcriptomics has recently emerged as a powerful tool to analyze cellular heterogeneity, discover new cell types, and infer putative differentiation routes. The technique has been rapidly embraced by the hematopoiesis research community, and like other technologies before, single-cell molecular profiling is widely expected to make important contributions to our understanding of the hematopoietic hierarchy. Much of this new interpretation relies on inference of the transcriptomic landscape as a representation of existing cellular states and associated transitions among them. Here we review how this model allows, under certain assumptions, charting of time-resolved differentiation trajectories with unparalleled resolution and how the landscape of multipotent cells may be rather devoid of discrete structures, challenging our preconceptions about stem and progenitor cell types and their organization. Finally, we highlight how promising technological advances may convert static differentiation landscapes into a dynamic cell flux model and thus provide a more holistic understanding of normal hematopoiesis and blood disorders.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1510&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd24616d328f8a36e70e2e750475eab5827fe194\",\n",
      "        \"title\": \"Dissecting CLL through high-dimensional single-cell technologies.\",\n",
      "        \"abstract\": \"We now have the potential to undertake detailed analysis of the inner workings of thousands of cancer cells, one cell at a time, through the emergence of a range of techniques that probe the genome, transcriptome, and proteome combined with the development of bioinformatics pipelines that enable their interpretation. This provides an unprecedented opportunity to better understand the heterogeneity of chronic lymphocytic leukemia and how mutations, activation states, and protein expression at the single-cell level have an impact on disease course, response to treatment, and outcomes. Herein, we review the emerging application of these new techniques to chronic lymphocytic leukemia and examine the insights already attained through this transformative technology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1511&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b8ce412d8818a1eae73933fc5f6daf4925d4ac72\",\n",
      "        \"title\": \"Orchestrating single-cell analysis with Bioconductor\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1512&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"59f3161e1cf69830d92021ba50bc673aa5a00a43\",\n",
      "        \"title\": \"Direct genome editing of patient-derived xenografts using CRISPR-Cas9 enables rapid in vivo functional genomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1513&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3a1970d88db4b5813db26c3eae4640e33e3d7b44\",\n",
      "        \"title\": \"Engineered microscale hydrogels for drug delivery, cell therapy, and sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1514&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7c06f105a4b2a94b72700d53fefb86f28aa6e2d1\",\n",
      "        \"title\": \"Functional genomics: paving the way for more successful cancer immunotherapy\",\n",
      "        \"abstract\": \"Immunotherapies have revolutionized cancer treatment. Immunotherapy is effective for the treatment of a wide range of cancer types and can mediate complete and durable tumor regression. Nonetheless, the field still faces many significant challenges, such as the need for personalized therapeutic strategies and better biomarkers, the difficulty of selecting the right combination therapy, and resistance to currently available immunotherapies. Both cancer and host immunity comprise significantly diverse and complex ecosystems, making immunogenomics an ideal field for functional genomics analysis. In this review, we describe the cancer-immunity cycle, how cancer cells manage to evade immune attack and the current hurdles in the path of cancer immunotherapy. Then, we discuss how functional genomics approaches can pave the way for more successful cancer immunotherapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1515&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f248cd23d15b81505553dd1d48244cb0f32d83c4\",\n",
      "        \"title\": \"Scalable analysis of cell-type composition from single-cell transcriptomics using deep recurrent learning\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1516&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a0df505a3ff944639b6f2baf421d0b40e1e0cc2f\",\n",
      "        \"title\": \"Engineered T Cell Therapies from a Drug Development Viewpoint\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1517&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6ecf1dfa2688d949a309e2cc336ddd929a69b12b\",\n",
      "        \"title\": \"Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1518&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"56f04f9c0d12974b84de5fbd300e6da5187e9b6e\",\n",
      "        \"title\": \"Parallel analysis of tri-molecular biosynthesis with cell identity and function in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1519&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bd6833fe0e3090e949f4e532a7bfdbd943eeb42f\",\n",
      "        \"title\": \"Systems immunology: Integrating multi-omics data to infer regulatory networks and hidden drivers of immunity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1520&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"96c08a77ed4ba2b708fc47003e61cd1edf26dd22\",\n",
      "        \"title\": \"Genotype-free demultiplexing of pooled single-cell RNA-seq\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1521&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4ad204a79e98bdd09e578c15c06657a974b8a60\",\n",
      "        \"title\": \"The inflammasome of circulatory collapse: single cell analysis of survival on extracorporeal life support\",\n",
      "        \"abstract\": \"Background Despite being a lifesaving intervention for the most critically ill and circulatory compromised patients, veno-arterial extra-corporeal life support (VA-ECLS) is associated with a mortality rate of nearly 60%. Understanding how the immune response to VA-ECLS either promotes or impedes survival would both enhance risk stratification and uncover new therapeutic strategies for these patients. However, conventional enumeration of peripheral blood mononuclear cells (PBMCs) and their subsets have failed to identify determinants of outcome among these cells. Methods Flow cytometry and plasma cytokine measurement was combined with single cell RNASeq analysis of PBMCs from patients in circulatory shock being started on VA-ECLS to identify clinical, laboratory, and cellular features associated with 72 hour survival. Results Non-surviving patients exhibited higher plasma levels of the tissue aggressive inflammatory cytokines IL-1, IL-6, IL-12 and TNF-\\u03b1. Distribution of cells between conventional PBMC subtypes was not predictive of survival. Single cell RNASeq analysis of discriminatory markers within each PBMC subtype revealed that the proportion of CD8+ Natural Killer T-cells (NKT) that expressed CD52, a known immune-modulator, was associated with improved survival. This cell population correlated inversely with IL-6 production. CD8+/CD52+ NKT cells were quantified by flow cytometry in a second, validation cohort. Those patients with a high proportion of CD52+ cells among all CD8+ NKT cells had more severe disease relative to the low CD52+ group, but nevertheless were nearly 5 time less likely to die in the first 72 hours of VA-ECLS (p=0.043 by log rank test). Conclusions CD8+/CD52+ NKT cells are associated with survival in patients undergoing VA-ECLS. Fluidics based scRNASeq can reveal important aspects of pathophysiology in complex disease states such as circulatory collapse and VA-ECLS. Further studies in animal models will be required to determine if stimulation of CD8+/CD52+ NKT cell expansion may be an effective therapeutic strategy in this patient population.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1522&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d78b7feb90125fc8013f1ee6e9ce2407fd1a243e\",\n",
      "        \"title\": \"SuperCT: a supervised-learning framework for enhanced characterization of single-cell transcriptomic profiles\",\n",
      "        \"abstract\": \"Abstract Characterization of individual cell types is fundamental to the study of multicellular samples. Single-cell RNAseq techniques, which allow high-throughput expression profiling of individual cells, have significantly advanced our ability of this task. Currently, most of the scRNA-seq data analyses are commenced with unsupervised clustering. Clusters are often assigned to different cell types based on the enriched canonical markers. However, this process is inefficient and arbitrary. In this study, we present a technical framework of training the expandable supervised-classifier in order to reveal the single-cell identities as soon as the single-cell expression profile is input. Using multiple scRNA-seq datasets we demonstrate the superior accuracy, robustness, compatibility and expandability of this new solution compared to the traditional methods. We use two examples of the model upgrade to demonstrate how the projected evolution of the cell-type classifier is realized.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1523&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8ab4c74923be50239a0ff94baca4233220d374f3\",\n",
      "        \"title\": \"Revolutionizing immunology with single-cell RNA sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1524&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5a2d29791aadd313b3485bfced8e13c3dafb8fd6\",\n",
      "        \"title\": \"CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing\",\n",
      "        \"abstract\": \"Cell type identification is essential for single-cell RNA sequencing (scRNA-seq) studies that are currently transforming the life sciences. CHETAH (CHaracterization of cEll Types Aided by Hierarchical clustering) is an accurate cell type identification algorithm that is rapid and selective, including the possibility of intermediate or unassigned categories. Evidence for assignment is based on a classification tree of previously available scRNA-seq reference data and includes a confidence score based on the variance in gene expression per cell type. For cell types represented in the reference data, CHETAH\\u2019s accuracy is as good as existing methods. Its specificity is superior when cells of an unknown type are encountered, such as malignant cells in tumor samples which it pinpoints as intermediate or unassigned. Although designed for tumor samples in particular, the use of unassigned and intermediate types is also valuable in other exploratory studies. This is exemplified in pancreas datasets where CHETAH highlights cell populations not well represented in the reference dataset, including cells with profiles that lie on a continuum between that of acinar and ductal cell types. Having the possibility of unassigned and intermediate cell types is pivotal for preventing misclassification and can yield important biological information for previously unexplored tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1525&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3609f3009b5c99c0504a7dc0a4fa98f8dd568aee\",\n",
      "        \"title\": \"Single\\u2010cell immune profiling reveals new insights into colorectal cancer\",\n",
      "        \"abstract\": \"Over the past few years, work in both the laboratory and the clinic has shown that harnessing and manipulating the immune system is a powerful approach to treating a variety of cancers. In a newly published study, Zhang et al. demonstrate a new integrated analytical approach that uses single-cell sequencing methods to assess multiple aspects of the immune response to colorectal cancer. The authors were able to understand the fates of tumor-infiltrating T cells and to map their relationships at the level of individual clonal groups. This presents a framework within which to consider T-cell responses to tumors and to understand how these responses can illuminate differential responses to treatment. We have seen many compelling demonstrations of the therapeutic potential of immuno-oncology. However, we do not fully understand the relevant interactions between immune cells and tumors; our success in this area will continue to be improved by increasing our knowledge of the basic mechanisms underlying the cancer immune response. One of the remaining burning questions is why treatments that release the immune \\u201cbrake\\u201d by inhibiting the PD-1 immune checkpoint are highly efficient in some cases, but fail in others. Such variable responses to treatment are also seen in colorectal cancer, with microsatellite-instable (MSI) forms of the disease generally responding to checkpoint inhibition, while the microsatellite stable (MSS) forms of the same cancer do not. When considering the T-cell response to cancers, many important facets of the cells\\u2019 behavior can be studied by measuring gene expression at the single-cell level. While the gene expression profiles first and foremost reflect cellular identities, they can also be used for much more detailed deduction of cellular regulation and function. Indeed, this study achieved a strikingly fine-grained dissection of CD8 and CD4 T-cell subtypes by single-cell RNA-sequencing (scRNAseq). Zhang et al. were able to identify naive T cells alongside multiple memory and effector states, and regulatory T cells. Among these were \\u201cexhausted\\u201d cells, an unresponsive state resulting from chronic stimulation that often develops during a T-cell response against cancer. Furthermore, they were able to identify the specific transcriptional signatures of tissue residency in cells that had left the blood and adjusted to life in the mucosa. T cells also inherently provide us with an exciting opportunity to understand their fine-grained population structure by studying their T-cell receptors (TCRs). These heterodimeric proteins bind to antigenic peptides (presented as complexes with major histocompatibility molecules) and are encoded by genes generated by the process of V(D)J recombination during T-cell development. Gene segments are randomly chosen from a pool such that the DNA is \\u201ccut-andpasted\\u201d together; this combinatorial diversity is combined with random editing of nucleotides at the junctions between segments to permit an enormous space of possible TCR sequences and antigen specificities. The vast combinatorial diversity of TCR sequences means that we can make the assumption that two cells with exactly the same set of recombinant sequences at both of their TCR loci almost certainly arose from the same original development event in the thymus. This allows us to use TCR sequences as cellular lineage barcodes, providing an additional layer of information typically missing from scRNAseq data. By studying the Correspondence Tapio L\\u20ac onnberg, Turku Centre for Biotechnology, University of Turku and Abo Akademi University, Tykist\\u20ac okatu 6, FI-20520 Turku, Finland. E-mail: taplon@utu.fi and Michael JT Stubbington, 10x Genomics Inc., 6230 Stoneridge Mall Road, Pleasanton, CA 94588, USA. E-mail: mike.stubbington@10xgenomics.com\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1526&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe38d95eef1b0d7dc57cc9abd19cedb651207319\",\n",
      "        \"title\": \"Dissecting Cellular Heterogeneity Using Single-Cell RNA Sequencing\",\n",
      "        \"abstract\": \"Cell-to-cell variability in gene expression exists even in a homogeneous population of cells. Dissecting such cellular heterogeneity within a biological system is a prerequisite for understanding how a biological system is developed, homeo-statically regulated, and responds to external perturbations. Single-cell RNA sequencing (scRNA-seq) allows the quantitative and unbiased characterization of cellular heterogeneity by providing genome-wide molecular profiles from tens of thousands of individual cells. A major question in analyzing scRNA-seq data is how to account for the observed cell-to-cell variability. In this review, we provide an overview of scRNA-seq protocols, computational approaches for dissecting cellular heterogeneity, and future directions of single-cell transcriptomic analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1527&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5d1c57659b845c99383291cee4a7a45d34e8759a\",\n",
      "        \"title\": \"The role of systems biology approaches in determining molecular signatures for the development of more effective vaccines\",\n",
      "        \"abstract\": \"ABSTRACT Introduction: Emerging infectious diseases are a major threat to public health, and while vaccines have proven to be one of the most effective preventive measures for infectious diseases, we still do not have safe and effective vaccines against many human pathogens, and emerging diseases continually pose new threats. The purpose of this review is to discuss how the creation of vaccines for these new threats has been hindered by limitations in the current approach to vaccine development. Recent advances in high-throughput technologies have enabled scientists to apply systems biology approaches to collect and integrate increasingly large datasets that capture comprehensive biological changes induced by vaccines, and then decipher the complex immune response to those vaccines. Areas covered: This review covers advances in these technologies and recent publications that describe systems biology approaches to understanding vaccine immune responses and to understanding the rational design of new vaccine candidates. Expert opinion: Systems biology approaches to vaccine development provide novel information regarding both the immune response and the underlying mechanisms and can inform vaccine development.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1528&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"02bb0069d6c02d0224bdf2464c9feb2fd4a29c84\",\n",
      "        \"title\": \"Lymphocyte innateness defined by transcriptional states reflects a balance between proliferation and effector functions\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1529&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"aff4a5299ff72047affcc5cecc07ef6b36983a50\",\n",
      "        \"title\": \"Multiplexed profiling of RNA and protein expression signatures in individual cells using flow or mass cytometry\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1530&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c7b90df43f1a87c167d1a7cbae3cc8b98a24689b\",\n",
      "        \"title\": \"Data mining for mutation-specific targets in acute myeloid leukemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1531&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5159d51a4e27a85dd311029d082c27e16435fe3b\",\n",
      "        \"title\": \"Comparative analysis of commercially available single-cell RNA sequencing platforms for their performance in complex human tissues\",\n",
      "        \"abstract\": \"The past five years have witnessed a tremendous growth of single-cell RNA-seq methodologies. Currently, there are three major commercial platforms for single-cell RNA-seq: Fluidigm C1, Clontech iCell8 (formerly Wafergen) and 10x Genomics Chromium. Here, we provide a systematic comparison of the throughput, sensitivity, cost and other performance statistics for these three platforms using single cells from primary human islets. The primary human islets represent a complex biological system where multiple cell types coexist, with varying cellular abundance, diverse transcriptomic profiles and differing total RNA contents. We apply standard pipelines optimized for each system to derive gene expression matrices. We further evaluate the performance of each system by benchmarking single-cell data with bulk RNA-seq data from sorted cell fractions. Our analyses can be generalized to a variety of complex biological systems and serve as a guide to newcomers to the field of single-cell RNA-seq when selecting platforms.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1532&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e0164f6c05a48da93d6f7019a24d58305770706c\",\n",
      "        \"title\": \"Self-Reporting Transposons Enable Simultaneous\\u00a0Readout of Gene Expression and Transcription\\u00a0Factor Binding in Single Cells\",\n",
      "        \"abstract\": \"In situ measurements of transcription factor (TF) binding are confounded by cellular heterogeneity and represent averaged profiles in complex tissues. Single cell RNA-seq (scRNA-seq) is capable of resolving different cell types based on gene expression profiles, but no technology exists to directly link specific cell types to the binding pattern of TFs in those cell types. Here, we present self-reporting transposons (SRTs) and their use in single cell calling cards (scCC), a novel assay for simultaneously capturing gene expression profiles and mapping TF binding sites in single cells. First, we show how the genomic locations of SRTs can be recovered from mRNA. Next, we demonstrate that SRTs deposited by the piggyBac transposase can be used to map the genome-wide localization of the TFs SP1, through a direct fusion of the two proteins, and BRD4, through its native affinity for piggyBac. We then present the scCC method, which maps SRTs from scRNA-seq libraries, thus enabling concomitant identification of cell types and TF binding sites in those same cells. As a proof-of-concept, we show recovery of cell type-specific BRD4 and SP1 binding sites from cultured cells. Finally, we map Brd4 binding sites in the mouse cortex at single cell resolution, thus establishing a new technique for studying TF biology in situ.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1533&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a810b2ccd8a1e891e9b23c748409871d9bbed7e6\",\n",
      "        \"title\": \"Exploiting Molecular Barcodes in High-Throughput Cellular Assays\",\n",
      "        \"abstract\": \"Multiplexing strategies, which greatly increase the number of simultaneously measured parameters in single experiments, are now being widely implemented by both the pharmaceutical industry and academic researchers. Color has long been used to identify biological signals and, when combined with molecular barcodes, has substantially enhanced the depth of multiplexed sample characterization. Moreover, the recent advent of DNA barcodes has led to an explosion of innovative cell sequencing approaches. Novel barcoding strategies also show great promise for encoding spatial information in transcriptomic studies, and for precise assessment of molecular abundance. Both color- and DNA-based barcodes can be conveniently analyzed with either a microscope or a cytometer, or via DNA sequencing. Here we review the basic principles of several technologies used to create barcodes and detail the type of samples that can be identified with such tags.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1534&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"577aad100ab2a39dbaa7b08841cee9edb16bdd0e\",\n",
      "        \"title\": \"Revolutionizing Cancer Immunology: The Power of Next-Generation Sequencing Technologies\",\n",
      "        \"abstract\": \"It has long been appreciated that tumors are diverse, varying in mutational status, composition of cellular infiltrate, and organizational architecture. For the most part, the information embedded in this diversity has gone untapped due to the limited resolution and dimensionality of assays for analyzing nucleic acid expression in cells. The advent of high-throughput, next-generation sequencing (NGS) technologies that measure nucleic acids, particularly at the single-cell level, is fueling the characterization of the many components that comprise the tumor microenvironment (TME), with a strong focus on immune composition. Understanding the immune and nonimmune components of the TME, how they interact, and how this shapes their functional properties requires the development of novel computational methods and, eventually, the application of systems-based approaches. The continued development and application of NGS technologies holds great promise for accelerating discovery in the cancer immunology field.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1535&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4542baa028362f36aa8ae42574029215fdd05966\",\n",
      "        \"title\": \"Microfluidics as an Emerging Precision Tool in Developmental Biology.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1536&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58c22899af553a611bbd3acfd8d2b43f86d2b3fe\",\n",
      "        \"title\": \"Probabilistic harmonization and annotation of single\\u2010cell transcriptomics data with deep generative models\",\n",
      "        \"abstract\": \"As single-cell transcriptomics becomes a mainstream technology, the natural next step is to integrate the accumulating data in order to achieve a common ontology of cell types and states. However, owing to various nuisance factors of variation, it is not straightforward how to compare gene expression levels across data sets and how to automatically assign cell type labels in a new data set based on existing annotations. In this manuscript, we demonstrate that our previously developed method, scVI, provides an effective and fully probabilistic approach for joint representation and analysis of cohorts of single-cell RNA-seq data sets, while accounting for uncertainty caused by biological and measurement noise. We also introduce single-cell ANnotation using Variational Inference (scANVI), a semi-supervised variant of scVI designed to leverage any available cell state annotations \\u2014 for instance when only one data set in a cohort is annotated, or when only a few cells in a single data set can be labeled using marker genes. We demonstrate that scVI and scANVI compare favorably to the existing methods for data integration and cell state annotation in terms of accuracy, scalability, and adaptability to challenging settings such as a hierarchical structure of cell state labels. We further show that different from existing methods, scVI and scANVI represent the integrated datasets with a single generative model that can be directly used for any probabilistic decision making task, using differential expression as our case study. scVI and scANVI are available as open source software and can be readily used to facilitate cell state annotation and help ensure consistency and reproducibility across studies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1537&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a12e9b29da9f3dcaaf02c0e5c8d69aee94c7112\",\n",
      "        \"title\": \"Integrative single-cell analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1538&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"39b61a39dbc508e3f746b16b9295b6f5b6c3bc05\",\n",
      "        \"title\": \"Advancing single-cell proteomics and metabolomics with microfluidic technologies.\",\n",
      "        \"abstract\": \"Recent advances in single-cell analysis have unraveled substantial heterogeneity among seemingly identical cells at genomic and transcriptomic levels. These discoveries have urged scientists to develop new tools that are capable of investigating single cells from a broader set of \\\"omics\\\". Proteomics and metabolomics, for instance, are of particular interest as they are closely correlated with a dynamic picture of cellular behaviors and phenotypic identities. The development of such tools requires highly efficient isolation and processing of a large number of individual cells, where techniques such as microfluidics are extremely useful. Here, we review the recent advances in single-cell proteomics and metabolomics, with a focus on microfluidics-based platforms. We highlight a vast array of emerging microfluidic formats for single-cell isolation and manipulation, and how the state-of-the-art analytical tools are coupled with such platforms for proteomic and metabolomic profiling.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1539&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe09af18531de36a3f2649db70664b221b7ba931\",\n",
      "        \"title\": \"Charting the single-cell transcriptional landscape of haematopoiesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1540&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"bbeaf63ef2c4ee6439d82d0aa311d01e12c19077\",\n",
      "        \"title\": \"Single-Cell Applications of Next-Generation Sequencing.\",\n",
      "        \"abstract\": \"The single cell is considered the basic unit of biology, and the pursuit of understanding how heterogeneous populations of cells can functionally coexist in tissues, organisms, microbial ecosystems, and even cancer, makes them the subject of intense study. Next-generation sequencing (NGS) of RNA and DNA has opened a new frontier of (single)-cell biology. Hundreds to millions of cells now can be assayed in parallel, providing the molecular profile of each cell in its milieu inexpensively and in a manner that can be analyzed mathematically. The goal of this article is to provide a high-level overview of single-cell sequencing for the nonexpert and show how its applications are influencing both basic and applied clinical studies in embryology, developmental genetics, and cancer.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1541&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3912fd6f60185599768133b650a75192a6f827bf\",\n",
      "        \"title\": \"Beyond the message: advantages of snapshot proteomics with single\\u2010cell mass cytometry in solid tumors\",\n",
      "        \"abstract\": \"Single\\u2010cell technologies that can quantify features of individual cells within a tumor are critical for treatment strategies aiming to target cancer cells while sparing or activating beneficial cells. Given that key players in protein networks are often the primary targets of precision oncology strategies, it is imperative to transcend the nucleic acid message and read cellular actions in human solid tumors. Here, we review the advantages of multiplex, single\\u2010cell mass cytometry in tissue and solid tumor investigations. Mass cytometry can quantitatively probe nearly any cellular feature or target. In discussing the ability of mass cytometry to reveal and characterize a broad spectrum of cell types, identify rare cells, and study functional behavior through protein signaling networks in millions of individual cells from a tumor, this review surveys publications of scientific advances in solid tumor biology made with the aid of mass cytometry. Advances discussed include functional identification of rare tumor and tumor\\u2010infiltrating immune cells and dissection of cellular mechanisms of immunotherapy in solid tumors and the periphery. The review concludes by highlighting ways to incorporate single\\u2010cell mass cytometry in solid tumor precision oncology efforts and rapidly developing cytometry techniques for quantifying cell location and sequenced nucleic acids.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1542&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"afbd9bec9c5ec46378d9077e236dbf31b773bd2b\",\n",
      "        \"title\": \"Kidney and organoid single-cell transcriptomics: the end of the beginning\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1543&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e9e774330f3255e733c608380c92f69a7173cbdf\",\n",
      "        \"title\": \"Current perspectives in assessing humoral immunity after measles vaccination\",\n",
      "        \"abstract\": \"ABSTRACT Introduction: Repeated measles outbreaks in countries with relatively high vaccine coverage are mainly due to failure to vaccinate and importation; however, cases in immunized individuals exist raising questions about suboptimal measles vaccine-induced humoral immunity and/or waning immunity in a low measles-exposure environment. Areas covered: The plaque reduction neutralization measurement of functional measles-specific antibodies correlates with protection is the gold standard in measles serology, but it does not assess cellular-immune or other parameters that may be associated with durable and/or protective immunity after vaccination. Additional correlates of protection and long-term immunity and new determinants/signatures of vaccine responsiveness such as specific CD46 and IFI44L genetic variants associated with neutralizing antibody titers after measles vaccination are under investigation. Current and future systems biology studies, coupled with new technology/assays and analytical approaches, will lead to an increasingly sophisticated understanding of measles vaccine-induced humoral immunity and will identify \\u2018signatures\\u2019 of protective and durable immune responses. Expert opinion: This will translate into the development of highly predictive assays of measles vaccine efficacy, effectiveness, and durability for prospective identification of potential low/non-responders and susceptible individuals who require additional vaccine doses. Such new advances may drive insights into the development of new/improved vaccine formulations and delivery systems.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1544&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1a61530160018e2b6d44d11ff93614394ec42410\",\n",
      "        \"title\": \"Highly multiplexed in situ protein imaging with signal amplification by Immuno-SABER\",\n",
      "        \"abstract\": \"Probing the molecular organization of tissues requires in situ analysis by microscopy. However current limitations in multiplexing, sensitivity, and throughput collectively constitute a major barrier for comprehensive single-cell profiling of proteins. Here, we report Immunostaining with Signal Amplification By Exchange Reaction (Immuno-SABER), a rapid, highly multiplexed signal amplification method that simultaneously tackles these key challenges. Immuno-SABER utilizes DNA-barcoded antibodies and provides a method for highly multiplexed signal amplification via modular orthogonal DNA concatemers generated by Primer Exchange Reaction. This approach offers the capability to preprogram and control the amplification level independently for multiple targets without in situ enzymatic reactions, and the intrinsic scalability to rapidly amplify and image a large number of protein targets. We validated our approach in diverse sample types including cultured cells, cryosections, FFPE sections, and whole mount tissues. We demonstrated independently tunable 5-180-fold amplification for multiple targets, covering the full signal range conventionally achieved by secondary antibodies to tyramide signal amplification, as well as simultaneous signal amplification for 10 different proteins using standard equipment and workflow. We further combined Immuno-SABER with Expansion Microscopy to enable rapid and highly multiplexed super-resolution tissue imaging. Overall, Immuno-SABER presents an effective and accessible platform for rapid, multiplexed imaging of proteins across scales with high sensitivity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1545&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e5bd365f23c8f2c637b83ce64076f2628e62763b\",\n",
      "        \"title\": \"Cell Type Purification by Single-Cell Transcriptome-Trained Sorting\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1546&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0579926d3adabc8eee494e3a62617741d8f08ab8\",\n",
      "        \"title\": \"Human Skin Equivalents Demonstrate Need for Neuro\\u2010Immuno\\u2010Cutaneous System\",\n",
      "        \"abstract\": \"A variety of human skin equivalents (HSEs) has been designed for clinical use or for exploratory skin research. In vitro HSE models have been used to target relationships between the skin and nervous or immune systems but have not yet considered the neuro\\u2010immuno\\u2010cutaneous (NIC) system. In this study, HSEs are described, with and without neural and immune components, to discern these types of effects. These systems are composed of only primary human cells and contain an epidermis, dermis, hypodermis (with immune cells), and human induced neural stem cells for the neuronal component. RNA sequencing is utilized to confirm differences between sample groups and to identify unique or important genes with respect to sample type. Only samples with both neural and immune components result in the upregulation of genes in all the key biological pathways explored. The analysis of protein secretion confirms that this group has measurable functions related to all key cell types. Overall, this novel skin tissue system confirms that designing HSEs that include the NIC system results in a tissue model that reflects key functions. These systems could be used to identify selected targets of interest in skin research related to healthy or diseased states.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1547&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e4e04b1301a3d7b7c1424c1d9c57f1021b49ca63\",\n",
      "        \"title\": \"A chromatin integration labelling method enables epigenomic profiling with lower input\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1548&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"86977d37097db206523c72e4a71ff959507e12a8\",\n",
      "        \"title\": \"Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1549&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4163215a4343f0cd0236524e07b687edef1f24e8\",\n",
      "        \"title\": \"Understanding tumor ecosystems by single-cell sequencing: promises and limitations\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1550&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1334dee67cad8f427b2df862c2ffbb1e9f067840\",\n",
      "        \"title\": \"An integrative approach for building personalized gene regulatory networks for precision medicine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1551&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6688ab095a3a56de10cb31c6b1107dd68aaef13c\",\n",
      "        \"title\": \"Single-Cell Sequencing in Cancer: Recent Applications to Immunogenomics and Multi-omics Tools\",\n",
      "        \"abstract\": \"Tumor heterogeneity, the cellular mosaic of multiple lineages arising from the process of clonal evolution, has continued to thwart multi-omics analyses using traditional bulk sequencing methods. The application of single-cell sequencing, in concert with existing genomics methods, has enabled high-resolution interrogation of the genome, transcriptome, epigenome, and proteome. Applied to cancers, these single-cell multi-omics methods bypass previous limitations on data resolution and have enabled a more nuanced understanding of the evolutionary dynamics of tumor progression, immune evasion, metastasis, and treatment resistance. This review details the growing number of novel single-cell multi-omics methods applied to tumors and further discusses recent discoveries emerging from these approaches, especially in regard to immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1552&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e046e9de7765ab79027c87a60dabdd65f5b8b47e\",\n",
      "        \"title\": \"The biology of the cell \\u2013 insights from mass cytometry\",\n",
      "        \"abstract\": \"In single\\u2010cell biology there is an ongoing push toward more and more comprehensive measurements made in individual cells. By combining measurements of proteins, mRNA, genetic, and epigenetic traits, in contexts of cell stimulation or drug treatment, novel single\\u2010cell analyses will help understand the determinants of cellular identity, differentiation decisions, and regulatory mechanisms governing cellular function in health and disease. Mass cytometry is a flexible methodology not only developed toward this aim, allowing analyses of cells in suspension such as blood but also extended for the analyses of tissue sections. Here, I will discuss the different measurements possible by mass cytometry and not only discuss its potential but also its limitations now and in the years to come.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1553&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"deea74c59b21622d228c43adea5340e8a2447081\",\n",
      "        \"title\": \"Systems vaccinology and big data in the vaccine development chain\",\n",
      "        \"abstract\": \"Systems vaccinology has proven a fascinating development in the last decade. Where traditionally vaccine development has been dominated by trial and error, systems vaccinology is a tool that provides novel and comprehensive understanding if properly used. Data sets retrieved from systems\\u2010based studies endorse rational design and effective development of safe and efficacious vaccines. In this review we first describe different omics\\u2010techniques that form the pillars of systems vaccinology. In the second part, the application of systems vaccinology in the different stages of vaccine development is described. Overall, this review shows that systems vaccinology has become an important tool anywhere in the vaccine development chain.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1554&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"69508db17c4e95c91011198fade99e6c2ac5ff96\",\n",
      "        \"title\": \"Recent Advances in Microfluidic Techniques for Systems Biology.\",\n",
      "        \"abstract\": \"Systems biology seeks to understand complex biological systems using a holistic approach instead of the reductionist approach, which is to study each part of the system individually.1 By examining the differences, interaction, and dynamics between cellular or molecular components, systems biology researchers aim to decipher the complexity of biological systems and to understand the functional networks that form living organisms. Such an understanding has implications in practical applications in many fields, including cancer biology, immunology, microbiology, neuroscience, precision medicine, therapeutic management, and many more.2-3 Thanks to the advances in analytical techniques and high-throughput experimentations, the field of system biology has undergone rapid growth in recent years. With enhanced sensitivity and specificity, a wide spectrum of single-cell and single-molecule analysis techniques, including optical, electrochemical, spectrometric, and sequencing-based methods, enable us to understand the structures and functions of individual biological building blocks.4-5 In order to generate sufficient amount of data to analyze the complexity of biological systems, one needs to integrate analysis techniques with rapid sample handling and perform high-throughput experimentations to collect data, while preserving individual heterogeneities. For the study of certain living biological systems, such as living cells, tissues, or whole organisms, it is often useful to characterize at multiple time-points to probe dynamics of the system in response to perturbations, all in a controlled culturing environment.2, 6.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1555&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c2075db536fa71f2198329adae2525a5c8aa1add\",\n",
      "        \"title\": \"Expanding the CITE-seq tool-kit: Detection of proteins, transcriptomes, clonotypes and CRISPR perturbations with multiplexing, in a single assay\",\n",
      "        \"abstract\": \"Rapid technological progress in the recent years has allowed the high-throughput interrogation of different types of biomolecules from single cells. Combining several of these readouts into integrated multi-omic assays is essential to comprehensively understand and model cellular processes. Here, we report the development of Expanded CRISPR-compatible Cellular Indexing of Transcriptomes and Epitopes by sequencing (ECCITE-seq) for the high-throughput characterization of at least five modalities of information from each single cell: transcriptome, immune receptor clonotypes, surface markers, sample identity and sgRNAs. We demonstrate the use of ECCITE-seq to directly and efficiently capture sgRNA molecules and measure their effects on gene expression and protein levels, opening the possibility of performing high throughput single cell CRISPR screens with multimodal readout using existing libraries and commonly used vectors. Finally, by utilizing the combined phenotyping of clonotype and cell surface markers in immune cells, we apply ECCITE to study a lymphoma sample to discriminate cells and define molecular signatures of malignant cells within a heterogeneous population.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1556&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"47b7dc4e3ba01b5ec21757c037afa439d28e8846\",\n",
      "        \"title\": \"A deadly dance: the choreography of host\\u2013pathogen interactions, as revealed by single-cell technologies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1557&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2287a3930a7568a956aae5f3f037efe8fed675e7\",\n",
      "        \"title\": \"Comprehensive Integration of Single-Cell Data\",\n",
      "        \"abstract\": \"Single cell transcriptomics (scRNA-seq) has transformed our ability to discover and annotate cell types and states, but deep biological understanding requires more than a taxonomic listing of clusters. As new methods arise to measure distinct cellular modalities, including high-dimensional immunophenotypes, chromatin accessibility, and spatial positioning, a key analytical challenge is to integrate these datasets into a harmonized atlas that can be used to better understand cellular identity and function. Here, we develop a computational strategy to \\u201canchor\\u201d diverse datasets together, enabling us to integrate and compare single cell measurements not only across scRNA-seq technologies, but different modalities as well. After demonstrating substantial improvement over existing methods for data integration, we anchor scRNA-seq experiments with scATAC-seq datasets to explore chromatin differences in closely related interneuron subsets, and project single cell protein measurements onto a human bone marrow atlas to annotate and characterize lymphocyte populations. Lastly, we demonstrate how anchoring can harmonize in-situ gene expression and scRNA-seq datasets, allowing for the transcriptome-wide imputation of spatial gene expression patterns, and the identification of spatial relationships between mapped cell types in the visual cortex. Our work presents a strategy for comprehensive integration of single cell data, including the assembly of harmonized references, and the transfer of information across datasets. Availability: Installation instructions, documentation, and tutorials are available at: https://www.satijalab.org/seurat\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1558&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"14659d99861f857affce9f75dc3c25f0603be557\",\n",
      "        \"title\": \"Protein Barcodes Enable High-Dimensional Single-Cell CRISPR Screens\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1559&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ed5d6be2eb8fc1d42f9a0e3c50abaf5439bbd8dd\",\n",
      "        \"title\": \"Deep Generative Modeling for Single-cell Transcriptomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1560&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6e12a2745a030c41a6909d9ab99ac5bbf098e5ed\",\n",
      "        \"title\": \"Multiparameter cell characterization using nanofluidic technology facilitates real-time phenotypic and genotypic elucidation of intratumor heterogeneity\",\n",
      "        \"abstract\": \"Genetic and functional complexity of bulk tumor has become evident through rapid advances in sequencing technologies. As a unique integrated approach to characterizing tumor heterogeneity, we demonstrate the multifaceted capabilities of a novel nanofluidic platform to enable single-cell phenotypic and genetic profiling of ovarian cancer patient-derived tumor cells. This approach has enabled increased resolution of tumor cell phenotypic and genetic heterogeneity, providing a better understanding of underlying biological drivers of the disease. A range of CA-125 expression levels is observed within cells from individuals, demonstrating clonal diversity consistent with other phenotypic data. Further, TP53 mutation analysis demonstrates a sub-population of cells exhibiting high mutation frequency that likely drives downstream growth kinetics and protein expression. Finally, genomic data is orthogonally used to address clonal heterogeneity across ovarian tumors when compared to bulk sequencing, illustrating the potential for single-cell sequencing data integrated with cellular functional and growth data toward future therapeutic intervention.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1561&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7c2dbb32de2a0822bba7d419bf81d13bbdd3fe73\",\n",
      "        \"title\": \"Transfer learning in single-cell transcriptomics improves data denoising and pattern discovery\",\n",
      "        \"abstract\": \"Although single-cell RNA sequencing (scRNA-seq) technologies have shed light on the role of cellular diversity in human pathophysiology1\\u20133, the resulting data remains noisy and sparse, making reliable quantification of gene expression challenging. Here, we show that a deep autoencoder coupled to a Bayesian model remarkably improves UMI-based scRNA-seq data quality by transfer learning across datasets. This new technology, SAVER-X, outperforms existing state-of-the-art tools. The deep learning model in SAVER-X extracts transferable gene expression features across data from different labs, generated by varying technologies, and obtained from divergent species. Through this framework, we explore the limits of transfer learning in a diverse testbed and demonstrate that future human sequencing projects will unequivocally benefit from the accumulation of publicly available data. We further show, through examples in immunology and neurodevelopment, that SAVER-X can harness existing public data to enhance downstream analysis of new data, such as those collected in clinical settings.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1562&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2dc8a86c9a834c10787a4474f07f77ecbc3f9f8b\",\n",
      "        \"title\": \"Gene detection models outperform gene expression for large-scale scRNA-seq analysis\",\n",
      "        \"abstract\": \"The number of cells to be sequenced is one of the most important considerations in single cell RNA experiment design, and we find that experiments designed to maximize the number of sequenced cells tends to also increase the measurement noise within individual cells. This noise is detrimental to the performance of downstream scRNA-seq analysis tools, but can be mitigated by ignoring precise expression measurements and focusing on the detection of a gene. This observation that throwing away data can increase signal is observed across multiple capture technologies, can be addressed using as little as one line of code, and results in execution times that are up to three orders of magnitude faster, therefore making this observation important as newer capture technologies continue to drive the number of sequenced cells higher.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1563&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3482410a861a8bfbf7b57dd8e0b4f79f1b5eda9d\",\n",
      "        \"title\": \"Tumour heterogeneity and metastasis at single-cell resolution\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1564&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"89b2abcd21a9d6f27aed220bd4f2176fd021cb2c\",\n",
      "        \"title\": \"Automated Optimal Parameters for T-Distributed Stochastic Neighbor Embedding Improve Visualization and Allow Analysis of Large Datasets\",\n",
      "        \"abstract\": \"Accurate and comprehensive extraction of information from high-dimensional single cell datasets necessitates faithful visualizations to assess biological populations. A state-of-the-art algorithm for non-linear dimension reduction, t-SNE, requires multiple heuristics and fails to produce clear representations of datasets when millions of cells are projected. We developed opt-SNE, an automated toolkit for optimal t-SNE parameter selection that utilizes Kullback-Liebler divergence evaluation in real time to tailor the early exaggeration and overall number of gradient descent iterations in a dataset-specific manner. The precise calibration of early exaggeration together with opt-SNE adjustment of gradient descent learning rate dramatically improves computation time and enables high-quality visualization of large cytometry and transcriptomics datasets, overcoming limitations of analysis tools with hard-coded parameters that often produce poorly resolved or misleading maps of fluorescent and mass cytometry data. In summary, opt-SNE enables optimal data resolution in t-SNE space and more precise data interpretation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1565&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2da0810bbb073c283f3f9451a12264a8b57eb1fd\",\n",
      "        \"title\": \"Large-scale reconstruction of cell lineages using single-cell readout of transcriptomes and CRISPR\\u2013Cas9 barcodes by scGESTALT\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1566&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8715090cf4194fa57a0fafc29a98ac2e5b2415ef\",\n",
      "        \"title\": \"A Single-Cell Sequencing Guide for Immunologists\",\n",
      "        \"abstract\": \"In recent years there has been a rapid increase in the use of single-cell sequencing (scRNA-seq) approaches in the field of immunology. With the wide range of technologies available, it is becoming harder for users to select the best scRNA-seq protocol/platform to address their biological questions of interest. Here, we compared the advantages and limitations of four commonly used scRNA-seq platforms in order to clarify their suitability for different experimental applications. We also address how the datasets generated by different scRNA-seq platforms can be integrated, and how to identify unknown populations of single cells using unbiased bioinformatics methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1567&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"42320a52b7400f72b1dd1ed62cba701de85cebee\",\n",
      "        \"title\": \"Integrated measurement of intracellular proteins and transcripts in single cells.\",\n",
      "        \"abstract\": \"Biological function arises from the interplay of proteins, transcripts, and metabolites. An ongoing revolution in miniaturization technologies has created tools to analyze any one of these species in single cells, thus resolving the heterogeneity of tissues previously invisible to bulk measurements. An emerging frontier is single cell multi-omics, which is the measurement of multiple classes of analytes from single cells. Here, we combine bead-based transcriptomics with microchip-based proteomics to measure intracellular proteins and transcripts from single cells and defined small numbers of cells. The transcripts and proteins are independently measured by sequencing and fluorescent immunoassays respectively, to preserve their optimal measurement modes, and linked by encoding the physical address locations of the cells into digital sequencing space using spatially patterned DNA barcodes. We resolve cell-type-specific protein and transcript signatures and present a path forward to scaling the platform to high-throughput.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1568&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7e4f67c856114989d32761e592c8b95e47ce2376\",\n",
      "        \"title\": \"Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1569&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8fcdf06a6701579f3951945bf232d25d36662a4f\",\n",
      "        \"title\": \"Mixed-effects association of single cells identifies an expanded effector CD4+ T cell subset in rheumatoid arthritis\",\n",
      "        \"abstract\": \"Mixed-effects regression of single-cell data accounts for confounding variation and reveals an expanded CD4+ T cell population in rheumatoid arthritis. Pinpointing culprits in single-cell data New techniques analyzing single cells are being applied to clinical samples. These techniques generate large datasets, and it can be challenging to identify rare cell types associated with disease. Fonseka et al. developed a simple statistical method to do just that while simultaneously controlling for confounding biological and technical variation. Using their method on single-cell mass and flow cytometry data revealed an expanded CD4+ T cell subset in the blood of rheumatoid arthritis patients that was later seen to contract upon treatment. The authors\\u2019 method performed better than the current gold standard and could be a potentially widely applied tool for the analysis of high-dimensional single-cell data. High-dimensional single-cell analyses have improved the ability to resolve complex mixtures of cells from human disease samples; however, identifying disease-associated cell types or cell states in patient samples remains challenging because of technical and interindividual variation. Here, we present mixed-effects modeling of associations of single cells (MASC), a reverse single-cell association strategy for testing whether case-control status influences the membership of single cells in any of multiple cellular subsets while accounting for technical confounders and biological variation. Applying MASC to mass cytometry analyses of CD4+ T cells from the blood of rheumatoid arthritis (RA) patients and controls revealed a significantly expanded population of CD4+ T cells, identified as CD27\\u2212 HLA-DR+ effector memory cells, in RA patients (odds ratio, 1.7; P = 1.1 \\u00d7 10\\u22123). The frequency of CD27\\u2212 HLA-DR+ cells was similarly elevated in blood samples from a second RA patient cohort, and CD27\\u2212 HLA-DR+ cell frequency decreased in RA patients who responded to immunosuppressive therapy. Mass cytometry and flow cytometry analyses indicated that CD27\\u2212 HLA-DR+ cells were associated with RA (meta-analysis P = 2.3 \\u00d7 10\\u22124). Compared to peripheral blood, synovial fluid and synovial tissue samples from RA patients contained about fivefold higher frequencies of CD27\\u2212 HLA-DR+ cells, which comprised ~10% of synovial CD4+ T cells. CD27\\u2212 HLA-DR+ cells expressed a distinctive effector memory transcriptomic program with T helper 1 (TH1)\\u2013 and cytotoxicity-associated features and produced abundant interferon-\\u03b3 (IFN-\\u03b3) and granzyme A protein upon stimulation. We propose that MASC is a broadly applicable method to identify disease-associated cell populations in high-dimensional single-cell data.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1570&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6a60db685e7db8fbde901dadc7c676b08f86184c\",\n",
      "        \"title\": \"High throughput droplet single-cell Genotyping of Transcriptomes (GoT) reveals the cell identity dependency of the impact of somatic mutations\",\n",
      "        \"abstract\": \"Defining the transcriptomic identity of clonally related malignant cells is challenging in the absence of cell surface markers that distinguish cancer clones from one another or from admixed non-neoplastic cells. While single-cell methods have been devised to capture both the transcriptome and genotype, these methods are not compatible with droplet-based single-cell transcriptomics, limiting their throughput. To overcome this limitation, we present single-cell Genotyping of Transcriptomes (GoT), which integrates cDNA genotyping with high-throughput droplet-based single-cell RNA-seq. We further demonstrate that multiplexed GoT can interrogate multiple genotypes for distinguishing subclonal transcriptomic identity. We apply GoT to 26,039 CD34+ cells across six patients with myeloid neoplasms, in which the complex process of hematopoiesis is corrupted by CALR-mutated stem and progenitor cells. We define high-resolution maps of malignant versus normal hematopoietic progenitors, and show that while mutant cells are comingled with wildtype cells throughout the hematopoietic progenitor landscape, their frequency increases with differentiation. We identify the unfolded protein response as a predominant outcome of CALR mutations, with significant cell identity dependency. Furthermore, we identify that CALR mutations lead to NF-\\u03baB pathway upregulation specifically in uncommitted early stem cells. Collectively, GoT provides high-throughput linkage of single-cell genotypes with transcriptomes and reveals that the transcriptional output of somatic mutations is heavily dependent on the native cell identity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1571&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a1c16e11821532a662a13722db6b1e9a5ae2a096\",\n",
      "        \"title\": \"Multi-Omics Profiling of the Tumor Microenvironment: Paving the Way to Precision Immuno-Oncology\",\n",
      "        \"abstract\": \"The tumor microenvironment (TME) is a multifaceted ecosystem characterized by profound cellular heterogeneity, dynamicity, and complex intercellular cross-talk. The striking responses obtained with immune checkpoint blockers, i.e., antibodies targeting immune-cell regulators to boost antitumor immunity, have demonstrated the enormous potential of anticancer treatments that target TME components other than tumor cells. However, as checkpoint blockade is currently beneficial only to a limited fraction of patients, there is an urgent need to understand the mechanisms orchestrating the immune response in the TME to guide the rational design of more effective anticancer therapies. In this Mini Review, we give an overview of the methodologies that allow studying the heterogeneity of the TME from multi-omics data generated from bulk samples, single cells, or images of tumor-tissue slides. These include approaches for the characterization of the different cell phenotypes and for the reconstruction of their spatial organization and inter-cellular cross-talk. We discuss how this broader vision of the cellular heterogeneity and plasticity of tumors, which is emerging thanks to these methodologies, offers the opportunity to rationally design precision immuno-oncology treatments. These developments are fundamental to overcome the current limitations of targeted agents and checkpoint blockers and to bring long-term clinical benefits to a larger fraction of cancer patients.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1572&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8d712778a1af42b181763d14311415ce9d3fec4c\",\n",
      "        \"title\": \"RNA Velocity: Molecular Kinetics from Single-Cell RNA-Seq.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1573&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3e06d2797a9f9daac8e460ce057d2bd89528d30b\",\n",
      "        \"title\": \"Multivariate Control of Transcript to Protein Variability in Single Mammalian Cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1574&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"08ad396d6dd4f2de14d65b15d1baf003cd3f72da\",\n",
      "        \"title\": \"Macrophage Biology, Classification, and Phenotype in Cardiovascular Disease: JACC Macrophage in CVD Series (Part 1).\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1575&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e97dcb5f92a86a99c545ebf144555228c5738683\",\n",
      "        \"title\": \"High-throughput targeted long-read single cell sequencing reveals the clonal and transcriptional landscape of lymphocytes\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1576&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4b4e715e63f3c56dcf3d085f3af37f3ae780c933\",\n",
      "        \"title\": \"Tutorial: guidelines for the experimental design of single-cell RNA sequencing studies\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1577&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5cafe2e690af3794f730c225690fea4a164d59b5\",\n",
      "        \"title\": \"SuperCT: A supervised-learning-framework to enhance the characterization of single-cell transcriptomic profiles\",\n",
      "        \"abstract\": \"Characterization of individual cell types is fundamental to the study of multicellular samples such as tumor tissues. Single-cell RNAseq techniques, which allow high-throughput expression profiling of individual cells, have significantly advanced our ability of this task. Currently, most of the scRNA-seq data analyses are commenced with unsupervised clustering of cells followed by visualization of clusters in a low-dimensional space. Clusters are often assigned to different cell types based on canonical markers. However, the efficiency of characterizing the known cell types in this way is low and limited by the investigator[s] knowledge. In this study, we present a technical framework of training the expandable supervised-classifier in order to reveal the single-cell identities based on their RNA expression profiles. Using multiple scRNA-seq datasets we demonstrate the superior accuracy, robustness, compatibility and expandability of this new solution compared to the traditional methods. We use two examples of model upgrade to demonstrate how the projected evolution of the cell-type classifier is realized.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1578&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3841129a04b9237c1cf234aaa4c90a9f5e6a156e\",\n",
      "        \"title\": \"Human Bone Marrow Assessment by Single Cell RNA Sequencing, Mass Cytometry and Flow Cytometry\",\n",
      "        \"abstract\": \"New techniques for single-cell analysis have led to insights into hematopoiesis and the immune system, but the ability of these techniques to cross-validate and reproducibly identify the biological variation in diverse human samples is currently unproven. We therefore performed a comprehensive assessment of human bone marrow cells using both single-cell RNA sequencing and multiparameter flow cytometry from twenty healthy adult human donors across a broad age range. These data characterize variation between healthy donors as well as age-associated changes in cell population frequencies. Direct comparison of techniques revealed discrepancy in the quantification of T lymphocyte and natural killer cell populations. Orthogonal validation of immunophenotyping using mass cytometry demonstrated good correlation with flow cytometry. Technical replicates using single-cell RNA sequencing matched robustly, while biological replicates showed variation. Given the increasing use of single-cell technologies in translational research, this resource serves as an important reference dataset and highlights opportunities for further refinement.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1579&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f3074bf6fce6950d1a79047ca93504c0c32878ab\",\n",
      "        \"title\": \"Understanding the Biology and Pathogenesis of the Kidney by Single-Cell Transcriptomic Analysis\",\n",
      "        \"abstract\": \"Background: Single-cell RNA-seq (scRNA-seq) has recently emerged as a revolutionary and powerful tool for biomedical research. However, there have been relatively few studies using scRNA-seq in the field of kidney study. Summary: scRNA-seq achieves gene expression profiling at single-cell resolution in contrast with the conventional methods of gene expression profiling, which are based on cell population and give averaged values of gene expression of the cells. Single-cell transcriptomic analysis is crucial because individual cells of the same type are highly heterogeneous in gene expression, which reflects the existence of subpopulations, different cellular states, or molecular dynamics, of the cells, and should be resolved for further insights. In addition, gene expression analysis of tissues or organs that usually comprise multiple cell types or subtypes results in data that are not fully applicable to any given cell type. scRNA-seq is capable of identifying all cell types and subtypes in a tissue, including those that are new or present in small quantity. With these unique capabilities, scRNA-seq has been used to dissect molecular processes in cell differentiation and to trace cell lineages in development. It is also used to analyze the cells in a lesion of disease to identify the cell types and molecular dynamics implicated in the injury. With continuous technical improvement, scRNA-seq has become extremely high throughput and cost effective, making it accessible to all laboratories. In the present review article, we provide an overall review of scRNA-seq concerning its history, improvements, and applications. In addition, we describe the available studies in which scRNA-seq was employed in the field of kidney research. Lastly, we discuss other potential uses of scRNA-seq for kidney research. Key Message: This review article provides general information on scRNA-seq and its various uses. Particularly, we summarize the studies in the field of kidney diseases in which scRNA-seq was used and discuss potential additional uses of scRNA-seq for kidney research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1580&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e752323e02e679ccec8e80a86c86cc8d0169649e\",\n",
      "        \"title\": \"Single-Cell Multi-omics: An Engine for New Quantitative Models of Gene Regulation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1581&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7851bac7a0496a15ecf87a40fd18b143bb4b547f\",\n",
      "        \"title\": \"Computational approaches for high\\u2010throughput single\\u2010cell data analysis\",\n",
      "        \"abstract\": \"During the past decade, the number of novel technologies to interrogate biological systems at the single\\u2010cell level has skyrocketed. Numerous approaches for measuring the proteome, genome, transcriptome and epigenome at the single\\u2010cell level have been pioneered, using a variety of technologies. All these methods have one thing in common: they generate large and high\\u2010dimensional datasets that require advanced computational modelling tools to highlight and interpret interesting patterns in these data, potentially leading to novel biological insights and hypotheses. In this work, we provide an overview of the computational approaches used to interpret various types of single\\u2010cell data in an automated and unbiased way.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1582&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4dc4b1273f5df61800366a5eaf68d8c7ec7340b0\",\n",
      "        \"title\": \"Functional interpretation of single cell similarity maps\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1583&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a757de35a2592c42094e50ec2a162f50d056e01d\",\n",
      "        \"title\": \"Co-expression in Single-Cell Analysis: Saving Grace or Original Sin?\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1584&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9059090f61c369dd59aa0fe4b5412c05affdfdef\",\n",
      "        \"title\": \"Ultrasensitive profiling of multiple biomarkers from single cells by signal amplification mass spectrometry.\",\n",
      "        \"abstract\": \"A signal amplification protocol based on mass spectrometry (MS) was developed to profile simultaneously multiple biomarkers from a single cell using various mass label (ML)-modified Au nanoparticles (AuNPs). The strategy with ultrahigh sensitivity and specificity has potential prospects in the deep exploration of molecular and cellular characterization.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1585&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1b835cd760df686348f8aa2f5cc9480e547d57a4\",\n",
      "        \"title\": \"MULTI-seq: Scalable sample multiplexing for single-cell RNA sequencing using lipid-tagged indices\",\n",
      "        \"abstract\": \"We describe MULTI-seq: A rapid, modular, and universal scRNA-seq sample multiplexing strategy using lipid-tagged indices. MULTI-seq reagents can barcode any cell type from any species with an accessible plasma membrane. The method is compatible with enzymatic tissue dissociation, and also preserves viability and endogenous gene expression patterns. We leverage these features to multiplex the analysis of multiple solid tissues comprising human and mouse cells isolated from patient-derived xenograft mouse models. We also utilize MULTI-seq\\u2019s modular design to perform a 96-plex perturbation experiment with human mammary epithelial cells. MULTI-seq also enables robust doublet identification, which improves data quality and increases scRNA-seq cell throughput by minimizing the negative effects of Poisson loading. We anticipate that the sample throughput and reagent savings enabled by MULTI-seq will expand the purview of scRNA-seq and democratize the application of these technologies within the scientific community.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1586&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"11652b1969a75beecd5a6af2c22d996ad78f85c8\",\n",
      "        \"title\": \"From the Human Cell Atlas to dynamic immune maps in human disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1587&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f9f64fed03dbec8f046f714fa83604503865cf15\",\n",
      "        \"title\": \"Designer T-cells and T-cell receptors for customized cancer immunotherapies.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1588&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"47ce69ce7469a367a74dd8d79460fbf637c285c7\",\n",
      "        \"title\": \"Single-Cell RNA Sequencing: A New Window into Cell Scale Dynamics.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1589&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2134b04cae1a0600c183f9e20e4d0b3031d392b4\",\n",
      "        \"title\": \"Machine Learning for Integrating Data in Biology and Medicine: Principles, Practice, and Opportunities\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1590&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ae491975b775d0023bb74c6d7b40481d544e374\",\n",
      "        \"title\": \"Combined quantification of intracellular (phospho-)proteins and transcriptomics from fixed single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1591&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"533cebc8a1e5b671bc16af8a9c772929ff46f2ce\",\n",
      "        \"title\": \"Immune Cell Dynamics Unfolded by Single-Cell Technologies\",\n",
      "        \"abstract\": \"The single-cell revolution is paving the way towards the molecular characterisation of every cell type in the human body, revealing relationships between cell types and states at high resolution. Changes in cellular phenotypes are particularly prevalent in the immune system and can be observed in its continuous remodelling up to adulthood, response to disease and development of immunological memory. In this review, we delve into the world of cellular dynamics of the immune system. We discuss current single-cell experimental and computational approaches in this area, giving insights into plasticity and commitment of cell fates. Finally, we provide an outlook on upcoming technological developments and predict how these will improve our understanding of the immune system.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1592&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b019fb9f84c733ea8ff938ebe2c5a584fb158044\",\n",
      "        \"title\": \"Human adaptive immune receptor repertoire analysis\\u2014Past, present, and future\",\n",
      "        \"abstract\": \"The genes encoding adaptive immune antigen receptors, namely the immunoglobulins expressed in membrane\\u2010bound or secreted forms by B cells, and the cell surface T cell receptors, are unique in human biology because they are generated by combinatorial rearrangement of the genomic DNA. The diversity of receptors so generated in populations of lymphocytes enables the human immune system to recognize antigens expressed by pathogens, but also underlies the pathological specificity of autoimmune diseases and the mistargeted immunity in allergies. Several recent technological developments, foremost among them the invention of high\\u2010throughput DNA sequencing instruments, have enabled much deeper and thorough evaluation of clones of human B cells and T cells and the antigen receptors they express during physiological and pathogenic immune responses. The evolutionary struggles between host adaptive immune responses and populations of pathogens are now open to greater scrutiny, elucidation of the underlying reasons for successful or failed immunity, and potential predictive modeling, than ever before. Here we give an overview of the foundations, recent progress, and future prospects in this dynamic area of research.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1593&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"47b25b41fce186817fdd9d7f48379be5bbedd64d\",\n",
      "        \"title\": \"High-throughput magnetic particle washing in nanoliter droplets using serial injection and splitting\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1594&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0ef77c109f58fc418aa5a2f70eff1839050b8e30\",\n",
      "        \"title\": \"Blood-based analyses of cancer: Circulating myeloid-derived suppressor cells \\u2013 is a new era coming?\",\n",
      "        \"abstract\": \"Abstract Progress in cancer treatment made by the beginning of the 21st century has shifted the paradigm from one-size-fits-all to tailor-made treatment. The popular vision, to study solid tumors through the relatively noninvasive sampling of blood, is one of the most thrilling and rapidly advancing fields in global cancer diagnostics. From this perspective, immune-cell analysis in cancer could play a pivotal role in oncology practice. This approach is driven both by rapid technological developments, including the analysis of circulating myeloid-derived suppressor cells (cMDSCs), and by the increasing application of (immune) therapies, the success or failure of which may depend on effective and timely measurements of relevant biomarkers. Although the implementation of these powerful noninvasive diagnostic capabilities in guiding precision cancer treatment is poised to change the ways in which we select and monitor cancer therapy, challenges remain. Here, we discuss the challenges associated with the analysis and clinical aspects of cMDSCs and assess whether the problems in implementing tumor-evolution monitoring as a global tool in personalized oncology can be overcome.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1595&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a6b2e7d25f3363089e735633cad6b53b96651b6\",\n",
      "        \"title\": \"An atlas of the aging lung mapped by single cell transcriptomics and deep tissue proteomics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1596&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"af63c4cf90a5cc6cfa80f56d17805e985fa985c9\",\n",
      "        \"title\": \"Defining Inflammatory Cell States in Rheumatoid Arthritis Joint Synovial Tissues by Integrating Single-cell Transcriptomics and Mass Cytometry\",\n",
      "        \"abstract\": \"To define the cell populations in rheumatoid arthritis (RA) driving joint inflammation, we applied single-cell RNA-seq (scRNA-seq), mass cytometry, bulk RNA-seq, and flow cytometry to sorted T cells, B cells, monocytes, and fibroblasts from 51 synovial tissue RA and osteoarthritis (OA) patient samples. Utilizing an integrated computational strategy based on canonical correlation analysis to 5,452 scRNA-seq profiles, we identified 18 unique cell populations. Combining mass cytometry and transcriptomics together revealed cell states expanded in RA synovia: THY1+HLAhigh sublining fibroblasts (OR=33.8), IL1B+ pro-inflammatory monocytes (OR=7.8), CD11c+T-bet+ autoimmune-associated B cells (OR=5.7), and PD-1+Tph/Tfh (OR=3.0). We also defined CD8+ T cell subsets characterized by GZMK+, GZMB+, and GNLY+ expression. Using bulk and single-cell data, we mapped inflammatory mediators to source cell populations, for example attributing IL6 production to THY1+HLAhigh fibroblasts and na\\u00efve B cells, and IL1B to pro-inflammatory monocytes. These populations are potentially key mediators of RA pathogenesis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1597&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"460bb6b0aeba7fb984589a0e74b8f0967f94ee23\",\n",
      "        \"title\": \"diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1598&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fae873d77ec51fb332958a3e5ff6911185e217b2\",\n",
      "        \"title\": \"Single cell protein analysis for systems biology.\",\n",
      "        \"abstract\": \"The cellular abundance of proteins can vary even between isogenic single cells. This variability between single-cell protein levels can have regulatory roles, such as controlling cell fate during apoptosis induction or the proliferation/quiescence decision. Here, we review examples connecting protein levels and their dynamics in single cells to cellular functions. Such findings were made possible by the introduction of antibodies, and subsequently fluorescent proteins, for tracking protein levels in single cells. However, in heterogeneous cell populations, such as tumors or differentiating stem cells, cellular decisions are controlled by hundreds, even thousands of proteins acting in concert. Characterizing such complex systems demands measurements of thousands of proteins across thousands of single cells. This demand has inspired the development of new methods for single-cell protein analysis, and we discuss their trade-offs, with an emphasis on their specificity and coverage. We finish by highlighting the potential of emerging mass-spec methods to enable systems-level measurement of single-cell proteomes with unprecedented coverage and specificity. Combining such methods with methods for quantitating the transcriptomes and metabolomes of single cells will provide essential data for advancing quantitative systems biology.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1599&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"925847f156d2d9e8e3798d0295dd7a351dbdfc13\",\n",
      "        \"title\": \"Defining Cell Identity with Single\\u2010Cell Omics\",\n",
      "        \"abstract\": \"Cells are a fundamental unit of life, and the ability to study the phenotypes and behaviors of individual cells is crucial to understanding the workings of complex biological systems. Cell phenotypes (epigenomic, transcriptomic, proteomic, and metabolomic) exhibit dramatic heterogeneity between and within the different cell types and states underlying cellular functional diversity. Cell genotypes can also display heterogeneity throughout an organism, in the form of somatic genetic variation\\u2014most notably in the emergence and evolution of tumors. Recent technical advances in single\\u2010cell isolation and the development of omics approaches sensitive enough to reveal these aspects of cell identity have enabled a revolution in the study of multicellular systems. In this review, we discuss the technologies available to resolve the genomes, epigenomes, transcriptomes, proteomes, and metabolomes of single cells from a wide variety of living systems.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1600&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9ce7454d0c58419f0f213b102d8fe3dbeda3ad51\",\n",
      "        \"title\": \"Linking single-cell measurements of mass, growth rate, and gene expression\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1601&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"01f696356c5b89fd5712612ab461e133eb5dbe44\",\n",
      "        \"title\": \"S\\ufeffingle-cell approaches to immune profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1602&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fdddf19d4535842a2cedc24f42954f4cef9caf05\",\n",
      "        \"title\": \"Highly Multiplexed Single-Cell Protein Analysis.\",\n",
      "        \"abstract\": \"Single-cell proteomic analysis is crucial to advance our understanding of normal physiology and disease pathogenesis. The comprehensive protein profiling in individual cells of a heterogeneous sample can provide new insights into many important biological issues, such as the regulation of inter- and intracellular signaling pathways or the varied cellular compositions of normal and diseased tissues. With highly multiplexed molecular imaging of many different protein biomarkers in patient biopsies, diseases can be accurately diagnosed to guide the selection of the ideal treatment. In this Minireview, we will describe the recent technological advances of single-cell proteomic assays, discuss their advantages and limitations, highlight their applications in biology and precision medicine, and present the current challenges and potential solutions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1603&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9e1bc018319d1fd958921a0e8d9909106ae7538a\",\n",
      "        \"title\": \"Highly Multiplexed Single-Cell RNA-seq for Defining Cell Population and Transcriptional Spaces\",\n",
      "        \"abstract\": \"We describe a universal sample multiplexing method for single-cell RNA-seq in which cells are chemically labeled with identifying DNA oligonucleotides. Analysis of a 96-plex perturbation experiment revealed changes in cell population structure and transcriptional states that cannot be discerned from bulk measurements, establishing a cost effective means to survey cell populations from large experiments and clinical samples with the depth and resolution of single-cell RNA-seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1604&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cc077df5b82dd8418bc30c7eab3e77ad1a95d31b\",\n",
      "        \"title\": \"Single Cell Multi-Omics Technology: Methodology and Application\",\n",
      "        \"abstract\": \"In the era of precision medicine, multi-omics approaches enable the integration of data from diverse omics platforms, providing multi-faceted insight into the interrelation of these omics layers on disease processes. Single cell sequencing technology can dissect the genotypic and phenotypic heterogeneity of bulk tissue and promises to deepen our understanding of the underlying mechanisms governing both health and disease. Through modification and combination of single cell assays available for transcriptome, genome, epigenome, and proteome profiling, single cell multi-omics approaches have been developed to simultaneously and comprehensively study not only the unique genotypic and phenotypic characteristics of single cells, but also the combined regulatory mechanisms evident only at single cell resolution. In this review, we summarize the state-of-the-art single cell multi-omics methods and discuss their applications, challenges, and future directions.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1605&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d9c208af563534ed6add7b8e1e9372909e850f83\",\n",
      "        \"title\": \"Single-cell RNA-seq denoising using a deep count autoencoder\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1606&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"330605408e0e78762406156de53bcec2306d3f6a\",\n",
      "        \"title\": \"Analyse de la variabilit\\u00e9 de l\\u2019expression g\\u00e9nique et du m\\u00e9tabolisme glycolytique au cours du processus de diff\\u00e9renciation \\u00e9rythrocytaire : de l\\u2019analyse \\u00e0 grande \\u00e9chelle aux questions m\\u00e9canistiques\",\n",
      "        \"abstract\": \"La prise de decision cellulaire se traduit par la capacite de toute cellule vivante a integrer les differentes informations provenant de son environnement, et a les transformer en une reponse biologique coherente. Il est aujourd'hui de plus en plus demontre que les populations cellulaires presentent une heterogeneite quantitative et qualitative significative, qui pourrait jouer un role essentiel dans le fonctionnement des organismes vivants. La premiere partie de ma these a ainsi consiste a etudier la variabilite de l'expression genique au cours de la differenciation de progeniteurs erythrocytaires aviaires primaires, a l'echelle de la cellule unique. L'expression de 92 genes a ete analysee par RT-qPCR dans des cellules isolees a differents temps de differenciation. Les principaux resultats de cette etude ont montre que la variabilite de l'expression des genes, mesuree par l'entropie de Shannon, atteint un niveau maximal a 8h-24h de differenciation, simultanement a une chute du nombre de genes correles. Cette augmentation de la variabilite genique precede l'engagement irreversible des cellules dans le processus de differenciation erythrocytaire identifie entre 24 et 48h. Cette etude a egalement mis en lumiere le gene LDHA (Lactate dehydrogenase A), codant pour une enzyme de la glycolyse anaerobie, dans les progeniteurs erythrocytaires en etat d'auto-renouvellement et aux points critiques, 8h et 24h, de la differenciation. La deuxieme partie de ma these a donc consiste a analyser le role precis de LDHA dans l'auto-renouvellement des progeniteurs erythrocytaires, ainsi que les variations du metabolisme du glucose au cours de la differenciation. Nos premiers resultats suggerent que le processus de differenciation erythrocytaire s'accompagne d'un changement metabolique correspondant au passage de la glycolyse anaerobie dependante de LDHA, vers une production d'energie aerobie, reposant sur la phosphorylation oxydative\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1607&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0c437ead1aa1c4adc7fc329d8fc02794a23a6fb9\",\n",
      "        \"title\": \"Mass cytometry: a powerful tool for dissecting the immune landscape.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1608&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c43948bc6ccf55c8a78827d792113446dae9ecf1\",\n",
      "        \"title\": \"The use of single-cell RNA-Seq to understand virus-host interactions.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1609&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a82e68fe28ddcc08ffd3a8422013af61fd05aecd\",\n",
      "        \"title\": \"Bayesian Inference for a Generative Model of Transcriptome Profiles from Single-cell RNA Sequencing\",\n",
      "        \"abstract\": \"Transcriptome profiles of individual cells reflect true and often unexplored biological diversity, but are also affected by noise of biological and technical nature. This raises the need to explicitly model the resulting uncertainty and take it into account in any downstream analysis, such as dimensionality reduction, clustering, and differential expression. Here, we introduce Single-cell Variational Inference (scVI), a scalable framework for probabilistic representation and analysis of gene expression in single cells. Our model uses variational inference and stochastic optimization of deep neural networks to approximate the parameters that govern the distribution of expression values of each gene in every cell, using a non-linear mapping between the observations and a low-dimensional latent space. By doing so, scVI pools information between similar cells or genes while taking nuisance factors of variation such as batch effects and limited sensitivity into account. To evaluate scVI, we conducted a comprehensive comparative analysis to existing methods for distributional modeling and dimensionality reduction, all of which rely on generalized linear models. We first show that scVI scales to over one million cells, whereas competing algorithms can process at most tens of thousands of cells. Next, we show that scVI fits unseen data more closely and can impute missing data more accurately, both indicative of a better generalization capacity. We then utilize scVI to conduct a set of fundamental analysis tasks \\u2013 including batch correction, visualization, clustering and differential expression \\u2013 and demonstrate its accuracy in comparison to the state-of-the-art tools in each task. scVI is publicly available, and can be readily used as a principled and inclusive solution for multiple tasks of single-cell RNA sequencing data analysis.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1610&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8f7fd1e462296c3ea09edb8aabc7fe486e6e21f1\",\n",
      "        \"title\": \"Single cell transcriptomics reveals molecular subtype and functional heterogeneity in models of breast cancer\",\n",
      "        \"abstract\": \"Breast cancer has long been classified into a number of molecular subtypes that predict prognosis and therefore influence clinical treatment decisions. Cellular heterogeneity is also evident in breast cancers and plays a key role in the development, evolution and metastatic progression of many cancers. How clinical heterogeneity relates to cellular heterogeneity is poorly understood, so we approached this question using single cell gene expression analysis of well established in vitro and in vivo models of disease. To explore the cellular heterogeneity in breast cancer we first examined a panel of genes that define the PAM50 classifier of molecular subtype. Five breast cancer cell line models (MCF7, BT474, SKBR3, MDA-MB-231, and MDA-MB-468) were selected as representatives of the intrinsic molecular subtypes (luminal A and B, basal-like, and Her2-enriched). Single cell multiplex RT-PCR was used to isolate and quantify the gene expression of single cells from each of these models, and the PAM50 classifier applied. Using this approach, we identified heterogeneity of intrinsic subtypes at single-cell level, indicating that cells with different subtypes exist within a cell line. Using the Chromium 10X system, this study was extended into thousands of cells from the MCF7 cell-line and an ER+ patient derived xenograft (PDX) model and again identified significant intra-tumour heterogeneity of molecular subtype. Estrogen Receptor (ER) is an important driver and therapeutic target in many breast cancers. It is heterogeneously expressed in a proportion of clinical cases but the significance of this to ER activity is unknown. Significant heterogeneity in the transcriptional activation of ER regulated genes was observed within tumours. This differential activation of the ER cistrome aligned with expression of two known transcriptional co-regulatory factors of ER (FOXA1 and PGR). To examine the degree of heterogeneity for other important phenotypic traits, we used an unsupervised clustering approach to identify cellular sub-populations with diverse cancer associated transcriptional properties, such as: proliferation; hypoxia; and treatment resistance. In particular, we show that we can identify two distinct sub-populations of cells that may have denovo resistance to endocrine therapies in a treatment na\\u00efve PDX model of ER+ breast cancer. One of these consists of cells with a non-proliferative transcriptional phenotype that is enriched for transcriptional properties of ERBB2 tumours. The other is heavily enriched for components of the primary cilia. Gene regulatory networks were used to identify transcription factor regulons that are active in each cell, leading us to identify potential transcriptional drivers (such as E2F7, MYB and RFX3) of the cilia associated endocrine resistant cells. This rare subpopulation of cells also has a highly heterogenous mix of intrinsic subtypes highlighting a potential role of intra-tumour subtype heterogeneity in endocrine resistance and metastatic potential. Overall, These results suggest a high degree of cellular heterogeneity within breast cancer models, even cell lines, that can be functionally dissected into sub-populations of cells with transcriptional phenotypes of potential clinical relevance.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1611&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ba418ddb78f8c467bbd7c256a2c2f527d7bcb821\",\n",
      "        \"title\": \"Implementation of a Multiplex and Quantitative Proteomics Platform for Assessing Protein Lysates Using DNA-Barcoded Antibodies*\",\n",
      "        \"abstract\": \"Molecular analysis of tumors forms the basis for personalized cancer medicine and increasingly guides patient selection for targeted therapy. Future opportunities for personalized medicine are highlighted by the measurement of protein expression levels via immunohistochemistry, protein arrays, and other approaches; however, sample type, sample quantity, batch effects, and \\u201ctime to result\\u201d are limiting factors for clinical application. Here, we present a development pipeline for a novel multiplexed DNA-labeled antibody platform which digitally quantifies protein expression from lysate samples. We implemented a rigorous validation process for each antibody and show that the platform is amenable to multiple protocols covering nitrocellulose and plate-based methods. Results are highly reproducible across technical and biological replicates, and there are no observed \\u201cbatch effects\\u201d which are common for most multiplex molecular assays. Tests from basal and perturbed cancer cell lines indicate that this platform is comparable to orthogonal proteomic assays such as Reverse-Phase Protein Array, and applicable to measuring the pharmacodynamic effects of clinically-relevant cancer therapeutics. Furthermore, we demonstrate the potential clinical utility of the platform with protein profiling from breast cancer patient samples to identify molecular subtypes. Together, these findings highlight the potential of this platform for enhancing our understanding of cancer biology in a clinical translation setting.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1612&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"24aa4bb21f8cc5d4078cad3a885f60e3a6abf58d\",\n",
      "        \"title\": \"Multiplexed in situ protein imaging using DNA-barcoded antibodies with extended hybridization chain reactions\",\n",
      "        \"abstract\": \"Immunofluorescence (IF) imaging using antibodies to visualize specific biomolecules is a widely used technique in both biological and clinical laboratories. Standard IF imaging methods using primary antibodies followed by secondary antibodies have low multiplexing capability due to limited availability of primary antibodies raised in different animal species. Here, we used a DNA-based signal amplification method, Hybridization Chain Reaction (HCR), to replace secondary antibodies to achieve multiplexed imaging using primary antibodies of the same species with superior signal intensity. To enable imaging with DNA-conjugated antibodies, we developed a new antibody staining protocol to minimize nonspecific binding of antibodies caused by conjugated DNA oligonucleotides. We also expanded the HCR hairpin pool from previously published 5 to 13 for highly multiplexed in situ imaging. We finally demonstrated multiplexed in situ protein imaging using the technique in both cultured cells and mouse retina sections.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1613&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"117e02c50a01854813d117ad4b27016e3bbd895b\",\n",
      "        \"title\": \"Single-Cell Sequencing in Normal and Malignant Hematopoiesis\",\n",
      "        \"abstract\": \"Abstract Hematopoiesis is one of the best studied adult stem-cell systems, with a differentiation hierarchy progressing from immature hematopoietic stem cells to over 10 distinct mature cell types. Recent technological breakthroughs now make it possible to define transcriptional profiles in thousands of individual cells. Facilitated by the wealth of prior data on cell purification and analysis strategies, hematopoiesis has been one of the earliest experimental systems to which many of these new single-cell sequencing technologies have been applied. In this review, the authors focus on recent studies, which have shed light on heterogeneity within individual populations as well as the relationships between populations, and also attempt to characterize the differences between normal and disease/perturbed states.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1614&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2bd156b43c08c53cc3e133aaca9f2e4a1450a78d\",\n",
      "        \"title\": \"Sampling from Single Cells\",\n",
      "        \"abstract\": \"The cell is the fundamental unit of biology. Major methodological advances in engineering and molecular biology have enabled the 'omics analysis of individual cells and supported biologists in understanding the deepest difference between health and disease. These advancements were based on the assumption that a single cell needs to be lysed or fixed before any in\\u2010depth analysis can be performed. Here, a picture is painted of the innovative methods used for extracting the contents of living cells without affecting their viability. These novel methods are now empowering the biological community to repeatedly interrogate a single cell over time, thus giving a dynamic representation of the cell's 'omics rather than a snapshot at a particular time point.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1615&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4dbab0ff601765a9314bc8f83a388cfea4534772\",\n",
      "        \"title\": \"Human dendritic cell subsets: an update\",\n",
      "        \"abstract\": \"Dendritic cells (DC) are a class of bone\\u2010marrow\\u2010derived cells arising from lympho\\u2010myeloid haematopoiesis that form an essential interface between the innate sensing of pathogens and the activation of adaptive immunity. This task requires a wide range of mechanisms and responses, which are divided between three major DC subsets: plasmacytoid DC (pDC), myeloid/conventional DC1 (cDC1) and myeloid/conventional DC2 (cDC2). Each DC subset develops under the control of a specific repertoire of transcription factors involving differential levels of IRF8 and IRF4 in collaboration with PU.1, ID2, E2\\u20102, ZEB2, KLF4, IKZF1 and BATF3. DC haematopoiesis is conserved between mammalian species and is distinct from monocyte development. Although monocytes can differentiate into DC, especially during inflammation, most quiescent tissues contain significant resident populations of DC lineage cells. An extended range of surface markers facilitates the identification of specific DC subsets although it remains difficult to dissociate cDC2 from monocyte\\u2010derived DC in some settings. Recent studies based on an increasing level of resolution of phenotype and gene expression have identified pre\\u2010DC in human blood and heterogeneity among cDC2. These advances facilitate the integration of mouse and human immunology, support efforts to unravel human DC function in vivo and continue to present new translational opportunities to medicine.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1616&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"091c65bc00674b396dd2cc8b8ba60cded8778b8d\",\n",
      "        \"title\": \"The Peripheral and Intratumoral Immune Cell Landscape in Cancer Patients: A Proxy for Tumor Biology and a Tool for Outcome Prediction\",\n",
      "        \"abstract\": \"Functional systemic and local immunity is required for effective anti-tumor responses. In addition to an active engagement with cancer cells and tumor stroma, immune cells can be affected and are often found to be dysregulated in cancer patients. The impact of tumors on local and systemic immunity can be assessed using a variety of approaches ranging from low-dimensional analyses that are performed on large patient cohorts to multi-dimensional assays that are technically and logistically challenging and are therefore confined to a limited sample size. Many of these strategies have been established in recent years leading to exciting findings. Not only were analyses of immune cells in tumor patients able to predict the clinical course of the disease and patients\\u2019 survival, numerous studies also detected changes in the immune landscape that correlated with responses to novel immunotherapies. This review will provide an overview of established and novel tools for assessing immune cells in tumor patients and will discuss exemplary studies that utilized these techniques to predict patient outcomes.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1617&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"90565abb8cd36e2ff785fb312a75ea2c48bb26c1\",\n",
      "        \"title\": \"Methods and challenges in the analysis of single-cell RNA-sequencing data\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1618&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5921065e8fe73ed93dc8c78cf3617132429bdeb7\",\n",
      "        \"title\": \"Statistical single cell multi-omics integration\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1619&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b5840f4b8bc1fc286aeec56ccea0be9edc4980b8\",\n",
      "        \"title\": \"Single-cell RNA-sequencing resolves self-antigen expression during mTEC development\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1620&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9f71e2327f53716f7f085fe739f335d1e5fab242\",\n",
      "        \"title\": \"Using single-cell multiple omics approaches to resolve tumor heterogeneity\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1621&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2aba23446cc8c1b3b844c0cdd6c4171214ee3d90\",\n",
      "        \"title\": \"Cell \\u201chashing\\u201d with barcoded antibodies enables multiplexing and doublet detection for single cell genomics\",\n",
      "        \"abstract\": \"Despite rapid developments in single cell sequencing technology, sample-specific batch effects, detection of cell doublets, and the cost of generating massive datasets remain outstanding challenges. Here, we introduce cell \\u201chashing\\u201d, where oligo-tagged antibodies against ubiquitously expressed surface proteins are used to uniquely label cells from distinct samples, which can be subsequently pooled. By sequencing these tags alongside the cellular transcriptome, we can assign each cell to its sample of origin, and robustly identify doublets originating from multiple samples. We demonstrate our approach by pooling eight human PBMC samples on a single run of the 10x Chromium system, substantially reducing our per-cell costs for library generation. Cell \\u201chashing\\u201d is inspired by, and complementary to, elegant multiplexing strategies based on genetic variation, which we also leverage to validate our results. We therefore envision that our approach will help to generalize the benefits of single cell multiplexing to diverse samples and experimental designs.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1622&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"1ce71b8f12b4e856a0a743ae296cb26942ad08f6\",\n",
      "        \"title\": \"Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq\",\n",
      "        \"abstract\": \"Due to the presence of systematic measurement biases, data normalization is an essential preprocessing step in the analysis of single-cell RNA sequencing (scRNA-seq) data. While a variety of normalization procedures are available for bulk RNA-seq, their suitability with respect to single-cell data is still largely unexplored. Furthermore, there may be multiple, competing considerations behind the assessment of normalization performance, some of them study-specific. The choice of normalization method can have a large impact on the results of downstream analyses (e.g., clustering, inference of cell lineages, differential expression analysis), and thus it is critically important to assess the performance of competing methods in order to select a suitable procedure for the study at hand. We have developed scone \\u2013 a framework that implements a wide range of normalization procedures in the context of scRNA-seq, and enables the assessment of their performance based on a comprehensive set of data-driven performance metrics. The accompanying open-source Bioconductor R software package scone (available at https://bioconductor.org/packages/scone) also provides numerical and graphical summaries of expression measures, data quality assessment, and data-adaptive gene and sample filtering criteria. We demonstrate the effectiveness of scone on a selection of scRNA-seq datasets across a variety of protocols, ranging from plate- to droplet-based methods. We show that scone is able to correctly rank normalization methods according to their performance in a given dataset and that selecting the best performing normalization leads to higher agreement with independent validation data than lowly-ranked methods.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1623&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e624772eb056e0d77711cec2ee3e91248fd15965\",\n",
      "        \"title\": \"Decoration of trastuzumab with short oligonucleotides: synthesis and detailed characterization.\",\n",
      "        \"abstract\": \"Trastuzumab (Herceptin\\u00ae) is an FDA-approved therapeutic antibody currently employed in the treatment of metastatic stages of breast cancer. Herein, we propose a simple, fast and cost-effective methodology to conjugate trastuzumab with 22-mer 5' thiol-modified oligonucleotides using a bifunctional crosslinker. The conjugates were successfully characterized by MALDI-ToF MS and SDS-PAGE, obviating the need for enzymatic digestion and difficult chromatographic separations. Furthermore, ELISA was performed to ensure that trastuzumab activity is not affected by oligonucleotide conjugation.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1624&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"05e405cba0f50e5e73039c9036e47c0a36f32bcb\",\n",
      "        \"title\": \"High-Throughput Approaches to Pinpoint Function within the Noncoding Genome.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1625&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b9c0006388f48175ee629ed7e67e9d89028ad696\",\n",
      "        \"title\": \"Determining T-cell specificity to understand and treat disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1626&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"271bd73ebed27650505123380f821d57ce510f52\",\n",
      "        \"title\": \"Single-cell transcriptomics to explore the immune system in health and disease\",\n",
      "        \"abstract\": \"The immune system varies in cell types, states, and locations. The complex networks, interactions, and responses of immune cells produce diverse cellular ecosystems composed of multiple cell types, accompanied by genetic diversity in antigen receptors. Within this ecosystem, innate and adaptive immune cells maintain and protect tissue function, integrity, and homeostasis upon changes in functional demands and diverse insults. Characterizing this inherent complexity requires studies at single-cell resolution. Recent advances such as massively parallel single-cell RNA sequencing and sophisticated computational methods are catalyzing a revolution in our understanding of immunology. Here we provide an overview of the state of single-cell genomics methods and an outlook on the use of single-cell techniques to decipher the adaptive and innate components of immunity.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1627&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ba4c83e781eb01eac665aea6a4954b2833b534e5\",\n",
      "        \"title\": \"Microfluidic single-cell technology in immunology and antibody screening.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1628&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4cbee2ef50c5f224c877bfade743528683f4d414\",\n",
      "        \"title\": \"The Author File: Marlon Stoeckius\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1629&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"efebd3eb2717297b198ebef1f23bae7921b3e1ae\",\n",
      "        \"title\": \"Multiplexed quantification of proteins and transcripts in single cells\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1630&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"17d7c1c363da80a4bf2187ee8d02f90a41923577\",\n",
      "        \"title\": \"New skin for the old RNA-Seq ceremony: the age of single-cell multi-omics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1631&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cbda2699d39e0f769354467c3a101b7e733e0a7f\",\n",
      "        \"title\": \"MOGSA: Integrative Single Sample Gene-set Analysis of Multiple Omics Data\",\n",
      "        \"abstract\": \"Gene-set analysis (GSA) summarizes information from molecule to gene-set level to facilitate biological interpretation of molecular profiling experiments. We present a statistical framework for single sample GSA of multiple 'omic data. MOGSA learns the most variant biomolecules and integrates data sets to generate gene set scores (GSS) for each sample. It extracts the contribution of individual data sets and biomolecules to GSS. MOGSA is rigorously benchmarked with simulated and real biological data and is implemented in the Bioconductor package \\u201cmogsa.\\u201d Graphical Abstract Highlights MOGSA is a new integrative multi 'omics single sample gene set analysis method. Integrating diverse biomolecular data increases the power to discover gene sets. MOGSA evaluates the contribution of biomolecules, samples and data sets to gene sets. The method and tutorial are available in the R/Bioconductor package mogsa. Gene-set analysis (GSA) summarizes individual molecular measurements to more interpretable pathways or gene-sets and has become an indispensable step in the interpretation of large-scale omics data. However, GSA methods are limited to the analysis of single omics data. Here, we introduce a new computation method termed multi-omics gene-set analysis (MOGSA), a multivariate single sample gene-set analysis method that integrates multiple experimental and molecular data types measured over the same set of samples. The method learns a low dimensional representation of most variant correlated features (genes, proteins, etc.) across multiple omics data sets, transforms the features onto the same scale and calculates an integrated gene-set score from the most informative features in each data type. MOGSA does not require filtering data to the intersection of features (gene IDs), therefore, all molecular features, including those that lack annotation may be included in the analysis. Using simulated data, we demonstrate that integrating multiple diverse sources of molecular data increases the power to discover subtle changes in gene-sets and may reduce the impact of unreliable information in any single data type. Using real experimental data, we demonstrate three use-cases of MOGSA. First, we show how to remove a source of noise (technical or biological) in integrative MOGSA of NCI60 transcriptome and proteome data. Second, we apply MOGSA to discover similarities and differences in mRNA, protein and phosphorylation profiles of a small study of stem cell lines and assess the influence of each data type or feature on the total gene-set score. Finally, we apply MOGSA to cluster analysis and show that three molecular subtypes are robustly discovered when copy number variation and mRNA data of 308 bladder cancers from The Cancer Genome Atlas are integrated using MOGSA. MOGSA is available in the Bioconductor R package \\u201cmogsa.\\u201d\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1632&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2630bb8cc2ee2c26bb421384cadac4c91175aceb\",\n",
      "        \"title\": \"across human immune cells\",\n",
      "        \"abstract\": \"Cells experience intrinsic and extrinsic pressures that affect their proclivity to expand and persist in vivo . In congenital disorders caused by loss-of-function mutations in mitochondrial DNA (mtDNA), metabolic vulnerabilities may result in cell-type specific phenotypes and depletion of pathogenic alleles, contributing to purifying selection. However, the impact of pathogenic mtDNA mutations on the cellular hematopoietic landscape is not well understood. Here, we establish a multi-omics approach to quantify deletions in mtDNA alongside cell state features in single cells derived from Pearson syndrome patients. We resolve the interdependence between pathogenic mtDNA and lineage, including purifying selection against deletions in effector/memory CD8 T-cell populations and recent thymic emigrants and dynamics in other hematopoietic populations. Our mapping of lineage-specific purifying selection dynamics in primary cells from patients carrying pathogenic heteroplasmy provides a new perspective on recurrent clinical phenotypes in mitochondrial disorders, including cancer and infection, with potential broader relevance to age-related immune dysfunction.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1633&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0cccaed25ed7e5528ea8469328c205c4b0db06b7\",\n",
      "        \"title\": \"Repository of the Max Delbr\\u00fcck Center for Molecular Medicine (MDC) in the Helmholtz Association\",\n",
      "        \"abstract\": \"Fiskin et al. (2021) developed a \\u201c mul\\ufffd -omics\\u201d approach that integrates phage-displayed single-domain an\\ufffdbodies (\\u201cnanobodies\\u201d) with the assay for transposase-accessible chroma\\ufffdn (PHAGE-ATAC) to simultaneously determine protein expression, chroma\\ufffdn accessibility\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1634&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a942cdbd240e05633d86626e30ce9e966a6c581f\",\n",
      "        \"title\": \"Into the Microscale: Low-input sequencing technologies and applications in medicine\",\n",
      "        \"abstract\": \"Sequencing technology has undergone major breakthroughs over time and has become indispensable in biological and medical research. Advances in protocols and analysis algorithms allow the analysis of small sample inputs, enabling the characterization of complex networks that govern biological processes in physiology and pathology, driving the development of precision medicine and targeted therapy. In this review, we comprehensively summarize low-input sequencing technologies that include methods for profiling the genome, epigenome, transcriptome, translatome, RNA modification, RNA-RNA interaction, RNA-protein interaction, and multi-omics. The key steps and innovations in different protocols are highlighted, and their advantages, limitations, and scope of application are described. With a focus on the impact of low-input sequencing technologies in biomedical fields, including embryonic development, prenatal and preimplantation genetic testing, oncology and immunology, and pandemic research, we discuss their potential to improve embryo implantation rates, prevent birth defects, develop prospective therapies, and predict prognosis. Lastly, we discuss current limitations and future prospects, providing new insights for medical research.\\n\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1635&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4db00c2db8afe41f4f3f92fb728aad817c153fe8\",\n",
      "        \"title\": \"An Unexpected Single-step Glycosylation Enables the Construction of Antibody-Biomacromolecule Conjugates as Therapeutics\",\n",
      "        \"abstract\": \"Glycoengineering has been demonstrated to be a powerful tool to construct site-specific antibody conjugates. However, with most glycoengineering strategies, several hours to days are needed to complete the reaction, and the payloads are limited to small molecules. Herein, we show that reducing the complexity of Fc glycan could dramatically boost the enzymatic glycoengineering of IgG molecules to achieve unexpectedly high efficiency and payload capacity. Notably, antibodies with the modified Fc glycan were successfully conjugated with biomacromolecules, e.g. , oligonucleotides and nanobodies, in one step by an engineered fucosyltransferase. Moreover, the conversion of this chemoenzymatic engineering could be finished in minutes, or up to hours, affording homogeneous products. The trisaccharide linker of these antibody conjugates exhibits excellent hydrophilicity and stability and reduced ADCC activity. Using this platform, we successfully synthesized an antibody-conjugate-based HER2/CD3 bispecific antibody and applied it to selectively destroy patient-derived cancer organoids by reactivating endogenous T cells inside the organoid. These results suggest that this bispecific antibody format has a high (Biolegend, Clone FN50, 310910), and CD25-Percp/Cy5.5 (Biolegend, Clone BC96, 302626) to assess CD69 and CD25 upregulation. Cells were acquired using an Agilent Novocyte Quanton and analyzed using FlowJo 10.5.3. Graphs were obtained using GraphPad Prism 8. This study was approved by the Ethics Committee of Nanjing Drum Tower Hospital (2021-394-01).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1636&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fa73c7381134ed7de0b27a441c495e27398f3195\",\n",
      "        \"title\": \"Cellular sources and targets of type I interferons that drive susceptibility to tuberculosis\",\n",
      "        \"abstract\": \"Mycobacterium tuberculosis ( Mtb ) causes 1.5 million deaths annually. Active tuberculosis correlates with a neutrophil-driven type I interferon (IFN) signature, but the cellular mechanisms underlying tuberculosis pathogenesis remain poorly understood. We found interstitial macrophages (IMs) and plasmacytoid dendritic cells (pDCs) are dominant producers of type I IFN during Mtb infection in mice and non-human primates, and pDCs localize near human Mtb granulomas. Depletion of pDCs reduces Mtb burdens, implicating pDCs in tuberculosis pathogenesis. During IFN-driven disease, we observe abundant DNA-containing neutrophil extracellular traps (NETs) known to activate pDCs. Single cell RNA-seq reveals type I IFNs act on IMs to impair their responses to IFN \\u03b3 , a cytokine critical for Mtb control. Cell type-specific disruption of the type I IFN receptor suggests IFNs act on IMs to inhibit Mtb control. We propose pDC-derived type I IFNs, driven by NETs, act on IMs to drive bacterial replication, further neutrophil recruitment, and active tuberculosis disease. were Data were pathology 8 lung samples and 8 node samples were analyzed. Each sample had previously culture Mycobacterium tuberculosis The samples were formalin-fixed and paraffin-embedded. Sections were cut and stained with anti-CD123 (6h6, Thermo Fisher Scientific), anti-CD303 (124B3.13, Dendritics), or hematoxylin and eosin 94,95 . Primary antibodies were detected by immunoperoxidase staining with the LSAB+ System and a standard DAB reaction following manufacturer\\u2019s instructions (DakoCytomation). Sections were counterstained with hematoxlyin prior to mounting and microscopy. pDCs were assessed in multiple each singe and CX3CR1 (SA011F11, BioLegend). All samples were also stained with a unique anti-mouse TotalSeq-A Hashtag antibody (1-6; BioLegend) to allow up to 6 populations to be multiplexed in a single lane on a 10X Genomics Chromium Next GEM Chip 96 . Enriched cells from infected mice were also stained with PE-labeled B220 (RA3-6B2, Tonbo), PE-labeled CD90.2 (30-H12, Tonbo), and BV785-labeled CD45.2 (104, BioLegend) anti-mouse antibodies. Enriched cells from na\\u00efve mice were stained with Pacific Blue-labeled B220 (RA3-6B2, BioLegend), Pacific Blue-labeled CD90.2 (53-2.1, BioLegend), PE-labeled F4/80 (BM8, Thermo Fisher Scientific), and BV786-labeled CD45.2 (104, BioLegend) anti-mouse antibodies. All post-enrichment antibody staining was performed on ice for 45 minutes in the presence of True-Stain Monocyte Blocker (BioLegend). Following staining, cells were resuspended in PBS with 2% Newborn Calf Serum and Sytox Blue Dead Cell Stain (Thermo Fisher Scientific). Cells were sort purified using a 100 \\u00b5m microfluidic sorting chip in a 4 laser SH-800 cell sorter (Sony) on the purity setting. The isolated populations from infected lungs were Mtb -infected cells and bystander myeloid cells. Macrophages and a mixture of neutrophils and monocytes were isolated from the na\\u00efve lungs and the macrophages were combined with neutrophil/monocyte mixture at a 1:2 ratio for better macrophage representation in the resulting dataset.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1637&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d2bda280bdbaa197829499d1de09f4ecfb364db8\",\n",
      "        \"title\": \"Single cell profiling of COVID-19 patients: an international data resource from multiple tissues\",\n",
      "        \"abstract\": \"underpinnings of biological processes. Early foundational work within the single-cell community, including the Human Cell Atlas, utilized published and unpublished data to characterize the putative target cells of SARS-CoV-2 sampled from diverse organs based on expression of the viral receptor ACE2 and associated entry factors TMPRSS2 and CTSL (Muus et al., 2020; Sungnak et al., 2020; Ziegler et al., 2020). This initial characterization of reference data provided an important foundation for framing infection and pathology in the airway as well as other organs. However, initial community analysis was limited to samples derived from uninfected donors and other previously-sampled disease indications. This report provides an overview of a single-cell data resource derived from samples from COVID-19 patients along with initial observations and guidance on data reuse and exploration.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1638&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b7f90392dd62db2f09473a4ae21e13894ccfa85\",\n",
      "        \"title\": \"A Single-Cell Molecular Atlas of White Adipose Tissue Shows Differences in Myeloid and Lymphoid Cell Polarization in Type 2 Diabetes and HIV Infection\",\n",
      "        \"abstract\": \"Subcutaneous white adipose tissue (scWAT) is a critical regulator of systemic metabolic homeostasis. HIV infection is accompanied by alterations in scWAT immune cells, which may contribute to the increased risk for cardiometabolic diseases in persons with HIV (PWH) and serve as a model to investigate cellular relationships. We generated the largest molecular atlas of scWAT in humans to date from 91 individuals using paired single-cell transcriptomic and proteogenomics to determine the scWAT immune profile in non-diabetic, pre-diabetic, and diabetic PWH, and HIV-negative diabetic persons. We identified a substantial increase in lipid-associated macrophages (LAMs) with glucose intolerance, and, in PWH only, CD4 + and CD8 + T effector memory cells were correlated with LAM proportion. Finally, we resolved adipogenic and fibrogenic cells at high resolution and found a strong association between LAMs and myofibroblasts. These results provide a high-resolution, multi-platform characterization of changes in scWAT composition that occur with metabolic disease and HIV. interquartile Categorical The whiskers are 1.5x the interquartile range. Abbreviations: ILC, innate lymphoid cells; mNK, mature natural killer; NK, natural killer; TCM, T central memory; TEM, T effector memory, TEMRA, T effector memory RA +\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1639&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dd1480c5429430e303540e42e59333cb18a116e7\",\n",
      "        \"title\": \"Unifying genomics and transcriptomics in single cells with ResolveOME amplification chemistry to illuminate oncogenic and drug resistance mechanisms\",\n",
      "        \"abstract\": \"Discovering genomic variation in the absence of information about transcriptional consequence of that variation or, conversely, a transcriptional signature without understanding underlying genomic contributions, hinders understanding of molecular mechanisms of disease. To assess this genomic and transcriptomic coordination, we developed a new chemistry and method, ResolveOME, to extract this information out of the individual cell. The workflow unifies template-switching full-transcript RNA-Seq chemistry and whole genome amplification (WGA), followed by affinity purification of first-strand cDNA and subsequent separation of the RNA/DNA fractions for sequencing library preparation. In the ResolveOME methodology we leverage the attributes of primary template-directed amplification (PTA) 1 to enable accurate assessment of single-nucleotide variation as a DNA feature\\u2014not achieved with existing workflows to assess DNA + RNA information in the same cell. We demonstrated the validity of the technique in the context of two major phenomena in oncology: tumor heterogeneity (leading to cancer progression) and treatment resistance. Material from a primary patient breast cancer and an acute myeloid leukemia (AML) cell line, MOLM-13, was used to highlight multiomic biomarker paradigms enabled by this chemistry. Performance of the PTA-enabled genome amplification was largely unaffected by addition of RNA enrichment, with control WGS results showing > 95% genome coverage, precision > 0.99 and allele drop out < 15%. In the RNA fraction of the chemistry, we were able to routinely retrieve full-length transcripts that demonstrate a ratio of 1 for 5\\u2019/3\\u2019 bias, with increased coverage of intronic regions and 5\\u2019 regions that are indicative of novel transcripts, showing strength of the template switching mechanism to capture isoform information with sparsity rates < 75%. We find remarkable cellular variability of revealed biomarkers at both in the genome and transcriptome despite employing a relatively small number of individual cells. In our primary patient sample of ductal carcinoma in situ (DCIS)/invasive ductal carcinoma (IDC) we observed oncogenic PIK3CA driver mutations and prototypical DCIS copy number alterations binned into heterogenous single-cell classes of genomic lesions. Within our quizartinib-treated MOLM-13 cells, we identified multiple potential mechanisms of resistance within N841 and K841. These data suggest that FLT3 K841 existed as an extremely rare clone in the original MOLM-13 cell line which upon the selective pressure of quizartinib was enriched to domination of the with RNA sequencing in single cells and show its utility by cataloging putative regulatory SNVs affecting gene expression. in between washes and passed through a 35 micron filter prior to loading for FACS. Singlet (FSC-A / FSC-H, BSC-A / BSC-W) and live-cell (Calcein AM) gating was defined followed by daughter EpCAM high and EpCAM low gates. EpCAM High and Low cells were sorted into the same 96 well plates as described above for to minimize potential batch effects of downstream genomic/transcriptomic amplification. analyses coming from the genomic fraction of ResolveOME, we leveraged an internal analytics pipeline modified from Sentieon driver-based tools. Initial FASTQ pairs were trimmed against low quality and library artifacts using fastp 45 (v0.20.1) Alignment was performed using BWA (Sentieon-202112), followed by deduplication (locus_collector v202112 / dedup v202112 ) of identically-aligned reads. Alignment-based\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1640&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3e43e1054b583e0105f54e1de54edb943b4aaef9\",\n",
      "        \"title\": \"The ResolveOME Platform for Comprehensive Analysis of Multi-omic Layers of Single-cell Biology\",\n",
      "        \"abstract\": \"The central dogma of molecular biology states that the genome of every single cell contains the complete set of instructions for all biological life. The omic layers, comprised of the entire genome, transcriptome and proteome, synchronize to translate and decode this genetic code to dictate cellular fate. Normal and pathologic cellular development are dependent on the fidelity of this transformative process. They are dependent on the OME, or the sum of ALL of the parts.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1641&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2883ee72bcdac0102980ec7806cb2841c62d8d54\",\n",
      "        \"title\": \"Biosensors for single-cell proteomic characterization\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1642&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ac338060cd503cf172e6e9c408967069980da3c3\",\n",
      "        \"title\": \"Spatial mapping of the tumor immune microenvironment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1643&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e7e73c389e0f94b525fca03fcc0282ac08cfda8b\",\n",
      "        \"title\": \"Analysing Genetic Programs of Cell Differentiation to Study Cardiac Cell Diversification\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1644&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d8023efeac94549a6cbbc8b386a1845f969c6676\",\n",
      "        \"title\": \"Microfluidics technology: past, present, and future prospects for biomarker diagnostics\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1645&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"38b09c250ef4a32843ddaefab6f1d9ccdb2072b1\",\n",
      "        \"title\": \"Gene Editing Through CRISPR-Based Technology\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1646&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7cbe40e7dc6bf80edf35ccd851817d6bfe12f15d\",\n",
      "        \"title\": \"Comprehensive analysis of long non-coding RNAs in breast cancer using topic modeling.\",\n",
      "        \"abstract\": \"Single-cell RNA sequencing is a powerful tool to explore cancer heterogeneity. However, the expression of lncRNAs in single cells is still to be studied extensively and methods to deal with the sparsity of this type of data are lacking. Here, we propose a topic modeling approach to investigate the transcriptional heterogeneity of luminal and triple negative breast cancer cells using patient-derived xenograft models of acquired resistance to chemotherapy and targeted therapy. We show that using an integrative clustering that combines the information coming from mRNAs and lncRNAs treated as disjoint omic layers greatly improves the accuracy of cell classification. Topics associated with specific breast cancer subpopulations show a clear enrichment for pathways involved in subtyping and progression of breast cancer and to sets of lncRNA encoded in the open chromatin regions of breast cancer cell lines. We identified lncRNAs strongly associated with cell clusters already well known in the literature, such as MALAT1 and NEAT1, and highlighted some others that may be clinically relevant. A larger scale study based on Smart-seq-total technology (8) has assayed a broad spectrum of mRNAs and lncRNAs from single cells and showed that the lncRNA content of cells significantly differs across cell types and dynamically changes throughout cellular processes such as cell cycle and cell differentiation. Another large-scale study, combining bulk tissue RNA-seq and scRNA-seq to deeply profile lncRNA expression during neocortical development, found that many lncRNAs are specific to distinct cell types and abundantly expressed in individual cells (9), thus supporting that cell type-specific expression of lncRNAs contributes to the low levels of lncRNAs observed in tissues.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1647&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b923fadc62f23cf84b152211fdb11e33d352b3b4\",\n",
      "        \"title\": \"Single-cell multi-omics analysis reveals IFN-driven alterations in T lymphocytes and natural killer cells in systemic lupus erythematosus [version 1; peer review: 1 approved, 2 approved with reservations]\",\n",
      "        \"abstract\": \"Background: The characterisation of the peripheral immune system in the autoimmune disease systemic lupus erythematosus (SLE) at the single-cell level has been limited by the reduced sensitivity of current whole-transcriptomic technologies. Here we employ a targeted single-cell multi-omics approach, combining protein and mRNA quantification, to generate a high-resolution map of the T lymphocyte and natural killer (NK) cell populations in blood from SLE patients. Methods: We designed a custom panel to quantify the transcription of 534 genes in parallel with the expression of 51 surface protein targets using the BD Rhapsody AbSeq single-cell system. We applied this technology to profile 20,656 T and NK cells isolated from peripheral blood from an SLE patient with a type I interferon (IFN)-induced gene expression signature (IFN hi ), and an age- and sex- matched IFN low SLE patient and healthy donor. Results: We confirmed the presence of a rare cytotoxic CD4 + T cell (CTL) subset, which was exclusively present in the IFN hi patient. Furthermore, we identified additional alterations consistent with increased immune activation in this patient, most notably a shift towards terminally differentiated CD57 + CD8 + T cell and CD16 + NK dim phenotypes, and the Consequently, we were able to find evidence for novel cellular peripheral biomarkers of SLE disease activity, including a subpopulation of CD57 + CD4 + CTLs. Abbreviations CTL, cytotoxic T lymphocyte; FACS, fluorescence-activated cell sorting; IFN, type I interferon; IFN hi , SLE patient with a detectable type I interferon-inducible gene expression signature in blood; ISG, type I interferon-stimulated gene; NK, natural killer; PBMC, peripheral blood mononuclear cells; PMA, phorbol 12-myristate 13-acetate; scRNA-seq, single-cell RNA-sequencing; SLE, systemic lupus erythematosus; T1D, type 1 diabetes; Tcm, memory T cell; effector T cell; effector T cell; TEMRA, terminally differentiated effector memory T Tfh, follicular helper T This is a nice piece of work using RNA and protein profiling of T and NK cells in 2 SLA patients with high and low interferon signatures and a non-SLA control patient. The authors accepted that the patient numbers are low, and that this is essentially a proof-of-concept study. There is certainly merit in increasing the patient numbers to validate this work, but this does not detract from the proof-of-concept findings of the current analysis. My main comment relates to the disease state of the IFNhi patient and the implications for biomarker discovery. Does the IFN signature relate to current SLA disease activity or to future prognosis? The context here is that there are established clinical scoring criteria for scoring disease activity in SLE (e.g., Thanou et al ., 2019 1 ), therefore if the findings of this study (ideally replicated in a larger cohort) apply to current disease state, then the question arises as to whether it adds anything more to what is obtained through simple clinical scoring. These are questions that can be explored through larger cohort analysis. If however the multi-omics analysis brings additional information about prognosis, this would be a more intriguing outcome of this analysis. Minor comments relate to the length of the discussion which can be more focused. This study describes the profiling of immune cell subpopulations in SLE, using a combined transcriptomic and proteomic methodology, down to the single-cell level. The main weakness of the study is the very limited number of subjects, which dampens the enthusiasm of the reviewer. Conclusions are drawn on 1 donor per study population (1 patient IFNhi, 1 patient IFNlo, 1 HD, with only limited clinical characteristics provided). It would have been highly recommended to validate the findings obtained by the transcriptomic/Proteomic analyses in additional samples from IFNhi and IFNlo SLE patients, by a flow cytometry based methodology. Moreover, while the authors repeatedly stress the fact that they are utilizing a novel method combining protein and mRNA quantification, this type of analysis has also been largely used by others, eg., CITE-seq, on unsorted single immune cells, with good resolution and identification of discrete, rare immune cell states (such as CD4 CTLs). Finally, while the inclusion of a \\\"stimulated\\\" condition appears very interesting to the reviewer, the rationale to use PMA/Ionomycin instead of aCD3/CD28 activation (at least for T cell subsets) is unclear. This would have provided a more physiological activation, that would better reflect the in vivo events occurring in patients. This is an interesting and detailed study of the immune cell features in patients with systemic lupus erythematosus (SLE). The authors have shown the strength of combined transcriptomic and proteomic profiling in single-cell analyses. The findings suggest that standard lymphocyte subsets, such as CD4 and NK cells, are in fact very complex and that subsets of these cells, even rare ones, have functional features that likely contribute to disease pathogenesis. Strengths of this work include the detailed analyses, the very clearly presented data in multiple figures, and the fact that the proteomic results reinforce functional implications of the various phenotypes defined by flow cytometry. The major weakness, which is discussed by the authors, is that this is a very limited analysis in terms of the patients sampled: one with high IFN signature, one with low IFN signature, and one healthy subject. The other available data on the two patients, levels of IL2 and circulating sCD25 are consistent with active disease. However, the IFN signature genes were not elevated as expected in the IFNhi individual. The explanation offered is that the IFN levels were determined at a separate time, prior to the sample taken for this study. This might imply that the cellular changes are more prolonged than the signature expression, but in the absence of longitudinal data, that is speculative. In addition, clinical features of the patients to confirm active disease are not described. The results will require further validation in larger patient samples, but the data presented suggest\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1648&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"31d134b6e344b4b51a36aff22c75e3a5357fe20b\",\n",
      "        \"title\": \"Aberrant extrafollicular B cells, immune dysfunction, myeloid inflammation and MyD88-mutant progenitors precede malignancy in Waldenstrom macroglobulinemia\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1649&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"0813e6068b694862db72e4a7459bbf628d3f85fd\",\n",
      "        \"title\": \"High-Dimensional Analysis of Circulating and Tissue-Derived Myeloid-Derived Suppressor Cells from Patients with Glioblastoma.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1650&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4ccfac957d6660d8a501174e0eb311c1a4896b4c\",\n",
      "        \"title\": \"Infection Diverse Mucosal-Associated Invariant TCR Usage in HIV\",\n",
      "        \"abstract\": \"Mucosal-associated invariant T (MAIT) cells are innate-like T cells that specifically target bacterial metabolites but are also identified as innate-like sensors of viral infection. Individuals with chronic HIV-1 infection have lower numbers of circulating MAIT cells compared with healthy individuals, yet the features of the MAIT TCR repertoire are not well known. We isolated and stimulated human PBMCs from healthy non-HIV\\u2013infected donors (HD), HIV-infected progressors on antiretroviral therapy, and HIV-infected elite controllers (EC). We sorted MAIT cells using flow cytometry and used a high-throughput sequencing method with bar coding to link the expression of TCRa, TCRb, and functional genes of interest at the single-cell level. We show differential patterns of MAIT TCR usage among the groups. We observed expansions of certain dominant MAIT clones in HIV-infected individuals upon Escherichia coli stimulation, which was not observed in clones of HD. We also found different patterns of CDR3 amino acid distributions among the three groups. Furthermore, we found blunted expression of phenotypic genes in HIV individuals; most notably, HD mounted a robust IFNG response to stimulation, whereas both HIV-infected progressors and EC did not. In conclusion, our study describes the diverse MAIT TCR repertoire of persons with chronic HIV-1 infection and suggest that MAIT clones of HIV-infected persons may be primed for expansion more than that of noninfected persons. Further studies are needed to examine the functional significance of unique MAIT cell TCR usage in EC. ImmunoHorizons, 2021, 5: 360\\u2013369.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1651&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d48080a604274fc9720eee1668e32231a1252d97\",\n",
      "        \"title\": \"An Introduction to Single-Cell RNA-Seq Analysis and its Applications\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1652&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3364caeff07b363b00399539dd624d94a83ebb13\",\n",
      "        \"title\": \"Transcriptomic and epigenetic changes after the phenotypic rescue of a Rett Syndrome mouse model\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1653&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"87eab02db49215dffaa76334922c520da1c08ef1\",\n",
      "        \"title\": \"Correlated expression of protein and RNA reveals a unique molecular signature in Th1-polarized cells\",\n",
      "        \"abstract\": \"Bulk RNA sequencing (RNA-Seq) has become a widely used tool for measuring gene expression, whereas flow cytometry is considered the gold standard for studying protein expression at a cellular level. Recent developments in single-cell sequencing technologies have combined the analysis of RNA and protein in a technique called cellular indexing of transcriptomes and epitopes by sequencing (CITE-Seq). This method uses antibodies conjugated to oligonucleotides, available as TotalSeq antibodies from BioLegend. These oligo-conjugated antibodies have been incorporated into an RNA-Seq workflow for simultaneously obtaining RNA and protein data using Illumina next-generation sequencing (NGS), described as bulk epitope and nucleic acid sequencing (BEN-Seq), rendering a complete proteogenomic profile of the sample. To demonstrate the utility of this approach, we performed CITE-Seq and bulk RNA and protein sequencing with 280 TotalSeq antibodies on T helper 1 (Th1)-polarized human peripheral blood mononuclear cells (PBMCs). We found a unique molecular profile that characterized Th1 cells with a strong correlation between both methods, demonstrating how protein analysis can be used to enrich bulk RNA-Seq data sets.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1654&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"be7b881fedfa08d3fd52cc6f32e25a46319cc826\",\n",
      "        \"title\": \"Single Cell Genomics for Tumor Heterogeneity.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1655&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8cbf3a0d0e158d9addbd46febeba52bc9f26d047\",\n",
      "        \"title\": \"Single-cell RNA sequencing in cardiovascular science\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1656&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a7b5b7516d026dac788930cc0879e4ddbc9511ce\",\n",
      "        \"title\": \"Single-Cell Gene Expression, Clonality, and Feature Barcoding of Melanoma Tumor-Infiltrating Lymphocytes.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1657&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"62b2ab770c0ad396909c63eed0db9a291ad69325\",\n",
      "        \"title\": \"Systems Immunology Approaches for Understanding of Primary Dendritic Cell Subpopulations in the Past, Present and Future\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1658&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4049ae8185de76dc35138a84a4b8cabef52785b2\",\n",
      "        \"title\": \"Next Generation Sequencing Methods: Pushing the Boundaries\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1659&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4345deb3745a251c17a48a0675d72a221ee9660\",\n",
      "        \"title\": \"Normalization of Single-Cell RNA-Seq Data.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1660&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4fb7a5a1ae20b0c70134f108d72bd2efaf5e1900\",\n",
      "        \"title\": \"Endocrinology of a Single Cell: Tools and Insights\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1661&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"fe90a705ff381c6b5e7e72c6bd718f6e354f6f13\",\n",
      "        \"title\": \"Single-Cell Analysis of Hematopoietic Stem Cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1662&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3a0905b8e1ba1f8642b35fdb2ba5ac1ccf2bc67a\",\n",
      "        \"title\": \"New Sensing Technologies: Microtas/NEMS/MEMS\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1663&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"39849c79799a65d651cc87dc4152a409e8731574\",\n",
      "        \"title\": \"Elucidation of host-virus surfaceome interactions using spatial proteotyping.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1664&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"d0b2738b4b5a68ac130829ebb5d6ef762ab0c117\",\n",
      "        \"title\": \"A broad introduction to RNA-Seq\",\n",
      "        \"abstract\": \"RNA-Seq, named as an abbreviation of \\\"RNA sequencing\\\" and sometimes spelled RNA-seq, RNAseq, or RNASeq, uses next-generation sequencing (NGS) to reveal the presence and quantity of ribonucleic acid (RNA) in a biological sample at a given moment.[1][2] RNA-Seq is used to analyze the continuously changing cellular transcriptome (Figure 1). Specifically, RNA-Seq facilitates the ability to look at alternative gene spliced transcripts, post-transcriptional modifications, gene fusion, mutations/single nucleotide polymorphisms (SNPs) and changes in gene expression over time, or differences in gene expression in different groups or treatments.[3] In addition to messenger RNA (mRNA) transcripts, RNA-Seq can look at different populations of RNA to include total RNA, small RNA, such as microRNA (miRNA), transfer RNA (tRNA), and ribosomal profiling.[4] RNA-Seq can also be used to determine exon/intron boundaries and verify or amend previously annotated 5' and 3' gene boundaries. Recent advances in RNA-Seq include single cell sequencing, in situ sequencing of fixed tissue, and native RNA molecule sequencing with single-molecule real-time sequencing.[5] Prior to RNA-Seq, gene expression studies were done with hybridization-based microarrays. Issues with microarrays include cross-hybridization artifacts, poor quantification of lowly and highly expressed genes, and needing to know the sequence a priori.[6] Because of these technical issues, transcriptomics transitioned to sequencing-based methods. These progressed from Sanger sequencing of Expressed Sequence Tag libraries, to chemical tag-based methods (e.g., serial analysis of gene expression), and finally to the current technology, next-gen sequencing of complementary DNA ( cDNA), notably RNA-Seq.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1665&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c4641299978beb32bf20e7100603fb0a3ce9f8ce\",\n",
      "        \"title\": \"Single-cell RNA sequencing reveals a novel cell type and immunotherapeutic 1 targets in papillary thyroid cancer\",\n",
      "        \"abstract\": \"30 Papillary thyroid cancer (PTC) is the most common thyroid malignancy. Although 31 PTC usually has a favorable prognosis, some aggressive PTC subtypes and lymph 32 node (LN) metastasis contribute to high rates of recurrence and poor clinical 33 outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from 15 34 samples, including primary tumors of PTC, metastatic LNs, and paracancerous tissues. 35 After quality filtering, 28,205 cells were detected. Of these, 13,390 cells originated 36 from 7 tumor tissues, 2,869 cells from 2 metastatic LNs, and 11,945 cells from 6 37 paracancerous tissues. The increase in the proportion of CD4 + Tregs may be a key 38 factor responsible for the immunosuppressive property of PTC. A novel cell type was 39 identified, named Protective EGR1 + CD4 + T cell, which might be antagonistic to the 40 CD4 + Tregs and inhibit the formation of the immunosuppressive microenvironment 41 and tumor immune evasion. Inhibitory checkpoints TIGIT and CD96 were found to be 42 better targets than PD-1 for immune therapy in PTC patients with LN metastasis. For 43 PTC patients without LN metastasis, however, PD-1, TIGIT, and CD96 could be 44 suitable targets of immunotherapy. These findings would contribute to the further 45 understanding of molecular mechanisms resulting in occurrence and development of 46 PTC, and provide a theoretical rationale for targeted therapy and immunotherapy. 47 51 metastasis than PDCD1 (PD-1). These findings would contribute to the further understanding of molecular mechanisms resulting in occurrence and development of PTC, and provide a theoretical rationale for targeted therapy and immunotherapy.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1666&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f91eae4b7c5a392a1627a427960c751ee0768c73\",\n",
      "        \"title\": \"Transcription factors and gene regulatory elements in the human malaria parasite. Plasmodium falciparum\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1667&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3590a3cb33c46cd3ebf4d252858b01036c7da675\",\n",
      "        \"title\": \"Circulating Tumor Cells in Breast Cancer Metastatic Disease\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1668&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ce9c0e88ded33b7309626a1f0904a454e0456492\",\n",
      "        \"title\": \"Advances in the Characterization of Circulating Tumor Cells in Metastatic Breast Cancer: Single Cell Analyses and Interactions, and Patient-Derived Models for Drug Testing.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1669&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f556fcbb2e602ce7c512da498b7aa171c60bc64d\",\n",
      "        \"title\": \"Simultaneous Measurement of Surface Proteins and Gene Expression from Single Cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1670&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"75826cc127cde993a704edc3e40f79bc012d33aa\",\n",
      "        \"title\": \"Methods for High-Dimensional Flow Cytometry Analysis of Human MAIT Cells in Tissues and Peripheral Blood.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1671&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2350cc758d0482e3dec58563a90f1967ca7f3038\",\n",
      "        \"title\": \"The Conjugation of Antibodies for the Simultaneous Detection of Surface Proteins and Transcriptome Analysis at a Single-Cell Level.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1672&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"da037fcfa241dffbaddca6f1cfe7c1555ff8bc04\",\n",
      "        \"title\": \"Development of Proximity Ligation Assay variants for single cell proteomics\",\n",
      "        \"abstract\": \"________________________________________________________________________________ 5 ZUSAMMENFASSUNG _______________________________________________________________________ 7 CHAPTER 1: GENERAL INTRODUCTION __________________________________________________________ 9 1.1 QUANTITATIVE SINGLE CELL PROTEOMICS ________________________________________________________ 9 1.1.1 Advantages of Single Cell Analysis _____________________________________________________ 9 1.1.2 Importance of Protein Quantification _________________________________________________ 11 1.1.3 Proximity Ligation Assay ___________________________________________________________ 13 1.1.4 Methods for Quantitative Single Cell Proteomics ________________________________________ 16 1.2 HIGH DIMENSIONAL MULTIOMIC SINGLE CELL ANALYSIS _____________________________________________ 18 1.2.1 Single cell RNA-Seq is dominating the field _____________________________________________ 18 1.2.2 Proteomics Methods for high dimensional Single Cell Analysis _____________________________ 19 1.2.3 The Growth in other Omics Methods __________________________________________________ 21 1.2.4 PLA as an overlooked tool for Multiomics Single Cell Research _____________________________ 23 CHAPTER 2: DIGITAL QUANTIFICATION OF PROTEINS AND MRNA IN SINGLE MAMMALIAN CELLS. ________ 26 2.1 SUMMARY ____________________________________________________________________________ 26 2.2 INTRODUCTION _________________________________________________________________________ 26 2.3 DESIGN ______________________________________________________________________________ 27 2.4 RESULTS ______________________________________________________________________________ 29 2.5 DISCUSSION ___________________________________________________________________________ 39 2.6 EXPERIMENTAL PROCEDURES ________________________________________________________________ 42 CHAPTER 3: DEVELOPMENT OF PLA-SEQ OLIGOMER ARCHITECTURE, AND TROUBLESHOOTING OF NGS LIBRARY PREPARATION _____________________________________________________________________ 46 3.1 ABSTRACT ____________________________________________________________________________ 46 3.2 INTRODUCTION _________________________________________________________________________ 46 3.3 RESULTS ______________________________________________________________________________ 48 3.4 DISCUSSION ___________________________________________________________________________ 56 3.5 METHODS ____________________________________________________________________________ 58 CHAPTER 4: PROFILING OF IMMUNE CELL MEMBRANE PROTEIN INTERACTIONS USING NGS COUPLED PLA 61 4.1 ABSTRACT ____________________________________________________________________________ 61 4.2 INTRODUCTION _________________________________________________________________________ 61 4.3 RESULTS SECTION _______________________________________________________________________ 63 4.4 DISCUSSION ___________________________________________________________________________ 70 4.5 METHODS ____________________________________________________________________________ 73\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1673&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"8ce4d0821395da908810ef15031ea861c509a57c\",\n",
      "        \"title\": \"Functional screens identify vulnerabilities in acute leukemia\",\n",
      "        \"abstract\": \"Acute leukemia refers to a group of aggressive hematological malignancies of myeloid and lymphoid lineages termed acute myeloid leukemia (AML) and acute lymphoid leukemia (ALL) respectively. Acute leukemia is characterized by the presence of underlying genetic aberrations which alter the biology of normal hematopoietic cells resulting in the accumulation of immature abnormally differentiated blast cells. In this thesis, we have used advanced molecular techniques to identify vulnerabilities in acute leukemia. In paper I, we performed an in vivo CRISPR-Cas9 screen targeting cell surface genes in murine AML stem cells and showed that CXCR4 is a top cell surface regulator of AML cell growth and survival. Notably, loss of CXCR4 signaling in leukemia cells leads to oxidative stress and differentiation in vivo. In contrast, the CXCR4 ligand CXCL12 is dispensable for leukemia development in recipient mice. To identify key regulators of AML, in paper II, we performed an ex vivo cytokine screen on arrayed molecularly barcoded murine AML cells with a competitive in vivo read-out of their leukemia-initiating capacity. We identified TNFSF13 as a positive regulator of leukemia-initiating cells. We confirmed that TNFSF13 supports leukemia initiation under physiological conditions using Tnfsf13 mice. We further showed that TNFSF13 suppresses apoptosis and promotes AML cell proliferation in an NF-\\u03baB dependent manner. DUX4-rearranged BCP-ALL is a recently identified molecular subtype characterized by the expression of the IGHDUX4 fusion gene. With the aim of identifying biological dependencies of this subtype, in paper III, we performed a genome-wide CRISPR-Cas9 screen in the NALM6 cell line, driven by the IGH-DUX4 fusion gene, and two control cell lines. We showed that FNIP1, IRF4 and SYNCRIP are selectively important for the growth and survival of NALM6 cells and that their expression is under the control of the IGH-DUX4 fusion gene. While the deletion of FNIP1 led to the enrichment of transcriptional signatures associated with metabolic dysregulation, loss of IRF4 resulted in upregulation of genes involved in differentiation and apoptosis of NALM6 cells. Moreover, disruption of SYNCRIP caused downregulation of the TGF\\u03b2-SMAD signaling pathways in NALM6 cells. In paper IV, we explored the immune-mediated anti-leukemic activity of the cytokine interleukin 4 (IL4) in a murine AML model. Overexpression of IL4 in AML cells resulted in a strong anti-leukemic effect accompanied by an expansion of macrophages in the bone marrow and spleen of the recipient mice. Depletion of macrophages in vivo eliminated the antileukemic effect of IL4. In addition, IL4 directly activates murine macrophages resulting in enhanced phagocytosis of AML cells in vitro. Interestingly, IL4 also induced Stat6-dependent upregulation of CD47 in AML cells thereby inhibiting phagocytosis. Consistent with this finding, IL4 stimulation combined with CD47 blockade enhanced macrophage-mediated phagocytosis of AML cells. Taken together, the studies included in this thesis employed high-throughput functional screens using CRISPRCas9 and molecular barcoding techniques to identify key regulators of AML cells. These findings improve our understanding of the disease and may translate into the development of new therapies for acute leukemia.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1674&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cd8cd62b005a7497658565106b2146f561076189\",\n",
      "        \"title\": \"University of Groningen High-Resolution Transcriptomic and Proteomic Profiling of Heterogeneity of Brain-Derived Microglia in Multiple Sclerosis Miedema,\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1675&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"10c25a03a9c72ff76b4a64e59528d0b3aa75f8ac\",\n",
      "        \"title\": \"Translational interest of immune profiling\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1676&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7b69c31f3e70907352d27e26a579833d379b8681\",\n",
      "        \"title\": \"Pluripotent stem cell biology and engineering\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1677&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"2a2c0c6d29b3dc947a3b0b97d58149c103592914\",\n",
      "        \"title\": \"Single-cell genomic analysis of human cerebral organoids.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1678&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"69c2d7bbe781c1d14ce107030a20446e2e15f692\",\n",
      "        \"title\": \"Human neurogenesis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1679&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"cb70df8089c5108cae48c83fc78f4ca6b5185ec9\",\n",
      "        \"title\": \"Methods for Single-Cell Isolation and Preparation.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1680&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f73be6110c6d99745d92771004a6775d0128c625\",\n",
      "        \"title\": \"Advances in single-cell epigenomics of the immune system\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1681&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9c6a7d233e72222f8ba14ffe81efb99d6d79ea1c\",\n",
      "        \"title\": \"IBEX: A versatile multiplex optical imaging approach for deep phenotyping and spatial analysis of cells in complex tissues.\",\n",
      "        \"abstract\": \"The diverse composition of mammalian tissues poses challenges for understanding the 40 cell-cell interactions required for organ homeostasis and how spatial relationships are perturbed 41 during disease. Existing methods such as single-cell genomics, lacking a spatial context, and 42 traditional immunofluorescence, capturing only 2-6 molecular features, cannot resolve these 43 issues. Imaging technologies have been developed to address these problems, but each 44 possesses limitations that constrain widespread use. Here we report a new method that overcomes 45 major impediments to highly multi-plex tissue imaging. Iterative Bleaching Extends multi-pleXity 46 (IBEX) uses an iterative staining and chemical bleaching method to enable high resolution imaging 47 of >65 parameters in the same tissue section without physical degradation. IBEX can be employed 48 with various types of conventional microscopes and permits use of both commercially available and 49 user-generated antibodies in an \\u2018open\\u2019 system to allow easy adjustment of staining panels based 50 on ongoing marker discovery efforts. We show how IBEX can also be used with amplified staining 51 methods for imaging strongly fixed tissues with limited epitope retention and with oligonucleotide- 52 based staining, allowing potential cross-referencing between flow cytometry, Cellular Indexing of 53 Transcriptomes and Epitopes by Sequencing (CITE-Seq), and IBEX analysis of the same tissue. 54 To facilitate data processing, we provide an open source platform for automated registration of 55 iterative images. IBEX thus represents a technology that can be rapidly integrated into most current 56 laboratory workflows to achieve high content imaging to reveal the complex cellular landscape of 57 diverse organs and tissues. image tissue preparations and a method to facilitate direct integration of the imaging data with flow cytometry and sequencing technologies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1682&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"f2bb2379a5b18d49552f4dfb3e419143ec7288e3\",\n",
      "        \"title\": \"1 Robust single-cell discovery of RNA targets of RNA binding proteins and ribosomes 2 3\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1683&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"695310e698f160c0e19ec8ca0339f0a86b87fbdf\",\n",
      "        \"title\": \"Single-Cell Pro \\ufb01 ling Reveals Divergent, Globally Patterned Immune Responses in Murine Skin In \\ufb02 ammation\",\n",
      "        \"abstract\": \"Biolegend), anti-CD4 (1:100, GK1.5, Biolegend), anti-CD45 (1:100, 30-F11, Biolegend), anti-CD8\\u03b1 (1:100, 53-6.7, Biolegend), anti-F4/80 (1:25, BM8, Biolegend), anti-\\u03b3\\u03b4TCR (1:100, GL3, Biolegend), and anti-I-A/I-E (1:100, M5/114, BD bioscience)), then fixed, permeabilized, and stained for intracellular antigens with anti-CCL5 (1:100, 2E9/CCL5, Biolegend), anti-IL-17A (1:50, eBio17B7, eBioscience), anti-IL-22 (1:100, Poly5164, Biolegend), and anti-Foxp3 (1:50, FJK-16s, eBioscience). Data were acquired on an LSRFortessa Flow Cytometer (BD) and analyzed using FlowJo 10.0 software (Tree Star).\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1684&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ddb42e6e6f16408e6a77a31f9fe593c1d904c368\",\n",
      "        \"title\": \"Computational Methods for Single-Cell Data Analysis\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1685&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"dcb0b162145daa498f5f3b4724ae5489f8f9b8a1\",\n",
      "        \"title\": \"Single Molecule and Single Cell Sequencing\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1686&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6747eef7b2ecf5eb157e7daf0e97983418788e19\",\n",
      "        \"title\": \"Biological Insights from Geometry and Structure of Single-Cell Data\",\n",
      "        \"abstract\": \"Biological Insights from Geometry and Structure of Single-Cell Data\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1687&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"3144162acda8d62d074a685c10da8e32ff37d590\",\n",
      "        \"title\": \"Integrated Fluidic Circuits for Single-Cell Omics and Multi-omics Applications.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1688&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4ff91ac6bb0cf8c2043c9fa2048d5ce0484fcfee\",\n",
      "        \"title\": \"Quantitation of mRNA Transcripts and Proteins Using the BD Rhapsody\\u2122 Single-Cell Analysis System.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1689&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"7ffa940dd8ab7da8026c4d0df87d925879402f99\",\n",
      "        \"title\": \"A Bioinformatic Toolkit for Single-Cell mRNA Analysis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1690&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"63e57c8facadd93633f1a51cf8ad66e0bce07137\",\n",
      "        \"title\": \"Single-Cell RNA Sequencing with Drop-Seq.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1691&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"6de6e8a6d558adf9c31191f9735aa8296d93b17d\",\n",
      "        \"title\": \"Identification of Cell Types from Single-Cell Transcriptomic Data.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1692&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c29b9d36cc50b3e5d74c664fc0a11c84c3affc5a\",\n",
      "        \"title\": \"An Informative Approach to Single-Cell Sequencing Analysis.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1693&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5a1e9f0ce14ca6441263d8f78b0c8b51bec0fd8f\",\n",
      "        \"title\": \"Modern Techniques for DNA, RNA, and Protein Assessment\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1694&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"abc8a931c18a134cc5e6e9081f6d5e0e0efb9307\",\n",
      "        \"title\": \"Immunophenotyping of Tissue Samples Using Multicolor Flow Cytometry.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1695&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"9a80e44c83e84f9d4ca3bfdfbe79c531d06cc3ce\",\n",
      "        \"title\": \"Novel technologies for interrogating T cell recognition\",\n",
      "        \"abstract\": \"The studies presented in this thesis have all involved the development of novel technologies for interrogating different aspects of antigen-specific T cell recognition. The immune system\\u2019s potential for eliminating intruding pathogens or malignancies is largely governed by the ability of T cells to specifically recognize and respond to foreign molecular structures. The activity of T cells is guided by specific interactions of the surface expressed T cell receptor (TCR) with peptides presented in the context of MHC molecules on the surface of target cells. Hence, for broad protection of virtually any encountered pathogen or cellular malignancy, the TCR repertoire of each individual must be able to recognize and adapt to an extensively large and unpredictable range of foreign peptides, while avoiding immunopathology caused by autoreactivity. The scope of this PhD has been to develop novel technologies that enable us in describing peptide-MHC-TCR interactions from a broader and more detailed view, which aims at integrating the complexity of the range of peptide-MHCs potentially recognized by T cells, as well as the cross-recognition potential of the individual T cell. The implementation of molecular barcodes as labels to trace peptide-MHC-TCR interactions provide the foundation for all the research included in this thesis. In a first study, I show that unique DNA barcodes can be used to label specific peptide-MHC multimer reagents, and subsequently enable tracing of the antigen-specificities of T cells within a heterogeneous sample. With this strategy I show that it is possible to screen for more than 1,000 different peptideMHC specificities in parallel, which provides completely new possibilities for assessing immune recognition in an individualized manner. In a second study, I discuss the advantages of assessing T cell recognition at such complexities, as well as current shortcomings. In a third study I address the cross-recognition potential of individual T cells through establishment of a technology that assess the relative affinity of a TCR to several MHC multimers carrying related peptide. This in turn enabled the identification of essential molecular interaction-points of TCRs and identification of cross-recognized peptides. The fourth study comprises a protocol which should enable other researchers to understand details of the technologies reported in study one and three, and ultimately assist them in performing the same types of analyses. The data reported in the fifth study comprise an early finding that show the simultaneous identification of peptide-MHC specificity and TCR sequence of many single cells in parallel. This will be implemented in a highthroughput format and hereby facilitate the pairing of numerous peptide-MHC specificities with the corresponding TCR sequences, which in turn can provide a first step towards resolving the relationship between the TCR sequence and the peptide-MHC binding motif. Collectively, the studies reported in this thesis provide a foundation for others to develop new and personalized\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1696&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"73b7ed0dea65f3a0fe29fc62c561f0c1dd386a95\",\n",
      "        \"title\": \"Redefining Cellular Phenotyping: Comprehensive Characterization and Resolution of the Antigen-Specific T Cell Response\",\n",
      "        \"abstract\": \"Adaptive immunity requires the ability to respond to different types of antigens, activate pathways that eliminate specific pathogens, and develop immunological memory so that subsequent infections or disease can be quickly identified and removed. This complex process is mediated through the specific recognition of antigens by T and B cells, and the dynamics of the immune response are fueled by the genetic diversity of Tand B-cell receptors.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1697&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"58cca1326399e20e009f0cb5ce332cff441ed88a\",\n",
      "        \"title\": \"Single-Cell Omics: Deciphering Tumor Clonal Architecture\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1698&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"b856d2e7fea4abec9be0d1e6c41dab70bca7c473\",\n",
      "        \"title\": \"Priming and Maturation of the Human Fetal Mucosal Immunity Occurs Early in Fetal Development\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1699&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"97367a0cd1286c00013f65b551db8b2ab365e009\",\n",
      "        \"title\": \"Single-cell screening using microfluidic systems\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1700&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e18309caa9ae447605d0cbf63c9015396cfc7bf1\",\n",
      "        \"title\": \"Angiogenesis inhibition in non-small cell lung cancer: a critical appraisal, basic concepts and updates from American Society for Clinical Oncology 2019.\",\n",
      "        \"abstract\": \"PURPOSE OF REVIEW\\nRecently, the combination of antiangiogenic agents, chemotherapy and immunotherapy has shown synergistic anticancer effects in non-small cell lung cancer (NSCLC). The future for this approach appears bright in lung cancer treatment; however, many challenges remain to be overcome regarding its true potential, optimal sequence and timing of therapy, and safety profile. In this review, we will discuss the current status and future direction of antiangiogenic therapy for the treatment of NSCLC, and highlight emerging strategies, such as tumor vessel normalization (TVN).\\n\\n\\nRECENT FINDINGS\\nBevacizumab was the first antiangiogenic agent approved for the treatment of advanced NSCLC. Recently, the combination of chemotherapy/antiangiogenic therapy with immunotherapy showed high efficacy in first-line settings. A subgroup of patients with liver metastasis and driver mutation-addicted tumors benefited most, suggesting that the metastatic location, as well as the genetic background of the tumor, are key determinants for therapy responses.\\n\\n\\nSUMMARY\\nThe efficacy of antiangiogenic therapies in unselected patients is rather limited. The tumor microenvironment has appeared to be more complex and heterogeneous than previously assumed. Only a contextual rather than a cell-specific approach might provide valuable insights towards the clinical validation of combinational therapies.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1701&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e441a0f3a28a260f3fc2bb2c3db83f491418d6b1\",\n",
      "        \"title\": \"Single-Cell Omics in Personalized Medicine\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1702&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"5705ef783b13ac9e13b63baf27d7581b084a3082\",\n",
      "        \"title\": \"High-throughput discovery of cancer-targeting TCRs.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1703&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"a1d903ae6dda7c4a1586d19ccdf965a265859b60\",\n",
      "        \"title\": \"Chemical probes for spatially resolved measurement of active enzymes in single cells.\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1704&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"ff7d19762b634063fa90187658538ee6addd6d06\",\n",
      "        \"title\": \"Immunogenicity and Immune Silence in Human Cancer\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1705&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"4c8d3c73f7e35397dc196731d003e56a448f7229\",\n",
      "        \"title\": \"Spatially Resolved Gene Expression Analysis\",\n",
      "        \"abstract\": \"Spatially resolved transcriptomics has greatly expanded our knowledge of complex multicellular biological systems. To date, several technologies have been developed that combine gene expression dat ...\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1706&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"da6db7fe03407a9746134c42e33217555e41a71c\",\n",
      "        \"title\": \"Application of multi-omics in single cells 1\",\n",
      "        \"abstract\": \"In recent years, single cell assays have made exciting progresses, overcoming the issue of heterogeneity associated with bulk populations. The fast-developing sequencing methods now enable unbiased, high-throughput and highresolution view of the heterogeneity from individual cell within a population, in terms of its fate decisions, identity and function. The cell\\u2019s state is regulated at different levels, such as DNA, RNA and protein, by complex interplay of intrinsic molecules existing in the organism and extrinsic stimuli such as local environment. Comprehensive profiling of single cell requires a simultaneously dissection from different levels (multi-omics) to avoid incomplete information generated from single cell. In this short review, we first examine the whole genome amplification methods, and then survey the features of the single cell approaches for genome, epigenome, transcriptome, proteome and metabolome profiling. Finally, we briefly analyze advantages of multi-omics measurement from single cells as compared to separate measurement of each molecular type, and discuss opportunities and challenges of combining single cell multiomics information on resolving phenotype variants.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1707&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"81927e286bf7540d1c8977f367f5fda459b1ab18\",\n",
      "        \"title\": \"Multivariate Control of Tra nscript to Protein Variability in Single Mammalian Cells Graphical\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1708&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"e8e4376f02131c36980ae4089d9ef38af8db7c3e\",\n",
      "        \"title\": \"Lab on a Chip\",\n",
      "        \"abstract\": \"Microlasers integrated with biological systems have received tremendous attention for their intense light intensity and narrow linewidth recently, serving as a powerful tool for studying complex dynamics and interactions in scattered biological micro-environments. However, manipulation of microlasers with controllable motions and versatile functions remains elusive. Herein, we introduce the concept of motor-like microlasers formed by magnetic-doped liquid crystal droplets, in which the direction and velocity could be controlled by altering internal magnetic nanoparticles or external magnetic fields. Both translational and rotatory motions of the lasing resonator could be continually changed in real-time. Lasing-encoded motors carrying different functions and lasing wavelengths were also achieved. Finally, we demonstrate the potential of motor-like microlasers by functioning as a localized stimulation emission light source to stimulate or illuminate living cells, providing a novel approach for switching on/off light emissions and subcellular imaging. Laser emitting micromotors offer a facile system for precise manipulation of microlasers in biological fluids, providing new insight into the development of programmable on-chip laser devices and laser-emitting intelligent systems.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1709&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"167633d4603fde1577107c6be7b0cc03d6df2044\",\n",
      "        \"title\": \"Towards comprehensive cellular atlases\",\n",
      "        \"abstract\": \"With recent technological advancements in single-cell biology, many aspects of individual cells are characterized with unprecedented resolution and details. Cell types in human and model organisms are redefined, and multiple organ-wide atlases are proposed to integrate different types of data to provide a comprehensive view of biological systems at cellular resolution. Incorporating location information of cells in such atlases is crucial to understanding the structure and functions. Several spatially resolved transcriptomics technologies may serve this purpose, and in situ sequencing (ISS) is among the most powerful ones. ISS detects the expression of tens to hundreds of genes in situ, i.e. inside preserved cells and tissues. ISS is a targeted approach, using probes designed to identify specific transcripts. Its key advantages, as compared to other spatially resolved gene expression analysis methods, are high throughput, cellular resolution and tissue compatibility, making it a tool ideally suited for spatial cell mapping. The work included in this thesis aims to develop tools and methods for this application. In paper I, a network analysis tool was developed to analyze ISS and other spatially resolved data. The tool enables smooth visualization of large datasets and generates networks based on colocalization. It also includes functions to test statistical significance and resolve tissue heterogeneity. In paper II, we studied spatio-temporal patterns of immune response in tuberculosis granuloma by targeting immune markers with ISS. Using the tool developed in paper I together with other methods, we established an immune response time course at the granuloma sites and found histologically different granulomas based on transcriptional information. The paper demonstrated that ISS can robustly detect transcripts in formalin-fixed paraffin-embedded tissues across biological samples and reveal biologically relevant structures. In paper III, we developed probabilistic cell typing by in situ sequencing (pciSeq), a method to spatially map cell types defined by single-cell RNA-sequencing. pciSeq is an integrated pipeline that includes gene selection, image analysis, barcode calling and cell type calling. We mapped closely related interneuron cell types of the mouse hippocampal CA1 region in 14 coronal sections and validated the results against ground truth. In paper IV, we investigated the quantification bias of ISS resulting from the probe target selection. We developed a method to sequence in situ synthesized cDNA and found that the read coverage of in situ cDNA library reflected ISS counts more closely than conventional RNA sequencing, making it possible, to some extent, to predict a probe\\u2019s performance and guide the probe design. Taken together, the developments described in this thesis comprise several tools that make ISS suitable for building cellular atlases via large-scale spatial mapping.\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1710&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": \"c8c9d1039797ebaea731ec63aff2597f1f59ff3f\",\n",
      "        \"title\": \"Title : Large field of view-spatially resolved transcriptomics at nanoscale resolution Short title : DNA nanoball stereo-sequencing\",\n",
      "        \"abstract\": \"High-throughput profiling of in situ gene expression represents a major advance towards the systematic understanding of tissue complexity. Applied with enough 3 capture area and high sample throughput it will help to define the spatio-temporal dynamics of gene expression in tissues and organisms. Yet, current technologies have considerable bottlenecks that limit widespread application. Here, we have combined DNA nanoball (DNB) patterned array chips and in situ RNA capture to develop Stereo-seq (Spatio-Temporal Enhanced REsolution Omics-sequencing). This approach allows high sample throughput transcriptomic profiling of histological sections at unprecedented (nanoscale) resolution with areas expandable to centimeter scale, high sensitivity and homogenous capture rate. As proof of principle, we applied Stereo-seq to the adult mouse brain and sagittal sections of E11.5 and E16.5 mouse embryos. Thanks to its unique features and amenability to additional modifications, Stereo-seq can pave the way for the systematic spatially resolved-omics characterization of tissues and organisms. MAIN TEXT Single-cell RNA-sequencing (scRNA-seq) technologies have unveiled an unexpectedly large degree of heterogeneity in cell types and cell states among tissues (1-5). Because multicellular organization and cell-cell interactions determine individual cell function, spatially resolved transcriptomic information is necessary to fully understand tissue complexity in normal physiology or under perturbation. This is best illustrated in embryonic development, where the spatio-temporal coordination of cells and signaling cues determine the dynamics of self-organized patterning, but in fact applies to any tissue and biological process. For example, spatially resolved transcriptomic information of the complex interactions between cancer cells, immune cells and stromal cells in the tumor niche (6, 7) could be useful to improve anticancer therapies. Similarly, revealing the dynamic changes of gene expression responsible for the spreading pattern of certain neurodegenerative diseases (8, 9) could help prevent this process. A number of remarkable methodologies have been recently developed that enable the high-throughput transcriptomic profiling of in situ gene expression. These techniques can be grouped into three main categories depending on the RNA 4 capture approach: a) physical segmentation using laser microdissection, b) pre-designed targeted probes (for either single-molecule hybridization or amplification and in situ decoding) or c) DNA-barcoded probes. Examples of the former approach include LCM-seq (laser capture microscopy coupled with full-length mRNA-sequencing) (10) and GEO-seq (geographical position sequencing) (11), which are highly operable but have low capture throughput, limited resolution (center-to-center distance of each array component) and the accuracy of spatial information is subjected to experimental variability. Examples of techniques based on pre-designed probes are MERFISH (multiplexed error-robust FISH) (12) and seqFISH (sequential fluorescence in situ hybridization) (13) for hybridization-based methods, and FISSEQ (fluorescent in situ RNA sequencing) (14) and STARmap (spatially-resolved transcript amplicon readout mapping) (15) for in situ decoding. General caveats of these techniques are that they rely on a finite number of sequences, can only be practically applied to a small field of view and require complex instrumentation. As for the latter group, Slide-seq (16), spatial transcriptomics (ST, Visium) (17) and their optimized versions Slide-seqV2 and HDST (high-definition spatial transcriptome) (18, 19), respectively, use barcoded arrays and have attracted significant attention. Similarly, DBiT-seq (deterministic barcoding in tissue for spatial omics sequencing) uses microfluidic channels for delivering barcoded probes directly into tissues and can simultaneously capture transcripts and selected protein targets (20). Although unbiased in the nature of captured transcripts, barcoded probe-based techniques suffer from an inadequate balance between resolution, average numbers of gene/transcript per bin and capture rate across multiple bins. Besides, as for hybridization-based methods, the measured areas are small (e.g., Visium 42.5 mm2, Slide-seq/Slide-seqV2 7 mm2, HDST 13.68 mm2 and DBiT-seq 25 mm2) (17-20) and face a major bottleneck of scalability. Increasing the resolution is important because it can help define transcriptomic gradients within tissues more accurately and allow detection of the subcellular localization of transcripts. This could also reduce problems associated with RNA diffusion, capture rate or disturbed patterns caused by unmeasured inter-bin space. Likewise, enlarging the capture area could allow the 5 measurement of whole mammalian embryos, human organs, or model organisms or the simultaneous assessment of multiple smaller samples (e.g., biopsies) on the same chip, avoiding the batch effect problem caused by separate runs and facilitating automation. Our aim was to develop a spatially resolved transcriptomic technology that combines billions of tightly packed barcodes in known positions with large-captured areas, high capture rate and high reproducibility. We deposited DNA nanoball (DNB) containing random barcoded sequences onto a modified silicon surface (chip) photolithographically etched with a grid-patterned array of spots where the DNB are efficiently docked in (21), each spot being approximately 220 nm in diameter and with a center-to-center distance of 500 or 715 nm (Fig. 1, step 1). The use of random barcode-labeled DNB generated by rolling circle amplification allows a much larger spatial barcode pool size (425) compared with bead-based approaches and at the same time maintains sequence fidelity. The array was then microphotographed, incubated with primers and sequenced to obtain the data matrix containing the coordinate identity (CID) of every etched DNB (Fig. 1, step 2). Then, molecular identifiers (MID) and polyT sequence-containing oligonucleotides were ligated on each spot through hybridization with the CID (Fig. 1, step 3). This DNB-based strategy enables the generation of large chips containing barcoded probes at a much higher density than any other previous method. The next step consisted in the capture of tissue polyA-tailed RNA, which was achieved in situ by loading fresh-nitrogen frozen tissue sections onto the chip surface, followed by fixation, permeabilization and finally reverse transcription and amplification (Fig. 1, step 4). Amplified cDNAs were collected, used as template for library preparation and sequenced together with the CID (Fig. 1, step 5). Computational analysis of the sequencing data allows spatially resolved transcriptomics with a resolution of 500 or 715 nm and minimal inter-bin space (Fig. 1, step 6), resulting in a significantly higher number of spots per 100 \\u03bcm2 (average size of a mammalian single cell) than any other available method (Fig. 2a). Currently, we have used Stereo-seq chips of 50, 100 and 200 mm2 in tissues like the mouse olfactory bulb (~11.7 mm2), half mouse brain (~26.7 mm2) or a whole E11.5 6 (~10.5 mm2) and E16.5 (~76.9 mm2) mouse embryos (fig. S1a). Notably, we have also developed DNB patterned array chips of up to 42.25 cm2 for potential application to large tissues such as whole human brain tissue sections (fig. S1a, b). We termed our approach Stereo-seq (Spatio-Temporal Enhanced REsolution Omics-sequencing), We first profiled the mouse olfactory bulb with Stereo-seq because it is a widely used model tissue for spatially resolved transcriptomic approaches (18, 19). We used patterned Stereo-seq chips (715 nm resolution) in combination with DAPI staining, which confirmed that the spatial transcriptomic information reflects the cellular structure of the tissue (Fig. 2b). Stereo-seq captured numbers ranging on average from 9,342 genes and 34,435 transcripts per 143 \\uf06dm bin (bin 200, 200 \\u00d7 200 DNB) to 275 genes and 346 transcripts per 14 \\uf06dm bin (bin 20, 20 \\u00d7 20 DNB, equivalent to ~1 cell) (Fig. 2c). Our reanalysis of previously reported datasets showed that Visium captures an average of 5,735 genes and 25,598 transcripts at 100 \\uf06dm resolution (mouse brain), Slide-seq 37 genes and 45 transcripts at 10 \\uf06dm resolution (hippocampus) (16), Slide-seqV2 426 genes and 494 transcripts at 10 \\uf06dm resolution (hippocampus) (18), and DBiT-seq 3,301 genes and 7,603 transcripts at 100 \\uf06dm resolution (E10 mouse embryo) and 1,801 genes and 3,572 transcripts at 20 \\uf06dm resolution (E10 mouse embryo eye/brain) (Fig. 2c). Gene and transcript capture of Stereo-seq per 2 \\uf06dm-diameter bin (bin 3, 3 \\u00d7 3 DNB, subcellular resolution) was substantially higher than HDST (mouse olfactory bulb; 2 genes and 2 transcripts at 2 \\uf06dm resolution) (19) (Fig. 2d), which is so far the only sequencing-based spatially resolved transcriptomic technology with subcellular resolution. We then performed unsupervised clustering to computationally reconstruct the spatial identity of the olfactory bulb profiled with Stereo-seq. We observed clustering consistent with known anatomical structures including the outer plexiform layer (OPL), granular cell zone deep (GCL-D), granular cell layer internal (GCL-I), internal plexiform layer (IPL), subependymal zone (SEZ), olfactory nerve layer (ONL), glomerular layer (GL) and granular cell layer externa (GCL-E) (Fig. 2e). This also identified two distinct sublayers within the GL and five in the ONL, suggesting potential sub-regional differences. The consistency between biological replicates using two adjacent sections 7 of the mouse olfactory bulb profiled with Stereo-seq was high (fig. S1c-f). In addition, there was also high correlation between similar anatomical areas in each of the olfactory bulb halves from a single section (fig. S2a). Analysis of reported mouse olfactory bulb datasets obtained with HDST or Slide-seqV2 did not reflect tissue organi\"\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1711&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": null,\n",
      "        \"title\": \"mole-rat hematlas by employing cross-reactive antibodies for fluorescence-activated cell sorting\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n",
      "https://api.semanticscholar.org/graph/v1/paper/e6ac2e5fdeb3f2d94efeeee5639c06cba2248f4e/citations?fields=abstract,title&offset=1712&limit=1\n",
      "{\n",
      "    \"citingPaper\": {\n",
      "        \"paperId\": null,\n",
      "        \"title\": \"The Transcriptome of SH \\u2010 SY 5 Y at Single \\u2010 Cell Resolution :\",\n",
      "        \"abstract\": null\n",
      "    }\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "# Getting ALL the citations\n",
    "i = 0\n",
    "total_citations = paper_details['citationCount']\n",
    "while (i < total_citations):\n",
    "    try:\n",
    "        query = 'https://api.semanticscholar.org/graph/v1/paper/' + paper_details['paperId'] + '/' + 'citations?' + f'fields=abstract,title&offset={i}&limit=1'\n",
    "        print(query)\n",
    "        cites = requests.get(query)\n",
    "        \n",
    "        title = cites.json()['data'][0]['citingPaper']['title'].replace('/', '') + '.json'\n",
    "        json_data = json.dumps(cites.json()['data'][0], indent=4)\n",
    "        \n",
    "        # Write the JSON string to a file\n",
    "        outpath = os.path.join(out_dir, title)\n",
    "        with open(outpath, 'w') as file:\n",
    "            file.write(json_data)\n",
    "        print(json_data)\n",
    "        time.sleep(3)\n",
    "    except:\n",
    "        continue\n",
    "    \n",
    "    finally:\n",
    "        i+=1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://europepmc.org/articles/pmc5669064?pdf=render\n"
     ]
    }
   ],
   "source": [
    "# if paper_details:\n",
    "#     if paper_details['isOpenAccess']:\n",
    "#         pdf_url = paper_details['openAccessPdf']['url'] \n",
    "#         print(pdf_url)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File downloaded successfully: test.pdf\n"
     ]
    }
   ],
   "source": [
    "# download_file(pdf_url, \"test.pdf\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
